
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>Bbassiana_JEF-350 - 51 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/fungi.css">
<meta property="og:title" content="Bbassiana_JEF-350 - 8 record(s) - 51 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/fungi_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://fungismash.secondarymetabolites.org/upload/fungi-1ab29bfc-2912-45b1-9231-c9236abb2b1a/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://fungismash.secondarymetabolites.org/">
<img src="images/fungi_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://fungismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="Bbassiana_JEF-350.zip">Download all results</a></li>

<li><a href="Bbassiana_JEF-350.gbk">Download GenBank summary file</a></li>
<li><a href="Bbassiana_JEF-350.json">Download JSON results file</a></li>

<li><a href="fungi-1ab29bfc-2912-45b1-9231-c9236abb2b1a.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton hybrid r1c1">
<a href="#r1c1">1.1</a>
</div>

<div class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">1.2</a>
</div>

<div class="regbutton PKS T3PKS r1c3">
<a href="#r1c3">1.3</a>
</div>

<div class="regbutton NRPS NRPS r1c4">
<a href="#r1c4">1.4</a>
</div>

<div class="regbutton PKS T1PKS r1c5">
<a href="#r1c5">1.5</a>
</div>

<div class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">1.6</a>
</div>

<div class="regbutton NRPS NAPAA r1c7">
<a href="#r1c7">1.7</a>
</div>

<div class="regbutton NRPS NRPS r1c8">
<a href="#r1c8">1.8</a>
</div>



<div class="regbutton NRPS NRPS r2c1">
<a href="#r2c1">2.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r2c2">
<a href="#r2c2">2.2</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r2c3">
<a href="#r2c3">2.3</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r2c4">
<a href="#r2c4">2.4</a>
</div>

<div class="regbutton hybrid r2c5">
<a href="#r2c5">2.5</a>
</div>

<div class="regbutton terpene terpene r2c6">
<a href="#r2c6">2.6</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r2c7">
<a href="#r2c7">2.7</a>
</div>

<div class="regbutton terpene terpene r2c8">
<a href="#r2c8">2.8</a>
</div>



<div class="regbutton NRPS NRPS-like r3c1">
<a href="#r3c1">3.1</a>
</div>

<div class="regbutton PKS T1PKS r3c2">
<a href="#r3c2">3.2</a>
</div>

<div class="regbutton hybrid r3c3">
<a href="#r3c3">3.3</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r3c4">
<a href="#r3c4">3.4</a>
</div>

<div class="regbutton NRPS NRPS-like r3c5">
<a href="#r3c5">3.5</a>
</div>



<div class="regbutton NRPS NRPS r4c1">
<a href="#r4c1">4.1</a>
</div>

<div class="regbutton hybrid r4c2">
<a href="#r4c2">4.2</a>
</div>

<div class="regbutton PKS T1PKS r4c3">
<a href="#r4c3">4.3</a>
</div>

<div class="regbutton NRPS NRPS r4c4">
<a href="#r4c4">4.4</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r4c5">
<a href="#r4c5">4.5</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r4c6">
<a href="#r4c6">4.6</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r4c7">
<a href="#r4c7">4.7</a>
</div>



<div class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">5.1</a>
</div>

<div class="regbutton terpene terpene r5c2">
<a href="#r5c2">5.2</a>
</div>

<div class="regbutton NRPS NRPS r5c3">
<a href="#r5c3">5.3</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r5c4">
<a href="#r5c4">5.4</a>
</div>

<div class="regbutton PKS T1PKS r5c5">
<a href="#r5c5">5.5</a>
</div>

<div class="regbutton NRPS NRPS r5c6">
<a href="#r5c6">5.6</a>
</div>

<div class="regbutton NRPS NRPS-like r5c7">
<a href="#r5c7">5.7</a>
</div>

<div class="regbutton NRPS r5c8">
<a href="#r5c8">5.8</a>
</div>

<div class="regbutton other isocyanide r5c9">
<a href="#r5c9">5.9</a>
</div>

<div class="regbutton NRPS NRPS-like r5c10">
<a href="#r5c10">5.10</a>
</div>



<div class="regbutton NRPS isocyanide-nrp r6c1">
<a href="#r6c1">6.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r6c2">
<a href="#r6c2">6.2</a>
</div>

<div class="regbutton hybrid r6c3">
<a href="#r6c3">6.3</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r6c4">
<a href="#r6c4">6.4</a>
</div>

<div class="regbutton NRPS NRPS r6c5">
<a href="#r6c5">6.5</a>
</div>

<div class="regbutton PKS T1PKS r6c6">
<a href="#r6c6">6.6</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r6c7">
<a href="#r6c7">6.7</a>
</div>

<div class="regbutton terpene terpene r6c8">
<a href="#r6c8">6.8</a>
</div>

<div class="regbutton PKS T1PKS r6c9">
<a href="#r6c9">6.9</a>
</div>



<div class="regbutton terpene terpene r7c1">
<a href="#r7c1">7.1</a>
</div>

<div class="regbutton NRPS NRPS r7c2">
<a href="#r7c2">7.2</a>
</div>

<div class="regbutton hybrid r7c3">
<a href="#r7c3">7.3</a>
</div>



<div class="regbutton NRPS NRPS r8c1">
<a href="#r8c1">8.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>JAAZTN010000001.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="JAAZTN010000001.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton hybrid r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">48,045</td>
<td class="digits table-split-left">143,152</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">530,314</td>
<td class="digits table-split-left">578,242</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton PKS T3PKS r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">2,238,853</td>
<td class="digits table-split-left">2,279,775</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c4">
<td class="regbutton NRPS NRPS r1c4">
<a href="#r1c4">Region&nbsp1.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,395,725</td>
<td class="digits table-split-left">2,443,882</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c5">
<td class="regbutton PKS T1PKS r1c5">
<a href="#r1c5">Region&nbsp1.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,474,021</td>
<td class="digits table-split-left">2,517,924</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c6">
<td class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">Region&nbsp1.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,038,256</td>
<td class="digits table-split-left">5,088,211</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c7">
<td class="regbutton NRPS NAPAA r1c7">
<a href="#r1c7">Region&nbsp1.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">5,118,244</td>
<td class="digits table-split-left">5,152,499</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">ε-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c8">
<td class="regbutton NRPS NRPS r1c8">
<a href="#r1c8">Region&nbsp1.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,701,426</td>
<td class="digits table-split-left">5,746,814</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JAAZTN010000002.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="JAAZTN010000002.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton NRPS NRPS r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">368,177</td>
<td class="digits table-split-left">418,344</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000313/1" target="_blank">beauvericin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton RiPP fungal-RiPP-like r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">451,888</td>
<td class="digits table-split-left">512,608</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton RiPP fungal-RiPP-like r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">617,599</td>
<td class="digits table-split-left">678,443</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c4">
<td class="regbutton RiPP fungal-RiPP-like r2c4">
<a href="#r2c4">Region&nbsp2.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">988,357</td>
<td class="digits table-split-left">1,048,573</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c5">
<td class="regbutton hybrid r2c5">
<a href="#r2c5">Region&nbsp2.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,916,227</td>
<td class="digits table-split-left">4,983,225</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c6">
<td class="regbutton terpene terpene r2c6">
<a href="#r2c6">Region&nbsp2.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">5,342,184</td>
<td class="digits table-split-left">5,363,737</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c7">
<td class="regbutton RiPP fungal-RiPP-like r2c7">
<a href="#r2c7">Region&nbsp2.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">6,156,507</td>
<td class="digits table-split-left">6,221,691</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001278/1" target="_blank">nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c8">
<td class="regbutton terpene terpene r2c8">
<a href="#r2c8">Region&nbsp2.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">6,230,485</td>
<td class="digits table-split-left">6,253,898</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001811/1" target="_blank">trichodiene-11-one</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JAAZTN010000003.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="JAAZTN010000003.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton NRPS NRPS-like r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1,356,085</td>
<td class="digits table-split-left">1,399,625</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton PKS T1PKS r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,733,944</td>
<td class="digits table-split-left">2,781,143</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton hybrid r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,904,574</td>
<td class="digits table-split-left">2,966,490</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton RiPP fungal-RiPP-like r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,435,132</td>
<td class="digits table-split-left">4,501,526</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton NRPS NRPS-like r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">4,949,632</td>
<td class="digits table-split-left">4,993,501</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JAAZTN010000004.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="JAAZTN010000004.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton NRPS NRPS r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">327,623</td>
<td class="digits table-split-left">372,983</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton hybrid r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,257,280</td>
<td class="digits table-split-left">2,326,779</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002203/1" target="_blank">beauveriolide B/beauveriolide C/beauveriolide D</a></td>
<td>NRP+Polyketide+Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c3">
<td class="regbutton PKS T1PKS r4c3">
<a href="#r4c3">Region&nbsp4.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,400,244</td>
<td class="digits table-split-left">2,447,344</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001720/1" target="_blank">oosporein</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c4">
<td class="regbutton NRPS NRPS r4c4">
<a href="#r4c4">Region&nbsp4.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,746,561</td>
<td class="digits table-split-left">2,790,231</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c5">
<td class="regbutton RiPP fungal-RiPP-like r4c5">
<a href="#r4c5">Region&nbsp4.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,734,693</td>
<td class="digits table-split-left">3,796,740</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c6">
<td class="regbutton RiPP fungal-RiPP-like r4c6">
<a href="#r4c6">Region&nbsp4.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,245,240</td>
<td class="digits table-split-left">4,305,988</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c7">
<td class="regbutton RiPP fungal-RiPP-like r4c7">
<a href="#r4c7">Region&nbsp4.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,427,358</td>
<td class="digits table-split-left">4,489,081</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JAAZTN010000005.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="JAAZTN010000005.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">317,504</td>
<td class="digits table-split-left">364,300</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002175/1" target="_blank">YWA1</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton terpene terpene r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">531,129</td>
<td class="digits table-split-left">554,392</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001248/1" target="_blank">clavaric acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton NRPS NRPS r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,316,394</td>
<td class="digits table-split-left">1,365,962</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000312/1" target="_blank">bassianolide</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>

</tr>
<tr class="linked-row even" data-anchor="#r5c4">
<td class="regbutton RiPP fungal-RiPP-like r5c4">
<a href="#r5c4">Region&nbsp5.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,690,230</td>
<td class="digits table-split-left">1,750,801</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c5">
<td class="regbutton PKS T1PKS r5c5">
<a href="#r5c5">Region&nbsp5.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,738,257</td>
<td class="digits table-split-left">2,785,904</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c6">
<td class="regbutton NRPS NRPS r5c6">
<a href="#r5c6">Region&nbsp5.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,042,285</td>
<td class="digits table-split-left">3,098,625</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c7">
<td class="regbutton NRPS NRPS-like r5c7">
<a href="#r5c7">Region&nbsp5.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,567,082</td>
<td class="digits table-split-left">3,610,671</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c8">
<td class="regbutton NRPS r5c8">
<a href="#r5c8">Region&nbsp5.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,257,421</td>
<td class="digits table-split-left">4,317,347</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c9">
<td class="regbutton other isocyanide r5c9">
<a href="#r5c9">Region&nbsp5.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide" target="_blank">isocyanide</a>
</td>
<td class="digits">4,348,960</td>
<td class="digits table-split-left">4,391,085</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c10">
<td class="regbutton NRPS NRPS-like r5c10">
<a href="#r5c10">Region&nbsp5.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">4,586,794</td>
<td class="digits table-split-left">4,630,328</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JAAZTN010000006.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="JAAZTN010000006.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton NRPS isocyanide-nrp r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide-nrp" target="_blank">isocyanide-nrp</a>
</td>
<td class="digits">221,188</td>
<td class="digits table-split-left">276,076</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton RiPP fungal-RiPP-like r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">276,865</td>
<td class="digits table-split-left">337,882</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c3">
<td class="regbutton hybrid r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">373,965</td>
<td class="digits table-split-left">426,949</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001049/1" target="_blank">tenellin</a></td>
<td>NRP+Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c4">
<td class="regbutton RiPP fungal-RiPP-like r6c4">
<a href="#r6c4">Region&nbsp6.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">594,524</td>
<td class="digits table-split-left">657,071</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c5">
<td class="regbutton NRPS NRPS r6c5">
<a href="#r6c5">Region&nbsp6.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">839,905</td>
<td class="digits table-split-left">884,155</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c6">
<td class="regbutton PKS T1PKS r6c6">
<a href="#r6c6">Region&nbsp6.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">3,097,978</td>
<td class="digits table-split-left">3,145,368</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c7">
<td class="regbutton RiPP fungal-RiPP-like r6c7">
<a href="#r6c7">Region&nbsp6.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,570,863</td>
<td class="digits table-split-left">3,631,695</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c8">
<td class="regbutton terpene terpene r6c8">
<a href="#r6c8">Region&nbsp6.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">3,813,802</td>
<td class="digits table-split-left">3,836,575</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c9">
<td class="regbutton PKS T1PKS r6c9">
<a href="#r6c9">Region&nbsp6.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">3,976,721</td>
<td class="digits table-split-left">4,007,226</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JAAZTN010000007.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-7" data-filename="JAAZTN010000007.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-7" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r7c1">
<td class="regbutton terpene terpene r7c1">
<a href="#r7c1">Region&nbsp7.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">44,679</td>
<td class="digits table-split-left">65,809</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c2">
<td class="regbutton NRPS NRPS r7c2">
<a href="#r7c2">Region&nbsp7.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,458,226</td>
<td class="digits table-split-left">1,503,935</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002710/1" target="_blank">metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c3">
<td class="regbutton hybrid r7c3">
<a href="#r7c3">Region&nbsp7.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,369,824</td>
<td class="digits table-split-left">2,455,224</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001141/1" target="_blank">4-epi-15-epi-brefeldin A</a></td>
<td>Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JAAZTN010000008.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-8" data-filename="JAAZTN010000008.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-8" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r8c1">
<td class="regbutton NRPS NRPS r8c1">
<a href="#r8c1">Region&nbsp8.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">592,927</td>
<td class="digits table-split-left">637,723</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton hybrid r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">48,045</td>
<td class="digits table-split-left">143,152</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">530,314</td>
<td class="digits table-split-left">578,242</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton PKS T3PKS r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">2,238,853</td>
<td class="digits table-split-left">2,279,775</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c4">
<td class="regbutton NRPS NRPS r1c4">
<a href="#r1c4">Region&nbsp1.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,395,725</td>
<td class="digits table-split-left">2,443,882</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c5">
<td class="regbutton PKS T1PKS r1c5">
<a href="#r1c5">Region&nbsp1.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,474,021</td>
<td class="digits table-split-left">2,517,924</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c6">
<td class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">Region&nbsp1.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,038,256</td>
<td class="digits table-split-left">5,088,211</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c7">
<td class="regbutton NRPS NAPAA r1c7">
<a href="#r1c7">Region&nbsp1.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">5,118,244</td>
<td class="digits table-split-left">5,152,499</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">ε-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c8">
<td class="regbutton NRPS NRPS r1c8">
<a href="#r1c8">Region&nbsp1.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,701,426</td>
<td class="digits table-split-left">5,746,814</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton NRPS NRPS r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">368,177</td>
<td class="digits table-split-left">418,344</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000313/1" target="_blank">beauvericin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton RiPP fungal-RiPP-like r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">451,888</td>
<td class="digits table-split-left">512,608</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton RiPP fungal-RiPP-like r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">617,599</td>
<td class="digits table-split-left">678,443</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c4">
<td class="regbutton RiPP fungal-RiPP-like r2c4">
<a href="#r2c4">Region&nbsp2.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">988,357</td>
<td class="digits table-split-left">1,048,573</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c5">
<td class="regbutton hybrid r2c5">
<a href="#r2c5">Region&nbsp2.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,916,227</td>
<td class="digits table-split-left">4,983,225</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c6">
<td class="regbutton terpene terpene r2c6">
<a href="#r2c6">Region&nbsp2.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">5,342,184</td>
<td class="digits table-split-left">5,363,737</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c7">
<td class="regbutton RiPP fungal-RiPP-like r2c7">
<a href="#r2c7">Region&nbsp2.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">6,156,507</td>
<td class="digits table-split-left">6,221,691</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001278/1" target="_blank">nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c8">
<td class="regbutton terpene terpene r2c8">
<a href="#r2c8">Region&nbsp2.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">6,230,485</td>
<td class="digits table-split-left">6,253,898</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001811/1" target="_blank">trichodiene-11-one</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton NRPS NRPS-like r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1,356,085</td>
<td class="digits table-split-left">1,399,625</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton PKS T1PKS r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,733,944</td>
<td class="digits table-split-left">2,781,143</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton hybrid r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,904,574</td>
<td class="digits table-split-left">2,966,490</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton RiPP fungal-RiPP-like r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,435,132</td>
<td class="digits table-split-left">4,501,526</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton NRPS NRPS-like r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">4,949,632</td>
<td class="digits table-split-left">4,993,501</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row even" data-anchor="#r4c1">
<td class="regbutton NRPS NRPS r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">327,623</td>
<td class="digits table-split-left">372,983</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c2">
<td class="regbutton hybrid r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,257,280</td>
<td class="digits table-split-left">2,326,779</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002203/1" target="_blank">beauveriolide B/beauveriolide C/beauveriolide D</a></td>
<td>NRP+Polyketide+Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c3">
<td class="regbutton PKS T1PKS r4c3">
<a href="#r4c3">Region&nbsp4.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,400,244</td>
<td class="digits table-split-left">2,447,344</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001720/1" target="_blank">oosporein</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c4">
<td class="regbutton NRPS NRPS r4c4">
<a href="#r4c4">Region&nbsp4.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,746,561</td>
<td class="digits table-split-left">2,790,231</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c5">
<td class="regbutton RiPP fungal-RiPP-like r4c5">
<a href="#r4c5">Region&nbsp4.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,734,693</td>
<td class="digits table-split-left">3,796,740</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c6">
<td class="regbutton RiPP fungal-RiPP-like r4c6">
<a href="#r4c6">Region&nbsp4.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,245,240</td>
<td class="digits table-split-left">4,305,988</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c7">
<td class="regbutton RiPP fungal-RiPP-like r4c7">
<a href="#r4c7">Region&nbsp4.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,427,358</td>
<td class="digits table-split-left">4,489,081</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">317,504</td>
<td class="digits table-split-left">364,300</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002175/1" target="_blank">YWA1</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton terpene terpene r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">531,129</td>
<td class="digits table-split-left">554,392</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001248/1" target="_blank">clavaric acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton NRPS NRPS r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,316,394</td>
<td class="digits table-split-left">1,365,962</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000312/1" target="_blank">bassianolide</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>

</tr>
<tr class="linked-row even" data-anchor="#r5c4">
<td class="regbutton RiPP fungal-RiPP-like r5c4">
<a href="#r5c4">Region&nbsp5.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,690,230</td>
<td class="digits table-split-left">1,750,801</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c5">
<td class="regbutton PKS T1PKS r5c5">
<a href="#r5c5">Region&nbsp5.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,738,257</td>
<td class="digits table-split-left">2,785,904</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c6">
<td class="regbutton NRPS NRPS r5c6">
<a href="#r5c6">Region&nbsp5.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,042,285</td>
<td class="digits table-split-left">3,098,625</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c7">
<td class="regbutton NRPS NRPS-like r5c7">
<a href="#r5c7">Region&nbsp5.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,567,082</td>
<td class="digits table-split-left">3,610,671</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c8">
<td class="regbutton NRPS r5c8">
<a href="#r5c8">Region&nbsp5.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,257,421</td>
<td class="digits table-split-left">4,317,347</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c9">
<td class="regbutton other isocyanide r5c9">
<a href="#r5c9">Region&nbsp5.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide" target="_blank">isocyanide</a>
</td>
<td class="digits">4,348,960</td>
<td class="digits table-split-left">4,391,085</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c10">
<td class="regbutton NRPS NRPS-like r5c10">
<a href="#r5c10">Region&nbsp5.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">4,586,794</td>
<td class="digits table-split-left">4,630,328</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton NRPS isocyanide-nrp r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide-nrp" target="_blank">isocyanide-nrp</a>
</td>
<td class="digits">221,188</td>
<td class="digits table-split-left">276,076</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton RiPP fungal-RiPP-like r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">276,865</td>
<td class="digits table-split-left">337,882</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c3">
<td class="regbutton hybrid r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">373,965</td>
<td class="digits table-split-left">426,949</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001049/1" target="_blank">tenellin</a></td>
<td>NRP+Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c4">
<td class="regbutton RiPP fungal-RiPP-like r6c4">
<a href="#r6c4">Region&nbsp6.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">594,524</td>
<td class="digits table-split-left">657,071</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c5">
<td class="regbutton NRPS NRPS r6c5">
<a href="#r6c5">Region&nbsp6.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">839,905</td>
<td class="digits table-split-left">884,155</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c6">
<td class="regbutton PKS T1PKS r6c6">
<a href="#r6c6">Region&nbsp6.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">3,097,978</td>
<td class="digits table-split-left">3,145,368</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c7">
<td class="regbutton RiPP fungal-RiPP-like r6c7">
<a href="#r6c7">Region&nbsp6.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,570,863</td>
<td class="digits table-split-left">3,631,695</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c8">
<td class="regbutton terpene terpene r6c8">
<a href="#r6c8">Region&nbsp6.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">3,813,802</td>
<td class="digits table-split-left">3,836,575</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c9">
<td class="regbutton PKS T1PKS r6c9">
<a href="#r6c9">Region&nbsp6.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">3,976,721</td>
<td class="digits table-split-left">4,007,226</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r7c1">
<td class="regbutton terpene terpene r7c1">
<a href="#r7c1">Region&nbsp7.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">44,679</td>
<td class="digits table-split-left">65,809</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c2">
<td class="regbutton NRPS NRPS r7c2">
<a href="#r7c2">Region&nbsp7.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,458,226</td>
<td class="digits table-split-left">1,503,935</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002710/1" target="_blank">metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>

</tr>
<tr class="linked-row even" data-anchor="#r7c3">
<td class="regbutton hybrid r7c3">
<a href="#r7c3">Region&nbsp7.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,369,824</td>
<td class="digits table-split-left">2,455,224</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001141/1" target="_blank">4-epi-15-epi-brefeldin A</a></td>
<td>Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r8c1">
<td class="regbutton NRPS NRPS r8c1">
<a href="#r8c1">Region&nbsp8.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">592,927</td>
<td class="digits table-split-left">637,723</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000001.1 - Region 1 - NRPS,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-455"></div> <span class="help-tooltip" id="region_svg-help-455">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000001.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="JAAZTN010000001.1_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 48,045 - 143,152 nt. (total: 95,108 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c1-rules">fungal-RiPP-like: DUF3328<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-456"></div> <span class="help-tooltip" id="focus-panel-help-456">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-nrps_pks" data-name="r1c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r1c1-nrps-pks-bubbles" data-name="r1c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c1-tigrfam-details" data-name="r1c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-9"></div> <span class="help-tooltip" id="gt-help-tooltip-help-9">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-details-svg" data-filename="JAAZTN010000001.1_r1c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407: 0-10802">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.34</td>
<td>RiPP</td>

<td>bicereucin</td>


<td>Bacillus cereus SJ1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.34</td>
<td>RiPP</td>

<td>cytolysin ClyLl, cytolysin ClyLs</td>


<td>Plasmid pAD1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002701: 0-9977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002701/index.html#r1c1" target="_new">BGC0002701</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.33</td>
<td>RiPP</td>

<td>paenithipeptin</td>


<td>Paenibacillus thiaminolyticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.32</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.32</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002382: 465-15791">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002382/index.html#r1c1" target="_new">BGC0002382</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.32</td>
<td>RiPP</td>

<td>grimoviridin</td>


<td>Grimontia marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.32</td>
<td>RiPP</td>

<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>


<td>Actinomadura namibiensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002491: 0-17030">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002491/index.html#r1c1" target="_new">BGC0002491</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>myxochelin B, myxochelin A, pseudochelin A</td>
<td>Pseudoalteromonas piscicida</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717: 0-19662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>okaramine B</td>
<td>Penicillium simplicissimum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>curacomycin</td>
<td>Streptomyces curacoi</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c1_1.svg">
NW_006271977 (184210-275183): Cordyceps militaris CM01 unplaced genomic scaff... (55% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_2.svg">
NW_025408660 (68205-155722): Fusarium poae strain DAOMC 252244 chromosome Unk... (21% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_3.svg">
NW_019154045 (358401-688746): Pochonia chlamydosporia 170 chromosome Unknown ... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_4.svg">
NC_054856 (3727884-3894446): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (6% of genes show similarity), NI-siderophore,NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_5.svg">
NW_011942151 (20096-254382): Metarhizium robertsii ARSEF 23 MAA Scf 11, whole... (4% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r1c1_6.svg">
NW_020290899 (240548-317044): Aspergillus niger CBS 101883 unplaced genomic s... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_7.svg">
NW_014574696 (225420-389931): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (7% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_8.svg">
NW_011942171 (314568-405746): Metarhizium robertsii ARSEF 23 MAA Scf 3, whole... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_9.svg">
NW_024467526 (7938-201848): Aspergillus versicolor CBS 583.65 unplaced genomi... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c1_10.svg">
NW_020291498 (165271-359753): Aspergillus aculeatinus CBS 121060 unplaced gen... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

</select>
<button id= "clusterblast-r1c1-download">Download graphic</button>
</div>
<div id="clusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r1c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-8"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-8">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-domain-bubble-svg" data-filename="JAAZTN010000001.1_r1c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (48044 - 143152): neighbouring fungal-RiPP-like-NRPS
</option>



<option value="CC3">
Candidate 3 (93596 - 143152): single NRPS
</option>


</select>

<div class="details-svg" id="r1c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-10"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-10">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="JAAZTN010000001.1_r1c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-11"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-11">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-tigrfam-details-svg" data-filename="JAAZTN010000001.1_r1c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c1-nrps_pks_products" data-name="r1c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c1-nrps_pks_monomers" data-name="r1c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r1c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 48044 - 143152:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Ala)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,ala*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 93596 - 143152:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Ala)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,ala*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene29</span></strong>: X - Ala
<div class="collapser collapser-target-input-path1-gene29 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (105..502)</strong>: X
<div class="collapser collapser-target-input-path1-gene29 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene29-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAPFLGMAYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DAMFLGCTYK</span> (53% 8Å match)</dd>
<dd>Leu or Ile <span class="serif">DAFFLGMTFK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1209..1608)</strong>: Ala
<div class="collapser collapser-target-input-path1-gene29 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-input-path1-gene29-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DTTDCSLAVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>

<div class="page" id='r1c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000001.1 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-457"></div> <span class="help-tooltip" id="region_svg-help-457">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000001.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c2-svg" data-filename="JAAZTN010000001.1_r1c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 530,314 - 578,242 nt. (total: 47,929 nt)

<a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r1c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-458"></div> <span class="help-tooltip" id="focus-panel-help-458">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c2-gene-table-details" data-name="r1c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c2-nrps_pks" data-name="r1c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c2-nrps-pks-bubbles" data-name="r1c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-20"></div> <span class="help-tooltip" id="gt-help-tooltip-help-20">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-12"></div> <span class="help-tooltip" id="nrps-domain-help-12">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-details-svg" data-filename="JAAZTN010000001.1_r1c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-13"></div> <span class="help-tooltip" id="MIBiG-help-13">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c2"></div>
<div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene, Polyketide</td>

<td>calidoustene A, calidoustene B, calidoustene C</td>


<td>Aspergillus calidoustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>phomoidride</td>


<td>fungal sp. ATCC 74256</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.46, order:0.51, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>flavoglaucin</td>


<td>Aspergillus ruber CBS 135680</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>trichoxide</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>phomoidride</td>
<td>fungal sp. ATCC 74256</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.46, order:0.51, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>flavoglaucin</td>
<td>Aspergillus ruber CBS 135680</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-14"></div> <span class="help-tooltip" id="cb-general-help-14">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c2_1.svg">
NW_006271974 (2321315-2405136): Cordyceps militaris CM01 unplaced genomic sca... (55% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c2_2.svg">
NW_026249875 (10804-78109): Alternaria incomplexa strain BMP 0042 chromosome ... (38% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_3.svg">
NC_049565 (2694595-2813931): Talaromyces rugulosus chromosome V, complete seq... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_4.svg">
NW_026249941 (3336813-3404118): Alternaria arbusti strain BMP 1465 chromosome... (33% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_5.svg">
NW_026055699 (512819-580124): Alternaria ventricosa strain BMP 2768 chromosom... (32% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_6.svg">
NW_025408687 (532325-600393): Hirsutella rhossiliensis strain HR02 chromosome... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_7.svg">
NW_017253364 (509203-576723): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_8.svg">
NW_025544748 (424962-492810): Colletotrichum gloeosporioides strain Lc1 chrom... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_9.svg">
NW_025953791 (566181-633989): Annulohypoxylon truncatum strain CBS 140777 unp... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_10.svg">
NC_054862 (2647617-2714965): Aspergillus puulaauensis MK2 DNA, chromosome 6, ... (18% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r1c2-download">Download graphic</button>
</div>
<div id="clusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-15"></div> <span class="help-tooltip" id="cb-known-help-15">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-16"></div> <span class="help-tooltip" id="cb-sub-help-16">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-19"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-19">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-domain-bubble-svg" data-filename="JAAZTN010000001.1_r1c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (530313 - 578242): single T1PKS
</option>


</select>

<div class="details-svg" id="r1c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-21"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-21">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="JAAZTN010000001.1_r1c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c2-nrps_pks_products" data-name="r1c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c2-nrps_pks_monomers" data-name="r1c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-17"></div> <span class="help-tooltip" id="nrps-structures-help-17">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 530313 - 578242:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c2-4" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c2-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-18"></div> <span class="help-tooltip" id="nrps-monomers-help-18">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene149</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene149 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (514..825)</strong>: pk
<div class="collapser collapser-target-input-path1-gene149 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene149-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Methoxymalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene149-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 72.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 64.9</dt>

<dd></dd><dt>Malonyl-CoA: 54.5</dt>

<dd></dd><dt>Propionyl-CoA: 38.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 34.8</dt>

<dd></dd><dt>fatty_acid: 30.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 27.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 23.0</dt>

<dd></dd><dt>inactive: 18.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.2</dt>

<dd></dd><dt>CHC-CoA: 15.0</dt>

<dd></dd><dt>Benzoyl-CoA: 14.1</dt>

<dd></dd><dt>Acetyl-CoA: -0.9</dt>

<dd></dd><dt>Ethylmalonyl-CoA: -5.2</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000001.1 - Region 3 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-459"></div> <span class="help-tooltip" id="region_svg-help-459">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000001.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c3-svg" data-filename="JAAZTN010000001.1_r1c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,238,853 - 2,279,775 nt. (total: 40,923 nt)

<a class="cluster-rules-header" id="r1c3-rules-header" href="#r1c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c3-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>

<div class="region-svg-container">
<div id='r1c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-460"></div> <span class="help-tooltip" id="focus-panel-help-460">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c3-gene-table-details" data-name="r1c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c3-nrps_pks" data-name="r1c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c3-MIBiG-cluster-compare" data-name="r1c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c3-clusterblast" data-name="r1c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c3-knownclusterblast" data-name="r1c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c3-subclusterblast" data-name="r1c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c3-pfam-details" data-name="r1c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c3-tigrfam-details" data-name="r1c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-28"></div> <span class="help-tooltip" id="gt-help-tooltip-help-28">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-22"></div> <span class="help-tooltip" id="nrps-domain-help-22">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-details-svg" data-filename="JAAZTN010000001.1_r1c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-23"></div> <span class="help-tooltip" id="MIBiG-help-23">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c3"></div>
<div id="comparison-MIBiG-r1c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T3PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>β-Diketones</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>lupeol</td>


<td>Lotus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002139: 487-22086">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002139/index.html#r1c1" target="_new">BGC0002139</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>burnettiene A, preburnettiene B, preburnettiene A</td>


<td>Aspergillus burnettii FRR 5400</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041: 0-14402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:1.00, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>oronofacic acid</td>


<td>Pseudomonas syringae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>β-Diketones</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>lupeol</td>
<td>Lotus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002139: 487-22086">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002139/index.html#r1c1" target="_new">BGC0002139</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>burnettiene A, preburnettiene B, preburnettiene A</td>
<td>Aspergillus burnettii FRR 5400</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041: 0-14402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:1.00, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>oronofacic acid</td>
<td>Pseudomonas syringae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-24"></div> <span class="help-tooltip" id="cb-general-help-24">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c3_1.svg">
NC_036439 (828576-889901): Aspergillus oryzae RIB40 DNA, chromosome 5 (11% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r1c3_2.svg">
NW_006917094 (2678871-2742279): Pestalotiopsis fici W106-1 unplaced genomic s... (19% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c3_3.svg">
NW_006763053 (142092-206720): Drepanopeziza brunnea f. sp. &#39;multigermtubi&#39; MB... (20% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r1c3_4.svg">
NC_049565 (503429-564663): Talaromyces rugulosus chromosome V, complete sequence (7% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r1c3_5.svg">
NW_026250021 (502671-612769): Hypoxylon trugodes strain CBS 135444 unplaced g... (6% of genes show similarity), T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r1c3_6.svg">
NW_025091270 (735960-804290): Aspergillus udagawae IFM 46973 DNA, scaffold 7,... (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c3_7.svg">
NC_054852 (254803-319299): Aspergillus luchuensis IFO 4308 DNA, chromosome 4,... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c3_8.svg">
NC_022785 (11126401-11244528): Streptomyces rapamycinicus NRRL 5491, complete... (18% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c3_9.svg">
NZ_CP030771 (7915496-7990387): Streptomyces sp. YIM 121038 chromosome, comple... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c3_10.svg">
NZ_CP023690 (7957332-8031858): Streptomyces spectabilis strain ATCC 27465 chr... (9% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r1c3-download">Download graphic</button>
</div>
<div id="clusterblast-r1c3-svg"></div>

</div>
</div>

<div class="body-details-section r1c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-25"></div> <span class="help-tooltip" id="cb-known-help-25">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-26"></div> <span class="help-tooltip" id="cb-sub-help-26">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-29"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-29">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-pfam-details-svg" data-filename="JAAZTN010000001.1_r1c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-30"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-30">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-tigrfam-details-svg" data-filename="JAAZTN010000001.1_r1c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c3-nrps_pks_monomers" data-name="r1c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r1c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-27"></div> <span class="help-tooltip" id="nrps-monomers-help-27">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r1c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000001.1 - Region 4 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-461"></div> <span class="help-tooltip" id="region_svg-help-461">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000001.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c4-svg" data-filename="JAAZTN010000001.1_r1c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,395,725 - 2,443,882 nt. (total: 48,158 nt)

<a class="cluster-rules-header" id="r1c4-rules-header" href="#r1c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-462"></div> <span class="help-tooltip" id="focus-panel-help-462">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c4-gene-table-details" data-name="r1c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c4-nrps_pks" data-name="r1c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c4-MIBiG-cluster-compare" data-name="r1c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c4-clusterblast" data-name="r1c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c4-knownclusterblast" data-name="r1c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c4-subclusterblast" data-name="r1c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c4-nrps-pks-bubbles" data-name="r1c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c4-pfam-details" data-name="r1c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-39"></div> <span class="help-tooltip" id="gt-help-tooltip-help-39">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-31"></div> <span class="help-tooltip" id="nrps-domain-help-31">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-details-svg" data-filename="JAAZTN010000001.1_r1c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-32"></div> <span class="help-tooltip" id="MIBiG-help-32">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c4"></div>
<div id="comparison-MIBiG-r1c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002248: 5-21557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002248/index.html#r1c1" target="_new">BGC0002248</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene, NRP</td>

<td>flavunoidine</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>serinocyclin A, serinocyclin B</td>


<td>Metarhizium robertsii</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002248: 5-21557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002248/index.html#r1c1" target="_new">BGC0002248</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene, NRP</td>
<td>flavunoidine</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>serinocyclin A, serinocyclin B</td>
<td>Metarhizium robertsii</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-33"></div> <span class="help-tooltip" id="cb-general-help-33">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c4_1.svg">
NW_020291486 (59597-203320): Aspergillus aculeatinus CBS 121060 unplaced geno... (3% of genes show similarity), NRPS,fungal-RiPP-like,isocyanide
</option>

<option value = "svg/clusterblast_r1c4_2.svg">
NC_057368 (37469-194929): Aspergillus chevalieri M1 DNA, chromosome 7, nearly... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c4_3.svg">
NW_019170479 (217386-317627): Aspergillus glaucus CBS 516.65 unplaced genomic... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c4_4.svg">
NZ_JACSQL010000010 (55695-96850): Paenibacillus gallinarum strain Sa2BVA9 NOD... (5% of genes show similarity), ranthipeptide,resorcinol
</option>

<option value = "svg/clusterblast_r1c4_5.svg">
NW_006271969 (1967760-2079673): Cordyceps militaris CM01 unplaced genomic sca... (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_6.svg">
NW_025408683 (1085865-1154571): Hirsutella rhossiliensis strain HR02 chromoso... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c4_7.svg">
NZ_KZ845664 (239911-328977): Thermoflavimicrobium daqui strain FBKL4.011 scaf... (6% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c4_8.svg">
NC_013158 (3006292-3049136): Halorhabdus utahensis DSM 12940, complete sequence (5% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r1c4_9.svg">
NZ_CP104322 (1641425-1680797): Halorhabdus sp. SVX81 chromosome, complete genome (5% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r1c4_10.svg">
NC_021921 (2541272-2592365): Halorhabdus tiamatea SARL4B chromosome I, comple... (4% of genes show similarity), thiopeptide
</option>

</select>
<button id= "clusterblast-r1c4-download">Download graphic</button>
</div>
<div id="clusterblast-r1c4-svg"></div>

</div>
</div>

<div class="body-details-section r1c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-34"></div> <span class="help-tooltip" id="cb-known-help-34">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-35"></div> <span class="help-tooltip" id="cb-sub-help-35">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-38"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-38">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-domain-bubble-svg" data-filename="JAAZTN010000001.1_r1c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (2395724 - 2443882): single NRPS
</option>


</select>

<div class="details-svg" id="r1c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-40"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-40">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-pfam-details-svg" data-filename="JAAZTN010000001.1_r1c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c4-nrps_pks_products" data-name="r1c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c4-nrps_pks_monomers" data-name="r1c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-36"></div> <span class="help-tooltip" id="nrps-structures-help-36">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 2395724 - 2443882:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c4-6" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c4-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-37"></div> <span class="help-tooltip" id="nrps-monomers-help-37">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene633</span></strong>: X
<div class="collapser collapser-target-input-path1-gene633 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (5..325)</strong>: X
<div class="collapser collapser-target-input-path1-gene633 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene633-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">VVTAVVLKGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000001.1 - Region 5 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-463"></div> <span class="help-tooltip" id="region_svg-help-463">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000001.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c5-svg" data-filename="JAAZTN010000001.1_r1c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,474,021 - 2,517,924 nt. (total: 43,904 nt)

<a class="cluster-rules-header" id="r1c5-rules-header" href="#r1c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c5-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r1c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-464"></div> <span class="help-tooltip" id="focus-panel-help-464">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c5-gene-table-details" data-name="r1c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c5-nrps_pks" data-name="r1c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c5-MIBiG-cluster-compare" data-name="r1c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c5-clusterblast" data-name="r1c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c5-knownclusterblast" data-name="r1c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c5-subclusterblast" data-name="r1c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c5-pfam-details" data-name="r1c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c5-tigrfam-details" data-name="r1c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-47"></div> <span class="help-tooltip" id="gt-help-tooltip-help-47">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-41"></div> <span class="help-tooltip" id="nrps-domain-help-41">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-details-svg" data-filename="JAAZTN010000001.1_r1c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-42"></div> <span class="help-tooltip" id="MIBiG-help-42">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c5"></div>
<div id="comparison-MIBiG-r1c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.50, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002036: 0-15983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002036/index.html#r1c1" target="_new">BGC0002036</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>dehydrofosmidomycin</td>


<td>Streptomyces lavendulae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.50, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002036: 0-15983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002036/index.html#r1c1" target="_new">BGC0002036</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>dehydrofosmidomycin</td>
<td>Streptomyces lavendulae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-43"></div> <span class="help-tooltip" id="cb-general-help-43">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c5_1.svg">
NZ_CP012672 (5491586-5600451): Sorangium cellulosum strain So ce836 chromosom... (21% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_2.svg">
NZ_SNXZ01000007 (57627-303701): Labedaea rhizosphaerae strain DSM 45361 Ga031... (11% of genes show similarity), LAP,NRPS,T1PKS,oligosaccharide
</option>

<option value = "svg/clusterblast_r1c5_3.svg">
NZ_CP046600 (2510203-2568468): Mycobacterium spongiae strain FSD4b-SM chromos... (8% of genes show similarity), T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r1c5_4.svg">
NZ_JAMTCR010000002 (2282409-2345201): Actinokineospora globicatena strain DSM... (9% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_5.svg">
NZ_RCDD01000001 (2060439-2123171): Actinokineospora cianjurensis strain DSM 4... (9% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_6.svg">
NZ_JAIMIC010000009 (33782-124218): Heliomarina baculiformis strain T40-3 T40-... (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_7.svg">
NZ_CP066801 (69486-238975): Streptomyces sp. HSG2 chromosome, complete genome (11% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_8.svg">
NZ_CP060828 (3814826-3920326): Streptomyces roseirectus strain CRXT-G-22 chro... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c5_9.svg">
NZ_CP011509 (8416504-8476008): Archangium gephyra strain DSM 2261 chromosome,... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c5_10.svg">
NZ_CP006850 (7237936-7295913): Nocardia nova SH22a chromosome, complete genome (6% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r1c5-download">Download graphic</button>
</div>
<div id="clusterblast-r1c5-svg"></div>

</div>
</div>

<div class="body-details-section r1c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-44"></div> <span class="help-tooltip" id="cb-known-help-44">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-45"></div> <span class="help-tooltip" id="cb-sub-help-45">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-48"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-48">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-pfam-details-svg" data-filename="JAAZTN010000001.1_r1c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-49"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-49">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-tigrfam-details-svg" data-filename="JAAZTN010000001.1_r1c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c5-nrps_pks_monomers" data-name="r1c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r1c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-46"></div> <span class="help-tooltip" id="nrps-monomers-help-46">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene666</span></strong>: mxmal
<div class="collapser collapser-target-input-path1-gene666 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (265..487)</strong>: mxmal
<div class="collapser collapser-target-input-path1-gene666 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-input-path1-gene666-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-input-path1-gene666-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 23.9</dt>

<dd></dd><dt>Methylmalonyl-CoA: 20.8</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 16.1</dt>

<dd></dd><dt>fatty_acid: 13.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 7.6</dt>

<dd></dd><dt>Malonyl-CoA: 7.1</dt>

<dd></dd><dt>Propionyl-CoA: 3.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r1c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000001.1 - Region 6 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-465"></div> <span class="help-tooltip" id="region_svg-help-465">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000001.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c6-svg" data-filename="JAAZTN010000001.1_r1c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,038,256 - 5,088,211 nt. (total: 49,956 nt)

<a class="cluster-rules-header" id="r1c6-rules-header" href="#r1c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c6-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-466"></div> <span class="help-tooltip" id="focus-panel-help-466">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c6-gene-table-details" data-name="r1c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c6-nrps_pks" data-name="r1c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c6-MIBiG-cluster-compare" data-name="r1c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c6-clusterblast" data-name="r1c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c6-knownclusterblast" data-name="r1c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c6-subclusterblast" data-name="r1c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c6-nrps-pks-bubbles" data-name="r1c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c6-pfam-details" data-name="r1c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c6-tigrfam-details" data-name="r1c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-58"></div> <span class="help-tooltip" id="gt-help-tooltip-help-58">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-50"></div> <span class="help-tooltip" id="nrps-domain-help-50">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-details-svg" data-filename="JAAZTN010000001.1_r1c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-51"></div> <span class="help-tooltip" id="MIBiG-help-51">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c6"></div>
<div id="comparison-MIBiG-r1c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-52"></div> <span class="help-tooltip" id="cb-general-help-52">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c6_1.svg">
NC_007196 (3980521-4047637): Aspergillus fumigatus Af293 chromosome 3, whole ... (19% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_2.svg">
NT_165983 (57601-125565): Chaetomium globosum CBS 148.51 scaffold 8 genomic s... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_3.svg">
NW_022984959 (833150-900692): Dissoconium aciculare CBS 342.82 unplaced genom... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_4.svg">
NW_025091284 (4592542-4724179): Aspergillus pseudoviridinutans IFM 55266 DNA,... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_5.svg">
NW_025091265 (4273679-4408020): Aspergillus udagawae IFM 46973 DNA, scaffold ... (12% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c6_6.svg">
NW_023336278 (1558416-1625499): Aspergillus tubingensis WU-2223L DNA, scaffol... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_7.svg">
NW_020291804 (923841-990927): Aspergillus costaricaensis CBS 115574 unplaced ... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_8.svg">
NW_020291598 (1079001-1146084): Aspergillus piperis CBS 112811 unplaced genom... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_9.svg">
NC_054853 (4826346-4893426): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_10.svg">
NW_020290888 (242228-309772): Aspergillus niger CBS 101883 unplaced genomic s... (21% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c6-download">Download graphic</button>
</div>
<div id="clusterblast-r1c6-svg"></div>

</div>
</div>

<div class="body-details-section r1c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-53"></div> <span class="help-tooltip" id="cb-known-help-53">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-54"></div> <span class="help-tooltip" id="cb-sub-help-54">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-57"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-57">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-domain-bubble-svg" data-filename="JAAZTN010000001.1_r1c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC8">
Candidate 8 (5038255 - 5088211): single NRPS
</option>


</select>

<div class="details-svg" id="r1c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-59"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-59">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-pfam-details-svg" data-filename="JAAZTN010000001.1_r1c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-60"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-60">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-tigrfam-details-svg" data-filename="JAAZTN010000001.1_r1c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c6-nrps_pks_products" data-name="r1c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c6-nrps_pks_monomers" data-name="r1c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-55"></div> <span class="help-tooltip" id="nrps-structures-help-55">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 8, location 5038255 - 5088211:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c6-8" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c6-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-56"></div> <span class="help-tooltip" id="nrps-monomers-help-56">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1291</span></strong>: X - X
<div class="collapser collapser-target-input-path1-gene1291 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (845..1223)</strong>: X
<div class="collapser collapser-target-input-path1-gene1291 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-input-path1-gene1291-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">SARDVGSQLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1892..2276)</strong>: X
<div class="collapser collapser-target-input-path1-gene1291 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-input-path1-gene1291-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGDNMGVICK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DTDDMGFIDK</span> (56% 8Å match)</dd>
<dd>OH-Orn <span class="serif">DMENMGLINK</span> (47% 8Å match)</dd>
<dd>n-oxoDec <span class="serif">DGLTCGVIIK</span> (38% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000001.1 - Region 7 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-467"></div> <span class="help-tooltip" id="region_svg-help-467">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000001.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c7-svg" data-filename="JAAZTN010000001.1_r1c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,118,244 - 5,152,499 nt. (total: 34,256 nt)

<a class="cluster-rules-header" id="r1c7-rules-header" href="#r1c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c7-rules">NAPAA: (AMP-binding and TIGR02353)</div>

<div class="region-svg-container">
<div id='r1c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-468"></div> <span class="help-tooltip" id="focus-panel-help-468">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c7-gene-table-details" data-name="r1c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c7-nrps_pks" data-name="r1c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c7-MIBiG-cluster-compare" data-name="r1c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c7-clusterblast" data-name="r1c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c7-knownclusterblast" data-name="r1c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c7-subclusterblast" data-name="r1c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c7-nrps-pks-bubbles" data-name="r1c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c7-pfam-details" data-name="r1c7-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c7-tigrfam-details" data-name="r1c7-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-69"></div> <span class="help-tooltip" id="gt-help-tooltip-help-69">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-61"></div> <span class="help-tooltip" id="nrps-domain-help-61">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c7-details-svg" data-filename="JAAZTN010000001.1_r1c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-62"></div> <span class="help-tooltip" id="MIBiG-help-62">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c7"></div>
<div id="comparison-MIBiG-r1c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NAPAA</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.54, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>ε-Poly-L-lysine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>γ-poly-L-2,4-diaminobutyric acid</td>


<td>Streptomyces celluloflavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>zealexin B2, zealexin C2</td>


<td>Zea mays</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001679: 12-12481">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001679/index.html#r1c1" target="_new">BGC0001679</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:0.52, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>N-Acetyltryptophan</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.54, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>ε-Poly-L-lysine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>γ-poly-L-2,4-diaminobutyric acid</td>
<td>Streptomyces celluloflavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>zealexin B2, zealexin C2</td>
<td>Zea mays</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001679: 12-12481">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001679/index.html#r1c1" target="_new">BGC0001679</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:0.52, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>N-Acetyltryptophan</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-63"></div> <span class="help-tooltip" id="cb-general-help-63">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c7_1.svg">
NW_006271975 (1702597-1751671): Cordyceps militaris CM01 unplaced genomic sca... (38% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r1c7_2.svg">
NW_011942149 (6466303-6515308): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (16% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r1c7_3.svg">
NC_054389 (12430017-12539558): Drechmeria coniospora strain ARSEF 6962 chromo... (7% of genes show similarity), NAPAA,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c7_4.svg">
NC_035791 (818908-980602): Pochonia chlamydosporia 170 chromosome 2, whole ge... (3% of genes show similarity), NAPAA,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c7_5.svg">
NC_018207 (430001-619419): Zymoseptoria tritici IPO323 chromosome 12, whole g... (3% of genes show similarity), NAPAA,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c7_6.svg">
NZ_NDXL01000001 (4615806-4695188): Streptomyces kasugaensis strain BCRC 12349... (6% of genes show similarity), NAPAA,NRPS
</option>

<option value = "svg/clusterblast_r1c7_7.svg">
NZ_CP023778 (520234-554272): Nocardia terpenica strain NC YFY NT001 chromosom... (12% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r1c7_8.svg">
NZ_KV411303 (2673555-2707593): Nocardia terpenica strain IFM 0406 scaffold1, ... (12% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r1c7_9.svg">
NZ_CP007155 (5665878-5722576): Kutzneria albida DSM 43870 chromosome, complet... (5% of genes show similarity), NAPAA,betalactone,other
</option>

<option value = "svg/clusterblast_r1c7_10.svg">
NZ_CP073318 (1841655-1875627): Kutzneria sp. CA-103260 chromosome, complete g... (10% of genes show similarity), NAPAA
</option>

</select>
<button id= "clusterblast-r1c7-download">Download graphic</button>
</div>
<div id="clusterblast-r1c7-svg"></div>

</div>
</div>

<div class="body-details-section r1c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-64"></div> <span class="help-tooltip" id="cb-known-help-64">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c7-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c7_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c7_1.svg">
ε-Poly-L-lysine
</option>

</select>
<button id="knownclusterblast-r1c7-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c7-svg"></div>

</div>
</div>

<div class="body-details-section r1c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-65"></div> <span class="help-tooltip" id="cb-sub-help-65">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-68"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-68">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c7-domain-bubble-svg" data-filename="JAAZTN010000001.1_r1c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC9">
Candidate 9 (5118243 - 5152499): single NAPAA
</option>


</select>

<div class="details-svg" id="r1c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-70"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-70">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c7-pfam-details-svg" data-filename="JAAZTN010000001.1_r1c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c7-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-71"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-71">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c7-tigrfam-details-svg" data-filename="JAAZTN010000001.1_r1c7_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c7-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c7-nrps_pks_products" data-name="r1c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c7-nrps_pks_monomers" data-name="r1c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-66"></div> <span class="help-tooltip" id="nrps-structures-help-66">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 9, location 5118243 - 5152499:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c7-9" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c7-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-67"></div> <span class="help-tooltip" id="nrps-monomers-help-67">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1312</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1312 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (57..432)</strong>: X
<div class="collapser collapser-target-input-path1-gene1312 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1312-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAESIGTVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DAEDIGTVVK</span> (68% 8Å match)</dd>
<dd>Gln or Tyr <span class="serif">DAESVGTVVK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000001.1 - Region 8 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-469"></div> <span class="help-tooltip" id="region_svg-help-469">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000001.1.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c8-svg" data-filename="JAAZTN010000001.1_r1c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,701,426 - 5,746,814 nt. (total: 45,389 nt)

<a class="cluster-rules-header" id="r1c8-rules-header" href="#r1c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c8-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-470"></div> <span class="help-tooltip" id="focus-panel-help-470">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c8-gene-table-details" data-name="r1c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c8-nrps_pks" data-name="r1c8-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c8-MIBiG-cluster-compare" data-name="r1c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c8-clusterblast" data-name="r1c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c8-knownclusterblast" data-name="r1c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c8-subclusterblast" data-name="r1c8-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c8-pfam-details" data-name="r1c8-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c8-tigrfam-details" data-name="r1c8-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-78"></div> <span class="help-tooltip" id="gt-help-tooltip-help-78">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c8-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-72"></div> <span class="help-tooltip" id="nrps-domain-help-72">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c8-details-svg" data-filename="JAAZTN010000001.1_r1c8_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c8-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-73"></div> <span class="help-tooltip" id="MIBiG-help-73">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c8"></div>
<div id="comparison-MIBiG-r1c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>phaseolotoxin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>serinocyclin A, serinocyclin B</td>


<td>Metarhizium robertsii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>phaseolotoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>serinocyclin A, serinocyclin B</td>
<td>Metarhizium robertsii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-74"></div> <span class="help-tooltip" id="cb-general-help-74">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c8_1.svg">
NC_072351 (3151891-3300466): Talaromyces marneffei chromosome 4, complete seq... (7% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c8_2.svg">
NC_026474 (147409-227940): Fusarium graminearum PH-1 chromosome 1, whole geno... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_3.svg">
NW_020291810 (881892-951425): Aspergillus costaricaensis CBS 115574 unplaced ... (40% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_4.svg">
NW_014574707 (383137-474713): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_5.svg">
NW_020290885 (1149889-1231194): Aspergillus niger CBS 101883 unplaced genomic... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_6.svg">
NW_014574611 (142817-224122): Penicillium digitatum Pd1 unplaced genomic scaf... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_7.svg">
NW_020291863 (32753-124302): Aspergillus costaricaensis CBS 115574 unplaced g... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_8.svg">
NW_025091281 (4887733-5033357): Aspergillus pseudoviridinutans IFM 55266 DNA,... (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c8_9.svg">
NC_049565 (568974-656341): Talaromyces rugulosus chromosome V, complete sequence (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_10.svg">
NC_036437 (3459599-3523616): Aspergillus oryzae RIB40 DNA, chromosome 3 (9% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c8-download">Download graphic</button>
</div>
<div id="clusterblast-r1c8-svg"></div>

</div>
</div>

<div class="body-details-section r1c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-75"></div> <span class="help-tooltip" id="cb-known-help-75">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-76"></div> <span class="help-tooltip" id="cb-sub-help-76">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-79"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-79">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c8-pfam-details-svg" data-filename="JAAZTN010000001.1_r1c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c8-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-80"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-80">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c8-tigrfam-details-svg" data-filename="JAAZTN010000001.1_r1c8_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c8-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c8-nrps_pks_monomers" data-name="r1c8-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r1c8-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-77"></div> <span class="help-tooltip" id="nrps-monomers-help-77">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

</dl>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000002.1 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-471"></div> <span class="help-tooltip" id="region_svg-help-471">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000002.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="JAAZTN010000002.1_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 368,177 - 418,344 nt. (total: 50,168 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-472"></div> <span class="help-tooltip" id="focus-panel-help-472">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-nrps_pks" data-name="r2c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c1-nrps-pks-bubbles" data-name="r2c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-89"></div> <span class="help-tooltip" id="gt-help-tooltip-help-89">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-81"></div> <span class="help-tooltip" id="nrps-domain-help-81">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-details-svg" data-filename="JAAZTN010000002.1_r2c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-82"></div> <span class="help-tooltip" id="MIBiG-help-82">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.77 (id:0.80, order:0.68, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>


<td>0.47</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.53, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>AbT1</td>


<td>Aureobasidium pullulans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.77 (id:0.80, order:0.68, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
<td>0.77</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.53, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>AbT1</td>
<td>Aureobasidium pullulans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-83"></div> <span class="help-tooltip" id="cb-general-help-83">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NW_006271972 (3966194-4059400): Cordyceps militaris CM01 unplaced genomic sca... (26% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NZ_BMRB01000010 (37666-131835): Actinokineospora fastidiosa strain JCM 3276 s... (4% of genes show similarity), NAPAA,NRPS
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NZ_CP031087 (3156779-3251358): Actinokineospora sp. UTMC 2448 chromosome, com... (4% of genes show similarity), NAPAA,NRPS
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NZ_CP096143 (4376050-4549469): Xanthomonas prunicola strain CIX97 chromosome,... (17% of genes show similarity), NRPS,T3PKS
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NZ_CP094827 (4889000-5061283): Xanthomonas prunicola strain Arg-1 chromosome,... (11% of genes show similarity), NRPS,T3PKS
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NZ_CP086333 (2267952-2438218): Xanthomonas translucens pv. arrhenatheri strai... (13% of genes show similarity), NRPS,T3PKS
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NZ_CP086332 (2341121-2517669): Xanthomonas translucens pv. phleipratensis str... (16% of genes show similarity), NRPS,T3PKS
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NZ_CP036251 (1599450-1772476): Xanthomonas oryzae strain X11-5A chromosome, c... (11% of genes show similarity), NRPS,T3PKS
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NZ_CP071090 (4176991-4379375): Pyxidicoccus sp. SCPEA002 chromosome (5% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NZ_CP114039 (6826819-6960069): Corallococcus sp. NCRR chromosome, complete ge... (6% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-84"></div> <span class="help-tooltip" id="cb-known-help-84">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c1_1.svg">
beauvericin
</option>

<option value = "svg/knownclusterblast_r2c1_2.svg">
enniatin
</option>

<option value = "svg/knownclusterblast_r2c1_3.svg">
AbT1
</option>

</select>
<button id="knownclusterblast-r2c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-85"></div> <span class="help-tooltip" id="cb-sub-help-85">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-88"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-88">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-domain-bubble-svg" data-filename="JAAZTN010000002.1_r2c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (368176 - 418344): single NRPS
</option>


</select>

<div class="details-svg" id="r2c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-90"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-90">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="JAAZTN010000002.1_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-91"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-91">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="JAAZTN010000002.1_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c1-nrps_pks_products" data-name="r2c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c1-nrps_pks_monomers" data-name="r2c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-86"></div> <span class="help-tooltip" id="nrps-structures-help-86">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 368176 - 418344:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-1" class="smiles-canvas" data-smiles="OC(C(C)C)C(=O)NC(Cc1ccccc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(D-Hiv - NMe-Phe)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-hiv,nme-phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-hiv*,nme-phe*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="OC(C(C)C)C(=O)NC(Cc1ccccc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-87"></div> <span class="help-tooltip" id="nrps-monomers-help-87">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene110</span></strong>: D-Hiv - Phe
<div class="collapser collapser-target-input-path1-gene110 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-hiv,phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-hiv*,phe*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (460..858)</strong>: D-Hiv
<div class="collapser collapser-target-input-path1-gene110 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  D-Hiv
<div class="collapser collapser-target-input-path1-gene110-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GALMIVGSIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>D-Hiv <span class="serif">GALMIVGSIK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1556..1962)</strong>: Phe
<div class="collapser collapser-target-input-path1-gene110 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-input-path1-gene110-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGYIMAAVMK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DGYIMAAVMK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000002.1 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-473"></div> <span class="help-tooltip" id="region_svg-help-473">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000002.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c2-svg" data-filename="JAAZTN010000002.1_r2c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 451,888 - 512,608 nt. (total: 60,721 nt)

<a class="cluster-rules-header" id="r2c2-rules-header" href="#r2c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r2c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-474"></div> <span class="help-tooltip" id="focus-panel-help-474">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c2-gene-table-details" data-name="r2c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c2-MIBiG-cluster-compare" data-name="r2c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c2-clusterblast" data-name="r2c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c2-knownclusterblast" data-name="r2c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c2-subclusterblast" data-name="r2c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c2-pfam-details" data-name="r2c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-96"></div> <span class="help-tooltip" id="gt-help-tooltip-help-96">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-92"></div> <span class="help-tooltip" id="MIBiG-help-92">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c2"></div>
<div id="comparison-MIBiG-r2c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002337: 0-2741">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002337/index.html#r1c1" target="_new">BGC0002337</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.22</td>
<td>RiPP</td>

<td>class IV lanthipeptide, SflA</td>


<td>Streptomyces sp. NRRL S-1022</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>


<td>Actinomadura namibiensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000584: 0-3682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000584/index.html#r1c1" target="_new">BGC0000584</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>colicin V</td>


<td>Escherichia coli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502: 0-15725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>


<td>Bacillus cereus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>carnolysin A1, carnolysin A2</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001494: 0-5185">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001494/index.html#r1c1" target="_new">BGC0001494</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>acinetodin</td>


<td>Acinetobacter gyllenbergii CIP 110306</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>auriculamide</td>


<td>Herpetosiphon aurantiacus DSM 785</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503: 0-17083">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>cinnamycin</td>


<td>Streptomyces cinnamoneus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002337: 0-2741">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002337/index.html#r1c1" target="_new">BGC0002337</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>RiPP</td>
<td>class IV lanthipeptide, SflA</td>
<td>Streptomyces sp. NRRL S-1022</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
<td>Actinomadura namibiensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>SRO15-3108</td>
<td>Streptomyces filamentosus NRRL 15998</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000584: 0-3682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000584/index.html#r1c1" target="_new">BGC0000584</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>colicin V</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502: 0-15725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
<td>Bacillus cereus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>carnolysin A1, carnolysin A2</td>
<td>Carnobacterium maltaromaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001494: 0-5185">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001494/index.html#r1c1" target="_new">BGC0001494</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>RiPP</td>
<td>acinetodin</td>
<td>Acinetobacter gyllenbergii CIP 110306</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP</td>
<td>auriculamide</td>
<td>Herpetosiphon aurantiacus DSM 785</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503: 0-17083">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>RiPP</td>
<td>cinnamycin</td>
<td>Streptomyces cinnamoneus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-93"></div> <span class="help-tooltip" id="cb-general-help-93">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c2_1.svg">
NW_006271970 (3761055-3936010): Cordyceps militaris CM01 unplaced genomic sca... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_2.svg">
NW_023336279 (145289-236294): Aspergillus tubingensis WU-2223L DNA, scaffold ... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_3.svg">
NC_035791 (6148325-6287407): Pochonia chlamydosporia 170 chromosome 2, whole ... (5% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_4.svg">
NW_020291612 (327439-419782): Aspergillus piperis CBS 112811 unplaced genomic... (8% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_5.svg">
NW_020290899 (145522-237868): Aspergillus niger CBS 101883 unplaced genomic s... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_6.svg">
NW_020292127 (15270-136339): Aspergillus ibericus CBS 121593 unplaced genomic... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r2c2_7.svg">
NT_166521 (362145-454491): Aspergillus niger CBS 513.88 supercontig An05 (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_8.svg">
NW_020290621 (328465-419468): Aspergillus saccharolyticus JOP 1030-1 unplaced... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_9.svg">
NC_054856 (3727884-3894446): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (3% of genes show similarity), NI-siderophore,NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_10.svg">
NW_020292053 (616043-708384): Aspergillus vadensis CBS 113365 unplaced genomi... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r2c2-download">Download graphic</button>
</div>
<div id="clusterblast-r2c2-svg"></div>

</div>
</div>

<div class="body-details-section r2c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-94"></div> <span class="help-tooltip" id="cb-known-help-94">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-95"></div> <span class="help-tooltip" id="cb-sub-help-95">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-97"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-97">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-pfam-details-svg" data-filename="JAAZTN010000002.1_r2c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r2c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000002.1 - Region 3 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-475"></div> <span class="help-tooltip" id="region_svg-help-475">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000002.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c3-svg" data-filename="JAAZTN010000002.1_r2c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 617,599 - 678,443 nt. (total: 60,845 nt)

<a class="cluster-rules-header" id="r2c3-rules-header" href="#r2c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c3-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r2c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-476"></div> <span class="help-tooltip" id="focus-panel-help-476">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c3-gene-table-details" data-name="r2c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c3-MIBiG-cluster-compare" data-name="r2c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c3-clusterblast" data-name="r2c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c3-knownclusterblast" data-name="r2c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c3-subclusterblast" data-name="r2c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c3-pfam-details" data-name="r2c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-102"></div> <span class="help-tooltip" id="gt-help-tooltip-help-102">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-98"></div> <span class="help-tooltip" id="MIBiG-help-98">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c3"></div>
<div id="comparison-MIBiG-r2c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-99"></div> <span class="help-tooltip" id="cb-general-help-99">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c3_1.svg">
NW_006271969 (3108218-3200931): Cordyceps militaris CM01 unplaced genomic sca... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c3_2.svg">
NW_025091286 (2285069-2376104): Aspergillus pseudoviridinutans IFM 55266 DNA,... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c3_3.svg">
NW_025091290 (21868-84973): Aspergillus pseudoviridinutans IFM 55266 DNA, sca... (9% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r2c3-download">Download graphic</button>
</div>
<div id="clusterblast-r2c3-svg"></div>

</div>
</div>

<div class="body-details-section r2c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-100"></div> <span class="help-tooltip" id="cb-known-help-100">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-101"></div> <span class="help-tooltip" id="cb-sub-help-101">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-103"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-103">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-pfam-details-svg" data-filename="JAAZTN010000002.1_r2c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r2c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000002.1 - Region 4 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-477"></div> <span class="help-tooltip" id="region_svg-help-477">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000002.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c4-svg" data-filename="JAAZTN010000002.1_r2c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 988,357 - 1,048,573 nt. (total: 60,217 nt)

<a class="cluster-rules-header" id="r2c4-rules-header" href="#r2c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c4-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r2c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-478"></div> <span class="help-tooltip" id="focus-panel-help-478">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c4-gene-table-details" data-name="r2c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c4-MIBiG-cluster-compare" data-name="r2c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c4-clusterblast" data-name="r2c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c4-knownclusterblast" data-name="r2c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c4-subclusterblast" data-name="r2c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c4-pfam-details" data-name="r2c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c4-tigrfam-details" data-name="r2c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-108"></div> <span class="help-tooltip" id="gt-help-tooltip-help-108">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-104"></div> <span class="help-tooltip" id="MIBiG-help-104">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c4"></div>
<div id="comparison-MIBiG-r2c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487: 0-9112">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other</td>

<td>3,7-dihydroxytropolone</td>


<td>Streptomyces cyaneofuscatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160: 0-27093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>terreic acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000628: 0-13800">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000628/index.html#r1c1" target="_new">BGC0000628</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>lactocillin</td>


<td>Lactobacillus paragasseri JV-V03</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.49, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Terpene</td>

<td>citreohybridonol</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002390: 227-643106">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002390/index.html#r1c1" target="_new">BGC0002390</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Terpene</td>

<td>zealexin A1</td>


<td>Zea mays</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000495: 41-38168">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000495/index.html#r1c1" target="_new">BGC0000495</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.42, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>RiPP</td>

<td>actagardine</td>


<td>Actinoplanes garbadinensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487: 0-9112">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other</td>
<td>3,7-dihydroxytropolone</td>
<td>Streptomyces cyaneofuscatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160: 0-27093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>terreic acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000628: 0-13800">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000628/index.html#r1c1" target="_new">BGC0000628</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>RiPP</td>
<td>lactocillin</td>
<td>Lactobacillus paragasseri JV-V03</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.49, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Terpene</td>
<td>citreohybridonol</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002390: 227-643106">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002390/index.html#r1c1" target="_new">BGC0002390</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Terpene</td>
<td>zealexin A1</td>
<td>Zea mays</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000495: 41-38168">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000495/index.html#r1c1" target="_new">BGC0000495</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.42, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>RiPP</td>
<td>actagardine</td>
<td>Actinoplanes garbadinensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-105"></div> <span class="help-tooltip" id="cb-general-help-105">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c4_1.svg">
NC_064676 (3053739-3126636): Colletotrichum lupini chromosome 3, complete seq... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c4_2.svg">
NC_066262 (2299164-2362581): Aspergillus nidulans FGSC A4 chromosome VI (15% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c4_3.svg">
NZ_CP074694 (1708839-1753498): Telmatocola sphagniphila strain SP2T chromosom... (6% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r2c4-download">Download graphic</button>
</div>
<div id="clusterblast-r2c4-svg"></div>

</div>
</div>

<div class="body-details-section r2c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-106"></div> <span class="help-tooltip" id="cb-known-help-106">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-107"></div> <span class="help-tooltip" id="cb-sub-help-107">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-109"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-109">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-pfam-details-svg" data-filename="JAAZTN010000002.1_r2c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-110"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-110">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-tigrfam-details-svg" data-filename="JAAZTN010000002.1_r2c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r2c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000002.1 - Region 5 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-479"></div> <span class="help-tooltip" id="region_svg-help-479">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000002.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c5-svg" data-filename="JAAZTN010000002.1_r2c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,916,227 - 4,983,225 nt. (total: 66,999 nt)

<a class="cluster-rules-header" id="r2c5-rules-header" href="#r2c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c5-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-480"></div> <span class="help-tooltip" id="focus-panel-help-480">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c5-gene-table-details" data-name="r2c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c5-nrps_pks" data-name="r2c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c5-MIBiG-cluster-compare" data-name="r2c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c5-clusterblast" data-name="r2c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c5-knownclusterblast" data-name="r2c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c5-subclusterblast" data-name="r2c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c5-nrps-pks-bubbles" data-name="r2c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c5-pfam-details" data-name="r2c5-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-119"></div> <span class="help-tooltip" id="gt-help-tooltip-help-119">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-111"></div> <span class="help-tooltip" id="nrps-domain-help-111">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-details-svg" data-filename="JAAZTN010000002.1_r2c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-112"></div> <span class="help-tooltip" id="MIBiG-help-112">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c5"></div>
<div id="comparison-MIBiG-r2c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.69</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002639: 0-8541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002639/index.html#r1c1" target="_new">BGC0002639</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP</td>

<td>phospho-tyrolose, tyrolose, tyrolitol, tyrocitabine-626, acyl-tyrocitabine-668, acyl-tyrocitabine-696, acyl-tyrocitabine-724, acyl-tyrocitabine-752</td>


<td>Lactobacillus iners LEAF 2052A-d</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP, Polyketide</td>

<td>himeic acid A</td>


<td>Aspergillus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide, NRP</td>

<td>pyranoviolin A</td>


<td>Aspergillus violaceofuscus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002639: 0-8541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002639/index.html#r1c1" target="_new">BGC0002639</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>phospho-tyrolose, tyrolose, tyrolitol, tyrocitabine-626, acyl-tyrocitabine-668, acyl-tyrocitabine-696, acyl-tyrocitabine-724, acyl-tyrocitabine-752</td>
<td>Lactobacillus iners LEAF 2052A-d</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>himeic acid A</td>
<td>Aspergillus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide, NRP</td>
<td>pyranoviolin A</td>
<td>Aspergillus violaceofuscus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-113"></div> <span class="help-tooltip" id="cb-general-help-113">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c5_1.svg">
NW_006271971 (3503471-3590640): Cordyceps militaris CM01 unplaced genomic sca... (65% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c5_2.svg">
NW_020290887 (673812-764439): Aspergillus niger CBS 101883 unplaced genomic s... (28% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c5_3.svg">
NW_023336287 (1029426-1127137): Aspergillus tubingensis WU-2223L DNA, scaffol... (29% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c5_4.svg">
NW_020292069 (316400-408453): Aspergillus vadensis CBS 113365 unplaced genomi... (30% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c5_5.svg">
NW_020291625 (99659-196841): Aspergillus piperis CBS 112811 unplaced genomic ... (29% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c5_6.svg">
NW_020291816 (340965-433020): Aspergillus costaricaensis CBS 115574 unplaced ... (29% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c5_7.svg">
NC_054853 (2119459-2216438): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (25% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c5_8.svg">
NW_013566983 (689939-757505): Aureobasidium subglaciale EXF-2481 unplaced gen... (19% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_9.svg">
NW_013581207 (483365-550922): Aureobasidium namibiae CBS 147.97 unplaced geno... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_10.svg">
NW_022474207 (2338451-2410505): Venustampulla echinocandica strain BP 5553 ch... (19% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r2c5-download">Download graphic</button>
</div>
<div id="clusterblast-r2c5-svg"></div>

</div>
</div>

<div class="body-details-section r2c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-114"></div> <span class="help-tooltip" id="cb-known-help-114">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-115"></div> <span class="help-tooltip" id="cb-sub-help-115">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-118"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-118">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-domain-bubble-svg" data-filename="JAAZTN010000002.1_r2c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (4916226 - 4983225): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r2c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-120"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-120">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-pfam-details-svg" data-filename="JAAZTN010000002.1_r2c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c5-nrps_pks_products" data-name="r2c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c5-nrps_pks_monomers" data-name="r2c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-116"></div> <span class="help-tooltip" id="nrps-structures-help-116">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 4916226 - 4983225:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c5-5" class="smiles-canvas" data-smiles="NC([*])C(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c5-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-117"></div> <span class="help-tooltip" id="nrps-monomers-help-117">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1325</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene1325 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (547..865)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1325 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1325-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1325-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 71.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 63.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 48.4</dt>

<dd></dd><dt>Malonyl-CoA: 27.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 26.5</dt>

<dd></dd><dt>fatty_acid: 25.3</dt>

<dd></dd><dt>Propionyl-CoA: 22.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 21.9</dt>

<dd></dd><dt>Benzoyl-CoA: 21.4</dt>

<dd></dd><dt>CHC-CoA: 11.0</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2996..3403)</strong>: X
<div class="collapser collapser-target-input-path1-gene1325 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene1325-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIDTNGNVWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGVVWK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene1328</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1328 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (229..666)</strong>: X
<div class="collapser collapser-target-input-path1-gene1328 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene1328-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLIFVGIMAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DALFVGLVAK</span> (62% 8Å match)</dd>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000002.1 - Region 6 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-481"></div> <span class="help-tooltip" id="region_svg-help-481">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000002.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c6-svg" data-filename="JAAZTN010000002.1_r2c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,342,184 - 5,363,737 nt. (total: 21,554 nt)

<a class="cluster-rules-header" id="r2c6-rules-header" href="#r2c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c6-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-482"></div> <span class="help-tooltip" id="focus-panel-help-482">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c6-gene-table-details" data-name="r2c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c6-MIBiG-cluster-compare" data-name="r2c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c6-clusterblast" data-name="r2c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c6-knownclusterblast" data-name="r2c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c6-subclusterblast" data-name="r2c6-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c6-pfam-details" data-name="r2c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c6-tigrfam-details" data-name="r2c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-125"></div> <span class="help-tooltip" id="gt-help-tooltip-help-125">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-121"></div> <span class="help-tooltip" id="MIBiG-help-121">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c6"></div>
<div id="comparison-MIBiG-r2c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.53, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>Terpene</td>

<td>squalestatin S1</td>


<td>Aspergillus sp. Z5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>cytolysin ClyLl, cytolysin ClyLs</td>


<td>Plasmid pAD1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>microcyclamide</td>


<td>Microcystis aeruginosa NIES-298</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>carbapenem MM4550</td>


<td>Streptomyces argenteolus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.52, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>squalestatin S1</td>


<td>Phoma sp. MF5453</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>gramicidin S</td>


<td>Aneurinibacillus migulanus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>





<td class="cc-heat-cell" title="0.15 (id:0.45, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>sorangipyranone</td>


<td>Myxococcales bacterium</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.53, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>Terpene</td>
<td>squalestatin S1</td>
<td>Aspergillus sp. Z5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>cytolysin ClyLl, cytolysin ClyLs</td>
<td>Plasmid pAD1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>RiPP</td>
<td>microcyclamide</td>
<td>Microcystis aeruginosa NIES-298</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>carbapenem MM4550</td>
<td>Streptomyces argenteolus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.52, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Polyketide</td>
<td>squalestatin S1</td>
<td>Phoma sp. MF5453</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP</td>
<td>gramicidin S</td>
<td>Aneurinibacillus migulanus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>


<td class="cc-heat-cell" title="0.15 (id:0.45, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
<td>0.15</td>
<td>Polyketide</td>
<td>sorangipyranone</td>
<td>Myxococcales bacterium</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-122"></div> <span class="help-tooltip" id="cb-general-help-122">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c6_1.svg">
NW_006271974 (1078266-1109834): Cordyceps militaris CM01 unplaced genomic sca... (27% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_2.svg">
NC_070529 (5465471-5497016): Fusarium keratoplasticum isolate Fu6.1 chromosom... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_3.svg">
NC_070544 (5794890-5826434): Fusarium falciforme chromosome 1, complete sequence (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_4.svg">
NC_036623 (3601013-3632563): Fusarium fujikuroi IMI 58289 draft genome, chrom... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_5.svg">
NC_058389 (1466023-1497573): Fusarium musae strain F31 chromosome 3, whole ge... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_6.svg">
NC_058402 (6845329-6876878): Fusarium poae strain DAOMC 252244 chromosome 4, ... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_7.svg">
NC_054390 (5438362-5469923): Drechmeria coniospora strain ARSEF 6962 chromoso... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_8.svg">
NC_026477 (6663266-6694813): Fusarium graminearum PH-1 chromosome 4, whole ge... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_9.svg">
NC_038015 (6636088-6667634): Fusarium venenatum strain A3/5 genome assembly, ... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_10.svg">
NW_025764705 (1077863-1109318): Emericellopsis atlantica unplaced genomic sca... (21% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r2c6-download">Download graphic</button>
</div>
<div id="clusterblast-r2c6-svg"></div>

</div>
</div>

<div class="body-details-section r2c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-123"></div> <span class="help-tooltip" id="cb-known-help-123">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c6-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c6_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c6_1.svg">
squalestatin S1
</option>

</select>
<button id="knownclusterblast-r2c6-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c6-svg"></div>

</div>
</div>

<div class="body-details-section r2c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-124"></div> <span class="help-tooltip" id="cb-sub-help-124">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-126"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-126">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-pfam-details-svg" data-filename="JAAZTN010000002.1_r2c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-127"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-127">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-tigrfam-details-svg" data-filename="JAAZTN010000002.1_r2c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r2c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000002.1 - Region 7 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-483"></div> <span class="help-tooltip" id="region_svg-help-483">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000002.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c7-svg" data-filename="JAAZTN010000002.1_r2c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,156,507 - 6,221,691 nt. (total: 65,185 nt)

<a class="cluster-rules-header" id="r2c7-rules-header" href="#r2c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c7-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r2c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-484"></div> <span class="help-tooltip" id="focus-panel-help-484">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c7-gene-table-details" data-name="r2c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c7-MIBiG-cluster-compare" data-name="r2c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c7-clusterblast" data-name="r2c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c7-knownclusterblast" data-name="r2c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c7-subclusterblast" data-name="r2c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c7-pfam-details" data-name="r2c7-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-132"></div> <span class="help-tooltip" id="gt-help-tooltip-help-132">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-128"></div> <span class="help-tooltip" id="MIBiG-help-128">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c7"></div>
<div id="comparison-MIBiG-r2c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000687: 3134-24057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000687/index.html#r1c1" target="_new">BGC0000687</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.36</td>
<td>Terpene</td>

<td>phyllocladan-16α-ol</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:0.57, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>betaestacin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002724: 0-63348">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002724/index.html#r1c1" target="_new">BGC0002724</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>casbene</td>


<td>Jatropha curcas</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>mangicol A</td>


<td>Fusarium equiseti</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000687: 3134-24057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000687/index.html#r1c1" target="_new">BGC0000687</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>phyllocladan-16α-ol</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:0.57, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002724: 0-63348">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002724/index.html#r1c1" target="_new">BGC0002724</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>casbene</td>
<td>Jatropha curcas</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>mangicol A</td>
<td>Fusarium equiseti</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-129"></div> <span class="help-tooltip" id="cb-general-help-129">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c7_1.svg">
NW_020291483 (499510-590382): Aspergillus aculeatinus CBS 121060 unplaced gen... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_2.svg">
NC_036441 (799518-862899): Aspergillus oryzae RIB40 DNA, chromosome 7 (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c7_3.svg">
NW_007360994 (690217-788751): Glarea lozoyensis ATCC 20868 chromosome Unknown... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_4.svg">
NW_017253372 (448547-632808): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_5.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (8% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_6.svg">
NW_011942150 (718356-830095): Metarhizium robertsii ARSEF 23 MAA Scf 10, whol... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c7_7.svg">
NC_054856 (3727884-3894446): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (6% of genes show similarity), NI-siderophore,NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_8.svg">
NC_030959 (3784443-3974228): Colletotrichum higginsianum IMI 349063 chromosom... (8% of genes show similarity), NRPS,fungal-RiPP-like,indole
</option>

<option value = "svg/clusterblast_r2c7_9.svg">
NW_022474205 (248150-404938): Venustampulla echinocandica strain BP 5553 chro... (6% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r2c7_10.svg">
NW_017264196 (550250-647543): Xylona heveae TC161 unplaced genomic scaffold L... (10% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r2c7-download">Download graphic</button>
</div>
<div id="clusterblast-r2c7-svg"></div>

</div>
</div>

<div class="body-details-section r2c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-130"></div> <span class="help-tooltip" id="cb-known-help-130">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c7-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c7_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c7_1.svg">
nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

<option value = "svg/knownclusterblast_r2c7_2.svg">
trichodiene-11-one
</option>

<option value = "svg/knownclusterblast_r2c7_3.svg">
deoxynivalenol
</option>

<option value = "svg/knownclusterblast_r2c7_4.svg">
nivalenol/vomitoxin/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-deacetylcalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

</select>
<button id="knownclusterblast-r2c7-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c7-svg"></div>

</div>
</div>

<div class="body-details-section r2c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-131"></div> <span class="help-tooltip" id="cb-sub-help-131">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-133"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-133">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-pfam-details-svg" data-filename="JAAZTN010000002.1_r2c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r2c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000002.1 - Region 8 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-485"></div> <span class="help-tooltip" id="region_svg-help-485">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000002.1.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c8-svg" data-filename="JAAZTN010000002.1_r2c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,230,485 - 6,253,898 nt. (total: 23,414 nt)

<a class="cluster-rules-header" id="r2c8-rules-header" href="#r2c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c8-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-486"></div> <span class="help-tooltip" id="focus-panel-help-486">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c8-gene-table-details" data-name="r2c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c8-MIBiG-cluster-compare" data-name="r2c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c8-clusterblast" data-name="r2c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c8-knownclusterblast" data-name="r2c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c8-subclusterblast" data-name="r2c8-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c8-pfam-details" data-name="r2c8-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c8-tigrfam-details" data-name="r2c8-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-138"></div> <span class="help-tooltip" id="gt-help-tooltip-help-138">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-134"></div> <span class="help-tooltip" id="MIBiG-help-134">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c8"></div>
<div id="comparison-MIBiG-r2c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>thalianol, thaliandiol</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>marneral</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.46, order:0.45, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>aspterric acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>armillyl orsellinate, 8α-hydroxy-6-protoilludene</td>


<td>Armillaria gallica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001811: 32-25672">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001811/index.html#r1c1" target="_new">BGC0001811</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.57, order:0.34, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>trichodiene-11-one</td>


<td>Fusarium asiaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000930: 2573-28230">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000930/index.html#r1c1" target="_new">BGC0000930</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.57, order:0.34, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>deoxynivalenol</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.57, order:0.34, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>


<td>Fusarium graminearum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>thalianol, thaliandiol</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.46, order:0.45, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>aspterric acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>armillyl orsellinate, 8α-hydroxy-6-protoilludene</td>
<td>Armillaria gallica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001811: 32-25672">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001811/index.html#r1c1" target="_new">BGC0001811</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.57, order:0.34, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>trichodiene-11-one</td>
<td>Fusarium asiaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000930: 2573-28230">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000930/index.html#r1c1" target="_new">BGC0000930</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.57, order:0.34, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>deoxynivalenol</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.57, order:0.34, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
<td>Fusarium graminearum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-135"></div> <span class="help-tooltip" id="cb-general-help-135">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c8_1.svg">
NC_026475 (6633069-6664256): Fusarium graminearum PH-1 chromosome 2, whole ge... (15% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_2.svg">
NC_058400 (5864309-5895503): Fusarium poae strain DAOMC 252244 chromosome 2, ... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_3.svg">
NC_038013 (5603203-5634406): Fusarium venenatum strain A3/5 genome assembly, ... (16% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_4.svg">
NC_035791 (633219-706856): Pochonia chlamydosporia 170 chromosome 2, whole ge... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c8_5.svg">
NC_064679 (5392829-5459152): Colletotrichum lupini chromosome 6, complete seq... (27% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r2c8_6.svg">
NW_025091284 (1355867-1424289): Aspergillus pseudoviridinutans IFM 55266 DNA,... (18% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r2c8_7.svg">
NC_054693 (3931870-3995086): Aspergillus flavus NRRL3357 chromosome 3, comple... (9% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r2c8_8.svg">
NC_036437 (4008631-4071847): Aspergillus oryzae RIB40 DNA, chromosome 3 (9% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r2c8_9.svg">
NC_070540 (2401382-2432591): Fusarium keratoplasticum isolate Fu6.1 chromosom... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_10.svg">
NW_006912905 (2600367-2631558): Capronia epimyces CBS 606.96 aczIQ-supercont1... (14% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r2c8-download">Download graphic</button>
</div>
<div id="clusterblast-r2c8-svg"></div>

</div>
</div>

<div class="body-details-section r2c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-136"></div> <span class="help-tooltip" id="cb-known-help-136">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c8-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c8_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c8_1.svg">
trichodiene-11-one
</option>

<option value = "svg/knownclusterblast_r2c8_2.svg">
nivalenol/vomitoxin/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-deacetylcalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

<option value = "svg/knownclusterblast_r2c8_3.svg">
nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

<option value = "svg/knownclusterblast_r2c8_4.svg">
deoxynivalenol
</option>

</select>
<button id="knownclusterblast-r2c8-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c8-svg"></div>

</div>
</div>

<div class="body-details-section r2c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-137"></div> <span class="help-tooltip" id="cb-sub-help-137">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-139"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-139">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c8-pfam-details-svg" data-filename="JAAZTN010000002.1_r2c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c8-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-140"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-140">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c8-tigrfam-details-svg" data-filename="JAAZTN010000002.1_r2c8_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c8-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000003.1 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-487"></div> <span class="help-tooltip" id="region_svg-help-487">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000003.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="JAAZTN010000003.1_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,356,085 - 1,399,625 nt. (total: 43,541 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-488"></div> <span class="help-tooltip" id="focus-panel-help-488">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-nrps_pks" data-name="r3c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-147"></div> <span class="help-tooltip" id="gt-help-tooltip-help-147">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-141"></div> <span class="help-tooltip" id="nrps-domain-help-141">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-details-svg" data-filename="JAAZTN010000003.1_r3c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-142"></div> <span class="help-tooltip" id="MIBiG-help-142">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544: 0-27727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.47, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>NRP, Polyketide</td>

<td>chlorflavonin</td>


<td>Aspergillus campestris IBT 28561</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.41, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>isoindolinone</td>


<td>Stachybotrys chlorohalonata IBT 40285</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.41, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Terpene, Polyketide</td>

<td>ascochlorin</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001404: 0-28758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001404/index.html#r1c1" target="_new">BGC0001404</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>sorbicillin</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>ustethylin A</td>


<td>Aspergillus ustus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544: 0-27727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.47, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>NRP, Polyketide</td>
<td>chlorflavonin</td>
<td>Aspergillus campestris IBT 28561</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.41, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>isoindolinone</td>
<td>Stachybotrys chlorohalonata IBT 40285</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.41, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Terpene, Polyketide</td>
<td>ascochlorin</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001404: 0-28758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001404/index.html#r1c1" target="_new">BGC0001404</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>sorbicillin</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Polyketide</td>
<td>ustethylin A</td>
<td>Aspergillus ustus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-143"></div> <span class="help-tooltip" id="cb-general-help-143">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<p>No significant ClusterBlast hits found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-144"></div> <span class="help-tooltip" id="cb-known-help-144">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-145"></div> <span class="help-tooltip" id="cb-sub-help-145">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-148"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-148">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="JAAZTN010000003.1_r3c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c1-nrps_pks_monomers" data-name="r3c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r3c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-146"></div> <span class="help-tooltip" id="nrps-monomers-help-146">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene356</span></strong>: X
<div class="collapser collapser-target-input-path1-gene356 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (22..317)</strong>: X
<div class="collapser collapser-target-input-path1-gene356 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r3c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000003.1 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-489"></div> <span class="help-tooltip" id="region_svg-help-489">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000003.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c2-svg" data-filename="JAAZTN010000003.1_r3c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,733,944 - 2,781,143 nt. (total: 47,200 nt)

<a class="cluster-rules-header" id="r3c2-rules-header" href="#r3c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r3c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-490"></div> <span class="help-tooltip" id="focus-panel-help-490">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c2-gene-table-details" data-name="r3c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c2-nrps_pks" data-name="r3c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c2-MIBiG-cluster-compare" data-name="r3c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c2-clusterblast" data-name="r3c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c2-knownclusterblast" data-name="r3c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c2-subclusterblast" data-name="r3c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c2-nrps-pks-bubbles" data-name="r3c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c2-pfam-details" data-name="r3c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-157"></div> <span class="help-tooltip" id="gt-help-tooltip-help-157">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-149"></div> <span class="help-tooltip" id="nrps-domain-help-149">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-details-svg" data-filename="JAAZTN010000003.1_r3c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-150"></div> <span class="help-tooltip" id="MIBiG-help-150">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c2"></div>
<div id="comparison-MIBiG-r3c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>phomoidride</td>


<td>fungal sp. ATCC 74256</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>epipyriculol</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene, Polyketide</td>

<td>calidoustene A, calidoustene B, calidoustene C</td>


<td>Aspergillus calidoustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>phomoidride</td>
<td>fungal sp. ATCC 74256</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>epipyriculol</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-151"></div> <span class="help-tooltip" id="cb-general-help-151">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c2_1.svg">
NW_006271970 (1369951-1438884): Cordyceps militaris CM01 unplaced genomic sca... (38% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_2.svg">
NW_006711153 (459708-526750): Trichoderma reesei QM6a unplaced genomic scaffo... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_3.svg">
NC_035790 (8523905-8591015): Pochonia chlamydosporia 170 chromosome 1, whole ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_4.svg">
NW_025091220 (124401-191227): Aspergillus viridinutans IFM 47045 DNA, scaffol... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_5.svg">
NW_026250018 (661526-728676): Hypoxylon trugodes strain CBS 135444 unplaced g... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_6.svg">
NW_025953055 (276846-344269): Daldinia caldariorum strain CBS 122874 unplaced... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_7.svg">
NW_025953257 (328883-396554): Annulohypoxylon maeteangense strain CBS 123835 ... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_8.svg">
NW_026054554 (165113-232305): Daldinia decipiens strain CBS 113046 unplaced g... (17% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_9.svg">
NW_025953787 (2032241-2099463): Annulohypoxylon truncatum strain CBS 140777 u... (17% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_10.svg">
NC_072351 (123460-190570): Talaromyces marneffei chromosome 4, complete sequence (12% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r3c2-download">Download graphic</button>
</div>
<div id="clusterblast-r3c2-svg"></div>

</div>
</div>

<div class="body-details-section r3c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-152"></div> <span class="help-tooltip" id="cb-known-help-152">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-153"></div> <span class="help-tooltip" id="cb-sub-help-153">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-156"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-156">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-domain-bubble-svg" data-filename="JAAZTN010000003.1_r3c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (2733943 - 2781143): single T1PKS
</option>


</select>

<div class="details-svg" id="r3c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-158"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-158">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-pfam-details-svg" data-filename="JAAZTN010000003.1_r3c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c2-nrps_pks_products" data-name="r3c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c2-nrps_pks_monomers" data-name="r3c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-154"></div> <span class="help-tooltip" id="nrps-structures-help-154">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 2733943 - 2781143:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c2-2" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-155"></div> <span class="help-tooltip" id="nrps-monomers-help-155">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene684</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene684 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (538..848)</strong>: pk
<div class="collapser collapser-target-input-path1-gene684 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene684-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Isobutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene684-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 67.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 61.3</dt>

<dd></dd><dt>Isobutyryl-CoA: 49.3</dt>

<dd></dd><dt>Propionyl-CoA: 45.6</dt>

<dd></dd><dt>Malonyl-CoA: 41.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 22.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 20.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 20.2</dt>

<dd></dd><dt>Benzoyl-CoA: 17.3</dt>

<dd></dd><dt>inactive: 9.3</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000003.1 - Region 3 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-491"></div> <span class="help-tooltip" id="region_svg-help-491">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000003.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c3-svg" data-filename="JAAZTN010000003.1_r3c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,904,574 - 2,966,490 nt. (total: 61,917 nt)

<a class="cluster-rules-header" id="r3c3-rules-header" href="#r3c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-492"></div> <span class="help-tooltip" id="focus-panel-help-492">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c3-gene-table-details" data-name="r3c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c3-nrps_pks" data-name="r3c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c3-MIBiG-cluster-compare" data-name="r3c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c3-clusterblast" data-name="r3c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c3-knownclusterblast" data-name="r3c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c3-subclusterblast" data-name="r3c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c3-nrps-pks-bubbles" data-name="r3c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c3-pfam-details" data-name="r3c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c3-tigrfam-details" data-name="r3c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-167"></div> <span class="help-tooltip" id="gt-help-tooltip-help-167">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-159"></div> <span class="help-tooltip" id="nrps-domain-help-159">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-details-svg" data-filename="JAAZTN010000003.1_r3c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-160"></div> <span class="help-tooltip" id="MIBiG-help-160">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c3"></div>
<div id="comparison-MIBiG-r3c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.60</td>
<td>Polyketide</td>

<td>6-MSA, aculin A</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.58</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.58</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000120: 194-40016">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000120/index.html#r1c1" target="_new">BGC0000120</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>Polyketide</td>

<td>patulin</td>


<td>Penicillium expansum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>Terpene</td>

<td>lupeol</td>


<td>Lotus japonicus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>6-MSA, aculin A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000120: 194-40016">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000120/index.html#r1c1" target="_new">BGC0000120</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>patulin</td>
<td>Penicillium expansum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>lupeol</td>
<td>Lotus japonicus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-161"></div> <span class="help-tooltip" id="cb-general-help-161">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c3_1.svg">
NW_006271971 (3503471-3590640): Cordyceps militaris CM01 unplaced genomic sca... (20% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c3_2.svg">
NW_023336287 (1029426-1127137): Aspergillus tubingensis WU-2223L DNA, scaffol... (14% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r3c3_3.svg">
NW_020292069 (316400-408453): Aspergillus vadensis CBS 113365 unplaced genomi... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c3_4.svg">
NW_020290887 (673812-764439): Aspergillus niger CBS 101883 unplaced genomic s... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c3_5.svg">
NW_013581207 (483365-550922): Aureobasidium namibiae CBS 147.97 unplaced geno... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_6.svg">
NW_013566983 (689939-757505): Aureobasidium subglaciale EXF-2481 unplaced gen... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_7.svg">
NW_020291625 (99659-196841): Aspergillus piperis CBS 112811 unplaced genomic ... (11% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r3c3_8.svg">
NW_020291816 (340965-433020): Aspergillus costaricaensis CBS 115574 unplaced ... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c3_9.svg">
NW_022474207 (2338451-2410505): Venustampulla echinocandica strain BP 5553 ch... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c3_10.svg">
NW_020292039 (3332166-3473157): Aspergillus vadensis CBS 113365 unplaced geno... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c3-download">Download graphic</button>
</div>
<div id="clusterblast-r3c3-svg"></div>

</div>
</div>

<div class="body-details-section r3c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-162"></div> <span class="help-tooltip" id="cb-known-help-162">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-163"></div> <span class="help-tooltip" id="cb-sub-help-163">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-166"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-166">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-domain-bubble-svg" data-filename="JAAZTN010000003.1_r3c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (2904573 - 2966490): neighbouring T1PKS-NRPS
</option>



<option value="CC4">
Candidate 4 (2904573 - 2951767): single T1PKS
</option>



<option value="CC5">
Candidate 5 (2919247 - 2966490): single NRPS
</option>


</select>

<div class="details-svg" id="r3c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-168"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-168">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-pfam-details-svg" data-filename="JAAZTN010000003.1_r3c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-169"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-169">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-tigrfam-details-svg" data-filename="JAAZTN010000003.1_r3c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c3-nrps_pks_products" data-name="r3c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c3-nrps_pks_monomers" data-name="r3c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-164"></div> <span class="help-tooltip" id="nrps-structures-help-164">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 2904573 - 2966490:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c3-3" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 2904573 - 2951767:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c3-4" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c3-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 5, location 2919247 - 2966490:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c3-5" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c3-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-165"></div> <span class="help-tooltip" id="nrps-monomers-help-165">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene722</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene722 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (571..889)</strong>: pk
<div class="collapser collapser-target-input-path1-gene722 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene722-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene722-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 59.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 52.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 39.4</dt>

<dd></dd><dt>fatty_acid: 23.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 23.1</dt>

<dd></dd><dt>Malonyl-CoA: 21.8</dt>

<dd></dd><dt>Propionyl-CoA: 18.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 15.6</dt>

<dd></dd><dt>CHC-CoA: 10.8</dt>

<dd></dd><dt>Benzoyl-CoA: 9.4</dt>

<dd></dd><dt>Acetyl-CoA: 8.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 6.5</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">input.path1.gene726</span></strong>: X
<div class="collapser collapser-target-input-path1-gene726 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (219..660)</strong>: X
<div class="collapser collapser-target-input-path1-gene726 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene726-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALTVGGIVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DALAVGEVVK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000003.1 - Region 4 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-493"></div> <span class="help-tooltip" id="region_svg-help-493">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000003.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c4-svg" data-filename="JAAZTN010000003.1_r3c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,435,132 - 4,501,526 nt. (total: 66,395 nt)

<a class="cluster-rules-header" id="r3c4-rules-header" href="#r3c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c4-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-494"></div> <span class="help-tooltip" id="focus-panel-help-494">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c4-gene-table-details" data-name="r3c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c4-MIBiG-cluster-compare" data-name="r3c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c4-clusterblast" data-name="r3c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c4-knownclusterblast" data-name="r3c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c4-subclusterblast" data-name="r3c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c4-pfam-details" data-name="r3c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c4-tigrfam-details" data-name="r3c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-174"></div> <span class="help-tooltip" id="gt-help-tooltip-help-174">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-170"></div> <span class="help-tooltip" id="MIBiG-help-170">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c4"></div>
<div id="comparison-MIBiG-r3c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Other</td>

<td>itaconic acid</td>


<td>Ustilago maydis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>aspterric acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660: 0-2468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>albaflavenone</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911: 591-10299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>raimonol</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Alkaloid</td>

<td>huperzine A</td>


<td>Shiraia sp. slf14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>heptelidic acid</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>HEx-tc3 terpene</td>


<td>Botrytis cinerea T4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Other</td>
<td>itaconic acid</td>
<td>Ustilago maydis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>aspterric acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660: 0-2468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>albaflavenone</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911: 591-10299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>raimonol</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Alkaloid</td>
<td>huperzine A</td>
<td>Shiraia sp. slf14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>heptelidic acid</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>HEx-tc3 terpene</td>
<td>Botrytis cinerea T4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-171"></div> <span class="help-tooltip" id="cb-general-help-171">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c4_1.svg">
NW_006271970 (2031211-2136459): Cordyceps militaris CM01 unplaced genomic sca... (47% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_2.svg">
NT_165982 (1008305-1103578): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (20% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_3.svg">
NW_011371361 (1937664-2035052): Paracoccidioides brasiliensis Pb18 unplaced g... (23% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_4.svg">
NW_015440948 (109188-206314): Paracoccidioides lutzii Pb01 unplaced genomic s... (21% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_5.svg">
NW_017263587 (265177-438368): Mollisia scopiformis unplaced genomic scaffold ... (8% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_6.svg">
NC_030959 (4385741-4495530): Colletotrichum higginsianum IMI 349063 chromosom... (14% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_7.svg">
NW_017263589 (1285573-1392383): Mollisia scopiformis unplaced genomic scaffol... (6% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_8.svg">
NW_026250024 (1787317-1950532): Hypoxylon trugodes strain CBS 135444 unplaced... (6% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_9.svg">
NC_036437 (172443-252235): Aspergillus oryzae RIB40 DNA, chromosome 3 (6% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r3c4_10.svg">
NC_054693 (237013-316681): Aspergillus flavus NRRL3357 chromosome 3, complete... (6% of genes show similarity), NRPS-like,terpene
</option>

</select>
<button id= "clusterblast-r3c4-download">Download graphic</button>
</div>
<div id="clusterblast-r3c4-svg"></div>

</div>
</div>

<div class="body-details-section r3c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-172"></div> <span class="help-tooltip" id="cb-known-help-172">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-173"></div> <span class="help-tooltip" id="cb-sub-help-173">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-175"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-175">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-pfam-details-svg" data-filename="JAAZTN010000003.1_r3c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-176"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-176">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-tigrfam-details-svg" data-filename="JAAZTN010000003.1_r3c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r3c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000003.1 - Region 5 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-495"></div> <span class="help-tooltip" id="region_svg-help-495">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000003.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c5-svg" data-filename="JAAZTN010000003.1_r3c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,949,632 - 4,993,501 nt. (total: 43,870 nt)

<a class="cluster-rules-header" id="r3c5-rules-header" href="#r3c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c5-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-496"></div> <span class="help-tooltip" id="focus-panel-help-496">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c5-gene-table-details" data-name="r3c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c5-nrps_pks" data-name="r3c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c5-MIBiG-cluster-compare" data-name="r3c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c5-clusterblast" data-name="r3c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c5-knownclusterblast" data-name="r3c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c5-subclusterblast" data-name="r3c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c5-nrps-pks-bubbles" data-name="r3c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c5-pfam-details" data-name="r3c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c5-tigrfam-details" data-name="r3c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-185"></div> <span class="help-tooltip" id="gt-help-tooltip-help-185">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-177"></div> <span class="help-tooltip" id="nrps-domain-help-177">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-details-svg" data-filename="JAAZTN010000003.1_r3c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-178"></div> <span class="help-tooltip" id="MIBiG-help-178">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c5"></div>
<div id="comparison-MIBiG-r3c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.55, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>choline</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000016: 226-5727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000016/index.html#r1c1" target="_new">BGC0000016</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Other</td>

<td>amphotericin B</td>


<td>Streptomyces nodosus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP, Saccharide</td>

<td>minimycin</td>


<td>Streptomyces hygroscopicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002160: 880-28810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002160/index.html#r1c1" target="_new">BGC0002160</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>aspergillicin A, aspergillicin F, aspergillicin G</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.55, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>choline</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000016: 226-5727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000016/index.html#r1c1" target="_new">BGC0000016</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Other</td>
<td>amphotericin B</td>
<td>Streptomyces nodosus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP, Saccharide</td>
<td>minimycin</td>
<td>Streptomyces hygroscopicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002160: 880-28810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002160/index.html#r1c1" target="_new">BGC0002160</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>aspergillicin A, aspergillicin F, aspergillicin G</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-179"></div> <span class="help-tooltip" id="cb-general-help-179">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c5_1.svg">
NW_014013657 (238627-342899): Trichoderma virens Gv29-8 chromosome Unknown TR... (20% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_2.svg">
NW_006711169 (17423-81332): Trichoderma reesei QM6a unplaced genomic scaffold... (35% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c5_3.svg">
NW_025764707 (691343-755195): Emericellopsis atlantica unplaced genomic scaff... (30% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c5_4.svg">
NC_070532 (4966088-5118746): Fusarium keratoplasticum isolate Fu6.1 chromosom... (16% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_5.svg">
NC_070547 (4757164-4894085): Fusarium falciforme chromosome 4, complete sequence (17% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_6.svg">
NC_054390 (745744-925575): Drechmeria coniospora strain ARSEF 6962 chromosome... (10% of genes show similarity), NRPS,NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_7.svg">
NW_011942160 (555242-619088): Metarhizium robertsii ARSEF 23 MAA Scf 2, whole... (40% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c5_8.svg">
NC_035791 (1143313-1234197): Pochonia chlamydosporia 170 chromosome 2, whole ... (19% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_9.svg">
NW_014574716 (3095388-3159234): Metarhizium brunneum ARSEF 3297 chromosome Un... (37% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c5_10.svg">
NW_025408669 (7090957-7154869): Hirsutella rhossiliensis strain HR02 chromoso... (46% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r3c5-download">Download graphic</button>
</div>
<div id="clusterblast-r3c5-svg"></div>

</div>
</div>

<div class="body-details-section r3c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-180"></div> <span class="help-tooltip" id="cb-known-help-180">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c5-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c5_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c5_1.svg">
choline
</option>

</select>
<button id="knownclusterblast-r3c5-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c5-svg"></div>

</div>
</div>

<div class="body-details-section r3c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-181"></div> <span class="help-tooltip" id="cb-sub-help-181">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-184"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-184">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-domain-bubble-svg" data-filename="JAAZTN010000003.1_r3c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (4949631 - 4993501): single NRPS-like
</option>


</select>

<div class="details-svg" id="r3c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-186"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-186">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-pfam-details-svg" data-filename="JAAZTN010000003.1_r3c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-187"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-187">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-tigrfam-details-svg" data-filename="JAAZTN010000003.1_r3c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c5-nrps_pks_products" data-name="r3c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c5-nrps_pks_monomers" data-name="r3c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-182"></div> <span class="help-tooltip" id="nrps-structures-help-182">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 4949631 - 4993501:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c5-7" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c5-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-183"></div> <span class="help-tooltip" id="nrps-monomers-help-183">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1295</span></strong>: Phe
<div class="collapser collapser-target-input-path1-gene1295 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (13..418)</strong>: Phe
<div class="collapser collapser-target-input-path1-gene1295 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-input-path1-gene1295-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>phe (Phe)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--WLWNVEAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r4c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000004.1 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-497"></div> <span class="help-tooltip" id="region_svg-help-497">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000004.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c1-svg" data-filename="JAAZTN010000004.1_r4c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 327,623 - 372,983 nt. (total: 45,361 nt)

<a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-498"></div> <span class="help-tooltip" id="focus-panel-help-498">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c1-gene-table-details" data-name="r4c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c1-nrps-pks-bubbles" data-name="r4c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c1-tigrfam-details" data-name="r4c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-196"></div> <span class="help-tooltip" id="gt-help-tooltip-help-196">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-188"></div> <span class="help-tooltip" id="nrps-domain-help-188">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="JAAZTN010000004.1_r4c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-189"></div> <span class="help-tooltip" id="MIBiG-help-189">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c1"></div>
<div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002248: 5-21557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002248/index.html#r1c1" target="_new">BGC0002248</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene, NRP</td>

<td>flavunoidine</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002248: 5-21557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002248/index.html#r1c1" target="_new">BGC0002248</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, NRP</td>
<td>flavunoidine</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-190"></div> <span class="help-tooltip" id="cb-general-help-190">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c1_1.svg">
NW_025091266 (1102530-1174184): Aspergillus udagawae IFM 46973 DNA, scaffold ... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_2.svg">
NW_011942151 (1102940-1173579): Metarhizium robertsii ARSEF 23 MAA Scf 11, wh... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_3.svg">
NW_025091283 (359926-455746): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c1_4.svg">
NW_014574707 (383137-474713): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_5.svg">
NW_011942171 (5162432-5375378): Metarhizium robertsii ARSEF 23 MAA Scf 3, who... (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c1_6.svg">
NW_006271970 (4182342-4255710): Cordyceps militaris CM01 unplaced genomic sca... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_7.svg">
NC_054389 (10725158-10792532): Drechmeria coniospora strain ARSEF 6962 chromo... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_8.svg">
NW_025408671 (1651054-1734405): Hirsutella rhossiliensis strain HR02 chromoso... (13% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r4c1_9.svg">
NC_026474 (5505232-5623287): Fusarium graminearum PH-1 chromosome 1, whole ge... (6% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c1_10.svg">
NW_020291486 (59597-203320): Aspergillus aculeatinus CBS 121060 unplaced geno... (5% of genes show similarity), NRPS,fungal-RiPP-like,isocyanide
</option>

</select>
<button id= "clusterblast-r4c1-download">Download graphic</button>
</div>
<div id="clusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-191"></div> <span class="help-tooltip" id="cb-known-help-191">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-192"></div> <span class="help-tooltip" id="cb-sub-help-192">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-195"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-195">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-domain-bubble-svg" data-filename="JAAZTN010000004.1_r4c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (327622 - 372983): single NRPS
</option>


</select>

<div class="details-svg" id="r4c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-197"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-197">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="JAAZTN010000004.1_r4c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-198"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-198">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-tigrfam-details-svg" data-filename="JAAZTN010000004.1_r4c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c1-nrps_pks_monomers" data-name="r4c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-193"></div> <span class="help-tooltip" id="nrps-structures-help-193">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 327622 - 372983:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-194"></div> <span class="help-tooltip" id="nrps-monomers-help-194">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene69</span></strong>: X
<div class="collapser collapser-target-input-path1-gene69 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (195..594)</strong>: X
<div class="collapser collapser-target-input-path1-gene69 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene69-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVCDVGLPNK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (71% 8Å match)</dd>
<dd>Pro <span class="serif">DVMLVGAVNK</span> (56% 8Å match)</dd>
<dd>Ala <span class="serif">DVFFVGLVAK</span> (47% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000004.1 - Region 2 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-499"></div> <span class="help-tooltip" id="region_svg-help-499">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000004.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c2-svg" data-filename="JAAZTN010000004.1_r4c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,257,280 - 2,326,779 nt. (total: 69,500 nt)

<a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r4c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-500"></div> <span class="help-tooltip" id="focus-panel-help-500">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c2-gene-table-details" data-name="r4c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c2-nrps_pks" data-name="r4c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c2-clusterblast" data-name="r4c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c2-nrps-pks-bubbles" data-name="r4c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c2-pfam-details" data-name="r4c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c2-tigrfam-details" data-name="r4c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-207"></div> <span class="help-tooltip" id="gt-help-tooltip-help-207">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-199"></div> <span class="help-tooltip" id="nrps-domain-help-199">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-details-svg" data-filename="JAAZTN010000004.1_r4c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-200"></div> <span class="help-tooltip" id="MIBiG-help-200">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c2"></div>
<div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>





<td class="cc-heat-cell" title="0.75 (id:0.60, order:0.88, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.62, order:0.76, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.68</td>
<td>Polyketide, NRP</td>

<td>beauveriolide I, beauveriolide III</td>


<td>Cordyceps militaris CM01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002203: 0-33767">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002203/index.html#r1c1" target="_new">BGC0002203</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.75, order:0.61, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.89, order:0.57, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.65</td>
<td>NRP, Polyketide, Other</td>

<td>beauveriolide B, beauveriolide C, beauveriolide D</td>


<td>Beauveria bassiana ARSEF 2860</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.50</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001290: 0-37424">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001290/index.html#r1c1" target="_new">BGC0001290</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.50, order:0.58, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.52, order:0.64, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.44</td>
<td>NRP</td>

<td>emericellamide A, emericellamide B</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.41</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.40</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.46, order:0.57, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.46, order:0.57, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.40</td>
<td>Polyketide</td>

<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.38</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>


<td class="cc-heat-cell" title="0.75 (id:0.60, order:0.88, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
<td>0.75</td>
<td>Polyketide, NRP</td>
<td>beauveriolide I, beauveriolide III</td>
<td>Cordyceps militaris CM01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002203: 0-33767">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002203/index.html#r1c1" target="_new">BGC0002203</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.75, order:0.63, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP, Polyketide, Other</td>
<td>beauveriolide B, beauveriolide C, beauveriolide D</td>
<td>Beauveria bassiana ARSEF 2860</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001290: 0-37424">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001290/index.html#r1c1" target="_new">BGC0001290</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.50, order:0.58, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>emericellamide A, emericellamide B</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002199: 0-17425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002199/index.html#r1c1" target="_new">BGC0002199</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Alkaloid</td>
<td>nanangelenin A</td>
<td>Aspergillus nanangensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>serinocyclin A, serinocyclin B</td>
<td>Metarhizium robertsii</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-201"></div> <span class="help-tooltip" id="cb-general-help-201">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c2_1.svg">
NW_006271969 (4087046-4175862): Cordyceps militaris CM01 unplaced genomic sca... (23% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_2.svg">
NW_025091271 (18416-119910): Aspergillus udagawae IFM 46973 DNA, scaffold 8, ... (21% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_3.svg">
NC_066263 (4200452-4292984): Aspergillus nidulans FGSC A4 chromosome VII (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_4.svg">
NT_165979 (2272028-2354621): Chaetomium globosum CBS 148.51 scaffold 4 genomi... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_5.svg">
NC_038012 (6505506-6611069): Fusarium venenatum strain A3/5 genome assembly, ... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_6.svg">
NC_058399 (6406982-6512577): Fusarium poae strain DAOMC 252244 chromosome 1, ... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_7.svg">
NC_072354 (38624-176736): Talaromyces marneffei chromosome 7, complete sequence (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_8.svg">
NC_054859 (4418286-4502506): Aspergillus puulaauensis MK2 DNA, chromosome 3, ... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_9.svg">
NW_025408670 (875820-944244): Hirsutella rhossiliensis strain HR02 chromosome... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c2_10.svg">
NW_026250021 (502671-612769): Hypoxylon trugodes strain CBS 135444 unplaced g... (6% of genes show similarity), T1PKS,T3PKS
</option>

</select>
<button id= "clusterblast-r4c2-download">Download graphic</button>
</div>
<div id="clusterblast-r4c2-svg"></div>

</div>
</div>

<div class="body-details-section r4c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-202"></div> <span class="help-tooltip" id="cb-known-help-202">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c2_1.svg">
beauveriolide B/beauveriolide C/beauveriolide D
</option>

<option value = "svg/knownclusterblast_r4c2_2.svg">
beauveriolide I/beauveriolide III
</option>

<option value = "svg/knownclusterblast_r4c2_3.svg">
emericellamide A/emericellamide B
</option>

</select>
<button id="knownclusterblast-r4c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c2-svg"></div>

</div>
</div>

<div class="body-details-section r4c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-203"></div> <span class="help-tooltip" id="cb-sub-help-203">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-206"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-206">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-domain-bubble-svg" data-filename="JAAZTN010000004.1_r4c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (2257279 - 2326779): neighbouring NRPS-T1PKS
</option>



<option value="CC3">
Candidate 3 (2257279 - 2311718): single NRPS
</option>



<option value="CC4">
Candidate 4 (2279132 - 2326779): single T1PKS
</option>


</select>

<div class="details-svg" id="r4c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-208"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-208">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-pfam-details-svg" data-filename="JAAZTN010000004.1_r4c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-209"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-209">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-tigrfam-details-svg" data-filename="JAAZTN010000004.1_r4c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c2-nrps_pks_products" data-name="r4c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c2-nrps_pks_monomers" data-name="r4c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-204"></div> <span class="help-tooltip" id="nrps-structures-help-204">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 2257279 - 2326779:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c2-2" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe - D-Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,d-ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,d-ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 2257279 - 2311718:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c2-3" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe - D-Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,d-ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,d-ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c2-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 2279132 - 2326779:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c2-4" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe - D-Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,d-ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,d-ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c2-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-205"></div> <span class="help-tooltip" id="nrps-monomers-help-205">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene632</span></strong>: Phe - Ile
<div class="collapser collapser-target-input-path1-gene632 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,ile*" target="_new">relaxed</a>
</dd>
<br>







<dd><strong>AMP-binding (1556..1817)</strong>: Phe
<div class="collapser collapser-target-input-path1-gene632 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-input-path1-gene632-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPTIVGAIIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DPTIVGAIIK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2526..2919)</strong>: Ile
<div class="collapser collapser-target-input-path1-gene632 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-input-path1-gene632-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMHMMGVMIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DMHMMGVMIK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">input.path1.gene636</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene636 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (474..794)</strong>: pk
<div class="collapser collapser-target-input-path1-gene636 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene636-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
<dd>inactive: 54.2%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-input-path1-gene636-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 69.7</dt>

<dd></dd><dt>Methylmalonyl-CoA: 64.8</dt>

<dd></dd><dt>Malonyl-CoA: 52.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 41.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 29.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 29.3</dt>

<dd></dd><dt>Propionyl-CoA: 24.6</dt>

<dd></dd><dt>Benzoyl-CoA: 18.3</dt>

<dd></dd><dt>fatty_acid: 17.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 16.4</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 15.1</dt>

<dd></dd><dt>inactive: 11.9</dt>

<dd></dd><dt>CHC-CoA: 10.7</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000004.1 - Region 3 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-501"></div> <span class="help-tooltip" id="region_svg-help-501">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000004.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c3-svg" data-filename="JAAZTN010000004.1_r4c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,400,244 - 2,447,344 nt. (total: 47,101 nt)

<a class="cluster-rules-header" id="r4c3-rules-header" href="#r4c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r4c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-502"></div> <span class="help-tooltip" id="focus-panel-help-502">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c3-gene-table-details" data-name="r4c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c3-nrps_pks" data-name="r4c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c3-MIBiG-cluster-compare" data-name="r4c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c3-clusterblast" data-name="r4c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c3-knownclusterblast" data-name="r4c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c3-subclusterblast" data-name="r4c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c3-nrps-pks-bubbles" data-name="r4c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c3-pfam-details" data-name="r4c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r4c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-218"></div> <span class="help-tooltip" id="gt-help-tooltip-help-218">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-210"></div> <span class="help-tooltip" id="nrps-domain-help-210">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-details-svg" data-filename="JAAZTN010000004.1_r4c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-211"></div> <span class="help-tooltip" id="MIBiG-help-211">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c3"></div>
<div id="comparison-MIBiG-r4c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>





<td class="cc-heat-cell" title="0.80 (id:0.72, order:0.72, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>


<td>0.53</td>
<td>Polyketide</td>

<td>oosporein</td>


<td>Beauveria bassiana ARSEF 2860</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>cichorine</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>naphthalene</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Nodulisporium sp. ATCC74245</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>


<td class="cc-heat-cell" title="0.80 (id:0.72, order:0.72, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
<td>0.80</td>
<td>Polyketide</td>
<td>oosporein</td>
<td>Beauveria bassiana ARSEF 2860</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>cichorine</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>naphthalene</td>
<td>Daldinia eschscholzii IFB-TL01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Nodulisporium sp. ATCC74245</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-212"></div> <span class="help-tooltip" id="cb-general-help-212">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c3_1.svg">
NW_025091270 (468524-575944): Aspergillus udagawae IFM 46973 DNA, scaffold 7,... (8% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r4c3_2.svg">
NW_025091287 (1795699-1862693): Aspergillus pseudoviridinutans IFM 55266 DNA,... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_3.svg">
NC_054693 (2311669-2378695): Aspergillus flavus NRRL3357 chromosome 3, comple... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_4.svg">
NC_036437 (2261903-2328906): Aspergillus oryzae RIB40 DNA, chromosome 3 (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_5.svg">
NW_025091283 (4319746-4671991): Aspergillus pseudoviridinutans IFM 55266 DNA,... (2% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r4c3_6.svg">
NW_025953250 (1567429-1634321): Annulohypoxylon maeteangense strain CBS 12383... (10% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c3_7.svg">
NW_025953809 (474281-541195): Annulohypoxylon truncatum strain CBS 140777 unp... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_8.svg">
NZ_AP018288 (1487718-1540025): Nostoc sp. NIES-4103 DNA, nearly complete genome (10% of genes show similarity), NRPS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r4c3_9.svg">
NW_025091230 (495798-623533): Aspergillus viridinutans IFM 47045 DNA, scaffol... (5% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r4c3_10.svg">
NZ_CP022987 (3398338-3464677): Pusillimonas thiosulfatoxidans strain YE3 chro... (4% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r4c3-download">Download graphic</button>
</div>
<div id="clusterblast-r4c3-svg"></div>

</div>
</div>

<div class="body-details-section r4c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-213"></div> <span class="help-tooltip" id="cb-known-help-213">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c3_1.svg">
oosporein
</option>

</select>
<button id="knownclusterblast-r4c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c3-svg"></div>

</div>
</div>

<div class="body-details-section r4c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-214"></div> <span class="help-tooltip" id="cb-sub-help-214">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-217"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-217">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-domain-bubble-svg" data-filename="JAAZTN010000004.1_r4c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (2400243 - 2447344): single T1PKS
</option>


</select>

<div class="details-svg" id="r4c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-219"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-219">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-pfam-details-svg" data-filename="JAAZTN010000004.1_r4c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c3-nrps_pks_products" data-name="r4c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c3-nrps_pks_monomers" data-name="r4c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-215"></div> <span class="help-tooltip" id="nrps-structures-help-215">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 2400243 - 2447344:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c3-5" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c3-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-216"></div> <span class="help-tooltip" id="nrps-monomers-help-216">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene670</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene670 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (567..867)</strong>: mal
<div class="collapser collapser-target-input-path1-gene670 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene670-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>inactive: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene670-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 104.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 48.7</dt>

<dd></dd><dt>inactive: 47.9</dt>

<dd></dd><dt>Methylmalonyl-CoA: 44.3</dt>

<dd></dd><dt>fatty_acid: 30.7</dt>

<dd></dd><dt>Propionyl-CoA: 23.7</dt>

<dd></dd><dt>CHC-CoA: 19.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 17.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 15.6</dt>

<dd></dd><dt>Benzoyl-CoA: 14.3</dt>

<dd></dd><dt>Acetyl-CoA: 14.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 9.9</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 5.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000004.1 - Region 4 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-503"></div> <span class="help-tooltip" id="region_svg-help-503">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000004.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c4-svg" data-filename="JAAZTN010000004.1_r4c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,746,561 - 2,790,231 nt. (total: 43,671 nt)

<a class="cluster-rules-header" id="r4c4-rules-header" href="#r4c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-504"></div> <span class="help-tooltip" id="focus-panel-help-504">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c4-gene-table-details" data-name="r4c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c4-nrps_pks" data-name="r4c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c4-MIBiG-cluster-compare" data-name="r4c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c4-clusterblast" data-name="r4c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c4-knownclusterblast" data-name="r4c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c4-subclusterblast" data-name="r4c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c4-pfam-details" data-name="r4c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c4-tigrfam-details" data-name="r4c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-226"></div> <span class="help-tooltip" id="gt-help-tooltip-help-226">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-220"></div> <span class="help-tooltip" id="nrps-domain-help-220">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-details-svg" data-filename="JAAZTN010000004.1_r4c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-221"></div> <span class="help-tooltip" id="MIBiG-help-221">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c4"></div>
<div id="comparison-MIBiG-r4c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.55, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>zealexin B2, zealexin C2</td>


<td>Zea mays</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>β-Diketones</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>hydroxycinnamoyltyramine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>viguiepinol</td>


<td>Streptomyces sp. KO-3988</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.55, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Terpene</td>
<td>zealexin B2, zealexin C2</td>
<td>Zea mays</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>β-Diketones</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>hydroxycinnamoyltyramine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Polyketide</td>
<td>viguiepinol</td>
<td>Streptomyces sp. KO-3988</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-222"></div> <span class="help-tooltip" id="cb-general-help-222">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c4_1.svg">
NW_006271972 (846900-910624): Cordyceps militaris CM01 unplaced genomic scaff... (40% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_2.svg">
NW_025408674 (2031871-2095823): Hirsutella rhossiliensis strain HR02 chromoso... (30% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_3.svg">
NC_070531 (1081663-1145343): Fusarium keratoplasticum isolate Fu6.1 chromosom... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_4.svg">
NC_070546 (1097748-1161429): Fusarium falciforme chromosome 3, complete sequence (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_5.svg">
NC_036627 (1441489-1505176): Fusarium fujikuroi IMI 58289 draft genome, chrom... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_6.svg">
NW_017263587 (265177-438368): Mollisia scopiformis unplaced genomic scaffold ... (4% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c4_7.svg">
NC_072845 (1566921-1630612): Fusarium oxysporum Fo47 chromosome VI, complete ... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_8.svg">
NC_058400 (1372797-1436602): Fusarium poae strain DAOMC 252244 chromosome 2, ... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_9.svg">
NC_058392 (2751608-2815071): Fusarium musae strain F31 chromosome 6, whole ge... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_10.svg">
NC_038013 (1254253-1318050): Fusarium venenatum strain A3/5 genome assembly, ... (10% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r4c4-download">Download graphic</button>
</div>
<div id="clusterblast-r4c4-svg"></div>

</div>
</div>

<div class="body-details-section r4c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-223"></div> <span class="help-tooltip" id="cb-known-help-223">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-224"></div> <span class="help-tooltip" id="cb-sub-help-224">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-227"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-227">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-pfam-details-svg" data-filename="JAAZTN010000004.1_r4c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-228"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-228">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-tigrfam-details-svg" data-filename="JAAZTN010000004.1_r4c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c4-nrps_pks_monomers" data-name="r4c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r4c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-225"></div> <span class="help-tooltip" id="nrps-monomers-help-225">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene771</span></strong>: X
<div class="collapser collapser-target-input-path1-gene771 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (45..473)</strong>: X
<div class="collapser collapser-target-input-path1-gene771 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene771-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">IFNSTAAPLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DFESTAAVYK</span> (29% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r4c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000004.1 - Region 5 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-505"></div> <span class="help-tooltip" id="region_svg-help-505">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000004.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c5-svg" data-filename="JAAZTN010000004.1_r4c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,734,693 - 3,796,740 nt. (total: 62,048 nt)

<a class="cluster-rules-header" id="r4c5-rules-header" href="#r4c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c5-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-506"></div> <span class="help-tooltip" id="focus-panel-help-506">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c5-gene-table-details" data-name="r4c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c5-MIBiG-cluster-compare" data-name="r4c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c5-clusterblast" data-name="r4c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c5-knownclusterblast" data-name="r4c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c5-subclusterblast" data-name="r4c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c5-pfam-details" data-name="r4c5-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r4c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-233"></div> <span class="help-tooltip" id="gt-help-tooltip-help-233">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-229"></div> <span class="help-tooltip" id="MIBiG-help-229">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c5"></div>
<div id="comparison-MIBiG-r4c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000048: 0-52827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000048/index.html#r1c1" target="_new">BGC0000048</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.45, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>dothistromin</td>


<td>Dothistroma septosporum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000048: 0-52827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000048/index.html#r1c1" target="_new">BGC0000048</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.45, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>Polyketide</td>
<td>dothistromin</td>
<td>Dothistroma septosporum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-230"></div> <span class="help-tooltip" id="cb-general-help-230">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c5_1.svg">
NW_024467325 (2734-93761): Aspergillus ruber CBS 135680 unplaced genomic scaf... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_2.svg">
NW_006917102 (1579939-1692472): Pestalotiopsis fici W106-1 unplaced genomic s... (5% of genes show similarity), fungal-RiPP-like,indole
</option>

<option value = "svg/clusterblast_r4c5_3.svg">
NC_057369 (2300771-2391818): Aspergillus chevalieri M1 DNA, chromosome 8, nea... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_4.svg">
NC_018214 (92350-159292): Zymoseptoria tritici IPO323 chromosome 5, whole gen... (9% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r4c5-download">Download graphic</button>
</div>
<div id="clusterblast-r4c5-svg"></div>

</div>
</div>

<div class="body-details-section r4c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-231"></div> <span class="help-tooltip" id="cb-known-help-231">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-232"></div> <span class="help-tooltip" id="cb-sub-help-232">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-234"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-234">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c5-pfam-details-svg" data-filename="JAAZTN010000004.1_r4c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r4c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000004.1 - Region 6 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-507"></div> <span class="help-tooltip" id="region_svg-help-507">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000004.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c6-svg" data-filename="JAAZTN010000004.1_r4c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,245,240 - 4,305,988 nt. (total: 60,749 nt)

<a class="cluster-rules-header" id="r4c6-rules-header" href="#r4c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c6-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-508"></div> <span class="help-tooltip" id="focus-panel-help-508">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c6-gene-table-details" data-name="r4c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c6-MIBiG-cluster-compare" data-name="r4c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c6-clusterblast" data-name="r4c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c6-knownclusterblast" data-name="r4c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c6-subclusterblast" data-name="r4c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c6-pfam-details" data-name="r4c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c6-tigrfam-details" data-name="r4c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-239"></div> <span class="help-tooltip" id="gt-help-tooltip-help-239">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-235"></div> <span class="help-tooltip" id="MIBiG-help-235">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c6"></div>
<div id="comparison-MIBiG-r4c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>betaestacin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>AK-toxin I</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>viguiepinol</td>


<td>Streptomyces sp. KO-3988</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>AK-toxin I</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>viguiepinol</td>
<td>Streptomyces sp. KO-3988</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-236"></div> <span class="help-tooltip" id="cb-general-help-236">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c6_1.svg">
NW_006271973 (1574268-1668763): Cordyceps militaris CM01 unplaced genomic sca... (29% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c6_2.svg">
NW_025408667 (6621596-6715308): Hirsutella rhossiliensis strain HR02 chromoso... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c6_3.svg">
NW_014013666 (749888-843435): Trichoderma virens Gv29-8 chromosome Unknown TR... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c6_4.svg">
NW_020291609 (269731-414026): Aspergillus piperis CBS 112811 unplaced genomic... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r4c6_5.svg">
NW_022987222 (405428-496756): Trematosphaeria pertusa strain CBS 122368 unpla... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c6_6.svg">
NW_020291486 (59597-203320): Aspergillus aculeatinus CBS 121060 unplaced geno... (3% of genes show similarity), NRPS,fungal-RiPP-like,isocyanide
</option>

<option value = "svg/clusterblast_r4c6_7.svg">
NW_007360994 (690217-788751): Glarea lozoyensis ATCC 20868 chromosome Unknown... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c6_8.svg">
NC_030958 (1139459-1267774): Colletotrichum higginsianum IMI 349063 chromosom... (4% of genes show similarity), T1PKS,fungal-RiPP-like,indole
</option>

<option value = "svg/clusterblast_r4c6_9.svg">
NW_024467835 (3914759-4005529): Aspergillus wentii DTO 134E9 unplaced genomic... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c6_10.svg">
NW_024467861 (1263516-1354336): Aspergillus sydowii CBS 593.65 unplaced genom... (4% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r4c6-download">Download graphic</button>
</div>
<div id="clusterblast-r4c6-svg"></div>

</div>
</div>

<div class="body-details-section r4c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-237"></div> <span class="help-tooltip" id="cb-known-help-237">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-238"></div> <span class="help-tooltip" id="cb-sub-help-238">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-240"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-240">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c6-pfam-details-svg" data-filename="JAAZTN010000004.1_r4c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-241"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-241">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c6-tigrfam-details-svg" data-filename="JAAZTN010000004.1_r4c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r4c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000004.1 - Region 7 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-509"></div> <span class="help-tooltip" id="region_svg-help-509">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000004.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c7-svg" data-filename="JAAZTN010000004.1_r4c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,427,358 - 4,489,081 nt. (total: 61,724 nt)

<a class="cluster-rules-header" id="r4c7-rules-header" href="#r4c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c7-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-510"></div> <span class="help-tooltip" id="focus-panel-help-510">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c7-gene-table-details" data-name="r4c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c7-MIBiG-cluster-compare" data-name="r4c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c7-clusterblast" data-name="r4c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c7-knownclusterblast" data-name="r4c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c7-subclusterblast" data-name="r4c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c7-pfam-details" data-name="r4c7-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c7-tigrfam-details" data-name="r4c7-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-246"></div> <span class="help-tooltip" id="gt-help-tooltip-help-246">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-242"></div> <span class="help-tooltip" id="MIBiG-help-242">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c7"></div>
<div id="comparison-MIBiG-r4c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>isoindolinone</td>


<td>Stachybotrys chlorohalonata IBT 40285</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001964: 2-41089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001964/index.html#r1c1" target="_new">BGC0001964</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.47, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>alkylpyrone-407, alkylpyrone-393</td>


<td>Cystobacterineae bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001902: 871-37015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001902/index.html#r1c1" target="_new">BGC0001902</a></td>





<td class="cc-heat-cell" title="0.15 (id:0.42, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>


<td>0.03</td>
<td>NRP, Polyketide</td>

<td>bengamide</td>


<td>Myxococcus virescens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>isoindolinone</td>
<td>Stachybotrys chlorohalonata IBT 40285</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001964: 2-41089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001964/index.html#r1c1" target="_new">BGC0001964</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.47, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>Polyketide</td>
<td>alkylpyrone-407, alkylpyrone-393</td>
<td>Cystobacterineae bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001902: 871-37015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001902/index.html#r1c1" target="_new">BGC0001902</a></td>


<td class="cc-heat-cell" title="0.15 (id:0.42, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
<td>0.15</td>
<td>NRP, Polyketide</td>
<td>bengamide</td>
<td>Myxococcus virescens</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-243"></div> <span class="help-tooltip" id="cb-general-help-243">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c7_1.svg">
NW_016017392 (82602-175191): Sphaerulina musiva SO2202 unplaced genomic scaff... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c7_2.svg">
NC_026475 (3314467-3405418): Fusarium graminearum PH-1 chromosome 2, whole ge... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c7_3.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (3% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c7_4.svg">
NW_006917093 (3054358-3117993): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c7_5.svg">
NW_024468937 (5549232-5621994): Cucurbitaria berberidis CBS 394.84 unplaced g... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c7_6.svg">
NW_013550615 (4368823-4474200): Fonsecaea pedrosoi CBS 271.37 unplaced genomi... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c7_7.svg">
NW_015971635 (1692172-1793674): Fonsecaea multimorphosa CBS 102226 unplaced g... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c7_8.svg">
NW_025764728 (301721-365002): Emericellopsis atlantica unplaced genomic scaff... (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c7_9.svg">
NW_022156716 (3888118-3951114): Pyricularia grisea strain NI907 chromosome Un... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c7_10.svg">
NC_066264 (3194691-3262661): Aspergillus nidulans FGSC A4 chromosome VIII (7% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r4c7-download">Download graphic</button>
</div>
<div id="clusterblast-r4c7-svg"></div>

</div>
</div>

<div class="body-details-section r4c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-244"></div> <span class="help-tooltip" id="cb-known-help-244">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-245"></div> <span class="help-tooltip" id="cb-sub-help-245">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-247"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-247">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c7-pfam-details-svg" data-filename="JAAZTN010000004.1_r4c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c7-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-248"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-248">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c7-tigrfam-details-svg" data-filename="JAAZTN010000004.1_r4c7_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c7-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r5c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-511"></div> <span class="help-tooltip" id="region_svg-help-511">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c1-svg" data-filename="JAAZTN010000005.1_r5c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 317,504 - 364,300 nt. (total: 46,797 nt)

<a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-512"></div> <span class="help-tooltip" id="focus-panel-help-512">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c1-gene-table-details" data-name="r5c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c1-nrps_pks" data-name="r5c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c1-nrps-pks-bubbles" data-name="r5c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c1-tigrfam-details" data-name="r5c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-257"></div> <span class="help-tooltip" id="gt-help-tooltip-help-257">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-249"></div> <span class="help-tooltip" id="nrps-domain-help-249">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-details-svg" data-filename="JAAZTN010000005.1_r5c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-250"></div> <span class="help-tooltip" id="MIBiG-help-250">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c1"></div>
<div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Nodulisporium sp. ATCC74245</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>grayanic acid</td>


<td>Cladonia grayi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000684: 369-21746">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000684/index.html#r1c1" target="_new">BGC0000684</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>asperthecin</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>emodin</td>


<td>Escovopsis weberi</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Nodulisporium sp. ATCC74245</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>grayanic acid</td>
<td>Cladonia grayi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000684: 369-21746">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000684/index.html#r1c1" target="_new">BGC0000684</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>asperthecin</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>emodin</td>
<td>Escovopsis weberi</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-251"></div> <span class="help-tooltip" id="cb-general-help-251">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c1_1.svg">
NW_006271971 (91199-157949): Cordyceps militaris CM01 unplaced genomic scaffo... (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_2.svg">
NW_025408670 (3613020-3679633): Hirsutella rhossiliensis strain HR02 chromoso... (23% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r5c1_3.svg">
NW_017263595 (627002-693843): Mollisia scopiformis unplaced genomic scaffold ... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_4.svg">
NC_049564 (4920448-4987078): Talaromyces rugulosus chromosome IV, complete se... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_5.svg">
NC_072350 (1386622-1453217): Talaromyces marneffei chromosome 3, complete seq... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_6.svg">
NW_017971432 (1168267-1234841): Talaromyces atroroseus strain IBT 11181 chrom... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_7.svg">
NC_063022 (1108505-1175063): Fulvia fulva chromosome 11, complete sequence (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_8.svg">
NZ_CP031455 (7073563-7290094): Streptomyces olivoreticuli subsp. olivoreticul... (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_9.svg">
NW_022985010 (1623664-1725341): Dothidotthia symphoricarpi CBS 119687 unplace... (9% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r5c1_10.svg">
NZ_JABUMT010000002 (1074426-1191054): Myxococcus sp. CA056 NODE 2 length 1734... (8% of genes show similarity), NRPS,T1PKS,T3PKS
</option>

</select>
<button id= "clusterblast-r5c1-download">Download graphic</button>
</div>
<div id="clusterblast-r5c1-svg"></div>

</div>
</div>

<div class="body-details-section r5c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-252"></div> <span class="help-tooltip" id="cb-known-help-252">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r5c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r5c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r5c1_1.svg">
YWA1
</option>

</select>
<button id="knownclusterblast-r5c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r5c1-svg"></div>

</div>
</div>

<div class="body-details-section r5c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-253"></div> <span class="help-tooltip" id="cb-sub-help-253">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-256"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-256">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-domain-bubble-svg" data-filename="JAAZTN010000005.1_r5c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (317503 - 364300): single T1PKS
</option>


</select>

<div class="details-svg" id="r5c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-258"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-258">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-259"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-259">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-tigrfam-details-svg" data-filename="JAAZTN010000005.1_r5c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c1-nrps_pks_products" data-name="r5c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c1-nrps_pks_monomers" data-name="r5c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-254"></div> <span class="help-tooltip" id="nrps-structures-help-254">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 317503 - 364300:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-255"></div> <span class="help-tooltip" id="nrps-monomers-help-255">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene87</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene87 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (654..950)</strong>: mal
<div class="collapser collapser-target-input-path1-gene87 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene87-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene87-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 75.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 45.0</dt>

<dd></dd><dt>inactive: 39.5</dt>

<dd></dd><dt>Methylmalonyl-CoA: 38.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 17.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 12.2</dt>

<dd></dd><dt>Propionyl-CoA: 12.0</dt>

<dd></dd><dt>Benzoyl-CoA: 11.9</dt>

<dd></dd><dt>fatty_acid: 10.5</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 9.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-513"></div> <span class="help-tooltip" id="region_svg-help-513">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c2-svg" data-filename="JAAZTN010000005.1_r5c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 531,129 - 554,392 nt. (total: 23,264 nt)

<a class="cluster-rules-header" id="r5c2-rules-header" href="#r5c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r5c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-514"></div> <span class="help-tooltip" id="focus-panel-help-514">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c2-gene-table-details" data-name="r5c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c2-MIBiG-cluster-compare" data-name="r5c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c2-clusterblast" data-name="r5c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c2-knownclusterblast" data-name="r5c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c2-subclusterblast" data-name="r5c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r5c2-pfam-details" data-name="r5c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c2-tigrfam-details" data-name="r5c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-264"></div> <span class="help-tooltip" id="gt-help-tooltip-help-264">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-260"></div> <span class="help-tooltip" id="MIBiG-help-260">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c2"></div>
<div id="comparison-MIBiG-r5c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>marneral</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001248: 1352-4192">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001248/index.html#r1c1" target="_new">BGC0001248</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>clavaric acid</td>


<td>Hypholoma sublateritium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>lupeol</td>


<td>Lotus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Other</td>

<td>sarasinoside</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>thalianol, thaliandiol</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000686: 0-16277">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000686/index.html#r1c1" target="_new">BGC0000686</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.46, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>helvolic acid</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314: 1711-56297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>tirucalla</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002402: 203-37725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002402/index.html#r1c1" target="_new">BGC0002402</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>yossosides</td>


<td>Spinacia oleracea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002722: 3-337945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002722/index.html#r1c1" target="_new">BGC0002722</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Alkaloid, Terpene, Saccharide</td>

<td>α-chaconine, α-solanine</td>


<td>Solanum tuberosum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000273: 50-42876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000273/index.html#r1c1" target="_new">BGC0000273</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.46, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Polyketide, Saccharide</td>

<td>steffimycin D</td>


<td>Streptomyces steffisburgensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001248: 1352-4192">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001248/index.html#r1c1" target="_new">BGC0001248</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>clavaric acid</td>
<td>Hypholoma sublateritium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>lupeol</td>
<td>Lotus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>sarasinoside</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>thalianol, thaliandiol</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000686: 0-16277">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000686/index.html#r1c1" target="_new">BGC0000686</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.46, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Terpene</td>
<td>helvolic acid</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314: 1711-56297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Terpene</td>
<td>tirucalla</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002402: 203-37725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002402/index.html#r1c1" target="_new">BGC0002402</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Terpene</td>
<td>yossosides</td>
<td>Spinacia oleracea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002722: 3-337945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002722/index.html#r1c1" target="_new">BGC0002722</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Alkaloid, Terpene, Saccharide</td>
<td>α-chaconine, α-solanine</td>
<td>Solanum tuberosum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000273: 50-42876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000273/index.html#r1c1" target="_new">BGC0000273</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.46, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Polyketide, Saccharide</td>
<td>steffimycin D</td>
<td>Streptomyces steffisburgensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-261"></div> <span class="help-tooltip" id="cb-general-help-261">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c2_1.svg">
NW_006271977 (650506-682810): Cordyceps militaris CM01 unplaced genomic scaff... (64% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r5c2_2.svg">
NC_043740 (4056696-4089123): Pyricularia pennisetigena strain Br36 chromosome... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r5c2_3.svg">
NW_014574706 (185039-217400): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (53% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r5c2_4.svg">
NC_035794 (3381868-3414218): Pochonia chlamydosporia 170 chromosome 5, whole ... (77% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r5c2_5.svg">
NW_014013668 (153970-222143): Trichoderma virens Gv29-8 chromosome Unknown TR... (31% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c2_6.svg">
NC_054389 (10607680-10640044): Drechmeria coniospora strain ARSEF 6962 chromo... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r5c2_7.svg">
NW_025408667 (5243071-5275419): Hirsutella rhossiliensis strain HR02 chromoso... (77% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r5c2_8.svg">
NW_006711154 (931670-999559): Trichoderma reesei QM6a unplaced genomic scaffo... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c2_9.svg">
NC_057320 (3125192-3157585): Ustilaginoidea virens chromosome 5, complete seq... (54% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r5c2_10.svg">
NW_023336201 (447629-481710): Geosmithia morbida isolate 1262 chromosome Unkn... (38% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r5c2-download">Download graphic</button>
</div>
<div id="clusterblast-r5c2-svg"></div>

</div>
</div>

<div class="body-details-section r5c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-262"></div> <span class="help-tooltip" id="cb-known-help-262">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r5c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r5c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r5c2_1.svg">
clavaric acid
</option>

</select>
<button id="knownclusterblast-r5c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r5c2-svg"></div>

</div>
</div>

<div class="body-details-section r5c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-263"></div> <span class="help-tooltip" id="cb-sub-help-263">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r5c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-265"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-265">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-266"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-266">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-tigrfam-details-svg" data-filename="JAAZTN010000005.1_r5c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r5c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-515"></div> <span class="help-tooltip" id="region_svg-help-515">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c3-svg" data-filename="JAAZTN010000005.1_r5c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,316,394 - 1,365,962 nt. (total: 49,569 nt)

<a class="cluster-rules-header" id="r5c3-rules-header" href="#r5c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-516"></div> <span class="help-tooltip" id="focus-panel-help-516">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c3-gene-table-details" data-name="r5c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c3-nrps_pks" data-name="r5c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c3-MIBiG-cluster-compare" data-name="r5c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c3-clusterblast" data-name="r5c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c3-knownclusterblast" data-name="r5c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c3-subclusterblast" data-name="r5c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c3-nrps-pks-bubbles" data-name="r5c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c3-pfam-details" data-name="r5c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c3-tigrfam-details" data-name="r5c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-275"></div> <span class="help-tooltip" id="gt-help-tooltip-help-275">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-267"></div> <span class="help-tooltip" id="nrps-domain-help-267">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-details-svg" data-filename="JAAZTN010000005.1_r5c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-268"></div> <span class="help-tooltip" id="MIBiG-help-268">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c3"></div>
<div id="comparison-MIBiG-r5c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.55, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.53, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000312: 0-41205">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000312/index.html#r1c1" target="_new">BGC0000312</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.68, order:0.42, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>bassianolide</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>AbT1</td>


<td>Aureobasidium pullulans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.55, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.53, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000312: 0-41205">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000312/index.html#r1c1" target="_new">BGC0000312</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.68, order:0.42, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>bassianolide</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>AbT1</td>
<td>Aureobasidium pullulans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-269"></div> <span class="help-tooltip" id="cb-general-help-269">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c3_1.svg">
NC_054696 (832977-904788): Aspergillus flavus NRRL3357 chromosome 6, complete... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c3_2.svg">
NZ_AP026074 (37933-149202): Streptomyces nigrescens strain HEK616 plasmid SNP1 (4% of genes show similarity), NRPS,NRPS-like,RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_3.svg">
NZ_JABFMP010000013 (49687-114322): Pseudomonas sp. C1C7 NODE 13 length 158710... (4% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r5c3_4.svg">
NZ_CP095481 (5894087-6205936): Amycolatopsis sp. WQ 127309 chromosome, comple... (1% of genes show similarity), NRPS,NRPS-like,T1PKS,nucleoside
</option>

<option value = "svg/clusterblast_r5c3_5.svg">
NZ_QLLT01000012 (11741-117910): Kitasatospora sp. SolWspMP-SS2h K353DRAFT sca... (2% of genes show similarity), NRPS,T1PKS,T3PKS,terpene
</option>

<option value = "svg/clusterblast_r5c3_6.svg">
NZ_JALAPV010000001 (714146-791926): Bacillus mojavensis strain TP8F7aA-2 cont... (5% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r5c3_7.svg">
NZ_CP051464 (1869836-1947616): Bacillus mojavensis strain UCMB5075 chromosome... (5% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r5c3_8.svg">
NZ_CP047140 (110451-286764): Streptomyces sp. Tu 2975 chromosome, complete ge... (1% of genes show similarity), NRPS,PKS-like,T2PKS,ectoine,transAT-PKS
</option>

<option value = "svg/clusterblast_r5c3_9.svg">
NW_007361001 (264244-388551): Glarea lozoyensis ATCC 20868 chromosome Unknown... (5% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_10.svg">
NW_024467525 (3144728-3217090): Aspergillus versicolor CBS 583.65 unplaced ge... (13% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r5c3-download">Download graphic</button>
</div>
<div id="clusterblast-r5c3-svg"></div>

</div>
</div>

<div class="body-details-section r5c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-270"></div> <span class="help-tooltip" id="cb-known-help-270">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r5c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r5c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r5c3_1.svg">
bassianolide
</option>

<option value = "svg/knownclusterblast_r5c3_2.svg">
enniatin
</option>

<option value = "svg/knownclusterblast_r5c3_3.svg">
AbT1
</option>

</select>
<button id="knownclusterblast-r5c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r5c3-svg"></div>

</div>
</div>

<div class="body-details-section r5c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-271"></div> <span class="help-tooltip" id="cb-sub-help-271">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-274"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-274">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-domain-bubble-svg" data-filename="JAAZTN010000005.1_r5c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (1316393 - 1365962): single NRPS
</option>


</select>

<div class="details-svg" id="r5c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-276"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-276">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-277"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-277">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-tigrfam-details-svg" data-filename="JAAZTN010000005.1_r5c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c3-nrps_pks_products" data-name="r5c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c3-nrps_pks_monomers" data-name="r5c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-272"></div> <span class="help-tooltip" id="nrps-structures-help-272">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 1316393 - 1365962:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c3-3" class="smiles-canvas" data-smiles="OC(C(C)C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(D-Hiv - NMe-Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-hiv,nme-leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-hiv*,nme-leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="OC(C(C)C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-273"></div> <span class="help-tooltip" id="nrps-monomers-help-273">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene359</span></strong>: D-Hiv - Leu
<div class="collapser collapser-target-input-path1-gene359 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-hiv,leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-hiv*,leu*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (348..744)</strong>: D-Hiv
<div class="collapser collapser-target-input-path1-gene359 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  D-Hiv
<div class="collapser collapser-target-input-path1-gene359-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GALMVVGSIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>D-Hiv <span class="serif">GALMVVGSIK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1423..1830)</strong>: Leu
<div class="collapser collapser-target-input-path1-gene359 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-input-path1-gene359-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGYIIGGVFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DGYIIGGVFK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 4 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-517"></div> <span class="help-tooltip" id="region_svg-help-517">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c4-svg" data-filename="JAAZTN010000005.1_r5c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,690,230 - 1,750,801 nt. (total: 60,572 nt)

<a class="cluster-rules-header" id="r5c4-rules-header" href="#r5c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c4-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r5c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-518"></div> <span class="help-tooltip" id="focus-panel-help-518">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c4-gene-table-details" data-name="r5c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c4-MIBiG-cluster-compare" data-name="r5c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c4-clusterblast" data-name="r5c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c4-knownclusterblast" data-name="r5c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c4-subclusterblast" data-name="r5c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c4-pfam-details" data-name="r5c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c4-tigrfam-details" data-name="r5c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-282"></div> <span class="help-tooltip" id="gt-help-tooltip-help-282">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-278"></div> <span class="help-tooltip" id="MIBiG-help-278">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c4"></div>
<div id="comparison-MIBiG-r5c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002262: 0-50372">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002262/index.html#r1c1" target="_new">BGC0002262</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>trienylfuranol A, sporothriolide, dihydrosporothriolide, deoxysporothric acid, dihydroisosporothric acid, sporothric acid, isosporothric acid, sporochartine B</td>


<td>Hypomontagnella monticulosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.44, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>NRP, Polyketide</td>

<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>


<td>Fusarium heterosporum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001423: 0-41128">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001423/index.html#r1c1" target="_new">BGC0001423</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide C</td>


<td>Myxococcus hansupus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001421: 0-47099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001421/index.html#r1c1" target="_new">BGC0001421</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide B</td>


<td>Myxococcus sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001424: 0-41589">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001424/index.html#r1c1" target="_new">BGC0001424</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide C</td>


<td>Myxococcus virescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001426: 0-44134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001426/index.html#r1c1" target="_new">BGC0001426</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001427: 0-44152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001427/index.html#r1c1" target="_new">BGC0001427</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001425: 0-44135">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001425/index.html#r1c1" target="_new">BGC0001425</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001428: 0-44682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001428/index.html#r1c1" target="_new">BGC0001428</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001996: 8-46512">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001996/index.html#r1c1" target="_new">BGC0001996</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Other</td>

<td>oryzine A, oryzine B</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002262: 0-50372">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002262/index.html#r1c1" target="_new">BGC0002262</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>trienylfuranol A, sporothriolide, dihydrosporothriolide, deoxysporothric acid, dihydroisosporothric acid, sporothric acid, isosporothric acid, sporochartine B</td>
<td>Hypomontagnella monticulosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.44, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>NRP, Polyketide</td>
<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>
<td>Fusarium heterosporum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001423: 0-41128">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001423/index.html#r1c1" target="_new">BGC0001423</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide C</td>
<td>Myxococcus hansupus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001421: 0-47099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001421/index.html#r1c1" target="_new">BGC0001421</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide B</td>
<td>Myxococcus sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001424: 0-41589">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001424/index.html#r1c1" target="_new">BGC0001424</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide C</td>
<td>Myxococcus virescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001426: 0-44134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001426/index.html#r1c1" target="_new">BGC0001426</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001427: 0-44152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001427/index.html#r1c1" target="_new">BGC0001427</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001425: 0-44135">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001425/index.html#r1c1" target="_new">BGC0001425</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001428: 0-44682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001428/index.html#r1c1" target="_new">BGC0001428</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001996: 8-46512">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001996/index.html#r1c1" target="_new">BGC0001996</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Other</td>
<td>oryzine A, oryzine B</td>
<td>Aspergillus oryzae RIB40</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-279"></div> <span class="help-tooltip" id="cb-general-help-279">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c4_1.svg">
NW_006271970 (3761055-3936010): Cordyceps militaris CM01 unplaced genomic sca... (25% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_2.svg">
NC_035794 (1709791-1793960): Pochonia chlamydosporia 170 chromosome 5, whole ... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c4_3.svg">
NW_014574622 (178902-269178): Penicillium digitatum Pd1 unplaced genomic scaf... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_4.svg">
NW_023336279 (145289-236294): Aspergillus tubingensis WU-2223L DNA, scaffold ... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_5.svg">
NC_054856 (3727884-3894446): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (3% of genes show similarity), NI-siderophore,NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_6.svg">
NW_020292053 (616043-708384): Aspergillus vadensis CBS 113365 unplaced genomi... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_7.svg">
NC_049561 (5202011-5292943): Talaromyces rugulosus chromosome I, complete seq... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_8.svg">
NC_072350 (2939594-3030676): Talaromyces marneffei chromosome 3, complete seq... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_9.svg">
NW_017253361 (212602-309441): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_10.svg">
NW_017253362 (1208407-1333171): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (3% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r5c4-download">Download graphic</button>
</div>
<div id="clusterblast-r5c4-svg"></div>

</div>
</div>

<div class="body-details-section r5c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-280"></div> <span class="help-tooltip" id="cb-known-help-280">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-281"></div> <span class="help-tooltip" id="cb-sub-help-281">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-283"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-283">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-284"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-284">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-tigrfam-details-svg" data-filename="JAAZTN010000005.1_r5c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r5c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 5 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-519"></div> <span class="help-tooltip" id="region_svg-help-519">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c5-svg" data-filename="JAAZTN010000005.1_r5c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,738,257 - 2,785,904 nt. (total: 47,648 nt)

<a class="cluster-rules-header" id="r5c5-rules-header" href="#r5c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c5-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-520"></div> <span class="help-tooltip" id="focus-panel-help-520">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c5-gene-table-details" data-name="r5c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c5-nrps_pks" data-name="r5c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c5-MIBiG-cluster-compare" data-name="r5c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c5-clusterblast" data-name="r5c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c5-knownclusterblast" data-name="r5c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c5-subclusterblast" data-name="r5c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c5-nrps-pks-bubbles" data-name="r5c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c5-pfam-details" data-name="r5c5-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-293"></div> <span class="help-tooltip" id="gt-help-tooltip-help-293">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-285"></div> <span class="help-tooltip" id="nrps-domain-help-285">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c5-details-svg" data-filename="JAAZTN010000005.1_r5c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-286"></div> <span class="help-tooltip" id="MIBiG-help-286">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c5"></div>
<div id="comparison-MIBiG-r5c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Terpene, Polyketide</td>

<td>calidoustene A, calidoustene B, calidoustene C</td>


<td>Aspergillus calidoustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>epipyriculol</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>epipyriculol</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-287"></div> <span class="help-tooltip" id="cb-general-help-287">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c5_1.svg">
NW_014013660 (102297-169956): Trichoderma virens Gv29-8 chromosome Unknown TR... (62% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_2.svg">
NC_063077 (1099162-1166854): Purpureocillium takamizusanense chromosome 10, c... (63% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_3.svg">
NW_025408670 (2580109-2647727): Hirsutella rhossiliensis strain HR02 chromoso... (55% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_4.svg">
NC_035793 (3629294-3696908): Pochonia chlamydosporia 170 chromosome 4, whole ... (55% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_5.svg">
NW_006711164 (601534-669174): Trichoderma reesei QM6a unplaced genomic scaffo... (64% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_6.svg">
NC_057317 (2166473-2234156): Ustilaginoidea virens chromosome 2, complete seq... (52% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_7.svg">
NC_036627 (3001008-3068498): Fusarium fujikuroi IMI 58289 draft genome, chrom... (50% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_8.svg">
NC_072845 (3069245-3136729): Fusarium oxysporum Fo47 chromosome VI, complete ... (43% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_9.svg">
NC_058392 (1189925-1257407): Fusarium musae strain F31 chromosome 6, whole ge... (53% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_10.svg">
NC_070546 (2092593-2160147): Fusarium falciforme chromosome 3, complete sequence (58% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r5c5-download">Download graphic</button>
</div>
<div id="clusterblast-r5c5-svg"></div>

</div>
</div>

<div class="body-details-section r5c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-288"></div> <span class="help-tooltip" id="cb-known-help-288">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-289"></div> <span class="help-tooltip" id="cb-sub-help-289">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-292"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-292">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c5-domain-bubble-svg" data-filename="JAAZTN010000005.1_r5c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (2738256 - 2785904): single T1PKS
</option>


</select>

<div class="details-svg" id="r5c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-294"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-294">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c5-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c5-nrps_pks_products" data-name="r5c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c5-nrps_pks_monomers" data-name="r5c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-290"></div> <span class="help-tooltip" id="nrps-structures-help-290">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 2738256 - 2785904:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c5-5" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c5-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-291"></div> <span class="help-tooltip" id="nrps-monomers-help-291">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene709</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene709 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (335..592)</strong>: pk
<div class="collapser collapser-target-input-path1-gene709 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene709-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>2-Methylbutyryl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-input-path1-gene709-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 59.8</dt>

<dd></dd><dt>Methylmalonyl-CoA: 56.1</dt>

<dd></dd><dt>Malonyl-CoA: 34.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 32.3</dt>

<dd></dd><dt>Propionyl-CoA: 27.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 23.6</dt>

<dd></dd><dt>CHC-CoA: 19.9</dt>

<dd></dd><dt>Benzoyl-CoA: 19.3</dt>

<dd></dd><dt>fatty_acid: 17.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 9.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.5</dt>

<dd></dd><dt>Acetyl-CoA: 2.9</dt>

<dd></dd><dt>inactive: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 6 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-521"></div> <span class="help-tooltip" id="region_svg-help-521">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c6-svg" data-filename="JAAZTN010000005.1_r5c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,042,285 - 3,098,625 nt. (total: 56,341 nt)

<a class="cluster-rules-header" id="r5c6-rules-header" href="#r5c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c6-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-522"></div> <span class="help-tooltip" id="focus-panel-help-522">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c6-gene-table-details" data-name="r5c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c6-nrps_pks" data-name="r5c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c6-MIBiG-cluster-compare" data-name="r5c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c6-clusterblast" data-name="r5c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c6-knownclusterblast" data-name="r5c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c6-subclusterblast" data-name="r5c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c6-nrps-pks-bubbles" data-name="r5c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c6-pfam-details" data-name="r5c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c6-tigrfam-details" data-name="r5c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-303"></div> <span class="help-tooltip" id="gt-help-tooltip-help-303">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-295"></div> <span class="help-tooltip" id="nrps-domain-help-295">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c6-details-svg" data-filename="JAAZTN010000005.1_r5c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-296"></div> <span class="help-tooltip" id="MIBiG-help-296">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c6"></div>
<div id="comparison-MIBiG-r5c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>oxepinamide D</td>


<td>Aspergillus ustus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>oxepinamide D</td>
<td>Aspergillus ustus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-297"></div> <span class="help-tooltip" id="cb-general-help-297">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c6_1.svg">
NW_006271970 (4182342-4255710): Cordyceps militaris CM01 unplaced genomic sca... (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c6_2.svg">
NW_011942171 (5162432-5375378): Metarhizium robertsii ARSEF 23 MAA Scf 3, who... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_3.svg">
NW_014574707 (383137-474713): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c6_4.svg">
NW_021167094 (207435-275256): Sodiomyces alkalinus F11 unplaced genomic scaff... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c6_5.svg">
NW_025091266 (1102530-1174184): Aspergillus udagawae IFM 46973 DNA, scaffold ... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c6_6.svg">
NW_025091283 (359926-455746): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_7.svg">
NW_011942151 (1102940-1173579): Metarhizium robertsii ARSEF 23 MAA Scf 11, wh... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c6_8.svg">
NC_035790 (380514-527376): Pochonia chlamydosporia 170 chromosome 1, whole ge... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c6_9.svg">
NW_025091281 (4887733-5033357): Aspergillus pseudoviridinutans IFM 55266 DNA,... (5% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_10.svg">
NW_020291492 (266674-426009): Aspergillus aculeatinus CBS 121060 unplaced gen... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r5c6-download">Download graphic</button>
</div>
<div id="clusterblast-r5c6-svg"></div>

</div>
</div>

<div class="body-details-section r5c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-298"></div> <span class="help-tooltip" id="cb-known-help-298">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-299"></div> <span class="help-tooltip" id="cb-sub-help-299">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-302"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-302">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c6-domain-bubble-svg" data-filename="JAAZTN010000005.1_r5c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (3042284 - 3098625): single NRPS
</option>


</select>

<div class="details-svg" id="r5c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-304"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-304">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c6-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-305"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-305">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c6-tigrfam-details-svg" data-filename="JAAZTN010000005.1_r5c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c6-nrps_pks_products" data-name="r5c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c6-nrps_pks_monomers" data-name="r5c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-300"></div> <span class="help-tooltip" id="nrps-structures-help-300">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 3042284 - 3098625:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c6-6" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X - Ala - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x,ala*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c6-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-301"></div> <span class="help-tooltip" id="nrps-monomers-help-301">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene787</span></strong>: X - X - Ala - X
<div class="collapser collapser-target-input-path1-gene787 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (40..445)</strong>: X
<div class="collapser collapser-target-input-path1-gene787 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-input-path1-gene787-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVMEVGTPTK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Å match)</dd>
<dd>Orn <span class="serif">DVMELSSITK</span> (47% 8Å match)</dd>
<dd>Pro <span class="serif">DVMLVGAVNK</span> (44% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1177..1589)</strong>: X
<div class="collapser collapser-target-input-path1-gene787 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene787-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALDTGMATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (71% 8Å match)</dd>
<dd>Leu <span class="serif">DALYLGMMCK</span> (41% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2278..2679)</strong>: Ala
<div class="collapser collapser-target-input-path1-gene787 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-input-path1-gene787-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLCLHIMTFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3366..3770)</strong>: X
<div class="collapser collapser-target-input-path1-gene787 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene787-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALDTGMATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Å match)</dd>
<dd>Phe or Trp <span class="serif">DAMIGGMVTK</span> (50% 8Å match)</dd>
<dd>Leu <span class="serif">DALYLGMMCK</span> (47% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 7 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-523"></div> <span class="help-tooltip" id="region_svg-help-523">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c7-svg" data-filename="JAAZTN010000005.1_r5c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,567,082 - 3,610,671 nt. (total: 43,590 nt)

<a class="cluster-rules-header" id="r5c7-rules-header" href="#r5c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c7-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-524"></div> <span class="help-tooltip" id="focus-panel-help-524">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c7-gene-table-details" data-name="r5c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c7-nrps_pks" data-name="r5c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c7-MIBiG-cluster-compare" data-name="r5c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c7-clusterblast" data-name="r5c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c7-knownclusterblast" data-name="r5c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c7-subclusterblast" data-name="r5c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c7-nrps-pks-bubbles" data-name="r5c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c7-pfam-details" data-name="r5c7-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c7-tigrfam-details" data-name="r5c7-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-314"></div> <span class="help-tooltip" id="gt-help-tooltip-help-314">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-306"></div> <span class="help-tooltip" id="nrps-domain-help-306">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c7-details-svg" data-filename="JAAZTN010000005.1_r5c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-307"></div> <span class="help-tooltip" id="MIBiG-help-307">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c7"></div>
<div id="comparison-MIBiG-r5c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>AbT1</td>


<td>Aureobasidium pullulans</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>AbT1</td>
<td>Aureobasidium pullulans</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-308"></div> <span class="help-tooltip" id="cb-general-help-308">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c7_1.svg">
NC_035790 (3811097-3874723): Pochonia chlamydosporia 170 chromosome 1, whole ... (40% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c7_2.svg">
NW_023336187 (263224-326908): Geosmithia morbida isolate 1262 chromosome Unkn... (39% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c7_3.svg">
NC_026476 (4083977-4147801): Fusarium graminearum PH-1 chromosome 3, whole ge... (27% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c7_4.svg">
NC_038014 (4378948-4442770): Fusarium venenatum strain A3/5 genome assembly, ... (29% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c7_5.svg">
NC_058401 (4731897-4795714): Fusarium poae strain DAOMC 252244 chromosome 3, ... (29% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c7_6.svg">
NW_026250020 (2828871-2896149): Hypoxylon trugodes strain CBS 135444 unplaced... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c7_7.svg">
NW_006917096 (193746-257181): Pestalotiopsis fici W106-1 unplaced genomic sca... (26% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c7_8.svg">
NW_015971648 (509725-573970): Fonsecaea multimorphosa CBS 102226 unplaced gen... (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c7_9.svg">
NW_015622516 (1296496-1389490): Exophiala spinifera strain CBS 89968 unplaced... (5% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c7_10.svg">
NC_018210 (1464053-1532264): Zymoseptoria tritici IPO323 chromosome 9, whole ... (10% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r5c7-download">Download graphic</button>
</div>
<div id="clusterblast-r5c7-svg"></div>

</div>
</div>

<div class="body-details-section r5c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-309"></div> <span class="help-tooltip" id="cb-known-help-309">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-310"></div> <span class="help-tooltip" id="cb-sub-help-310">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-313"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-313">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c7-domain-bubble-svg" data-filename="JAAZTN010000005.1_r5c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (3567081 - 3610671): single NRPS-like
</option>


</select>

<div class="details-svg" id="r5c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-315"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-315">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c7-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c7-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-316"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-316">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c7-tigrfam-details-svg" data-filename="JAAZTN010000005.1_r5c7_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c7-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c7-nrps_pks_products" data-name="r5c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c7-nrps_pks_monomers" data-name="r5c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-311"></div> <span class="help-tooltip" id="nrps-structures-help-311">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 3567081 - 3610671:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c7-7" class="smiles-canvas" data-smiles="NC(CCCC(=O)O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Aad)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c7-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCC(=O)O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-312"></div> <span class="help-tooltip" id="nrps-monomers-help-312">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene927</span></strong>: Aad
<div class="collapser collapser-target-input-path1-gene927 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (25..490)</strong>: Aad
<div class="collapser collapser-target-input-path1-gene927 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Aad
<div class="collapser collapser-target-input-path1-gene927-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPRHFVMRSK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Aad <span class="serif">DPRHFVMRSK</span> (91% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 8 - NRP-metallophore,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-525"></div> <span class="help-tooltip" id="region_svg-help-525">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c8-svg" data-filename="JAAZTN010000005.1_r5c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,257,421 - 4,317,347 nt. (total: 59,927 nt)

<a class="cluster-rules-header" id="r5c8-rules-header" href="#r5c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c8-rules">NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-526"></div> <span class="help-tooltip" id="focus-panel-help-526">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c8-gene-table-details" data-name="r5c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c8-nrps_pks" data-name="r5c8-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c8-MIBiG-cluster-compare" data-name="r5c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c8-clusterblast" data-name="r5c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c8-knownclusterblast" data-name="r5c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c8-subclusterblast" data-name="r5c8-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c8-nrps-pks-bubbles" data-name="r5c8-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c8-pfam-details" data-name="r5c8-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c8-tigrfam-details" data-name="r5c8-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-325"></div> <span class="help-tooltip" id="gt-help-tooltip-help-325">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c8-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-317"></div> <span class="help-tooltip" id="nrps-domain-help-317">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c8-details-svg" data-filename="JAAZTN010000005.1_r5c8_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c8-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-318"></div> <span class="help-tooltip" id="MIBiG-help-318">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c8"></div>
<div id="comparison-MIBiG-r5c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.61</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.61</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.40</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.36</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>pseudomonine</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349: 0-24145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>erythrochelin</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>pseudomonine</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349: 0-24145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>erythrochelin</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-319"></div> <span class="help-tooltip" id="cb-general-help-319">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c8_1.svg">
NW_014574715 (339608-420863): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (72% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c8_2.svg">
NW_011942149 (5526781-5608010): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (72% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c8_3.svg">
NW_006711178 (9878-84649): Trichoderma reesei QM6a unplaced genomic scaffold ... (54% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c8_4.svg">
NC_035790 (6855176-6936375): Pochonia chlamydosporia 170 chromosome 1, whole ... (66% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c8_5.svg">
NW_006271969 (5451891-5533992): Cordyceps militaris CM01 unplaced genomic sca... (64% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c8_6.svg">
NC_070530 (3742001-3826089): Fusarium keratoplasticum isolate Fu6.1 chromosom... (58% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c8_7.svg">
NC_070545 (3816730-3900832): Fusarium falciforme chromosome 2, complete sequence (58% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c8_8.svg">
NC_026476 (2004097-2087970): Fusarium graminearum PH-1 chromosome 3, whole ge... (41% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c8_9.svg">
NC_058389 (2698749-2782755): Fusarium musae strain F31 chromosome 3, whole ge... (61% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c8_10.svg">
NC_057319 (3036502-3117459): Ustilaginoidea virens chromosome 4, complete seq... (47% of genes show similarity), NRP-metallophore,NRPS
</option>

</select>
<button id= "clusterblast-r5c8-download">Download graphic</button>
</div>
<div id="clusterblast-r5c8-svg"></div>

</div>
</div>

<div class="body-details-section r5c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-320"></div> <span class="help-tooltip" id="cb-known-help-320">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-321"></div> <span class="help-tooltip" id="cb-sub-help-321">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c8-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-324"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-324">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c8-domain-bubble-svg" data-filename="JAAZTN010000005.1_r5c8_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c8-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC8">
Candidate 8 (4257420 - 4317347): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r5c8-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c8-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-326"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-326">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c8-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c8-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-327"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-327">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c8-tigrfam-details-svg" data-filename="JAAZTN010000005.1_r5c8_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c8-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c8-nrps_pks_products" data-name="r5c8-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c8-nrps_pks_monomers" data-name="r5c8-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c8-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-322"></div> <span class="help-tooltip" id="nrps-structures-help-322">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 8, location 4257420 - 4317347:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c8-8" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c8-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c8-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-323"></div> <span class="help-tooltip" id="nrps-monomers-help-323">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1111</span></strong>: X - X
<div class="collapser collapser-target-input-path1-gene1111 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (662..1056)</strong>: X
<div class="collapser collapser-target-input-path1-gene1111 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene1111-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFSVAGNIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DVFSVALVYK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2270..2669)</strong>: X
<div class="collapser collapser-target-input-path1-gene1111 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-input-path1-gene1111-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVLDIGAIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DVLSFGAIIK</span> (65% 8Å match)</dd>
<dd>Phe or Trp <span class="serif">DVLSIGAVVK</span> (62% 8Å match)</dd>
<dd>Arg <span class="serif">DVESIGAIAK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>














</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c9' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 9 - isocyanide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-527"></div> <span class="help-tooltip" id="region_svg-help-527">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region009.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c9-svg" data-filename="JAAZTN010000005.1_r5c9.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,348,960 - 4,391,085 nt. (total: 42,126 nt)

<a class="cluster-rules-header" id="r5c9-rules-header" href="#r5c9">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c9-rules">isocyanide: cds(TauD and DIT1_PvcA)</div>

<div class="region-svg-container">
<div id='r5c9-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-528"></div> <span class="help-tooltip" id="focus-panel-help-528">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c9">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c9-gene-table-details" data-name="r5c9-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c9-MIBiG-cluster-compare" data-name="r5c9-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c9-clusterblast" data-name="r5c9-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c9-knownclusterblast" data-name="r5c9-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c9-subclusterblast" data-name="r5c9-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r5c9-pfam-details" data-name="r5c9-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c9-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-332"></div> <span class="help-tooltip" id="gt-help-tooltip-help-332">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c9">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c9-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c9-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c9">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c9-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-328"></div> <span class="help-tooltip" id="MIBiG-help-328">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c9-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c9-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c9"></div>
<div id="comparison-MIBiG-r5c9-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">isocyanide</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000923: 943-6060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000923/index.html#r1c1" target="_new">BGC0000923</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Other</td>

<td>paerucumarin</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Alkaloid</td>

<td>fischerindole</td>


<td>Fischerella sp. SAG 46.79</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001594: 0-38842">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001594/index.html#r1c1" target="_new">BGC0001594</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Alkaloid</td>

<td>fischerindole L</td>


<td>Fischerella muscicola UTEX 1829</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000968: 0-25930">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000968/index.html#r1c1" target="_new">BGC0000968</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>chaetoglobosin A, chaetoglobosin C</td>


<td>Penicillium expansum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>


<td>Hapalosiphon welwitschii UTEX B 1830</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.45, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.45, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Alkaloid</td>

<td>ambiguine H isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>oviedomycin</td>


<td>Streptomyces antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000275: 58-12211">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000275/index.html#r1c1" target="_new">BGC0000275</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>tetracenomycin C</td>


<td>Streptomyces glaucescens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c9-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000923: 943-6060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000923/index.html#r1c1" target="_new">BGC0000923</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>paerucumarin</td>
<td>Pseudomonas aeruginosa PAO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Alkaloid</td>
<td>fischerindole</td>
<td>Fischerella sp. SAG 46.79</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001594: 0-38842">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001594/index.html#r1c1" target="_new">BGC0001594</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Alkaloid</td>
<td>fischerindole L</td>
<td>Fischerella muscicola UTEX 1829</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000968: 0-25930">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000968/index.html#r1c1" target="_new">BGC0000968</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>NRP, Polyketide</td>
<td>chaetoglobosin A, chaetoglobosin C</td>
<td>Penicillium expansum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>
<td>Hapalosiphon welwitschii UTEX B 1830</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.45, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>
<td>Fischerella ambigua UTEX 1903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.45, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Alkaloid</td>
<td>ambiguine H isonitrile</td>
<td>Fischerella ambigua UTEX 1903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>oviedomycin</td>
<td>Streptomyces antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000275: 58-12211">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000275/index.html#r1c1" target="_new">BGC0000275</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Polyketide</td>
<td>tetracenomycin C</td>
<td>Streptomyces glaucescens</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c9-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-329"></div> <span class="help-tooltip" id="cb-general-help-329">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c9-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c9_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c9_1.svg">
NC_054389 (8457870-8566158): Drechmeria coniospora strain ARSEF 6962 chromoso... (28% of genes show similarity), T1PKS,isocyanide
</option>

<option value = "svg/clusterblast_r5c9_2.svg">
NW_011942149 (5418619-5480759): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (38% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c9_3.svg">
NW_014574715 (467040-529181): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (33% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c9_4.svg">
NC_063070 (2362618-2424749): Purpureocillium takamizusanense chromosome 3, co... (33% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c9_5.svg">
NW_006271969 (5556555-5618595): Cordyceps militaris CM01 unplaced genomic sca... (50% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c9_6.svg">
NW_025408686 (565240-627429): Hirsutella rhossiliensis strain HR02 chromosome... (35% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c9_7.svg">
NC_035790 (6741031-6803169): Pochonia chlamydosporia 170 chromosome 1, whole ... (27% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c9_8.svg">
NW_006711178 (114520-207586): Trichoderma reesei QM6a unplaced genomic scaffo... (16% of genes show similarity), isocyanide,terpene
</option>

<option value = "svg/clusterblast_r5c9_9.svg">
NC_057319 (4219732-4281896): Ustilaginoidea virens chromosome 4, complete seq... (14% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c9_10.svg">
NW_024468916 (67569-143215): Cryphonectria parasitica EP155 unplaced genomic ... (16% of genes show similarity), isocyanide,terpene
</option>

</select>
<button id= "clusterblast-r5c9-download">Download graphic</button>
</div>
<div id="clusterblast-r5c9-svg"></div>

</div>
</div>

<div class="body-details-section r5c9-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-330"></div> <span class="help-tooltip" id="cb-known-help-330">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c9-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-331"></div> <span class="help-tooltip" id="cb-sub-help-331">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r5c9-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-333"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-333">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c9-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c9_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c9-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r5c10' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000005.1 - Region 10 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-529"></div> <span class="help-tooltip" id="region_svg-help-529">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000005.1.region010.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c10-svg" data-filename="JAAZTN010000005.1_r5c10.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,586,794 - 4,630,328 nt. (total: 43,535 nt)

<a class="cluster-rules-header" id="r5c10-rules-header" href="#r5c10">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c10-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c10-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-530"></div> <span class="help-tooltip" id="focus-panel-help-530">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c10">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c10-gene-table-details" data-name="r5c10-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c10-nrps_pks" data-name="r5c10-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c10-MIBiG-cluster-compare" data-name="r5c10-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c10-clusterblast" data-name="r5c10-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c10-knownclusterblast" data-name="r5c10-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c10-subclusterblast" data-name="r5c10-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c10-pfam-details" data-name="r5c10-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c10-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-340"></div> <span class="help-tooltip" id="gt-help-tooltip-help-340">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c10">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c10-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c10-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c10">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c10-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-334"></div> <span class="help-tooltip" id="nrps-domain-help-334">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c10-details-svg" data-filename="JAAZTN010000005.1_r5c10_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c10-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c10-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-335"></div> <span class="help-tooltip" id="MIBiG-help-335">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c10-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c10-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c10"></div>
<div id="comparison-MIBiG-r5c10-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001515: 0-63902">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001515/index.html#r1c1" target="_new">BGC0001515</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP</td>

<td>aspercryptins</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>NRP</td>

<td>nidulanin A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.41, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>isoindolinone</td>


<td>Stachybotrys chlorohalonata IBT 40285</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001854: 1-34076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001854/index.html#r1c1" target="_new">BGC0001854</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.44, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>tricholignan A</td>


<td>Trichoderma harzianum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>ustethylin A</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001190: 82-52910">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001190/index.html#r1c1" target="_new">BGC0001190</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.42, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>fusaric acid</td>


<td>Fusarium verticillioides 7600</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c10-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001515: 0-63902">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001515/index.html#r1c1" target="_new">BGC0001515</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP</td>
<td>aspercryptins</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>NRP</td>
<td>nidulanin A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.41, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>isoindolinone</td>
<td>Stachybotrys chlorohalonata IBT 40285</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001854: 1-34076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001854/index.html#r1c1" target="_new">BGC0001854</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.44, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>tricholignan A</td>
<td>Trichoderma harzianum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>ustethylin A</td>
<td>Aspergillus ustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001190: 82-52910">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001190/index.html#r1c1" target="_new">BGC0001190</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.42, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>fusaric acid</td>
<td>Fusarium verticillioides 7600</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c10-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-336"></div> <span class="help-tooltip" id="cb-general-help-336">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c10-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c10_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c10_1.svg">
NW_006271975 (1558381-1621743): Cordyceps militaris CM01 unplaced genomic sca... (20% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c10_2.svg">
NW_017264197 (1062739-1126153): Xylona heveae TC161 unplaced genomic scaffold... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c10_3.svg">
NC_064677 (4604767-4667923): Colletotrichum lupini chromosome 4, complete seq... (6% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c10_4.svg">
NW_025091222 (2358629-2426438): Aspergillus viridinutans IFM 47045 DNA, scaff... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c10_5.svg">
NC_035792 (887996-951264): Pochonia chlamydosporia 170 chromosome 3, whole ge... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c10_6.svg">
NW_011942171 (3680815-3744138): Metarhizium robertsii ARSEF 23 MAA Scf 3, who... (11% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c10_7.svg">
NW_014574699 (142223-205548): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c10_8.svg">
NW_011371366 (647325-794373): Paracoccidioides brasiliensis Pb18 unplaced gen... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c10_9.svg">
NW_024067568 (58665-121970): Penicillium roqueforti strain LCP96 04111 chromo... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c10_10.svg">
NC_072348 (1750133-1909935): Talaromyces marneffei chromosome 1, complete seq... (3% of genes show similarity), NRPS-like,fungal-RiPP-like,other
</option>

</select>
<button id= "clusterblast-r5c10-download">Download graphic</button>
</div>
<div id="clusterblast-r5c10-svg"></div>

</div>
</div>

<div class="body-details-section r5c10-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-337"></div> <span class="help-tooltip" id="cb-known-help-337">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c10-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-338"></div> <span class="help-tooltip" id="cb-sub-help-338">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c10-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-341"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-341">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c10-pfam-details-svg" data-filename="JAAZTN010000005.1_r5c10_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c10-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c10-nrps_pks_monomers" data-name="r5c10-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r5c10-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-339"></div> <span class="help-tooltip" id="nrps-monomers-help-339">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

</dl>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r6c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 1 - isocyanide-nrp <div class="help-container"> <div class="help-icon" data-id="region_svg-help-531"></div> <span class="help-tooltip" id="region_svg-help-531">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c1-svg" data-filename="JAAZTN010000006.1_r6c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 221,188 - 276,076 nt. (total: 54,889 nt)

<a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c1-rules">isocyanide-nrp: (DIT1_PvcA and minimum(2, [AMP-binding, PP-binding, Transferase]))</div>

<div class="region-svg-container">
<div id='r6c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-532"></div> <span class="help-tooltip" id="focus-panel-help-532">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c1-gene-table-details" data-name="r6c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c1-nrps-pks-bubbles" data-name="r6c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-350"></div> <span class="help-tooltip" id="gt-help-tooltip-help-350">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-342"></div> <span class="help-tooltip" id="nrps-domain-help-342">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="JAAZTN010000006.1_r6c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-343"></div> <span class="help-tooltip" id="MIBiG-help-343">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c1"></div>
<div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">isocyanide-nrp</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.47, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>KK-1</td>


<td>Curvularia clavata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043: 0-13097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>curacin A</td>


<td>Moorea producens 19L</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000292: 0-27623">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000292/index.html#r1c1" target="_new">BGC0000292</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>acetylaranotin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002594: 653-30873">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002594/index.html#r1c1" target="_new">BGC0002594</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>orcinol, orsellinic acid</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Other</td>

<td>fosfazinomycin A, fosfazinomycin B</td>


<td>Streptomyces sp. WM6372</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002267: 0-50173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002267/index.html#r1c1" target="_new">BGC0002267</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.44, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>azasperpyranone A, azasperpyranone B, azasperpyranone C, azasperpyranone D, azasperpyranone E, azasperpyranone F, azasperpyranone G, azasperpyranone H</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596: 0-46688">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.44, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>fluostatins M-Q</td>


<td>Streptomyces albus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001436: 25-56534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001436/index.html#r1c1" target="_new">BGC0001436</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>cornexistin</td>


<td>Paecilomyces divaricatus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.47, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>KK-1</td>
<td>Curvularia clavata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043: 0-13097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>curacin A</td>
<td>Moorea producens 19L</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000292: 0-27623">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000292/index.html#r1c1" target="_new">BGC0000292</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP</td>
<td>acetylaranotin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002594: 653-30873">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002594/index.html#r1c1" target="_new">BGC0002594</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>orcinol, orsellinic acid</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Other</td>
<td>fosfazinomycin A, fosfazinomycin B</td>
<td>Streptomyces sp. WM6372</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002267: 0-50173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002267/index.html#r1c1" target="_new">BGC0002267</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.44, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>azasperpyranone A, azasperpyranone B, azasperpyranone C, azasperpyranone D, azasperpyranone E, azasperpyranone F, azasperpyranone G, azasperpyranone H</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596: 0-46688">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.44, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Polyketide</td>
<td>fluostatins M-Q</td>
<td>Streptomyces albus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001436: 25-56534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001436/index.html#r1c1" target="_new">BGC0001436</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Polyketide</td>
<td>cornexistin</td>
<td>Paecilomyces divaricatus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-344"></div> <span class="help-tooltip" id="cb-general-help-344">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c1_1.svg">
NW_006271975 (307084-386893): Cordyceps militaris CM01 unplaced genomic scaff... (11% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_2.svg">
NW_022987217 (1562235-1642230): Trematosphaeria pertusa strain CBS 122368 unp... (6% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_3.svg">
NW_006912928 (910358-990720): Cladophialophora psammophila CBS 110553 aczJe-s... (7% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_4.svg">
NW_006912907 (2292264-2372421): Capronia epimyces CBS 606.96 aczIQ-supercont1... (8% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_5.svg">
NW_007360990 (964758-1055299): Glarea lozoyensis ATCC 20868 chromosome Unknow... (13% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_6.svg">
NW_008481816 (5057935-5146015): Cladophialophora carrionii CBS 160.54 unplace... (10% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_7.svg">
NC_035791 (3928355-4086848): Pochonia chlamydosporia 170 chromosome 2, whole ... (6% of genes show similarity), fungal-RiPP-like,isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_8.svg">
NW_011942160 (3275170-3355369): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (14% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_9.svg">
NW_014574716 (5768037-5848237): Metarhizium brunneum ARSEF 3297 chromosome Un... (14% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c1_10.svg">
NW_009276928 (273235-353324): Verticillium dahliae VdLs.17 supercont1.27 geno... (11% of genes show similarity), isocyanide-nrp
</option>

</select>
<button id= "clusterblast-r6c1-download">Download graphic</button>
</div>
<div id="clusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-345"></div> <span class="help-tooltip" id="cb-known-help-345">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-346"></div> <span class="help-tooltip" id="cb-sub-help-346">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-349"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-349">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-domain-bubble-svg" data-filename="JAAZTN010000006.1_r6c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (221187 - 276076): single isocyanide-nrp
</option>


</select>

<div class="details-svg" id="r6c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-351"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-351">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c1-nrps_pks_products" data-name="r6c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c1-nrps_pks_monomers" data-name="r6c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-347"></div> <span class="help-tooltip" id="nrps-structures-help-347">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 221187 - 276076:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-348"></div> <span class="help-tooltip" id="nrps-monomers-help-348">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene69</span></strong>: X
<div class="collapser collapser-target-input-path1-gene69 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (442..758)</strong>: X
<div class="collapser collapser-target-input-path1-gene69 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene69-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GANSCSLFIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-533"></div> <span class="help-tooltip" id="region_svg-help-533">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c2-svg" data-filename="JAAZTN010000006.1_r6c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 276,865 - 337,882 nt. (total: 61,018 nt)

<a class="cluster-rules-header" id="r6c2-rules-header" href="#r6c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r6c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-534"></div> <span class="help-tooltip" id="focus-panel-help-534">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c2-gene-table-details" data-name="r6c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c2-MIBiG-cluster-compare" data-name="r6c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c2-clusterblast" data-name="r6c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c2-knownclusterblast" data-name="r6c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c2-subclusterblast" data-name="r6c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c2-pfam-details" data-name="r6c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c2-tigrfam-details" data-name="r6c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-356"></div> <span class="help-tooltip" id="gt-help-tooltip-help-356">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-352"></div> <span class="help-tooltip" id="MIBiG-help-352">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c2"></div>
<div id="comparison-MIBiG-r6c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001888: 459-19007">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001888/index.html#r1c1" target="_new">BGC0001888</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other</td>

<td>mannosylerythritol lipid A, mannosylerythritol lipid B, mannosylerythritol lipid C</td>


<td>Ustilago maydis 521</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>emodin</td>


<td>Escovopsis weberi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002239: 1103-31038">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002239/index.html#r1c1" target="_new">BGC0002239</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.50, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide, NRP</td>

<td>GKK1032 A2, pyrrocidine A, pyrrocidine B</td>


<td>Acremonium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001026: 1-19101">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001026/index.html#r1c1" target="_new">BGC0001026</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>NG-391</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002208: 764-29749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002208/index.html#r1c1" target="_new">BGC0002208</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>oxepinamide F</td>


<td>Aspergillus ustus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001888: 459-19007">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001888/index.html#r1c1" target="_new">BGC0001888</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other</td>
<td>mannosylerythritol lipid A, mannosylerythritol lipid B, mannosylerythritol lipid C</td>
<td>Ustilago maydis 521</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>emodin</td>
<td>Escovopsis weberi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002239: 1103-31038">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002239/index.html#r1c1" target="_new">BGC0002239</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.50, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide, NRP</td>
<td>GKK1032 A2, pyrrocidine A, pyrrocidine B</td>
<td>Acremonium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001026: 1-19101">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001026/index.html#r1c1" target="_new">BGC0001026</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>NRP, Polyketide</td>
<td>NG-391</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002208: 764-29749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002208/index.html#r1c1" target="_new">BGC0002208</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>NRP</td>
<td>oxepinamide F</td>
<td>Aspergillus ustus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-353"></div> <span class="help-tooltip" id="cb-general-help-353">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c2_1.svg">
NW_006271975 (307084-386893): Cordyceps militaris CM01 unplaced genomic scaff... (22% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c2_2.svg">
NW_020290916 (183438-282887): Aspergillus niger CBS 101883 unplaced genomic s... (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_3.svg">
NT_166522 (170057-269564): Aspergillus niger CBS 513.88 supercontig An06 (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_4.svg">
NW_014574694 (19387-240297): Metarhizium brunneum ARSEF 3297 chromosome Unkno... (3% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_5.svg">
NW_007360988 (678454-840815): Glarea lozoyensis ATCC 20868 chromosome Unknown... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_6.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_7.svg">
NC_049566 (57470-149701): Talaromyces rugulosus chromosome VI, complete sequence (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_8.svg">
NC_064677 (1583587-1752421): Colletotrichum lupini chromosome 4, complete seq... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_9.svg">
NW_017263611 (213405-339732): Mollisia scopiformis unplaced genomic scaffold ... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_10.svg">
NW_025091266 (1883427-2035482): Aspergillus udagawae IFM 46973 DNA, scaffold ... (3% of genes show similarity), NRPS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c2-download">Download graphic</button>
</div>
<div id="clusterblast-r6c2-svg"></div>

</div>
</div>

<div class="body-details-section r6c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-354"></div> <span class="help-tooltip" id="cb-known-help-354">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-355"></div> <span class="help-tooltip" id="cb-sub-help-355">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-357"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-357">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-358"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-358">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-tigrfam-details-svg" data-filename="JAAZTN010000006.1_r6c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r6c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 3 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-535"></div> <span class="help-tooltip" id="region_svg-help-535">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c3-svg" data-filename="JAAZTN010000006.1_r6c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 373,965 - 426,949 nt. (total: 52,985 nt)

<a class="cluster-rules-header" id="r6c3-rules-header" href="#r6c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-536"></div> <span class="help-tooltip" id="focus-panel-help-536">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c3-gene-table-details" data-name="r6c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c3-nrps_pks" data-name="r6c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c3-MIBiG-cluster-compare" data-name="r6c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c3-clusterblast" data-name="r6c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c3-knownclusterblast" data-name="r6c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c3-subclusterblast" data-name="r6c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c3-nrps-pks-bubbles" data-name="r6c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c3-pfam-details" data-name="r6c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c3-tigrfam-details" data-name="r6c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-367"></div> <span class="help-tooltip" id="gt-help-tooltip-help-367">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-359"></div> <span class="help-tooltip" id="nrps-domain-help-359">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-details-svg" data-filename="JAAZTN010000006.1_r6c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-360"></div> <span class="help-tooltip" id="MIBiG-help-360">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c3"></div>
<div id="comparison-MIBiG-r6c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.71</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001049: 0-19176">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001049/index.html#r1c1" target="_new">BGC0001049</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.76, order:0.57, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.76, order:0.57, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.69</td>
<td>NRP, Polyketide</td>

<td>tenellin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002239: 1103-31038">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002239/index.html#r1c1" target="_new">BGC0002239</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Polyketide, NRP</td>

<td>GKK1032 A2, pyrrocidine A, pyrrocidine B</td>


<td>Acremonium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Polyketide</td>

<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>


<td>Chaetomium olivaceum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001136: 0-20147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001136/index.html#r1c1" target="_new">BGC0001136</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.71, order:0.51, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.71, order:0.51, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>NRP, Polyketide</td>

<td>desmethylbassianin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.61</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.60</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001049: 0-19176">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001049/index.html#r1c1" target="_new">BGC0001049</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.76, order:0.57, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP, Polyketide</td>
<td>tenellin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002239: 1103-31038">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002239/index.html#r1c1" target="_new">BGC0002239</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide, NRP</td>
<td>GKK1032 A2, pyrrocidine A, pyrrocidine B</td>
<td>Acremonium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>
<td>Chaetomium olivaceum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001136: 0-20147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001136/index.html#r1c1" target="_new">BGC0001136</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.71, order:0.51, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP, Polyketide</td>
<td>desmethylbassianin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-361"></div> <span class="help-tooltip" id="cb-general-help-361">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c3_1.svg">
NW_006271975 (27980-100581): Cordyceps militaris CM01 unplaced genomic scaffo... (50% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c3_2.svg">
NC_036629 (2646284-2784115): Fusarium fujikuroi IMI 58289 draft genome, chrom... (8% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_3.svg">
NC_072841 (2188172-2273307): Fusarium oxysporum Fo47 chromosome II, complete ... (10% of genes show similarity), NRPS,T1PKS,indole
</option>

<option value = "svg/clusterblast_r6c3_4.svg">
NC_054696 (3134268-3301036): Aspergillus flavus NRRL3357 chromosome 6, comple... (10% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c3_5.svg">
NC_038014 (6525771-6597702): Fusarium venenatum strain A3/5 genome assembly, ... (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c3_6.svg">
NC_072850 (2022235-2153706): Fusarium oxysporum Fo47 chromosome XI, complete ... (7% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_7.svg">
NW_007360993 (564614-637444): Glarea lozoyensis ATCC 20868 chromosome Unknown... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c3_8.svg">
NW_014013670 (106490-178646): Trichoderma virens Gv29-8 chromosome Unknown TR... (31% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c3_9.svg">
NC_036436 (608747-764003): Aspergillus oryzae RIB40 DNA, chromosome 2 (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c3_10.svg">
NC_064652 (3981714-4053762): Penicillium oxalicum strain HP7-1 chromosome III... (26% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r6c3-download">Download graphic</button>
</div>
<div id="clusterblast-r6c3-svg"></div>

</div>
</div>

<div class="body-details-section r6c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-362"></div> <span class="help-tooltip" id="cb-known-help-362">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c3_1.svg">
tenellin
</option>

<option value = "svg/knownclusterblast_r6c3_2.svg">
desmethylbassianin
</option>

<option value = "svg/knownclusterblast_r6c3_3.svg">
fumosorinone
</option>

<option value = "svg/knownclusterblast_r6c3_4.svg">
ilicicolin H
</option>

<option value = "svg/knownclusterblast_r6c3_5.svg">
ilicicolin H/ilicicolin J/8-epi-ilicicolin H
</option>

</select>
<button id="knownclusterblast-r6c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c3-svg"></div>

</div>
</div>

<div class="body-details-section r6c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-363"></div> <span class="help-tooltip" id="cb-sub-help-363">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-366"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-366">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-domain-bubble-svg" data-filename="JAAZTN010000006.1_r6c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (373964 - 426949): chemical hybrid T1PKS-NRPS
</option>


</select>

<div class="details-svg" id="r6c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-368"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-368">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-369"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-369">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-tigrfam-details-svg" data-filename="JAAZTN010000006.1_r6c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c3-nrps_pks_products" data-name="r6c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c3-nrps_pks_monomers" data-name="r6c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-364"></div> <span class="help-tooltip" id="nrps-structures-help-364">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 373964 - 426949:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c3-3" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC(Cc1ccc(O)cc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk - Tyr)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk,tyr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*,tyr*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC(Cc1ccc(O)cc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-365"></div> <span class="help-tooltip" id="nrps-monomers-help-365">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene106</span></strong>: pk - Tyr
<div class="collapser collapser-target-input-path1-gene106 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>PKS_AT (586..911)</strong>: pk
<div class="collapser collapser-target-input-path1-gene106 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene106-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>2-Methylbutyryl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene106-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 90.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 71.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 54.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 51.3</dt>

<dd></dd><dt>Benzoyl-CoA: 49.4</dt>

<dd></dd><dt>Propionyl-CoA: 36.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 33.1</dt>

<dd></dd><dt>Malonyl-CoA: 24.7</dt>

<dd></dd><dt>fatty_acid: 22.2</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 20.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.2</dt>

<dd></dd><dt>CHC-CoA: 16.6</dt>

<dd></dd><dt>Acetyl-CoA: 2.3</dt>

<dd></dd><dt>inactive: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>













<dd><strong>AMP-binding (3200..3625)</strong>: Tyr
<div class="collapser collapser-target-input-path1-gene106 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Tyr
<div class="collapser collapser-target-input-path1-gene106-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMVICGCAAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DMVICGCAAK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 4 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-537"></div> <span class="help-tooltip" id="region_svg-help-537">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c4-svg" data-filename="JAAZTN010000006.1_r6c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 594,524 - 657,071 nt. (total: 62,548 nt)

<a class="cluster-rules-header" id="r6c4-rules-header" href="#r6c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c4-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r6c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-538"></div> <span class="help-tooltip" id="focus-panel-help-538">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c4-gene-table-details" data-name="r6c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c4-MIBiG-cluster-compare" data-name="r6c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c4-clusterblast" data-name="r6c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c4-knownclusterblast" data-name="r6c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c4-subclusterblast" data-name="r6c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c4-pfam-details" data-name="r6c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-374"></div> <span class="help-tooltip" id="gt-help-tooltip-help-374">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-370"></div> <span class="help-tooltip" id="MIBiG-help-370">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c4"></div>
<div id="comparison-MIBiG-r6c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>betaestacin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>AK-toxin I</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:0.48, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>AK-toxin I</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:0.48, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-371"></div> <span class="help-tooltip" id="cb-general-help-371">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c4_1.svg">
NW_025953816 (281521-436136): Annulohypoxylon truncatum strain CBS 140777 unp... (5% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_2.svg">
NW_025953267 (201738-434522): Annulohypoxylon maeteangense strain CBS 123835 ... (3% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_3.svg">
NW_020194479 (1449493-1553698): Amorphotheca resinae ATCC 22711 unplaced geno... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_4.svg">
NW_006917091 (6617355-6715135): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_5.svg">
NW_017306195 (991961-1083838): Alternaria alternata strain SRC1lrK2f unplaced... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_6.svg">
NW_022987223 (375403-544458): Trematosphaeria pertusa strain CBS 122368 unpla... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_7.svg">
NW_022983484 (349700-470118): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_8.svg">
NW_024065901 (1362618-1459951): Colletotrichum karsti strain CkLH20 chromosom... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_9.svg">
NW_019154045 (358401-688746): Pochonia chlamydosporia 170 chromosome Unknown ... (2% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_10.svg">
NW_015971648 (293686-402495): Fonsecaea multimorphosa CBS 102226 unplaced gen... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c4-download">Download graphic</button>
</div>
<div id="clusterblast-r6c4-svg"></div>

</div>
</div>

<div class="body-details-section r6c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-372"></div> <span class="help-tooltip" id="cb-known-help-372">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-373"></div> <span class="help-tooltip" id="cb-sub-help-373">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-375"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-375">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c4-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r6c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 5 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-539"></div> <span class="help-tooltip" id="region_svg-help-539">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c5-svg" data-filename="JAAZTN010000006.1_r6c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 839,905 - 884,155 nt. (total: 44,251 nt)

<a class="cluster-rules-header" id="r6c5-rules-header" href="#r6c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c5-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-540"></div> <span class="help-tooltip" id="focus-panel-help-540">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c5-gene-table-details" data-name="r6c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c5-nrps_pks" data-name="r6c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c5-MIBiG-cluster-compare" data-name="r6c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c5-clusterblast" data-name="r6c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c5-knownclusterblast" data-name="r6c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c5-subclusterblast" data-name="r6c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c5-pfam-details" data-name="r6c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c5-tigrfam-details" data-name="r6c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-382"></div> <span class="help-tooltip" id="gt-help-tooltip-help-382">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-376"></div> <span class="help-tooltip" id="nrps-domain-help-376">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-details-svg" data-filename="JAAZTN010000006.1_r6c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-377"></div> <span class="help-tooltip" id="MIBiG-help-377">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c5"></div>
<div id="comparison-MIBiG-r6c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404: 1968-56662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>penicillin</td>


<td>Penicillium chrysogenum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>xenematide</td>


<td>Xenorhabdus nematophila AN6/1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404: 1968-56662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>penicillin</td>
<td>Penicillium chrysogenum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>xenematide</td>
<td>Xenorhabdus nematophila AN6/1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-378"></div> <span class="help-tooltip" id="cb-general-help-378">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c5_1.svg">
NW_006271969 (2236886-2301135): Cordyceps militaris CM01 unplaced genomic sca... (36% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_2.svg">
NZ_CP091381 (1436862-1565580): Amycolatopsis sp. FU40 chromosome, complete ge... (3% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r6c5_3.svg">
NW_020194479 (261473-327722): Amorphotheca resinae ATCC 22711 unplaced genomi... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_4.svg">
NW_025091284 (1488387-1554717): Aspergillus pseudoviridinutans IFM 55266 DNA,... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_5.svg">
NW_025091224 (385124-451454): Aspergillus viridinutans IFM 47045 DNA, scaffol... (11% of genes show similarity), NI-siderophore,NRPS
</option>

<option value = "svg/clusterblast_r6c5_6.svg">
NZ_LT607413 (4729322-4815787): Micromonospora echinospora strain DSM 43816 ch... (3% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r6c5_7.svg">
NZ_PNXQ01000016 (500910-548524): Paenibacillus terrae strain PIC167 Contig 01... (5% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r6c5_8.svg">
NZ_JPMW01000001 (6405501-6473515): Planobispora rosea strain ATCC 53733 scaff... (4% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r6c5_9.svg">
NZ_MAUJ01000002 (439421-527978): Pseudoalteromonas luteoviolacea strain IPB1 ... (3% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r6c5_10.svg">
NW_006271969 (1967760-2079673): Cordyceps militaris CM01 unplaced genomic sca... (6% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r6c5-download">Download graphic</button>
</div>
<div id="clusterblast-r6c5-svg"></div>

</div>
</div>

<div class="body-details-section r6c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-379"></div> <span class="help-tooltip" id="cb-known-help-379">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-380"></div> <span class="help-tooltip" id="cb-sub-help-380">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-383"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-383">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-384"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-384">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-tigrfam-details-svg" data-filename="JAAZTN010000006.1_r6c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c5-nrps_pks_monomers" data-name="r6c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r6c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-381"></div> <span class="help-tooltip" id="nrps-monomers-help-381">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene240</span></strong>: X
<div class="collapser collapser-target-input-path1-gene240 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (217..652)</strong>: X
<div class="collapser collapser-target-input-path1-gene240 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene240-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLMFVGIMCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAMLVGAACK</span> (59% 8Å match)</dd>
<dd>Hty <span class="serif">DLGFTGAVCK</span> (56% 8Å match)</dd>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r6c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 6 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-541"></div> <span class="help-tooltip" id="region_svg-help-541">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c6-svg" data-filename="JAAZTN010000006.1_r6c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,097,978 - 3,145,368 nt. (total: 47,391 nt)

<a class="cluster-rules-header" id="r6c6-rules-header" href="#r6c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c6-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r6c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-542"></div> <span class="help-tooltip" id="focus-panel-help-542">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c6-gene-table-details" data-name="r6c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c6-nrps_pks" data-name="r6c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c6-MIBiG-cluster-compare" data-name="r6c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c6-clusterblast" data-name="r6c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c6-knownclusterblast" data-name="r6c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c6-subclusterblast" data-name="r6c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c6-nrps-pks-bubbles" data-name="r6c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c6-pfam-details" data-name="r6c6-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-393"></div> <span class="help-tooltip" id="gt-help-tooltip-help-393">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-385"></div> <span class="help-tooltip" id="nrps-domain-help-385">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-details-svg" data-filename="JAAZTN010000006.1_r6c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-386"></div> <span class="help-tooltip" id="MIBiG-help-386">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c6"></div>
<div id="comparison-MIBiG-r6c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>scytalidin</td>


<td>Scytalidium album</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>scytalidin</td>
<td>Scytalidium album</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-387"></div> <span class="help-tooltip" id="cb-general-help-387">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c6_1.svg">
NW_024468938 (2652248-2719890): Cucurbitaria berberidis CBS 394.84 unplaced g... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c6_2.svg">
NW_022985440 (2305586-2373284): Mytilinidion resinicola strain CBS 304.34 unp... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c6_3.svg">
NW_025091222 (676393-744069): Aspergillus viridinutans IFM 47045 DNA, scaffol... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c6_4.svg">
NW_020290894 (433814-525041): Aspergillus niger CBS 101883 unplaced genomic s... (10% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r6c6_5.svg">
NW_020291608 (94205-186802): Aspergillus piperis CBS 112811 unplaced genomic ... (10% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c6_6.svg">
NW_020292048 (78288-172374): Aspergillus vadensis CBS 113365 unplaced genomic... (8% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c6_7.svg">
NW_023336280 (65196-169882): Aspergillus tubingensis WU-2223L DNA, scaffold 6... (8% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c6_8.svg">
NW_020291836 (40952-145610): Aspergillus costaricaensis CBS 115574 unplaced g... (8% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c6_9.svg">
NC_054849 (78324-143361): Aspergillus luchuensis IFO 4308 DNA, chromosome 1, ... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c6_10.svg">
NW_017264199 (1073767-1141168): Xylona heveae TC161 unplaced genomic scaffold... (13% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r6c6-download">Download graphic</button>
</div>
<div id="clusterblast-r6c6-svg"></div>

</div>
</div>

<div class="body-details-section r6c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-388"></div> <span class="help-tooltip" id="cb-known-help-388">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-389"></div> <span class="help-tooltip" id="cb-sub-help-389">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-392"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-392">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-domain-bubble-svg" data-filename="JAAZTN010000006.1_r6c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (3097977 - 3145368): single T1PKS
</option>


</select>

<div class="details-svg" id="r6c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-394"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-394">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c6-nrps_pks_products" data-name="r6c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c6-nrps_pks_monomers" data-name="r6c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-390"></div> <span class="help-tooltip" id="nrps-structures-help-390">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 3097977 - 3145368:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c6-6" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c6-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-391"></div> <span class="help-tooltip" id="nrps-monomers-help-391">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene830</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene830 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (393..699)</strong>: pk
<div class="collapser collapser-target-input-path1-gene830 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene830-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Methoxymalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene830-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 86.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 72.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 50.8</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 49.7</dt>

<dd></dd><dt>Propionyl-CoA: 45.4</dt>

<dd></dd><dt>Malonyl-CoA: 41.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 33.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 30.0</dt>

<dd></dd><dt>CHC-CoA: 28.2</dt>

<dd></dd><dt>fatty_acid: 20.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 19.6</dt>

<dd></dd><dt>Benzoyl-CoA: 15.4</dt>

<dd></dd><dt>inactive: 11.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 7 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-543"></div> <span class="help-tooltip" id="region_svg-help-543">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c7-svg" data-filename="JAAZTN010000006.1_r6c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,570,863 - 3,631,695 nt. (total: 60,833 nt)

<a class="cluster-rules-header" id="r6c7-rules-header" href="#r6c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c7-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r6c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-544"></div> <span class="help-tooltip" id="focus-panel-help-544">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c7-gene-table-details" data-name="r6c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c7-MIBiG-cluster-compare" data-name="r6c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c7-clusterblast" data-name="r6c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c7-knownclusterblast" data-name="r6c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c7-subclusterblast" data-name="r6c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c7-pfam-details" data-name="r6c7-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-399"></div> <span class="help-tooltip" id="gt-help-tooltip-help-399">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-395"></div> <span class="help-tooltip" id="MIBiG-help-395">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<div><p>No matches found.</p></div>

</div>
</div>



<div class="body-details-section r6c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-396"></div> <span class="help-tooltip" id="cb-general-help-396">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c7_1.svg">
NW_006271973 (266414-357915): Cordyceps militaris CM01 unplaced genomic scaff... (31% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c7_2.svg">
NW_025953267 (201738-434522): Annulohypoxylon maeteangense strain CBS 123835 ... (2% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c7_3.svg">
NC_072350 (525276-617173): Talaromyces marneffei chromosome 3, complete sequence (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c7_4.svg">
NW_022985510 (2150254-2227692): Neohortaea acidophila strain CBS 113389 unpla... (6% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r6c7_5.svg">
NW_022985463 (11775-102765): Mytilinidion resinicola strain CBS 304.34 unplac... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c7_6.svg">
NW_017253367 (1229921-1261277): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c7_7.svg">
NW_024467832 (4333687-4396473): Aspergillus wentii DTO 134E9 unplaced genomic... (6% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r6c7-download">Download graphic</button>
</div>
<div id="clusterblast-r6c7-svg"></div>

</div>
</div>

<div class="body-details-section r6c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-397"></div> <span class="help-tooltip" id="cb-known-help-397">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-398"></div> <span class="help-tooltip" id="cb-sub-help-398">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-400"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-400">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c7-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r6c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 8 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-545"></div> <span class="help-tooltip" id="region_svg-help-545">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c8-svg" data-filename="JAAZTN010000006.1_r6c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,813,802 - 3,836,575 nt. (total: 22,774 nt)

<a class="cluster-rules-header" id="r6c8-rules-header" href="#r6c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c8-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r6c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-546"></div> <span class="help-tooltip" id="focus-panel-help-546">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c8-gene-table-details" data-name="r6c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c8-MIBiG-cluster-compare" data-name="r6c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c8-clusterblast" data-name="r6c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c8-knownclusterblast" data-name="r6c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c8-subclusterblast" data-name="r6c8-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r6c8-pfam-details" data-name="r6c8-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-405"></div> <span class="help-tooltip" id="gt-help-tooltip-help-405">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-401"></div> <span class="help-tooltip" id="MIBiG-help-401">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c8"></div>
<div id="comparison-MIBiG-r6c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.46, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:0.69, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>preaspterpenacid I</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>aspterric acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002643: 0-9229">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002643/index.html#r1c1" target="_new">BGC0002643</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Alkaloid</td>

<td>aetokthonotoxin</td>


<td>Aetokthonos hydrillicola Thurmond2011</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>pimara-8(14),15-diene</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.46, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.46, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:0.69, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>preaspterpenacid I</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>aspterric acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002643: 0-9229">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002643/index.html#r1c1" target="_new">BGC0002643</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Alkaloid</td>
<td>aetokthonotoxin</td>
<td>Aetokthonos hydrillicola Thurmond2011</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>pimara-8(14),15-diene</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.46, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-402"></div> <span class="help-tooltip" id="cb-general-help-402">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c8_1.svg">
NW_025091271 (301767-334078): Aspergillus udagawae IFM 46973 DNA, scaffold 8,... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c8_2.svg">
NC_036439 (2814767-2846485): Aspergillus oryzae RIB40 DNA, chromosome 5 (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c8_3.svg">
NC_054695 (2772365-2804826): Aspergillus flavus NRRL3357 chromosome 5, comple... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c8_4.svg">
NC_049563 (446618-479408): Talaromyces rugulosus chromosome III, complete seq... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c8_5.svg">
NW_020291503 (31867-64708): Aspergillus aculeatinus CBS 121060 unplaced genom... (22% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c8_6.svg">
NW_022987219 (1855256-1946234): Trematosphaeria pertusa strain CBS 122368 unp... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c8_7.svg">
NC_072348 (5283352-5357754): Talaromyces marneffei chromosome 1, complete seq... (8% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r6c8_8.svg">
NW_022983490 (330572-424081): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (6% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r6c8_9.svg">
NW_026054585 (158434-189521): Daldinia decipiens strain CBS 113046 unplaced g... (16% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c8_10.svg">
NW_025953788 (1126759-1218489): Annulohypoxylon truncatum strain CBS 140777 u... (6% of genes show similarity), NRPS,indole,terpene
</option>

</select>
<button id= "clusterblast-r6c8-download">Download graphic</button>
</div>
<div id="clusterblast-r6c8-svg"></div>

</div>
</div>

<div class="body-details-section r6c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-403"></div> <span class="help-tooltip" id="cb-known-help-403">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-404"></div> <span class="help-tooltip" id="cb-sub-help-404">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r6c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-406"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-406">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c8-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r6c9' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000006.1 - Region 9 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-547"></div> <span class="help-tooltip" id="region_svg-help-547">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000006.1.region009.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c9-svg" data-filename="JAAZTN010000006.1_r6c9.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,976,721 - 4,007,226 nt. (total: 30,506 nt)

<a class="cluster-rules-header" id="r6c9-rules-header" href="#r6c9">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r6c9-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r6c9-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-548"></div> <span class="help-tooltip" id="focus-panel-help-548">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c9">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c9-gene-table-details" data-name="r6c9-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c9-nrps_pks" data-name="r6c9-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c9-MIBiG-cluster-compare" data-name="r6c9-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c9-clusterblast" data-name="r6c9-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c9-knownclusterblast" data-name="r6c9-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c9-subclusterblast" data-name="r6c9-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c9-nrps-pks-bubbles" data-name="r6c9-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c9-pfam-details" data-name="r6c9-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c9-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-415"></div> <span class="help-tooltip" id="gt-help-tooltip-help-415">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c9">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c9-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c9-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c9">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c9-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-407"></div> <span class="help-tooltip" id="nrps-domain-help-407">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c9-details-svg" data-filename="JAAZTN010000006.1_r6c9_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c9-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c9-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-408"></div> <span class="help-tooltip" id="MIBiG-help-408">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c9-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c9-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c9"></div>
<div id="comparison-MIBiG-r6c9-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.51, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002139: 487-22086">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002139/index.html#r1c1" target="_new">BGC0002139</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>burnettiene A, preburnettiene B, preburnettiene A</td>


<td>Aspergillus burnettii FRR 5400</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001305: 0-24253">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001305/index.html#r1c1" target="_new">BGC0001305</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>fujikurin A, fujikurin B, fujikurin C, fujikurin D</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002185: 100-28222">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002185/index.html#r1c1" target="_new">BGC0002185</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>gregatin A</td>


<td>Penicillium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001907: 74-38528">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001907/index.html#r1c1" target="_new">BGC0001907</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>chromane</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c9-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.51, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002139: 487-22086">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002139/index.html#r1c1" target="_new">BGC0002139</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>burnettiene A, preburnettiene B, preburnettiene A</td>
<td>Aspergillus burnettii FRR 5400</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001305: 0-24253">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001305/index.html#r1c1" target="_new">BGC0001305</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>fujikurin A, fujikurin B, fujikurin C, fujikurin D</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002185: 100-28222">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002185/index.html#r1c1" target="_new">BGC0002185</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>gregatin A</td>
<td>Penicillium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001907: 74-38528">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001907/index.html#r1c1" target="_new">BGC0001907</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>chromane</td>
<td>Daldinia eschscholzii IFB-TL01</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c9-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-409"></div> <span class="help-tooltip" id="cb-general-help-409">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c9-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c9_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c9_1.svg">
NW_020291503 (98376-258038): Aspergillus aculeatinus CBS 121060 unplaced geno... (4% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_2.svg">
NW_024468981 (193798-354462): Aspergillus fijiensis CBS 313.89 unplaced genom... (4% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_3.svg">
NW_026054642 (106430-178539): Ophidiomyces ophidiicola strain CBS 122913 chro... (23% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_4.svg">
NW_017263586 (566702-639094): Mollisia scopiformis unplaced genomic scaffold ... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_5.svg">
NW_025091265 (569809-675431): Aspergillus udagawae IFM 46973 DNA, scaffold 2,... (7% of genes show similarity), NI-siderophore,NRPS,RCDPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_6.svg">
NW_025091284 (549444-621270): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_7.svg">
NW_006711153 (4745-99477): Trichoderma reesei QM6a unplaced genomic scaffold ... (11% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_8.svg">
NC_070535 (927174-999058): Fusarium keratoplasticum isolate Fu6.1 chromosome ... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_9.svg">
NC_070550 (2172664-2244548): Fusarium falciforme chromosome 7, complete sequence (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c9_10.svg">
NW_011942171 (5060591-5132457): Metarhizium robertsii ARSEF 23 MAA Scf 3, who... (14% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r6c9-download">Download graphic</button>
</div>
<div id="clusterblast-r6c9-svg"></div>

</div>
</div>

<div class="body-details-section r6c9-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-410"></div> <span class="help-tooltip" id="cb-known-help-410">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c9-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-411"></div> <span class="help-tooltip" id="cb-sub-help-411">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c9-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-414"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-414">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c9-domain-bubble-svg" data-filename="JAAZTN010000006.1_r6c9_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c9-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC9">
Candidate 9 (3976720 - 4007226): single T1PKS
</option>


</select>

<div class="details-svg" id="r6c9-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c9-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c9-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-416"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-416">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c9-pfam-details-svg" data-filename="JAAZTN010000006.1_r6c9_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c9-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c9-nrps_pks_products" data-name="r6c9-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c9-nrps_pks_monomers" data-name="r6c9-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c9-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-412"></div> <span class="help-tooltip" id="nrps-structures-help-412">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 9, location 3976720 - 4007226:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c9-9" class="smiles-canvas" data-smiles="C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-Me-ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-me-ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-me-ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c9-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c9-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-413"></div> <span class="help-tooltip" id="nrps-monomers-help-413">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1054</span></strong>: mmal
<div class="collapser collapser-target-input-path1-gene1054 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (450..642)</strong>: mmal
<div class="collapser collapser-target-input-path1-gene1054 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-input-path1-gene1054-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1054-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 43.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 32.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 22.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 20.1</dt>

<dd></dd><dt>fatty_acid: 18.4</dt>

<dd></dd><dt>Propionyl-CoA: 18.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 14.6</dt>

<dd></dd><dt>Malonyl-CoA: 13.8</dt>

<dd></dd><dt>CHC-CoA: 11.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 10.9</dt>

<dd></dd><dt>Benzoyl-CoA: 9.3</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r7c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000007.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-549"></div> <span class="help-tooltip" id="region_svg-help-549">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000007.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c1-svg" data-filename="JAAZTN010000007.1_r7c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 44,679 - 65,809 nt. (total: 21,131 nt)

<a class="cluster-rules-header" id="r7c1-rules-header" href="#r7c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r7c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-550"></div> <span class="help-tooltip" id="focus-panel-help-550">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c1-gene-table-details" data-name="r7c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c1-MIBiG-cluster-compare" data-name="r7c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c1-clusterblast" data-name="r7c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c1-knownclusterblast" data-name="r7c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c1-subclusterblast" data-name="r7c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r7c1-pfam-details" data-name="r7c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-421"></div> <span class="help-tooltip" id="gt-help-tooltip-help-421">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-417"></div> <span class="help-tooltip" id="MIBiG-help-417">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c1"></div>
<div id="comparison-MIBiG-r7c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>pimara-8(14),15-diene</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.54, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>sartorypyrone A</td>


<td>Aspergillus felis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.51, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Terpene, Polyketide</td>

<td>subglutinol A, subglutinol B</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.50, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>


<td>Penicillium paxilli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.52, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>chevalone E</td>


<td>Aspergillus versicolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.51, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>shearinine D</td>


<td>Penicillium janthinellum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>pimara-8(14),15-diene</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.54, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>sartorypyrone A</td>
<td>Aspergillus felis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.51, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Terpene, Polyketide</td>
<td>subglutinol A, subglutinol B</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.50, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Terpene</td>
<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>
<td>Penicillium paxilli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.52, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>chevalone E</td>
<td>Aspergillus versicolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.51, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Terpene</td>
<td>shearinine D</td>
<td>Penicillium janthinellum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-418"></div> <span class="help-tooltip" id="cb-general-help-418">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c1_1.svg">
NW_006271971 (242847-274242): Cordyceps militaris CM01 unplaced genomic scaff... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_2.svg">
NC_070533 (2808830-2840090): Fusarium keratoplasticum isolate Fu6.1 chromosom... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_3.svg">
NC_070548 (2879257-2910520): Fusarium falciforme chromosome 5, complete sequence (22% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_4.svg">
NC_058399 (9867240-9898491): Fusarium poae strain DAOMC 252244 chromosome 1, ... (22% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_5.svg">
NC_058390 (2633992-2665249): Fusarium musae strain F31 chromosome 4, whole ge... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_6.svg">
NC_072844 (1849949-1881206): Fusarium oxysporum Fo47 chromosome V, complete s... (22% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_7.svg">
NW_006711170 (271878-302853): Trichoderma reesei QM6a unplaced genomic scaffo... (28% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_8.svg">
NC_036626 (1833202-1864459): Fusarium fujikuroi IMI 58289 draft genome, chrom... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_9.svg">
NC_038012 (9811194-9842445): Fusarium venenatum strain A3/5 genome assembly, ... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c1_10.svg">
NC_057321 (2150636-2181917): Ustilaginoidea virens chromosome 6, complete seq... (18% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r7c1-download">Download graphic</button>
</div>
<div id="clusterblast-r7c1-svg"></div>

</div>
</div>

<div class="body-details-section r7c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-419"></div> <span class="help-tooltip" id="cb-known-help-419">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-420"></div> <span class="help-tooltip" id="cb-sub-help-420">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r7c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-422"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-422">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-pfam-details-svg" data-filename="JAAZTN010000007.1_r7c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r7c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000007.1 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-551"></div> <span class="help-tooltip" id="region_svg-help-551">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000007.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c2-svg" data-filename="JAAZTN010000007.1_r7c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,458,226 - 1,503,935 nt. (total: 45,710 nt)

<a class="cluster-rules-header" id="r7c2-rules-header" href="#r7c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r7c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-552"></div> <span class="help-tooltip" id="focus-panel-help-552">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c2-gene-table-details" data-name="r7c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c2-nrps_pks" data-name="r7c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c2-MIBiG-cluster-compare" data-name="r7c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c2-clusterblast" data-name="r7c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c2-knownclusterblast" data-name="r7c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c2-subclusterblast" data-name="r7c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c2-nrps-pks-bubbles" data-name="r7c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c2-pfam-details" data-name="r7c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c2-tigrfam-details" data-name="r7c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-431"></div> <span class="help-tooltip" id="gt-help-tooltip-help-431">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-423"></div> <span class="help-tooltip" id="nrps-domain-help-423">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-details-svg" data-filename="JAAZTN010000007.1_r7c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-424"></div> <span class="help-tooltip" id="MIBiG-help-424">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c2"></div>
<div id="comparison-MIBiG-r7c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.55, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.55, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-425"></div> <span class="help-tooltip" id="cb-general-help-425">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c2_1.svg">
NW_006271973 (996861-1062288): Cordyceps militaris CM01 unplaced genomic scaf... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_2.svg">
NW_011942160 (3770676-3836079): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_3.svg">
NW_014574716 (6263628-6329031): Metarhizium brunneum ARSEF 3297 chromosome Un... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_4.svg">
NC_057319 (2334015-2399373): Ustilaginoidea virens chromosome 4, complete seq... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_5.svg">
NC_035791 (4601700-4667019): Pochonia chlamydosporia 170 chromosome 2, whole ... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_6.svg">
NW_006711167 (506611-572053): Trichoderma reesei QM6a unplaced genomic scaffo... (64% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_7.svg">
NC_070536 (2319263-2385135): Fusarium keratoplasticum isolate Fu6.1 chromosom... (56% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_8.svg">
NW_014013664 (48182-113471): Trichoderma virens Gv29-8 chromosome Unknown TRI... (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_9.svg">
NC_070551 (2536076-2601948): Fusarium falciforme chromosome 8, complete sequence (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_10.svg">
NW_021167080 (1267789-1333877): Sodiomyces alkalinus F11 unplaced genomic sca... (50% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r7c2-download">Download graphic</button>
</div>
<div id="clusterblast-r7c2-svg"></div>

</div>
</div>

<div class="body-details-section r7c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-426"></div> <span class="help-tooltip" id="cb-known-help-426">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r7c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r7c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r7c2_1.svg">
metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid
</option>

<option value = "svg/knownclusterblast_r7c2_2.svg">
dimethylcoprogen
</option>

</select>
<button id="knownclusterblast-r7c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r7c2-svg"></div>

</div>
</div>

<div class="body-details-section r7c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-427"></div> <span class="help-tooltip" id="cb-sub-help-427">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-430"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-430">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-domain-bubble-svg" data-filename="JAAZTN010000007.1_r7c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (1458225 - 1503935): single NRPS
</option>


</select>

<div class="details-svg" id="r7c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-432"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-432">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-pfam-details-svg" data-filename="JAAZTN010000007.1_r7c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-433"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-433">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-tigrfam-details-svg" data-filename="JAAZTN010000007.1_r7c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c2-nrps_pks_products" data-name="r7c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c2-nrps_pks_monomers" data-name="r7c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-428"></div> <span class="help-tooltip" id="nrps-structures-help-428">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 1458225 - 1503935:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-429"></div> <span class="help-tooltip" id="nrps-monomers-help-429">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene335</span></strong>: X
<div class="collapser collapser-target-input-path1-gene335 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (55..461)</strong>: X
<div class="collapser collapser-target-input-path1-gene335 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Z-ahmohOrn
<div class="collapser collapser-target-input-path1-gene335-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">FGLHIASTCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">FGLHIASTCK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene336</span></strong>: X
<div class="collapser collapser-target-input-path1-gene336 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (107..507)</strong>: X
<div class="collapser collapser-target-input-path1-gene336 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene336-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVDHGGGVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">DVDGGGGIGK</span> (76% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000007.1 - Region 3 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-553"></div> <span class="help-tooltip" id="region_svg-help-553">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000007.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c3-svg" data-filename="JAAZTN010000007.1_r7c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,369,824 - 2,455,224 nt. (total: 85,401 nt)

<a class="cluster-rules-header" id="r7c3-rules-header" href="#r7c3">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r7c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r7c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-554"></div> <span class="help-tooltip" id="focus-panel-help-554">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c3-gene-table-details" data-name="r7c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c3-nrps_pks" data-name="r7c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c3-MIBiG-cluster-compare" data-name="r7c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c3-clusterblast" data-name="r7c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c3-knownclusterblast" data-name="r7c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c3-subclusterblast" data-name="r7c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c3-nrps-pks-bubbles" data-name="r7c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c3-pfam-details" data-name="r7c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-442"></div> <span class="help-tooltip" id="gt-help-tooltip-help-442">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-434"></div> <span class="help-tooltip" id="nrps-domain-help-434">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-details-svg" data-filename="JAAZTN010000007.1_r7c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-435"></div> <span class="help-tooltip" id="MIBiG-help-435">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c3"></div>
<div id="comparison-MIBiG-r7c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.95</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.84</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.78</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.77</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.75</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.73</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.72</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001811: 32-25672">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001811/index.html#r1c1" target="_new">BGC0001811</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.70</td>
<td>Terpene</td>

<td>trichodiene-11-one</td>


<td>Fusarium asiaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.69</td>
<td>Polyketide</td>

<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>


<td>Chaetomium olivaceum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000120: 194-40016">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000120/index.html#r1c1" target="_new">BGC0000120</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>patulin</td>
<td>Penicillium expansum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>AK-toxin I</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>6-MSA, aculin A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-436"></div> <span class="help-tooltip" id="cb-general-help-436">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c3_1.svg">
NC_054862 (2647617-2714965): Aspergillus puulaauensis MK2 DNA, chromosome 6, ... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c3_2.svg">
NW_020194484 (7340-83776): Amorphotheca resinae ATCC 22711 unplaced genomic s... (41% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r7c3_3.svg">
NW_022474205 (7019281-7087036): Venustampulla echinocandica strain BP 5553 ch... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c3_4.svg">
NW_026249875 (10804-78109): Alternaria incomplexa strain BMP 0042 chromosome ... (38% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c3_5.svg">
NW_025091281 (4887733-5033357): Aspergillus pseudoviridinutans IFM 55266 DNA,... (12% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_6.svg">
NW_025091265 (865026-937041): Aspergillus udagawae IFM 46973 DNA, scaffold 2,... (23% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_7.svg">
NW_006271974 (2321315-2405136): Cordyceps militaris CM01 unplaced genomic sca... (40% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_8.svg">
NW_026249941 (3336813-3404118): Alternaria arbusti strain BMP 1465 chromosome... (40% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c3_9.svg">
NW_026055699 (512819-580124): Alternaria ventricosa strain BMP 2768 chromosom... (32% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c3_10.svg">
NC_049565 (5160731-5287596): Talaromyces rugulosus chromosome V, complete seq... (25% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r7c3-download">Download graphic</button>
</div>
<div id="clusterblast-r7c3-svg"></div>

</div>
</div>

<div class="body-details-section r7c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-437"></div> <span class="help-tooltip" id="cb-known-help-437">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r7c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r7c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r7c3_1.svg">
4-epi-15-epi-brefeldin A
</option>

</select>
<button id="knownclusterblast-r7c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r7c3-svg"></div>

</div>
</div>

<div class="body-details-section r7c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-438"></div> <span class="help-tooltip" id="cb-sub-help-438">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-441"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-441">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-domain-bubble-svg" data-filename="JAAZTN010000007.1_r7c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (2369823 - 2455224): neighbouring T1PKS-NRPS
</option>





<option value="CC5">
Candidate 5 (2411534 - 2455224): chemical hybrid T1PKS-NRPS
</option>


</select>

<div class="details-svg" id="r7c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-443"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-443">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-pfam-details-svg" data-filename="JAAZTN010000007.1_r7c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c3-nrps_pks_products" data-name="r7c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c3-nrps_pks_monomers" data-name="r7c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-439"></div> <span class="help-tooltip" id="nrps-structures-help-439">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 2369823 - 2455224:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c3-3" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>






<div>
<div class="prediction-text">
For candidate cluster 5, location 2411534 - 2455224:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c3-5" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c3-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-440"></div> <span class="help-tooltip" id="nrps-monomers-help-440">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene536</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene536 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (570..881)</strong>: pk
<div class="collapser collapser-target-input-path1-gene536 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene536-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>2-Methylbutyryl-CoA: 62.5%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene536-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 66.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 48.2</dt>

<dd></dd><dt>Malonyl-CoA: 31.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 26.7</dt>

<dd></dd><dt>Propionyl-CoA: 22.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 18.2</dt>

<dd></dd><dt>Benzoyl-CoA: 15.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 15.1</dt>

<dd></dd><dt>fatty_acid: 11.3</dt>

<dd></dd><dt>CHC-CoA: 11.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 10.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 9.1</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">input.path1.gene542</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene542 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (566..880)</strong>: pk
<div class="collapser collapser-target-input-path1-gene542 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene542-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene542-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 75.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 57.9</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 42.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 33.1</dt>

<dd></dd><dt>Propionyl-CoA: 32.6</dt>

<dd></dd><dt>Benzoyl-CoA: 32.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 22.3</dt>

<dd></dd><dt>Malonyl-CoA: 19.3</dt>

<dd></dd><dt>fatty_acid: 15.4</dt>

<dd></dd><dt>CHC-CoA: 14.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 14.7</dt>

<dd></dd><dt>Acetyl-CoA: 13.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 7.1</dt>

<dd></dd><dt>inactive: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2941..3345)</strong>: X
<div class="collapser collapser-target-input-path1-gene542 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene542-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAALMGAVAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>dh-Trp <span class="serif">DAALMGAVTK</span> (44% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r8c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JAAZTN010000008.1 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-555"></div> <span class="help-tooltip" id="region_svg-help-555">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JAAZTN010000008.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r8c1-svg" data-filename="JAAZTN010000008.1_r8c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 592,927 - 637,723 nt. (total: 44,797 nt)

<a class="cluster-rules-header" id="r8c1-rules-header" href="#r8c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r8c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r8c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-556"></div> <span class="help-tooltip" id="focus-panel-help-556">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r8c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r8c1-gene-table-details" data-name="r8c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r8c1-nrps_pks" data-name="r8c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r8c1-MIBiG-cluster-compare" data-name="r8c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r8c1-clusterblast" data-name="r8c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r8c1-knownclusterblast" data-name="r8c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r8c1-subclusterblast" data-name="r8c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r8c1-nrps-pks-bubbles" data-name="r8c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r8c1-pfam-details" data-name="r8c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r8c1-tigrfam-details" data-name="r8c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r8c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-452"></div> <span class="help-tooltip" id="gt-help-tooltip-help-452">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r8c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r8c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r8c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r8c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r8c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-444"></div> <span class="help-tooltip" id="nrps-domain-help-444">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c1-details-svg" data-filename="JAAZTN010000008.1_r8c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r8c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r8c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-445"></div> <span class="help-tooltip" id="MIBiG-help-445">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r8c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r8c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r8c1"></div>
<div id="comparison-MIBiG-r8c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>serinocyclin A, serinocyclin B</td>


<td>Metarhizium robertsii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001679: 12-12481">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001679/index.html#r1c1" target="_new">BGC0001679</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>N-Acetyltryptophan</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>xenortide A, xenortide B, xenortide C, xenortide D</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r8c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>serinocyclin A, serinocyclin B</td>
<td>Metarhizium robertsii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001679: 12-12481">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001679/index.html#r1c1" target="_new">BGC0001679</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>N-Acetyltryptophan</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>xenortide A, xenortide B, xenortide C, xenortide D</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r8c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-446"></div> <span class="help-tooltip" id="cb-general-help-446">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r8c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r8c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r8c1_1.svg">
NZ_CP017172 (4365701-4538043): Myxococcus xanthus strain KF4.3.9c1 chromosome... (6% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r8c1_2.svg">
NZ_CP114039 (5092993-5212306): Corallococcus sp. NCRR chromosome, complete ge... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_3.svg">
NZ_CP069396 (4617537-4766022): Archangium violaceum strain SDU8 chromosome, c... (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r8c1_4.svg">
NZ_JAIGPR010000001 (311775-488049): Pseudomonas cichorii strain LCYJ08 Scaffo... (3% of genes show similarity), NRP-metallophore,NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r8c1_5.svg">
NC_058401 (7711183-7779338): Fusarium poae strain DAOMC 252244 chromosome 3, ... (15% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r8c1_6.svg">
NZ_CP042220 (1139263-1263144): Dickeya poaceiphila strain NCPPB 569 chromosom... (3% of genes show similarity), NRP-metallophore,NRPS,transAT-PKS
</option>

<option value = "svg/clusterblast_r8c1_7.svg">
NZ_CM001858 (1438873-1515910): Dickeya zeae NCPPB 3532 chromosome, whole geno... (6% of genes show similarity), NRPS,transAT-PKS
</option>

<option value = "svg/clusterblast_r8c1_8.svg">
NZ_JAIZGC010000028 (45809-122833): Dickeya oryzae strain A642-S2-A17 A642 (pa... (9% of genes show similarity), NRPS,transAT-PKS
</option>

<option value = "svg/clusterblast_r8c1_9.svg">
NZ_CM001972 (1481019-1558136): Dickeya oryzae strain CSL RW192 chromosome, wh... (6% of genes show similarity), NRPS,transAT-PKS
</option>

<option value = "svg/clusterblast_r8c1_10.svg">
NZ_CP012109 (3929722-4013404): Myxococcus hansupus strain contaminant ex DSM ... (5% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r8c1-download">Download graphic</button>
</div>
<div id="clusterblast-r8c1-svg"></div>

</div>
</div>

<div class="body-details-section r8c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-447"></div> <span class="help-tooltip" id="cb-known-help-447">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r8c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-448"></div> <span class="help-tooltip" id="cb-sub-help-448">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r8c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-451"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-451">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c1-domain-bubble-svg" data-filename="JAAZTN010000008.1_r8c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r8c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (592926 - 637723): single NRPS
</option>


</select>

<div class="details-svg" id="r8c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r8c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r8c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-453"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-453">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c1-pfam-details-svg" data-filename="JAAZTN010000008.1_r8c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r8c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r8c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-454"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-454">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c1-tigrfam-details-svg" data-filename="JAAZTN010000008.1_r8c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r8c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r8c1-nrps_pks_products" data-name="r8c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r8c1-nrps_pks_monomers" data-name="r8c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r8c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-449"></div> <span class="help-tooltip" id="nrps-structures-help-449">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 592926 - 637723:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r8c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r8c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r8c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-450"></div> <span class="help-tooltip" id="nrps-monomers-help-450">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene181</span></strong>: X
<div class="collapser collapser-target-input-path1-gene181 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (100..335)</strong>: X
<div class="collapser collapser-target-input-path1-gene181 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene181-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">EKVLEGLVFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene182</span></strong>: X
<div class="collapser collapser-target-input-path1-gene182 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (38..249)</strong>: X
<div class="collapser collapser-target-input-path1-gene182 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene182-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>  <strong>NOTE: uncertain match</strong><br>
<dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVS------K</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene183</span></strong>: X
<div class="collapser collapser-target-input-path1-gene183 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (245..480)</strong>: X
<div class="collapser collapser-target-input-path1-gene183 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene183-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GVGSIVAIFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVESIGAIAK</span> (47% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/fungi_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://fungismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/fungi_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>