
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>Cordyceps_javanica_Apopka97.scaffolds - 51 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/fungi.css">
<meta property="og:title" content="Cordyceps_javanica_Apopka97.scaffolds - 7 record(s) - 51 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/fungi_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://fungismash.secondarymetabolites.org/upload/fungi-6fbc267d-7eeb-4e2b-879b-05c4695b36de/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://fungismash.secondarymetabolites.org/">
<img src="images/fungi_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://fungismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="Cordyceps_javanica_Apopka97.scaffolds.zip">Download all results</a></li>

<li><a href="Cordyceps_javanica_Apopka97.scaffolds.gbk">Download GenBank summary file</a></li>
<li><a href="Cordyceps_javanica_Apopka97.scaffolds.json">Download JSON results file</a></li>

<li><a href="fungi-6fbc267d-7eeb-4e2b-879b-05c4695b36de.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton RiPP fungal-RiPP-like r1c1">
<a href="#r1c1">1.1</a>
</div>

<div class="regbutton NRPS NRPS r1c2">
<a href="#r1c2">1.2</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r1c3">
<a href="#r1c3">1.3</a>
</div>

<div class="regbutton hybrid r1c4">
<a href="#r1c4">1.4</a>
</div>

<div class="regbutton hybrid r1c5">
<a href="#r1c5">1.5</a>
</div>

<div class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">1.6</a>
</div>

<div class="regbutton NRPS NRPS r1c7">
<a href="#r1c7">1.7</a>
</div>

<div class="regbutton NRPS NRPS r1c8">
<a href="#r1c8">1.8</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r1c9">
<a href="#r1c9">1.9</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r1c10">
<a href="#r1c10">1.10</a>
</div>

<div class="regbutton hybrid r1c11">
<a href="#r1c11">1.11</a>
</div>

<div class="regbutton NRPS NRPS-like r1c12">
<a href="#r1c12">1.12</a>
</div>



<div class="regbutton hybrid r2c1">
<a href="#r2c1">2.1</a>
</div>

<div class="regbutton NRPS NRPS-like r2c2">
<a href="#r2c2">2.2</a>
</div>

<div class="regbutton NRPS NRPS r2c3">
<a href="#r2c3">2.3</a>
</div>

<div class="regbutton NRPS r2c4">
<a href="#r2c4">2.4</a>
</div>

<div class="regbutton PKS T1PKS r2c5">
<a href="#r2c5">2.5</a>
</div>

<div class="regbutton terpene terpene r2c6">
<a href="#r2c6">2.6</a>
</div>

<div class="regbutton PKS T1PKS r2c7">
<a href="#r2c7">2.7</a>
</div>

<div class="regbutton hybrid r2c8">
<a href="#r2c8">2.8</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r2c9">
<a href="#r2c9">2.9</a>
</div>

<div class="regbutton PKS T1PKS r2c10">
<a href="#r2c10">2.10</a>
</div>

<div class="regbutton NRPS NRPS r2c11">
<a href="#r2c11">2.11</a>
</div>

<div class="regbutton hybrid r2c12">
<a href="#r2c12">2.12</a>
</div>



<div class="regbutton hybrid r3c1">
<a href="#r3c1">3.1</a>
</div>

<div class="regbutton hybrid r3c2">
<a href="#r3c2">3.2</a>
</div>

<div class="regbutton PKS T1PKS r3c3">
<a href="#r3c3">3.3</a>
</div>

<div class="regbutton NRPS NRPS-like r3c4">
<a href="#r3c4">3.4</a>
</div>

<div class="regbutton other isocyanide r3c5">
<a href="#r3c5">3.5</a>
</div>

<div class="regbutton NRPS NRPS-like r3c6">
<a href="#r3c6">3.6</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r3c7">
<a href="#r3c7">3.7</a>
</div>



<div class="regbutton hybrid r4c1">
<a href="#r4c1">4.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">4.2</a>
</div>

<div class="regbutton NRPS NRPS r4c3">
<a href="#r4c3">4.3</a>
</div>

<div class="regbutton hybrid r4c4">
<a href="#r4c4">4.4</a>
</div>

<div class="regbutton NRPS NRPS r4c5">
<a href="#r4c5">4.5</a>
</div>

<div class="regbutton NRPS NRPS r4c6">
<a href="#r4c6">4.6</a>
</div>

<div class="regbutton hybrid r4c7">
<a href="#r4c7">4.7</a>
</div>



<div class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">5.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r5c2">
<a href="#r5c2">5.2</a>
</div>

<div class="regbutton NRPS NRPS r5c3">
<a href="#r5c3">5.3</a>
</div>



<div class="regbutton PKS T1PKS r6c1">
<a href="#r6c1">6.1</a>
</div>

<div class="regbutton NRPS NRPS r6c2">
<a href="#r6c2">6.2</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r6c3">
<a href="#r6c3">6.3</a>
</div>

<div class="regbutton NRPS NAPAA r6c4">
<a href="#r6c4">6.4</a>
</div>

<div class="regbutton terpene terpene r6c5">
<a href="#r6c5">6.5</a>
</div>



<div class="regbutton PKS T1PKS r7c1">
<a href="#r7c1">7.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r7c2">
<a href="#r7c2">7.2</a>
</div>

<div class="regbutton NRPS NRPS r7c3">
<a href="#r7c3">7.3</a>
</div>

<div class="regbutton PKS T1PKS r7c4">
<a href="#r7c4">7.4</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r7c5">
<a href="#r7c5">7.5</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>scaffold_1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="scaffold_1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton RiPP fungal-RiPP-like r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">190,202</td>
<td class="digits table-split-left">250,798</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton NRPS NRPS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">501,911</td>
<td class="digits table-split-left">548,733</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton RiPP fungal-RiPP-like r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,093,141</td>
<td class="digits table-split-left">1,153,616</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c4">
<td class="regbutton hybrid r1c4">
<a href="#r1c4">Region&nbsp1.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,822,437</td>
<td class="digits table-split-left">2,874,607</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000063/1" target="_blank">fumonisin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 11%, #ffffff00 11%)">11%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c5">
<td class="regbutton hybrid r1c5">
<a href="#r1c5">Region&nbsp1.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,310,567</td>
<td class="digits table-split-left">3,365,192</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001738/1" target="_blank">phomasetin</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 57%, #ffffff00 57%)">57%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c6">
<td class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">Region&nbsp1.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,947,761</td>
<td class="digits table-split-left">3,993,208</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002710/1" target="_blank">metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c7">
<td class="regbutton NRPS NRPS r1c7">
<a href="#r1c7">Region&nbsp1.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,530,689</td>
<td class="digits table-split-left">4,576,376</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c8">
<td class="regbutton NRPS NRPS r1c8">
<a href="#r1c8">Region&nbsp1.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,598,008</td>
<td class="digits table-split-left">4,641,012</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c9">
<td class="regbutton RiPP fungal-RiPP-like r1c9">
<a href="#r1c9">Region&nbsp1.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">6,114,201</td>
<td class="digits table-split-left">6,174,942</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c10">
<td class="regbutton RiPP fungal-RiPP-like r1c10">
<a href="#r1c10">Region&nbsp1.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">6,990,917</td>
<td class="digits table-split-left">7,052,134</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c11">
<td class="regbutton hybrid r1c11">
<a href="#r1c11">Region&nbsp1.11</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">7,156,336</td>
<td class="digits table-split-left">7,227,294</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002259/1" target="_blank">beauveriolide I/beauveriolide III</a></td>
<td>Polyketide+NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c12">
<td class="regbutton NRPS NRPS-like r1c12">
<a href="#r1c12">Region&nbsp1.12</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">8,118,141</td>
<td class="digits table-split-left">8,163,807</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_2</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="scaffold_2_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton hybrid r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">163,991</td>
<td class="digits table-split-left">216,113</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton NRPS NRPS-like r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">642,561</td>
<td class="digits table-split-left">686,369</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton NRPS NRPS r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">943,118</td>
<td class="digits table-split-left">991,234</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c4">
<td class="regbutton NRPS r2c4">
<a href="#r2c4">Region&nbsp2.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,037,874</td>
<td class="digits table-split-left">1,099,097</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c5">
<td class="regbutton PKS T1PKS r2c5">
<a href="#r2c5">Region&nbsp2.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,402,466</td>
<td class="digits table-split-left">1,450,513</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c6">
<td class="regbutton terpene terpene r2c6">
<a href="#r2c6">Region&nbsp2.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,230,916</td>
<td class="digits table-split-left">2,253,026</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c7">
<td class="regbutton PKS T1PKS r2c7">
<a href="#r2c7">Region&nbsp2.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,410,850</td>
<td class="digits table-split-left">2,458,506</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c8">
<td class="regbutton hybrid r2c8">
<a href="#r2c8">Region&nbsp2.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,880,687</td>
<td class="digits table-split-left">2,949,376</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c9">
<td class="regbutton RiPP fungal-RiPP-like r2c9">
<a href="#r2c9">Region&nbsp2.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,573,753</td>
<td class="digits table-split-left">3,634,823</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c10">
<td class="regbutton PKS T1PKS r2c10">
<a href="#r2c10">Region&nbsp2.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,495,415</td>
<td class="digits table-split-left">4,550,701</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002255/1" target="_blank">lijiquinone</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 25%, #ffffff00 25%)">25%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c11">
<td class="regbutton NRPS NRPS r2c11">
<a href="#r2c11">Region&nbsp2.11</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,997,278</td>
<td class="digits table-split-left">5,041,527</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c12">
<td class="regbutton hybrid r2c12">
<a href="#r2c12">Region&nbsp2.12</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide-nrp" target="_blank">isocyanide-nrp</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">5,433,305</td>
<td class="digits table-split-left">5,528,524</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_3</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="scaffold_3_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton hybrid r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">58,616</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000146/1" target="_blank">solanapyrone D</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton hybrid r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">580,696</td>
<td class="digits table-split-left">674,169</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton PKS T1PKS r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,904,091</td>
<td class="digits table-split-left">2,951,445</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton NRPS NRPS-like r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,562,170</td>
<td class="digits table-split-left">3,606,393</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton other isocyanide r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide" target="_blank">isocyanide</a>
</td>
<td class="digits">4,132,053</td>
<td class="digits table-split-left">4,174,181</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c6">
<td class="regbutton NRPS NRPS-like r3c6">
<a href="#r3c6">Region&nbsp3.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">4,473,101</td>
<td class="digits table-split-left">4,520,263</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c7">
<td class="regbutton RiPP fungal-RiPP-like r3c7">
<a href="#r3c7">Region&nbsp3.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,900,141</td>
<td class="digits table-split-left">4,971,066</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_4</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="scaffold_4_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton hybrid r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">146,693</td>
<td class="digits table-split-left">241,447</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002258/1" target="_blank">shimalactone A/shimalactone B</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 42%, #ffffff00 42%)">42%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">722,715</td>
<td class="digits table-split-left">783,737</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c3">
<td class="regbutton NRPS NRPS r4c3">
<a href="#r4c3">Region&nbsp4.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,700,022</td>
<td class="digits table-split-left">1,744,001</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c4">
<td class="regbutton hybrid r4c4">
<a href="#r4c4">Region&nbsp4.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,932,254</td>
<td class="digits table-split-left">2,007,007</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c5">
<td class="regbutton NRPS NRPS r4c5">
<a href="#r4c5">Region&nbsp4.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,137,263</td>
<td class="digits table-split-left">4,187,527</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c6">
<td class="regbutton NRPS NRPS r4c6">
<a href="#r4c6">Region&nbsp4.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,324,099</td>
<td class="digits table-split-left">4,367,973</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002192/1" target="_blank">FR901512</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c7">
<td class="regbutton hybrid r4c7">
<a href="#r4c7">Region&nbsp4.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,376,382</td>
<td class="digits table-split-left">4,435,002</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002239/1" target="_blank">GKK1032 A2/pyrrocidine A/pyrrocidine B</a></td>
<td>Polyketide+NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_5</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="scaffold_5_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">70,132</td>
<td class="digits table-split-left">117,548</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton RiPP fungal-RiPP-like r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,053,084</td>
<td class="digits table-split-left">3,122,111</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton NRPS NRPS r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,684,161</td>
<td class="digits table-split-left">3,732,594</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_6</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="scaffold_6_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton PKS T1PKS r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">59,935</td>
<td class="digits table-split-left">106,303</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton NRPS NRPS r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">146,604</td>
<td class="digits table-split-left">200,634</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001251/1" target="_blank">epichloenin A</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c3">
<td class="regbutton RiPP fungal-RiPP-like r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">264,312</td>
<td class="digits table-split-left">330,763</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c4">
<td class="regbutton NRPS NAPAA r6c4">
<a href="#r6c4">Region&nbsp6.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">2,420,791</td>
<td class="digits table-split-left">2,454,831</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">Îµ-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c5">
<td class="regbutton terpene terpene r6c5">
<a href="#r6c5">Region&nbsp6.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,936,656</td>
<td class="digits table-split-left">2,958,213</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_7</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-7" data-filename="scaffold_7_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-7" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r7c1">
<td class="regbutton PKS T1PKS r7c1">
<a href="#r7c1">Region&nbsp7.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">35,828</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c2">
<td class="regbutton RiPP fungal-RiPP-like r7c2">
<a href="#r7c2">Region&nbsp7.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">260,813</td>
<td class="digits table-split-left">322,230</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c3">
<td class="regbutton NRPS NRPS r7c3">
<a href="#r7c3">Region&nbsp7.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,977,612</td>
<td class="digits table-split-left">2,031,136</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c4">
<td class="regbutton PKS T1PKS r7c4">
<a href="#r7c4">Region&nbsp7.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,292,634</td>
<td class="digits table-split-left">2,340,313</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c5">
<td class="regbutton RiPP fungal-RiPP-like r7c5">
<a href="#r7c5">Region&nbsp7.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">2,868,748</td>
<td class="digits table-split-left">2,929,547</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton RiPP fungal-RiPP-like r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">190,202</td>
<td class="digits table-split-left">250,798</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton NRPS NRPS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">501,911</td>
<td class="digits table-split-left">548,733</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton RiPP fungal-RiPP-like r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,093,141</td>
<td class="digits table-split-left">1,153,616</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c4">
<td class="regbutton hybrid r1c4">
<a href="#r1c4">Region&nbsp1.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,822,437</td>
<td class="digits table-split-left">2,874,607</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000063/1" target="_blank">fumonisin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 11%, #ffffff00 11%)">11%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c5">
<td class="regbutton hybrid r1c5">
<a href="#r1c5">Region&nbsp1.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,310,567</td>
<td class="digits table-split-left">3,365,192</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001738/1" target="_blank">phomasetin</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 57%, #ffffff00 57%)">57%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c6">
<td class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">Region&nbsp1.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,947,761</td>
<td class="digits table-split-left">3,993,208</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002710/1" target="_blank">metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c7">
<td class="regbutton NRPS NRPS r1c7">
<a href="#r1c7">Region&nbsp1.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,530,689</td>
<td class="digits table-split-left">4,576,376</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c8">
<td class="regbutton NRPS NRPS r1c8">
<a href="#r1c8">Region&nbsp1.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,598,008</td>
<td class="digits table-split-left">4,641,012</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c9">
<td class="regbutton RiPP fungal-RiPP-like r1c9">
<a href="#r1c9">Region&nbsp1.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">6,114,201</td>
<td class="digits table-split-left">6,174,942</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c10">
<td class="regbutton RiPP fungal-RiPP-like r1c10">
<a href="#r1c10">Region&nbsp1.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">6,990,917</td>
<td class="digits table-split-left">7,052,134</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c11">
<td class="regbutton hybrid r1c11">
<a href="#r1c11">Region&nbsp1.11</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">7,156,336</td>
<td class="digits table-split-left">7,227,294</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002259/1" target="_blank">beauveriolide I/beauveriolide III</a></td>
<td>Polyketide+NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c12">
<td class="regbutton NRPS NRPS-like r1c12">
<a href="#r1c12">Region&nbsp1.12</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">8,118,141</td>
<td class="digits table-split-left">8,163,807</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton hybrid r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">163,991</td>
<td class="digits table-split-left">216,113</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton NRPS NRPS-like r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">642,561</td>
<td class="digits table-split-left">686,369</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton NRPS NRPS r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">943,118</td>
<td class="digits table-split-left">991,234</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c4">
<td class="regbutton NRPS r2c4">
<a href="#r2c4">Region&nbsp2.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,037,874</td>
<td class="digits table-split-left">1,099,097</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c5">
<td class="regbutton PKS T1PKS r2c5">
<a href="#r2c5">Region&nbsp2.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,402,466</td>
<td class="digits table-split-left">1,450,513</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c6">
<td class="regbutton terpene terpene r2c6">
<a href="#r2c6">Region&nbsp2.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,230,916</td>
<td class="digits table-split-left">2,253,026</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c7">
<td class="regbutton PKS T1PKS r2c7">
<a href="#r2c7">Region&nbsp2.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,410,850</td>
<td class="digits table-split-left">2,458,506</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c8">
<td class="regbutton hybrid r2c8">
<a href="#r2c8">Region&nbsp2.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,880,687</td>
<td class="digits table-split-left">2,949,376</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c9">
<td class="regbutton RiPP fungal-RiPP-like r2c9">
<a href="#r2c9">Region&nbsp2.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,573,753</td>
<td class="digits table-split-left">3,634,823</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c10">
<td class="regbutton PKS T1PKS r2c10">
<a href="#r2c10">Region&nbsp2.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,495,415</td>
<td class="digits table-split-left">4,550,701</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002255/1" target="_blank">lijiquinone</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 25%, #ffffff00 25%)">25%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c11">
<td class="regbutton NRPS NRPS r2c11">
<a href="#r2c11">Region&nbsp2.11</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,997,278</td>
<td class="digits table-split-left">5,041,527</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c12">
<td class="regbutton hybrid r2c12">
<a href="#r2c12">Region&nbsp2.12</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide-nrp" target="_blank">isocyanide-nrp</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">5,433,305</td>
<td class="digits table-split-left">5,528,524</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton hybrid r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">58,616</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000146/1" target="_blank">solanapyrone D</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton hybrid r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">580,696</td>
<td class="digits table-split-left">674,169</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton PKS T1PKS r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,904,091</td>
<td class="digits table-split-left">2,951,445</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton NRPS NRPS-like r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,562,170</td>
<td class="digits table-split-left">3,606,393</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton other isocyanide r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide" target="_blank">isocyanide</a>
</td>
<td class="digits">4,132,053</td>
<td class="digits table-split-left">4,174,181</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c6">
<td class="regbutton NRPS NRPS-like r3c6">
<a href="#r3c6">Region&nbsp3.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">4,473,101</td>
<td class="digits table-split-left">4,520,263</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c7">
<td class="regbutton RiPP fungal-RiPP-like r3c7">
<a href="#r3c7">Region&nbsp3.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,900,141</td>
<td class="digits table-split-left">4,971,066</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r4c1">
<td class="regbutton hybrid r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">146,693</td>
<td class="digits table-split-left">241,447</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002258/1" target="_blank">shimalactone A/shimalactone B</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 42%, #ffffff00 42%)">42%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c2">
<td class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">722,715</td>
<td class="digits table-split-left">783,737</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c3">
<td class="regbutton NRPS NRPS r4c3">
<a href="#r4c3">Region&nbsp4.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,700,022</td>
<td class="digits table-split-left">1,744,001</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c4">
<td class="regbutton hybrid r4c4">
<a href="#r4c4">Region&nbsp4.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,932,254</td>
<td class="digits table-split-left">2,007,007</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c5">
<td class="regbutton NRPS NRPS r4c5">
<a href="#r4c5">Region&nbsp4.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,137,263</td>
<td class="digits table-split-left">4,187,527</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c6">
<td class="regbutton NRPS NRPS r4c6">
<a href="#r4c6">Region&nbsp4.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,324,099</td>
<td class="digits table-split-left">4,367,973</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002192/1" target="_blank">FR901512</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c7">
<td class="regbutton hybrid r4c7">
<a href="#r4c7">Region&nbsp4.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,376,382</td>
<td class="digits table-split-left">4,435,002</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002239/1" target="_blank">GKK1032 A2/pyrrocidine A/pyrrocidine B</a></td>
<td>Polyketide+NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">70,132</td>
<td class="digits table-split-left">117,548</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton RiPP fungal-RiPP-like r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,053,084</td>
<td class="digits table-split-left">3,122,111</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton NRPS NRPS r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,684,161</td>
<td class="digits table-split-left">3,732,594</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r6c1">
<td class="regbutton PKS T1PKS r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">59,935</td>
<td class="digits table-split-left">106,303</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c2">
<td class="regbutton NRPS NRPS r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">146,604</td>
<td class="digits table-split-left">200,634</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001251/1" target="_blank">epichloenin A</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c3">
<td class="regbutton RiPP fungal-RiPP-like r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">264,312</td>
<td class="digits table-split-left">330,763</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c4">
<td class="regbutton NRPS NAPAA r6c4">
<a href="#r6c4">Region&nbsp6.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">2,420,791</td>
<td class="digits table-split-left">2,454,831</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">Îµ-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c5">
<td class="regbutton terpene terpene r6c5">
<a href="#r6c5">Region&nbsp6.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,936,656</td>
<td class="digits table-split-left">2,958,213</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r7c1">
<td class="regbutton PKS T1PKS r7c1">
<a href="#r7c1">Region&nbsp7.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">35,828</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c2">
<td class="regbutton RiPP fungal-RiPP-like r7c2">
<a href="#r7c2">Region&nbsp7.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">260,813</td>
<td class="digits table-split-left">322,230</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c3">
<td class="regbutton NRPS NRPS r7c3">
<a href="#r7c3">Region&nbsp7.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,977,612</td>
<td class="digits table-split-left">2,031,136</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c4">
<td class="regbutton PKS T1PKS r7c4">
<a href="#r7c4">Region&nbsp7.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,292,634</td>
<td class="digits table-split-left">2,340,313</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c5">
<td class="regbutton RiPP fungal-RiPP-like r7c5">
<a href="#r7c5">Region&nbsp7.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">2,868,748</td>
<td class="digits table-split-left">2,929,547</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-470"></div> <span class="help-tooltip" id="region_svg-help-470">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="scaffold_1_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 190,202 - 250,798 nt. (total: 60,597 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-471"></div> <span class="help-tooltip" id="focus-panel-help-471">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-5"></div> <span class="help-tooltip" id="gt-help-tooltip-help-5">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000517: 0-9116">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000517/index.html#r1c1" target="_new">BGC0000517</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>RiPP</td>

<td>haloduracin Î², haloduracin Î±</td>


<td>Bacillus halodurans C-125</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:0.31, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>nidulanin A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002747: 549-28042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002747/index.html#r1c1" target="_new">BGC0002747</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>lentinulin A</td>


<td>Lentinula edodes</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001515: 0-63902">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001515/index.html#r1c1" target="_new">BGC0001515</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.45, order:0.30, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>aspercryptins</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001854: 1-34076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001854/index.html#r1c1" target="_new">BGC0001854</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>tricholignan A</td>


<td>Trichoderma harzianum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002172: 696-54762">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002172/index.html#r1c1" target="_new">BGC0002172</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>fusatrixin, fusapentaxin, fusaoctaxin A</td>


<td>Fusarium graminearum PH-1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000517: 0-9116">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000517/index.html#r1c1" target="_new">BGC0000517</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>RiPP</td>
<td>haloduracin Î², haloduracin Î±</td>
<td>Bacillus halodurans C-125</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:0.31, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>NRP</td>
<td>nidulanin A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002747: 549-28042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002747/index.html#r1c1" target="_new">BGC0002747</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>RiPP</td>
<td>lentinulin A</td>
<td>Lentinula edodes</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001515: 0-63902">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001515/index.html#r1c1" target="_new">BGC0001515</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.45, order:0.30, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP</td>
<td>aspercryptins</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001854: 1-34076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001854/index.html#r1c1" target="_new">BGC0001854</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>tricholignan A</td>
<td>Trichoderma harzianum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002172: 696-54762">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002172/index.html#r1c1" target="_new">BGC0002172</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>NRP</td>
<td>fusatrixin, fusapentaxin, fusaoctaxin A</td>
<td>Fusarium graminearum PH-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c1_1.svg">
NW_020290621 (328465-419468): Aspergillus saccharolyticus JOP 1030-1 unplaced... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_2.svg">
NW_020292053 (616043-708384): Aspergillus vadensis CBS 113365 unplaced genomi... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_3.svg">
NW_023336279 (145289-236294): Aspergillus tubingensis WU-2223L DNA, scaffold ... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_4.svg">
NW_020290899 (145522-237868): Aspergillus niger CBS 101883 unplaced genomic s... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_5.svg">
NW_024468978 (129841-221990): Aspergillus fijiensis CBS 313.89 unplaced genom... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_6.svg">
NW_020292127 (15270-136339): Aspergillus ibericus CBS 121593 unplaced genomic... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r1c1_7.svg">
NW_020291612 (327439-419782): Aspergillus piperis CBS 112811 unplaced genomic... (5% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_8.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_9.svg">
NC_054856 (3727884-3894446): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (3% of genes show similarity), NI-siderophore,NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_10.svg">
NW_025408669 (4163753-4299872): Hirsutella rhossiliensis strain HR02 chromoso... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r1c1-download">Download graphic</button>
</div>
<div id="clusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-6"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-6">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="scaffold_1_r1c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r1c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-472"></div> <span class="help-tooltip" id="region_svg-help-472">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c2-svg" data-filename="scaffold_1_r1c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 501,911 - 548,733 nt. (total: 46,823 nt)

<a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-473"></div> <span class="help-tooltip" id="focus-panel-help-473">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c2-gene-table-details" data-name="r1c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c2-nrps_pks" data-name="r1c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c2-nrps-pks-bubbles" data-name="r1c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c2-tigrfam-details" data-name="r1c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-15"></div> <span class="help-tooltip" id="gt-help-tooltip-help-15">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-7"></div> <span class="help-tooltip" id="nrps-domain-help-7">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-details-svg" data-filename="scaffold_1_r1c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-8"></div> <span class="help-tooltip" id="MIBiG-help-8">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c2"></div>
<div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002134: 0-5774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002134/index.html#r1c1" target="_new">BGC0002134</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other (Other)</td>

<td>leupeptin A, leupeptin B, leupeptin D, leupeptin C</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>serinocyclin A, serinocyclin B</td>


<td>Metarhizium robertsii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002134: 0-5774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002134/index.html#r1c1" target="_new">BGC0002134</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other (Other)</td>
<td>leupeptin A, leupeptin B, leupeptin D, leupeptin C</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>serinocyclin A, serinocyclin B</td>
<td>Metarhizium robertsii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-9"></div> <span class="help-tooltip" id="cb-general-help-9">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c2_1.svg">
NW_024467540 (77139-192266): Aspergillus versicolor CBS 583.65 unplaced genom... (12% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r1c2_2.svg">
NC_054691 (3592203-3657410): Aspergillus flavus NRRL3357 chromosome 1, comple... (23% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_3.svg">
NW_024467876 (196055-261189): Aspergillus sydowii CBS 593.65 unplaced genomic... (15% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r1c2_4.svg">
NW_006917092 (458237-537613): Pestalotiopsis fici W106-1 unplaced genomic sca... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_5.svg">
NW_025953265 (97314-162450): Annulohypoxylon maeteangense strain CBS 123835 u... (35% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r1c2_6.svg">
NC_036435 (3736891-3802098): Aspergillus oryzae RIB40 DNA, chromosome 1 (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_7.svg">
NC_054861 (186488-251632): Aspergillus puulaauensis MK2 DNA, chromosome 5, ne... (14% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r1c2_8.svg">
NW_020290621 (440448-519602): Aspergillus saccharolyticus JOP 1030-1 unplaced... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_9.svg">
NW_020291593 (584732-664064): Aspergillus piperis CBS 112811 unplaced genomic... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_10.svg">
NC_036440 (2602285-2678300): Aspergillus oryzae RIB40 DNA, chromosome 6 (11% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c2-download">Download graphic</button>
</div>
<div id="clusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-10"></div> <span class="help-tooltip" id="cb-known-help-10">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-11"></div> <span class="help-tooltip" id="cb-sub-help-11">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-14"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-14">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-domain-bubble-svg" data-filename="scaffold_1_r1c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (501910 - 548733): single NRPS
</option>


</select>

<div class="details-svg" id="r1c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-16"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-16">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="scaffold_1_r1c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-17"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-17">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-tigrfam-details-svg" data-filename="scaffold_1_r1c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c2-nrps_pks_products" data-name="r1c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c2-nrps_pks_monomers" data-name="r1c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-12"></div> <span class="help-tooltip" id="nrps-structures-help-12">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 501910 - 548733:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c2-2" class="smiles-canvas" data-smiles="NC(C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(D-Ala)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ala*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-13"></div> <span class="help-tooltip" id="nrps-monomers-help-13">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene151</span></strong>: X
<div class="collapser collapser-target-input-path1-gene151 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (73..282)</strong>: X
<div class="collapser collapser-target-input-path1-gene151 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene151-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>  <strong>NOTE: uncertain match</strong><br>
<dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DSPMVS---K</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene154</span></strong>: Ala
<div class="collapser collapser-target-input-path1-gene154 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*" target="_new">relaxed</a>
</dd>
<br>







<dd><strong>AMP-binding (553..951)</strong>: Ala
<div class="collapser collapser-target-input-path1-gene154 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-input-path1-gene154-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLALVAAIYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Aol <span class="serif">DVWYVAAIYK</span> (56% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">input.path1.gene155</span></strong>: X
<div class="collapser collapser-target-input-path1-gene155 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (13..207)</strong>: X
<div class="collapser collapser-target-input-path1-gene155 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene155-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>  <strong>NOTE: uncertain match</strong><br>
<dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">---HIAAAVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 3 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-474"></div> <span class="help-tooltip" id="region_svg-help-474">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c3-svg" data-filename="scaffold_1_r1c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,093,141 - 1,153,616 nt. (total: 60,476 nt)

<a class="cluster-rules-header" id="r1c3-rules-header" href="#r1c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c3-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r1c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-475"></div> <span class="help-tooltip" id="focus-panel-help-475">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c3-gene-table-details" data-name="r1c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c3-MIBiG-cluster-compare" data-name="r1c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c3-clusterblast" data-name="r1c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c3-knownclusterblast" data-name="r1c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c3-subclusterblast" data-name="r1c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c3-pfam-details" data-name="r1c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c3-tigrfam-details" data-name="r1c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-22"></div> <span class="help-tooltip" id="gt-help-tooltip-help-22">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-18"></div> <span class="help-tooltip" id="MIBiG-help-18">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c3"></div>
<div id="comparison-MIBiG-r1c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002283: 0-4572">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002283/index.html#r1c1" target="_new">BGC0002283</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>sodorifen</td>


<td>Serratia plymuthica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001361: 0-4580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001361/index.html#r1c1" target="_new">BGC0001361</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>sodorifen</td>


<td>Serratia plymuthica 4Rx13</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>polyhydroxyalkanoate</td>


<td>Burkholderia sp. DSM 9242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>PR-toxin</td>


<td>Penicillium roqueforti FM164</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002195: 156-5932">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002195/index.html#r1c1" target="_new">BGC0002195</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>kojic acid</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.51, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.46, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>heptelidic acid</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002262: 0-50372">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002262/index.html#r1c1" target="_new">BGC0002262</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>trienylfuranol A, sporothriolide, dihydrosporothriolide, deoxysporothric acid, dihydroisosporothric acid, sporothric acid, isosporothric acid, sporochartine B</td>


<td>Hypomontagnella monticulosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001738: 0-24326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001738/index.html#r1c1" target="_new">BGC0001738</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>phomasetin</td>


<td>Pyrenochaetopsis sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>


<td>Fusarium graminearum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002283: 0-4572">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002283/index.html#r1c1" target="_new">BGC0002283</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>sodorifen</td>
<td>Serratia plymuthica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001361: 0-4580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001361/index.html#r1c1" target="_new">BGC0001361</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>sodorifen</td>
<td>Serratia plymuthica 4Rx13</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>polyhydroxyalkanoate</td>
<td>Burkholderia sp. DSM 9242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Terpene</td>
<td>PR-toxin</td>
<td>Penicillium roqueforti FM164</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002195: 156-5932">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002195/index.html#r1c1" target="_new">BGC0002195</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>kojic acid</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.51, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.46, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Terpene</td>
<td>heptelidic acid</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002262: 0-50372">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002262/index.html#r1c1" target="_new">BGC0002262</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Polyketide</td>
<td>trienylfuranol A, sporothriolide, dihydrosporothriolide, deoxysporothric acid, dihydroisosporothric acid, sporothric acid, isosporothric acid, sporochartine B</td>
<td>Hypomontagnella monticulosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001738: 0-24326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001738/index.html#r1c1" target="_new">BGC0001738</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP, Polyketide</td>
<td>phomasetin</td>
<td>Pyrenochaetopsis sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
<td>Fusarium graminearum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-19"></div> <span class="help-tooltip" id="cb-general-help-19">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c3_1.svg">
NC_049561 (2011040-2059583): Talaromyces rugulosus chromosome I, complete seq... (21% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c3_2.svg">
NW_011942160 (1251508-1300367): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (20% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c3_3.svg">
NW_006913048 (3521323-3618515): Cladophialophora yegresii CBS 114405 aczIN-su... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c3_4.svg">
NW_022985511 (1077816-1171333): Neohortaea acidophila strain CBS 113389 unpla... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c3_5.svg">
NW_016017383 (3230246-3380441): Sphaerulina musiva SO2202 unplaced genomic sc... (3% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c3_6.svg">
NT_166522 (170057-269564): Aspergillus niger CBS 513.88 supercontig An06 (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c3_7.svg">
NC_018215 (1659185-1716249): Zymoseptoria tritici IPO323 chromosome 4, whole ... (8% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c3_8.svg">
NC_036439 (4081578-4137471): Aspergillus oryzae RIB40 DNA, chromosome 5 (9% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c3_9.svg">
NW_006921536 (1019168-1074511): Pseudocercospora fijiensis CIRAD86 unplaced g... (11% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c3_10.svg">
NC_063024 (1660782-1725193): Fulvia fulva chromosome 13, complete sequence (8% of genes show similarity), fungal-RiPP
</option>

</select>
<button id= "clusterblast-r1c3-download">Download graphic</button>
</div>
<div id="clusterblast-r1c3-svg"></div>

</div>
</div>

<div class="body-details-section r1c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-20"></div> <span class="help-tooltip" id="cb-known-help-20">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-21"></div> <span class="help-tooltip" id="cb-sub-help-21">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-23"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-23">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-pfam-details-svg" data-filename="scaffold_1_r1c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-24"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-24">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-tigrfam-details-svg" data-filename="scaffold_1_r1c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r1c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 4 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-476"></div> <span class="help-tooltip" id="region_svg-help-476">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c4-svg" data-filename="scaffold_1_r1c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,822,437 - 2,874,607 nt. (total: 52,171 nt)

<a class="cluster-rules-header" id="r1c4-rules-header" href="#r1c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r1c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-477"></div> <span class="help-tooltip" id="focus-panel-help-477">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c4-gene-table-details" data-name="r1c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c4-nrps_pks" data-name="r1c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c4-MIBiG-cluster-compare" data-name="r1c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c4-clusterblast" data-name="r1c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c4-knownclusterblast" data-name="r1c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c4-subclusterblast" data-name="r1c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c4-nrps-pks-bubbles" data-name="r1c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c4-pfam-details" data-name="r1c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-33"></div> <span class="help-tooltip" id="gt-help-tooltip-help-33">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-25"></div> <span class="help-tooltip" id="nrps-domain-help-25">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-details-svg" data-filename="scaffold_1_r1c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-26"></div> <span class="help-tooltip" id="MIBiG-help-26">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c4"></div>
<div id="comparison-MIBiG-r1c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.69</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>Terpene</td>

<td>betaestacin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001136: 0-20147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001136/index.html#r1c1" target="_new">BGC0001136</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>NRP, Polyketide</td>

<td>desmethylbassianin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP, Polyketide</td>

<td>chaetoglobosins</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001136: 0-20147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001136/index.html#r1c1" target="_new">BGC0001136</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Polyketide</td>
<td>desmethylbassianin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Polyketide</td>
<td>chaetoglobosins</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-27"></div> <span class="help-tooltip" id="cb-general-help-27">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c4_1.svg">
NW_024467841 (133513-205645): Aspergillus wentii DTO 134E9 unplaced genomic s... (28% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_2.svg">
NC_064680 (1077686-1149810): Colletotrichum lupini chromosome 7, complete seq... (50% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_3.svg">
NW_020291470 (1087367-1204862): Aspergillus aculeatinus CBS 121060 unplaced g... (7% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_4.svg">
NW_025953046 (865813-935176): Daldinia caldariorum strain CBS 122874 unplaced... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c4_5.svg">
NC_072850 (2022235-2153706): Fusarium oxysporum Fo47 chromosome XI, complete ... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c4_6.svg">
NC_036629 (2646284-2784115): Fusarium fujikuroi IMI 58289 draft genome, chrom... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c4_7.svg">
NC_064672 (6807692-6886490): Colletotrichum lupini chromosome 1, complete seq... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_8.svg">
NW_025953826 (72610-144857): Annulohypoxylon truncatum strain CBS 140777 unpl... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_9.svg">
NW_024067566 (1440324-1543594): Penicillium roqueforti strain LCP96 04111 chr... (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_10.svg">
NW_025953249 (406839-479078): Annulohypoxylon maeteangense strain CBS 123835 ... (14% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r1c4-download">Download graphic</button>
</div>
<div id="clusterblast-r1c4-svg"></div>

</div>
</div>

<div class="body-details-section r1c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-28"></div> <span class="help-tooltip" id="cb-known-help-28">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c4_1.svg">
fumonisin
</option>

<option value = "svg/knownclusterblast_r1c4_2.svg">
fumonisin B1
</option>

</select>
<button id="knownclusterblast-r1c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c4-svg"></div>

</div>
</div>

<div class="body-details-section r1c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-29"></div> <span class="help-tooltip" id="cb-sub-help-29">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-32"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-32">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-domain-bubble-svg" data-filename="scaffold_1_r1c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (2822436 - 2874607): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r1c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-34"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-34">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-pfam-details-svg" data-filename="scaffold_1_r1c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c4-nrps_pks_products" data-name="r1c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c4-nrps_pks_monomers" data-name="r1c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-30"></div> <span class="help-tooltip" id="nrps-structures-help-30">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 2822436 - 2874607:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c4-4" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-31"></div> <span class="help-tooltip" id="nrps-monomers-help-31">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene785</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene785 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (554..868)</strong>: pk
<div class="collapser collapser-target-input-path1-gene785 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene785-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene785-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 97.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 77.9</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 73.8</dt>

<dd></dd><dt>Malonyl-CoA: 54.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 47.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 36.1</dt>

<dd></dd><dt>Benzoyl-CoA: 35.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 25.5</dt>

<dd></dd><dt>Propionyl-CoA: 22.0</dt>

<dd></dd><dt>Acetyl-CoA: 21.5</dt>

<dd></dd><dt>fatty_acid: 21.3</dt>

<dd></dd><dt>CHC-CoA: 14.5</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.7</dt>

<dd></dd><dt>inactive: 9.3</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>













<dd><strong>AMP-binding (2891..3291)</strong>: X
<div class="collapser collapser-target-input-path1-gene785 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene785-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLTYLAVCGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DLGYLAGCFK</span> (53% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 5 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-478"></div> <span class="help-tooltip" id="region_svg-help-478">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c5-svg" data-filename="scaffold_1_r1c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,310,567 - 3,365,192 nt. (total: 54,626 nt)

<a class="cluster-rules-header" id="r1c5-rules-header" href="#r1c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c5-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-479"></div> <span class="help-tooltip" id="focus-panel-help-479">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c5-gene-table-details" data-name="r1c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c5-nrps_pks" data-name="r1c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c5-MIBiG-cluster-compare" data-name="r1c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c5-clusterblast" data-name="r1c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c5-knownclusterblast" data-name="r1c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c5-subclusterblast" data-name="r1c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c5-nrps-pks-bubbles" data-name="r1c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c5-pfam-details" data-name="r1c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c5-tigrfam-details" data-name="r1c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-43"></div> <span class="help-tooltip" id="gt-help-tooltip-help-43">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-35"></div> <span class="help-tooltip" id="nrps-domain-help-35">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-details-svg" data-filename="scaffold_1_r1c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-36"></div> <span class="help-tooltip" id="MIBiG-help-36">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c5"></div>
<div id="comparison-MIBiG-r1c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002539: 47-37339">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002539/index.html#r1c1" target="_new">BGC0002539</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>Alkaloid</td>

<td>chaetoglobosin P, chaetoglobosin K, chaetoglobosin A</td>


<td>Discosia rubi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002230: 9359-28771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002230/index.html#r1c1" target="_new">BGC0002230</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>Polyketide, NRP</td>

<td>tolypyridone C</td>


<td>Tolypocladium sp. 49Y</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000968: 0-25930">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000968/index.html#r1c1" target="_new">BGC0000968</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>NRP, Polyketide</td>

<td>chaetoglobosin A, chaetoglobosin C</td>


<td>Penicillium expansum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001738: 0-24326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001738/index.html#r1c1" target="_new">BGC0001738</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.54, order:0.43, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.54, order:0.43, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP, Polyketide</td>

<td>phomasetin</td>


<td>Pyrenochaetopsis sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.45</td>
<td>NRP, Polyketide</td>

<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>


<td>Fusarium heterosporum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002204: 1548-38253">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002204/index.html#r1c1" target="_new">BGC0002204</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.43</td>
<td>Polyketide, NRP</td>

<td>flavichalasine F, flavichalasine G, aspochalasin C, aspochalasin E, aspochalasin M, TMC-169, aspergillin PZ</td>


<td>Aspergillus flavipes</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>Polyketide</td>

<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>


<td>Chaetomium olivaceum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002539: 47-37339">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002539/index.html#r1c1" target="_new">BGC0002539</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Alkaloid</td>
<td>chaetoglobosin P, chaetoglobosin K, chaetoglobosin A</td>
<td>Discosia rubi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002230: 9359-28771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002230/index.html#r1c1" target="_new">BGC0002230</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide, NRP</td>
<td>tolypyridone C</td>
<td>Tolypocladium sp. 49Y</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000968: 0-25930">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000968/index.html#r1c1" target="_new">BGC0000968</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Polyketide</td>
<td>chaetoglobosin A, chaetoglobosin C</td>
<td>Penicillium expansum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001738: 0-24326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001738/index.html#r1c1" target="_new">BGC0001738</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.54, order:0.43, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>phomasetin</td>
<td>Pyrenochaetopsis sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>
<td>Fusarium heterosporum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002204: 1548-38253">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002204/index.html#r1c1" target="_new">BGC0002204</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide, NRP</td>
<td>flavichalasine F, flavichalasine G, aspochalasin C, aspochalasin E, aspochalasin M, TMC-169, aspergillin PZ</td>
<td>Aspergillus flavipes</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>
<td>Chaetomium olivaceum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-37"></div> <span class="help-tooltip" id="cb-general-help-37">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c5_1.svg">
NW_017264196 (2263112-2335444): Xylona heveae TC161 unplaced genomic scaffold... (35% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_2.svg">
NW_017263583 (13540-86418): Mollisia scopiformis unplaced genomic scaffold LY... (22% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_3.svg">
NW_024467862 (1993078-2065565): Aspergillus sydowii CBS 593.65 unplaced genom... (22% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_4.svg">
NW_022474219 (264912-357113): Venustampulla echinocandica strain BP 5553 chro... (22% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_5.svg">
NC_057369 (485067-557466): Aspergillus chevalieri M1 DNA, chromosome 8, nearl... (19% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_6.svg">
NW_020291497 (242872-444890): Aspergillus aculeatinus CBS 121060 unplaced gen... (5% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_7.svg">
NC_072842 (4536206-4608469): Fusarium oxysporum Fo47 chromosome III, complete... (20% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_8.svg">
NC_036624 (299708-371970): Fusarium fujikuroi IMI 58289 draft genome, chromos... (18% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_9.svg">
NW_017306203 (371193-443658): Alternaria alternata strain SRC1lrK2f unplaced ... (20% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_10.svg">
NW_014574610 (194963-267516): Penicillium digitatum Pd1 unplaced genomic scaf... (22% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r1c5-download">Download graphic</button>
</div>
<div id="clusterblast-r1c5-svg"></div>

</div>
</div>

<div class="body-details-section r1c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-38"></div> <span class="help-tooltip" id="cb-known-help-38">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c5-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c5_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c5_1.svg">
phomasetin
</option>

</select>
<button id="knownclusterblast-r1c5-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c5-svg"></div>

</div>
</div>

<div class="body-details-section r1c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-39"></div> <span class="help-tooltip" id="cb-sub-help-39">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-42"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-42">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-domain-bubble-svg" data-filename="scaffold_1_r1c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (3310566 - 3365192): chemical hybrid T1PKS-NRPS
</option>


</select>

<div class="details-svg" id="r1c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-44"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-44">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-pfam-details-svg" data-filename="scaffold_1_r1c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-45"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-45">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-tigrfam-details-svg" data-filename="scaffold_1_r1c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c5-nrps_pks_products" data-name="r1c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c5-nrps_pks_monomers" data-name="r1c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-40"></div> <span class="help-tooltip" id="nrps-structures-help-40">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 3310566 - 3365192:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c5-5" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c5-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-41"></div> <span class="help-tooltip" id="nrps-monomers-help-41">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene924</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene924 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (739..1057)</strong>: pk
<div class="collapser collapser-target-input-path1-gene924 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene924-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Ethylmalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene924-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 71.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 59.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 42.0</dt>

<dd></dd><dt>Malonyl-CoA: 36.7</dt>

<dd></dd><dt>Propionyl-CoA: 32.6</dt>

<dd></dd><dt>Isobutyryl-CoA: 30.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 22.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 16.6</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 14.4</dt>

<dd></dd><dt>Acetyl-CoA: 14.2</dt>

<dd></dd><dt>fatty_acid: 13.6</dt>

<dd></dd><dt>inactive: 12.0</dt>

<dd></dd><dt>Benzoyl-CoA: 12.0</dt>

<dd></dd><dt>CHC-CoA: 9.6</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (3085..3499)</strong>: X
<div class="collapser collapser-target-input-path1-gene924 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Orn, Lys, Arg
<div class="collapser collapser-target-input-path1-gene924-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Long positively charged side chain (Orn, Lys, Arg)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIDVFGNLAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DFESFGNIAK</span> (59% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 6 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-480"></div> <span class="help-tooltip" id="region_svg-help-480">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c6-svg" data-filename="scaffold_1_r1c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,947,761 - 3,993,208 nt. (total: 45,448 nt)

<a class="cluster-rules-header" id="r1c6-rules-header" href="#r1c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c6-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-481"></div> <span class="help-tooltip" id="focus-panel-help-481">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c6-gene-table-details" data-name="r1c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c6-nrps_pks" data-name="r1c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c6-MIBiG-cluster-compare" data-name="r1c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c6-clusterblast" data-name="r1c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c6-knownclusterblast" data-name="r1c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c6-subclusterblast" data-name="r1c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c6-nrps-pks-bubbles" data-name="r1c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c6-pfam-details" data-name="r1c6-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-54"></div> <span class="help-tooltip" id="gt-help-tooltip-help-54">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-46"></div> <span class="help-tooltip" id="nrps-domain-help-46">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-details-svg" data-filename="scaffold_1_r1c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-47"></div> <span class="help-tooltip" id="MIBiG-help-47">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c6"></div>
<div id="comparison-MIBiG-r1c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.55, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.55, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-48"></div> <span class="help-tooltip" id="cb-general-help-48">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c6_1.svg">
NW_006271973 (996861-1062288): Cordyceps militaris CM01 unplaced genomic scaf... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_2.svg">
NW_011942160 (3770676-3836079): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_3.svg">
NW_014574716 (6263628-6329031): Metarhizium brunneum ARSEF 3297 chromosome Un... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_4.svg">
NC_035791 (4601700-4667019): Pochonia chlamydosporia 170 chromosome 2, whole ... (42% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_5.svg">
NC_070536 (2319263-2385135): Fusarium keratoplasticum isolate Fu6.1 chromosom... (56% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_6.svg">
NW_014013664 (48182-113471): Trichoderma virens Gv29-8 chromosome Unknown TRI... (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_7.svg">
NW_006711167 (506611-572053): Trichoderma reesei QM6a unplaced genomic scaffo... (64% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_8.svg">
NC_057319 (2334015-2399373): Ustilaginoidea virens chromosome 4, complete seq... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_9.svg">
NC_070551 (2536076-2601948): Fusarium falciforme chromosome 8, complete sequence (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_10.svg">
NW_021167080 (1267789-1333877): Sodiomyces alkalinus F11 unplaced genomic sca... (50% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c6-download">Download graphic</button>
</div>
<div id="clusterblast-r1c6-svg"></div>

</div>
</div>

<div class="body-details-section r1c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-49"></div> <span class="help-tooltip" id="cb-known-help-49">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c6-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c6_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c6_1.svg">
metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid
</option>

<option value = "svg/knownclusterblast_r1c6_2.svg">
dimethylcoprogen
</option>

</select>
<button id="knownclusterblast-r1c6-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c6-svg"></div>

</div>
</div>

<div class="body-details-section r1c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-50"></div> <span class="help-tooltip" id="cb-sub-help-50">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-53"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-53">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-domain-bubble-svg" data-filename="scaffold_1_r1c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (3947760 - 3993208): single NRPS
</option>


</select>

<div class="details-svg" id="r1c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-55"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-55">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-pfam-details-svg" data-filename="scaffold_1_r1c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c6-nrps_pks_products" data-name="r1c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c6-nrps_pks_monomers" data-name="r1c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-51"></div> <span class="help-tooltip" id="nrps-structures-help-51">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 3947760 - 3993208:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c6-6" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c6-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-52"></div> <span class="help-tooltip" id="nrps-monomers-help-52">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1094</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1094 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (0..246)</strong>: X
<div class="collapser collapser-target-input-path1-gene1094 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1094-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DRVHIASTCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">FGLHIASTCK</span> (68% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene1095</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1095 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (93..442)</strong>: X
<div class="collapser collapser-target-input-path1-gene1095 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene1095-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVDHGGAVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">DVDGGGGIGK</span> (65% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 7 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-482"></div> <span class="help-tooltip" id="region_svg-help-482">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c7-svg" data-filename="scaffold_1_r1c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,530,689 - 4,576,376 nt. (total: 45,688 nt)

<a class="cluster-rules-header" id="r1c7-rules-header" href="#r1c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c7-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-483"></div> <span class="help-tooltip" id="focus-panel-help-483">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c7-gene-table-details" data-name="r1c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c7-nrps_pks" data-name="r1c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c7-MIBiG-cluster-compare" data-name="r1c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c7-clusterblast" data-name="r1c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c7-knownclusterblast" data-name="r1c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c7-subclusterblast" data-name="r1c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c7-nrps-pks-bubbles" data-name="r1c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c7-pfam-details" data-name="r1c7-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c7-tigrfam-details" data-name="r1c7-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-64"></div> <span class="help-tooltip" id="gt-help-tooltip-help-64">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-56"></div> <span class="help-tooltip" id="nrps-domain-help-56">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c7-details-svg" data-filename="scaffold_1_r1c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-57"></div> <span class="help-tooltip" id="MIBiG-help-57">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c7"></div>
<div id="comparison-MIBiG-r1c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-58"></div> <span class="help-tooltip" id="cb-general-help-58">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c7_1.svg">
NC_072354 (600326-666699): Talaromyces marneffei chromosome 7, complete sequence (19% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c7_2.svg">
NW_017971433 (2380693-2447085): Talaromyces atroroseus strain IBT 11181 chrom... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c7_3.svg">
NW_020167533 (924875-991241): Pseudogymnoascus destructans isolate 20631-21 u... (40% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c7_4.svg">
NW_020292131 (174943-242345): Aspergillus ibericus CBS 121593 unplaced genomi... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c7_5.svg">
NW_023336287 (930617-996817): Aspergillus tubingensis WU-2223L DNA, scaffold ... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c7_6.svg">
NC_054853 (2254582-2320833): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c7_7.svg">
NW_024468912 (725220-816586): Cryphonectria parasitica EP155 unplaced genomic... (21% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c7_8.svg">
NW_020292051 (818013-884264): Aspergillus vadensis CBS 113365 unplaced genomi... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c7_9.svg">
NW_020291816 (465557-531757): Aspergillus costaricaensis CBS 115574 unplaced ... (19% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c7_10.svg">
NW_020290890 (42074-108614): Aspergillus niger CBS 101883 unplaced genomic sc... (18% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c7-download">Download graphic</button>
</div>
<div id="clusterblast-r1c7-svg"></div>

</div>
</div>

<div class="body-details-section r1c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-59"></div> <span class="help-tooltip" id="cb-known-help-59">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-60"></div> <span class="help-tooltip" id="cb-sub-help-60">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-63"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-63">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c7-domain-bubble-svg" data-filename="scaffold_1_r1c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (4530688 - 4576376): single NRPS
</option>


</select>

<div class="details-svg" id="r1c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-65"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-65">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c7-pfam-details-svg" data-filename="scaffold_1_r1c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c7-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-66"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-66">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c7-tigrfam-details-svg" data-filename="scaffold_1_r1c7_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c7-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c7-nrps_pks_products" data-name="r1c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c7-nrps_pks_monomers" data-name="r1c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-61"></div> <span class="help-tooltip" id="nrps-structures-help-61">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 4530688 - 4576376:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c7-7" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c7-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-62"></div> <span class="help-tooltip" id="nrps-monomers-help-62">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1263</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1263 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (5..412)</strong>: X
<div class="collapser collapser-target-input-path1-gene1263 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Z-ahmohOrn
<div class="collapser collapser-target-input-path1-gene1263-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aromatic side chain (Phe, Trp, Ph-Gly, Tyr, bOH-Tyr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVDGGGGIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">DVDGGGGIGK</span> (91% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 8 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-484"></div> <span class="help-tooltip" id="region_svg-help-484">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c8-svg" data-filename="scaffold_1_r1c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,598,008 - 4,641,012 nt. (total: 43,005 nt)

<a class="cluster-rules-header" id="r1c8-rules-header" href="#r1c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c8-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-485"></div> <span class="help-tooltip" id="focus-panel-help-485">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c8-gene-table-details" data-name="r1c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c8-nrps_pks" data-name="r1c8-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c8-MIBiG-cluster-compare" data-name="r1c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c8-clusterblast" data-name="r1c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c8-knownclusterblast" data-name="r1c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c8-subclusterblast" data-name="r1c8-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c8-nrps-pks-bubbles" data-name="r1c8-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c8-pfam-details" data-name="r1c8-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-75"></div> <span class="help-tooltip" id="gt-help-tooltip-help-75">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c8-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-67"></div> <span class="help-tooltip" id="nrps-domain-help-67">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c8-details-svg" data-filename="scaffold_1_r1c8_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c8-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-68"></div> <span class="help-tooltip" id="MIBiG-help-68">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c8"></div>
<div id="comparison-MIBiG-r1c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-69"></div> <span class="help-tooltip" id="cb-general-help-69">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c8_1.svg">
NC_054850 (2918415-2983028): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_2.svg">
NC_054861 (1916756-1987814): Aspergillus puulaauensis MK2 DNA, chromosome 5, ... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c8_3.svg">
NW_024467524 (3810963-3881818): Aspergillus versicolor CBS 583.65 unplaced ge... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c8_4.svg">
NW_019170454 (685594-749291): Aspergillus glaucus CBS 516.65 unplaced genomic... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_5.svg">
NC_057367 (973431-1037220): Aspergillus chevalieri M1 DNA, chromosome 6, near... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_6.svg">
NW_024467304 (552618-615729): Aspergillus ruber CBS 135680 unplaced genomic s... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_7.svg">
NC_007196 (3573152-3653216): Aspergillus fumigatus Af293 chromosome 3, whole ... (6% of genes show similarity), NRPS,isocyanide-nrp
</option>

<option value = "svg/clusterblast_r1c8_8.svg">
NW_020291486 (59597-203320): Aspergillus aculeatinus CBS 121060 unplaced geno... (5% of genes show similarity), NRPS,fungal-RiPP-like,isocyanide
</option>

<option value = "svg/clusterblast_r1c8_9.svg">
NW_024467538 (321782-393387): Aspergillus versicolor CBS 583.65 unplaced geno... (6% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c8_10.svg">
NC_018295 (1958734-2022937): Candida orthopsilosis Co 90-125, chromosome 2 dr... (8% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r1c8-download">Download graphic</button>
</div>
<div id="clusterblast-r1c8-svg"></div>

</div>
</div>

<div class="body-details-section r1c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-70"></div> <span class="help-tooltip" id="cb-known-help-70">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-71"></div> <span class="help-tooltip" id="cb-sub-help-71">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c8-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-74"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-74">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c8-domain-bubble-svg" data-filename="scaffold_1_r1c8_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c8-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC8">
Candidate 8 (4598007 - 4641012): single NRPS
</option>


</select>

<div class="details-svg" id="r1c8-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c8-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-76"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-76">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c8-pfam-details-svg" data-filename="scaffold_1_r1c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c8-nrps_pks_products" data-name="r1c8-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c8-nrps_pks_monomers" data-name="r1c8-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c8-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-72"></div> <span class="help-tooltip" id="nrps-structures-help-72">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 8, location 4598007 - 4641012:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c8-8" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c8-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c8-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-73"></div> <span class="help-tooltip" id="nrps-monomers-help-73">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1279</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1279 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (6..265)</strong>: X
<div class="collapser collapser-target-input-path1-gene1279 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr
<div class="collapser collapser-target-input-path1-gene1279-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">-GGAVVLKGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c9' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 9 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-486"></div> <span class="help-tooltip" id="region_svg-help-486">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region009.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c9-svg" data-filename="scaffold_1_r1c9.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,114,201 - 6,174,942 nt. (total: 60,742 nt)

<a class="cluster-rules-header" id="r1c9-rules-header" href="#r1c9">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c9-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r1c9-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-487"></div> <span class="help-tooltip" id="focus-panel-help-487">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c9">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c9-gene-table-details" data-name="r1c9-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c9-MIBiG-cluster-compare" data-name="r1c9-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c9-clusterblast" data-name="r1c9-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c9-knownclusterblast" data-name="r1c9-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c9-subclusterblast" data-name="r1c9-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c9-pfam-details" data-name="r1c9-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c9-tigrfam-details" data-name="r1c9-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c9-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-81"></div> <span class="help-tooltip" id="gt-help-tooltip-help-81">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c9">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c9-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c9-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c9">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c9-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-77"></div> <span class="help-tooltip" id="MIBiG-help-77">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<div><p>No matches found.</p></div>

</div>
</div>



<div class="body-details-section r1c9-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-78"></div> <span class="help-tooltip" id="cb-general-help-78">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c9-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c9_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c9_1.svg">
NC_026475 (3314467-3405418): Fusarium graminearum PH-1 chromosome 2, whole ge... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c9_2.svg">
NW_006917093 (3054358-3117993): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c9_3.svg">
NW_024468937 (5549232-5621994): Cucurbitaria berberidis CBS 394.84 unplaced g... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c9_4.svg">
NW_024467839 (148457-211490): Aspergillus wentii DTO 134E9 unplaced genomic s... (6% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c9_5.svg">
NW_013550615 (4368823-4474200): Fonsecaea pedrosoi CBS 271.37 unplaced genomi... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c9_6.svg">
NW_015971635 (1692172-1793674): Fonsecaea multimorphosa CBS 102226 unplaced g... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c9_7.svg">
NW_022156716 (3888118-3951114): Pyricularia grisea strain NI907 chromosome Un... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c9_8.svg">
NW_006911270 (429097-460265): Baudoinia panamericana UAMH 10762 unplaced geno... (14% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c9_9.svg">
NC_066264 (3194691-3262661): Aspergillus nidulans FGSC A4 chromosome VIII (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c9_10.svg">
NW_006917100 (2218469-2283927): Pestalotiopsis fici W106-1 unplaced genomic s... (12% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r1c9-download">Download graphic</button>
</div>
<div id="clusterblast-r1c9-svg"></div>

</div>
</div>

<div class="body-details-section r1c9-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-79"></div> <span class="help-tooltip" id="cb-known-help-79">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c9-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-80"></div> <span class="help-tooltip" id="cb-sub-help-80">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c9-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-82"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-82">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c9-pfam-details-svg" data-filename="scaffold_1_r1c9_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c9-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c9-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-83"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-83">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c9-tigrfam-details-svg" data-filename="scaffold_1_r1c9_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c9-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r1c10' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 10 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-488"></div> <span class="help-tooltip" id="region_svg-help-488">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region010.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c10-svg" data-filename="scaffold_1_r1c10.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,990,917 - 7,052,134 nt. (total: 61,218 nt)

<a class="cluster-rules-header" id="r1c10-rules-header" href="#r1c10">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c10-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r1c10-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-489"></div> <span class="help-tooltip" id="focus-panel-help-489">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c10">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c10-gene-table-details" data-name="r1c10-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c10-MIBiG-cluster-compare" data-name="r1c10-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c10-clusterblast" data-name="r1c10-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c10-knownclusterblast" data-name="r1c10-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c10-subclusterblast" data-name="r1c10-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c10-pfam-details" data-name="r1c10-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c10-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-88"></div> <span class="help-tooltip" id="gt-help-tooltip-help-88">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c10">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c10-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c10-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c10">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c10-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-84"></div> <span class="help-tooltip" id="MIBiG-help-84">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<div><p>No matches found.</p></div>

</div>
</div>



<div class="body-details-section r1c10-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-85"></div> <span class="help-tooltip" id="cb-general-help-85">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c10-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c10_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c10_1.svg">
NW_006271973 (266414-357915): Cordyceps militaris CM01 unplaced genomic scaff... (43% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c10_2.svg">
NW_024468919 (637636-749466): Cryphonectria parasitica EP155 unplaced genomic... (10% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c10_3.svg">
NC_072350 (525276-617173): Talaromyces marneffei chromosome 3, complete sequence (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c10_4.svg">
NW_022985510 (2150254-2227692): Neohortaea acidophila strain CBS 113389 unpla... (6% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r1c10_5.svg">
NW_017253367 (1229921-1261277): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c10_6.svg">
NW_022985463 (11775-102765): Mytilinidion resinicola strain CBS 304.34 unplac... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c10_7.svg">
NW_024467832 (4333687-4396473): Aspergillus wentii DTO 134E9 unplaced genomic... (6% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r1c10-download">Download graphic</button>
</div>
<div id="clusterblast-r1c10-svg"></div>

</div>
</div>

<div class="body-details-section r1c10-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-86"></div> <span class="help-tooltip" id="cb-known-help-86">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c10-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-87"></div> <span class="help-tooltip" id="cb-sub-help-87">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c10-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-89"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-89">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c10-pfam-details-svg" data-filename="scaffold_1_r1c10_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c10-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r1c11' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 11 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-490"></div> <span class="help-tooltip" id="region_svg-help-490">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region011.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c11-svg" data-filename="scaffold_1_r1c11.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 7,156,336 - 7,227,294 nt. (total: 70,959 nt)

<a class="cluster-rules-header" id="r1c11-rules-header" href="#r1c11">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c11-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c11-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-491"></div> <span class="help-tooltip" id="focus-panel-help-491">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c11">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c11-gene-table-details" data-name="r1c11-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c11-nrps_pks" data-name="r1c11-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c11-MIBiG-cluster-compare" data-name="r1c11-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c11-clusterblast" data-name="r1c11-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c11-knownclusterblast" data-name="r1c11-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c11-subclusterblast" data-name="r1c11-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c11-nrps-pks-bubbles" data-name="r1c11-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c11-pfam-details" data-name="r1c11-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c11-tigrfam-details" data-name="r1c11-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c11-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-98"></div> <span class="help-tooltip" id="gt-help-tooltip-help-98">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c11">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c11-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c11-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c11">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c11-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-90"></div> <span class="help-tooltip" id="nrps-domain-help-90">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c11-details-svg" data-filename="scaffold_1_r1c11_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c11-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c11-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-91"></div> <span class="help-tooltip" id="MIBiG-help-91">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c11-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c11-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c11"></div>
<div id="comparison-MIBiG-r1c11-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.59, order:0.88, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.69 (id:0.60, order:0.86, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.70</td>
<td>Polyketide, NRP</td>

<td>beauveriolide I, beauveriolide III</td>


<td>Cordyceps militaris CM01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.60</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.56</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.54</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002203: 0-33767">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002203/index.html#r1c1" target="_new">BGC0002203</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.59, order:0.62, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.60, order:0.60, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.52</td>
<td>NRP, Polyketide, Other</td>

<td>beauveriolide B, beauveriolide C, beauveriolide D</td>


<td>Beauveria bassiana ARSEF 2860</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.50</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.49</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.48</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.46</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.46</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c11-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.60, order:0.88, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide, NRP</td>
<td>beauveriolide I, beauveriolide III</td>
<td>Cordyceps militaris CM01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c11-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-92"></div> <span class="help-tooltip" id="cb-general-help-92">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c11-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c11_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c11_1.svg">
NW_006271969 (4087046-4175862): Cordyceps militaris CM01 unplaced genomic sca... (23% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c11_2.svg">
NC_066263 (4200452-4292984): Aspergillus nidulans FGSC A4 chromosome VII (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c11_3.svg">
NW_025091271 (18416-119910): Aspergillus udagawae IFM 46973 DNA, scaffold 8, ... (21% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c11_4.svg">
NT_165979 (2272028-2354621): Chaetomium globosum CBS 148.51 scaffold 4 genomi... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c11_5.svg">
NC_072354 (38624-176736): Talaromyces marneffei chromosome 7, complete sequence (12% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c11_6.svg">
NC_058399 (6406982-6512577): Fusarium poae strain DAOMC 252244 chromosome 1, ... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c11_7.svg">
NC_054860 (308762-381067): Aspergillus puulaauensis MK2 DNA, chromosome 4, ne... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c11_8.svg">
NW_024467878 (18679-90994): Aspergillus sydowii CBS 593.65 unplaced genomic s... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c11_9.svg">
NW_003052500 (3318567-3442588): Uncinocarpus reesii 1704 scaffold 1 genomic s... (7% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c11_10.svg">
NW_022985363 (1607118-1709581): Macroventuria anomochaeta strain CBS 525.71 u... (9% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r1c11-download">Download graphic</button>
</div>
<div id="clusterblast-r1c11-svg"></div>

</div>
</div>

<div class="body-details-section r1c11-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-93"></div> <span class="help-tooltip" id="cb-known-help-93">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c11-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c11_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c11_1.svg">
beauveriolide I/beauveriolide III
</option>

<option value = "svg/knownclusterblast_r1c11_2.svg">
beauveriolide B/beauveriolide C/beauveriolide D
</option>

<option value = "svg/knownclusterblast_r1c11_3.svg">
emericellamide A/emericellamide B
</option>

</select>
<button id="knownclusterblast-r1c11-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c11-svg"></div>

</div>
</div>

<div class="body-details-section r1c11-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-94"></div> <span class="help-tooltip" id="cb-sub-help-94">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c11-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-97"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-97">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c11-domain-bubble-svg" data-filename="scaffold_1_r1c11_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c11-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC11">
Candidate 11 (7156335 - 7227294): neighbouring T1PKS-NRPS
</option>



<option value="CC12">
Candidate 12 (7156335 - 7206452): single T1PKS
</option>



<option value="CC13">
Candidate 13 (7172977 - 7227294): single NRPS
</option>


</select>

<div class="details-svg" id="r1c11-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c11-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c11-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-99"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-99">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c11-pfam-details-svg" data-filename="scaffold_1_r1c11_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c11-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c11-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-100"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-100">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c11-tigrfam-details-svg" data-filename="scaffold_1_r1c11_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c11-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c11-nrps_pks_products" data-name="r1c11-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c11-nrps_pks_monomers" data-name="r1c11-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c11-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-95"></div> <span class="help-tooltip" id="nrps-structures-help-95">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 11, location 7156335 - 7227294:
<div class="collapser collapser-target-CC11 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c11-11" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c11-11-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 12, location 7156335 - 7206452:
<div class="collapser collapser-target-CC12 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c11-12" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c11-12-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 13, location 7172977 - 7227294:
<div class="collapser collapser-target-CC13 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c11-13" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c11-13-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c11-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-96"></div> <span class="help-tooltip" id="nrps-monomers-help-96">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene2012</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene2012 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (495..815)</strong>: pk
<div class="collapser collapser-target-input-path1-gene2012 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene2012-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
<dd>Ethylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-input-path1-gene2012-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 63.3</dt>

<dd></dd><dt>Methylmalonyl-CoA: 59.5</dt>

<dd></dd><dt>Malonyl-CoA: 33.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 32.6</dt>

<dd></dd><dt>Isobutyryl-CoA: 29.2</dt>

<dd></dd><dt>Propionyl-CoA: 23.1</dt>

<dd></dd><dt>Benzoyl-CoA: 18.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 16.9</dt>

<dd></dd><dt>CHC-CoA: 10.4</dt>

<dd></dd><dt>fatty_acid: 10.2</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">input.path1.gene2016</span></strong>: X
<div class="collapser collapser-target-input-path1-gene2016 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>AMP-binding (495..889)</strong>: X
<div class="collapser collapser-target-input-path1-gene2016 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene2016-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAHFVIGIIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe or Trp <span class="serif">DAFCVIGIIK</span> (82% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene2017</span></strong>: X - Leu
<div class="collapser collapser-target-input-path1-gene2017 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (10..271)</strong>: X
<div class="collapser collapser-target-input-path1-gene2017 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene2017-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLTVVGGMIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DLGIVGGMIK</span> (91% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>





<dd><strong>AMP-binding (738..1129)</strong>: Leu
<div class="collapser collapser-target-input-path1-gene2017 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-input-path1-gene2017-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMHMLGAIIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DMHMLGVIIK</span> (88% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c12' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 12 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-492"></div> <span class="help-tooltip" id="region_svg-help-492">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region012.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c12-svg" data-filename="scaffold_1_r1c12.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 8,118,141 - 8,163,807 nt. (total: 45,667 nt)

<a class="cluster-rules-header" id="r1c12-rules-header" href="#r1c12">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c12-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c12-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-493"></div> <span class="help-tooltip" id="focus-panel-help-493">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c12">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c12-gene-table-details" data-name="r1c12-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c12-nrps_pks" data-name="r1c12-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c12-MIBiG-cluster-compare" data-name="r1c12-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c12-clusterblast" data-name="r1c12-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c12-knownclusterblast" data-name="r1c12-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c12-subclusterblast" data-name="r1c12-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c12-pfam-details" data-name="r1c12-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c12-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-107"></div> <span class="help-tooltip" id="gt-help-tooltip-help-107">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c12">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c12-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c12-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c12">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c12-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-101"></div> <span class="help-tooltip" id="nrps-domain-help-101">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c12-details-svg" data-filename="scaffold_1_r1c12_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c12-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c12-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-102"></div> <span class="help-tooltip" id="MIBiG-help-102">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c12-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c12-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c12"></div>
<div id="comparison-MIBiG-r1c12-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002555: 2045-61815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002555/index.html#r1c1" target="_new">BGC0002555</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP, Polyketide</td>

<td>emerimicin V, emerimicin VI, emerimicin VII, emerimicin VIII, emerimicin IX, emerimicin X</td>


<td>Acremonium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002306: 0-3536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002306/index.html#r1c1" target="_new">BGC0002306</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>freyrasin</td>


<td>Paenibacillus polymyxa ATCC 842</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001639: 0-5012">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001639/index.html#r1c1" target="_new">BGC0001639</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>klebsidin</td>


<td>Klebsiella pneumoniae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>cytolysin ClyLl, cytolysin ClyLs</td>


<td>Plasmid pAD1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001638: 626-5883">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001638/index.html#r1c1" target="_new">BGC0001638</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>klebsazolicin</td>


<td>Klebsiella pneumoniae subsp. ozaenae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001862: 368-5173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001862/index.html#r1c1" target="_new">BGC0001862</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>pallidocin</td>


<td>Aeribacillus pallidus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>carnolysin A1, carnolysin A2</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c12-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002555: 2045-61815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002555/index.html#r1c1" target="_new">BGC0002555</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP, Polyketide</td>
<td>emerimicin V, emerimicin VI, emerimicin VII, emerimicin VIII, emerimicin IX, emerimicin X</td>
<td>Acremonium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>SRO15-3108</td>
<td>Streptomyces filamentosus NRRL 15998</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002306: 0-3536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002306/index.html#r1c1" target="_new">BGC0002306</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>freyrasin</td>
<td>Paenibacillus polymyxa ATCC 842</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001639: 0-5012">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001639/index.html#r1c1" target="_new">BGC0001639</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>klebsidin</td>
<td>Klebsiella pneumoniae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>cytolysin ClyLl, cytolysin ClyLs</td>
<td>Plasmid pAD1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001638: 626-5883">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001638/index.html#r1c1" target="_new">BGC0001638</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>klebsazolicin</td>
<td>Klebsiella pneumoniae subsp. ozaenae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001862: 368-5173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001862/index.html#r1c1" target="_new">BGC0001862</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>pallidocin</td>
<td>Aeribacillus pallidus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>carnolysin A1, carnolysin A2</td>
<td>Carnobacterium maltaromaticum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c12-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-103"></div> <span class="help-tooltip" id="cb-general-help-103">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c12-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c12_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c12_1.svg">
NW_006271975 (1558381-1621743): Cordyceps militaris CM01 unplaced genomic sca... (30% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c12_2.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (3% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c12_3.svg">
NW_011371366 (647325-794373): Paracoccidioides brasiliensis Pb18 unplaced gen... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c12_4.svg">
NW_015440960 (427476-569911): Paracoccidioides lutzii Pb01 unplaced genomic s... (3% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c12_5.svg">
NW_024067568 (58665-121970): Penicillium roqueforti strain LCP96 04111 chromo... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c12_6.svg">
NW_024470072 (71955-135189): Aureobasidium melanogenum CBS 110374 unplaced ge... (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r1c12_7.svg">
NW_020291840 (131998-235388): Aspergillus costaricaensis CBS 115574 unplaced ... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c12_8.svg">
NW_023336277 (142329-245745): Aspergillus tubingensis WU-2223L DNA, scaffold ... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c12_9.svg">
NW_020291596 (1375571-1479003): Aspergillus piperis CBS 112811 unplaced genom... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c12_10.svg">
NW_020292074 (140873-244304): Aspergillus vadensis CBS 113365 unplaced genomi... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r1c12-download">Download graphic</button>
</div>
<div id="clusterblast-r1c12-svg"></div>

</div>
</div>

<div class="body-details-section r1c12-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-104"></div> <span class="help-tooltip" id="cb-known-help-104">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c12-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-105"></div> <span class="help-tooltip" id="cb-sub-help-105">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c12-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-108"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-108">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c12-pfam-details-svg" data-filename="scaffold_1_r1c12_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c12-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c12-nrps_pks_monomers" data-name="r1c12-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r1c12-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-106"></div> <span class="help-tooltip" id="nrps-monomers-help-106">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene2271</span></strong>: X
<div class="collapser collapser-target-input-path1-gene2271 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (442..732)</strong>: X
<div class="collapser collapser-target-input-path1-gene2271 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene2271-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>  <strong>NOTE: uncertain match</strong><br>
<dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GLFIAG-VDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-494"></div> <span class="help-tooltip" id="region_svg-help-494">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="scaffold_2_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 163,991 - 216,113 nt. (total: 52,123 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-495"></div> <span class="help-tooltip" id="focus-panel-help-495">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-nrps_pks" data-name="r2c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c1-nrps-pks-bubbles" data-name="r2c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-117"></div> <span class="help-tooltip" id="gt-help-tooltip-help-117">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-109"></div> <span class="help-tooltip" id="nrps-domain-help-109">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-details-svg" data-filename="scaffold_2_r2c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-110"></div> <span class="help-tooltip" id="MIBiG-help-110">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.62</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.51</td>
<td>Polyketide, NRP</td>

<td>xenoacremone A</td>


<td>Xenoacremonium sp. BF-2018a</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002139: 487-22086">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002139/index.html#r1c1" target="_new">BGC0002139</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Polyketide</td>

<td>burnettiene A, preburnettiene B, preburnettiene A</td>


<td>Aspergillus burnettii FRR 5400</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide, NRP</td>
<td>xenoacremone A</td>
<td>Xenoacremonium sp. BF-2018a</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002139: 487-22086">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002139/index.html#r1c1" target="_new">BGC0002139</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>burnettiene A, preburnettiene B, preburnettiene A</td>
<td>Aspergillus burnettii FRR 5400</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-111"></div> <span class="help-tooltip" id="cb-general-help-111">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NW_020292039 (3332166-3473157): Aspergillus vadensis CBS 113365 unplaced geno... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NW_023336282 (1703318-1900820): Aspergillus tubingensis WU-2223L DNA, scaffol... (4% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NW_024467862 (1993078-2065565): Aspergillus sydowii CBS 593.65 unplaced genom... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NW_025953831 (148948-216846): Annulohypoxylon truncatum strain CBS 140777 unp... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NW_025953261 (917015-984994): Annulohypoxylon maeteangense strain CBS 123835 ... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NC_058388 (4682141-4750888): Fusarium musae strain F31 chromosome 2, whole ge... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NC_072350 (3049070-3121085): Talaromyces marneffei chromosome 3, complete seq... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NW_024067574 (235773-308013): Penicillium roqueforti strain LCP96 04111 chrom... (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NW_017264196 (2263112-2335444): Xylona heveae TC161 unplaced genomic scaffold... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NC_072842 (4536206-4608469): Fusarium oxysporum Fo47 chromosome III, complete... (8% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-112"></div> <span class="help-tooltip" id="cb-known-help-112">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-113"></div> <span class="help-tooltip" id="cb-sub-help-113">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-116"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-116">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-domain-bubble-svg" data-filename="scaffold_2_r2c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (163990 - 216113): chemical hybrid T1PKS-NRPS
</option>


</select>

<div class="details-svg" id="r2c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-118"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-118">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="scaffold_2_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-119"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-119">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="scaffold_2_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c1-nrps_pks_products" data-name="r2c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c1-nrps_pks_monomers" data-name="r2c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-114"></div> <span class="help-tooltip" id="nrps-structures-help-114">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 163990 - 216113:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-115"></div> <span class="help-tooltip" id="nrps-monomers-help-115">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene55</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene55 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (391..705)</strong>: pk
<div class="collapser collapser-target-input-path1-gene55 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene55-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene55-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 85.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 71.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 61.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 59.1</dt>

<dd></dd><dt>Malonyl-CoA: 45.0</dt>

<dd></dd><dt>Benzoyl-CoA: 41.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 27.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 22.4</dt>

<dd></dd><dt>Propionyl-CoA: 22.2</dt>

<dd></dd><dt>Acetyl-CoA: 16.4</dt>

<dd></dd><dt>CHC-CoA: 14.2</dt>

<dd></dd><dt>fatty_acid: 11.8</dt>

<dd></dd><dt>inactive: 11.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 6.7</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>













<dd><strong>AMP-binding (2546..2939)</strong>: X
<div class="collapser collapser-target-input-path1-gene55 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  bMe-Phe
<div class="collapser collapser-target-input-path1-gene55-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFHLLGFIAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>bMe-Phe <span class="serif">DGWLLGFIAK</span> (47% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 2 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-496"></div> <span class="help-tooltip" id="region_svg-help-496">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c2-svg" data-filename="scaffold_2_r2c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 642,561 - 686,369 nt. (total: 43,809 nt)

<a class="cluster-rules-header" id="r2c2-rules-header" href="#r2c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-497"></div> <span class="help-tooltip" id="focus-panel-help-497">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c2-gene-table-details" data-name="r2c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c2-nrps_pks" data-name="r2c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c2-MIBiG-cluster-compare" data-name="r2c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c2-clusterblast" data-name="r2c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c2-knownclusterblast" data-name="r2c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c2-subclusterblast" data-name="r2c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c2-nrps-pks-bubbles" data-name="r2c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c2-pfam-details" data-name="r2c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c2-tigrfam-details" data-name="r2c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-128"></div> <span class="help-tooltip" id="gt-help-tooltip-help-128">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-120"></div> <span class="help-tooltip" id="nrps-domain-help-120">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-details-svg" data-filename="scaffold_2_r2c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-121"></div> <span class="help-tooltip" id="MIBiG-help-121">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c2"></div>
<div id="comparison-MIBiG-r2c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-122"></div> <span class="help-tooltip" id="cb-general-help-122">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c2_1.svg">
NW_025764721 (203011-307075): Emericellopsis atlantica unplaced genomic scaff... (7% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r2c2-download">Download graphic</button>
</div>
<div id="clusterblast-r2c2-svg"></div>

</div>
</div>

<div class="body-details-section r2c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-123"></div> <span class="help-tooltip" id="cb-known-help-123">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-124"></div> <span class="help-tooltip" id="cb-sub-help-124">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-127"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-127">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-domain-bubble-svg" data-filename="scaffold_2_r2c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (642560 - 686369): single NRPS-like
</option>


</select>

<div class="details-svg" id="r2c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-129"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-129">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-pfam-details-svg" data-filename="scaffold_2_r2c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-130"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-130">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-tigrfam-details-svg" data-filename="scaffold_2_r2c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c2-nrps_pks_products" data-name="r2c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c2-nrps_pks_monomers" data-name="r2c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-125"></div> <span class="help-tooltip" id="nrps-structures-help-125">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 642560 - 686369:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c2-2" class="smiles-canvas" data-smiles="NC(CCCC(=O)O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Aad)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCC(=O)O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-126"></div> <span class="help-tooltip" id="nrps-monomers-help-126">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene174</span></strong>: Aad
<div class="collapser collapser-target-input-path1-gene174 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (22..488)</strong>: Aad
<div class="collapser collapser-target-input-path1-gene174 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Aad
<div class="collapser collapser-target-input-path1-gene174-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPRHFVMRAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Aad <span class="serif">DPRHFVMRAK</span> (94% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-498"></div> <span class="help-tooltip" id="region_svg-help-498">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c3-svg" data-filename="scaffold_2_r2c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 943,118 - 991,234 nt. (total: 48,117 nt)

<a class="cluster-rules-header" id="r2c3-rules-header" href="#r2c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-499"></div> <span class="help-tooltip" id="focus-panel-help-499">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c3-gene-table-details" data-name="r2c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c3-nrps_pks" data-name="r2c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c3-MIBiG-cluster-compare" data-name="r2c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c3-clusterblast" data-name="r2c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c3-knownclusterblast" data-name="r2c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c3-subclusterblast" data-name="r2c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c3-nrps-pks-bubbles" data-name="r2c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c3-pfam-details" data-name="r2c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-139"></div> <span class="help-tooltip" id="gt-help-tooltip-help-139">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-131"></div> <span class="help-tooltip" id="nrps-domain-help-131">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-details-svg" data-filename="scaffold_2_r2c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-132"></div> <span class="help-tooltip" id="MIBiG-help-132">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c3"></div>
<div id="comparison-MIBiG-r2c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>nidulanin A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>nidulanin A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-133"></div> <span class="help-tooltip" id="cb-general-help-133">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c3_1.svg">
NW_020289984 (358881-432918): Jaminaea rosea strain MCA 5214 unplaced genomic... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c3_2.svg">
NC_026476 (5895028-5969337): Fusarium graminearum PH-1 chromosome 3, whole ge... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c3_3.svg">
NW_025408668 (7195256-7268741): Hirsutella rhossiliensis strain HR02 chromoso... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c3_4.svg">
NT_165977 (336072-410944): Chaetomium globosum CBS 148.51 scaffold 2 genomic ... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c3_5.svg">
NC_070554 (2084847-2158774): Fusarium falciforme chromosome 11, complete sequ... (23% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c3_6.svg">
NC_072850 (397562-471903): Fusarium oxysporum Fo47 chromosome XI, complete se... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c3_7.svg">
NC_058391 (4271687-4345998): Fusarium musae strain F31 chromosome 5, whole ge... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c3_8.svg">
NC_058401 (7122271-7196560): Fusarium poae strain DAOMC 252244 chromosome 3, ... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c3_9.svg">
NC_038014 (6703493-6777780): Fusarium venenatum strain A3/5 genome assembly, ... (12% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r2c3_10.svg">
NW_025764703 (74053-148318): Emericellopsis atlantica unplaced genomic scaffo... (17% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r2c3-download">Download graphic</button>
</div>
<div id="clusterblast-r2c3-svg"></div>

</div>
</div>

<div class="body-details-section r2c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-134"></div> <span class="help-tooltip" id="cb-known-help-134">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-135"></div> <span class="help-tooltip" id="cb-sub-help-135">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-138"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-138">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-domain-bubble-svg" data-filename="scaffold_2_r2c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (943117 - 991234): single NRPS
</option>


</select>

<div class="details-svg" id="r2c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-140"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-140">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-pfam-details-svg" data-filename="scaffold_2_r2c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c3-nrps_pks_products" data-name="r2c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c3-nrps_pks_monomers" data-name="r2c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-136"></div> <span class="help-tooltip" id="nrps-structures-help-136">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 943117 - 991234:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c3-3" class="smiles-canvas" data-smiles="NC(CS)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Cys)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-137"></div> <span class="help-tooltip" id="nrps-monomers-help-137">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene258</span></strong>: Cys - X
<div class="collapser collapser-target-input-path1-gene258 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (375..797)</strong>: Cys
<div class="collapser collapser-target-input-path1-gene258 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Cys
<div class="collapser collapser-target-input-path1-gene258-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVQHTITVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Cys <span class="serif">DVQHTITVVK</span> (74% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1553..1958)</strong>: X
<div class="collapser collapser-target-input-path1-gene258 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-input-path1-gene258-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPTGTGFIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DPLSTGAIGK</span> (62% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 4 - NRP-metallophore,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-500"></div> <span class="help-tooltip" id="region_svg-help-500">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c4-svg" data-filename="scaffold_2_r2c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,037,874 - 1,099,097 nt. (total: 61,224 nt)

<a class="cluster-rules-header" id="r2c4-rules-header" href="#r2c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c4-rules">NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-501"></div> <span class="help-tooltip" id="focus-panel-help-501">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c4-gene-table-details" data-name="r2c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c4-nrps_pks" data-name="r2c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c4-MIBiG-cluster-compare" data-name="r2c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c4-clusterblast" data-name="r2c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c4-knownclusterblast" data-name="r2c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c4-subclusterblast" data-name="r2c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c4-nrps-pks-bubbles" data-name="r2c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c4-pfam-details" data-name="r2c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c4-tigrfam-details" data-name="r2c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-149"></div> <span class="help-tooltip" id="gt-help-tooltip-help-149">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-141"></div> <span class="help-tooltip" id="nrps-domain-help-141">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-details-svg" data-filename="scaffold_2_r2c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-142"></div> <span class="help-tooltip" id="MIBiG-help-142">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c4"></div>
<div id="comparison-MIBiG-r2c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.40</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002424: 0-19621">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002424/index.html#r1c1" target="_new">BGC0002424</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>saccharochelin A, saccharochelin B, saccharochelin C, saccharochelin D, saccharochelin E, saccharochelin F, saccharochelin G, saccharochelin H</td>


<td>Saccharothrix sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>pseudomonine</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767: 114-29921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>salinichelins</td>


<td>Salinispora pacifica CNY331</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001211: 0-33663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001211/index.html#r1c1" target="_new">BGC0001211</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>albachelin</td>


<td>Amycolatopsis alba DSM 44262</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>sespendole</td>


<td>Cordana terrestris</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002424: 0-19621">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002424/index.html#r1c1" target="_new">BGC0002424</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>saccharochelin A, saccharochelin B, saccharochelin C, saccharochelin D, saccharochelin E, saccharochelin F, saccharochelin G, saccharochelin H</td>
<td>Saccharothrix sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>pseudomonine</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767: 114-29921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>NRP</td>
<td>salinichelins</td>
<td>Salinispora pacifica CNY331</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>tabtoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001211: 0-33663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001211/index.html#r1c1" target="_new">BGC0001211</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>NRP</td>
<td>albachelin</td>
<td>Amycolatopsis alba DSM 44262</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-143"></div> <span class="help-tooltip" id="cb-general-help-143">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c4_1.svg">
NW_006271969 (5451891-5533992): Cordyceps militaris CM01 unplaced genomic sca... (76% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r2c4_2.svg">
NC_035790 (6855176-6936375): Pochonia chlamydosporia 170 chromosome 1, whole ... (66% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r2c4_3.svg">
NW_006711178 (9878-84649): Trichoderma reesei QM6a unplaced genomic scaffold ... (50% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c4_4.svg">
NW_014574715 (339608-420863): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (66% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r2c4_5.svg">
NW_011942149 (5526781-5608010): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (66% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r2c4_6.svg">
NC_063070 (2187850-2275304): Purpureocillium takamizusanense chromosome 3, co... (56% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r2c4_7.svg">
NW_023336198 (10188-84865): Geosmithia morbida isolate 1262 chromosome Unknow... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c4_8.svg">
NC_070530 (3742001-3826089): Fusarium keratoplasticum isolate Fu6.1 chromosom... (52% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r2c4_9.svg">
NC_070545 (3816730-3900832): Fusarium falciforme chromosome 2, complete sequence (52% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r2c4_10.svg">
NC_026476 (2004097-2087970): Fusarium graminearum PH-1 chromosome 3, whole ge... (37% of genes show similarity), NRP-metallophore,NRPS
</option>

</select>
<button id= "clusterblast-r2c4-download">Download graphic</button>
</div>
<div id="clusterblast-r2c4-svg"></div>

</div>
</div>

<div class="body-details-section r2c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-144"></div> <span class="help-tooltip" id="cb-known-help-144">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-145"></div> <span class="help-tooltip" id="cb-sub-help-145">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-148"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-148">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-domain-bubble-svg" data-filename="scaffold_2_r2c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (1037873 - 1099097): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r2c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-150"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-150">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-pfam-details-svg" data-filename="scaffold_2_r2c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-151"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-151">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-tigrfam-details-svg" data-filename="scaffold_2_r2c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c4-nrps_pks_products" data-name="r2c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c4-nrps_pks_monomers" data-name="r2c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-146"></div> <span class="help-tooltip" id="nrps-structures-help-146">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 1037873 - 1099097:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c4-4" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-147"></div> <span class="help-tooltip" id="nrps-monomers-help-147">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene286</span></strong>: X - X - X
<div class="collapser collapser-target-input-path1-gene286 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (32..421)</strong>: X
<div class="collapser collapser-target-input-path1-gene286 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene286-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFELIMIHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Orn <span class="serif">DVSELGTINK</span> (59% 8Ã match)</dd>
<dd>Dab <span class="serif">DVGELSSIDK</span> (41% 8Ã match)</dd>
<dd>Ser <span class="serif">DLFNLGLIHK</span> (38% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>



<dd><strong>AMP-binding (865..1259)</strong>: X
<div class="collapser collapser-target-input-path1-gene286 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene286-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFSVAGNIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DVFSVALVYK</span> (56% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2474..2873)</strong>: X
<div class="collapser collapser-target-input-path1-gene286 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-input-path1-gene286-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVLDIGAIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DVLSFGAIIK</span> (65% 8Ã match)</dd>
<dd>Phe or Trp <span class="serif">DVLSIGAVVK</span> (62% 8Ã match)</dd>
<dd>Arg <span class="serif">DVESIGAIAK</span> (56% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 5 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-502"></div> <span class="help-tooltip" id="region_svg-help-502">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c5-svg" data-filename="scaffold_2_r2c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,402,466 - 1,450,513 nt. (total: 48,048 nt)

<a class="cluster-rules-header" id="r2c5-rules-header" href="#r2c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c5-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-503"></div> <span class="help-tooltip" id="focus-panel-help-503">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c5-gene-table-details" data-name="r2c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c5-nrps_pks" data-name="r2c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c5-MIBiG-cluster-compare" data-name="r2c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c5-clusterblast" data-name="r2c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c5-knownclusterblast" data-name="r2c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c5-subclusterblast" data-name="r2c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c5-nrps-pks-bubbles" data-name="r2c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c5-pfam-details" data-name="r2c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c5-tigrfam-details" data-name="r2c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-160"></div> <span class="help-tooltip" id="gt-help-tooltip-help-160">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-152"></div> <span class="help-tooltip" id="nrps-domain-help-152">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-details-svg" data-filename="scaffold_2_r2c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-153"></div> <span class="help-tooltip" id="MIBiG-help-153">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c5"></div>
<div id="comparison-MIBiG-r2c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>scytalidin</td>


<td>Scytalidium album</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:0.48, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>scytalidin</td>
<td>Scytalidium album</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:0.48, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-154"></div> <span class="help-tooltip" id="cb-general-help-154">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c5_1.svg">
NW_014574615 (637002-705039): Penicillium digitatum Pd1 unplaced genomic scaf... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_2.svg">
NW_025953251 (1891693-2006513): Annulohypoxylon maeteangense strain CBS 12383... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_3.svg">
NW_020290894 (331892-411939): Aspergillus niger CBS 101883 unplaced genomic s... (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_4.svg">
NC_054856 (1720508-1886717): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c5_5.svg">
NW_007360988 (1313-69766): Glarea lozoyensis ATCC 20868 chromosome Unknown GL... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_6.svg">
NW_022474215 (790980-859315): Venustampulla echinocandica strain BP 5553 chro... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_7.svg">
NW_017263607 (750023-818389): Mollisia scopiformis unplaced genomic scaffold ... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_8.svg">
NC_064655 (797649-865606): Penicillium oxalicum strain HP7-1 chromosome VI, w... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_9.svg">
NW_024467841 (224613-292322): Aspergillus wentii DTO 134E9 unplaced genomic s... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c5_10.svg">
NW_006711154 (185525-254521): Trichoderma reesei QM6a unplaced genomic scaffo... (9% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r2c5-download">Download graphic</button>
</div>
<div id="clusterblast-r2c5-svg"></div>

</div>
</div>

<div class="body-details-section r2c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-155"></div> <span class="help-tooltip" id="cb-known-help-155">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-156"></div> <span class="help-tooltip" id="cb-sub-help-156">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-159"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-159">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-domain-bubble-svg" data-filename="scaffold_2_r2c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (1402465 - 1450513): single T1PKS
</option>


</select>

<div class="details-svg" id="r2c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-161"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-161">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-pfam-details-svg" data-filename="scaffold_2_r2c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-162"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-162">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-tigrfam-details-svg" data-filename="scaffold_2_r2c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c5-nrps_pks_products" data-name="r2c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c5-nrps_pks_monomers" data-name="r2c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-157"></div> <span class="help-tooltip" id="nrps-structures-help-157">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 1402465 - 1450513:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c5-5" class="smiles-canvas" data-smiles="C(C)CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-redmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-redmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-redmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c5-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C(C)CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-158"></div> <span class="help-tooltip" id="nrps-monomers-help-158">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene376</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene376 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (522..843)</strong>: mal
<div class="collapser collapser-target-input-path1-gene376 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene376-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene376-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 85.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 76.8</dt>

<dd></dd><dt>Methylmalonyl-CoA: 71.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 63.3</dt>

<dd></dd><dt>inactive: 37.8</dt>

<dd></dd><dt>Propionyl-CoA: 23.1</dt>

<dd></dd><dt>fatty_acid: 14.8</dt>

<dd></dd><dt>Acetyl-CoA: 13.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 11.7</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.0</dt>

<dd></dd><dt>CHC-CoA: 8.8</dt>

<dd></dd><dt>Benzoyl-CoA: 6.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: -1.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 6 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-504"></div> <span class="help-tooltip" id="region_svg-help-504">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c6-svg" data-filename="scaffold_2_r2c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,230,916 - 2,253,026 nt. (total: 22,111 nt)

<a class="cluster-rules-header" id="r2c6-rules-header" href="#r2c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c6-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-505"></div> <span class="help-tooltip" id="focus-panel-help-505">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c6-gene-table-details" data-name="r2c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c6-MIBiG-cluster-compare" data-name="r2c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c6-clusterblast" data-name="r2c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c6-knownclusterblast" data-name="r2c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c6-subclusterblast" data-name="r2c6-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c6-pfam-details" data-name="r2c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c6-tigrfam-details" data-name="r2c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-167"></div> <span class="help-tooltip" id="gt-help-tooltip-help-167">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-163"></div> <span class="help-tooltip" id="MIBiG-help-163">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c6"></div>
<div id="comparison-MIBiG-r2c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.53, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>squalestatin S1</td>


<td>Aspergillus sp. Z5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>cytolysin ClyLl, cytolysin ClyLs</td>


<td>Plasmid pAD1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>microcyclamide</td>


<td>Microcystis aeruginosa NIES-298</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>carbapenem MM4550</td>


<td>Streptomyces argenteolus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.52, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>squalestatin S1</td>


<td>Phoma sp. MF5453</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>gramicidin S</td>


<td>Aneurinibacillus migulanus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>





<td class="cc-heat-cell" title="0.15 (id:0.46, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>sorangipyranone</td>


<td>Myxococcales bacterium</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.53, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>squalestatin S1</td>
<td>Aspergillus sp. Z5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>cytolysin ClyLl, cytolysin ClyLs</td>
<td>Plasmid pAD1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>RiPP</td>
<td>microcyclamide</td>
<td>Microcystis aeruginosa NIES-298</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>carbapenem MM4550</td>
<td>Streptomyces argenteolus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.52, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Polyketide</td>
<td>squalestatin S1</td>
<td>Phoma sp. MF5453</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP</td>
<td>gramicidin S</td>
<td>Aneurinibacillus migulanus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>


<td class="cc-heat-cell" title="0.15 (id:0.46, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
<td>0.15</td>
<td>Polyketide</td>
<td>sorangipyranone</td>
<td>Myxococcales bacterium</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-164"></div> <span class="help-tooltip" id="cb-general-help-164">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c6_1.svg">
NW_006271974 (1078266-1109834): Cordyceps militaris CM01 unplaced genomic sca... (36% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_2.svg">
NC_070529 (5465471-5497016): Fusarium keratoplasticum isolate Fu6.1 chromosom... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_3.svg">
NC_070544 (5794890-5826434): Fusarium falciforme chromosome 1, complete sequence (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_4.svg">
NC_036623 (3601013-3632563): Fusarium fujikuroi IMI 58289 draft genome, chrom... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_5.svg">
NC_058389 (1466023-1497573): Fusarium musae strain F31 chromosome 3, whole ge... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_6.svg">
NC_026477 (6663266-6694813): Fusarium graminearum PH-1 chromosome 4, whole ge... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_7.svg">
NC_038015 (6636088-6667634): Fusarium venenatum strain A3/5 genome assembly, ... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_8.svg">
NC_058402 (6845329-6876878): Fusarium poae strain DAOMC 252244 chromosome 4, ... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_9.svg">
NW_025764705 (1077863-1109318): Emericellopsis atlantica unplaced genomic sca... (28% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c6_10.svg">
NW_023336189 (374372-405746): Geosmithia morbida isolate 1262 chromosome Unkn... (44% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r2c6-download">Download graphic</button>
</div>
<div id="clusterblast-r2c6-svg"></div>

</div>
</div>

<div class="body-details-section r2c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-165"></div> <span class="help-tooltip" id="cb-known-help-165">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c6-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c6_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c6_1.svg">
squalestatin S1
</option>

</select>
<button id="knownclusterblast-r2c6-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c6-svg"></div>

</div>
</div>

<div class="body-details-section r2c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-166"></div> <span class="help-tooltip" id="cb-sub-help-166">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-168"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-168">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-pfam-details-svg" data-filename="scaffold_2_r2c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-169"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-169">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-tigrfam-details-svg" data-filename="scaffold_2_r2c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r2c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 7 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-506"></div> <span class="help-tooltip" id="region_svg-help-506">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c7-svg" data-filename="scaffold_2_r2c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,410,850 - 2,458,506 nt. (total: 47,657 nt)

<a class="cluster-rules-header" id="r2c7-rules-header" href="#r2c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c7-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-507"></div> <span class="help-tooltip" id="focus-panel-help-507">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c7-gene-table-details" data-name="r2c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c7-nrps_pks" data-name="r2c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c7-MIBiG-cluster-compare" data-name="r2c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c7-clusterblast" data-name="r2c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c7-knownclusterblast" data-name="r2c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c7-subclusterblast" data-name="r2c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c7-nrps-pks-bubbles" data-name="r2c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c7-pfam-details" data-name="r2c7-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c7-tigrfam-details" data-name="r2c7-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-178"></div> <span class="help-tooltip" id="gt-help-tooltip-help-178">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-170"></div> <span class="help-tooltip" id="nrps-domain-help-170">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-details-svg" data-filename="scaffold_2_r2c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-171"></div> <span class="help-tooltip" id="MIBiG-help-171">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c7"></div>
<div id="comparison-MIBiG-r2c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001037: 0-18799">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001037/index.html#r1c1" target="_new">BGC0001037</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP, Polyketide</td>

<td>pseurotin, azaspirene</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>PR-toxin</td>


<td>Penicillium roqueforti FM164</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001037: 0-18799">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001037/index.html#r1c1" target="_new">BGC0001037</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP, Polyketide</td>
<td>pseurotin, azaspirene</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>PR-toxin</td>
<td>Penicillium roqueforti FM164</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-172"></div> <span class="help-tooltip" id="cb-general-help-172">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c7_1.svg">
NW_025953066 (59356-127683): Daldinia caldariorum strain CBS 122874 unplaced ... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c7_2.svg">
NW_025091264 (5340181-5432359): Aspergillus udagawae IFM 46973 DNA, scaffold ... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c7_3.svg">
NW_025091217 (93705-162168): Aspergillus viridinutans IFM 47045 DNA, scaffold... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c7_4.svg">
NW_022983478 (652553-879885): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (4% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_5.svg">
NW_025091284 (1562771-1677454): Aspergillus pseudoviridinutans IFM 55266 DNA,... (8% of genes show similarity), NI-siderophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c7_6.svg">
NC_030957 (146150-210453): Colletotrichum higginsianum IMI 349063 chromosome ... (20% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r2c7_7.svg">
NW_021167080 (4131403-4218796): Sodiomyces alkalinus F11 unplaced genomic sca... (19% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c7_8.svg">
NW_023336281 (1676508-1751599): Aspergillus tubingensis WU-2223L DNA, scaffol... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c7_9.svg">
NW_020291807 (494786-574110): Aspergillus costaricaensis CBS 115574 unplaced ... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c7_10.svg">
NW_007361002 (231437-369047): Glarea lozoyensis ATCC 20868 chromosome Unknown... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r2c7-download">Download graphic</button>
</div>
<div id="clusterblast-r2c7-svg"></div>

</div>
</div>

<div class="body-details-section r2c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-173"></div> <span class="help-tooltip" id="cb-known-help-173">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-174"></div> <span class="help-tooltip" id="cb-sub-help-174">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-177"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-177">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-domain-bubble-svg" data-filename="scaffold_2_r2c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (2410849 - 2458506): single T1PKS
</option>


</select>

<div class="details-svg" id="r2c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-179"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-179">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-pfam-details-svg" data-filename="scaffold_2_r2c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c7-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-180"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-180">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-tigrfam-details-svg" data-filename="scaffold_2_r2c7_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c7-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c7-nrps_pks_products" data-name="r2c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c7-nrps_pks_monomers" data-name="r2c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-175"></div> <span class="help-tooltip" id="nrps-structures-help-175">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 2410849 - 2458506:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c7-7" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c7-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-176"></div> <span class="help-tooltip" id="nrps-monomers-help-176">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene654</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene654 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (370..685)</strong>: pk
<div class="collapser collapser-target-input-path1-gene654 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene654-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>CE-Malonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene654-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 68.3</dt>

<dd></dd><dt>Malonyl-CoA: 59.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 59.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 31.3</dt>

<dd></dd><dt>Propionyl-CoA: 30.7</dt>

<dd></dd><dt>inactive: 28.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 28.2</dt>

<dd></dd><dt>Benzoyl-CoA: 21.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 21.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 15.1</dt>

<dd></dd><dt>Acetyl-CoA: 13.7</dt>

<dd></dd><dt>CHC-CoA: 11.1</dt>

<dd></dd><dt>fatty_acid: 7.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 8 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-508"></div> <span class="help-tooltip" id="region_svg-help-508">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c8-svg" data-filename="scaffold_2_r2c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,880,687 - 2,949,376 nt. (total: 68,690 nt)

<a class="cluster-rules-header" id="r2c8-rules-header" href="#r2c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c8-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-509"></div> <span class="help-tooltip" id="focus-panel-help-509">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c8-gene-table-details" data-name="r2c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c8-nrps_pks" data-name="r2c8-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c8-MIBiG-cluster-compare" data-name="r2c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c8-clusterblast" data-name="r2c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c8-knownclusterblast" data-name="r2c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c8-subclusterblast" data-name="r2c8-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c8-nrps-pks-bubbles" data-name="r2c8-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c8-pfam-details" data-name="r2c8-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-189"></div> <span class="help-tooltip" id="gt-help-tooltip-help-189">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c8-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-181"></div> <span class="help-tooltip" id="nrps-domain-help-181">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c8-details-svg" data-filename="scaffold_2_r2c8_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c8-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-182"></div> <span class="help-tooltip" id="MIBiG-help-182">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c8"></div>
<div id="comparison-MIBiG-r2c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.69</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP, Polyketide</td>

<td>himeic acid A</td>


<td>Aspergillus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide, NRP</td>

<td>pyranoviolin A</td>


<td>Aspergillus violaceofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-Î²-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001563: 1377-52987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001563/index.html#r1c1" target="_new">BGC0001563</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP, Polyketide</td>

<td>curvupallide-B</td>


<td>Cochliobolus pallescens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>6-MSA, aculin A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>himeic acid A</td>
<td>Aspergillus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide, NRP</td>
<td>pyranoviolin A</td>
<td>Aspergillus violaceofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-Î²-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-183"></div> <span class="help-tooltip" id="cb-general-help-183">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c8_1.svg">
NW_006271971 (3503471-3590640): Cordyceps militaris CM01 unplaced genomic sca... (70% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c8_2.svg">
NW_020290887 (673812-764439): Aspergillus niger CBS 101883 unplaced genomic s... (28% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c8_3.svg">
NW_023336287 (1029426-1127137): Aspergillus tubingensis WU-2223L DNA, scaffol... (29% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c8_4.svg">
NW_020292069 (316400-408453): Aspergillus vadensis CBS 113365 unplaced genomi... (30% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c8_5.svg">
NW_020291625 (99659-196841): Aspergillus piperis CBS 112811 unplaced genomic ... (29% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c8_6.svg">
NW_020291816 (340965-433020): Aspergillus costaricaensis CBS 115574 unplaced ... (29% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c8_7.svg">
NC_054853 (2119459-2216438): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (25% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c8_8.svg">
NW_013581207 (483365-550922): Aureobasidium namibiae CBS 147.97 unplaced geno... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c8_9.svg">
NW_022474207 (2338451-2410505): Venustampulla echinocandica strain BP 5553 ch... (19% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c8_10.svg">
NW_013566983 (689939-757505): Aureobasidium subglaciale EXF-2481 unplaced gen... (15% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r2c8-download">Download graphic</button>
</div>
<div id="clusterblast-r2c8-svg"></div>

</div>
</div>

<div class="body-details-section r2c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-184"></div> <span class="help-tooltip" id="cb-known-help-184">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-185"></div> <span class="help-tooltip" id="cb-sub-help-185">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c8-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-188"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-188">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c8-domain-bubble-svg" data-filename="scaffold_2_r2c8_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c8-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC8">
Candidate 8 (2880686 - 2949376): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r2c8-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c8-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-190"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-190">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c8-pfam-details-svg" data-filename="scaffold_2_r2c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c8-nrps_pks_products" data-name="r2c8-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c8-nrps_pks_monomers" data-name="r2c8-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c8-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-186"></div> <span class="help-tooltip" id="nrps-structures-help-186">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 8, location 2880686 - 2949376:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c8-8" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c8-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c8-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-187"></div> <span class="help-tooltip" id="nrps-monomers-help-187">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene787</span></strong>: X
<div class="collapser collapser-target-input-path1-gene787 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (260..697)</strong>: X
<div class="collapser collapser-target-input-path1-gene787 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene787-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLIFVGIMAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DALFVGLVAK</span> (59% 8Ã match)</dd>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (56% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">input.path1.gene791</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene791 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (547..865)</strong>: pk
<div class="collapser collapser-target-input-path1-gene791 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene791-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene791-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 69.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 60.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 44.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 26.7</dt>

<dd></dd><dt>Malonyl-CoA: 24.9</dt>

<dd></dd><dt>fatty_acid: 23.7</dt>

<dd></dd><dt>Propionyl-CoA: 22.0</dt>

<dd></dd><dt>Benzoyl-CoA: 21.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 21.2</dt>

<dd></dd><dt>CHC-CoA: 10.8</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2997..3404)</strong>: X
<div class="collapser collapser-target-input-path1-gene791 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene791-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIDTNGNVWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGVVWK</span> (56% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c9' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 9 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-510"></div> <span class="help-tooltip" id="region_svg-help-510">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region009.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c9-svg" data-filename="scaffold_2_r2c9.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,573,753 - 3,634,823 nt. (total: 61,071 nt)

<a class="cluster-rules-header" id="r2c9-rules-header" href="#r2c9">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c9-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r2c9-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-511"></div> <span class="help-tooltip" id="focus-panel-help-511">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c9">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c9-gene-table-details" data-name="r2c9-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c9-MIBiG-cluster-compare" data-name="r2c9-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c9-clusterblast" data-name="r2c9-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c9-knownclusterblast" data-name="r2c9-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c9-subclusterblast" data-name="r2c9-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c9-pfam-details" data-name="r2c9-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c9-tigrfam-details" data-name="r2c9-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c9-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-195"></div> <span class="help-tooltip" id="gt-help-tooltip-help-195">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c9">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c9-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c9-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c9">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c9-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-191"></div> <span class="help-tooltip" id="MIBiG-help-191">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c9-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c9-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c9"></div>
<div id="comparison-MIBiG-r2c9-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001423: 0-41128">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001423/index.html#r1c1" target="_new">BGC0001423</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide C</td>


<td>Myxococcus hansupus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001421: 0-47099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001421/index.html#r1c1" target="_new">BGC0001421</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide B</td>


<td>Myxococcus sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001426: 0-44134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001426/index.html#r1c1" target="_new">BGC0001426</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001427: 0-44152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001427/index.html#r1c1" target="_new">BGC0001427</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001425: 0-44135">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001425/index.html#r1c1" target="_new">BGC0001425</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001424: 0-41589">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001424/index.html#r1c1" target="_new">BGC0001424</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide C</td>


<td>Myxococcus virescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001428: 0-44682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001428/index.html#r1c1" target="_new">BGC0001428</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>NRP, Polyketide</td>

<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>


<td>Fusarium heterosporum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001422: 0-42748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001422/index.html#r1c1" target="_new">BGC0001422</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.47, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus fulvus HW-1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c9-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001423: 0-41128">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001423/index.html#r1c1" target="_new">BGC0001423</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide C</td>
<td>Myxococcus hansupus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001421: 0-47099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001421/index.html#r1c1" target="_new">BGC0001421</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide B</td>
<td>Myxococcus sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001426: 0-44134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001426/index.html#r1c1" target="_new">BGC0001426</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001427: 0-44152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001427/index.html#r1c1" target="_new">BGC0001427</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001425: 0-44135">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001425/index.html#r1c1" target="_new">BGC0001425</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001424: 0-41589">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001424/index.html#r1c1" target="_new">BGC0001424</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide C</td>
<td>Myxococcus virescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001428: 0-44682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001428/index.html#r1c1" target="_new">BGC0001428</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>NRP, Polyketide</td>
<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>
<td>Fusarium heterosporum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001422: 0-42748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001422/index.html#r1c1" target="_new">BGC0001422</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.47, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus fulvus HW-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c9-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-192"></div> <span class="help-tooltip" id="cb-general-help-192">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c9-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c9_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c9_1.svg">
NW_006271970 (3761055-3936010): Cordyceps militaris CM01 unplaced genomic sca... (23% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c9_2.svg">
NC_035794 (1709791-1793960): Pochonia chlamydosporia 170 chromosome 5, whole ... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_3.svg">
NC_035791 (818908-980602): Pochonia chlamydosporia 170 chromosome 2, whole ge... (3% of genes show similarity), NAPAA,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c9_4.svg">
NC_018212 (827083-917413): Zymoseptoria tritici IPO323 chromosome 7, whole ge... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c9_5.svg">
NW_014013658 (1705881-1809848): Trichoderma virens Gv29-8 chromosome Unknown ... (5% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c9_6.svg">
NW_022985439 (868864-959842): Mytilinidion resinicola strain CBS 304.34 unpla... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c9_7.svg">
NW_014574622 (178902-269178): Penicillium digitatum Pd1 unplaced genomic scaf... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c9_8.svg">
NW_024067574 (514885-683420): Penicillium roqueforti strain LCP96 04111 chrom... (3% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c9_9.svg">
NC_049561 (5202011-5292943): Talaromyces rugulosus chromosome I, complete seq... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c9_10.svg">
NC_072350 (2939594-3030676): Talaromyces marneffei chromosome 3, complete seq... (6% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r2c9-download">Download graphic</button>
</div>
<div id="clusterblast-r2c9-svg"></div>

</div>
</div>

<div class="body-details-section r2c9-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-193"></div> <span class="help-tooltip" id="cb-known-help-193">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c9-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-194"></div> <span class="help-tooltip" id="cb-sub-help-194">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c9-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-196"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-196">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c9-pfam-details-svg" data-filename="scaffold_2_r2c9_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c9-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c9-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-197"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-197">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c9-tigrfam-details-svg" data-filename="scaffold_2_r2c9_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c9-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r2c10' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 10 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-512"></div> <span class="help-tooltip" id="region_svg-help-512">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region010.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c10-svg" data-filename="scaffold_2_r2c10.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,495,415 - 4,550,701 nt. (total: 55,287 nt)

<a class="cluster-rules-header" id="r2c10-rules-header" href="#r2c10">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c10-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c10-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-513"></div> <span class="help-tooltip" id="focus-panel-help-513">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c10">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c10-gene-table-details" data-name="r2c10-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c10-nrps_pks" data-name="r2c10-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c10-MIBiG-cluster-compare" data-name="r2c10-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c10-clusterblast" data-name="r2c10-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c10-knownclusterblast" data-name="r2c10-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c10-subclusterblast" data-name="r2c10-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c10-nrps-pks-bubbles" data-name="r2c10-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c10-pfam-details" data-name="r2c10-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c10-tigrfam-details" data-name="r2c10-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c10-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-206"></div> <span class="help-tooltip" id="gt-help-tooltip-help-206">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c10">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c10-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c10-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c10">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c10-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-198"></div> <span class="help-tooltip" id="nrps-domain-help-198">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c10-details-svg" data-filename="scaffold_2_r2c10_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c10-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c10-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-199"></div> <span class="help-tooltip" id="MIBiG-help-199">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c10-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c10-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c10"></div>
<div id="comparison-MIBiG-r2c10-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013: 0-7849">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c10-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013: 0-7849">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c10-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-200"></div> <span class="help-tooltip" id="cb-general-help-200">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c10-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c10_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c10_1.svg">
NC_064678 (3752676-3843730): Colletotrichum lupini chromosome 5, complete seq... (25% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c10_2.svg">
NW_022156719 (1039998-1181379): Pyricularia grisea strain NI907 chromosome V ... (19% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c10_3.svg">
NC_054695 (189610-273160): Aspergillus flavus NRRL3357 chromosome 5, complete... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c10_4.svg">
NC_036439 (153923-237433): Aspergillus oryzae RIB40 DNA, chromosome 5 (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c10_5.svg">
NW_025091287 (155600-237358): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c10_6.svg">
NW_025091270 (2458662-2536359): Aspergillus udagawae IFM 46973 DNA, scaffold ... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c10_7.svg">
NW_024467537 (92677-199333): Aspergillus versicolor CBS 583.65 unplaced genom... (6% of genes show similarity), T1PKS,fungal-RiPP
</option>

<option value = "svg/clusterblast_r2c10_8.svg">
NC_066264 (1638236-1719104): Aspergillus nidulans FGSC A4 chromosome VIII (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c10_9.svg">
NC_066263 (2555307-2633145): Aspergillus nidulans FGSC A4 chromosome VII (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c10_10.svg">
NW_022983475 (2655524-2740477): Aaosphaeria arxii CBS 175.79 unplaced genomic... (10% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r2c10-download">Download graphic</button>
</div>
<div id="clusterblast-r2c10-svg"></div>

</div>
</div>

<div class="body-details-section r2c10-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-201"></div> <span class="help-tooltip" id="cb-known-help-201">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c10-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c10_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c10_1.svg">
lijiquinone
</option>

<option value = "svg/knownclusterblast_r2c10_2.svg">
harziphilone/t22azaphilone/isoharziphilone-1/isoharziphilone-2/compound 4/compound 1
</option>

</select>
<button id="knownclusterblast-r2c10-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c10-svg"></div>

</div>
</div>

<div class="body-details-section r2c10-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-202"></div> <span class="help-tooltip" id="cb-sub-help-202">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c10-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-205"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-205">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c10-domain-bubble-svg" data-filename="scaffold_2_r2c10_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c10-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC10">
Candidate 10 (4495414 - 4550701): single T1PKS
</option>


</select>

<div class="details-svg" id="r2c10-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c10-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c10-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-207"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-207">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c10-pfam-details-svg" data-filename="scaffold_2_r2c10_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c10-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c10-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-208"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-208">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c10-tigrfam-details-svg" data-filename="scaffold_2_r2c10_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c10-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c10-nrps_pks_products" data-name="r2c10-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c10-nrps_pks_monomers" data-name="r2c10-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c10-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-203"></div> <span class="help-tooltip" id="nrps-structures-help-203">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 10, location 4495414 - 4550701:
<div class="collapser collapser-target-CC10 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c10-10" class="smiles-canvas" data-smiles="C(C)C(O)C(OC)C(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS linker matching; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-ohmal) + (mxmal)</dd>

<dt><b>Non-colinear ordering:</b></dt><dd>

<span class="serif">input.path1.gene1218</span>

+ <span class="serif">input.path1.gene1219</span>

+ <span class="serif">input.path1.gene1216</span>

</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-ohmal&amp;nrps2=mxmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-ohmal*&amp;nrps2=mxmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c10-10-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C(C)C(O)C(OC)C(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c10-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-204"></div> <span class="help-tooltip" id="nrps-monomers-help-204">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1216</span></strong>: mxmal
<div class="collapser collapser-target-input-path1-gene1216 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (1364..1570)</strong>: mxmal
<div class="collapser collapser-target-input-path1-gene1216 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-input-path1-gene1216-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1216-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 19.0</dt>

<dd></dd><dt>Malonyl-CoA: 18.2</dt>

<dd></dd><dt>inactive: 15.0</dt>

<dd></dd><dt>Methylmalonyl-CoA: 9.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>Propionyl-CoA: 0.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene1218</span></strong>: mmal
<div class="collapser collapser-target-input-path1-gene1218 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (487..699)</strong>: mmal
<div class="collapser collapser-target-input-path1-gene1218 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-input-path1-gene1218-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1218-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 42.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 38.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 24.6</dt>

<dd></dd><dt>Propionyl-CoA: 19.1</dt>

<dd></dd><dt>Isobutyryl-CoA: 8.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 8.3</dt>

<dd></dd><dt>Malonyl-CoA: 0.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c11' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 11 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-514"></div> <span class="help-tooltip" id="region_svg-help-514">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region011.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c11-svg" data-filename="scaffold_2_r2c11.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,997,278 - 5,041,527 nt. (total: 44,250 nt)

<a class="cluster-rules-header" id="r2c11-rules-header" href="#r2c11">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c11-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c11-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-515"></div> <span class="help-tooltip" id="focus-panel-help-515">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c11">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c11-gene-table-details" data-name="r2c11-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c11-nrps_pks" data-name="r2c11-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c11-MIBiG-cluster-compare" data-name="r2c11-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c11-clusterblast" data-name="r2c11-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c11-knownclusterblast" data-name="r2c11-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c11-subclusterblast" data-name="r2c11-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c11-pfam-details" data-name="r2c11-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c11-tigrfam-details" data-name="r2c11-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c11-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-215"></div> <span class="help-tooltip" id="gt-help-tooltip-help-215">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c11">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c11-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c11-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c11">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c11-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-209"></div> <span class="help-tooltip" id="nrps-domain-help-209">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c11-details-svg" data-filename="scaffold_2_r2c11_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c11-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c11-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-210"></div> <span class="help-tooltip" id="MIBiG-help-210">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c11-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c11-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c11"></div>
<div id="comparison-MIBiG-r2c11-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404: 1968-56662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>penicillin</td>


<td>Penicillium chrysogenum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>xenematide</td>


<td>Xenorhabdus nematophila AN6/1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>sansalvamide</td>


<td>[Nectria] haematococca mpVI 77-13-4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c11-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404: 1968-56662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>penicillin</td>
<td>Penicillium chrysogenum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>xenematide</td>
<td>Xenorhabdus nematophila AN6/1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>sansalvamide</td>
<td>[Nectria] haematococca mpVI 77-13-4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c11-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-211"></div> <span class="help-tooltip" id="cb-general-help-211">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c11-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c11_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c11_1.svg">
NW_006271969 (2236886-2301135): Cordyceps militaris CM01 unplaced genomic sca... (46% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c11_2.svg">
NZ_CP043317 (792756-856900): Streptomyces olivaceus strain SCSIO T05 chromoso... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c11_3.svg">
NZ_LT607413 (4729322-4815787): Micromonospora echinospora strain DSM 43816 ch... (3% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r2c11_4.svg">
NZ_PNXQ01000016 (500910-548524): Paenibacillus terrae strain PIC167 Contig 01... (5% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r2c11_5.svg">
NZ_MAUJ01000002 (439421-527978): Pseudoalteromonas luteoviolacea strain IPB1 ... (3% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r2c11_6.svg">
NC_016935 (4455805-4606301): Paenibacillus mucilaginosus 3016, complete sequence (3% of genes show similarity), HR-T2PKS,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c11_7.svg">
NC_017672 (4614434-4764897): Paenibacillus mucilaginosus K02, complete sequence (3% of genes show similarity), HR-T2PKS,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c11_8.svg">
NZ_BMNF01000004 (720280-789466): Micromonospora parathelypteridis strain CGMC... (3% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r2c11_9.svg">
NW_022986853 (1646865-1714956): Westerdykella ornata strain CBS 379.55 unplac... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c11_10.svg">
NC_070534 (1074625-1138530): Fusarium keratoplasticum isolate Fu6.1 chromosom... (10% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r2c11-download">Download graphic</button>
</div>
<div id="clusterblast-r2c11-svg"></div>

</div>
</div>

<div class="body-details-section r2c11-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-212"></div> <span class="help-tooltip" id="cb-known-help-212">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c11-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-213"></div> <span class="help-tooltip" id="cb-sub-help-213">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c11-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-216"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-216">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c11-pfam-details-svg" data-filename="scaffold_2_r2c11_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c11-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c11-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-217"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-217">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c11-tigrfam-details-svg" data-filename="scaffold_2_r2c11_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c11-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c11-nrps_pks_monomers" data-name="r2c11-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r2c11-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-214"></div> <span class="help-tooltip" id="nrps-monomers-help-214">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1360</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1360 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (225..660)</strong>: X
<div class="collapser collapser-target-input-path1-gene1360 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene1360-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLMFVGIMCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAMLVGAACK</span> (59% 8Ã match)</dd>
<dd>Hty <span class="serif">DLGFTGAVCK</span> (56% 8Ã match)</dd>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (50% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r2c12' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 12 - fungal-RiPP-like,isocyanide-nrp <div class="help-container"> <div class="help-icon" data-id="region_svg-help-516"></div> <span class="help-tooltip" id="region_svg-help-516">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region012.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c12-svg" data-filename="scaffold_2_r2c12.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,433,305 - 5,528,524 nt. (total: 95,220 nt)

<a class="cluster-rules-header" id="r2c12-rules-header" href="#r2c12">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c12-rules">isocyanide-nrp: (DIT1_PvcA and minimum(2, [AMP-binding, PP-binding, Transferase]))<br>fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r2c12-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-517"></div> <span class="help-tooltip" id="focus-panel-help-517">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c12">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c12-gene-table-details" data-name="r2c12-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c12-nrps_pks" data-name="r2c12-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c12-MIBiG-cluster-compare" data-name="r2c12-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c12-clusterblast" data-name="r2c12-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c12-knownclusterblast" data-name="r2c12-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c12-subclusterblast" data-name="r2c12-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r2c12-nrps-pks-bubbles" data-name="r2c12-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c12-pfam-details" data-name="r2c12-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c12-tigrfam-details" data-name="r2c12-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c12-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-226"></div> <span class="help-tooltip" id="gt-help-tooltip-help-226">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c12">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c12-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c12-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c12">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c12-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-218"></div> <span class="help-tooltip" id="nrps-domain-help-218">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c12-details-svg" data-filename="scaffold_2_r2c12_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c12-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c12-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-219"></div> <span class="help-tooltip" id="MIBiG-help-219">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c12-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c12-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c12"></div>
<div id="comparison-MIBiG-r2c12-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">isocyanide-nrp</span></th>

<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000933: 0-5189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000933/index.html#r1c1" target="_new">BGC0000933</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other</td>

<td>xiamenmycin A</td>


<td>Streptomyces xiamenensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>KK-1</td>


<td>Curvularia clavata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001218: 1349-26398">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001218/index.html#r1c1" target="_new">BGC0001218</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.41 (id:0.66, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>fumosorinone</td>


<td>Cordyceps fumosorosea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>citreohybridonol</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>yanuthone D</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002594: 653-30873">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002594/index.html#r1c1" target="_new">BGC0002594</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>orcinol, orsellinic acid</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c12-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000933: 0-5189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000933/index.html#r1c1" target="_new">BGC0000933</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>xiamenmycin A</td>
<td>Streptomyces xiamenensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001218: 1349-26398">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001218/index.html#r1c1" target="_new">BGC0001218</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.66, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP, Polyketide</td>
<td>fumosorinone</td>
<td>Cordyceps fumosorosea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>KK-1</td>
<td>Curvularia clavata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Terpene</td>
<td>citreohybridonol</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002594: 653-30873">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002594/index.html#r1c1" target="_new">BGC0002594</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>orcinol, orsellinic acid</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001508: 25-29346">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001508/index.html#r1c1" target="_new">BGC0001508</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>andrastin A</td>
<td>Penicillium roqueforti</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c12-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-220"></div> <span class="help-tooltip" id="cb-general-help-220">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c12-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c12_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c12_1.svg">
NW_006271975 (307084-386893): Cordyceps militaris CM01 unplaced genomic scaff... (37% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r2c12_2.svg">
NW_019154045 (358401-688746): Pochonia chlamydosporia 170 chromosome Unknown ... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c12_3.svg">
NW_006911257 (1345885-1409023): Baudoinia panamericana UAMH 10762 unplaced ge... (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c12_4.svg">
NW_014574695 (32560-164814): Metarhizium brunneum ARSEF 3297 chromosome Unkno... (9% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c12_5.svg">
NW_011942182 (203866-337752): Metarhizium robertsii ARSEF 23 MAA Scf 4, whole... (13% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c12_6.svg">
NC_072847 (367100-458042): Fusarium oxysporum Fo47 chromosome VIII, complete ... (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c12_7.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (8% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c12_8.svg">
NC_036441 (684002-775165): Aspergillus oryzae RIB40 DNA, chromosome 7 (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c12_9.svg">
NW_013581183 (53074-116224): Aureobasidium namibiae CBS 147.97 unplaced genom... (16% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c12_10.svg">
NW_024468912 (725220-816586): Cryphonectria parasitica EP155 unplaced genomic... (10% of genes show similarity), NRPS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r2c12-download">Download graphic</button>
</div>
<div id="clusterblast-r2c12-svg"></div>

</div>
</div>

<div class="body-details-section r2c12-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-221"></div> <span class="help-tooltip" id="cb-known-help-221">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c12-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-222"></div> <span class="help-tooltip" id="cb-sub-help-222">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r2c12-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-225"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-225">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c12-domain-bubble-svg" data-filename="scaffold_2_r2c12_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c12-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC12">
Candidate 12 (5433304 - 5528524): neighbouring isocyanide-nrp-fungal-RiPP-like
</option>



<option value="CC13">
Candidate 13 (5433304 - 5492659): single isocyanide-nrp
</option>


</select>

<div class="details-svg" id="r2c12-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c12-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c12-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-227"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-227">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c12-pfam-details-svg" data-filename="scaffold_2_r2c12_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c12-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c12-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-228"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-228">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c12-tigrfam-details-svg" data-filename="scaffold_2_r2c12_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c12-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c12-nrps_pks_products" data-name="r2c12-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c12-nrps_pks_monomers" data-name="r2c12-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r2c12-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-223"></div> <span class="help-tooltip" id="nrps-structures-help-223">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 12, location 5433304 - 5528524:
<div class="collapser collapser-target-CC12 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c12-12" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c12-12-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 13, location 5433304 - 5492659:
<div class="collapser collapser-target-CC13 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c12-13" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c12-13-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c12-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-224"></div> <span class="help-tooltip" id="nrps-monomers-help-224">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1486</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1486 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (436..789)</strong>: X
<div class="collapser collapser-target-input-path1-gene1486 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene1486-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GANSCSLFIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>





<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 1 - T1PKS,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-518"></div> <span class="help-tooltip" id="region_svg-help-518">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="scaffold_3_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 58,616 nt. (total: 58,616 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r3c1-rules">fungal-RiPP-like: DUF3328<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-519"></div> <span class="help-tooltip" id="focus-panel-help-519">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-nrps_pks" data-name="r3c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r3c1-nrps-pks-bubbles" data-name="r3c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-237"></div> <span class="help-tooltip" id="gt-help-tooltip-help-237">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-229"></div> <span class="help-tooltip" id="nrps-domain-help-229">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-details-svg" data-filename="scaffold_3_r3c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-230"></div> <span class="help-tooltip" id="MIBiG-help-230">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>






<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.74</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.64</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.63</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.50, order:0.52, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.50, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.61</td>
<td>Polyketide</td>

<td>solanapyrone D</td>


<td>Alternaria solani</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.50, order:0.58, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.51, order:0.58, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.61</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.58</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.56</td>
<td>Polyketide</td>

<td>abscisic acid</td>


<td>Leptosphaeria maculans JN3</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.50, order:0.63, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.51, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>abscisic acid</td>
<td>Leptosphaeria maculans JN3</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.51, order:0.58, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-231"></div> <span class="help-tooltip" id="cb-general-help-231">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c1_1.svg">
NW_017253364 (1410639-1535276): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (11% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r3c1_2.svg">
NW_026250020 (2828871-2896149): Hypoxylon trugodes strain CBS 135444 unplaced... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_3.svg">
NW_022986863 (466665-534682): Westerdykella ornata strain CBS 379.55 unplaced... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_4.svg">
NW_024468938 (1558469-1626421): Cucurbitaria berberidis CBS 394.84 unplaced g... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_5.svg">
NW_026054546 (1062713-1130726): Daldinia decipiens strain CBS 113046 unplaced... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_6.svg">
NW_006917091 (1555707-1623710): Pestalotiopsis fici W106-1 unplaced genomic s... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_7.svg">
NW_025408670 (2446028-2514009): Hirsutella rhossiliensis strain HR02 chromoso... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_8.svg">
NT_165982 (3358274-3426284): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_9.svg">
NW_021167080 (3127431-3195689): Sodiomyces alkalinus F11 unplaced genomic sca... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_10.svg">
NW_025953250 (276988-345443): Annulohypoxylon maeteangense strain CBS 123835 ... (14% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r3c1-download">Download graphic</button>
</div>
<div id="clusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-232"></div> <span class="help-tooltip" id="cb-known-help-232">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c1_1.svg">
solanapyrone D
</option>

<option value = "svg/knownclusterblast_r3c1_2.svg">
solanapyrone A
</option>

</select>
<button id="knownclusterblast-r3c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-233"></div> <span class="help-tooltip" id="cb-sub-help-233">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r3c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-236"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-236">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-domain-bubble-svg" data-filename="scaffold_3_r3c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 58616): neighbouring fungal-RiPP-like-T1PKS
</option>



<option value="CC3">
Candidate 3 (10559 - 58616): single T1PKS
</option>


</select>

<div class="details-svg" id="r3c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-238"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-238">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="scaffold_3_r3c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c1-nrps_pks_products" data-name="r3c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c1-nrps_pks_monomers" data-name="r3c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r3c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-234"></div> <span class="help-tooltip" id="nrps-structures-help-234">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 58616:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 10559 - 58616:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c1-3" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-235"></div> <span class="help-tooltip" id="nrps-monomers-help-235">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene8</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene8 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (640..958)</strong>: pk
<div class="collapser collapser-target-input-path1-gene8 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene8-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene8-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 73.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 70.7</dt>

<dd></dd><dt>Malonyl-CoA: 53.9</dt>

<dd></dd><dt>Propionyl-CoA: 42.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 31.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 22.2</dt>

<dd></dd><dt>CHC-CoA: 18.1</dt>

<dd></dd><dt>fatty_acid: 17.1</dt>

<dd></dd><dt>inactive: 16.3</dt>

<dd></dd><dt>Isobutyryl-CoA: 15.8</dt>

<dd></dd><dt>trans-1,2-CPDA: 8.3</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>

<div class="page" id='r3c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 2 - NRPS-like,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-520"></div> <span class="help-tooltip" id="region_svg-help-520">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c2-svg" data-filename="scaffold_3_r3c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 580,696 - 674,169 nt. (total: 93,474 nt)

<a class="cluster-rules-header" id="r3c2-rules-header" href="#r3c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-521"></div> <span class="help-tooltip" id="focus-panel-help-521">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c2-gene-table-details" data-name="r3c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c2-nrps_pks" data-name="r3c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c2-MIBiG-cluster-compare" data-name="r3c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c2-clusterblast" data-name="r3c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c2-knownclusterblast" data-name="r3c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c2-subclusterblast" data-name="r3c2-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r3c2-pfam-details" data-name="r3c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-245"></div> <span class="help-tooltip" id="gt-help-tooltip-help-245">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-239"></div> <span class="help-tooltip" id="nrps-domain-help-239">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-details-svg" data-filename="scaffold_3_r3c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-240"></div> <span class="help-tooltip" id="MIBiG-help-240">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c2"></div>
<div id="comparison-MIBiG-r3c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000841: 0-11023">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000841/index.html#r1c1" target="_new">BGC0000841</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.46, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>






<td class="cc-heat-cell" title="0.37 (id:0.46, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.21</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate, clavulanic acid</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>fragin</td>


<td>Burkholderia cenocepacia H111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.16</td>
<td>NRP</td>

<td>choline</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002555: 2045-61815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002555/index.html#r1c1" target="_new">BGC0002555</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.16</td>
<td>NRP, Polyketide</td>

<td>emerimicin V, emerimicin VI, emerimicin VII, emerimicin VIII, emerimicin IX, emerimicin X</td>


<td>Acremonium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.15</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.14</td>
<td>Polyketide, NRP</td>

<td>xenoacremone A</td>


<td>Xenoacremonium sp. BF-2018a</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001085: 0-10646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001085/index.html#r1c1" target="_new">BGC0001085</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.13</td>
<td>NRP, Terpene</td>

<td>teleocidin B1</td>


<td>Streptomyces blastmyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.13</td>
<td>NRP</td>

<td>guanipiperazine A, guanipiperazine B</td>


<td>Streptomyces chrestomyceticus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>fragin</td>
<td>Burkholderia cenocepacia H111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>NRP</td>
<td>choline</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002555: 2045-61815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002555/index.html#r1c1" target="_new">BGC0002555</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>NRP, Polyketide</td>
<td>emerimicin V, emerimicin VI, emerimicin VII, emerimicin VIII, emerimicin IX, emerimicin X</td>
<td>Acremonium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001085: 0-10646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001085/index.html#r1c1" target="_new">BGC0001085</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>NRP, Terpene</td>
<td>teleocidin B1</td>
<td>Streptomyces blastmyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>NRP</td>
<td>guanipiperazine A, guanipiperazine B</td>
<td>Streptomyces chrestomyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001627: 0-10806">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001627/index.html#r1c1" target="_new">BGC0001627</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>NRP</td>
<td>isonitrile lipopeptides</td>
<td>Mycobacterium tuberculosis H37Rv</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002706: 0-13879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002706/index.html#r1c1" target="_new">BGC0002706</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>NRP</td>
<td>saframycin Mx1</td>
<td>Myxococcus xanthus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-241"></div> <span class="help-tooltip" id="cb-general-help-241">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c2_1.svg">
NW_025764704 (762962-853638): Emericellopsis atlantica unplaced genomic scaff... (22% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_2.svg">
NW_024067566 (523062-613732): Penicillium roqueforti strain LCP96 04111 chrom... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_3.svg">
NW_022985018 (595640-686314): Dothidotthia symphoricarpi CBS 119687 unplaced ... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_4.svg">
NW_024468936 (4943083-5033757): Cucurbitaria berberidis CBS 394.84 unplaced g... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_5.svg">
NC_064672 (2020083-2159961): Colletotrichum lupini chromosome 1, complete seq... (9% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_6.svg">
NW_024467220 (23402-114183): Dacryopinax primogenitus strain DJM-731 SS1 chro... (12% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_7.svg">
NZ_CP023522 (4100844-4174348): Burkholderia gladioli pv. gladioli strain FDAA... (4% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c2_8.svg">
NZ_CP065595 (383197-456701): Burkholderia gladioli strain FDAARGOS 951 chromo... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c2_9.svg">
NZ_CADEVR010000004 (18265-65915): Burkholderia gladioli strain BCC1619, whole... (6% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c2_10.svg">
NZ_CADEWF010000007 (18253-85433): Burkholderia gladioli strain BCC1663, whole... (6% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r3c2-download">Download graphic</button>
</div>
<div id="clusterblast-r3c2-svg"></div>

</div>
</div>

<div class="body-details-section r3c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-242"></div> <span class="help-tooltip" id="cb-known-help-242">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-243"></div> <span class="help-tooltip" id="cb-sub-help-243">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r3c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-246"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-246">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-pfam-details-svg" data-filename="scaffold_3_r3c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c2-nrps_pks_monomers" data-name="r3c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-244"></div> <span class="help-tooltip" id="nrps-monomers-help-244">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 3 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-522"></div> <span class="help-tooltip" id="region_svg-help-522">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c3-svg" data-filename="scaffold_3_r3c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,904,091 - 2,951,445 nt. (total: 47,355 nt)

<a class="cluster-rules-header" id="r3c3-rules-header" href="#r3c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r3c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-523"></div> <span class="help-tooltip" id="focus-panel-help-523">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c3-gene-table-details" data-name="r3c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c3-nrps_pks" data-name="r3c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c3-MIBiG-cluster-compare" data-name="r3c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c3-clusterblast" data-name="r3c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c3-knownclusterblast" data-name="r3c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c3-subclusterblast" data-name="r3c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c3-nrps-pks-bubbles" data-name="r3c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c3-pfam-details" data-name="r3c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-255"></div> <span class="help-tooltip" id="gt-help-tooltip-help-255">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-247"></div> <span class="help-tooltip" id="nrps-domain-help-247">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-details-svg" data-filename="scaffold_3_r3c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-248"></div> <span class="help-tooltip" id="MIBiG-help-248">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c3"></div>
<div id="comparison-MIBiG-r3c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>phomoidride</td>


<td>fungal sp. ATCC 74256</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>epipyriculol</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene, Polyketide</td>

<td>calidoustene A, calidoustene B, calidoustene C</td>


<td>Aspergillus calidoustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>phomoidride</td>
<td>fungal sp. ATCC 74256</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>epipyriculol</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-249"></div> <span class="help-tooltip" id="cb-general-help-249">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c3_1.svg">
NW_006271970 (1369951-1438884): Cordyceps militaris CM01 unplaced genomic sca... (38% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_2.svg">
NW_006711153 (459708-526750): Trichoderma reesei QM6a unplaced genomic scaffo... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_3.svg">
NC_070539 (1227558-1294574): Fusarium keratoplasticum isolate Fu6.1 chromosom... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_4.svg">
NC_070554 (1471044-1538049): Fusarium falciforme chromosome 11, complete sequ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_5.svg">
NC_035790 (8523905-8591015): Pochonia chlamydosporia 170 chromosome 1, whole ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_6.svg">
NW_025953257 (328883-396554): Annulohypoxylon maeteangense strain CBS 123835 ... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_7.svg">
NW_026250018 (661526-728676): Hypoxylon trugodes strain CBS 135444 unplaced g... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_8.svg">
NW_025953055 (276846-344269): Daldinia caldariorum strain CBS 122874 unplaced... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_9.svg">
NW_025953787 (2032241-2099463): Annulohypoxylon truncatum strain CBS 140777 u... (17% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_10.svg">
NC_072351 (123460-190570): Talaromyces marneffei chromosome 4, complete sequence (12% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r3c3-download">Download graphic</button>
</div>
<div id="clusterblast-r3c3-svg"></div>

</div>
</div>

<div class="body-details-section r3c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-250"></div> <span class="help-tooltip" id="cb-known-help-250">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-251"></div> <span class="help-tooltip" id="cb-sub-help-251">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-254"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-254">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-domain-bubble-svg" data-filename="scaffold_3_r3c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (2904090 - 2951445): single T1PKS
</option>


</select>

<div class="details-svg" id="r3c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-256"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-256">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-pfam-details-svg" data-filename="scaffold_3_r3c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c3-nrps_pks_products" data-name="r3c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c3-nrps_pks_monomers" data-name="r3c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-252"></div> <span class="help-tooltip" id="nrps-structures-help-252">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 2904090 - 2951445:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c3-7" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c3-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-253"></div> <span class="help-tooltip" id="nrps-monomers-help-253">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene810</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene810 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (537..847)</strong>: pk
<div class="collapser collapser-target-input-path1-gene810 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene810-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Isobutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene810-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 65.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 61.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 47.1</dt>

<dd></dd><dt>Propionyl-CoA: 46.7</dt>

<dd></dd><dt>Malonyl-CoA: 37.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 21.1</dt>

<dd></dd><dt>Benzoyl-CoA: 19.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 18.7</dt>

<dd></dd><dt>inactive: 9.6</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 4 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-524"></div> <span class="help-tooltip" id="region_svg-help-524">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c4-svg" data-filename="scaffold_3_r3c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,562,170 - 3,606,393 nt. (total: 44,224 nt)

<a class="cluster-rules-header" id="r3c4-rules-header" href="#r3c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c4-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-525"></div> <span class="help-tooltip" id="focus-panel-help-525">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c4-gene-table-details" data-name="r3c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c4-nrps_pks" data-name="r3c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c4-MIBiG-cluster-compare" data-name="r3c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c4-clusterblast" data-name="r3c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c4-knownclusterblast" data-name="r3c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c4-subclusterblast" data-name="r3c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c4-nrps-pks-bubbles" data-name="r3c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c4-pfam-details" data-name="r3c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c4-tigrfam-details" data-name="r3c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-265"></div> <span class="help-tooltip" id="gt-help-tooltip-help-265">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-257"></div> <span class="help-tooltip" id="nrps-domain-help-257">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-details-svg" data-filename="scaffold_3_r3c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-258"></div> <span class="help-tooltip" id="MIBiG-help-258">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c4"></div>
<div id="comparison-MIBiG-r3c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger CBS 101883</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP, Polyketide</td>
<td>pyrophen, campyrone B</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP, Polyketide</td>
<td>pyrophen, campyrone B</td>
<td>Aspergillus niger CBS 101883</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-259"></div> <span class="help-tooltip" id="cb-general-help-259">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c4_1.svg">
NW_025408667 (4420833-4486911): Hirsutella rhossiliensis strain HR02 chromoso... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c4_2.svg">
NC_054390 (4169657-4235204): Drechmeria coniospora strain ARSEF 6962 chromoso... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c4_3.svg">
NW_025408678 (2092784-2157825): Hirsutella rhossiliensis strain HR02 chromoso... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c4_4.svg">
NZ_JTHL01000001 (3790742-3863553): Streptomyces sp. 150FB scaffold00001, whol... (3% of genes show similarity), NRPS,PKS-like,T1PKS,amglyccycl,phenazine
</option>

<option value = "svg/clusterblast_r3c4_5.svg">
NZ_CP061072 (3693368-3835908): Streptomyces sp. CB00271 chromosome, complete ... (3% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine
</option>

<option value = "svg/clusterblast_r3c4_6.svg">
NZ_CP086102 (3933946-4073561): Streptomyces anulatus strain YINM00001 chromos... (3% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine
</option>

<option value = "svg/clusterblast_r3c4_7.svg">
NZ_LIVW01000001 (1805343-1943895): Streptomyces sp. CB02115 scaffold1, whole ... (3% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine
</option>

<option value = "svg/clusterblast_r3c4_8.svg">
NZ_KZ195572 (3997844-4136010): Streptomyces sp. CS057 scaffold00001, whole ge... (3% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine
</option>

<option value = "svg/clusterblast_r3c4_9.svg">
NZ_CP080029 (4359383-4493253): Streptomyces anulatus strain VUW1 chromosome, ... (3% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone
</option>

<option value = "svg/clusterblast_r3c4_10.svg">
NZ_CM003601 (4499262-4637594): Streptomyces anulatus strain ATCC 11523 chromo... (3% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine
</option>

</select>
<button id= "clusterblast-r3c4-download">Download graphic</button>
</div>
<div id="clusterblast-r3c4-svg"></div>

</div>
</div>

<div class="body-details-section r3c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-260"></div> <span class="help-tooltip" id="cb-known-help-260">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-261"></div> <span class="help-tooltip" id="cb-sub-help-261">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-264"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-264">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-domain-bubble-svg" data-filename="scaffold_3_r3c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC8">
Candidate 8 (3562169 - 3606393): single NRPS-like
</option>


</select>

<div class="details-svg" id="r3c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-266"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-266">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-pfam-details-svg" data-filename="scaffold_3_r3c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-267"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-267">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-tigrfam-details-svg" data-filename="scaffold_3_r3c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c4-nrps_pks_products" data-name="r3c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c4-nrps_pks_monomers" data-name="r3c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-262"></div> <span class="help-tooltip" id="nrps-structures-help-262">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 8, location 3562169 - 3606393:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c4-8" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c4-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-263"></div> <span class="help-tooltip" id="nrps-monomers-help-263">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene987</span></strong>: Leu
<div class="collapser collapser-target-input-path1-gene987 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (31..418)</strong>: Leu
<div class="collapser collapser-target-input-path1-gene987 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-input-path1-gene987-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALQSGAVAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DALHSGAVAK</span> (65% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 5 - isocyanide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-526"></div> <span class="help-tooltip" id="region_svg-help-526">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c5-svg" data-filename="scaffold_3_r3c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,132,053 - 4,174,181 nt. (total: 42,129 nt)

<a class="cluster-rules-header" id="r3c5-rules-header" href="#r3c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c5-rules">isocyanide: cds(TauD and DIT1_PvcA)</div>

<div class="region-svg-container">
<div id='r3c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-527"></div> <span class="help-tooltip" id="focus-panel-help-527">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c5-gene-table-details" data-name="r3c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c5-MIBiG-cluster-compare" data-name="r3c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c5-clusterblast" data-name="r3c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c5-knownclusterblast" data-name="r3c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c5-subclusterblast" data-name="r3c5-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r3c5-pfam-details" data-name="r3c5-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-272"></div> <span class="help-tooltip" id="gt-help-tooltip-help-272">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-268"></div> <span class="help-tooltip" id="MIBiG-help-268">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c5"></div>
<div id="comparison-MIBiG-r3c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">isocyanide</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000923: 943-6060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000923/index.html#r1c1" target="_new">BGC0000923</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Other</td>

<td>paerucumarin</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001594: 0-38842">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001594/index.html#r1c1" target="_new">BGC0001594</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Alkaloid</td>

<td>fischerindole L</td>


<td>Fischerella muscicola UTEX 1829</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Alkaloid</td>

<td>fischerindole</td>


<td>Fischerella sp. SAG 46.79</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.44, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other</td>

<td>schizokinen</td>


<td>Leptolyngbya sp. PCC 7376</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000968: 0-25930">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000968/index.html#r1c1" target="_new">BGC0000968</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP, Polyketide</td>

<td>chaetoglobosin A, chaetoglobosin C</td>


<td>Penicillium expansum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.45, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>


<td>Hapalosiphon welwitschii UTEX B 1830</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Alkaloid</td>

<td>ambiguine H isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001966: 0-84810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001966/index.html#r1c1" target="_new">BGC0001966</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.46, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>BII-rafflesfungin</td>


<td>Phoma sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000923: 943-6060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000923/index.html#r1c1" target="_new">BGC0000923</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>paerucumarin</td>
<td>Pseudomonas aeruginosa PAO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001594: 0-38842">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001594/index.html#r1c1" target="_new">BGC0001594</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Alkaloid</td>
<td>fischerindole L</td>
<td>Fischerella muscicola UTEX 1829</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Alkaloid</td>
<td>fischerindole</td>
<td>Fischerella sp. SAG 46.79</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.44, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other</td>
<td>schizokinen</td>
<td>Leptolyngbya sp. PCC 7376</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000968: 0-25930">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000968/index.html#r1c1" target="_new">BGC0000968</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP, Polyketide</td>
<td>chaetoglobosin A, chaetoglobosin C</td>
<td>Penicillium expansum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.45, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>
<td>Hapalosiphon welwitschii UTEX B 1830</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>
<td>Fischerella ambigua UTEX 1903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Alkaloid</td>
<td>ambiguine H isonitrile</td>
<td>Fischerella ambigua UTEX 1903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001966: 0-84810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001966/index.html#r1c1" target="_new">BGC0001966</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.46, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>NRP, Polyketide</td>
<td>BII-rafflesfungin</td>
<td>Phoma sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-269"></div> <span class="help-tooltip" id="cb-general-help-269">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c5_1.svg">
NC_063070 (2362618-2424749): Purpureocillium takamizusanense chromosome 3, co... (33% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c5_2.svg">
NC_054389 (8457870-8566158): Drechmeria coniospora strain ARSEF 6962 chromoso... (21% of genes show similarity), T1PKS,isocyanide
</option>

<option value = "svg/clusterblast_r3c5_3.svg">
NW_014574715 (467040-529181): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (33% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c5_4.svg">
NW_011942149 (5418619-5480759): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (33% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c5_5.svg">
NC_035790 (6741031-6803169): Pochonia chlamydosporia 170 chromosome 1, whole ... (27% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c5_6.svg">
NW_006711178 (114520-207586): Trichoderma reesei QM6a unplaced genomic scaffo... (16% of genes show similarity), isocyanide,terpene
</option>

<option value = "svg/clusterblast_r3c5_7.svg">
NW_006271969 (5556555-5618595): Cordyceps militaris CM01 unplaced genomic sca... (40% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c5_8.svg">
NW_025408686 (565240-627429): Hirsutella rhossiliensis strain HR02 chromosome... (29% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c5_9.svg">
NC_057319 (4219732-4281896): Ustilaginoidea virens chromosome 4, complete seq... (14% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c5_10.svg">
NW_024468916 (67569-143215): Cryphonectria parasitica EP155 unplaced genomic ... (16% of genes show similarity), isocyanide,terpene
</option>

</select>
<button id= "clusterblast-r3c5-download">Download graphic</button>
</div>
<div id="clusterblast-r3c5-svg"></div>

</div>
</div>

<div class="body-details-section r3c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-270"></div> <span class="help-tooltip" id="cb-known-help-270">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-271"></div> <span class="help-tooltip" id="cb-sub-help-271">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r3c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-273"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-273">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-pfam-details-svg" data-filename="scaffold_3_r3c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r3c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 6 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-528"></div> <span class="help-tooltip" id="region_svg-help-528">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c6-svg" data-filename="scaffold_3_r3c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,473,101 - 4,520,263 nt. (total: 47,163 nt)

<a class="cluster-rules-header" id="r3c6-rules-header" href="#r3c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c6-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-529"></div> <span class="help-tooltip" id="focus-panel-help-529">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c6-gene-table-details" data-name="r3c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c6-nrps_pks" data-name="r3c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c6-MIBiG-cluster-compare" data-name="r3c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c6-clusterblast" data-name="r3c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c6-knownclusterblast" data-name="r3c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c6-subclusterblast" data-name="r3c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c6-nrps-pks-bubbles" data-name="r3c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c6-pfam-details" data-name="r3c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c6-tigrfam-details" data-name="r3c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-282"></div> <span class="help-tooltip" id="gt-help-tooltip-help-282">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-274"></div> <span class="help-tooltip" id="nrps-domain-help-274">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c6-details-svg" data-filename="scaffold_3_r3c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-275"></div> <span class="help-tooltip" id="MIBiG-help-275">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c6"></div>
<div id="comparison-MIBiG-r3c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>





<td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>


<td>0.43</td>
<td>NRP</td>

<td>choline</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308: 0-7851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>aureusimine A, aureusimine B, aureusimine C</td>


<td>Staphylococcus aureus subsp. aureus str. JKD6008</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>


<td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
<td>0.74</td>
<td>NRP</td>
<td>choline</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308: 0-7851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>aureusimine A, aureusimine B, aureusimine C</td>
<td>Staphylococcus aureus subsp. aureus str. JKD6008</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-276"></div> <span class="help-tooltip" id="cb-general-help-276">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c6_1.svg">
NC_070532 (4966088-5118746): Fusarium keratoplasticum isolate Fu6.1 chromosom... (23% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c6_2.svg">
NC_070547 (4757164-4894085): Fusarium falciforme chromosome 4, complete sequence (24% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c6_3.svg">
NW_006271970 (2419165-2483026): Cordyceps militaris CM01 unplaced genomic sca... (32% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c6_4.svg">
NC_072842 (587831-651686): Fusarium oxysporum Fo47 chromosome III, complete s... (32% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c6_5.svg">
NC_036624 (4507840-4571695): Fusarium fujikuroi IMI 58289 draft genome, chrom... (32% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c6_6.svg">
NC_058388 (524455-588310): Fusarium musae strain F31 chromosome 2, whole geno... (33% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c6_7.svg">
NC_026477 (249523-313372): Fusarium graminearum PH-1 chromosome 4, whole geno... (40% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c6_8.svg">
NW_014013657 (238627-342899): Trichoderma virens Gv29-8 chromosome Unknown TR... (22% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c6_9.svg">
NW_011942160 (555242-619088): Metarhizium robertsii ARSEF 23 MAA Scf 2, whole... (46% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c6_10.svg">
NC_035791 (1143313-1234197): Pochonia chlamydosporia 170 chromosome 2, whole ... (22% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c6-download">Download graphic</button>
</div>
<div id="clusterblast-r3c6-svg"></div>

</div>
</div>

<div class="body-details-section r3c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-277"></div> <span class="help-tooltip" id="cb-known-help-277">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c6-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c6_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c6_1.svg">
choline
</option>

</select>
<button id="knownclusterblast-r3c6-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c6-svg"></div>

</div>
</div>

<div class="body-details-section r3c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-278"></div> <span class="help-tooltip" id="cb-sub-help-278">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-281"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-281">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c6-domain-bubble-svg" data-filename="scaffold_3_r3c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC10">
Candidate 10 (4473100 - 4520263): single NRPS-like
</option>


</select>

<div class="details-svg" id="r3c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-283"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-283">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c6-pfam-details-svg" data-filename="scaffold_3_r3c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-284"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-284">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c6-tigrfam-details-svg" data-filename="scaffold_3_r3c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c6-nrps_pks_products" data-name="r3c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c6-nrps_pks_monomers" data-name="r3c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-279"></div> <span class="help-tooltip" id="nrps-structures-help-279">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 10, location 4473100 - 4520263:
<div class="collapser collapser-target-CC10 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c6-10" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c6-10-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-280"></div> <span class="help-tooltip" id="nrps-monomers-help-280">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1232</span></strong>: Phe
<div class="collapser collapser-target-input-path1-gene1232 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (13..418)</strong>: Phe
<div class="collapser collapser-target-input-path1-gene1232 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-input-path1-gene1232-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>phe (Phe)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--WLWNVEAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 7 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-530"></div> <span class="help-tooltip" id="region_svg-help-530">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c7-svg" data-filename="scaffold_3_r3c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,900,141 - 4,971,066 nt. (total: 70,926 nt)

<a class="cluster-rules-header" id="r3c7-rules-header" href="#r3c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c7-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-531"></div> <span class="help-tooltip" id="focus-panel-help-531">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c7-gene-table-details" data-name="r3c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c7-MIBiG-cluster-compare" data-name="r3c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c7-clusterblast" data-name="r3c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c7-knownclusterblast" data-name="r3c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c7-subclusterblast" data-name="r3c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c7-pfam-details" data-name="r3c7-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c7-tigrfam-details" data-name="r3c7-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-289"></div> <span class="help-tooltip" id="gt-help-tooltip-help-289">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-285"></div> <span class="help-tooltip" id="MIBiG-help-285">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c7"></div>
<div id="comparison-MIBiG-r3c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>aspterric acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.46, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>HEx-tc3 terpene</td>


<td>Botrytis cinerea T4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>heptelidic acid</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>aspterric acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.46, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Terpene</td>
<td>HEx-tc3 terpene</td>
<td>Botrytis cinerea T4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Terpene</td>
<td>heptelidic acid</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-286"></div> <span class="help-tooltip" id="cb-general-help-286">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c7_1.svg">
NW_006271970 (2031211-2136459): Cordyceps militaris CM01 unplaced genomic sca... (52% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_2.svg">
NW_011371361 (1937664-2035052): Paracoccidioides brasiliensis Pb18 unplaced g... (20% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_3.svg">
NW_015440948 (109188-206314): Paracoccidioides lutzii Pb01 unplaced genomic s... (21% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_4.svg">
NC_030959 (4385741-4495530): Colletotrichum higginsianum IMI 349063 chromosom... (14% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_5.svg">
NW_017263587 (265177-438368): Mollisia scopiformis unplaced genomic scaffold ... (8% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_6.svg">
NT_165982 (1008305-1103578): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (24% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_7.svg">
NW_026250024 (1787317-1950532): Hypoxylon trugodes strain CBS 135444 unplaced... (6% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_8.svg">
NW_020290631 (343013-434294): Aspergillus saccharolyticus JOP 1030-1 unplaced... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_9.svg">
NW_020291606 (579604-818783): Aspergillus piperis CBS 112811 unplaced genomic... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c7_10.svg">
NC_035796 (212423-303459): Pochonia chlamydosporia 170 chromosome 7, whole ge... (13% of genes show similarity), fungal-RiPP-like,terpene
</option>

</select>
<button id= "clusterblast-r3c7-download">Download graphic</button>
</div>
<div id="clusterblast-r3c7-svg"></div>

</div>
</div>

<div class="body-details-section r3c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-287"></div> <span class="help-tooltip" id="cb-known-help-287">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-288"></div> <span class="help-tooltip" id="cb-sub-help-288">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-290"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-290">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c7-pfam-details-svg" data-filename="scaffold_3_r3c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c7-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-291"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-291">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c7-tigrfam-details-svg" data-filename="scaffold_3_r3c7_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c7-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r4c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-532"></div> <span class="help-tooltip" id="region_svg-help-532">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c1-svg" data-filename="scaffold_4_r4c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 146,693 - 241,447 nt. (total: 94,755 nt)

<a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-533"></div> <span class="help-tooltip" id="focus-panel-help-533">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c1-gene-table-details" data-name="r4c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c1-nrps-pks-bubbles" data-name="r4c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c1-tigrfam-details" data-name="r4c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-300"></div> <span class="help-tooltip" id="gt-help-tooltip-help-300">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-292"></div> <span class="help-tooltip" id="nrps-domain-help-292">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="scaffold_4_r4c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-293"></div> <span class="help-tooltip" id="MIBiG-help-293">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c1"></div>
<div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.48, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.48, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.67</td>
<td>Terpene, Polyketide</td>

<td>calidoustene A, calidoustene B, calidoustene C</td>


<td>Aspergillus calidoustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.48, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.48, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.67</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.52, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.52, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.66</td>
<td>Polyketide</td>

<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:0.29, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.66</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.52, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.52, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.63</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.63</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.63</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.62</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.49, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.49, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.62</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.61</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.48, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002729: 0-27318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002729/index.html#r1c1" target="_new">BGC0002729</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>sordarial</td>
<td>Neurospora crassa OR74A</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.48, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>flavoglaucin</td>
<td>Aspergillus ruber CBS 135680</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001067: 0-35154">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001067/index.html#r1c1" target="_new">BGC0001067</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide, Terpene</td>
<td>fumagillin, Î²-trans-bergamotene, fumagillol</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.49, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-294"></div> <span class="help-tooltip" id="cb-general-help-294">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c1_1.svg">
NW_006271974 (2321315-2405136): Cordyceps militaris CM01 unplaced genomic sca... (75% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c1_2.svg">
NW_026249875 (10804-78109): Alternaria incomplexa strain BMP 0042 chromosome ... (38% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_3.svg">
NC_049565 (2694595-2813931): Talaromyces rugulosus chromosome V, complete seq... (24% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_4.svg">
NW_020194484 (7340-83776): Amorphotheca resinae ATCC 22711 unplaced genomic s... (35% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c1_5.svg">
NW_026055699 (512819-580124): Alternaria ventricosa strain BMP 2768 chromosom... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_6.svg">
NW_026249941 (3336813-3404118): Alternaria arbusti strain BMP 1465 chromosome... (33% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_7.svg">
NW_022474205 (7019281-7087036): Venustampulla echinocandica strain BP 5553 ch... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_8.svg">
NW_025408687 (532325-600393): Hirsutella rhossiliensis strain HR02 chromosome... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_9.svg">
NW_025544748 (424962-492810): Colletotrichum gloeosporioides strain Lc1 chrom... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_10.svg">
NW_021167084 (1837567-1907553): Sodiomyces alkalinus F11 unplaced genomic sca... (20% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r4c1-download">Download graphic</button>
</div>
<div id="clusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-295"></div> <span class="help-tooltip" id="cb-known-help-295">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c1_1.svg">
shimalactone A/shimalactone B
</option>

<option value = "svg/knownclusterblast_r4c1_2.svg">
fusarin
</option>

<option value = "svg/knownclusterblast_r4c1_3.svg">
fusarin C
</option>

</select>
<button id="knownclusterblast-r4c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-296"></div> <span class="help-tooltip" id="cb-sub-help-296">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-299"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-299">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-domain-bubble-svg" data-filename="scaffold_4_r4c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (146692 - 241447): neighbouring T1PKS-NRPS
</option>



<option value="CC2">
Candidate 2 (146692 - 194370): single T1PKS
</option>



<option value="CC3">
Candidate 3 (190106 - 241447): chemical hybrid T1PKS-NRPS
</option>


</select>

<div class="details-svg" id="r4c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-301"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-301">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="scaffold_4_r4c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-302"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-302">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-tigrfam-details-svg" data-filename="scaffold_4_r4c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c1-nrps_pks_monomers" data-name="r4c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-297"></div> <span class="help-tooltip" id="nrps-structures-help-297">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 146692 - 241447:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk) + (pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 146692 - 194370:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-2" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 190106 - 241447:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-3" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-298"></div> <span class="help-tooltip" id="nrps-monomers-help-298">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene48</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene48 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (512..823)</strong>: pk
<div class="collapser collapser-target-input-path1-gene48 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene48-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Methoxymalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene48-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 76.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 67.5</dt>

<dd></dd><dt>Malonyl-CoA: 49.2</dt>

<dd></dd><dt>Propionyl-CoA: 43.5</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 34.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 33.9</dt>

<dd></dd><dt>fatty_acid: 30.7</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 26.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 19.2</dt>

<dd></dd><dt>inactive: 17.1</dt>

<dd></dd><dt>CHC-CoA: 16.0</dt>

<dd></dd><dt>Benzoyl-CoA: 9.5</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: -5.5</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

<strong><span class="serif">input.path1.gene55</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene55 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (318..630)</strong>: pk
<div class="collapser collapser-target-input-path1-gene55 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene55-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene55-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 89.6</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 79.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 67.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 61.1</dt>

<dd></dd><dt>Propionyl-CoA: 42.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 33.2</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 32.5</dt>

<dd></dd><dt>Malonyl-CoA: 32.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 27.1</dt>

<dd></dd><dt>inactive: 25.3</dt>

<dd></dd><dt>Benzoyl-CoA: 21.9</dt>

<dd></dd><dt>CHC-CoA: 20.0</dt>

<dd></dd><dt>fatty_acid: 19.2</dt>

<dd></dd><dt>Acetyl-CoA: 14.6</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2506..2915)</strong>: X
<div class="collapser collapser-target-input-path1-gene55 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene55-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIAYSIESQK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-534"></div> <span class="help-tooltip" id="region_svg-help-534">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c2-svg" data-filename="scaffold_4_r4c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 722,715 - 783,737 nt. (total: 61,023 nt)

<a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-535"></div> <span class="help-tooltip" id="focus-panel-help-535">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c2-gene-table-details" data-name="r4c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c2-clusterblast" data-name="r4c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c2-pfam-details" data-name="r4c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r4c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-307"></div> <span class="help-tooltip" id="gt-help-tooltip-help-307">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-303"></div> <span class="help-tooltip" id="MIBiG-help-303">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c2"></div>
<div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002150: 0-3220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002150/index.html#r1c1" target="_new">BGC0002150</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>triculamin</td>


<td>Streptomyces triculaminicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002151: 0-3226">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002151/index.html#r1c1" target="_new">BGC0002151</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>alboverticillin</td>


<td>Streptomyces griseocarneus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000802: 0-10580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000802/index.html#r1c1" target="_new">BGC0000802</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Saccharide</td>

<td>succinoglycan</td>


<td>Sinorhizobium meliloti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002322: 0-4106">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002322/index.html#r1c1" target="_new">BGC0002322</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>citrocin</td>


<td>Citrobacter pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>pseudomycoicidin</td>


<td>Bacillus pseudomycoides DSM 12442</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001862: 368-5173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001862/index.html#r1c1" target="_new">BGC0001862</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>pallidocin</td>


<td>Aeribacillus pallidus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000543: 87-7506">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000543/index.html#r1c1" target="_new">BGC0000543</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>pep5</td>


<td>Staphylococcus epidermidis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000560: 1037-9106">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000560/index.html#r1c1" target="_new">BGC0000560</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>subtilomycin</td>


<td>Bacillus subtilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001311: 105-15400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001311/index.html#r1c1" target="_new">BGC0001311</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>flavecins</td>


<td>Ruminococcus flavefaciens FD-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002698: 0-12831">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002698/index.html#r1c1" target="_new">BGC0002698</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>RiPP</td>

<td>phaeornamide</td>


<td>Pseudophaeobacter arcticus DSM 23566</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002150: 0-3220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002150/index.html#r1c1" target="_new">BGC0002150</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>triculamin</td>
<td>Streptomyces triculaminicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002151: 0-3226">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002151/index.html#r1c1" target="_new">BGC0002151</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>alboverticillin</td>
<td>Streptomyces griseocarneus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000802: 0-10580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000802/index.html#r1c1" target="_new">BGC0000802</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Saccharide</td>
<td>succinoglycan</td>
<td>Sinorhizobium meliloti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002322: 0-4106">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002322/index.html#r1c1" target="_new">BGC0002322</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>citrocin</td>
<td>Citrobacter pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>pseudomycoicidin</td>
<td>Bacillus pseudomycoides DSM 12442</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001862: 368-5173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001862/index.html#r1c1" target="_new">BGC0001862</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>pallidocin</td>
<td>Aeribacillus pallidus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000543: 87-7506">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000543/index.html#r1c1" target="_new">BGC0000543</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>pep5</td>
<td>Staphylococcus epidermidis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000560: 1037-9106">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000560/index.html#r1c1" target="_new">BGC0000560</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>subtilomycin</td>
<td>Bacillus subtilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001311: 105-15400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001311/index.html#r1c1" target="_new">BGC0001311</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>flavecins</td>
<td>Ruminococcus flavefaciens FD-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002698: 0-12831">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002698/index.html#r1c1" target="_new">BGC0002698</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>RiPP</td>
<td>phaeornamide</td>
<td>Pseudophaeobacter arcticus DSM 23566</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-304"></div> <span class="help-tooltip" id="cb-general-help-304">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c2_1.svg">
NW_006271977 (184210-275183): Cordyceps militaris CM01 unplaced genomic scaff... (37% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_2.svg">
NW_011942171 (314568-405746): Metarhizium robertsii ARSEF 23 MAA Scf 3, whole... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_3.svg">
NW_025091265 (422738-551496): Aspergillus udagawae IFM 46973 DNA, scaffold 2,... (4% of genes show similarity), T1PKS,fungal-RiPP-like,indole
</option>

<option value = "svg/clusterblast_r4c2_4.svg">
NW_019154045 (358401-688746): Pochonia chlamydosporia 170 chromosome Unknown ... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_5.svg">
NW_006271970 (3761055-3936010): Cordyceps militaris CM01 unplaced genomic sca... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_6.svg">
NW_020291606 (579604-818783): Aspergillus piperis CBS 112811 unplaced genomic... (3% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_7.svg">
NW_011942151 (20096-254382): Metarhizium robertsii ARSEF 23 MAA Scf 11, whole... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r4c2_8.svg">
NC_018215 (1659185-1716249): Zymoseptoria tritici IPO323 chromosome 4, whole ... (8% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r4c2_9.svg">
NW_020194479 (1449493-1553698): Amorphotheca resinae ATCC 22711 unplaced geno... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_10.svg">
NW_017253364 (1855051-1945948): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r4c2-download">Download graphic</button>
</div>
<div id="clusterblast-r4c2-svg"></div>

</div>
</div>

<div class="body-details-section r4c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-305"></div> <span class="help-tooltip" id="cb-known-help-305">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-306"></div> <span class="help-tooltip" id="cb-sub-help-306">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-308"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-308">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-pfam-details-svg" data-filename="scaffold_4_r4c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r4c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-536"></div> <span class="help-tooltip" id="region_svg-help-536">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c3-svg" data-filename="scaffold_4_r4c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,700,022 - 1,744,001 nt. (total: 43,980 nt)

<a class="cluster-rules-header" id="r4c3-rules-header" href="#r4c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-537"></div> <span class="help-tooltip" id="focus-panel-help-537">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c3-gene-table-details" data-name="r4c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c3-nrps_pks" data-name="r4c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c3-MIBiG-cluster-compare" data-name="r4c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c3-clusterblast" data-name="r4c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c3-knownclusterblast" data-name="r4c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c3-subclusterblast" data-name="r4c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c3-pfam-details" data-name="r4c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c3-tigrfam-details" data-name="r4c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-315"></div> <span class="help-tooltip" id="gt-help-tooltip-help-315">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-309"></div> <span class="help-tooltip" id="nrps-domain-help-309">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-details-svg" data-filename="scaffold_4_r4c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-310"></div> <span class="help-tooltip" id="MIBiG-help-310">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c3"></div>
<div id="comparison-MIBiG-r4c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.55, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392: 0-27717">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.48, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>mirubactin</td>


<td>Actinosynnema mirum DSM 43827</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001806: 0-60453">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001806/index.html#r1c1" target="_new">BGC0001806</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>tolaasin A</td>


<td>Pseudomonas tolaasii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000435: 0-32188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000435/index.html#r1c1" target="_new">BGC0000435</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.41, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>syringafactin</td>


<td>Pseudomonas syringae pv. tomato str. DC3000</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000454: 24-19884">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000454/index.html#r1c1" target="_new">BGC0000454</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.49, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>vanchrobactin</td>


<td>Vibrio anguillarum RV22</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438: 137-73725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>syringopeptin 25A</td>


<td>Pseudomonas syringae pv. syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000385: 0-49278">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000385/index.html#r1c1" target="_new">BGC0000385</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>lysobactin</td>


<td>Lysobacter sp. ATCC 53042</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.49, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>1,6-phenazinedimethanol, fontizine A</td>


<td>Serratia fonticola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000460: 0-23711">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000460/index.html#r1c1" target="_new">BGC0000460</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.49, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>vulnibactin</td>


<td>Vibrio vulnificus CMCP6</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.55, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392: 0-27717">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.48, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>NRP</td>
<td>mirubactin</td>
<td>Actinosynnema mirum DSM 43827</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001806: 0-60453">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001806/index.html#r1c1" target="_new">BGC0001806</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP</td>
<td>tolaasin A</td>
<td>Pseudomonas tolaasii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000435: 0-32188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000435/index.html#r1c1" target="_new">BGC0000435</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.41, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>NRP</td>
<td>syringafactin</td>
<td>Pseudomonas syringae pv. tomato str. DC3000</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000454: 24-19884">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000454/index.html#r1c1" target="_new">BGC0000454</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.49, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>NRP</td>
<td>vanchrobactin</td>
<td>Vibrio anguillarum RV22</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438: 137-73725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>NRP</td>
<td>syringopeptin 25A</td>
<td>Pseudomonas syringae pv. syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000385: 0-49278">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000385/index.html#r1c1" target="_new">BGC0000385</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>NRP</td>
<td>lysobactin</td>
<td>Lysobacter sp. ATCC 53042</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.49, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>NRP</td>
<td>1,6-phenazinedimethanol, fontizine A</td>
<td>Serratia fonticola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000460: 0-23711">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000460/index.html#r1c1" target="_new">BGC0000460</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.49, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>NRP</td>
<td>vulnibactin</td>
<td>Vibrio vulnificus CMCP6</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-311"></div> <span class="help-tooltip" id="cb-general-help-311">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c3_1.svg">
NW_006271972 (846900-910624): Cordyceps militaris CM01 unplaced genomic scaff... (45% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c3_2.svg">
NW_025408674 (2031871-2095823): Hirsutella rhossiliensis strain HR02 chromoso... (34% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c3_3.svg">
NC_070546 (1097748-1161429): Fusarium falciforme chromosome 3, complete sequence (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c3_4.svg">
NC_070531 (1081663-1145343): Fusarium keratoplasticum isolate Fu6.1 chromosom... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c3_5.svg">
NC_072845 (1566921-1630612): Fusarium oxysporum Fo47 chromosome VI, complete ... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c3_6.svg">
NC_058392 (2751608-2815071): Fusarium musae strain F31 chromosome 6, whole ge... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c3_7.svg">
NC_030957 (4696242-4786854): Colletotrichum higginsianum IMI 349063 chromosom... (14% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c3_8.svg">
NC_015953 (1915584-1983417): Streptomyces sp. SirexAA-E, complete sequence (3% of genes show similarity), LAP,NRPS-like,PKS-like
</option>

<option value = "svg/clusterblast_r4c3_9.svg">
NW_021940921 (898993-983846): Pyricularia pennisetigena strain Br36 chromosom... (7% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r4c3-download">Download graphic</button>
</div>
<div id="clusterblast-r4c3-svg"></div>

</div>
</div>

<div class="body-details-section r4c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-312"></div> <span class="help-tooltip" id="cb-known-help-312">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-313"></div> <span class="help-tooltip" id="cb-sub-help-313">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-316"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-316">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-pfam-details-svg" data-filename="scaffold_4_r4c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-317"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-317">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-tigrfam-details-svg" data-filename="scaffold_4_r4c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c3-nrps_pks_monomers" data-name="r4c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r4c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-314"></div> <span class="help-tooltip" id="nrps-monomers-help-314">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene485</span></strong>: X
<div class="collapser collapser-target-input-path1-gene485 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (44..469)</strong>: X
<div class="collapser collapser-target-input-path1-gene485 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene485-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">IFNSSAAPLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r4c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 4 - T1PKS,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-538"></div> <span class="help-tooltip" id="region_svg-help-538">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c4-svg" data-filename="scaffold_4_r4c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,932,254 - 2,007,007 nt. (total: 74,754 nt)

<a class="cluster-rules-header" id="r4c4-rules-header" href="#r4c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c4-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-539"></div> <span class="help-tooltip" id="focus-panel-help-539">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c4-gene-table-details" data-name="r4c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c4-nrps_pks" data-name="r4c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c4-MIBiG-cluster-compare" data-name="r4c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c4-clusterblast" data-name="r4c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c4-knownclusterblast" data-name="r4c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c4-subclusterblast" data-name="r4c4-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r4c4-pfam-details" data-name="r4c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c4-tigrfam-details" data-name="r4c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-324"></div> <span class="help-tooltip" id="gt-help-tooltip-help-324">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-318"></div> <span class="help-tooltip" id="nrps-domain-help-318">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-details-svg" data-filename="scaffold_4_r4c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-319"></div> <span class="help-tooltip" id="MIBiG-help-319">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c4"></div>
<div id="comparison-MIBiG-r4c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.40</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.36</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.34</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.34</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.34</td>
<td>Polyketide</td>

<td>naphthalene</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>phaseolotoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
<td>Fusarium graminearum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-320"></div> <span class="help-tooltip" id="cb-general-help-320">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c4_1.svg">
NW_006271971 (2425175-2516249): Cordyceps militaris CM01 unplaced genomic sca... (38% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c4_2.svg">
NW_006271969 (1967760-2079673): Cordyceps militaris CM01 unplaced genomic sca... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c4_3.svg">
NW_022985016 (85063-152593): Dothidotthia symphoricarpi CBS 119687 unplaced g... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c4_4.svg">
NW_006271969 (4860304-4927848): Cordyceps militaris CM01 unplaced genomic sca... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c4_5.svg">
NW_013550604 (1398013-1484027): Rhinocladiella mackenziei CBS 650.93 unplaced... (23% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r4c4_6.svg">
NC_049565 (1715696-1783101): Talaromyces rugulosus chromosome V, complete seq... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c4_7.svg">
NW_021167090 (669974-737132): Sodiomyces alkalinus F11 unplaced genomic scaff... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c4_8.svg">
NW_025091264 (5340181-5432359): Aspergillus udagawae IFM 46973 DNA, scaffold ... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c4_9.svg">
NW_008481799 (388930-472807): Cyphellophora europaea CBS 101466 unplaced geno... (12% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r4c4_10.svg">
NW_026249877 (17373-85239): Alternaria incomplexa strain BMP 0042 chromosome ... (14% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r4c4-download">Download graphic</button>
</div>
<div id="clusterblast-r4c4-svg"></div>

</div>
</div>

<div class="body-details-section r4c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-321"></div> <span class="help-tooltip" id="cb-known-help-321">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-322"></div> <span class="help-tooltip" id="cb-sub-help-322">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r4c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-325"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-325">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-pfam-details-svg" data-filename="scaffold_4_r4c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-326"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-326">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-tigrfam-details-svg" data-filename="scaffold_4_r4c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c4-nrps_pks_monomers" data-name="r4c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-323"></div> <span class="help-tooltip" id="nrps-monomers-help-323">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene561</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene561 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (532..865)</strong>: pk
<div class="collapser collapser-target-input-path1-gene561 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene561-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>2-Methylbutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene561-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 82.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 74.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 59.3</dt>

<dd></dd><dt>Malonyl-CoA: 46.9</dt>

<dd></dd><dt>Propionyl-CoA: 43.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 41.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 30.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 27.7</dt>

<dd></dd><dt>CHC-CoA: 19.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 17.7</dt>

<dd></dd><dt>fatty_acid: 17.6</dt>

<dd></dd><dt>Benzoyl-CoA: 12.4</dt>

<dd></dd><dt>Acetyl-CoA: 9.9</dt>

<dd></dd><dt>inactive: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 5 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-540"></div> <span class="help-tooltip" id="region_svg-help-540">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c5-svg" data-filename="scaffold_4_r4c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,137,263 - 4,187,527 nt. (total: 50,265 nt)

<a class="cluster-rules-header" id="r4c5-rules-header" href="#r4c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c5-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-541"></div> <span class="help-tooltip" id="focus-panel-help-541">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c5-gene-table-details" data-name="r4c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c5-nrps_pks" data-name="r4c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c5-MIBiG-cluster-compare" data-name="r4c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c5-clusterblast" data-name="r4c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c5-knownclusterblast" data-name="r4c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c5-subclusterblast" data-name="r4c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c5-nrps-pks-bubbles" data-name="r4c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c5-pfam-details" data-name="r4c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c5-tigrfam-details" data-name="r4c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-335"></div> <span class="help-tooltip" id="gt-help-tooltip-help-335">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-327"></div> <span class="help-tooltip" id="nrps-domain-help-327">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c5-details-svg" data-filename="scaffold_4_r4c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-328"></div> <span class="help-tooltip" id="MIBiG-help-328">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c5"></div>
<div id="comparison-MIBiG-r4c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>xenortide A, xenortide B, xenortide C, xenortide D</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.41, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:0.69, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.41, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>xenortide A, xenortide B, xenortide C, xenortide D</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.41, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:0.69, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.41, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-329"></div> <span class="help-tooltip" id="cb-general-help-329">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c5_1.svg">
NC_054859 (4418286-4502506): Aspergillus puulaauensis MK2 DNA, chromosome 3, ... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c5_2.svg">
NC_066264 (8229-79458): Aspergillus nidulans FGSC A4 chromosome VIII (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c5_3.svg">
NC_058399 (7551854-7691715): Fusarium poae strain DAOMC 252244 chromosome 1, ... (4% of genes show similarity), NRPS,NRPS-like,T1PKS,isocyanide
</option>

<option value = "svg/clusterblast_r4c5_4.svg">
NW_025408674 (1561057-1628305): Hirsutella rhossiliensis strain HR02 chromoso... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c5_5.svg">
NW_024467526 (7938-201848): Aspergillus versicolor CBS 583.65 unplaced genomi... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c5_6.svg">
NW_014013674 (156254-238190): Trichoderma virens Gv29-8 chromosome Unknown TR... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c5_7.svg">
NW_020291486 (59597-203320): Aspergillus aculeatinus CBS 121060 unplaced geno... (3% of genes show similarity), NRPS,fungal-RiPP-like,isocyanide
</option>

<option value = "svg/clusterblast_r4c5_8.svg">
NW_025091281 (4887733-5033357): Aspergillus pseudoviridinutans IFM 55266 DNA,... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c5_9.svg">
NW_020290620 (309344-399291): Aspergillus saccharolyticus JOP 1030-1 unplaced... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c5_10.svg">
NW_022987215 (2142177-2223208): Trematosphaeria pertusa strain CBS 122368 unp... (6% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r4c5-download">Download graphic</button>
</div>
<div id="clusterblast-r4c5-svg"></div>

</div>
</div>

<div class="body-details-section r4c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-330"></div> <span class="help-tooltip" id="cb-known-help-330">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-331"></div> <span class="help-tooltip" id="cb-sub-help-331">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-334"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-334">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c5-domain-bubble-svg" data-filename="scaffold_4_r4c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC9">
Candidate 9 (4137262 - 4187527): single NRPS
</option>


</select>

<div class="details-svg" id="r4c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-336"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-336">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c5-pfam-details-svg" data-filename="scaffold_4_r4c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-337"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-337">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c5-tigrfam-details-svg" data-filename="scaffold_4_r4c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c5-nrps_pks_products" data-name="r4c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c5-nrps_pks_monomers" data-name="r4c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-332"></div> <span class="help-tooltip" id="nrps-structures-help-332">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 9, location 4137262 - 4187527:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c5-9" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c5-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-333"></div> <span class="help-tooltip" id="nrps-monomers-help-333">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1216</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1216 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (161..558)</strong>: X
<div class="collapser collapser-target-input-path1-gene1216 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene1216-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">LIRNLTASLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">input.path1.gene1217</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1217 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (247..651)</strong>: X
<div class="collapser collapser-target-input-path1-gene1217 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene1217-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIACIVAIFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DIAILVAILK</span> (53% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 6 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-542"></div> <span class="help-tooltip" id="region_svg-help-542">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c6-svg" data-filename="scaffold_4_r4c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,324,099 - 4,367,973 nt. (total: 43,875 nt)

<a class="cluster-rules-header" id="r4c6-rules-header" href="#r4c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c6-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-543"></div> <span class="help-tooltip" id="focus-panel-help-543">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c6-gene-table-details" data-name="r4c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c6-nrps_pks" data-name="r4c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c6-MIBiG-cluster-compare" data-name="r4c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c6-clusterblast" data-name="r4c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c6-knownclusterblast" data-name="r4c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c6-subclusterblast" data-name="r4c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c6-pfam-details" data-name="r4c6-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r4c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-344"></div> <span class="help-tooltip" id="gt-help-tooltip-help-344">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-338"></div> <span class="help-tooltip" id="nrps-domain-help-338">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c6-details-svg" data-filename="scaffold_4_r4c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-339"></div> <span class="help-tooltip" id="MIBiG-help-339">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c6"></div>
<div id="comparison-MIBiG-r4c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.51, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>lupeol</td>


<td>Lotus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.53, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.52, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002218: 3-22615">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002218/index.html#r1c1" target="_new">BGC0002218</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>(-)-Cyatha-3,12-diene</td>


<td>Dentipellis sp. KUC8613</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.51, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>lupeol</td>
<td>Lotus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.53, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.52, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002218: 3-22615">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002218/index.html#r1c1" target="_new">BGC0002218</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>(-)-Cyatha-3,12-diene</td>
<td>Dentipellis sp. KUC8613</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-340"></div> <span class="help-tooltip" id="cb-general-help-340">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c6_1.svg">
NW_023336287 (1029426-1127137): Aspergillus tubingensis WU-2223L DNA, scaffol... (14% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c6_2.svg">
NW_020291625 (99659-196841): Aspergillus piperis CBS 112811 unplaced genomic ... (14% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c6_3.svg">
NW_020291816 (340965-433020): Aspergillus costaricaensis CBS 115574 unplaced ... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c6_4.svg">
NW_020292069 (316400-408453): Aspergillus vadensis CBS 113365 unplaced genomi... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c6_5.svg">
NW_006271971 (3503471-3590640): Cordyceps militaris CM01 unplaced genomic sca... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c6_6.svg">
NC_054853 (2119459-2216438): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (12% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c6_7.svg">
NZ_CP070223 (1894925-2023629): Hydrogenophaga sp. YM1 chromosome, complete ge... (10% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c6_8.svg">
NZ_MIYM01000002 (248438-375235): Hydrogenophaga sp. PML113 NODE 2 length 5640... (11% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c6_9.svg">
NZ_CP022098 (7786475-7871429): Cystobacter fuscus strain DSM 52655 chromosome... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c6_10.svg">
NZ_CP046173 (6837938-7022382): Nocardia terpenica strain AUSMDU00012715 chrom... (12% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r4c6-download">Download graphic</button>
</div>
<div id="clusterblast-r4c6-svg"></div>

</div>
</div>

<div class="body-details-section r4c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-341"></div> <span class="help-tooltip" id="cb-known-help-341">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c6-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c6_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c6_1.svg">
FR901512
</option>

<option value = "svg/knownclusterblast_r4c6_2.svg">
fusarin C
</option>

<option value = "svg/knownclusterblast_r4c6_3.svg">
fusarin
</option>

<option value = "svg/knownclusterblast_r4c6_4.svg">
ACT-Toxin II
</option>

</select>
<button id="knownclusterblast-r4c6-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c6-svg"></div>

</div>
</div>

<div class="body-details-section r4c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-342"></div> <span class="help-tooltip" id="cb-sub-help-342">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-345"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-345">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c6-pfam-details-svg" data-filename="scaffold_4_r4c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c6-nrps_pks_monomers" data-name="r4c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r4c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-343"></div> <span class="help-tooltip" id="nrps-monomers-help-343">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1268</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1268 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (42..424)</strong>: X
<div class="collapser collapser-target-input-path1-gene1268 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1268-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALTVGGIVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DALAVGEVVK</span> (50% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">input.path1.gene1271</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene1271 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (0..308)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1271 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1271-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1271-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 63.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 49.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 42.2</dt>

<dd></dd><dt>Malonyl-CoA: 28.3</dt>

<dd></dd><dt>fatty_acid: 26.8</dt>

<dd></dd><dt>Propionyl-CoA: 26.1</dt>

<dd></dd><dt>Acetyl-CoA: 14.9</dt>

<dd></dd><dt>Benzoyl-CoA: 11.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 10.7</dt>

<dd></dd><dt>CHC-CoA: 8.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 5.7</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 5.6</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r4c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 7 - NRPS,T1PKS,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-544"></div> <span class="help-tooltip" id="region_svg-help-544">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c7-svg" data-filename="scaffold_4_r4c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,376,382 - 4,435,002 nt. (total: 58,621 nt)

<a class="cluster-rules-header" id="r4c7-rules-header" href="#r4c7">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r4c7-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-545"></div> <span class="help-tooltip" id="focus-panel-help-545">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c7-gene-table-details" data-name="r4c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c7-nrps_pks" data-name="r4c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c7-MIBiG-cluster-compare" data-name="r4c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c7-clusterblast" data-name="r4c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c7-knownclusterblast" data-name="r4c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c7-subclusterblast" data-name="r4c7-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r4c7-pfam-details" data-name="r4c7-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r4c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-352"></div> <span class="help-tooltip" id="gt-help-tooltip-help-352">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-346"></div> <span class="help-tooltip" id="nrps-domain-help-346">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c7-details-svg" data-filename="scaffold_4_r4c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-347"></div> <span class="help-tooltip" id="MIBiG-help-347">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c7"></div>
<div id="comparison-MIBiG-r4c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.47, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.47, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.48, order:0.69, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.70</td>
<td>Polyketide</td>

<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>


<td>Chaetomium olivaceum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.54</td>
<td>Polyketide</td>

<td>emodin</td>


<td>Escovopsis weberi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001888: 459-19007">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001888/index.html#r1c1" target="_new">BGC0001888</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>






<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>






<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.44</td>
<td>Other</td>

<td>mannosylerythritol lipid A, mannosylerythritol lipid B, mannosylerythritol lipid C</td>


<td>Ustilago maydis 521</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.51, order:0.33, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.51, order:0.33, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.39 (id:0.51, order:0.34, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.43</td>
<td>Polyketide, NRP</td>

<td>xenoacremone A</td>


<td>Xenoacremonium sp. BF-2018a</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.41</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.40</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.40</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.40</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002208: 764-29749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002208/index.html#r1c1" target="_new">BGC0002208</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>oxepinamide F</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:0.36, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>






<td class="cc-heat-cell" title="0.44 (id:0.43, order:0.36, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>






<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.40</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.47, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>
<td>Chaetomium olivaceum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>emodin</td>
<td>Escovopsis weberi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001874: 440-16375">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001874/index.html#r1c1" target="_new">BGC0001874</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP, Polyketide</td>
<td>cyclopiazonic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>
<td>Aspergillus oryzae RIB40</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-348"></div> <span class="help-tooltip" id="cb-general-help-348">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c7_1.svg">
NC_016457 (6563485-6636075): Thermothielavioides terrestris NRRL 8126 chromos... (30% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c7_2.svg">
NW_017263586 (566702-639094): Mollisia scopiformis unplaced genomic scaffold ... (27% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c7_3.svg">
NW_026249941 (389502-461772): Alternaria arbusti strain BMP 1465 chromosome U... (23% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c7_4.svg">
NW_026055684 (1807404-1879675): Alternaria ventricosa strain BMP 2768 chromos... (20% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c7_5.svg">
NW_026054642 (106430-178539): Ophidiomyces ophidiicola strain CBS 122913 chro... (30% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c7_6.svg">
NW_007360994 (1741133-1834732): Glarea lozoyensis ATCC 20868 chromosome Unkno... (14% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c7_7.svg">
NW_007360988 (678454-840815): Glarea lozoyensis ATCC 20868 chromosome Unknown... (6% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c7_8.svg">
NC_016474 (1151824-1224022): Thermothelomyces thermophilus ATCC 42464 chromos... (20% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c7_9.svg">
NW_021940922 (928677-1070633): Pyricularia pennisetigena strain Br36 chromoso... (23% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c7_10.svg">
NW_024067566 (1440324-1543594): Penicillium roqueforti strain LCP96 04111 chr... (13% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r4c7-download">Download graphic</button>
</div>
<div id="clusterblast-r4c7-svg"></div>

</div>
</div>

<div class="body-details-section r4c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-349"></div> <span class="help-tooltip" id="cb-known-help-349">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c7-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c7_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c7_1.svg">
GKK1032 A2/pyrrocidine A/pyrrocidine B
</option>

<option value = "svg/knownclusterblast_r4c7_2.svg">
xenoacremone A
</option>

</select>
<button id="knownclusterblast-r4c7-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c7-svg"></div>

</div>
</div>

<div class="body-details-section r4c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-350"></div> <span class="help-tooltip" id="cb-sub-help-350">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r4c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-353"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-353">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c7-pfam-details-svg" data-filename="scaffold_4_r4c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c7-nrps_pks_monomers" data-name="r4c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-351"></div> <span class="help-tooltip" id="nrps-monomers-help-351">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1278</span></strong>: pk - Tyr
<div class="collapser collapser-target-input-path1-gene1278 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>PKS_AT (561..882)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1278 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1278-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Methoxymalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1278-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 78.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 64.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 50.2</dt>

<dd></dd><dt>Malonyl-CoA: 46.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 35.0</dt>

<dd></dd><dt>Benzoyl-CoA: 24.0</dt>

<dd></dd><dt>Propionyl-CoA: 21.1</dt>

<dd></dd><dt>fatty_acid: 18.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 16.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 12.3</dt>

<dd></dd><dt>CHC-CoA: 9.6</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>









<dd><strong>AMP-binding (2591..3004)</strong>: Tyr
<div class="collapser collapser-target-input-path1-gene1278 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Tyr
<div class="collapser collapser-target-input-path1-gene1278-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMNSTGALSK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DMNSTGALSK</span> (88% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r5c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-546"></div> <span class="help-tooltip" id="region_svg-help-546">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c1-svg" data-filename="scaffold_5_r5c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 70,132 - 117,548 nt. (total: 47,417 nt)

<a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-547"></div> <span class="help-tooltip" id="focus-panel-help-547">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c1-gene-table-details" data-name="r5c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c1-nrps_pks" data-name="r5c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c1-nrps-pks-bubbles" data-name="r5c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c1-tigrfam-details" data-name="r5c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-362"></div> <span class="help-tooltip" id="gt-help-tooltip-help-362">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-354"></div> <span class="help-tooltip" id="nrps-domain-help-354">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-details-svg" data-filename="scaffold_5_r5c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-355"></div> <span class="help-tooltip" id="MIBiG-help-355">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c1"></div>
<div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Terpene</td>

<td>eupenifeldin</td>


<td>Phoma sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>citrinin</td>


<td>Monascus ruber</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000154: 266-31301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000154/index.html#r1c1" target="_new">BGC0000154</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>stipitatic acid</td>


<td>Talaromyces stipitatus ATCC 10500</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene, Polyketide</td>

<td>subglutinol A, subglutinol B</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>chevalone E</td>


<td>Aspergillus versicolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002483: 1330-45854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002483/index.html#r1c1" target="_new">BGC0002483</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>usnic acid</td>


<td>Cladonia uncialis subsp. uncialis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000027: 10301-76156">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000027/index.html#r1c1" target="_new">BGC0000027</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>ankaflavin, monascin, rubropunctatine, monascorubrin</td>


<td>Monascus pilosus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>eupenifeldin</td>
<td>Phoma sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>citrinin</td>
<td>Monascus ruber</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000154: 266-31301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000154/index.html#r1c1" target="_new">BGC0000154</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>stipitatic acid</td>
<td>Talaromyces stipitatus ATCC 10500</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, Polyketide</td>
<td>subglutinol A, subglutinol B</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>chevalone E</td>
<td>Aspergillus versicolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002483: 1330-45854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002483/index.html#r1c1" target="_new">BGC0002483</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>usnic acid</td>
<td>Cladonia uncialis subsp. uncialis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000027: 10301-76156">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000027/index.html#r1c1" target="_new">BGC0000027</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>ankaflavin, monascin, rubropunctatine, monascorubrin</td>
<td>Monascus pilosus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-356"></div> <span class="help-tooltip" id="cb-general-help-356">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c1_1.svg">
NW_025091264 (437383-553415): Aspergillus udagawae IFM 46973 DNA, scaffold 1,... (4% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_2.svg">
NZ_CP012949 (7450605-7620269): Streptomyces ambofaciens strain DSM 40697 chro... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_3.svg">
NW_024467534 (27161-95248): Aspergillus versicolor CBS 583.65 unplaced genomi... (6% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_4.svg">
NW_025953832 (31975-100050): Annulohypoxylon truncatum strain CBS 140777 unpl... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_5.svg">
NW_025953254 (1383196-1451301): Annulohypoxylon maeteangense strain CBS 12383... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_6.svg">
NW_024467861 (3251492-3338124): Aspergillus sydowii CBS 593.65 unplaced genom... (6% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c1_7.svg">
NW_024467526 (3357689-3444377): Aspergillus versicolor CBS 583.65 unplaced ge... (7% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c1_8.svg">
NC_054859 (3464231-3550852): Aspergillus puulaauensis MK2 DNA, chromosome 3, ... (6% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c1_9.svg">
NZ_CP012382 (7665381-7835031): Streptomyces ambofaciens ATCC 23877 chromosome... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_10.svg">
NW_017263599 (967676-1034636): Mollisia scopiformis unplaced genomic scaffold... (9% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r5c1-download">Download graphic</button>
</div>
<div id="clusterblast-r5c1-svg"></div>

</div>
</div>

<div class="body-details-section r5c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-357"></div> <span class="help-tooltip" id="cb-known-help-357">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-358"></div> <span class="help-tooltip" id="cb-sub-help-358">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-361"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-361">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-domain-bubble-svg" data-filename="scaffold_5_r5c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (70131 - 117548): single T1PKS
</option>


</select>

<div class="details-svg" id="r5c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-363"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-363">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="scaffold_5_r5c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-364"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-364">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-tigrfam-details-svg" data-filename="scaffold_5_r5c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c1-nrps_pks_products" data-name="r5c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c1-nrps_pks_monomers" data-name="r5c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-359"></div> <span class="help-tooltip" id="nrps-structures-help-359">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 70131 - 117548:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c1-1" class="smiles-canvas" data-smiles="C(OC)C(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mxmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mxmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mxmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C(OC)C(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-360"></div> <span class="help-tooltip" id="nrps-monomers-help-360">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene22</span></strong>: mxmal
<div class="collapser collapser-target-input-path1-gene22 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (745..1038)</strong>: mxmal
<div class="collapser collapser-target-input-path1-gene22 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-input-path1-gene22-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-input-path1-gene22-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 51.0</dt>

<dd></dd><dt>Methylmalonyl-CoA: 47.3</dt>

<dd></dd><dt>Malonyl-CoA: 40.2</dt>

<dd></dd><dt>Propionyl-CoA: 34.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 31.8</dt>

<dd></dd><dt>CHC-CoA: 26.5</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 23.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 21.6</dt>

<dd></dd><dt>inactive: 17.7</dt>

<dd></dd><dt>Benzoyl-CoA: 14.9</dt>

<dd></dd><dt>Acetyl-CoA: 13.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 12.1</dt>

<dd></dd><dt>fatty_acid: 11.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.9</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-548"></div> <span class="help-tooltip" id="region_svg-help-548">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c2-svg" data-filename="scaffold_5_r5c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,053,084 - 3,122,111 nt. (total: 69,028 nt)

<a class="cluster-rules-header" id="r5c2-rules-header" href="#r5c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r5c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-549"></div> <span class="help-tooltip" id="focus-panel-help-549">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c2-gene-table-details" data-name="r5c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c2-MIBiG-cluster-compare" data-name="r5c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c2-clusterblast" data-name="r5c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c2-knownclusterblast" data-name="r5c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c2-subclusterblast" data-name="r5c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c2-pfam-details" data-name="r5c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c2-tigrfam-details" data-name="r5c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-369"></div> <span class="help-tooltip" id="gt-help-tooltip-help-369">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-365"></div> <span class="help-tooltip" id="MIBiG-help-365">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c2"></div>
<div id="comparison-MIBiG-r5c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000509: 4195-15261">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000509/index.html#r1c1" target="_new">BGC0000509</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>epilancin 15x</td>


<td>Staphylococcus epidermidis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>falcarindiol</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002701: 0-9977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002701/index.html#r1c1" target="_new">BGC0002701</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>paenithipeptin</td>


<td>Paenibacillus thiaminolyticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002579: 0-7721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002579/index.html#r1c1" target="_new">BGC0002579</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>carnobacteriocin XY</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002585: 998-10750">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002585/index.html#r1c1" target="_new">BGC0002585</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>ubericin K</td>


<td>Streptococcus uberis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>itaconic acid</td>


<td>Ustilago maydis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002747: 549-28042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002747/index.html#r1c1" target="_new">BGC0002747</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>lentinulin A</td>


<td>Lentinula edodes</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.49, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>isoindolinone</td>


<td>Stachybotrys chlorohalonata IBT 40285</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000509: 4195-15261">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000509/index.html#r1c1" target="_new">BGC0000509</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>epilancin 15x</td>
<td>Staphylococcus epidermidis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>falcarindiol</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002701: 0-9977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002701/index.html#r1c1" target="_new">BGC0002701</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>paenithipeptin</td>
<td>Paenibacillus thiaminolyticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002579: 0-7721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002579/index.html#r1c1" target="_new">BGC0002579</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>carnobacteriocin XY</td>
<td>Carnobacterium maltaromaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002585: 998-10750">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002585/index.html#r1c1" target="_new">BGC0002585</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>ubericin K</td>
<td>Streptococcus uberis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>itaconic acid</td>
<td>Ustilago maydis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002747: 549-28042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002747/index.html#r1c1" target="_new">BGC0002747</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>RiPP</td>
<td>lentinulin A</td>
<td>Lentinula edodes</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.49, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>isoindolinone</td>
<td>Stachybotrys chlorohalonata IBT 40285</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-366"></div> <span class="help-tooltip" id="cb-general-help-366">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c2_1.svg">
NT_165982 (1008305-1103578): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (20% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_2.svg">
NC_030959 (4385741-4495530): Colletotrichum higginsianum IMI 349063 chromosom... (14% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_3.svg">
NW_017263587 (265177-438368): Mollisia scopiformis unplaced genomic scaffold ... (5% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_4.svg">
NW_006271970 (2031211-2136459): Cordyceps militaris CM01 unplaced genomic sca... (18% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_5.svg">
NW_011371361 (1937664-2035052): Paracoccidioides brasiliensis Pb18 unplaced g... (17% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_6.svg">
NW_015440948 (109188-206314): Paracoccidioides lutzii Pb01 unplaced genomic s... (18% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_7.svg">
NW_025953267 (201738-434522): Annulohypoxylon maeteangense strain CBS 123835 ... (2% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_8.svg">
NW_025953816 (281521-436136): Annulohypoxylon truncatum strain CBS 140777 unp... (3% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_9.svg">
NC_066257 (109084-244310): Aspergillus nidulans FGSC A4 chromosome I (6% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r5c2_10.svg">
NW_017263589 (1285573-1392383): Mollisia scopiformis unplaced genomic scaffol... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r5c2-download">Download graphic</button>
</div>
<div id="clusterblast-r5c2-svg"></div>

</div>
</div>

<div class="body-details-section r5c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-367"></div> <span class="help-tooltip" id="cb-known-help-367">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-368"></div> <span class="help-tooltip" id="cb-sub-help-368">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-370"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-370">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-pfam-details-svg" data-filename="scaffold_5_r5c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-371"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-371">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-tigrfam-details-svg" data-filename="scaffold_5_r5c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r5c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-550"></div> <span class="help-tooltip" id="region_svg-help-550">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c3-svg" data-filename="scaffold_5_r5c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,684,161 - 3,732,594 nt. (total: 48,434 nt)

<a class="cluster-rules-header" id="r5c3-rules-header" href="#r5c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-551"></div> <span class="help-tooltip" id="focus-panel-help-551">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c3-gene-table-details" data-name="r5c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c3-nrps_pks" data-name="r5c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c3-MIBiG-cluster-compare" data-name="r5c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c3-clusterblast" data-name="r5c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c3-knownclusterblast" data-name="r5c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c3-subclusterblast" data-name="r5c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c3-nrps-pks-bubbles" data-name="r5c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c3-pfam-details" data-name="r5c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c3-tigrfam-details" data-name="r5c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-380"></div> <span class="help-tooltip" id="gt-help-tooltip-help-380">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-372"></div> <span class="help-tooltip" id="nrps-domain-help-372">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-details-svg" data-filename="scaffold_5_r5c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-373"></div> <span class="help-tooltip" id="MIBiG-help-373">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c3"></div>
<div id="comparison-MIBiG-r5c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002622: 258-56047">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002622/index.html#r1c1" target="_new">BGC0002622</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.36</td>
<td>Other</td>

<td>hydroxycinnamoylputrescine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000293: 0-12765">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000293/index.html#r1c1" target="_new">BGC0000293</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>acetylaszonalenin</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002622: 258-56047">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002622/index.html#r1c1" target="_new">BGC0002622</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>hydroxycinnamoylputrescine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000293: 0-12765">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000293/index.html#r1c1" target="_new">BGC0000293</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>acetylaszonalenin</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-374"></div> <span class="help-tooltip" id="cb-general-help-374">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c3_1.svg">
NZ_CP010524 (2159044-2236915): Bacillus paralicheniformis strain BL-09 chromo... (7% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r5c3_2.svg">
NZ_CP012831 (5403821-5499692): Pseudomonas fluorescens strain FW300-N2C3 chro... (5% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c3_3.svg">
NZ_CP041745 (2013782-2112080): Paraburkholderia megapolitana strain LMG 23650... (4% of genes show similarity), NRPS,NRPS-like,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c3_4.svg">
NC_015711 (7333590-7403723): Corallococcus macrosporus, complete sequence (5% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c3_5.svg">
NZ_KE383842 (1649933-1738497): Paenibacillus harenae DSM 16969 H581DRAFT scaf... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c3_6.svg">
NZ_QGHB01000007 (176720-232526): Lentzea atacamensis strain DSM 45480 Ga02156... (4% of genes show similarity), NRPS,T1PKS,indole
</option>

<option value = "svg/clusterblast_r5c3_7.svg">
NZ_QLTT01000004 (532095-587907): Lentzea atacamensis strain DSM 45479 Ga02156... (4% of genes show similarity), NRPS,T1PKS,indole
</option>

<option value = "svg/clusterblast_r5c3_8.svg">
NZ_CP045701 (2604900-2681691): Pseudomonas brassicacearum strain S-1 chromoso... (5% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c3_9.svg">
NZ_LT629706 (446761-522933): Pseudomonas poae strain LMG 21465 chromosome I (5% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c3_10.svg">
NZ_CP098738 (358672-424068): Bacillus halotolerans strain XE48 chromosome, co... (4% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r5c3-download">Download graphic</button>
</div>
<div id="clusterblast-r5c3-svg"></div>

</div>
</div>

<div class="body-details-section r5c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-375"></div> <span class="help-tooltip" id="cb-known-help-375">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-376"></div> <span class="help-tooltip" id="cb-sub-help-376">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-379"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-379">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-domain-bubble-svg" data-filename="scaffold_5_r5c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (3684160 - 3732594): single NRPS
</option>


</select>

<div class="details-svg" id="r5c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-381"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-381">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-pfam-details-svg" data-filename="scaffold_5_r5c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-382"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-382">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-tigrfam-details-svg" data-filename="scaffold_5_r5c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c3-nrps_pks_products" data-name="r5c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c3-nrps_pks_monomers" data-name="r5c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-377"></div> <span class="help-tooltip" id="nrps-structures-help-377">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 3684160 - 3732594:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c3-3" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-378"></div> <span class="help-tooltip" id="nrps-monomers-help-378">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1014</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1014 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (206..471)</strong>: X
<div class="collapser collapser-target-input-path1-gene1014 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-input-path1-gene1014-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGRDTASISK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r6c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-552"></div> <span class="help-tooltip" id="region_svg-help-552">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c1-svg" data-filename="scaffold_6_r6c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 59,935 - 106,303 nt. (total: 46,369 nt)

<a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r6c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-553"></div> <span class="help-tooltip" id="focus-panel-help-553">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c1-gene-table-details" data-name="r6c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c1-nrps-pks-bubbles" data-name="r6c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c1-tigrfam-details" data-name="r6c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-391"></div> <span class="help-tooltip" id="gt-help-tooltip-help-391">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-383"></div> <span class="help-tooltip" id="nrps-domain-help-383">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="scaffold_6_r6c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-384"></div> <span class="help-tooltip" id="MIBiG-help-384">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c1"></div>
<div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002709: 890-28246">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002709/index.html#r1c1" target="_new">BGC0002709</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>aurofusarin</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002594: 653-30873">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002594/index.html#r1c1" target="_new">BGC0002594</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>orcinol, orsellinic acid</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Nodulisporium sp. ATCC74245</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002709: 890-28246">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002709/index.html#r1c1" target="_new">BGC0002709</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>aurofusarin</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002594: 653-30873">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002594/index.html#r1c1" target="_new">BGC0002594</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>orcinol, orsellinic acid</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Nodulisporium sp. ATCC74245</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-385"></div> <span class="help-tooltip" id="cb-general-help-385">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c1_1.svg">
NZ_CP048641 (1021290-1244509): Streptomyces aureoverticillatus strain HN6 chr... (17% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c1_2.svg">
NZ_MKZR01000001 (6570674-6673737): Moorena producens 3L Ga0081465 101, whole ... (19% of genes show similarity), NRPS,PKS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r6c1_3.svg">
NZ_CP017599 (7892181-7994187): Moorena producens PAL-8-15-08-1 chromosome, co... (18% of genes show similarity), NRPS,PKS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r6c1_4.svg">
NZ_CP017708 (6448824-6538425): Moorena producens JHB chromosome, complete genome (9% of genes show similarity), NRPS,PKS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r6c1_5.svg">
NW_025408669 (8000078-8067585): Hirsutella rhossiliensis strain HR02 chromoso... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c1_6.svg">
NW_020290890 (450315-638141): Aspergillus niger CBS 101883 unplaced genomic s... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c1_7.svg">
NW_023336287 (400215-591232): Aspergillus tubingensis WU-2223L DNA, scaffold ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c1_8.svg">
NC_054853 (2681752-2888163): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c1_9.svg">
NW_020291592 (339371-529932): Aspergillus piperis CBS 112811 unplaced genomic... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c1_10.svg">
NC_016458 (5074403-5202679): Thermothielavioides terrestris NRRL 8126 chromos... (5% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c1-download">Download graphic</button>
</div>
<div id="clusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-386"></div> <span class="help-tooltip" id="cb-known-help-386">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-387"></div> <span class="help-tooltip" id="cb-sub-help-387">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-390"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-390">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-domain-bubble-svg" data-filename="scaffold_6_r6c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (59934 - 106303): single T1PKS
</option>


</select>

<div class="details-svg" id="r6c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-392"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-392">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="scaffold_6_r6c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-393"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-393">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-tigrfam-details-svg" data-filename="scaffold_6_r6c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c1-nrps_pks_products" data-name="r6c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c1-nrps_pks_monomers" data-name="r6c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-388"></div> <span class="help-tooltip" id="nrps-structures-help-388">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 59934 - 106303:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-389"></div> <span class="help-tooltip" id="nrps-monomers-help-389">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene22</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene22 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (639..935)</strong>: mal
<div class="collapser collapser-target-input-path1-gene22 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene22-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 87.5%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
<dd>inactive: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene22-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 89.8</dt>

<dd></dd><dt>Methylmalonyl-CoA: 71.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 66.8</dt>

<dd></dd><dt>inactive: 51.3</dt>

<dd></dd><dt>Propionyl-CoA: 38.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 30.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 26.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 26.3</dt>

<dd></dd><dt>CHC-CoA: 26.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 25.6</dt>

<dd></dd><dt>Benzoyl-CoA: 18.5</dt>

<dd></dd><dt>Acetyl-CoA: 16.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 15.7</dt>

<dd></dd><dt>fatty_acid: 15.7</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-554"></div> <span class="help-tooltip" id="region_svg-help-554">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c2-svg" data-filename="scaffold_6_r6c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 146,604 - 200,634 nt. (total: 54,031 nt)

<a class="cluster-rules-header" id="r6c2-rules-header" href="#r6c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-555"></div> <span class="help-tooltip" id="focus-panel-help-555">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c2-gene-table-details" data-name="r6c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c2-nrps_pks" data-name="r6c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c2-MIBiG-cluster-compare" data-name="r6c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c2-clusterblast" data-name="r6c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c2-knownclusterblast" data-name="r6c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c2-subclusterblast" data-name="r6c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c2-nrps-pks-bubbles" data-name="r6c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c2-pfam-details" data-name="r6c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c2-tigrfam-details" data-name="r6c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-402"></div> <span class="help-tooltip" id="gt-help-tooltip-help-402">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-394"></div> <span class="help-tooltip" id="nrps-domain-help-394">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-details-svg" data-filename="scaffold_6_r6c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-395"></div> <span class="help-tooltip" id="MIBiG-help-395">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c2"></div>
<div id="comparison-MIBiG-r6c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.53, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.53, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>pseudomonine</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>KK-1</td>


<td>Curvularia clavata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.53, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.53, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>pseudomonine</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>KK-1</td>
<td>Curvularia clavata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-396"></div> <span class="help-tooltip" id="cb-general-help-396">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c2_1.svg">
NW_001820829 (1536004-1610195): Sclerotinia sclerotiorum 1980 UF-70 scaffold ... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_2.svg">
NC_049561 (2485042-2559251): Talaromyces rugulosus chromosome I, complete seq... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_3.svg">
NW_024467859 (5395537-5469740): Aspergillus sydowii CBS 593.65 unplaced genom... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_4.svg">
NC_070554 (2627617-2701809): Fusarium falciforme chromosome 11, complete sequ... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_5.svg">
NW_024467525 (4378280-4483078): Aspergillus versicolor CBS 583.65 unplaced ge... (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c2_6.svg">
NC_054858 (33608-143759): Aspergillus puulaauensis MK2 DNA, chromosome 2, nea... (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c2_7.svg">
NC_072849 (2666227-2740414): Fusarium oxysporum Fo47 chromosome X, complete s... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_8.svg">
NC_063022 (1445291-1519509): Fulvia fulva chromosome 11, complete sequence (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_9.svg">
NW_022985570 (41698-115964): Pseudovirgaria hyperparasitica strain CBS 121739... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_10.svg">
NW_024467835 (160214-260681): Aspergillus wentii DTO 134E9 unplaced genomic s... (5% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r6c2-download">Download graphic</button>
</div>
<div id="clusterblast-r6c2-svg"></div>

</div>
</div>

<div class="body-details-section r6c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-397"></div> <span class="help-tooltip" id="cb-known-help-397">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c2_1.svg">
epichloenin A
</option>

<option value = "svg/knownclusterblast_r6c2_2.svg">
epichloenin A
</option>

</select>
<button id="knownclusterblast-r6c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c2-svg"></div>

</div>
</div>

<div class="body-details-section r6c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-398"></div> <span class="help-tooltip" id="cb-sub-help-398">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-401"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-401">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-domain-bubble-svg" data-filename="scaffold_6_r6c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (146603 - 200634): single NRPS
</option>


</select>

<div class="details-svg" id="r6c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-403"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-403">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-pfam-details-svg" data-filename="scaffold_6_r6c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-404"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-404">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-tigrfam-details-svg" data-filename="scaffold_6_r6c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c2-nrps_pks_products" data-name="r6c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c2-nrps_pks_monomers" data-name="r6c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-399"></div> <span class="help-tooltip" id="nrps-structures-help-399">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 146603 - 200634:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c2-2" class="smiles-canvas" data-smiles="NC(CCC(=O)N)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gln)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCC(=O)N)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-400"></div> <span class="help-tooltip" id="nrps-monomers-help-400">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene49</span></strong>: X - Gln - X
<div class="collapser collapser-target-input-path1-gene49 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (217..633)</strong>: X
<div class="collapser collapser-target-input-path1-gene49 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene49-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMTLWGGIFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Cyc-D-Gly <span class="serif">DAYLWGGVFK</span> (59% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>





<dd><strong>AMP-binding (1328..1739)</strong>: Gln
<div class="collapser collapser-target-input-path1-gene49 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gln
<div class="collapser collapser-target-input-path1-gene49-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVRSTGVINK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gln <span class="serif">DACSTGVINK</span> (85% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>





<dd><strong>AMP-binding (2380..2787)</strong>: X
<div class="collapser collapser-target-input-path1-gene49 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene49-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVGGAGAVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>E-ahmohOrn <span class="serif">DVGGGGVIGK</span> (91% 8Ã match)</dd>
<dd>Trp <span class="serif">DVGMVGAVAK</span> (50% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>














</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 3 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-556"></div> <span class="help-tooltip" id="region_svg-help-556">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c3-svg" data-filename="scaffold_6_r6c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 264,312 - 330,763 nt. (total: 66,452 nt)

<a class="cluster-rules-header" id="r6c3-rules-header" href="#r6c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c3-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r6c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-557"></div> <span class="help-tooltip" id="focus-panel-help-557">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c3-gene-table-details" data-name="r6c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c3-MIBiG-cluster-compare" data-name="r6c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c3-clusterblast" data-name="r6c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c3-knownclusterblast" data-name="r6c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c3-subclusterblast" data-name="r6c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c3-pfam-details" data-name="r6c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c3-tigrfam-details" data-name="r6c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-409"></div> <span class="help-tooltip" id="gt-help-tooltip-help-409">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-405"></div> <span class="help-tooltip" id="MIBiG-help-405">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c3"></div>
<div id="comparison-MIBiG-r6c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002609: 249-18325">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002609/index.html#r1c1" target="_new">BGC0002609</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>ochratoxin A</td>


<td>Aspergillus welwitschiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002298: 82-4971">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002298/index.html#r1c1" target="_new">BGC0002298</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>lassomycin-like peptide</td>


<td>Sanguibacter keddieii DSM 10542</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>shearinine D</td>


<td>Penicillium janthinellum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>


<td>Fusarium heterosporum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002268: 0-26854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002268/index.html#r1c1" target="_new">BGC0002268</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Polyketide, NRP</td>

<td>CJ-16,173, CJ-15,696, citridone B, citridone A, citridone B&#39;</td>


<td>Penicillium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002062: 1607-32928">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002062/index.html#r1c1" target="_new">BGC0002062</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>agnestin A, agnestin B</td>


<td>Paecilomyces divaricatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000372: 0-22264">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000372/index.html#r1c1" target="_new">BGC0000372</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>hexadehydroastechrome, terezine-D, astechrome</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001677: 0-23325">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001677/index.html#r1c1" target="_new">BGC0001677</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>mycophenolic acid</td>


<td>Penicillium roqueforti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002616: 0-23320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002616/index.html#r1c1" target="_new">BGC0002616</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>mycophenolic acid</td>


<td>Penicillium roqueforti</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002609: 249-18325">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002609/index.html#r1c1" target="_new">BGC0002609</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Polyketide</td>
<td>ochratoxin A</td>
<td>Aspergillus welwitschiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002298: 82-4971">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002298/index.html#r1c1" target="_new">BGC0002298</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>lassomycin-like peptide</td>
<td>Sanguibacter keddieii DSM 10542</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Terpene</td>
<td>shearinine D</td>
<td>Penicillium janthinellum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP, Polyketide</td>
<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>
<td>Fusarium heterosporum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002268: 0-26854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002268/index.html#r1c1" target="_new">BGC0002268</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide, NRP</td>
<td>CJ-16,173, CJ-15,696, citridone B, citridone A, citridone B&#39;</td>
<td>Penicillium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002062: 1607-32928">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002062/index.html#r1c1" target="_new">BGC0002062</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>agnestin A, agnestin B</td>
<td>Paecilomyces divaricatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000372: 0-22264">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000372/index.html#r1c1" target="_new">BGC0000372</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>NRP</td>
<td>hexadehydroastechrome, terezine-D, astechrome</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001677: 0-23325">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001677/index.html#r1c1" target="_new">BGC0001677</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>mycophenolic acid</td>
<td>Penicillium roqueforti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002616: 0-23320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002616/index.html#r1c1" target="_new">BGC0002616</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>mycophenolic acid</td>
<td>Penicillium roqueforti</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-406"></div> <span class="help-tooltip" id="cb-general-help-406">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c3_1.svg">
NW_025953793 (855062-949334): Annulohypoxylon truncatum strain CBS 140777 unp... (19% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_2.svg">
NW_024468911 (2241962-2337395): Cryphonectria parasitica EP155 unplaced genom... (13% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_3.svg">
NW_020291494 (131618-251935): Aspergillus aculeatinus CBS 121060 unplaced gen... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_4.svg">
NW_006271975 (2582207-2649723): Cordyceps militaris CM01 unplaced genomic sca... (29% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c3_5.svg">
NW_001820812 (429792-524980): Sclerotinia sclerotiorum 1980 UF-70 scaffold 24... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_6.svg">
NW_024469024 (26525-267638): Aspergillus fijiensis CBS 313.89 unplaced genomi... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r6c3_7.svg">
NW_022985024 (157613-272496): Dothidotthia symphoricarpi CBS 119687 unplaced ... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_8.svg">
NW_020291814 (6202-188758): Aspergillus costaricaensis CBS 115574 unplaced ge... (2% of genes show similarity), NRP-metallophore,NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r6c3_9.svg">
NC_064650 (2277246-2380079): Penicillium oxalicum strain HP7-1 chromosome I, ... (6% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_10.svg">
NW_024468998 (250549-341477): Aspergillus fijiensis CBS 313.89 unplaced genom... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c3-download">Download graphic</button>
</div>
<div id="clusterblast-r6c3-svg"></div>

</div>
</div>

<div class="body-details-section r6c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-407"></div> <span class="help-tooltip" id="cb-known-help-407">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-408"></div> <span class="help-tooltip" id="cb-sub-help-408">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-410"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-410">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-pfam-details-svg" data-filename="scaffold_6_r6c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-411"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-411">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-tigrfam-details-svg" data-filename="scaffold_6_r6c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r6c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 4 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-558"></div> <span class="help-tooltip" id="region_svg-help-558">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c4-svg" data-filename="scaffold_6_r6c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,420,791 - 2,454,831 nt. (total: 34,041 nt)

<a class="cluster-rules-header" id="r6c4-rules-header" href="#r6c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c4-rules">NAPAA: (AMP-binding and TIGR02353)</div>

<div class="region-svg-container">
<div id='r6c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-559"></div> <span class="help-tooltip" id="focus-panel-help-559">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c4-gene-table-details" data-name="r6c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c4-nrps_pks" data-name="r6c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c4-MIBiG-cluster-compare" data-name="r6c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c4-clusterblast" data-name="r6c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c4-knownclusterblast" data-name="r6c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c4-subclusterblast" data-name="r6c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c4-nrps-pks-bubbles" data-name="r6c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c4-pfam-details" data-name="r6c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c4-tigrfam-details" data-name="r6c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-420"></div> <span class="help-tooltip" id="gt-help-tooltip-help-420">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-412"></div> <span class="help-tooltip" id="nrps-domain-help-412">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c4-details-svg" data-filename="scaffold_6_r6c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-413"></div> <span class="help-tooltip" id="MIBiG-help-413">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c4"></div>
<div id="comparison-MIBiG-r6c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NAPAA</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>Îµ-Poly-L-lysine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>Î³-poly-L-2,4-diaminobutyric acid</td>


<td>Streptomyces celluloflavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>Îµ-Poly-L-lysine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>Î³-poly-L-2,4-diaminobutyric acid</td>
<td>Streptomyces celluloflavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-414"></div> <span class="help-tooltip" id="cb-general-help-414">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c4_1.svg">
NW_006271975 (1702597-1751671): Cordyceps militaris CM01 unplaced genomic sca... (23% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r6c4_2.svg">
NC_054389 (12430017-12539558): Drechmeria coniospora strain ARSEF 6962 chromo... (7% of genes show similarity), NAPAA,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r6c4_3.svg">
NZ_CP034766 (1418729-1452668): Rhodococcus sp. X156 chromosome, complete genome (6% of genes show similarity), NAPAA,betalactone
</option>

<option value = "svg/clusterblast_r6c4_4.svg">
NC_057321 (2571856-2620759): Ustilaginoidea virens chromosome 6, complete seq... (14% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r6c4_5.svg">
NZ_CP116355 (1431563-1465472): Streptomyces xanthophaeus strain KPP03845 chro... (8% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r6c4_6.svg">
NZ_CP092183 (3421851-3593095): Couchioplanes caeruleus strain TL65 chromosome... (1% of genes show similarity), NAPAA,PKS-like,T1PKS,T2PKS,terpene
</option>

<option value = "svg/clusterblast_r6c4_7.svg">
NZ_CP087710 (63425-97325): Nocardioides okcheonensis strain MMS20-HV4-12 chro... (6% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r6c4_8.svg">
NZ_CP059399 (5697705-5731719): Nocardia huaxiensis strain WCH-YHL-001 chromos... (6% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r6c4_9.svg">
NZ_CP088007 (1616227-1650241): Nocardia huaxiensis strain BCHNH01 chromosome,... (6% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r6c4_10.svg">
NZ_CP051627 (3428619-3490248): Thermobifida alba strain DSM 43795 chromosome,... (14% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r6c4-download">Download graphic</button>
</div>
<div id="clusterblast-r6c4-svg"></div>

</div>
</div>

<div class="body-details-section r6c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-415"></div> <span class="help-tooltip" id="cb-known-help-415">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c4_1.svg">
Îµ-Poly-L-lysine
</option>

</select>
<button id="knownclusterblast-r6c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c4-svg"></div>

</div>
</div>

<div class="body-details-section r6c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-416"></div> <span class="help-tooltip" id="cb-sub-help-416">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-419"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-419">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c4-domain-bubble-svg" data-filename="scaffold_6_r6c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (2420790 - 2454831): single NAPAA
</option>


</select>

<div class="details-svg" id="r6c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-421"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-421">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c4-pfam-details-svg" data-filename="scaffold_6_r6c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-422"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-422">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c4-tigrfam-details-svg" data-filename="scaffold_6_r6c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c4-nrps_pks_products" data-name="r6c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c4-nrps_pks_monomers" data-name="r6c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-417"></div> <span class="help-tooltip" id="nrps-structures-help-417">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 2420790 - 2454831:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c4-4" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-418"></div> <span class="help-tooltip" id="nrps-monomers-help-418">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene628</span></strong>: X
<div class="collapser collapser-target-input-path1-gene628 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (55..433)</strong>: X
<div class="collapser collapser-target-input-path1-gene628 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene628-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAESIGTVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DAEDIGTVVK</span> (68% 8Ã match)</dd>
<dd>Gln or Tyr <span class="serif">DAESVGTVVK</span> (65% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 5 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-560"></div> <span class="help-tooltip" id="region_svg-help-560">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c5-svg" data-filename="scaffold_6_r6c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,936,656 - 2,958,213 nt. (total: 21,558 nt)

<a class="cluster-rules-header" id="r6c5-rules-header" href="#r6c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c5-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r6c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-561"></div> <span class="help-tooltip" id="focus-panel-help-561">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c5-gene-table-details" data-name="r6c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c5-MIBiG-cluster-compare" data-name="r6c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c5-clusterblast" data-name="r6c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c5-knownclusterblast" data-name="r6c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c5-subclusterblast" data-name="r6c5-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r6c5-pfam-details" data-name="r6c5-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-427"></div> <span class="help-tooltip" id="gt-help-tooltip-help-427">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-423"></div> <span class="help-tooltip" id="MIBiG-help-423">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c5"></div>
<div id="comparison-MIBiG-r6c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>pimara-8(14),15-diene</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.51, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16Î²-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.53, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>sartorypyrone A</td>


<td>Aspergillus felis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.51, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Terpene, Polyketide</td>

<td>subglutinol A, subglutinol B</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.50, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>


<td>Penicillium paxilli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.52, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>chevalone E</td>


<td>Aspergillus versicolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.51, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>shearinine D</td>


<td>Penicillium janthinellum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>pimara-8(14),15-diene</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.51, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16Î²-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.53, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>sartorypyrone A</td>
<td>Aspergillus felis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.51, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Terpene, Polyketide</td>
<td>subglutinol A, subglutinol B</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.50, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Terpene</td>
<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>
<td>Penicillium paxilli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.52, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>chevalone E</td>
<td>Aspergillus versicolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.51, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Terpene</td>
<td>shearinine D</td>
<td>Penicillium janthinellum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-424"></div> <span class="help-tooltip" id="cb-general-help-424">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c5_1.svg">
NW_006271971 (242847-274242): Cordyceps militaris CM01 unplaced genomic scaff... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c5_2.svg">
NC_070533 (2808830-2840090): Fusarium keratoplasticum isolate Fu6.1 chromosom... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c5_3.svg">
NC_070548 (2879257-2910520): Fusarium falciforme chromosome 5, complete sequence (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c5_4.svg">
NW_011942182 (2894477-2966843): Metarhizium robertsii ARSEF 23 MAA Scf 4, who... (13% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r6c5_5.svg">
NW_014574713 (814784-887446): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (15% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r6c5_6.svg">
NW_006711170 (271878-302853): Trichoderma reesei QM6a unplaced genomic scaffo... (42% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c5_7.svg">
NW_024065901 (1622993-1655154): Colletotrichum karsti strain CkLH20 chromosom... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c5_8.svg">
NW_006917094 (1335307-1366507): Pestalotiopsis fici W106-1 unplaced genomic s... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c5_9.svg">
NW_021167089 (220979-252224): Sodiomyces alkalinus F11 unplaced genomic scaff... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c5_10.svg">
NC_043743 (1411668-1442907): Pyricularia pennisetigena strain Br36 chromosome... (27% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r6c5-download">Download graphic</button>
</div>
<div id="clusterblast-r6c5-svg"></div>

</div>
</div>

<div class="body-details-section r6c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-425"></div> <span class="help-tooltip" id="cb-known-help-425">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-426"></div> <span class="help-tooltip" id="cb-sub-help-426">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r6c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-428"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-428">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-pfam-details-svg" data-filename="scaffold_6_r6c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r7c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-562"></div> <span class="help-tooltip" id="region_svg-help-562">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c1-svg" data-filename="scaffold_7_r7c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 35,828 nt. (total: 35,828 nt)

<a class="cluster-rules-header" id="r7c1-rules-header" href="#r7c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r7c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r7c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-563"></div> <span class="help-tooltip" id="focus-panel-help-563">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c1-gene-table-details" data-name="r7c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c1-nrps_pks" data-name="r7c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c1-MIBiG-cluster-compare" data-name="r7c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c1-clusterblast" data-name="r7c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c1-knownclusterblast" data-name="r7c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c1-subclusterblast" data-name="r7c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c1-pfam-details" data-name="r7c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-435"></div> <span class="help-tooltip" id="gt-help-tooltip-help-435">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-429"></div> <span class="help-tooltip" id="nrps-domain-help-429">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-details-svg" data-filename="scaffold_7_r7c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-430"></div> <span class="help-tooltip" id="MIBiG-help-430">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c1"></div>
<div id="comparison-MIBiG-r7c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>Other</td>

<td>falcarindiol</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858: 715-19483">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>falcarindiol</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858: 715-19483">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-431"></div> <span class="help-tooltip" id="cb-general-help-431">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c1_1.svg">
NW_019154050 (438681-506939): Pochonia chlamydosporia 170 chromosome Unknown ... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_2.svg">
NW_013566976 (286405-371170): Aureobasidium subglaciale EXF-2481 unplaced gen... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c1_3.svg">
NC_036436 (1264208-1374657): Aspergillus oryzae RIB40 DNA, chromosome 2 (6% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r7c1_4.svg">
NW_024469024 (26525-267638): Aspergillus fijiensis CBS 313.89 unplaced genomi... (2% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r7c1_5.svg">
NW_020290659 (1722-70409): Aspergillus saccharolyticus JOP 1030-1 unplaced ge... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_6.svg">
NW_020291498 (165271-359753): Aspergillus aculeatinus CBS 121060 unplaced gen... (2% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r7c1_7.svg">
NW_025953261 (590289-740142): Annulohypoxylon maeteangense strain CBS 123835 ... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c1_8.svg">
NW_017253360 (3662836-3730996): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_9.svg">
NC_070531 (2061168-2131965): Fusarium keratoplasticum isolate Fu6.1 chromosom... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_10.svg">
NW_017263607 (750023-818389): Mollisia scopiformis unplaced genomic scaffold ... (12% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r7c1-download">Download graphic</button>
</div>
<div id="clusterblast-r7c1-svg"></div>

</div>
</div>

<div class="body-details-section r7c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-432"></div> <span class="help-tooltip" id="cb-known-help-432">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-433"></div> <span class="help-tooltip" id="cb-sub-help-433">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-436"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-436">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-pfam-details-svg" data-filename="scaffold_7_r7c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c1-nrps_pks_monomers" data-name="r7c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r7c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-434"></div> <span class="help-tooltip" id="nrps-monomers-help-434">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene3</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene3 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (346..662)</strong>: pk
<div class="collapser collapser-target-input-path1-gene3 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene3-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Ethylmalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene3-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 87.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 82.3</dt>

<dd></dd><dt>Malonyl-CoA: 61.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 50.6</dt>

<dd></dd><dt>Isobutyryl-CoA: 46.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 38.1</dt>

<dd></dd><dt>Propionyl-CoA: 36.7</dt>

<dd></dd><dt>CHC-CoA: 17.8</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.1</dt>

<dd></dd><dt>Benzoyl-CoA: 14.1</dt>

<dd></dd><dt>fatty_acid: 12.1</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r7c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-564"></div> <span class="help-tooltip" id="region_svg-help-564">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c2-svg" data-filename="scaffold_7_r7c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 260,813 - 322,230 nt. (total: 61,418 nt)

<a class="cluster-rules-header" id="r7c2-rules-header" href="#r7c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r7c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-565"></div> <span class="help-tooltip" id="focus-panel-help-565">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c2-gene-table-details" data-name="r7c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c2-MIBiG-cluster-compare" data-name="r7c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c2-clusterblast" data-name="r7c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c2-knownclusterblast" data-name="r7c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c2-subclusterblast" data-name="r7c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c2-pfam-details" data-name="r7c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c2-tigrfam-details" data-name="r7c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-441"></div> <span class="help-tooltip" id="gt-help-tooltip-help-441">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-437"></div> <span class="help-tooltip" id="MIBiG-help-437">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c2"></div>
<div id="comparison-MIBiG-r7c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>RiPP</td>

<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002630: 0-4291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002630/index.html#r1c1" target="_new">BGC0002630</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>thatisin</td>


<td>Lysobacter antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>auriculamide</td>


<td>Herpetosiphon aurantiacus DSM 785</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502: 0-15725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>


<td>Bacillus cereus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527: 0-12324">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>mersacidin</td>


<td>Bacillus sp. HIL-Y85/54728</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000542: 0-9130">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000542/index.html#r1c1" target="_new">BGC0000542</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>paenicidin B</td>


<td>Paenibacillus terrae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289: 16-10643">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>RiPP</td>

<td>penisin</td>


<td>Paenibacillus ehimensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>RiPP</td>
<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002630: 0-4291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002630/index.html#r1c1" target="_new">BGC0002630</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>thatisin</td>
<td>Lysobacter antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>auriculamide</td>
<td>Herpetosiphon aurantiacus DSM 785</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502: 0-15725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
<td>Bacillus cereus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527: 0-12324">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>mersacidin</td>
<td>Bacillus sp. HIL-Y85/54728</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000542: 0-9130">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000542/index.html#r1c1" target="_new">BGC0000542</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>paenicidin B</td>
<td>Paenibacillus terrae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289: 16-10643">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>RiPP</td>
<td>penisin</td>
<td>Paenibacillus ehimensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-438"></div> <span class="help-tooltip" id="cb-general-help-438">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c2_1.svg">
NW_017264196 (550250-647543): Xylona heveae TC161 unplaced genomic scaffold L... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c2_2.svg">
NW_025091265 (422738-551496): Aspergillus udagawae IFM 46973 DNA, scaffold 2,... (4% of genes show similarity), T1PKS,fungal-RiPP-like,indole
</option>

<option value = "svg/clusterblast_r7c2_3.svg">
NW_006271972 (3966194-4059400): Cordyceps militaris CM01 unplaced genomic sca... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c2_4.svg">
NW_026249860 (525987-616886): Alternaria incomplexa strain BMP 0042 chromosom... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c2_5.svg">
NW_026249944 (919773-1010501): Alternaria arbusti strain BMP 1465 chromosome ... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c2_6.svg">
NW_006917092 (5143447-5234370): Pestalotiopsis fici W106-1 unplaced genomic s... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c2_7.svg">
NW_020291606 (579604-818783): Aspergillus piperis CBS 112811 unplaced genomic... (2% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c2_8.svg">
NW_014013658 (1705881-1809848): Trichoderma virens Gv29-8 chromosome Unknown ... (5% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c2_9.svg">
NC_049566 (4842785-4935536): Talaromyces rugulosus chromosome VI, complete se... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c2_10.svg">
NW_011942171 (314568-405746): Metarhizium robertsii ARSEF 23 MAA Scf 3, whole... (6% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r7c2-download">Download graphic</button>
</div>
<div id="clusterblast-r7c2-svg"></div>

</div>
</div>

<div class="body-details-section r7c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-439"></div> <span class="help-tooltip" id="cb-known-help-439">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-440"></div> <span class="help-tooltip" id="cb-sub-help-440">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-442"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-442">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-pfam-details-svg" data-filename="scaffold_7_r7c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-443"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-443">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-tigrfam-details-svg" data-filename="scaffold_7_r7c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r7c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-566"></div> <span class="help-tooltip" id="region_svg-help-566">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c3-svg" data-filename="scaffold_7_r7c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,977,612 - 2,031,136 nt. (total: 53,525 nt)

<a class="cluster-rules-header" id="r7c3-rules-header" href="#r7c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r7c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-567"></div> <span class="help-tooltip" id="focus-panel-help-567">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c3-gene-table-details" data-name="r7c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c3-nrps_pks" data-name="r7c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c3-MIBiG-cluster-compare" data-name="r7c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c3-clusterblast" data-name="r7c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c3-knownclusterblast" data-name="r7c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c3-subclusterblast" data-name="r7c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c3-nrps-pks-bubbles" data-name="r7c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c3-pfam-details" data-name="r7c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c3-tigrfam-details" data-name="r7c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-452"></div> <span class="help-tooltip" id="gt-help-tooltip-help-452">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-444"></div> <span class="help-tooltip" id="nrps-domain-help-444">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-details-svg" data-filename="scaffold_7_r7c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-445"></div> <span class="help-tooltip" id="MIBiG-help-445">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c3"></div>
<div id="comparison-MIBiG-r7c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>serinocyclin A, serinocyclin B</td>


<td>Metarhizium robertsii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>serinocyclin A, serinocyclin B</td>
<td>Metarhizium robertsii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-446"></div> <span class="help-tooltip" id="cb-general-help-446">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c3_1.svg">
NW_006271970 (4182342-4255710): Cordyceps militaris CM01 unplaced genomic sca... (26% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c3_2.svg">
NC_030955 (2361593-2450139): Colletotrichum higginsianum IMI 349063 chromosom... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c3_3.svg">
NW_006917099 (1200604-1277164): Pestalotiopsis fici W106-1 unplaced genomic s... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c3_4.svg">
NW_007360987 (190861-288838): Glarea lozoyensis ATCC 20868 chromosome Unknown... (8% of genes show similarity), NRPS,T1PKS,betalactone
</option>

<option value = "svg/clusterblast_r7c3_5.svg">
NC_036442 (316924-392616): Aspergillus oryzae RIB40 DNA, chromosome 8 (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c3_6.svg">
NW_025408676 (151720-222395): Hirsutella rhossiliensis strain HR02 chromosome... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c3_7.svg">
NC_064654 (32337-153510): Penicillium oxalicum strain HP7-1 chromosome V, who... (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_8.svg">
NW_011942149 (5633-135060): Metarhizium robertsii ARSEF 23 MAA Scf 1, whole g... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_9.svg">
NC_016458 (108795-194185): Thermothielavioides terrestris NRRL 8126 chromosom... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c3_10.svg">
NC_070543 (329392-412354): Fusarium keratoplasticum isolate Fu6.1 chromosome ... (11% of genes show similarity), NRPS,indole
</option>

</select>
<button id= "clusterblast-r7c3-download">Download graphic</button>
</div>
<div id="clusterblast-r7c3-svg"></div>

</div>
</div>

<div class="body-details-section r7c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-447"></div> <span class="help-tooltip" id="cb-known-help-447">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-448"></div> <span class="help-tooltip" id="cb-sub-help-448">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-451"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-451">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-domain-bubble-svg" data-filename="scaffold_7_r7c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (1977611 - 2031136): single NRPS
</option>


</select>

<div class="details-svg" id="r7c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-453"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-453">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-pfam-details-svg" data-filename="scaffold_7_r7c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-454"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-454">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-tigrfam-details-svg" data-filename="scaffold_7_r7c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c3-nrps_pks_products" data-name="r7c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c3-nrps_pks_monomers" data-name="r7c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-449"></div> <span class="help-tooltip" id="nrps-structures-help-449">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 1977611 - 2031136:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c3-3" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Leu - Ala - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,leu*,ala*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-450"></div> <span class="help-tooltip" id="nrps-monomers-help-450">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene572</span></strong>: X - Leu - Ala - X
<div class="collapser collapser-target-input-path1-gene572 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,ala*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (30..434)</strong>: X
<div class="collapser collapser-target-input-path1-gene572 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-input-path1-gene572-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVMEVGTPTK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Ã match)</dd>
<dd>Orn <span class="serif">DVMELSSITK</span> (47% 8Ã match)</dd>
<dd>Pro <span class="serif">DVMLVGAVNK</span> (44% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1128..1524)</strong>: Leu
<div class="collapser collapser-target-input-path1-gene572 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-input-path1-gene572-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALDTGMATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Ã match)</dd>
<dd>Phe or Trp <span class="serif">DAMIGGMVTK</span> (47% 8Ã match)</dd>
<dd>Leu <span class="serif">DALYLGMMCK</span> (44% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2212..2615)</strong>: Ala
<div class="collapser collapser-target-input-path1-gene572 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-input-path1-gene572-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLCLHIMTFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3302..3705)</strong>: X
<div class="collapser collapser-target-input-path1-gene572 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene572-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALDTGMATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Ã match)</dd>
<dd>Phe or Trp <span class="serif">DAMIGGMVTK</span> (50% 8Ã match)</dd>
<dd>Leu <span class="serif">DALYLGMMCK</span> (47% 8Ã match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 4 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-568"></div> <span class="help-tooltip" id="region_svg-help-568">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c4-svg" data-filename="scaffold_7_r7c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,292,634 - 2,340,313 nt. (total: 47,680 nt)

<a class="cluster-rules-header" id="r7c4-rules-header" href="#r7c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c4-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r7c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-569"></div> <span class="help-tooltip" id="focus-panel-help-569">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c4-gene-table-details" data-name="r7c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c4-nrps_pks" data-name="r7c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c4-MIBiG-cluster-compare" data-name="r7c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c4-clusterblast" data-name="r7c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c4-knownclusterblast" data-name="r7c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c4-subclusterblast" data-name="r7c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c4-pfam-details" data-name="r7c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-461"></div> <span class="help-tooltip" id="gt-help-tooltip-help-461">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-455"></div> <span class="help-tooltip" id="nrps-domain-help-455">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c4-details-svg" data-filename="scaffold_7_r7c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-456"></div> <span class="help-tooltip" id="MIBiG-help-456">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c4"></div>
<div id="comparison-MIBiG-r7c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858: 715-19483">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.44, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002186: 329-29262">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002186/index.html#r1c1" target="_new">BGC0002186</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.47, order:0.37, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>NRP, Polyketide</td>

<td>fusaristatin A</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.44, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001246: 1296-28428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001246/index.html#r1c1" target="_new">BGC0001246</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.44, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>trans-resorcylide</td>


<td>Sarocladium zeae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Polyketide, NRP</td>

<td>beauveriolide I, beauveriolide III</td>


<td>Cordyceps militaris CM01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.44, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858: 715-19483">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.44, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002186: 329-29262">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002186/index.html#r1c1" target="_new">BGC0002186</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.47, order:0.37, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>NRP, Polyketide</td>
<td>fusaristatin A</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.44, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001246: 1296-28428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001246/index.html#r1c1" target="_new">BGC0001246</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.44, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>trans-resorcylide</td>
<td>Sarocladium zeae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Polyketide, NRP</td>
<td>beauveriolide I, beauveriolide III</td>
<td>Cordyceps militaris CM01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.44, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-457"></div> <span class="help-tooltip" id="cb-general-help-457">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c4_1.svg">
NW_006711164 (601534-669174): Trichoderma reesei QM6a unplaced genomic scaffo... (57% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_2.svg">
NC_035793 (3629294-3696908): Pochonia chlamydosporia 170 chromosome 4, whole ... (50% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_3.svg">
NW_014013660 (102297-169956): Trichoderma virens Gv29-8 chromosome Unknown TR... (56% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_4.svg">
NC_063077 (1099162-1166854): Purpureocillium takamizusanense chromosome 10, c... (57% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_5.svg">
NW_025408670 (2580109-2647727): Hirsutella rhossiliensis strain HR02 chromoso... (50% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_6.svg">
NC_057317 (2166473-2234156): Ustilaginoidea virens chromosome 2, complete seq... (47% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_7.svg">
NC_036627 (3001008-3068498): Fusarium fujikuroi IMI 58289 draft genome, chrom... (42% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_8.svg">
NC_072845 (3069245-3136729): Fusarium oxysporum Fo47 chromosome VI, complete ... (37% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_9.svg">
NC_058392 (1189925-1257407): Fusarium musae strain F31 chromosome 6, whole ge... (46% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c4_10.svg">
NC_070546 (2092593-2160147): Fusarium falciforme chromosome 3, complete sequence (50% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r7c4-download">Download graphic</button>
</div>
<div id="clusterblast-r7c4-svg"></div>

</div>
</div>

<div class="body-details-section r7c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-458"></div> <span class="help-tooltip" id="cb-known-help-458">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-459"></div> <span class="help-tooltip" id="cb-sub-help-459">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-462"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-462">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c4-pfam-details-svg" data-filename="scaffold_7_r7c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c4-nrps_pks_monomers" data-name="r7c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r7c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-460"></div> <span class="help-tooltip" id="nrps-monomers-help-460">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene638</span></strong>: mmal
<div class="collapser collapser-target-input-path1-gene638 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (390..616)</strong>: mmal
<div class="collapser collapser-target-input-path1-gene638 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-input-path1-gene638-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene638-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 49.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 47.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 27.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 24.5</dt>

<dd></dd><dt>Malonyl-CoA: 22.1</dt>

<dd></dd><dt>Propionyl-CoA: 21.9</dt>

<dd></dd><dt>CHC-CoA: 16.6</dt>

<dd></dd><dt>Acetyl-CoA: 13.1</dt>

<dd></dd><dt>Benzoyl-CoA: 12.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 12.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.3</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r7c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 5 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-570"></div> <span class="help-tooltip" id="region_svg-help-570">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c5-svg" data-filename="scaffold_7_r7c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,868,748 - 2,929,547 nt. (total: 60,800 nt)

<a class="cluster-rules-header" id="r7c5-rules-header" href="#r7c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c5-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r7c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-571"></div> <span class="help-tooltip" id="focus-panel-help-571">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c5-gene-table-details" data-name="r7c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c5-MIBiG-cluster-compare" data-name="r7c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c5-clusterblast" data-name="r7c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c5-knownclusterblast" data-name="r7c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c5-subclusterblast" data-name="r7c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c5-pfam-details" data-name="r7c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c5-tigrfam-details" data-name="r7c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-467"></div> <span class="help-tooltip" id="gt-help-tooltip-help-467">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-463"></div> <span class="help-tooltip" id="MIBiG-help-463">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c5"></div>
<div id="comparison-MIBiG-r7c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-Î²-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>Î²-Diketones</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001811: 32-25672">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001811/index.html#r1c1" target="_new">BGC0001811</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>trichodiene-11-one</td>


<td>Fusarium asiaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000930: 2573-28230">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000930/index.html#r1c1" target="_new">BGC0000930</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>deoxynivalenol</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>armillyl orsellinate, 8Î±-hydroxy-6-protoilludene</td>


<td>Armillaria gallica</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-Î²-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>Î²-Diketones</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001811: 32-25672">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001811/index.html#r1c1" target="_new">BGC0001811</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>trichodiene-11-one</td>
<td>Fusarium asiaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000930: 2573-28230">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000930/index.html#r1c1" target="_new">BGC0000930</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>deoxynivalenol</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>armillyl orsellinate, 8Î±-hydroxy-6-protoilludene</td>
<td>Armillaria gallica</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-464"></div> <span class="help-tooltip" id="cb-general-help-464">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c5_1.svg">
NW_020291609 (269731-414026): Aspergillus piperis CBS 112811 unplaced genomic... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r7c5_2.svg">
NW_014574694 (19387-240297): Metarhizium brunneum ARSEF 3297 chromosome Unkno... (5% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c5_3.svg">
NW_011942182 (4343813-4440659): Metarhizium robertsii ARSEF 23 MAA Scf 4, who... (10% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c5_4.svg">
NW_017253360 (1149599-1251306): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c5_5.svg">
NW_011942151 (20096-254382): Metarhizium robertsii ARSEF 23 MAA Scf 11, whole... (3% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r7c5_6.svg">
NW_017971443 (484511-631944): Talaromyces atroroseus strain IBT 11181 chromos... (3% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c5_7.svg">
NW_024467298 (1389016-1495134): Aspergillus ruber CBS 135680 unplaced genomic... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c5_8.svg">
NC_057368 (37469-194929): Aspergillus chevalieri M1 DNA, chromosome 7, nearly... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c5_9.svg">
NC_072850 (2022235-2153706): Fusarium oxysporum Fo47 chromosome XI, complete ... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c5_10.svg">
NC_036629 (2646284-2784115): Fusarium fujikuroi IMI 58289 draft genome, chrom... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r7c5-download">Download graphic</button>
</div>
<div id="clusterblast-r7c5-svg"></div>

</div>
</div>

<div class="body-details-section r7c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-465"></div> <span class="help-tooltip" id="cb-known-help-465">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-466"></div> <span class="help-tooltip" id="cb-sub-help-466">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-468"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-468">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-pfam-details-svg" data-filename="scaffold_7_r7c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-469"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-469">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-tigrfam-details-svg" data-filename="scaffold_7_r7c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/fungi_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://fungismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/fungi_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>