
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>Cordyceps_cateniannulata_FRD24.scaffolds - 49 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/fungi.css">
<meta property="og:title" content="Cordyceps_cateniannulata_FRD24.scaffolds - 8 record(s) - 49 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/fungi_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://fungismash.secondarymetabolites.org/upload/fungi-3844f018-fb9c-4685-a777-654f2d12179e/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://fungismash.secondarymetabolites.org/">
<img src="images/fungi_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://fungismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="Cordyceps_cateniannulata_FRD24.scaffolds.zip">Download all results</a></li>

<li><a href="Cordyceps_cateniannulata_FRD24.scaffolds.gbk">Download GenBank summary file</a></li>
<li><a href="Cordyceps_cateniannulata_FRD24.scaffolds.json">Download JSON results file</a></li>

<li><a href="fungi-3844f018-fb9c-4685-a777-654f2d12179e.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton RiPP fungal-RiPP r1c1">
<a href="#r1c1">1.1</a>
</div>

<div class="regbutton NRPS NRPS r1c2">
<a href="#r1c2">1.2</a>
</div>

<div class="regbutton NRPS NRPS r1c3">
<a href="#r1c3">1.3</a>
</div>

<div class="regbutton hybrid r1c4">
<a href="#r1c4">1.4</a>
</div>

<div class="regbutton hybrid r1c5">
<a href="#r1c5">1.5</a>
</div>

<div class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">1.6</a>
</div>



<div class="regbutton NRPS NRPS r2c1">
<a href="#r2c1">2.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r2c2">
<a href="#r2c2">2.2</a>
</div>

<div class="regbutton other indole r2c3">
<a href="#r2c3">2.3</a>
</div>

<div class="regbutton hybrid r2c4">
<a href="#r2c4">2.4</a>
</div>

<div class="regbutton terpene terpene r2c5">
<a href="#r2c5">2.5</a>
</div>

<div class="regbutton PKS T1PKS r2c6">
<a href="#r2c6">2.6</a>
</div>

<div class="regbutton NRPS NRPS-like r2c7">
<a href="#r2c7">2.7</a>
</div>

<div class="regbutton terpene terpene r2c8">
<a href="#r2c8">2.8</a>
</div>

<div class="regbutton hybrid r2c9">
<a href="#r2c9">2.9</a>
</div>

<div class="regbutton hybrid r2c10">
<a href="#r2c10">2.10</a>
</div>



<div class="regbutton NRPS NRPS-like r3c1">
<a href="#r3c1">3.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r3c2">
<a href="#r3c2">3.2</a>
</div>

<div class="regbutton hybrid r3c3">
<a href="#r3c3">3.3</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r3c4">
<a href="#r3c4">3.4</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r3c5">
<a href="#r3c5">3.5</a>
</div>

<div class="regbutton PKS T1PKS r3c6">
<a href="#r3c6">3.6</a>
</div>

<div class="regbutton other isocyanide r3c7">
<a href="#r3c7">3.7</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r3c8">
<a href="#r3c8">3.8</a>
</div>

<div class="regbutton NRPS NRPS r3c9">
<a href="#r3c9">3.9</a>
</div>

<div class="regbutton hybrid r3c10">
<a href="#r3c10">3.10</a>
</div>



<div class="regbutton NRPS NRPS-like r4c1">
<a href="#r4c1">4.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">4.2</a>
</div>

<div class="regbutton hybrid r4c3">
<a href="#r4c3">4.3</a>
</div>



<div class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">5.1</a>
</div>

<div class="regbutton NRPS NRPS r5c2">
<a href="#r5c2">5.2</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r5c3">
<a href="#r5c3">5.3</a>
</div>

<div class="regbutton PKS T1PKS r5c4">
<a href="#r5c4">5.4</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r6c1">
<a href="#r6c1">6.1</a>
</div>

<div class="regbutton hybrid r6c2">
<a href="#r6c2">6.2</a>
</div>

<div class="regbutton NRPS NRPS r6c3">
<a href="#r6c3">6.3</a>
</div>

<div class="regbutton hybrid r6c4">
<a href="#r6c4">6.4</a>
</div>

<div class="regbutton NRPS NRPS r6c5">
<a href="#r6c5">6.5</a>
</div>

<div class="regbutton NRPS NRPS-like r6c6">
<a href="#r6c6">6.6</a>
</div>

<div class="regbutton NRPS NRPS r6c7">
<a href="#r6c7">6.7</a>
</div>



<div class="regbutton NRPS NRPS r7c1">
<a href="#r7c1">7.1</a>
</div>

<div class="regbutton terpene terpene r7c2">
<a href="#r7c2">7.2</a>
</div>

<div class="regbutton PKS T1PKS r7c3">
<a href="#r7c3">7.3</a>
</div>

<div class="regbutton terpene terpene r7c4">
<a href="#r7c4">7.4</a>
</div>

<div class="regbutton NRPS NRPS r7c5">
<a href="#r7c5">7.5</a>
</div>

<div class="regbutton NRPS r7c6">
<a href="#r7c6">7.6</a>
</div>

<div class="regbutton PKS T1PKS r7c7">
<a href="#r7c7">7.7</a>
</div>



<div class="regbutton NRPS NRPS r8c1">
<a href="#r8c1">8.1</a>
</div>

<div class="regbutton hybrid r8c2">
<a href="#r8c2">8.2</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>scaffold_1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="scaffold_1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton RiPP fungal-RiPP r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp" target="_blank">fungal-RiPP</a>
</td>
<td class="digits">524,832</td>
<td class="digits table-split-left">566,680</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000627/1" target="_blank">ustiloxin B</a></td>
<td>RiPP:Dikaritin</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 31%, #ffffff00 31%)">31%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton NRPS NRPS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">751,113</td>
<td class="digits table-split-left">800,710</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000313/1" target="_blank">beauvericin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton NRPS NRPS r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">801,212</td>
<td class="digits table-split-left">856,665</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000313/1" target="_blank">beauvericin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 30%, #ffffff00 30%)">30%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c4">
<td class="regbutton hybrid r1c4">
<a href="#r1c4">Region&nbsp1.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">1,897,982</td>
<td class="digits table-split-left">1,955,052</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">ε-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c5">
<td class="regbutton hybrid r1c5">
<a href="#r1c5">Region&nbsp1.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,207,850</td>
<td class="digits table-split-left">2,275,850</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c6">
<td class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">Region&nbsp1.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,140,021</td>
<td class="digits table-split-left">3,185,595</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002710/1" target="_blank">metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_2</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="scaffold_2_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton NRPS NRPS r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">308,796</td>
<td class="digits table-split-left">353,700</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton RiPP fungal-RiPP-like r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">720,724</td>
<td class="digits table-split-left">789,034</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton other indole r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">1,195,929</td>
<td class="digits table-split-left">1,216,674</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c4">
<td class="regbutton hybrid r2c4">
<a href="#r2c4">Region&nbsp2.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,326,254</td>
<td class="digits table-split-left">1,394,906</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c5">
<td class="regbutton terpene terpene r2c5">
<a href="#r2c5">Region&nbsp2.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,102,079</td>
<td class="digits table-split-left">2,123,663</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c6">
<td class="regbutton PKS T1PKS r2c6">
<a href="#r2c6">Region&nbsp2.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,224,468</td>
<td class="digits table-split-left">2,294,753</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c7">
<td class="regbutton NRPS NRPS-like r2c7">
<a href="#r2c7">Region&nbsp2.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">2,360,062</td>
<td class="digits table-split-left">2,403,928</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c8">
<td class="regbutton terpene terpene r2c8">
<a href="#r2c8">Region&nbsp2.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,629,960</td>
<td class="digits table-split-left">2,653,418</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001248/1" target="_blank">clavaric acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c9">
<td class="regbutton hybrid r2c9">
<a href="#r2c9">Region&nbsp2.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,774,261</td>
<td class="digits table-split-left">2,826,282</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c10">
<td class="regbutton hybrid r2c10">
<a href="#r2c10">Region&nbsp2.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,669,093</td>
<td class="digits table-split-left">4,731,138</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_3</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="scaffold_3_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton NRPS NRPS-like r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">198,793</td>
<td class="digits table-split-left">244,284</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton RiPP fungal-RiPP-like r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">835,515</td>
<td class="digits table-split-left">897,274</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton hybrid r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">958,534</td>
<td class="digits table-split-left">1,011,036</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton RiPP fungal-RiPP-like r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,645,613</td>
<td class="digits table-split-left">1,706,689</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton RiPP fungal-RiPP-like r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">2,103,594</td>
<td class="digits table-split-left">2,164,593</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c6">
<td class="regbutton PKS T1PKS r3c6">
<a href="#r3c6">Region&nbsp3.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,298,444</td>
<td class="digits table-split-left">2,342,900</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c7">
<td class="regbutton other isocyanide r3c7">
<a href="#r3c7">Region&nbsp3.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide" target="_blank">isocyanide</a>
</td>
<td class="digits">3,381,358</td>
<td class="digits table-split-left">3,423,497</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c8">
<td class="regbutton RiPP fungal-RiPP-like r3c8">
<a href="#r3c8">Region&nbsp3.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,827,689</td>
<td class="digits table-split-left">3,888,385</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c9">
<td class="regbutton NRPS NRPS r3c9">
<a href="#r3c9">Region&nbsp3.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,052,500</td>
<td class="digits table-split-left">4,096,745</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c10">
<td class="regbutton hybrid r3c10">
<a href="#r3c10">Region&nbsp3.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide-nrp" target="_blank">isocyanide-nrp</a>
</td>
<td class="digits">4,402,068</td>
<td class="digits table-split-left">4,530,840</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001218/1" target="_blank">fumosorinone</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_4</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="scaffold_4_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton NRPS NRPS-like r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">248,399</td>
<td class="digits table-split-left">293,974</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,417,666</td>
<td class="digits table-split-left">1,478,319</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c3">
<td class="regbutton hybrid r4c3">
<a href="#r4c3">Region&nbsp4.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,938,255</td>
<td class="digits table-split-left">4,057,426</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002258/1" target="_blank">shimalactone A/shimalactone B</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 42%, #ffffff00 42%)">42%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_5</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="scaffold_5_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">50,556</td>
<td class="digits table-split-left">98,255</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton NRPS NRPS r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">463,589</td>
<td class="digits table-split-left">516,413</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton RiPP fungal-RiPP-like r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">578,855</td>
<td class="digits table-split-left">639,595</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c4">
<td class="regbutton PKS T1PKS r5c4">
<a href="#r5c4">Region&nbsp5.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,851,347</td>
<td class="digits table-split-left">1,898,221</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_6</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="scaffold_6_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton RiPP fungal-RiPP-like r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">57,797</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton hybrid r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">292,495</td>
<td class="digits table-split-left">380,465</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c3">
<td class="regbutton NRPS NRPS r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">864,414</td>
<td class="digits table-split-left">909,766</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c4">
<td class="regbutton hybrid r6c4">
<a href="#r6c4">Region&nbsp6.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">966,154</td>
<td class="digits table-split-left">1,011,305</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001044/1" target="_blank">sirodesmin PL</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c5">
<td class="regbutton NRPS NRPS r6c5">
<a href="#r6c5">Region&nbsp6.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,397,566</td>
<td class="digits table-split-left">1,453,710</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c6">
<td class="regbutton NRPS NRPS-like r6c6">
<a href="#r6c6">Region&nbsp6.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,458,851</td>
<td class="digits table-split-left">3,502,639</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c7">
<td class="regbutton NRPS NRPS r6c7">
<a href="#r6c7">Region&nbsp6.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,511,972</td>
<td class="digits table-split-left">3,565,093</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001250/1" target="_blank">epichloenin A</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_7</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-7" data-filename="scaffold_7_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-7" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r7c1">
<td class="regbutton NRPS NRPS r7c1">
<a href="#r7c1">Region&nbsp7.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,607</td>
<td class="digits table-split-left">51,243</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c2">
<td class="regbutton terpene terpene r7c2">
<a href="#r7c2">Region&nbsp7.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">120,190</td>
<td class="digits table-split-left">144,256</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001811/1" target="_blank">trichodiene-11-one</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c3">
<td class="regbutton PKS T1PKS r7c3">
<a href="#r7c3">Region&nbsp7.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">534,078</td>
<td class="digits table-split-left">581,937</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001600/1" target="_blank">fusarielin H</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row even" data-anchor="#r7c4">
<td class="regbutton terpene terpene r7c4">
<a href="#r7c4">Region&nbsp7.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">700,447</td>
<td class="digits table-split-left">721,871</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c5">
<td class="regbutton NRPS NRPS r7c5">
<a href="#r7c5">Region&nbsp7.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">996,901</td>
<td class="digits table-split-left">1,046,628</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c6">
<td class="regbutton NRPS r7c6">
<a href="#r7c6">Region&nbsp7.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,735,616</td>
<td class="digits table-split-left">1,795,950</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c7">
<td class="regbutton PKS T1PKS r7c7">
<a href="#r7c7">Region&nbsp7.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,231,056</td>
<td class="digits table-split-left">2,280,851</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_8</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-8" data-filename="scaffold_8_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-8" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r8c1">
<td class="regbutton NRPS NRPS r8c1">
<a href="#r8c1">Region&nbsp8.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,913,883</td>
<td class="digits table-split-left">1,959,398</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r8c2">
<td class="regbutton hybrid r8c2">
<a href="#r8c2">Region&nbsp8.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">2,125,889</td>
<td class="digits table-split-left">2,201,955</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton RiPP fungal-RiPP r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp" target="_blank">fungal-RiPP</a>
</td>
<td class="digits">524,832</td>
<td class="digits table-split-left">566,680</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000627/1" target="_blank">ustiloxin B</a></td>
<td>RiPP:Dikaritin</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 31%, #ffffff00 31%)">31%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton NRPS NRPS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">751,113</td>
<td class="digits table-split-left">800,710</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000313/1" target="_blank">beauvericin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton NRPS NRPS r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">801,212</td>
<td class="digits table-split-left">856,665</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000313/1" target="_blank">beauvericin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 30%, #ffffff00 30%)">30%</td>

</tr>
<tr class="linked-row even" data-anchor="#r1c4">
<td class="regbutton hybrid r1c4">
<a href="#r1c4">Region&nbsp1.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">1,897,982</td>
<td class="digits table-split-left">1,955,052</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">ε-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c5">
<td class="regbutton hybrid r1c5">
<a href="#r1c5">Region&nbsp1.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,207,850</td>
<td class="digits table-split-left">2,275,850</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c6">
<td class="regbutton NRPS NRPS r1c6">
<a href="#r1c6">Region&nbsp1.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,140,021</td>
<td class="digits table-split-left">3,185,595</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002710/1" target="_blank">metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton NRPS NRPS r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">308,796</td>
<td class="digits table-split-left">353,700</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton RiPP fungal-RiPP-like r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">720,724</td>
<td class="digits table-split-left">789,034</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton other indole r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">1,195,929</td>
<td class="digits table-split-left">1,216,674</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c4">
<td class="regbutton hybrid r2c4">
<a href="#r2c4">Region&nbsp2.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,326,254</td>
<td class="digits table-split-left">1,394,906</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c5">
<td class="regbutton terpene terpene r2c5">
<a href="#r2c5">Region&nbsp2.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,102,079</td>
<td class="digits table-split-left">2,123,663</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c6">
<td class="regbutton PKS T1PKS r2c6">
<a href="#r2c6">Region&nbsp2.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,224,468</td>
<td class="digits table-split-left">2,294,753</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c7">
<td class="regbutton NRPS NRPS-like r2c7">
<a href="#r2c7">Region&nbsp2.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">2,360,062</td>
<td class="digits table-split-left">2,403,928</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c8">
<td class="regbutton terpene terpene r2c8">
<a href="#r2c8">Region&nbsp2.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,629,960</td>
<td class="digits table-split-left">2,653,418</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001248/1" target="_blank">clavaric acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c9">
<td class="regbutton hybrid r2c9">
<a href="#r2c9">Region&nbsp2.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,774,261</td>
<td class="digits table-split-left">2,826,282</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c10">
<td class="regbutton hybrid r2c10">
<a href="#r2c10">Region&nbsp2.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,669,093</td>
<td class="digits table-split-left">4,731,138</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton NRPS NRPS-like r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">198,793</td>
<td class="digits table-split-left">244,284</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton RiPP fungal-RiPP-like r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">835,515</td>
<td class="digits table-split-left">897,274</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton hybrid r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">958,534</td>
<td class="digits table-split-left">1,011,036</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton RiPP fungal-RiPP-like r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,645,613</td>
<td class="digits table-split-left">1,706,689</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton RiPP fungal-RiPP-like r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">2,103,594</td>
<td class="digits table-split-left">2,164,593</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c6">
<td class="regbutton PKS T1PKS r3c6">
<a href="#r3c6">Region&nbsp3.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,298,444</td>
<td class="digits table-split-left">2,342,900</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c7">
<td class="regbutton other isocyanide r3c7">
<a href="#r3c7">Region&nbsp3.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide" target="_blank">isocyanide</a>
</td>
<td class="digits">3,381,358</td>
<td class="digits table-split-left">3,423,497</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c8">
<td class="regbutton RiPP fungal-RiPP-like r3c8">
<a href="#r3c8">Region&nbsp3.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,827,689</td>
<td class="digits table-split-left">3,888,385</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c9">
<td class="regbutton NRPS NRPS r3c9">
<a href="#r3c9">Region&nbsp3.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,052,500</td>
<td class="digits table-split-left">4,096,745</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c10">
<td class="regbutton hybrid r3c10">
<a href="#r3c10">Region&nbsp3.10</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide-nrp" target="_blank">isocyanide-nrp</a>
</td>
<td class="digits">4,402,068</td>
<td class="digits table-split-left">4,530,840</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001218/1" target="_blank">fumosorinone</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton NRPS NRPS-like r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">248,399</td>
<td class="digits table-split-left">293,974</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,417,666</td>
<td class="digits table-split-left">1,478,319</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c3">
<td class="regbutton hybrid r4c3">
<a href="#r4c3">Region&nbsp4.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,938,255</td>
<td class="digits table-split-left">4,057,426</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002258/1" target="_blank">shimalactone A/shimalactone B</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 42%, #ffffff00 42%)">42%</td>

</tr>


<tr class="linked-row even" data-anchor="#r5c1">
<td class="regbutton PKS T1PKS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">50,556</td>
<td class="digits table-split-left">98,255</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c2">
<td class="regbutton NRPS NRPS r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">463,589</td>
<td class="digits table-split-left">516,413</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c3">
<td class="regbutton RiPP fungal-RiPP-like r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">578,855</td>
<td class="digits table-split-left">639,595</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c4">
<td class="regbutton PKS T1PKS r5c4">
<a href="#r5c4">Region&nbsp5.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,851,347</td>
<td class="digits table-split-left">1,898,221</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r6c1">
<td class="regbutton RiPP fungal-RiPP-like r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">57,797</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c2">
<td class="regbutton hybrid r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">292,495</td>
<td class="digits table-split-left">380,465</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c3">
<td class="regbutton NRPS NRPS r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">864,414</td>
<td class="digits table-split-left">909,766</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c4">
<td class="regbutton hybrid r6c4">
<a href="#r6c4">Region&nbsp6.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">966,154</td>
<td class="digits table-split-left">1,011,305</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001044/1" target="_blank">sirodesmin PL</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c5">
<td class="regbutton NRPS NRPS r6c5">
<a href="#r6c5">Region&nbsp6.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,397,566</td>
<td class="digits table-split-left">1,453,710</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c6">
<td class="regbutton NRPS NRPS-like r6c6">
<a href="#r6c6">Region&nbsp6.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,458,851</td>
<td class="digits table-split-left">3,502,639</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c7">
<td class="regbutton NRPS NRPS r6c7">
<a href="#r6c7">Region&nbsp6.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,511,972</td>
<td class="digits table-split-left">3,565,093</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001250/1" target="_blank">epichloenin A</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r7c1">
<td class="regbutton NRPS NRPS r7c1">
<a href="#r7c1">Region&nbsp7.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,607</td>
<td class="digits table-split-left">51,243</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c2">
<td class="regbutton terpene terpene r7c2">
<a href="#r7c2">Region&nbsp7.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">120,190</td>
<td class="digits table-split-left">144,256</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001811/1" target="_blank">trichodiene-11-one</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c3">
<td class="regbutton PKS T1PKS r7c3">
<a href="#r7c3">Region&nbsp7.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">534,078</td>
<td class="digits table-split-left">581,937</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001600/1" target="_blank">fusarielin H</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row even" data-anchor="#r7c4">
<td class="regbutton terpene terpene r7c4">
<a href="#r7c4">Region&nbsp7.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">700,447</td>
<td class="digits table-split-left">721,871</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c5">
<td class="regbutton NRPS NRPS r7c5">
<a href="#r7c5">Region&nbsp7.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">996,901</td>
<td class="digits table-split-left">1,046,628</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c6">
<td class="regbutton NRPS r7c6">
<a href="#r7c6">Region&nbsp7.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,735,616</td>
<td class="digits table-split-left">1,795,950</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c7">
<td class="regbutton PKS T1PKS r7c7">
<a href="#r7c7">Region&nbsp7.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,231,056</td>
<td class="digits table-split-left">2,280,851</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r8c1">
<td class="regbutton NRPS NRPS r8c1">
<a href="#r8c1">Region&nbsp8.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,913,883</td>
<td class="digits table-split-left">1,959,398</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r8c2">
<td class="regbutton hybrid r8c2">
<a href="#r8c2">Region&nbsp8.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">2,125,889</td>
<td class="digits table-split-left">2,201,955</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>scaffold_9</strong></dd>

<dd><strong>scaffold_10</strong></dd>

<dd><strong>scaffold_11</strong></dd>

<dd><strong>scaffold_12</strong></dd>

<dd><strong>scaffold_13</strong></dd>

<dd><strong>scaffold_14</strong></dd>

<dd><strong>scaffold_15</strong></dd>

<dd><strong>scaffold_16</strong></dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 1 - fungal-RiPP <div class="help-container"> <div class="help-icon" data-id="region_svg-help-454"></div> <span class="help-tooltip" id="region_svg-help-454">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="scaffold_1_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 524,832 - 566,680 nt. (total: 41,849 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c1-rules">fungal-RiPP: ((pop or ustH) and (p450 or TP_methylase) or DUF3328 and (Peptidase_S41 or Peptidase_S28 or pop or ustH))</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-455"></div> <span class="help-tooltip" id="focus-panel-help-455">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-5"></div> <span class="help-tooltip" id="gt-help-tooltip-help-5">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002219: 0-26397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002219/index.html#r1c1" target="_new">BGC0002219</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>(-)-(R)-nephthenol</td>


<td>Stereum hirsutum FP-91666 SS1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000584: 0-3682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000584/index.html#r1c1" target="_new">BGC0000584</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>colicin V</td>


<td>Escherichia coli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890: 0-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Other (Aminocoumarin)</td>

<td>pentabromopseudilin</td>


<td>Pseudoalteromonas phenolica O-BC30</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465: 0-12220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Other</td>

<td>bromophene, bistribromopyrrole, pentabromopseudilin</td>


<td>Marinomonas mediterranea MMB-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.49, order:0.27, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001518: 0-23499">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001518/index.html#r1c1" target="_new">BGC0001518</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.42, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>astellolide A</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.50, order:0.07, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002046: 0-69559">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002046/index.html#r1c1" target="_new">BGC0002046</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>NRP, Polyketide</td>

<td>lobatamide A</td>


<td>Gynuella sunshinyii YC6258</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001625: 0-22706">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001625/index.html#r1c1" target="_new">BGC0001625</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>isofuranonaphthoquinone</td>


<td>Streptomyces sp. CB01883</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001386: 193-25212">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001386/index.html#r1c1" target="_new">BGC0001386</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>JBIR-76, JBIR-77</td>


<td>Streptomyces sp. RI-77</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002219: 0-26397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002219/index.html#r1c1" target="_new">BGC0002219</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>(-)-(R)-nephthenol</td>
<td>Stereum hirsutum FP-91666 SS1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000584: 0-3682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000584/index.html#r1c1" target="_new">BGC0000584</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>colicin V</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890: 0-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Other (Aminocoumarin)</td>
<td>pentabromopseudilin</td>
<td>Pseudoalteromonas phenolica O-BC30</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465: 0-12220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Other</td>
<td>bromophene, bistribromopyrrole, pentabromopseudilin</td>
<td>Marinomonas mediterranea MMB-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.49, order:0.27, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001518: 0-23499">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001518/index.html#r1c1" target="_new">BGC0001518</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.42, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Terpene</td>
<td>astellolide A</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.50, order:0.07, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002046: 0-69559">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002046/index.html#r1c1" target="_new">BGC0002046</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>NRP, Polyketide</td>
<td>lobatamide A</td>
<td>Gynuella sunshinyii YC6258</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001625: 0-22706">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001625/index.html#r1c1" target="_new">BGC0001625</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>isofuranonaphthoquinone</td>
<td>Streptomyces sp. CB01883</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001386: 193-25212">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001386/index.html#r1c1" target="_new">BGC0001386</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>JBIR-76, JBIR-77</td>
<td>Streptomyces sp. RI-77</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c1_1.svg">
NW_006271970 (340089-399189): Cordyceps militaris CM01 unplaced genomic scaff... (45% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c1_2.svg">
NC_036439 (4081578-4137471): Aspergillus oryzae RIB40 DNA, chromosome 5 (45% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c1_3.svg">
NC_054695 (3982357-4038287): Aspergillus flavus NRRL3357 chromosome 5, comple... (42% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c1_4.svg">
NW_006921536 (1019168-1074511): Pseudocercospora fijiensis CIRAD86 unplaced g... (23% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c1_5.svg">
NW_009799103 (3395741-3458187): Neurospora tetrasperma FGSC 2508 unplaced gen... (16% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c1_6.svg">
NW_011942160 (1251508-1300367): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (20% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c1_7.svg">
NC_063024 (1660782-1725193): Fulvia fulva chromosome 13, complete sequence (12% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c1_8.svg">
NC_018215 (1659185-1716249): Zymoseptoria tritici IPO323 chromosome 4, whole ... (12% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r1c1_9.svg">
NC_030958 (1139459-1267774): Colletotrichum higginsianum IMI 349063 chromosom... (6% of genes show similarity), T1PKS,fungal-RiPP-like,indole
</option>

<option value = "svg/clusterblast_r1c1_10.svg">
NC_049561 (2011040-2059583): Talaromyces rugulosus chromosome I, complete seq... (10% of genes show similarity), fungal-RiPP
</option>

</select>
<button id= "clusterblast-r1c1-download">Download graphic</button>
</div>
<div id="clusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c1_1.svg">
ustiloxin B
</option>

<option value = "svg/knownclusterblast_r1c1_2.svg">
phomopsin A/phomopsin B/phomopsin E
</option>

</select>
<button id="knownclusterblast-r1c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-6"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-6">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="scaffold_1_r1c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r1c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-456"></div> <span class="help-tooltip" id="region_svg-help-456">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c2-svg" data-filename="scaffold_1_r1c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 751,113 - 800,710 nt. (total: 49,598 nt)

<a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-457"></div> <span class="help-tooltip" id="focus-panel-help-457">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c2-gene-table-details" data-name="r1c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c2-nrps_pks" data-name="r1c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c2-nrps-pks-bubbles" data-name="r1c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c2-tigrfam-details" data-name="r1c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-15"></div> <span class="help-tooltip" id="gt-help-tooltip-help-15">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-7"></div> <span class="help-tooltip" id="nrps-domain-help-7">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-details-svg" data-filename="scaffold_1_r1c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-8"></div> <span class="help-tooltip" id="MIBiG-help-8">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c2"></div>
<div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.54, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.61, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000312: 0-41205">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000312/index.html#r1c1" target="_new">BGC0000312</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.56, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>bassianolide</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>AbT1</td>


<td>Aureobasidium pullulans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.54, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.61, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000312: 0-41205">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000312/index.html#r1c1" target="_new">BGC0000312</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.56, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>bassianolide</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>AbT1</td>
<td>Aureobasidium pullulans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-9"></div> <span class="help-tooltip" id="cb-general-help-9">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c2_1.svg">
NC_072848 (2904413-2973827): Fusarium oxysporum Fo47 chromosome IX, complete ... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_2.svg">
NC_036630 (2636149-2705557): Fusarium fujikuroi IMI 58289 draft genome, chrom... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_3.svg">
NC_038015 (4276053-4345437): Fusarium venenatum strain A3/5 genome assembly, ... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_4.svg">
NC_058402 (4355119-4424503): Fusarium poae strain DAOMC 252244 chromosome 4, ... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_5.svg">
NW_026089542 (1840708-1920109): Schizophyllum commune H4-8 unplaced genomic s... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c2_6.svg">
NZ_CP031087 (3156779-3251358): Actinokineospora sp. UTMC 2448 chromosome, com... (4% of genes show similarity), NAPAA,NRPS
</option>

<option value = "svg/clusterblast_r1c2_7.svg">
NZ_BMRB01000010 (37666-131835): Actinokineospora fastidiosa strain JCM 3276 s... (4% of genes show similarity), NAPAA,NRPS
</option>

<option value = "svg/clusterblast_r1c2_8.svg">
NZ_CP115450 (525430-594583): Kitasatospora sp. HUAS 3-15 chromosome, complete... (7% of genes show similarity), NI-siderophore,NRPS
</option>

<option value = "svg/clusterblast_r1c2_9.svg">
NZ_CP088911 (7969830-8075560): Yinghuangia sp. ASG 101 chromosome, complete g... (3% of genes show similarity), NRPS,T2PKS
</option>

<option value = "svg/clusterblast_r1c2_10.svg">
NZ_NNBK01000012 (159469-222554): Streptomyces sp. CB01373 scaffold12, whole g... (7% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c2-download">Download graphic</button>
</div>
<div id="clusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-10"></div> <span class="help-tooltip" id="cb-known-help-10">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c2_1.svg">
beauvericin
</option>

<option value = "svg/knownclusterblast_r1c2_2.svg">
cyclosporin C
</option>

<option value = "svg/knownclusterblast_r1c2_3.svg">
enniatin
</option>

<option value = "svg/knownclusterblast_r1c2_4.svg">
AbT1
</option>

</select>
<button id="knownclusterblast-r1c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-11"></div> <span class="help-tooltip" id="cb-sub-help-11">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-14"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-14">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-domain-bubble-svg" data-filename="scaffold_1_r1c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (751112 - 800710): single NRPS
</option>


</select>

<div class="details-svg" id="r1c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-16"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-16">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="scaffold_1_r1c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-17"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-17">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-tigrfam-details-svg" data-filename="scaffold_1_r1c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c2-nrps_pks_products" data-name="r1c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c2-nrps_pks_monomers" data-name="r1c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-12"></div> <span class="help-tooltip" id="nrps-structures-help-12">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 751112 - 800710:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(Cc1ccccc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - NMe-Phe)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=nme-phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,nme-phe*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(Cc1ccccc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-13"></div> <span class="help-tooltip" id="nrps-monomers-help-13">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene242</span></strong>: X - Phe
<div class="collapser collapser-target-input-path1-gene242 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (482..878)</strong>: X
<div class="collapser collapser-target-input-path1-gene242 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene242-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GALMVVGSVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>D-Hiv <span class="serif">GALMVVGSIK</span> (94% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1565..1972)</strong>: Phe
<div class="collapser collapser-target-input-path1-gene242 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-input-path1-gene242-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGYIMAAVMK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DGYIMAAVMK</span> (97% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-458"></div> <span class="help-tooltip" id="region_svg-help-458">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c3-svg" data-filename="scaffold_1_r1c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 801,212 - 856,665 nt. (total: 55,454 nt)

<a class="cluster-rules-header" id="r1c3-rules-header" href="#r1c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-459"></div> <span class="help-tooltip" id="focus-panel-help-459">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c3-gene-table-details" data-name="r1c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c3-nrps_pks" data-name="r1c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c3-MIBiG-cluster-compare" data-name="r1c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c3-clusterblast" data-name="r1c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c3-knownclusterblast" data-name="r1c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c3-subclusterblast" data-name="r1c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c3-nrps-pks-bubbles" data-name="r1c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c3-pfam-details" data-name="r1c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c3-tigrfam-details" data-name="r1c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-26"></div> <span class="help-tooltip" id="gt-help-tooltip-help-26">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-18"></div> <span class="help-tooltip" id="nrps-domain-help-18">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-details-svg" data-filename="scaffold_1_r1c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-19"></div> <span class="help-tooltip" id="MIBiG-help-19">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c3"></div>
<div id="comparison-MIBiG-r1c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>xenematide</td>


<td>Xenorhabdus nematophila AN6/1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>xenematide</td>
<td>Xenorhabdus nematophila AN6/1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-20"></div> <span class="help-tooltip" id="cb-general-help-20">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c3_1.svg">
NW_006271972 (3966194-4059400): Cordyceps militaris CM01 unplaced genomic sca... (23% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c3_2.svg">
NC_064673 (3815898-3935302): Colletotrichum lupini chromosome 10, complete se... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_3.svg">
NW_022474206 (1449467-1550275): Venustampulla echinocandica strain BP 5553 ch... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_4.svg">
NW_025091281 (4887733-5033357): Aspergillus pseudoviridinutans IFM 55266 DNA,... (5% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c3_5.svg">
NC_043742 (7983415-8058089): Pyricularia pennisetigena strain Br36 chromosome... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_6.svg">
NW_024467540 (77139-192266): Aspergillus versicolor CBS 583.65 unplaced genom... (6% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r1c3_7.svg">
NC_064672 (5757548-5826257): Colletotrichum lupini chromosome 1, complete seq... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_8.svg">
NW_024467876 (196055-261189): Aspergillus sydowii CBS 593.65 unplaced genomic... (6% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r1c3_9.svg">
NC_035793 (899347-971704): Pochonia chlamydosporia 170 chromosome 4, whole ge... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c3_10.svg">
NW_024468995 (361260-526376): Aspergillus fijiensis CBS 313.89 unplaced genom... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r1c3-download">Download graphic</button>
</div>
<div id="clusterblast-r1c3-svg"></div>

</div>
</div>

<div class="body-details-section r1c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-21"></div> <span class="help-tooltip" id="cb-known-help-21">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c3_1.svg">
beauvericin
</option>

</select>
<button id="knownclusterblast-r1c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c3-svg"></div>

</div>
</div>

<div class="body-details-section r1c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-22"></div> <span class="help-tooltip" id="cb-sub-help-22">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-25"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-25">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-domain-bubble-svg" data-filename="scaffold_1_r1c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (801211 - 856665): single NRPS
</option>


</select>

<div class="details-svg" id="r1c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-27"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-27">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-pfam-details-svg" data-filename="scaffold_1_r1c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-28"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-28">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-tigrfam-details-svg" data-filename="scaffold_1_r1c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c3-nrps_pks_products" data-name="r1c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c3-nrps_pks_monomers" data-name="r1c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-23"></div> <span class="help-tooltip" id="nrps-structures-help-23">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 801211 - 856665:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c3-3" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X - X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x,x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-24"></div> <span class="help-tooltip" id="nrps-monomers-help-24">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene257</span></strong>: X - X - X - X
<div class="collapser collapser-target-input-path1-gene257 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (722..1122)</strong>: X
<div class="collapser collapser-target-input-path1-gene257 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-input-path1-gene257-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAYLIACVSK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DALLVGAVSK</span> (65% 8Å match)</dd>
<dd>Tyr <span class="serif">DASTIAAVCK</span> (59% 8Å match)</dd>
<dd>Phe <span class="serif">DAWTIAGVCK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1821..2217)</strong>: X
<div class="collapser collapser-target-input-path1-gene257 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene257-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVSMIGGIYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DVGMVGGIYK</span> (65% 8Å match)</dd>
<dd>Val <span class="serif">DASQIGGIYK</span> (62% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2913..3296)</strong>: X
<div class="collapser collapser-target-input-path1-gene257 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene257-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">FLVIGNLICK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3979..4380)</strong>: X
<div class="collapser collapser-target-input-path1-gene257 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene257-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFLAGACIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Abu <span class="serif">DVALAGVCVK</span> (50% 8Å match)</dd>
<dd>Leu <span class="serif">DVWLLGAVIK</span> (41% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 4 - NAPAA,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-460"></div> <span class="help-tooltip" id="region_svg-help-460">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c4-svg" data-filename="scaffold_1_r1c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,897,982 - 1,955,052 nt. (total: 57,071 nt)

<a class="cluster-rules-header" id="r1c4-rules-header" href="#r1c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c4-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NAPAA: (AMP-binding and TIGR02353)</div>

<div class="region-svg-container">
<div id='r1c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-461"></div> <span class="help-tooltip" id="focus-panel-help-461">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c4-gene-table-details" data-name="r1c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c4-nrps_pks" data-name="r1c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c4-MIBiG-cluster-compare" data-name="r1c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c4-clusterblast" data-name="r1c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c4-knownclusterblast" data-name="r1c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c4-subclusterblast" data-name="r1c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c4-nrps-pks-bubbles" data-name="r1c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c4-pfam-details" data-name="r1c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c4-tigrfam-details" data-name="r1c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-37"></div> <span class="help-tooltip" id="gt-help-tooltip-help-37">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-29"></div> <span class="help-tooltip" id="nrps-domain-help-29">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-details-svg" data-filename="scaffold_1_r1c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-30"></div> <span class="help-tooltip" id="MIBiG-help-30">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c4"></div>
<div id="comparison-MIBiG-r1c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NAPAA</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>






<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.83</td>
<td>NRP</td>

<td>ε-Poly-L-lysine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.51</td>
<td>NRP</td>

<td>γ-poly-L-2,4-diaminobutyric acid</td>


<td>Streptomyces celluloflavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.52, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.48</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>NRP, Saccharide</td>

<td>minimycin</td>


<td>Streptomyces hygroscopicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>ε-Poly-L-lysine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>γ-poly-L-2,4-diaminobutyric acid</td>
<td>Streptomyces celluloflavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>abscisic acid</td>
<td>Leptosphaeria maculans JN3</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>scytalidin</td>
<td>Scytalidium album</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-31"></div> <span class="help-tooltip" id="cb-general-help-31">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c4_1.svg">
NW_006271975 (1702597-1751671): Cordyceps militaris CM01 unplaced genomic sca... (46% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r1c4_2.svg">
NZ_CP031455 (1608048-1824050): Streptomyces olivoreticuli subsp. olivoreticul... (5% of genes show similarity), NAPAA,NRPS,T1PKS,terpene,transAT-PKS
</option>

<option value = "svg/clusterblast_r1c4_3.svg">
NZ_CP023694 (8334466-8506561): Streptomyces coeruleorubidus strain ATCC 13740... (2% of genes show similarity), NI-siderophore,NRPS,NRPS-like,T1PKS,transAT-PKS
</option>

<option value = "svg/clusterblast_r1c4_4.svg">
NZ_CP012752 (5672926-5843995): Kibdelosporangium phytohabitans strain KLBMP11... (4% of genes show similarity), NRPS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r1c4_5.svg">
NZ_CP010407 (740142-846731): Streptomyces vietnamensis strain GIMV4.0001 chro... (5% of genes show similarity), NRPS,T1PKS,butyrolactone
</option>

<option value = "svg/clusterblast_r1c4_6.svg">
NZ_BMUK01000002 (159096-237724): Streptomyces purpurascens strain JCM 4509 se... (4% of genes show similarity), NI-siderophore,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_7.svg">
NZ_AP018268 (1523090-1714633): Scytonema sp. NIES-4073 DNA, nearly complete g... (8% of genes show similarity), NRP-metallophore,NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c4_8.svg">
NW_014013668 (153970-222143): Trichoderma virens Gv29-8 chromosome Unknown TR... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c4_9.svg">
NW_006711154 (931670-999559): Trichoderma reesei QM6a unplaced genomic scaffo... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c4_10.svg">
NW_024467835 (160214-260681): Aspergillus wentii DTO 134E9 unplaced genomic s... (5% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r1c4-download">Download graphic</button>
</div>
<div id="clusterblast-r1c4-svg"></div>

</div>
</div>

<div class="body-details-section r1c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-32"></div> <span class="help-tooltip" id="cb-known-help-32">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c4_1.svg">
ε-Poly-L-lysine
</option>

</select>
<button id="knownclusterblast-r1c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c4-svg"></div>

</div>
</div>

<div class="body-details-section r1c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-33"></div> <span class="help-tooltip" id="cb-sub-help-33">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-36"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-36">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-domain-bubble-svg" data-filename="scaffold_1_r1c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (1897981 - 1955052): neighbouring T1PKS-NAPAA
</option>



<option value="CC5">
Candidate 5 (1897981 - 1946150): single T1PKS
</option>



<option value="CC6">
Candidate 6 (1920993 - 1955052): single NAPAA
</option>


</select>

<div class="details-svg" id="r1c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-38"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-38">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-pfam-details-svg" data-filename="scaffold_1_r1c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-39"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-39">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c4-tigrfam-details-svg" data-filename="scaffold_1_r1c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c4-nrps_pks_products" data-name="r1c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c4-nrps_pks_monomers" data-name="r1c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-34"></div> <span class="help-tooltip" id="nrps-structures-help-34">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 1897981 - 1955052:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c4-4" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 5, location 1897981 - 1946150:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c4-5" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c4-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 6, location 1920993 - 1955052:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c4-6" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c4-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-35"></div> <span class="help-tooltip" id="nrps-monomers-help-35">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene609</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene609 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (379..657)</strong>: pk
<div class="collapser collapser-target-input-path1-gene609 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene609-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene609-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 85.1</dt>

<dd></dd><dt>Malonyl-CoA: 79.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 76.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 70.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 49.8</dt>

<dd></dd><dt>Propionyl-CoA: 43.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 39.4</dt>

<dd></dd><dt>CHC-CoA: 25.3</dt>

<dd></dd><dt>Benzoyl-CoA: 24.7</dt>

<dd></dd><dt>inactive: 21.8</dt>

<dd></dd><dt>fatty_acid: 19.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.6</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

<strong><span class="serif">input.path1.gene613</span></strong>: X
<div class="collapser collapser-target-input-path1-gene613 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (54..431)</strong>: X
<div class="collapser collapser-target-input-path1-gene613 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene613-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAESIGTVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DAEDIGTVVK</span> (68% 8Å match)</dd>
<dd>Gln or Tyr <span class="serif">DAESVGTVVK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 5 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-462"></div> <span class="help-tooltip" id="region_svg-help-462">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c5-svg" data-filename="scaffold_1_r1c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,207,850 - 2,275,850 nt. (total: 68,001 nt)

<a class="cluster-rules-header" id="r1c5-rules-header" href="#r1c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c5-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r1c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-463"></div> <span class="help-tooltip" id="focus-panel-help-463">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c5-gene-table-details" data-name="r1c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c5-nrps_pks" data-name="r1c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c5-MIBiG-cluster-compare" data-name="r1c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c5-clusterblast" data-name="r1c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c5-knownclusterblast" data-name="r1c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c5-subclusterblast" data-name="r1c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c5-nrps-pks-bubbles" data-name="r1c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c5-pfam-details" data-name="r1c5-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-48"></div> <span class="help-tooltip" id="gt-help-tooltip-help-48">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-40"></div> <span class="help-tooltip" id="nrps-domain-help-40">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-details-svg" data-filename="scaffold_1_r1c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-41"></div> <span class="help-tooltip" id="MIBiG-help-41">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c5"></div>
<div id="comparison-MIBiG-r1c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.69</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP, Polyketide</td>

<td>himeic acid A</td>


<td>Aspergillus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide, NRP</td>

<td>pyranoviolin A</td>


<td>Aspergillus violaceofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001563: 1377-52987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001563/index.html#r1c1" target="_new">BGC0001563</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP, Polyketide</td>

<td>curvupallide-B</td>


<td>Cochliobolus pallescens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>himeic acid A</td>
<td>Aspergillus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide, NRP</td>
<td>pyranoviolin A</td>
<td>Aspergillus violaceofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001563: 1377-52987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001563/index.html#r1c1" target="_new">BGC0001563</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>curvupallide-B</td>
<td>Cochliobolus pallescens</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-42"></div> <span class="help-tooltip" id="cb-general-help-42">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c5_1.svg">
NW_006271971 (3503471-3590640): Cordyceps militaris CM01 unplaced genomic sca... (70% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_2.svg">
NW_020290887 (673812-764439): Aspergillus niger CBS 101883 unplaced genomic s... (28% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_3.svg">
NW_023336287 (1029426-1127137): Aspergillus tubingensis WU-2223L DNA, scaffol... (29% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r1c5_4.svg">
NW_020292069 (316400-408453): Aspergillus vadensis CBS 113365 unplaced genomi... (30% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_5.svg">
NW_020291816 (340965-433020): Aspergillus costaricaensis CBS 115574 unplaced ... (29% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_6.svg">
NW_020291625 (99659-196841): Aspergillus piperis CBS 112811 unplaced genomic ... (29% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r1c5_7.svg">
NC_054853 (2119459-2216438): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (25% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r1c5_8.svg">
NW_013581207 (483365-550922): Aureobasidium namibiae CBS 147.97 unplaced geno... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c5_9.svg">
NW_022474207 (2338451-2410505): Venustampulla echinocandica strain BP 5553 ch... (19% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c5_10.svg">
NW_013566983 (689939-757505): Aureobasidium subglaciale EXF-2481 unplaced gen... (15% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r1c5-download">Download graphic</button>
</div>
<div id="clusterblast-r1c5-svg"></div>

</div>
</div>

<div class="body-details-section r1c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-43"></div> <span class="help-tooltip" id="cb-known-help-43">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-44"></div> <span class="help-tooltip" id="cb-sub-help-44">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-47"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-47">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-domain-bubble-svg" data-filename="scaffold_1_r1c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (2207849 - 2275850): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r1c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-49"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-49">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c5-pfam-details-svg" data-filename="scaffold_1_r1c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c5-nrps_pks_products" data-name="r1c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c5-nrps_pks_monomers" data-name="r1c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-45"></div> <span class="help-tooltip" id="nrps-structures-help-45">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 2207849 - 2275850:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c5-7" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c5-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-46"></div> <span class="help-tooltip" id="nrps-monomers-help-46">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene691</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene691 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (547..865)</strong>: pk
<div class="collapser collapser-target-input-path1-gene691 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene691-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene691-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 69.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 61.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 44.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 27.0</dt>

<dd></dd><dt>Malonyl-CoA: 24.9</dt>

<dd></dd><dt>fatty_acid: 23.7</dt>

<dd></dd><dt>Propionyl-CoA: 22.1</dt>

<dd></dd><dt>Benzoyl-CoA: 22.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 21.3</dt>

<dd></dd><dt>CHC-CoA: 10.7</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2996..3404)</strong>: X
<div class="collapser collapser-target-input-path1-gene691 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene691-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIDTNGNVWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGVVWK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene695</span></strong>: X
<div class="collapser collapser-target-input-path1-gene695 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (235..672)</strong>: X
<div class="collapser collapser-target-input-path1-gene695 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene695-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLIFVGIMAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DALFVGLVAK</span> (59% 8Å match)</dd>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 6 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-464"></div> <span class="help-tooltip" id="region_svg-help-464">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c6-svg" data-filename="scaffold_1_r1c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,140,021 - 3,185,595 nt. (total: 45,575 nt)

<a class="cluster-rules-header" id="r1c6-rules-header" href="#r1c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c6-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-465"></div> <span class="help-tooltip" id="focus-panel-help-465">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c6-gene-table-details" data-name="r1c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c6-nrps_pks" data-name="r1c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c6-MIBiG-cluster-compare" data-name="r1c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c6-clusterblast" data-name="r1c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c6-knownclusterblast" data-name="r1c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c6-subclusterblast" data-name="r1c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c6-nrps-pks-bubbles" data-name="r1c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c6-pfam-details" data-name="r1c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c6-tigrfam-details" data-name="r1c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-58"></div> <span class="help-tooltip" id="gt-help-tooltip-help-58">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-50"></div> <span class="help-tooltip" id="nrps-domain-help-50">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-details-svg" data-filename="scaffold_1_r1c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-51"></div> <span class="help-tooltip" id="MIBiG-help-51">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c6"></div>
<div id="comparison-MIBiG-r1c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.55, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>curacomycin</td>


<td>Streptomyces curacoi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.55, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>curacomycin</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-52"></div> <span class="help-tooltip" id="cb-general-help-52">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c6_1.svg">
NW_006271973 (996861-1062288): Cordyceps militaris CM01 unplaced genomic scaf... (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_2.svg">
NW_006711167 (506611-572053): Trichoderma reesei QM6a unplaced genomic scaffo... (64% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_3.svg">
NW_014013664 (48182-113471): Trichoderma virens Gv29-8 chromosome Unknown TRI... (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_4.svg">
NW_011942160 (3770676-3836079): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (42% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_5.svg">
NW_014574716 (6263628-6329031): Metarhizium brunneum ARSEF 3297 chromosome Un... (42% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_6.svg">
NC_070551 (2536076-2601948): Fusarium falciforme chromosome 8, complete sequence (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_7.svg">
NC_070536 (2319263-2385135): Fusarium keratoplasticum isolate Fu6.1 chromosom... (50% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_8.svg">
NC_057319 (2334015-2399373): Ustilaginoidea virens chromosome 4, complete seq... (42% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_9.svg">
NC_035791 (4601700-4667019): Pochonia chlamydosporia 170 chromosome 2, whole ... (42% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r1c6_10.svg">
NW_021167080 (1267789-1333877): Sodiomyces alkalinus F11 unplaced genomic sca... (50% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r1c6-download">Download graphic</button>
</div>
<div id="clusterblast-r1c6-svg"></div>

</div>
</div>

<div class="body-details-section r1c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-53"></div> <span class="help-tooltip" id="cb-known-help-53">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c6-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c6_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c6_1.svg">
metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid
</option>

<option value = "svg/knownclusterblast_r1c6_2.svg">
dimethylcoprogen
</option>

</select>
<button id="knownclusterblast-r1c6-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c6-svg"></div>

</div>
</div>

<div class="body-details-section r1c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-54"></div> <span class="help-tooltip" id="cb-sub-help-54">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-57"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-57">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-domain-bubble-svg" data-filename="scaffold_1_r1c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC8">
Candidate 8 (3140020 - 3185595): single NRPS
</option>


</select>

<div class="details-svg" id="r1c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-59"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-59">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-pfam-details-svg" data-filename="scaffold_1_r1c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-60"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-60">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c6-tigrfam-details-svg" data-filename="scaffold_1_r1c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c6-nrps_pks_products" data-name="r1c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c6-nrps_pks_monomers" data-name="r1c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-55"></div> <span class="help-tooltip" id="nrps-structures-help-55">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 8, location 3140020 - 3185595:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c6-8" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c6-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-56"></div> <span class="help-tooltip" id="nrps-monomers-help-56">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene966</span></strong>: X
<div class="collapser collapser-target-input-path1-gene966 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (13..397)</strong>: X
<div class="collapser collapser-target-input-path1-gene966 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Z-ahmohOrn
<div class="collapser collapser-target-input-path1-gene966-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">FGLHIASTCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">FGLHIASTCK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene967</span></strong>: X
<div class="collapser collapser-target-input-path1-gene967 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (0..336)</strong>: X
<div class="collapser collapser-target-input-path1-gene967 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene967-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVDHGGAVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">DVDGGGGIGK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-466"></div> <span class="help-tooltip" id="region_svg-help-466">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="scaffold_2_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 308,796 - 353,700 nt. (total: 44,905 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-467"></div> <span class="help-tooltip" id="focus-panel-help-467">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-nrps_pks" data-name="r2c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c1-nrps-pks-bubbles" data-name="r2c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-69"></div> <span class="help-tooltip" id="gt-help-tooltip-help-69">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-61"></div> <span class="help-tooltip" id="nrps-domain-help-61">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-details-svg" data-filename="scaffold_2_r2c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-62"></div> <span class="help-tooltip" id="MIBiG-help-62">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>pyrrolizixenamide A</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058: 11-12613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide, NRP</td>

<td>mutanocyclin</td>


<td>Streptococcus mutans B04Sm5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002287: 24540-40025">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002287/index.html#r1c1" target="_new">BGC0002287</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>mutanocyclin, leuvalin, tyrvalin</td>


<td>Streptococcus mutans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002158: 177-13328">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002158/index.html#r1c1" target="_new">BGC0002158</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.47, order:0.57, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP, Polyketide</td>

<td>tenuazonic acid</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>pyrrolizixenamide A</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058: 11-12613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide, NRP</td>
<td>mutanocyclin</td>
<td>Streptococcus mutans B04Sm5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002287: 24540-40025">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002287/index.html#r1c1" target="_new">BGC0002287</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>mutanocyclin, leuvalin, tyrvalin</td>
<td>Streptococcus mutans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002158: 177-13328">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002158/index.html#r1c1" target="_new">BGC0002158</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.47, order:0.57, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP, Polyketide</td>
<td>tenuazonic acid</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-63"></div> <span class="help-tooltip" id="cb-general-help-63">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NW_006271970 (1632104-1696976): Cordyceps militaris CM01 unplaced genomic sca... (40% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NC_016472 (10589241-10691973): Thermothelomyces thermophilus ATCC 42464 chrom... (6% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NW_022156716 (287324-376248): Pyricularia grisea strain NI907 chromosome Unkn... (8% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NW_022985021 (395532-486580): Dothidotthia symphoricarpi CBS 119687 unplaced ... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NC_049565 (2694595-2813931): Talaromyces rugulosus chromosome V, complete seq... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NC_049561 (857603-965575): Talaromyces rugulosus chromosome I, complete sequence (5% of genes show similarity), NRPS-like,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NW_026089539 (1161221-1218974): Schizophyllum commune H4-8 unplaced genomic s... (8% of genes show similarity), fungal-RiPP
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-64"></div> <span class="help-tooltip" id="cb-known-help-64">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-65"></div> <span class="help-tooltip" id="cb-sub-help-65">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-68"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-68">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-domain-bubble-svg" data-filename="scaffold_2_r2c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (308795 - 353700): single NRPS
</option>


</select>

<div class="details-svg" id="r2c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-70"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-70">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="scaffold_2_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-71"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-71">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="scaffold_2_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c1-nrps_pks_products" data-name="r2c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c1-nrps_pks_monomers" data-name="r2c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-66"></div> <span class="help-tooltip" id="nrps-structures-help-66">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 308795 - 353700:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-1" class="smiles-canvas" data-smiles="NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-67"></div> <span class="help-tooltip" id="nrps-monomers-help-67">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene96</span></strong>: Ile
<div class="collapser collapser-target-input-path1-gene96 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (499..895)</strong>: Ile
<div class="collapser collapser-target-input-path1-gene96 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-input-path1-gene96-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALHSGAVIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DALHSGAVIK</span> (82% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-468"></div> <span class="help-tooltip" id="region_svg-help-468">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c2-svg" data-filename="scaffold_2_r2c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 720,724 - 789,034 nt. (total: 68,311 nt)

<a class="cluster-rules-header" id="r2c2-rules-header" href="#r2c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r2c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-469"></div> <span class="help-tooltip" id="focus-panel-help-469">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c2-gene-table-details" data-name="r2c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c2-MIBiG-cluster-compare" data-name="r2c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c2-clusterblast" data-name="r2c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c2-knownclusterblast" data-name="r2c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c2-subclusterblast" data-name="r2c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c2-pfam-details" data-name="r2c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c2-tigrfam-details" data-name="r2c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-76"></div> <span class="help-tooltip" id="gt-help-tooltip-help-76">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-72"></div> <span class="help-tooltip" id="MIBiG-help-72">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c2"></div>
<div id="comparison-MIBiG-r2c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.34</td>
<td>Other</td>

<td>itaconic acid</td>


<td>Ustilago maydis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002721: 182-542486">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002721/index.html#r1c1" target="_new">BGC0002721</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Saccharide</td>

<td>α-hydroxynitrile glucoside, β-hydroxynitrile glucoside, γ-hydroxynitrile glucoside</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Alkaloid</td>

<td>huperzine A</td>


<td>Shiraia sp. slf14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>mangicol A</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>heptelidic acid</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>armillyl orsellinate, 8α-hydroxy-6-protoilludene</td>


<td>Armillaria gallica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>HEx-tc3 terpene</td>


<td>Botrytis cinerea T4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001267: 3-35442">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001267/index.html#r1c1" target="_new">BGC0001267</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>lysergic acid, elymoclavine</td>


<td>Claviceps fusiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.46, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160: 0-27093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>terreic acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>itaconic acid</td>
<td>Ustilago maydis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002721: 182-542486">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002721/index.html#r1c1" target="_new">BGC0002721</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Saccharide</td>
<td>α-hydroxynitrile glucoside, β-hydroxynitrile glucoside, γ-hydroxynitrile glucoside</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Alkaloid</td>
<td>huperzine A</td>
<td>Shiraia sp. slf14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>mangicol A</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>heptelidic acid</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>armillyl orsellinate, 8α-hydroxy-6-protoilludene</td>
<td>Armillaria gallica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>HEx-tc3 terpene</td>
<td>Botrytis cinerea T4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001267: 3-35442">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001267/index.html#r1c1" target="_new">BGC0001267</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Terpene</td>
<td>lysergic acid, elymoclavine</td>
<td>Claviceps fusiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.46, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160: 0-27093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Polyketide</td>
<td>terreic acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-73"></div> <span class="help-tooltip" id="cb-general-help-73">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c2_1.svg">
NW_006271970 (2031211-2136459): Cordyceps militaris CM01 unplaced genomic sca... (55% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_2.svg">
NT_165982 (1008305-1103578): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (24% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_3.svg">
NW_017263587 (265177-438368): Mollisia scopiformis unplaced genomic scaffold ... (6% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_4.svg">
NC_030959 (4385741-4495530): Colletotrichum higginsianum IMI 349063 chromosom... (14% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_5.svg">
NW_011371361 (1937664-2035052): Paracoccidioides brasiliensis Pb18 unplaced g... (20% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_6.svg">
NW_015440948 (109188-206314): Paracoccidioides lutzii Pb01 unplaced genomic s... (21% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_7.svg">
NC_035796 (212423-303459): Pochonia chlamydosporia 170 chromosome 7, whole ge... (13% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r2c2_8.svg">
NW_026250024 (1787317-1950532): Hypoxylon trugodes strain CBS 135444 unplaced... (6% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_9.svg">
NW_020290631 (343013-434294): Aspergillus saccharolyticus JOP 1030-1 unplaced... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_10.svg">
NC_064675 (1394974-1486678): Colletotrichum lupini chromosome 2, complete seq... (10% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r2c2-download">Download graphic</button>
</div>
<div id="clusterblast-r2c2-svg"></div>

</div>
</div>

<div class="body-details-section r2c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-74"></div> <span class="help-tooltip" id="cb-known-help-74">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-75"></div> <span class="help-tooltip" id="cb-sub-help-75">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-77"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-77">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-pfam-details-svg" data-filename="scaffold_2_r2c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-78"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-78">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-tigrfam-details-svg" data-filename="scaffold_2_r2c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r2c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 3 - indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-470"></div> <span class="help-tooltip" id="region_svg-help-470">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c3-svg" data-filename="scaffold_2_r2c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,195,929 - 1,216,674 nt. (total: 20,746 nt)

<a class="cluster-rules-header" id="r2c3-rules-header" href="#r2c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c3-rules">indole: (indsynth or dmat or indole_PTase)</div>

<div class="region-svg-container">
<div id='r2c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-471"></div> <span class="help-tooltip" id="focus-panel-help-471">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c3-gene-table-details" data-name="r2c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c3-MIBiG-cluster-compare" data-name="r2c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c3-clusterblast" data-name="r2c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c3-knownclusterblast" data-name="r2c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c3-subclusterblast" data-name="r2c3-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c3-pfam-details" data-name="r2c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c3-tigrfam-details" data-name="r2c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-83"></div> <span class="help-tooltip" id="gt-help-tooltip-help-83">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-79"></div> <span class="help-tooltip" id="MIBiG-help-79">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c3"></div>
<div id="comparison-MIBiG-r2c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">indole</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.50, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000372: 0-22264">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000372/index.html#r1c1" target="_new">BGC0000372</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.46, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>NRP</td>

<td>hexadehydroastechrome, terezine-D, astechrome</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.14 (id:0.42, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.14), rgb(0, 0, 0, 0.14) 14%, white 14%)"></td>


<td>0.03</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.50, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000372: 0-22264">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000372/index.html#r1c1" target="_new">BGC0000372</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.46, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>NRP</td>
<td>hexadehydroastechrome, terezine-D, astechrome</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.14 (id:0.42, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.14), rgb(0, 0, 0, 0.14) 14%, white 14%)"></td>
<td>0.14</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-80"></div> <span class="help-tooltip" id="cb-general-help-80">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<p>No significant ClusterBlast hits found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-81"></div> <span class="help-tooltip" id="cb-known-help-81">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-82"></div> <span class="help-tooltip" id="cb-sub-help-82">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-84"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-84">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-pfam-details-svg" data-filename="scaffold_2_r2c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-85"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-85">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-tigrfam-details-svg" data-filename="scaffold_2_r2c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r2c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 4 - NRPS,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-472"></div> <span class="help-tooltip" id="region_svg-help-472">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c4-svg" data-filename="scaffold_2_r2c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,326,254 - 1,394,906 nt. (total: 68,653 nt)

<a class="cluster-rules-header" id="r2c4-rules-header" href="#r2c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r2c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-473"></div> <span class="help-tooltip" id="focus-panel-help-473">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c4-gene-table-details" data-name="r2c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c4-nrps_pks" data-name="r2c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c4-MIBiG-cluster-compare" data-name="r2c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c4-clusterblast" data-name="r2c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c4-knownclusterblast" data-name="r2c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c4-subclusterblast" data-name="r2c4-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r2c4-nrps-pks-bubbles" data-name="r2c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c4-pfam-details" data-name="r2c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c4-tigrfam-details" data-name="r2c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-94"></div> <span class="help-tooltip" id="gt-help-tooltip-help-94">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-86"></div> <span class="help-tooltip" id="nrps-domain-help-86">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-details-svg" data-filename="scaffold_2_r2c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-87"></div> <span class="help-tooltip" id="MIBiG-help-87">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c4"></div>
<div id="comparison-MIBiG-r2c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002199: 0-17425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002199/index.html#r1c1" target="_new">BGC0002199</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.60</td>
<td>Alkaloid</td>

<td>nanangelenin A</td>


<td>Aspergillus nanangensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.60</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.54</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.51</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002199: 0-17425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002199/index.html#r1c1" target="_new">BGC0002199</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Alkaloid</td>
<td>nanangelenin A</td>
<td>Aspergillus nanangensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901: 1761-23020">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-88"></div> <span class="help-tooltip" id="cb-general-help-88">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c4_1.svg">
NW_006271970 (3761055-3936010): Cordyceps militaris CM01 unplaced genomic sca... (13% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c4_2.svg">
NC_035791 (818908-980602): Pochonia chlamydosporia 170 chromosome 2, whole ge... (3% of genes show similarity), NAPAA,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c4_3.svg">
NW_024468995 (361260-526376): Aspergillus fijiensis CBS 313.89 unplaced genom... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c4_4.svg">
NW_020291492 (266674-426009): Aspergillus aculeatinus CBS 121060 unplaced gen... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c4_5.svg">
NC_072351 (3151891-3300466): Talaromyces marneffei chromosome 4, complete seq... (3% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c4_6.svg">
NW_020292049 (6038-140297): Aspergillus vadensis CBS 113365 unplaced genomic ... (4% of genes show similarity), NRPS,fungal-RiPP-like,indole,terpene
</option>

<option value = "svg/clusterblast_r2c4_7.svg">
NW_023336289 (1100952-1236029): Aspergillus tubingensis WU-2223L DNA, scaffol... (4% of genes show similarity), NRPS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r2c4_8.svg">
NW_014574694 (19387-240297): Metarhizium brunneum ARSEF 3297 chromosome Unkno... (3% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c4_9.svg">
NW_011942151 (20096-254382): Metarhizium robertsii ARSEF 23 MAA Scf 11, whole... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r2c4_10.svg">
NC_054390 (745744-925575): Drechmeria coniospora strain ARSEF 6962 chromosome... (3% of genes show similarity), NRPS,NRPS-like,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r2c4-download">Download graphic</button>
</div>
<div id="clusterblast-r2c4-svg"></div>

</div>
</div>

<div class="body-details-section r2c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-89"></div> <span class="help-tooltip" id="cb-known-help-89">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-90"></div> <span class="help-tooltip" id="cb-sub-help-90">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r2c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-93"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-93">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-domain-bubble-svg" data-filename="scaffold_2_r2c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (1326253 - 1394906): neighbouring NRPS-fungal-RiPP-like
</option>



<option value="CC5">
Candidate 5 (1326253 - 1378710): single NRPS
</option>


</select>

<div class="details-svg" id="r2c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-95"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-95">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-pfam-details-svg" data-filename="scaffold_2_r2c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-96"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-96">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-tigrfam-details-svg" data-filename="scaffold_2_r2c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c4-nrps_pks_products" data-name="r2c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c4-nrps_pks_monomers" data-name="r2c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r2c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-91"></div> <span class="help-tooltip" id="nrps-structures-help-91">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 1326253 - 1394906:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c4-4" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 5, location 1326253 - 1378710:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c4-5" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c4-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-92"></div> <span class="help-tooltip" id="nrps-monomers-help-92">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene414</span></strong>: X - X - X
<div class="collapser collapser-target-input-path1-gene414 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (146..529)</strong>: X
<div class="collapser collapser-target-input-path1-gene414 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene414-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">LIRNLTASLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1215..1619)</strong>: X
<div class="collapser collapser-target-input-path1-gene414 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene414-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVAFVGVVYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Trp <span class="serif">DVALVGVVQK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2310..2712)</strong>: X
<div class="collapser collapser-target-input-path1-gene414 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala
<div class="collapser collapser-target-input-path1-gene414-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIMSIVAIFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVSSVFAIFK</span> (65% 8Å match)</dd>
<dd>Thr <span class="serif">DAQTIVAIHK</span> (56% 8Å match)</dd>
<dd>Arg <span class="serif">DVESIGAIAK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>

<div class="page" id='r2c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 5 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-474"></div> <span class="help-tooltip" id="region_svg-help-474">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c5-svg" data-filename="scaffold_2_r2c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,102,079 - 2,123,663 nt. (total: 21,585 nt)

<a class="cluster-rules-header" id="r2c5-rules-header" href="#r2c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c5-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-475"></div> <span class="help-tooltip" id="focus-panel-help-475">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c5-gene-table-details" data-name="r2c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c5-MIBiG-cluster-compare" data-name="r2c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c5-clusterblast" data-name="r2c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c5-knownclusterblast" data-name="r2c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c5-subclusterblast" data-name="r2c5-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c5-pfam-details" data-name="r2c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c5-tigrfam-details" data-name="r2c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-101"></div> <span class="help-tooltip" id="gt-help-tooltip-help-101">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-97"></div> <span class="help-tooltip" id="MIBiG-help-97">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c5"></div>
<div id="comparison-MIBiG-r2c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.53, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>squalestatin S1</td>


<td>Aspergillus sp. Z5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>microcyclamide</td>


<td>Microcystis aeruginosa NIES-298</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>carbapenem MM4550</td>


<td>Streptomyces argenteolus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.52, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>squalestatin S1</td>


<td>Phoma sp. MF5453</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>gramicidin S</td>


<td>Aneurinibacillus migulanus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>





<td class="cc-heat-cell" title="0.15 (id:0.45, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>sorangipyranone</td>


<td>Myxococcales bacterium</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.53, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>squalestatin S1</td>
<td>Aspergillus sp. Z5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.44, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>RiPP</td>
<td>microcyclamide</td>
<td>Microcystis aeruginosa NIES-298</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>carbapenem MM4550</td>
<td>Streptomyces argenteolus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.52, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Polyketide</td>
<td>squalestatin S1</td>
<td>Phoma sp. MF5453</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>NRP</td>
<td>gramicidin S</td>
<td>Aneurinibacillus migulanus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>


<td class="cc-heat-cell" title="0.15 (id:0.45, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
<td>0.15</td>
<td>Polyketide</td>
<td>sorangipyranone</td>
<td>Myxococcales bacterium</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-98"></div> <span class="help-tooltip" id="cb-general-help-98">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c5_1.svg">
NW_006271974 (1078266-1109834): Cordyceps militaris CM01 unplaced genomic sca... (45% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_2.svg">
NC_036623 (3601013-3632563): Fusarium fujikuroi IMI 58289 draft genome, chrom... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_3.svg">
NC_058389 (1466023-1497573): Fusarium musae strain F31 chromosome 3, whole ge... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_4.svg">
NC_058402 (6845329-6876878): Fusarium poae strain DAOMC 252244 chromosome 4, ... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_5.svg">
NC_026477 (6663266-6694813): Fusarium graminearum PH-1 chromosome 4, whole ge... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_6.svg">
NC_038015 (6636088-6667634): Fusarium venenatum strain A3/5 genome assembly, ... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_7.svg">
NW_023336189 (374372-405746): Geosmithia morbida isolate 1262 chromosome Unkn... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_8.svg">
NC_054390 (5438362-5469923): Drechmeria coniospora strain ARSEF 6962 chromoso... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_9.svg">
NW_006711161 (132460-164034): Trichoderma reesei QM6a unplaced genomic scaffo... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c5_10.svg">
NW_022984651 (919341-950958): Daldinia childiae strain JS-1345 chromosome Unk... (60% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r2c5-download">Download graphic</button>
</div>
<div id="clusterblast-r2c5-svg"></div>

</div>
</div>

<div class="body-details-section r2c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-99"></div> <span class="help-tooltip" id="cb-known-help-99">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c5-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c5_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c5_1.svg">
squalestatin S1
</option>

</select>
<button id="knownclusterblast-r2c5-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c5-svg"></div>

</div>
</div>

<div class="body-details-section r2c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-100"></div> <span class="help-tooltip" id="cb-sub-help-100">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-102"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-102">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-pfam-details-svg" data-filename="scaffold_2_r2c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-103"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-103">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c5-tigrfam-details-svg" data-filename="scaffold_2_r2c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r2c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 6 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-476"></div> <span class="help-tooltip" id="region_svg-help-476">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c6-svg" data-filename="scaffold_2_r2c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,224,468 - 2,294,753 nt. (total: 70,286 nt)

<a class="cluster-rules-header" id="r2c6-rules-header" href="#r2c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c6-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-477"></div> <span class="help-tooltip" id="focus-panel-help-477">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c6-gene-table-details" data-name="r2c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c6-nrps_pks" data-name="r2c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c6-MIBiG-cluster-compare" data-name="r2c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c6-clusterblast" data-name="r2c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c6-knownclusterblast" data-name="r2c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c6-subclusterblast" data-name="r2c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c6-nrps-pks-bubbles" data-name="r2c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c6-pfam-details" data-name="r2c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c6-tigrfam-details" data-name="r2c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-112"></div> <span class="help-tooltip" id="gt-help-tooltip-help-112">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-104"></div> <span class="help-tooltip" id="nrps-domain-help-104">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-details-svg" data-filename="scaffold_2_r2c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-105"></div> <span class="help-tooltip" id="MIBiG-help-105">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c6"></div>
<div id="comparison-MIBiG-r2c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Other</td>

<td>falcarindiol</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076: 6-3180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>4,5-didehydroacaterin, acaterin</td>


<td>Pseudomonas jessenii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002253: 0-33620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002253/index.html#r1c1" target="_new">BGC0002253</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>waikikiamide A, waikikiamide B, waikikiamide C, (+)-semivioxanthin, semivioxanthin</td>


<td>Aspergillus sp. FM242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>falcarindiol</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076: 6-3180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>4,5-didehydroacaterin, acaterin</td>
<td>Pseudomonas jessenii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002253: 0-33620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002253/index.html#r1c1" target="_new">BGC0002253</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>waikikiamide A, waikikiamide B, waikikiamide C, (+)-semivioxanthin, semivioxanthin</td>
<td>Aspergillus sp. FM242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-106"></div> <span class="help-tooltip" id="cb-general-help-106">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c6_1.svg">
NW_021167080 (4131403-4218796): Sodiomyces alkalinus F11 unplaced genomic sca... (42% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c6_2.svg">
NW_026250021 (1864413-1931647): Hypoxylon trugodes strain CBS 135444 unplaced... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c6_3.svg">
NW_006271971 (91199-157949): Cordyceps militaris CM01 unplaced genomic scaffo... (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c6_4.svg">
NW_025408670 (3613020-3679633): Hirsutella rhossiliensis strain HR02 chromoso... (23% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r2c6_5.svg">
NW_024469024 (26525-267638): Aspergillus fijiensis CBS 313.89 unplaced genomi... (4% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r2c6_6.svg">
NW_017263595 (627002-693843): Mollisia scopiformis unplaced genomic scaffold ... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c6_7.svg">
NC_030957 (3250063-3360781): Colletotrichum higginsianum IMI 349063 chromosom... (8% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c6_8.svg">
NZ_KK106989 (375446-502971): Streptomyces sp. Tu 6176 scaffold00002, whole ge... (6% of genes show similarity), NRPS,T1PKS,ranthipeptide
</option>

<option value = "svg/clusterblast_r2c6_9.svg">
NW_024468940 (1557228-1624623): Cucurbitaria berberidis CBS 394.84 unplaced g... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c6_10.svg">
NW_025091288 (768347-846628): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (10% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r2c6-download">Download graphic</button>
</div>
<div id="clusterblast-r2c6-svg"></div>

</div>
</div>

<div class="body-details-section r2c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-107"></div> <span class="help-tooltip" id="cb-known-help-107">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-108"></div> <span class="help-tooltip" id="cb-sub-help-108">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-111"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-111">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-domain-bubble-svg" data-filename="scaffold_2_r2c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC8">
Candidate 8 (2224467 - 2294753): single T1PKS
</option>


</select>

<div class="details-svg" id="r2c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-113"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-113">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-pfam-details-svg" data-filename="scaffold_2_r2c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-114"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-114">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-tigrfam-details-svg" data-filename="scaffold_2_r2c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c6-nrps_pks_products" data-name="r2c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c6-nrps_pks_monomers" data-name="r2c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-109"></div> <span class="help-tooltip" id="nrps-structures-help-109">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 8, location 2224467 - 2294753:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c6-8" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c6-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-110"></div> <span class="help-tooltip" id="nrps-monomers-help-110">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene690</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene690 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (890..1182)</strong>: mal
<div class="collapser collapser-target-input-path1-gene690 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene690-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene690-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 90.3</dt>

<dd></dd><dt>inactive: 55.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 37.5</dt>

<dd></dd><dt>Methylmalonyl-CoA: 37.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 20.0</dt>

<dd></dd><dt>Propionyl-CoA: 16.5</dt>

<dd></dd><dt>Benzoyl-CoA: 15.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 15.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 11.8</dt>

<dd></dd><dt>fatty_acid: 11.5</dt>

<dd></dd><dt>CHC-CoA: 10.1</dt>

<dd></dd><dt>Acetyl-CoA: 9.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene696</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene696 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (448..757)</strong>: pk
<div class="collapser collapser-target-input-path1-gene696 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene696-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>CE-Malonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene696-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 64.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 54.2</dt>

<dd></dd><dt>Malonyl-CoA: 50.0</dt>

<dd></dd><dt>Propionyl-CoA: 33.9</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 26.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 23.2</dt>

<dd></dd><dt>Acetyl-CoA: 19.6</dt>

<dd></dd><dt>Isobutyryl-CoA: 18.4</dt>

<dd></dd><dt>inactive: 18.2</dt>

<dd></dd><dt>CHC-CoA: 17.5</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 12.3</dt>

<dd></dd><dt>Benzoyl-CoA: 9.8</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 7 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-478"></div> <span class="help-tooltip" id="region_svg-help-478">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c7-svg" data-filename="scaffold_2_r2c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,360,062 - 2,403,928 nt. (total: 43,867 nt)

<a class="cluster-rules-header" id="r2c7-rules-header" href="#r2c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c7-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-479"></div> <span class="help-tooltip" id="focus-panel-help-479">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c7-gene-table-details" data-name="r2c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c7-nrps_pks" data-name="r2c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c7-MIBiG-cluster-compare" data-name="r2c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c7-clusterblast" data-name="r2c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c7-knownclusterblast" data-name="r2c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c7-subclusterblast" data-name="r2c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c7-nrps-pks-bubbles" data-name="r2c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c7-pfam-details" data-name="r2c7-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c7-tigrfam-details" data-name="r2c7-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-123"></div> <span class="help-tooltip" id="gt-help-tooltip-help-123">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-115"></div> <span class="help-tooltip" id="nrps-domain-help-115">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-details-svg" data-filename="scaffold_2_r2c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-116"></div> <span class="help-tooltip" id="MIBiG-help-116">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c7"></div>
<div id="comparison-MIBiG-r2c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>





<td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>


<td>0.43</td>
<td>NRP</td>

<td>choline</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP, Saccharide</td>

<td>minimycin</td>


<td>Streptomyces hygroscopicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308: 0-7851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>aureusimine A, aureusimine B, aureusimine C</td>


<td>Staphylococcus aureus subsp. aureus str. JKD6008</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>


<td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
<td>0.74</td>
<td>NRP</td>
<td>choline</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP, Saccharide</td>
<td>minimycin</td>
<td>Streptomyces hygroscopicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308: 0-7851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>aureusimine A, aureusimine B, aureusimine C</td>
<td>Staphylococcus aureus subsp. aureus str. JKD6008</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-117"></div> <span class="help-tooltip" id="cb-general-help-117">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c7_1.svg">
NW_025764707 (691343-755195): Emericellopsis atlantica unplaced genomic scaff... (40% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c7_2.svg">
NC_070532 (4966088-5118746): Fusarium keratoplasticum isolate Fu6.1 chromosom... (19% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_3.svg">
NC_070547 (4757164-4894085): Fusarium falciforme chromosome 4, complete sequence (19% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_4.svg">
NW_014013657 (238627-342899): Trichoderma virens Gv29-8 chromosome Unknown TR... (20% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_5.svg">
NC_035791 (1143313-1234197): Pochonia chlamydosporia 170 chromosome 2, whole ... (22% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c7_6.svg">
NW_025408669 (7090957-7154869): Hirsutella rhossiliensis strain HR02 chromoso... (53% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c7_7.svg">
NW_006711169 (17423-81332): Trichoderma reesei QM6a unplaced genomic scaffold... (35% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c7_8.svg">
NC_072842 (587831-651686): Fusarium oxysporum Fo47 chromosome III, complete s... (24% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c7_9.svg">
NC_036624 (4507840-4571695): Fusarium fujikuroi IMI 58289 draft genome, chrom... (24% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r2c7_10.svg">
NC_058388 (524455-588310): Fusarium musae strain F31 chromosome 2, whole geno... (25% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r2c7-download">Download graphic</button>
</div>
<div id="clusterblast-r2c7-svg"></div>

</div>
</div>

<div class="body-details-section r2c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-118"></div> <span class="help-tooltip" id="cb-known-help-118">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c7-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c7_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c7_1.svg">
choline
</option>

</select>
<button id="knownclusterblast-r2c7-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c7-svg"></div>

</div>
</div>

<div class="body-details-section r2c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-119"></div> <span class="help-tooltip" id="cb-sub-help-119">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-122"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-122">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-domain-bubble-svg" data-filename="scaffold_2_r2c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC9">
Candidate 9 (2360061 - 2403928): single NRPS-like
</option>


</select>

<div class="details-svg" id="r2c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-124"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-124">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-pfam-details-svg" data-filename="scaffold_2_r2c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c7-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-125"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-125">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-tigrfam-details-svg" data-filename="scaffold_2_r2c7_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c7-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c7-nrps_pks_products" data-name="r2c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c7-nrps_pks_monomers" data-name="r2c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-120"></div> <span class="help-tooltip" id="nrps-structures-help-120">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 9, location 2360061 - 2403928:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c7-9" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c7-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-121"></div> <span class="help-tooltip" id="nrps-monomers-help-121">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene730</span></strong>: Phe
<div class="collapser collapser-target-input-path1-gene730 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (13..417)</strong>: Phe
<div class="collapser collapser-target-input-path1-gene730 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-input-path1-gene730-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>phe (Phe)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--WLWNVEAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 8 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-480"></div> <span class="help-tooltip" id="region_svg-help-480">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c8-svg" data-filename="scaffold_2_r2c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,629,960 - 2,653,418 nt. (total: 23,459 nt)

<a class="cluster-rules-header" id="r2c8-rules-header" href="#r2c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c8-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-481"></div> <span class="help-tooltip" id="focus-panel-help-481">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c8-gene-table-details" data-name="r2c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c8-MIBiG-cluster-compare" data-name="r2c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c8-clusterblast" data-name="r2c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c8-knownclusterblast" data-name="r2c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c8-subclusterblast" data-name="r2c8-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c8-pfam-details" data-name="r2c8-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c8-tigrfam-details" data-name="r2c8-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-130"></div> <span class="help-tooltip" id="gt-help-tooltip-help-130">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-126"></div> <span class="help-tooltip" id="MIBiG-help-126">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c8"></div>
<div id="comparison-MIBiG-r2c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001248: 1352-4192">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001248/index.html#r1c1" target="_new">BGC0001248</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>clavaric acid</td>


<td>Hypholoma sublateritium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>marneral</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Other</td>

<td>sarasinoside</td>


<td>uncultured bacterium</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001248: 1352-4192">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001248/index.html#r1c1" target="_new">BGC0001248</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>clavaric acid</td>
<td>Hypholoma sublateritium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Other</td>
<td>sarasinoside</td>
<td>uncultured bacterium</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-127"></div> <span class="help-tooltip" id="cb-general-help-127">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c8_1.svg">
NW_006271977 (650506-682810): Cordyceps militaris CM01 unplaced genomic scaff... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_2.svg">
NW_014574706 (185039-217400): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (46% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_3.svg">
NC_035794 (3381868-3414218): Pochonia chlamydosporia 170 chromosome 5, whole ... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_4.svg">
NW_014013668 (153970-222143): Trichoderma virens Gv29-8 chromosome Unknown TR... (27% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c8_5.svg">
NC_054389 (10607680-10640044): Drechmeria coniospora strain ARSEF 6962 chromo... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_6.svg">
NW_025408667 (5243071-5275419): Hirsutella rhossiliensis strain HR02 chromoso... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_7.svg">
NC_043740 (4056696-4089123): Pyricularia pennisetigena strain Br36 chromosome... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_8.svg">
NW_006711154 (931670-999559): Trichoderma reesei QM6a unplaced genomic scaffo... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c8_9.svg">
NC_057320 (3125192-3157585): Ustilaginoidea virens chromosome 5, complete seq... (45% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c8_10.svg">
NW_023336201 (447629-481710): Geosmithia morbida isolate 1262 chromosome Unkn... (30% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r2c8-download">Download graphic</button>
</div>
<div id="clusterblast-r2c8-svg"></div>

</div>
</div>

<div class="body-details-section r2c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-128"></div> <span class="help-tooltip" id="cb-known-help-128">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c8-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c8_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c8_1.svg">
clavaric acid
</option>

</select>
<button id="knownclusterblast-r2c8-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c8-svg"></div>

</div>
</div>

<div class="body-details-section r2c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-129"></div> <span class="help-tooltip" id="cb-sub-help-129">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-131"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-131">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c8-pfam-details-svg" data-filename="scaffold_2_r2c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c8-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-132"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-132">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c8-tigrfam-details-svg" data-filename="scaffold_2_r2c8_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c8-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r2c9' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 9 - NRPS,NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-482"></div> <span class="help-tooltip" id="region_svg-help-482">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region009.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c9-svg" data-filename="scaffold_2_r2c9.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,774,261 - 2,826,282 nt. (total: 52,022 nt)

<a class="cluster-rules-header" id="r2c9-rules-header" href="#r2c9">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c9-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c9-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-483"></div> <span class="help-tooltip" id="focus-panel-help-483">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c9">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c9-gene-table-details" data-name="r2c9-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c9-nrps_pks" data-name="r2c9-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c9-MIBiG-cluster-compare" data-name="r2c9-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c9-clusterblast" data-name="r2c9-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c9-knownclusterblast" data-name="r2c9-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c9-subclusterblast" data-name="r2c9-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c9-nrps-pks-bubbles" data-name="r2c9-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c9-pfam-details" data-name="r2c9-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c9-tigrfam-details" data-name="r2c9-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c9-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-141"></div> <span class="help-tooltip" id="gt-help-tooltip-help-141">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c9">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c9-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c9-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c9">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c9-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-133"></div> <span class="help-tooltip" id="nrps-domain-help-133">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c9-details-svg" data-filename="scaffold_2_r2c9_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c9-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c9-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-134"></div> <span class="help-tooltip" id="MIBiG-help-134">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c9-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c9-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c9"></div>
<div id="comparison-MIBiG-r2c9-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.02</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.98</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.94</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.94</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.90</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.90</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger CBS 101883</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.88</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.86</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.86</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.86</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c9-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP, Polyketide</td>
<td>pyrophen, campyrone B</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP, Polyketide</td>
<td>pyrophen, campyrone B</td>
<td>Aspergillus niger CBS 101883</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c9-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-135"></div> <span class="help-tooltip" id="cb-general-help-135">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c9-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c9_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c9_1.svg">
NZ_CP011509 (5423897-5570224): Archangium gephyra strain DSM 2261 chromosome,... (3% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_2.svg">
NZ_BDUC01000001 (716583-802583): Tolypothrix sp. NIES-4075, whole genome shot... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_3.svg">
NZ_CP011799 (2655858-2717809): Streptomyces sp. PBH53 chromosome (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_4.svg">
NZ_LR594664 (174868-226900): Variovorax sp. RA8 plasmid 3 (5% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_5.svg">
NZ_LR594691 (178519-230482): Variovorax sp. WDL1 plasmid 3 (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_6.svg">
NZ_CP040438 (3361872-3455938): Stenotrophomonas maltophilia strain PEG-173 ch... (4% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_7.svg">
NC_017030 (3441885-3495141): Corallococcus coralloides DSM 2259, complete seq... (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_8.svg">
NZ_CP079946 (3522109-3575284): Corallococcus sp. EGB chromosome, complete genome (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_9.svg">
NZ_CP034669 (3230815-3284035): Corallococcus coralloides strain B035 chromoso... (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c9_10.svg">
NZ_CP114039 (5390394-5443611): Corallococcus sp. NCRR chromosome, complete ge... (4% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r2c9-download">Download graphic</button>
</div>
<div id="clusterblast-r2c9-svg"></div>

</div>
</div>

<div class="body-details-section r2c9-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-136"></div> <span class="help-tooltip" id="cb-known-help-136">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c9-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-137"></div> <span class="help-tooltip" id="cb-sub-help-137">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c9-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-140"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-140">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c9-domain-bubble-svg" data-filename="scaffold_2_r2c9_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c9-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC11">
Candidate 11 (2774260 - 2826282): neighbouring T1PKS-NRPS-like-NRPS
</option>



<option value="CC12">
Candidate 12 (2774260 - 2821315): chemical hybrid T1PKS-NRPS-like
</option>



<option value="CC13">
Candidate 13 (2782657 - 2826282): single NRPS
</option>


</select>

<div class="details-svg" id="r2c9-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c9-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c9-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-142"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-142">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c9-pfam-details-svg" data-filename="scaffold_2_r2c9_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c9-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c9-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-143"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-143">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c9-tigrfam-details-svg" data-filename="scaffold_2_r2c9_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c9-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c9-nrps_pks_products" data-name="r2c9-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c9-nrps_pks_monomers" data-name="r2c9-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c9-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-138"></div> <span class="help-tooltip" id="nrps-structures-help-138">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 11, location 2774260 - 2826282:
<div class="collapser collapser-target-CC11 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c9-11" class="smiles-canvas" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly - mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c9-11-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 12, location 2774260 - 2821315:
<div class="collapser collapser-target-CC12 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c9-12" class="smiles-canvas" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly - mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c9-12-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 13, location 2782657 - 2826282:
<div class="collapser collapser-target-CC13 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c9-13" class="smiles-canvas" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly - mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c9-13-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c9-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-139"></div> <span class="help-tooltip" id="nrps-monomers-help-139">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene852</span></strong>: Gly - mal
<div class="collapser collapser-target-input-path1-gene852 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (37..430)</strong>: Gly
<div class="collapser collapser-target-input-path1-gene852 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-input-path1-gene852-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>gly (Gly)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DILQLGLVWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGLVWK</span> (79% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>PKS_AT (1184..1465)</strong>: mal
<div class="collapser collapser-target-input-path1-gene852 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene852-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>inactive: 62.5%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene852-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 146.0</dt>

<dd></dd><dt>inactive: 74.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 72.1</dt>

<dd></dd><dt>Methylmalonyl-CoA: 70.7</dt>

<dd></dd><dt>Acetyl-CoA: 35.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 35.0</dt>

<dd></dd><dt>Propionyl-CoA: 33.8</dt>

<dd></dd><dt>fatty_acid: 29.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 27.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 25.1</dt>

<dd></dd><dt>CHC-CoA: 24.4</dt>

<dd></dd><dt>Benzoyl-CoA: 24.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.7</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 12.8</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c10' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 10 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-484"></div> <span class="help-tooltip" id="region_svg-help-484">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region010.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c10-svg" data-filename="scaffold_2_r2c10.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,669,093 - 4,731,138 nt. (total: 62,046 nt)

<a class="cluster-rules-header" id="r2c10-rules-header" href="#r2c10">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c10-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c10-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-485"></div> <span class="help-tooltip" id="focus-panel-help-485">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c10">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c10-gene-table-details" data-name="r2c10-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c10-nrps_pks" data-name="r2c10-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c10-MIBiG-cluster-compare" data-name="r2c10-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c10-clusterblast" data-name="r2c10-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c10-knownclusterblast" data-name="r2c10-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c10-subclusterblast" data-name="r2c10-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c10-pfam-details" data-name="r2c10-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c10-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-150"></div> <span class="help-tooltip" id="gt-help-tooltip-help-150">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c10">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c10-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c10-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c10">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c10-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-144"></div> <span class="help-tooltip" id="nrps-domain-help-144">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c10-details-svg" data-filename="scaffold_2_r2c10_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c10-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c10-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-145"></div> <span class="help-tooltip" id="MIBiG-help-145">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c10-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c10-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c10"></div>
<div id="comparison-MIBiG-r2c10-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.64</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002644: 0-7903">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002644/index.html#r1c1" target="_new">BGC0002644</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Other, Terpene</td>

<td>domoic acid, isodomoic acid A, isodomoic acid B, isodomoic acid C, 7&#39;-carboxy-L-NGG, prekainic acid, kainic acid</td>


<td>Chondria armata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911: 591-10299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>Terpene</td>

<td>raimonol</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.45</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910: 237-11923">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.45</td>
<td>Terpene</td>

<td>cyslabdan</td>


<td>Streptomyces cyslabdanicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>β-Diketones</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.43, order:0.27, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.41</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001375: 649-10840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001375/index.html#r1c1" target="_new">BGC0001375</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.38</td>
<td>Terpene</td>

<td>penitrem A, penitrem B, penitrem C, penitrem D, penitrem E, penitrem F</td>


<td>Penicillium simplicissimum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002218: 3-22615">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002218/index.html#r1c1" target="_new">BGC0002218</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.38</td>
<td>Terpene</td>

<td>(-)-Cyatha-3,12-diene</td>


<td>Dentipellis sp. KUC8613</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c10-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001267: 3-35442">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001267/index.html#r1c1" target="_new">BGC0001267</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>lysergic acid, elymoclavine</td>
<td>Claviceps fusiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>thalianol, thaliandiol</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002644: 0-7903">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002644/index.html#r1c1" target="_new">BGC0002644</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other, Terpene</td>
<td>domoic acid, isodomoic acid A, isodomoic acid B, isodomoic acid C, 7&#39;-carboxy-L-NGG, prekainic acid, kainic acid</td>
<td>Chondria armata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910: 237-11923">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>cyslabdan</td>
<td>Streptomyces cyslabdanicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002218: 3-22615">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002218/index.html#r1c1" target="_new">BGC0002218</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>(-)-Cyatha-3,12-diene</td>
<td>Dentipellis sp. KUC8613</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c10-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-146"></div> <span class="help-tooltip" id="cb-general-help-146">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c10-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c10_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c10_1.svg">
NW_006271971 (3503471-3590640): Cordyceps militaris CM01 unplaced genomic sca... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c10_2.svg">
NW_023336287 (1029426-1127137): Aspergillus tubingensis WU-2223L DNA, scaffol... (14% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c10_3.svg">
NW_020291625 (99659-196841): Aspergillus piperis CBS 112811 unplaced genomic ... (14% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r2c10_4.svg">
NW_020291816 (340965-433020): Aspergillus costaricaensis CBS 115574 unplaced ... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c10_5.svg">
NW_020290887 (673812-764439): Aspergillus niger CBS 101883 unplaced genomic s... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c10_6.svg">
NW_020292069 (316400-408453): Aspergillus vadensis CBS 113365 unplaced genomi... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c10_7.svg">
NW_024467525 (3144728-3217090): Aspergillus versicolor CBS 583.65 unplaced ge... (17% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c10_8.svg">
NW_013581207 (483365-550922): Aureobasidium namibiae CBS 147.97 unplaced geno... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c10_9.svg">
NW_013566983 (689939-757505): Aureobasidium subglaciale EXF-2481 unplaced gen... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c10_10.svg">
NW_020292039 (3332166-3473157): Aspergillus vadensis CBS 113365 unplaced geno... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r2c10-download">Download graphic</button>
</div>
<div id="clusterblast-r2c10-svg"></div>

</div>
</div>

<div class="body-details-section r2c10-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-147"></div> <span class="help-tooltip" id="cb-known-help-147">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c10-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-148"></div> <span class="help-tooltip" id="cb-sub-help-148">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c10-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-151"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-151">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c10-pfam-details-svg" data-filename="scaffold_2_r2c10_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c10-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c10-nrps_pks_monomers" data-name="r2c10-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r2c10-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-149"></div> <span class="help-tooltip" id="nrps-monomers-help-149">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1443</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene1443 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (553..869)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1443 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1443-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1443-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 76.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 62.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 52.5</dt>

<dd></dd><dt>Malonyl-CoA: 38.5</dt>

<dd></dd><dt>Propionyl-CoA: 37.3</dt>

<dd></dd><dt>fatty_acid: 35.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 22.4</dt>

<dd></dd><dt>Benzoyl-CoA: 15.9</dt>

<dd></dd><dt>CHC-CoA: 15.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 14.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 12.2</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene1447</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1447 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (132..575)</strong>: X
<div class="collapser collapser-target-input-path1-gene1447 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1447-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALTVGGIVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DALAVGEVVK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-486"></div> <span class="help-tooltip" id="region_svg-help-486">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="scaffold_3_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 198,793 - 244,284 nt. (total: 45,492 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-487"></div> <span class="help-tooltip" id="focus-panel-help-487">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-nrps_pks" data-name="r3c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c1-nrps-pks-bubbles" data-name="r3c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-160"></div> <span class="help-tooltip" id="gt-help-tooltip-help-160">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-152"></div> <span class="help-tooltip" id="nrps-domain-help-152">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-details-svg" data-filename="scaffold_3_r3c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-153"></div> <span class="help-tooltip" id="MIBiG-help-153">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.45, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002172: 696-54762">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002172/index.html#r1c1" target="_new">BGC0002172</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>NRP</td>

<td>fusatrixin, fusapentaxin, fusaoctaxin A</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001515: 0-63902">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001515/index.html#r1c1" target="_new">BGC0001515</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.41, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>NRP</td>

<td>aspercryptins</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.45, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002172: 696-54762">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002172/index.html#r1c1" target="_new">BGC0002172</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>NRP</td>
<td>fusatrixin, fusapentaxin, fusaoctaxin A</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001515: 0-63902">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001515/index.html#r1c1" target="_new">BGC0001515</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.41, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>NRP</td>
<td>aspercryptins</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-154"></div> <span class="help-tooltip" id="cb-general-help-154">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c1_1.svg">
NW_006271975 (1558381-1621743): Cordyceps militaris CM01 unplaced genomic sca... (35% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c1_2.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_3.svg">
NC_035792 (887996-951264): Pochonia chlamydosporia 170 chromosome 3, whole ge... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c1_4.svg">
NW_011942171 (3680815-3744138): Metarhizium robertsii ARSEF 23 MAA Scf 3, who... (11% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c1_5.svg">
NW_014574699 (142223-205548): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c1_6.svg">
NW_011371366 (647325-794373): Paracoccidioides brasiliensis Pb18 unplaced gen... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_7.svg">
NW_024468919 (637636-749466): Cryphonectria parasitica EP155 unplaced genomic... (6% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_8.svg">
NW_015440960 (427476-569911): Paracoccidioides lutzii Pb01 unplaced genomic s... (3% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_9.svg">
NC_072348 (1750133-1909935): Talaromyces marneffei chromosome 1, complete seq... (3% of genes show similarity), NRPS-like,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r3c1_10.svg">
NC_007196 (616905-727670): Aspergillus fumigatus Af293 chromosome 3, whole ge... (10% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r3c1-download">Download graphic</button>
</div>
<div id="clusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-155"></div> <span class="help-tooltip" id="cb-known-help-155">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-156"></div> <span class="help-tooltip" id="cb-sub-help-156">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-159"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-159">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-domain-bubble-svg" data-filename="scaffold_3_r3c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (198792 - 244284): single NRPS-like
</option>


</select>

<div class="details-svg" id="r3c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-161"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-161">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="scaffold_3_r3c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c1-nrps_pks_products" data-name="r3c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c1-nrps_pks_monomers" data-name="r3c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-157"></div> <span class="help-tooltip" id="nrps-structures-help-157">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 198792 - 244284:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-158"></div> <span class="help-tooltip" id="nrps-monomers-help-158">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene64</span></strong>: X
<div class="collapser collapser-target-input-path1-gene64 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (575..892)</strong>: X
<div class="collapser collapser-target-input-path1-gene64 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene64-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">INTIVG-ISK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DPTIVGAIIK</span> (29% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-488"></div> <span class="help-tooltip" id="region_svg-help-488">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c2-svg" data-filename="scaffold_3_r3c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 835,515 - 897,274 nt. (total: 61,760 nt)

<a class="cluster-rules-header" id="r3c2-rules-header" href="#r3c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-489"></div> <span class="help-tooltip" id="focus-panel-help-489">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c2-gene-table-details" data-name="r3c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c2-MIBiG-cluster-compare" data-name="r3c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c2-clusterblast" data-name="r3c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c2-knownclusterblast" data-name="r3c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c2-subclusterblast" data-name="r3c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c2-pfam-details" data-name="r3c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-166"></div> <span class="help-tooltip" id="gt-help-tooltip-help-166">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-162"></div> <span class="help-tooltip" id="MIBiG-help-162">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<div><p>No matches found.</p></div>

</div>
</div>



<div class="body-details-section r3c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-163"></div> <span class="help-tooltip" id="cb-general-help-163">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c2_1.svg">
NW_006271973 (266414-357915): Cordyceps militaris CM01 unplaced genomic scaff... (46% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_2.svg">
NC_072350 (525276-617173): Talaromyces marneffei chromosome 3, complete sequence (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_3.svg">
NW_024468919 (637636-749466): Cryphonectria parasitica EP155 unplaced genomic... (10% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_4.svg">
NW_022985510 (2150254-2227692): Neohortaea acidophila strain CBS 113389 unpla... (6% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r3c2_5.svg">
NW_022985463 (11775-102765): Mytilinidion resinicola strain CBS 304.34 unplac... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_6.svg">
NW_024467832 (4333687-4396473): Aspergillus wentii DTO 134E9 unplaced genomic... (6% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_7.svg">
NW_017253367 (1229921-1261277): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (20% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r3c2-download">Download graphic</button>
</div>
<div id="clusterblast-r3c2-svg"></div>

</div>
</div>

<div class="body-details-section r3c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-164"></div> <span class="help-tooltip" id="cb-known-help-164">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-165"></div> <span class="help-tooltip" id="cb-sub-help-165">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-167"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-167">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-pfam-details-svg" data-filename="scaffold_3_r3c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r3c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 3 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-490"></div> <span class="help-tooltip" id="region_svg-help-490">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c3-svg" data-filename="scaffold_3_r3c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 958,534 - 1,011,036 nt. (total: 52,503 nt)

<a class="cluster-rules-header" id="r3c3-rules-header" href="#r3c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r3c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-491"></div> <span class="help-tooltip" id="focus-panel-help-491">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c3-gene-table-details" data-name="r3c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c3-nrps_pks" data-name="r3c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c3-MIBiG-cluster-compare" data-name="r3c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c3-clusterblast" data-name="r3c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c3-knownclusterblast" data-name="r3c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c3-subclusterblast" data-name="r3c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c3-nrps-pks-bubbles" data-name="r3c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c3-pfam-details" data-name="r3c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c3-tigrfam-details" data-name="r3c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-176"></div> <span class="help-tooltip" id="gt-help-tooltip-help-176">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-168"></div> <span class="help-tooltip" id="nrps-domain-help-168">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-details-svg" data-filename="scaffold_3_r3c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-169"></div> <span class="help-tooltip" id="MIBiG-help-169">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c3"></div>
<div id="comparison-MIBiG-r3c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.51</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.51</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>Polyketide, NRP</td>

<td>xenoacremone A</td>


<td>Xenoacremonium sp. BF-2018a</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001563: 1377-52987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001563/index.html#r1c1" target="_new">BGC0001563</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>NRP, Polyketide</td>

<td>curvupallide-B</td>


<td>Cochliobolus pallescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>Polyketide, NRP</td>

<td>pyranoviolin A</td>


<td>Aspergillus violaceofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002250: 637-40051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002250/index.html#r1c1" target="_new">BGC0002250</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide, NRP</td>

<td>triticone D, triticone E, triticone A, triticone B, triticone F, triticone C</td>


<td>Pyrenophora tritici-repentis Pt-1C-BFP</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide, NRP</td>
<td>xenoacremone A</td>
<td>Xenoacremonium sp. BF-2018a</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001563: 1377-52987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001563/index.html#r1c1" target="_new">BGC0001563</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP, Polyketide</td>
<td>curvupallide-B</td>
<td>Cochliobolus pallescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide, NRP</td>
<td>pyranoviolin A</td>
<td>Aspergillus violaceofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002250: 637-40051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002250/index.html#r1c1" target="_new">BGC0002250</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide, NRP</td>
<td>triticone D, triticone E, triticone A, triticone B, triticone F, triticone C</td>
<td>Pyrenophora tritici-repentis Pt-1C-BFP</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-170"></div> <span class="help-tooltip" id="cb-general-help-170">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c3_1.svg">
NW_025764716 (588555-656757): Emericellopsis atlantica unplaced genomic scaff... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_2.svg">
NW_011942171 (2100194-2164118): Metarhizium robertsii ARSEF 23 MAA Scf 3, who... (25% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c3_3.svg">
NW_014574702 (214578-278514): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (24% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c3_4.svg">
NW_020290884 (537866-609889): Aspergillus niger CBS 101883 unplaced genomic s... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c3_5.svg">
NZ_LT607413 (6800632-6913222): Micromonospora echinospora strain DSM 43816 ch... (12% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r3c3_6.svg">
NW_014013670 (106490-178646): Trichoderma virens Gv29-8 chromosome Unknown TR... (18% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c3_7.svg">
NC_072850 (2022235-2153706): Fusarium oxysporum Fo47 chromosome XI, complete ... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_8.svg">
NC_036629 (2646284-2784115): Fusarium fujikuroi IMI 58289 draft genome, chrom... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_9.svg">
NW_020290907 (289711-383298): Aspergillus niger CBS 101883 unplaced genomic s... (6% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r3c3_10.svg">
NW_023336276 (543081-628370): Aspergillus tubingensis WU-2223L DNA, scaffold ... (8% of genes show similarity), NRPS,T1PKS,terpene
</option>

</select>
<button id= "clusterblast-r3c3-download">Download graphic</button>
</div>
<div id="clusterblast-r3c3-svg"></div>

</div>
</div>

<div class="body-details-section r3c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-171"></div> <span class="help-tooltip" id="cb-known-help-171">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-172"></div> <span class="help-tooltip" id="cb-sub-help-172">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-175"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-175">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-domain-bubble-svg" data-filename="scaffold_3_r3c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (958533 - 1011036): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r3c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-177"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-177">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-pfam-details-svg" data-filename="scaffold_3_r3c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-178"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-178">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-tigrfam-details-svg" data-filename="scaffold_3_r3c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c3-nrps_pks_products" data-name="r3c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c3-nrps_pks_monomers" data-name="r3c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-173"></div> <span class="help-tooltip" id="nrps-structures-help-173">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 958533 - 1011036:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c3-3" class="smiles-canvas" data-smiles="C([*])C(-O)NC(CCCN)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - Orn)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC(CCCN)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-174"></div> <span class="help-tooltip" id="nrps-monomers-help-174">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene310</span></strong>: pk - Orn
<div class="collapser collapser-target-input-path1-gene310 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>PKS_AT (556..869)</strong>: pk
<div class="collapser collapser-target-input-path1-gene310 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene310-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methoxymalonyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene310-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 88.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 72.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 60.7</dt>

<dd></dd><dt>Malonyl-CoA: 40.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 39.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 35.3</dt>

<dd></dd><dt>Propionyl-CoA: 35.1</dt>

<dd></dd><dt>Benzoyl-CoA: 31.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 23.9</dt>

<dd></dd><dt>fatty_acid: 22.6</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 17.5</dt>

<dd></dd><dt>CHC-CoA: 15.4</dt>

<dd></dd><dt>Acetyl-CoA: 11.5</dt>

<dd></dd><dt>inactive: 9.6</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2590..2998)</strong>: Orn
<div class="collapser collapser-target-input-path1-gene310 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Orn
<div class="collapser collapser-target-input-path1-gene310-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Long positively charged side chain (Orn, Lys, Arg)</dd>
<dt>Small clusters prediction:</dt>
<dd>Orn and hydroxy-Orn specific (Orn, OH-Orn)</dd>
<dt>Single AA prediction:</dt>
<dd>orn (Orn)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFDRFGSISK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>MalAc <span class="serif">NHDKFGSISK</span> (74% 8Å match)</dd>
<dd>Ser <span class="serif">DFDTFGGILK</span> (62% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 4 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-492"></div> <span class="help-tooltip" id="region_svg-help-492">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c4-svg" data-filename="scaffold_3_r3c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,645,613 - 1,706,689 nt. (total: 61,077 nt)

<a class="cluster-rules-header" id="r3c4-rules-header" href="#r3c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c4-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-493"></div> <span class="help-tooltip" id="focus-panel-help-493">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c4-gene-table-details" data-name="r3c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c4-MIBiG-cluster-compare" data-name="r3c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c4-clusterblast" data-name="r3c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c4-knownclusterblast" data-name="r3c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c4-subclusterblast" data-name="r3c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c4-pfam-details" data-name="r3c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c4-tigrfam-details" data-name="r3c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-183"></div> <span class="help-tooltip" id="gt-help-tooltip-help-183">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-179"></div> <span class="help-tooltip" id="MIBiG-help-179">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c4"></div>
<div id="comparison-MIBiG-r3c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001964: 2-41089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001964/index.html#r1c1" target="_new">BGC0001964</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.47, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>alkylpyrone-407, alkylpyrone-393</td>


<td>Cystobacterineae bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001443: 0-23990">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001443/index.html#r1c1" target="_new">BGC0001443</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>matlystatin A</td>


<td>Actinomadura atramentaria DSM 43919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001945: 511-47350">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001945/index.html#r1c1" target="_new">BGC0001945</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.41, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>chloromyxamide</td>


<td>Myxococcus sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001406: 39-82088">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001406/index.html#r1c1" target="_new">BGC0001406</a></td>





<td class="cc-heat-cell" title="0.15 (id:0.43, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>


<td>0.03</td>
<td>NRP</td>

<td>telomycin</td>


<td>Streptomyces canus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001964: 2-41089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001964/index.html#r1c1" target="_new">BGC0001964</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.47, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Polyketide</td>
<td>alkylpyrone-407, alkylpyrone-393</td>
<td>Cystobacterineae bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001443: 0-23990">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001443/index.html#r1c1" target="_new">BGC0001443</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP, Polyketide</td>
<td>matlystatin A</td>
<td>Actinomadura atramentaria DSM 43919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001945: 511-47350">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001945/index.html#r1c1" target="_new">BGC0001945</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.41, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>NRP</td>
<td>chloromyxamide</td>
<td>Myxococcus sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001406: 39-82088">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001406/index.html#r1c1" target="_new">BGC0001406</a></td>


<td class="cc-heat-cell" title="0.15 (id:0.43, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
<td>0.15</td>
<td>NRP</td>
<td>telomycin</td>
<td>Streptomyces canus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-180"></div> <span class="help-tooltip" id="cb-general-help-180">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c4_1.svg">
NW_016017392 (82602-175191): Sphaerulina musiva SO2202 unplaced genomic scaff... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_2.svg">
NC_070545 (1284871-1351441): Fusarium falciforme chromosome 2, complete sequence (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c4_3.svg">
NC_070530 (1234874-1301442): Fusarium keratoplasticum isolate Fu6.1 chromosom... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c4_4.svg">
NW_006917093 (3054358-3117993): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c4_5.svg">
NW_024468937 (5549232-5621994): Cucurbitaria berberidis CBS 394.84 unplaced g... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c4_6.svg">
NW_014574698 (207239-280433): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (7% of genes show similarity), NRPS,indole
</option>

<option value = "svg/clusterblast_r3c4_7.svg">
NW_011942151 (608845-681943): Metarhizium robertsii ARSEF 23 MAA Scf 11, whol... (8% of genes show similarity), NRPS,indole
</option>

<option value = "svg/clusterblast_r3c4_8.svg">
NW_024467531 (1115674-1206760): Aspergillus versicolor CBS 583.65 unplaced ge... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_9.svg">
NC_054857 (688182-779289): Aspergillus puulaauensis MK2 DNA, chromosome 1, ne... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_10.svg">
NW_024467868 (678273-769342): Aspergillus sydowii CBS 593.65 unplaced genomic... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c4-download">Download graphic</button>
</div>
<div id="clusterblast-r3c4-svg"></div>

</div>
</div>

<div class="body-details-section r3c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-181"></div> <span class="help-tooltip" id="cb-known-help-181">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-182"></div> <span class="help-tooltip" id="cb-sub-help-182">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-184"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-184">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-pfam-details-svg" data-filename="scaffold_3_r3c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-185"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-185">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-tigrfam-details-svg" data-filename="scaffold_3_r3c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r3c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 5 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-494"></div> <span class="help-tooltip" id="region_svg-help-494">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c5-svg" data-filename="scaffold_3_r3c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,103,594 - 2,164,593 nt. (total: 61,000 nt)

<a class="cluster-rules-header" id="r3c5-rules-header" href="#r3c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c5-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-495"></div> <span class="help-tooltip" id="focus-panel-help-495">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c5-gene-table-details" data-name="r3c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c5-MIBiG-cluster-compare" data-name="r3c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c5-clusterblast" data-name="r3c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c5-knownclusterblast" data-name="r3c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c5-subclusterblast" data-name="r3c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c5-pfam-details" data-name="r3c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c5-tigrfam-details" data-name="r3c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-190"></div> <span class="help-tooltip" id="gt-help-tooltip-help-190">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-186"></div> <span class="help-tooltip" id="MIBiG-help-186">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c5"></div>
<div id="comparison-MIBiG-r3c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000622: 395-4938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000622/index.html#r1c1" target="_new">BGC0000622</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>RiPP</td>

<td>megacin</td>


<td>Bacillus megaterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>auriculamide</td>


<td>Herpetosiphon aurantiacus DSM 785</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002509: 94-21985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002509/index.html#r1c1" target="_new">BGC0002509</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>NRP</td>

<td>montamide A, capsimycin, clifednamide A, frontalamide A, combamide A</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002294: 412-30500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002294/index.html#r1c1" target="_new">BGC0002294</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other</td>

<td>streptozotocin</td>


<td>Streptomyces achromogenes subsp. streptozoticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000622: 395-4938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000622/index.html#r1c1" target="_new">BGC0000622</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>RiPP</td>
<td>megacin</td>
<td>Bacillus megaterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>auriculamide</td>
<td>Herpetosiphon aurantiacus DSM 785</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002509: 94-21985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002509/index.html#r1c1" target="_new">BGC0002509</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>NRP</td>
<td>montamide A, capsimycin, clifednamide A, frontalamide A, combamide A</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002294: 412-30500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002294/index.html#r1c1" target="_new">BGC0002294</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other</td>
<td>streptozotocin</td>
<td>Streptomyces achromogenes subsp. streptozoticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-187"></div> <span class="help-tooltip" id="cb-general-help-187">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c5_1.svg">
NW_006271977 (184210-275183): Cordyceps militaris CM01 unplaced genomic scaff... (51% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_2.svg">
NW_019154045 (358401-688746): Pochonia chlamydosporia 170 chromosome Unknown ... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_3.svg">
NW_011942171 (314568-405746): Metarhizium robertsii ARSEF 23 MAA Scf 3, whole... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_4.svg">
NW_025091265 (422738-551496): Aspergillus udagawae IFM 46973 DNA, scaffold 2,... (4% of genes show similarity), T1PKS,fungal-RiPP-like,indole
</option>

<option value = "svg/clusterblast_r3c5_5.svg">
NW_020291606 (579604-818783): Aspergillus piperis CBS 112811 unplaced genomic... (2% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_6.svg">
NW_020292039 (3332166-3473157): Aspergillus vadensis CBS 113365 unplaced geno... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_7.svg">
NW_022985457 (182988-273904): Mytilinidion resinicola strain CBS 304.34 unpla... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_8.svg">
NW_014574716 (2113724-2204326): Metarhizium brunneum ARSEF 3297 chromosome Un... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_9.svg">
NW_025408683 (24432-115490): Hirsutella rhossiliensis strain HR02 chromosome ... (20% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_10.svg">
NW_014574622 (178902-269178): Penicillium digitatum Pd1 unplaced genomic scaf... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c5-download">Download graphic</button>
</div>
<div id="clusterblast-r3c5-svg"></div>

</div>
</div>

<div class="body-details-section r3c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-188"></div> <span class="help-tooltip" id="cb-known-help-188">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-189"></div> <span class="help-tooltip" id="cb-sub-help-189">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-191"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-191">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-pfam-details-svg" data-filename="scaffold_3_r3c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-192"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-192">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-tigrfam-details-svg" data-filename="scaffold_3_r3c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r3c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 6 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-496"></div> <span class="help-tooltip" id="region_svg-help-496">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c6-svg" data-filename="scaffold_3_r3c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,298,444 - 2,342,900 nt. (total: 44,457 nt)

<a class="cluster-rules-header" id="r3c6-rules-header" href="#r3c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c6-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r3c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-497"></div> <span class="help-tooltip" id="focus-panel-help-497">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c6-gene-table-details" data-name="r3c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c6-nrps_pks" data-name="r3c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c6-MIBiG-cluster-compare" data-name="r3c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c6-clusterblast" data-name="r3c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c6-knownclusterblast" data-name="r3c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c6-subclusterblast" data-name="r3c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c6-pfam-details" data-name="r3c6-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-199"></div> <span class="help-tooltip" id="gt-help-tooltip-help-199">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-193"></div> <span class="help-tooltip" id="nrps-domain-help-193">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c6-details-svg" data-filename="scaffold_3_r3c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-194"></div> <span class="help-tooltip" id="MIBiG-help-194">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c6"></div>
<div id="comparison-MIBiG-r3c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>solanapyrone D</td>


<td>Alternaria solani</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858: 715-19483">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002619: 0-42148">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002619/index.html#r1c1" target="_new">BGC0002619</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>mycophenolic acid</td>


<td>Penicillium brevicompactum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001578: 0-31150">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001578/index.html#r1c1" target="_new">BGC0001578</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>duclauxin</td>


<td>Talaromyces stipitatus ATCC 10500</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858: 715-19483">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002619: 0-42148">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002619/index.html#r1c1" target="_new">BGC0002619</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>mycophenolic acid</td>
<td>Penicillium brevicompactum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001578: 0-31150">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001578/index.html#r1c1" target="_new">BGC0001578</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>duclauxin</td>
<td>Talaromyces stipitatus ATCC 10500</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-195"></div> <span class="help-tooltip" id="cb-general-help-195">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c6_1.svg">
NC_049565 (1715696-1783101): Talaromyces rugulosus chromosome V, complete seq... (29% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c6_2.svg">
NW_022985016 (85063-152593): Dothidotthia symphoricarpi CBS 119687 unplaced g... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c6_3.svg">
NW_006271969 (4860304-4927848): Cordyceps militaris CM01 unplaced genomic sca... (27% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c6_4.svg">
NW_021167090 (669974-737132): Sodiomyces alkalinus F11 unplaced genomic scaff... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c6_5.svg">
NW_024467836 (90960-158318): Aspergillus wentii DTO 134E9 unplaced genomic sc... (6% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c6_6.svg">
NW_020292051 (280608-471430): Aspergillus vadensis CBS 113365 unplaced genomi... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c6_7.svg">
NW_020291592 (339371-529932): Aspergillus piperis CBS 112811 unplaced genomic... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c6_8.svg">
NC_054853 (2681752-2888163): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (3% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c6_9.svg">
NW_020290890 (450315-638141): Aspergillus niger CBS 101883 unplaced genomic s... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c6_10.svg">
NW_023336287 (400215-591232): Aspergillus tubingensis WU-2223L DNA, scaffold ... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c6-download">Download graphic</button>
</div>
<div id="clusterblast-r3c6-svg"></div>

</div>
</div>

<div class="body-details-section r3c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-196"></div> <span class="help-tooltip" id="cb-known-help-196">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-197"></div> <span class="help-tooltip" id="cb-sub-help-197">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-200"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-200">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c6-pfam-details-svg" data-filename="scaffold_3_r3c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c6-nrps_pks_monomers" data-name="r3c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r3c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-198"></div> <span class="help-tooltip" id="nrps-monomers-help-198">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene727</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene727 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (207..470)</strong>: pk
<div class="collapser collapser-target-input-path1-gene727 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene727-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
<dd>Ethylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene727-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 65.6</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 63.1</dt>

<dd></dd><dt>Malonyl-CoA: 52.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 43.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 34.2</dt>

<dd></dd><dt>Propionyl-CoA: 22.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 17.7</dt>

<dd></dd><dt>inactive: 15.0</dt>

<dd></dd><dt>fatty_acid: 12.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 10.7</dt>

<dd></dd><dt>Benzoyl-CoA: 10.3</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r3c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 7 - isocyanide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-498"></div> <span class="help-tooltip" id="region_svg-help-498">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c7-svg" data-filename="scaffold_3_r3c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,381,358 - 3,423,497 nt. (total: 42,140 nt)

<a class="cluster-rules-header" id="r3c7-rules-header" href="#r3c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c7-rules">isocyanide: cds(TauD and DIT1_PvcA)</div>

<div class="region-svg-container">
<div id='r3c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-499"></div> <span class="help-tooltip" id="focus-panel-help-499">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c7-gene-table-details" data-name="r3c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c7-MIBiG-cluster-compare" data-name="r3c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c7-clusterblast" data-name="r3c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c7-knownclusterblast" data-name="r3c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c7-subclusterblast" data-name="r3c7-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r3c7-pfam-details" data-name="r3c7-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-205"></div> <span class="help-tooltip" id="gt-help-tooltip-help-205">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-201"></div> <span class="help-tooltip" id="MIBiG-help-201">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c7"></div>
<div id="comparison-MIBiG-r3c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">isocyanide</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000923: 943-6060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000923/index.html#r1c1" target="_new">BGC0000923</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>paerucumarin</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001594: 0-38842">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001594/index.html#r1c1" target="_new">BGC0001594</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.46, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Alkaloid</td>

<td>fischerindole L</td>


<td>Fischerella muscicola UTEX 1829</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Alkaloid</td>

<td>fischerindole</td>


<td>Fischerella sp. SAG 46.79</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Other</td>

<td>schizokinen</td>


<td>Leptolyngbya sp. PCC 7376</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>


<td>Hapalosiphon welwitschii UTEX B 1830</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Alkaloid</td>

<td>ambiguine H isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>oviedomycin</td>


<td>Streptomyces antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002066: 0-32436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002066/index.html#r1c1" target="_new">BGC0002066</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>harzianopyridone</td>


<td>Trichoderma harzianum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000923: 943-6060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000923/index.html#r1c1" target="_new">BGC0000923</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>paerucumarin</td>
<td>Pseudomonas aeruginosa PAO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001594: 0-38842">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001594/index.html#r1c1" target="_new">BGC0001594</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.46, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Alkaloid</td>
<td>fischerindole L</td>
<td>Fischerella muscicola UTEX 1829</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Alkaloid</td>
<td>fischerindole</td>
<td>Fischerella sp. SAG 46.79</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Other</td>
<td>schizokinen</td>
<td>Leptolyngbya sp. PCC 7376</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>
<td>Hapalosiphon welwitschii UTEX B 1830</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>
<td>Fischerella ambigua UTEX 1903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Alkaloid</td>
<td>ambiguine H isonitrile</td>
<td>Fischerella ambigua UTEX 1903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>oviedomycin</td>
<td>Streptomyces antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002066: 0-32436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002066/index.html#r1c1" target="_new">BGC0002066</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>harzianopyridone</td>
<td>Trichoderma harzianum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-202"></div> <span class="help-tooltip" id="cb-general-help-202">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c7_1.svg">
NC_063070 (2362618-2424749): Purpureocillium takamizusanense chromosome 3, co... (47% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c7_2.svg">
NW_011942149 (5418619-5480759): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (38% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c7_3.svg">
NW_014574715 (467040-529181): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (38% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c7_4.svg">
NC_035790 (6741031-6803169): Pochonia chlamydosporia 170 chromosome 1, whole ... (31% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c7_5.svg">
NC_054389 (8457870-8566158): Drechmeria coniospora strain ARSEF 6962 chromoso... (21% of genes show similarity), T1PKS,isocyanide
</option>

<option value = "svg/clusterblast_r3c7_6.svg">
NW_006711178 (114520-207586): Trichoderma reesei QM6a unplaced genomic scaffo... (16% of genes show similarity), isocyanide,terpene
</option>

<option value = "svg/clusterblast_r3c7_7.svg">
NC_057319 (4219732-4281896): Ustilaginoidea virens chromosome 4, complete seq... (19% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c7_8.svg">
NW_006271969 (5556555-5618595): Cordyceps militaris CM01 unplaced genomic sca... (30% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c7_9.svg">
NW_025953794 (771521-833710): Annulohypoxylon truncatum strain CBS 140777 unp... (13% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c7_10.svg">
NW_025953253 (1464233-1545264): Annulohypoxylon maeteangense strain CBS 12383... (11% of genes show similarity), NRPS,isocyanide
</option>

</select>
<button id= "clusterblast-r3c7-download">Download graphic</button>
</div>
<div id="clusterblast-r3c7-svg"></div>

</div>
</div>

<div class="body-details-section r3c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-203"></div> <span class="help-tooltip" id="cb-known-help-203">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-204"></div> <span class="help-tooltip" id="cb-sub-help-204">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r3c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-206"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-206">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c7-pfam-details-svg" data-filename="scaffold_3_r3c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r3c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 8 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-500"></div> <span class="help-tooltip" id="region_svg-help-500">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c8-svg" data-filename="scaffold_3_r3c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,827,689 - 3,888,385 nt. (total: 60,697 nt)

<a class="cluster-rules-header" id="r3c8-rules-header" href="#r3c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c8-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-501"></div> <span class="help-tooltip" id="focus-panel-help-501">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c8-gene-table-details" data-name="r3c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c8-MIBiG-cluster-compare" data-name="r3c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c8-clusterblast" data-name="r3c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c8-knownclusterblast" data-name="r3c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c8-subclusterblast" data-name="r3c8-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c8-pfam-details" data-name="r3c8-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-211"></div> <span class="help-tooltip" id="gt-help-tooltip-help-211">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-207"></div> <span class="help-tooltip" id="MIBiG-help-207">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c8"></div>
<div id="comparison-MIBiG-r3c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>nidulanin A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001907: 74-38528">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001907/index.html#r1c1" target="_new">BGC0001907</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>chromane</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.44, order:0.27, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>NRP</td>
<td>nidulanin A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001907: 74-38528">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001907/index.html#r1c1" target="_new">BGC0001907</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>chromane</td>
<td>Daldinia eschscholzii IFB-TL01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.44, order:0.27, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-208"></div> <span class="help-tooltip" id="cb-general-help-208">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c8_1.svg">
NW_023336279 (145289-236294): Aspergillus tubingensis WU-2223L DNA, scaffold ... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c8_2.svg">
NW_020291612 (327439-419782): Aspergillus piperis CBS 112811 unplaced genomic... (8% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c8_3.svg">
NW_020290621 (328465-419468): Aspergillus saccharolyticus JOP 1030-1 unplaced... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c8_4.svg">
NW_020292053 (616043-708384): Aspergillus vadensis CBS 113365 unplaced genomi... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c8_5.svg">
NW_020290899 (145522-237868): Aspergillus niger CBS 101883 unplaced genomic s... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c8_6.svg">
NW_024468978 (129841-221990): Aspergillus fijiensis CBS 313.89 unplaced genom... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c8_7.svg">
NW_020292127 (15270-136339): Aspergillus ibericus CBS 121593 unplaced genomic... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r3c8_8.svg">
NT_166521 (362145-454491): Aspergillus niger CBS 513.88 supercontig An05 (9% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c8_9.svg">
NC_054856 (3727884-3894446): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (4% of genes show similarity), NI-siderophore,NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c8_10.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c8-download">Download graphic</button>
</div>
<div id="clusterblast-r3c8-svg"></div>

</div>
</div>

<div class="body-details-section r3c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-209"></div> <span class="help-tooltip" id="cb-known-help-209">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-210"></div> <span class="help-tooltip" id="cb-sub-help-210">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-212"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-212">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c8-pfam-details-svg" data-filename="scaffold_3_r3c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r3c9' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 9 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-502"></div> <span class="help-tooltip" id="region_svg-help-502">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region009.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c9-svg" data-filename="scaffold_3_r3c9.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,052,500 - 4,096,745 nt. (total: 44,246 nt)

<a class="cluster-rules-header" id="r3c9-rules-header" href="#r3c9">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c9-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c9-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-503"></div> <span class="help-tooltip" id="focus-panel-help-503">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c9">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c9-gene-table-details" data-name="r3c9-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c9-nrps_pks" data-name="r3c9-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c9-MIBiG-cluster-compare" data-name="r3c9-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c9-clusterblast" data-name="r3c9-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c9-knownclusterblast" data-name="r3c9-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c9-subclusterblast" data-name="r3c9-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c9-pfam-details" data-name="r3c9-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c9-tigrfam-details" data-name="r3c9-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c9-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-219"></div> <span class="help-tooltip" id="gt-help-tooltip-help-219">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c9">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c9-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c9-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c9">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c9-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-213"></div> <span class="help-tooltip" id="nrps-domain-help-213">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c9-details-svg" data-filename="scaffold_3_r3c9_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c9-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c9-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-214"></div> <span class="help-tooltip" id="MIBiG-help-214">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c9-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c9-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c9"></div>
<div id="comparison-MIBiG-r3c9-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>pyrrolomycin</td>


<td>Streptomyces sp. UC 11065</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>xenematide</td>


<td>Xenorhabdus nematophila AN6/1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404: 1968-56662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>penicillin</td>


<td>Penicillium chrysogenum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>sansalvamide</td>


<td>[Nectria] haematococca mpVI 77-13-4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c9-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>xenematide</td>
<td>Xenorhabdus nematophila AN6/1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404: 1968-56662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>penicillin</td>
<td>Penicillium chrysogenum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>sansalvamide</td>
<td>[Nectria] haematococca mpVI 77-13-4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c9-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-215"></div> <span class="help-tooltip" id="cb-general-help-215">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c9-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c9_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c9_1.svg">
NW_006271969 (2236886-2301135): Cordyceps militaris CM01 unplaced genomic sca... (43% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c9_2.svg">
NZ_LT607413 (4729322-4815787): Micromonospora echinospora strain DSM 43816 ch... (4% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r3c9_3.svg">
NZ_AP023438 (6154260-6239493): Micromonospora sagamiensis strain JCM 3310 chr... (3% of genes show similarity), NRPS,T1PKS,betalactone
</option>

<option value = "svg/clusterblast_r3c9_4.svg">
NZ_LT607413 (6504232-6664363): Micromonospora echinospora strain DSM 43816 ch... (1% of genes show similarity), NRPS,PKS-like,T1PKS,betalactone,lassopeptide
</option>

<option value = "svg/clusterblast_r3c9_5.svg">
NZ_PNXQ01000016 (500910-548524): Paenibacillus terrae strain PIC167 Contig 01... (5% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r3c9_6.svg">
NZ_LN850107 (1679126-1853953): Alloactinosynnema sp. L-07 isolate Alloactinos... (2% of genes show similarity), NRP-metallophore,NRPS,NRPS-like,other
</option>

<option value = "svg/clusterblast_r3c9_7.svg">
NC_030959 (128983-262697): Colletotrichum higginsianum IMI 349063 chromosome ... (5% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c9_8.svg">
NZ_MAUJ01000002 (439421-527978): Pseudoalteromonas luteoviolacea strain IPB1 ... (3% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r3c9_9.svg">
NC_017672 (4614434-4764897): Paenibacillus mucilaginosus K02, complete sequence (3% of genes show similarity), HR-T2PKS,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c9_10.svg">
NZ_CP016279 (5371274-5445973): Streptomyces griseochromogenes strain ATCC 145... (2% of genes show similarity), NRPS,NRPS-like
</option>

</select>
<button id= "clusterblast-r3c9-download">Download graphic</button>
</div>
<div id="clusterblast-r3c9-svg"></div>

</div>
</div>

<div class="body-details-section r3c9-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-216"></div> <span class="help-tooltip" id="cb-known-help-216">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c9-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-217"></div> <span class="help-tooltip" id="cb-sub-help-217">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c9-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-220"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-220">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c9-pfam-details-svg" data-filename="scaffold_3_r3c9_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c9-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c9-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-221"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-221">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c9-tigrfam-details-svg" data-filename="scaffold_3_r3c9_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c9-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c9-nrps_pks_monomers" data-name="r3c9-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r3c9-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-218"></div> <span class="help-tooltip" id="nrps-monomers-help-218">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1242</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1242 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (227..662)</strong>: X
<div class="collapser collapser-target-input-path1-gene1242 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene1242-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLMFVGIMCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAMLVGAACK</span> (59% 8Å match)</dd>
<dd>Hty <span class="serif">DLGFTGAVCK</span> (56% 8Å match)</dd>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r3c10' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 10 - NRPS,T1PKS,fungal-RiPP-like,isocyanide-nrp <div class="help-container"> <div class="help-icon" data-id="region_svg-help-504"></div> <span class="help-tooltip" id="region_svg-help-504">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region010.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c10-svg" data-filename="scaffold_3_r3c10.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,402,068 - 4,530,840 nt. (total: 128,773 nt)

<a class="cluster-rules-header" id="r3c10-rules-header" href="#r3c10">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c10-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>fungal-RiPP-like: DUF3328<br>isocyanide-nrp: (DIT1_PvcA and minimum(2, [AMP-binding, PP-binding, Transferase]))</div>

<div class="region-svg-container">
<div id='r3c10-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-505"></div> <span class="help-tooltip" id="focus-panel-help-505">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c10">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c10-gene-table-details" data-name="r3c10-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c10-nrps_pks" data-name="r3c10-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c10-MIBiG-cluster-compare" data-name="r3c10-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c10-clusterblast" data-name="r3c10-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c10-knownclusterblast" data-name="r3c10-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c10-subclusterblast" data-name="r3c10-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r3c10-nrps-pks-bubbles" data-name="r3c10-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c10-pfam-details" data-name="r3c10-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c10-tigrfam-details" data-name="r3c10-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c10-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-230"></div> <span class="help-tooltip" id="gt-help-tooltip-help-230">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c10">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c10-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c10-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c10">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c10-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-222"></div> <span class="help-tooltip" id="nrps-domain-help-222">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c10-details-svg" data-filename="scaffold_3_r3c10_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c10-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c10-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-223"></div> <span class="help-tooltip" id="MIBiG-help-223">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c10-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c10-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c10"></div>
<div id="comparison-MIBiG-r3c10-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">fungal-RiPP-like</span></th>

<th><span class="cc-heat-product">isocyanide-nrp</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001049: 0-19176">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001049/index.html#r1c1" target="_new">BGC0001049</a></td>





<td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>






<td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>1.17</td>
<td>NRP, Polyketide</td>

<td>tenellin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001218: 1349-26398">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001218/index.html#r1c1" target="_new">BGC0001218</a></td>





<td class="cc-heat-cell" title="0.74 (id:0.60, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>






<td class="cc-heat-cell" title="0.74 (id:0.60, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>






<td class="cc-heat-cell" title="0.34 (id:0.68, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.96</td>
<td>NRP, Polyketide</td>

<td>fumosorinone</td>


<td>Cordyceps fumosorosea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001136: 0-20147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001136/index.html#r1c1" target="_new">BGC0001136</a></td>





<td class="cc-heat-cell" title="0.77 (id:0.58, order:0.80, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>






<td class="cc-heat-cell" title="0.77 (id:0.58, order:0.80, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.94</td>
<td>NRP, Polyketide</td>

<td>desmethylbassianin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.70</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.68</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.67</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.67</td>
<td>NRP, Polyketide</td>

<td>chaetoglobosins</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.65</td>
<td>Polyketide, NRP</td>

<td>xenoacremone A</td>


<td>Xenoacremonium sp. BF-2018a</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.65</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.64</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c10-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001049: 0-19176">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001049/index.html#r1c1" target="_new">BGC0001049</a></td>


<td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
<td>0.83</td>
<td>NRP, Polyketide</td>
<td>tenellin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001136: 0-20147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001136/index.html#r1c1" target="_new">BGC0001136</a></td>


<td class="cc-heat-cell" title="0.77 (id:0.58, order:0.80, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
<td>0.77</td>
<td>NRP, Polyketide</td>
<td>desmethylbassianin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001218: 1349-26398">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001218/index.html#r1c1" target="_new">BGC0001218</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.61, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP, Polyketide</td>
<td>fumosorinone</td>
<td>Cordyceps fumosorosea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>chaetoglobosins</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide, NRP</td>
<td>xenoacremone A</td>
<td>Xenoacremonium sp. BF-2018a</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c10-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-224"></div> <span class="help-tooltip" id="cb-general-help-224">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c10-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c10_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c10_1.svg">
NW_006271975 (27980-100581): Cordyceps militaris CM01 unplaced genomic scaffo... (50% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c10_2.svg">
NW_006271975 (307084-386893): Cordyceps militaris CM01 unplaced genomic scaff... (29% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r3c10_3.svg">
NC_036629 (2646284-2784115): Fusarium fujikuroi IMI 58289 draft genome, chrom... (10% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c10_4.svg">
NC_072850 (2022235-2153706): Fusarium oxysporum Fo47 chromosome XI, complete ... (9% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c10_5.svg">
NW_020290887 (673812-764439): Aspergillus niger CBS 101883 unplaced genomic s... (17% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c10_6.svg">
NW_014013670 (106490-178646): Trichoderma virens Gv29-8 chromosome Unknown TR... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c10_7.svg">
NC_030959 (4385741-4495530): Colletotrichum higginsianum IMI 349063 chromosom... (11% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c10_8.svg">
NW_006911257 (1345885-1409023): Baudoinia panamericana UAMH 10762 unplaced ge... (15% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c10_9.svg">
NC_038014 (6525771-6597702): Fusarium venenatum strain A3/5 genome assembly, ... (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c10_10.svg">
NC_064672 (6807692-6886490): Colletotrichum lupini chromosome 1, complete seq... (25% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r3c10-download">Download graphic</button>
</div>
<div id="clusterblast-r3c10-svg"></div>

</div>
</div>

<div class="body-details-section r3c10-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-225"></div> <span class="help-tooltip" id="cb-known-help-225">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c10-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c10_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c10_1.svg">
fumosorinone
</option>

<option value = "svg/knownclusterblast_r3c10_2.svg">
desmethylbassianin
</option>

<option value = "svg/knownclusterblast_r3c10_3.svg">
tenellin
</option>

<option value = "svg/knownclusterblast_r3c10_4.svg">
ilicicolin H/ilicicolin J/8-epi-ilicicolin H
</option>

<option value = "svg/knownclusterblast_r3c10_5.svg">
ilicicolin H
</option>

<option value = "svg/knownclusterblast_r3c10_6.svg">
atpenin B
</option>

</select>
<button id="knownclusterblast-r3c10-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c10-svg"></div>

</div>
</div>

<div class="body-details-section r3c10-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-226"></div> <span class="help-tooltip" id="cb-sub-help-226">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r3c10-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-229"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-229">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c10-domain-bubble-svg" data-filename="scaffold_3_r3c10_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c10-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC10">
Candidate 10 (4402067 - 4530840): neighbouring NRPS-T1PKS-fungal-RiPP-like-isocyanide-nrp
</option>



<option value="CC11">
Candidate 11 (4402067 - 4454618): chemical hybrid T1PKS-NRPS
</option>



<option value="CC13">
Candidate 13 (4475938 - 4530840): single isocyanide-nrp
</option>


</select>

<div class="details-svg" id="r3c10-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c10-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c10-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-231"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-231">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c10-pfam-details-svg" data-filename="scaffold_3_r3c10_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c10-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c10-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-232"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-232">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c10-tigrfam-details-svg" data-filename="scaffold_3_r3c10_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c10-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c10-nrps_pks_products" data-name="r3c10-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c10-nrps_pks_monomers" data-name="r3c10-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r3c10-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-227"></div> <span class="help-tooltip" id="nrps-structures-help-227">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 10, location 4402067 - 4530840:
<div class="collapser collapser-target-CC10 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c10-10" class="smiles-canvas" data-smiles="NC([*])C(=O)C([*])C(-O)(C)NC(Cc1ccc(O)cc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (Me-pk - Tyr)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=me-pk,tyr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=me-pk*,tyr*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c10-10-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C([*])C(-O)(C)NC(Cc1ccc(O)cc1)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 11, location 4402067 - 4454618:
<div class="collapser collapser-target-CC11 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c10-11" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC(Cc1ccc(O)cc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk - Tyr)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk,tyr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*,tyr*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c10-11-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC(Cc1ccc(O)cc1)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 13, location 4475938 - 4530840:
<div class="collapser collapser-target-CC13 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c10-13" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c10-13-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c10-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-228"></div> <span class="help-tooltip" id="nrps-monomers-help-228">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1365</span></strong>: pk - Tyr
<div class="collapser collapser-target-input-path1-gene1365 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>PKS_AT (587..907)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1365 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1365-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>2-Methylbutyryl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1365-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 100.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 84.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 68.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 58.8</dt>

<dd></dd><dt>Malonyl-CoA: 55.4</dt>

<dd></dd><dt>Benzoyl-CoA: 49.9</dt>

<dd></dd><dt>Propionyl-CoA: 46.7</dt>

<dd></dd><dt>Acetyl-CoA: 33.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 23.2</dt>

<dd></dd><dt>CHC-CoA: 21.6</dt>

<dd></dd><dt>fatty_acid: 16.7</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 15.9</dt>

<dd></dd><dt>inactive: 11.6</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>













<dd><strong>AMP-binding (3164..3579)</strong>: Tyr
<div class="collapser collapser-target-input-path1-gene1365 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Tyr
<div class="collapser collapser-target-input-path1-gene1365-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMVICGCAAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DMVICGCAAK</span> (97% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene1389</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1389 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (523..811)</strong>: X
<div class="collapser collapser-target-input-path1-gene1389 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene1389-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GAMSCSLFIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>





<div class="page" id='r4c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-506"></div> <span class="help-tooltip" id="region_svg-help-506">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c1-svg" data-filename="scaffold_4_r4c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 248,399 - 293,974 nt. (total: 45,576 nt)

<a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-507"></div> <span class="help-tooltip" id="focus-panel-help-507">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c1-gene-table-details" data-name="r4c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c1-nrps-pks-bubbles" data-name="r4c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c1-tigrfam-details" data-name="r4c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-241"></div> <span class="help-tooltip" id="gt-help-tooltip-help-241">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-233"></div> <span class="help-tooltip" id="nrps-domain-help-233">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="scaffold_4_r4c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-234"></div> <span class="help-tooltip" id="MIBiG-help-234">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c1"></div>
<div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>curacomycin</td>


<td>Streptomyces curacoi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002248: 5-21557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002248/index.html#r1c1" target="_new">BGC0002248</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Terpene, NRP</td>

<td>flavunoidine</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002606: 0-21662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002606/index.html#r1c1" target="_new">BGC0002606</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP, Polyketide</td>

<td>ochratoxin A</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>curacomycin</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002248: 5-21557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002248/index.html#r1c1" target="_new">BGC0002248</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Terpene, NRP</td>
<td>flavunoidine</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002606: 0-21662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002606/index.html#r1c1" target="_new">BGC0002606</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP, Polyketide</td>
<td>ochratoxin A</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-235"></div> <span class="help-tooltip" id="cb-general-help-235">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c1_1.svg">
NZ_AP021903 (2769248-2846759): Pseudomonas sp. Seg1 chromosome, complete genome (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_2.svg">
NZ_JACOPX010000001 (291706-369233): Pseudomonas neuropathica strain P155 Seq1... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_3.svg">
NZ_CABVJC010000014 (34930-112451): Pseudomonas fluorescens isolate PS941, who... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_4.svg">
NZ_CP087175 (3650521-3728046): Pseudomonas sp. B21-041 chromosome, complete g... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_5.svg">
NZ_CP073104 (60305-137835): Pseudomonas sp. SCA2728.1 7 chromosome (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_6.svg">
NZ_CABVHD010000011 (38525-116053): Pseudomonas fluorescens isolate PS619, who... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_7.svg">
NZ_CP077089 (4029858-4107383): Pseudomonas tensinigenes strain ZA 5.3 chromos... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_8.svg">
NZ_CP019432 (3905331-3982838): Pseudomonas sp. S49 chromosome, complete genome (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_9.svg">
NZ_CP054880 (3275786-3353311): Pseudomonas sp. 43A chromosome, complete genome (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c1_10.svg">
NZ_CP071586 (4043138-4120663): Pseudomonas germanica strain FIT28 chromosome,... (8% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r4c1-download">Download graphic</button>
</div>
<div id="clusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-236"></div> <span class="help-tooltip" id="cb-known-help-236">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-237"></div> <span class="help-tooltip" id="cb-sub-help-237">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-240"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-240">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-domain-bubble-svg" data-filename="scaffold_4_r4c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (248398 - 293974): single NRPS-like
</option>


</select>

<div class="details-svg" id="r4c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-242"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-242">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="scaffold_4_r4c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-243"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-243">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-tigrfam-details-svg" data-filename="scaffold_4_r4c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c1-nrps_pks_monomers" data-name="r4c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-238"></div> <span class="help-tooltip" id="nrps-structures-help-238">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 248398 - 293974:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-239"></div> <span class="help-tooltip" id="nrps-monomers-help-239">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene86</span></strong>: X
<div class="collapser collapser-target-input-path1-gene86 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (220..468)</strong>: X
<div class="collapser collapser-target-input-path1-gene86 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene86-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DTTDVSLAVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys or 4S-ohLys <span class="serif">DVTEVSLCAK</span> (65% 8Å match)</dd>
<dd>bLys <span class="serif">DTENVGTAVK</span> (41% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-508"></div> <span class="help-tooltip" id="region_svg-help-508">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c2-svg" data-filename="scaffold_4_r4c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,417,666 - 1,478,319 nt. (total: 60,654 nt)

<a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-509"></div> <span class="help-tooltip" id="focus-panel-help-509">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c2-gene-table-details" data-name="r4c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c2-clusterblast" data-name="r4c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c2-pfam-details" data-name="r4c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c2-tigrfam-details" data-name="r4c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-248"></div> <span class="help-tooltip" id="gt-help-tooltip-help-248">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-244"></div> <span class="help-tooltip" id="MIBiG-help-244">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c2"></div>
<div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002630: 0-4291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002630/index.html#r1c1" target="_new">BGC0002630</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>thatisin</td>


<td>Lysobacter antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501: 0-7208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>catenulipeptin</td>


<td>Catenulispora acidiphila DSM 44928</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002698: 0-12831">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002698/index.html#r1c1" target="_new">BGC0002698</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>RiPP</td>

<td>phaeornamide</td>


<td>Pseudophaeobacter arcticus DSM 23566</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496: 0-7345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>RiPP</td>

<td>AmfS</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000539: 155-11740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000539/index.html#r1c1" target="_new">BGC0000539</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>RiPP</td>

<td>nukacin ISK-1</td>


<td>Staphylococcus warneri</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>cytolysin ClyLl, cytolysin ClyLs</td>


<td>Plasmid pAD1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>carnolysin A1, carnolysin A2</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002630: 0-4291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002630/index.html#r1c1" target="_new">BGC0002630</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>thatisin</td>
<td>Lysobacter antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501: 0-7208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>catenulipeptin</td>
<td>Catenulispora acidiphila DSM 44928</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002698: 0-12831">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002698/index.html#r1c1" target="_new">BGC0002698</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>RiPP</td>
<td>phaeornamide</td>
<td>Pseudophaeobacter arcticus DSM 23566</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496: 0-7345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>RiPP</td>
<td>AmfS</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000539: 155-11740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000539/index.html#r1c1" target="_new">BGC0000539</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>RiPP</td>
<td>nukacin ISK-1</td>
<td>Staphylococcus warneri</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>cytolysin ClyLl, cytolysin ClyLs</td>
<td>Plasmid pAD1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>carnolysin A1, carnolysin A2</td>
<td>Carnobacterium maltaromaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-245"></div> <span class="help-tooltip" id="cb-general-help-245">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c2_1.svg">
NW_022156719 (1039998-1181379): Pyricularia grisea strain NI907 chromosome V ... (19% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_2.svg">
NC_064678 (3752676-3843730): Colletotrichum lupini chromosome 5, complete seq... (21% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_3.svg">
NW_017263602 (8884-99481): Mollisia scopiformis unplaced genomic scaffold LY8... (14% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_4.svg">
NW_007360999 (1055008-1145942): Glarea lozoyensis ATCC 20868 chromosome Unkno... (18% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_5.svg">
NW_017264196 (550250-647543): Xylona heveae TC161 unplaced genomic scaffold L... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_6.svg">
NW_025953267 (201738-434522): Annulohypoxylon maeteangense strain CBS 123835 ... (3% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_7.svg">
NW_025953254 (723627-790553): Annulohypoxylon maeteangense strain CBS 123835 ... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c2_8.svg">
NW_025953815 (1874-68796): Annulohypoxylon truncatum strain CBS 140777 unplac... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c2_9.svg">
NW_026250019 (1727835-1794739): Hypoxylon trugodes strain CBS 135444 unplaced... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c2_10.svg">
NW_025953049 (269746-336708): Daldinia caldariorum strain CBS 122874 unplaced... (25% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r4c2-download">Download graphic</button>
</div>
<div id="clusterblast-r4c2-svg"></div>

</div>
</div>

<div class="body-details-section r4c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-246"></div> <span class="help-tooltip" id="cb-known-help-246">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-247"></div> <span class="help-tooltip" id="cb-sub-help-247">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-249"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-249">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-pfam-details-svg" data-filename="scaffold_4_r4c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-250"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-250">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-tigrfam-details-svg" data-filename="scaffold_4_r4c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r4c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 3 - NRPS,NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-510"></div> <span class="help-tooltip" id="region_svg-help-510">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c3-svg" data-filename="scaffold_4_r4c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,938,255 - 4,057,426 nt. (total: 119,172 nt)

<a class="cluster-rules-header" id="r4c3-rules-header" href="#r4c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-511"></div> <span class="help-tooltip" id="focus-panel-help-511">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c3-gene-table-details" data-name="r4c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c3-nrps_pks" data-name="r4c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c3-MIBiG-cluster-compare" data-name="r4c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c3-clusterblast" data-name="r4c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c3-knownclusterblast" data-name="r4c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c3-subclusterblast" data-name="r4c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c3-nrps-pks-bubbles" data-name="r4c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c3-pfam-details" data-name="r4c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c3-tigrfam-details" data-name="r4c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-259"></div> <span class="help-tooltip" id="gt-help-tooltip-help-259">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-251"></div> <span class="help-tooltip" id="nrps-domain-help-251">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-details-svg" data-filename="scaffold_4_r4c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-252"></div> <span class="help-tooltip" id="MIBiG-help-252">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c3"></div>
<div id="comparison-MIBiG-r4c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.75</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.51, order:0.47, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.51, order:0.47, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.53, order:0.53, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.74</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.67</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.65</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001037: 0-18799">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001037/index.html#r1c1" target="_new">BGC0001037</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.64</td>
<td>NRP, Polyketide</td>

<td>pseurotin, azaspirene</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.64</td>
<td>Polyketide</td>

<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>


<td>Chaetomium olivaceum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.61</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.60</td>
<td>Polyketide</td>

<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002239: 1103-31038">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002239/index.html#r1c1" target="_new">BGC0002239</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.59</td>
<td>Polyketide, NRP</td>

<td>GKK1032 A2, pyrrocidine A, pyrrocidine B</td>


<td>Acremonium sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002729: 0-27318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002729/index.html#r1c1" target="_new">BGC0002729</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>sordarial</td>
<td>Neurospora crassa OR74A</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>phomoidride</td>
<td>fungal sp. ATCC 74256</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-253"></div> <span class="help-tooltip" id="cb-general-help-253">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c3_1.svg">
NW_006271974 (2321315-2405136): Cordyceps militaris CM01 unplaced genomic sca... (80% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c3_2.svg">
NW_026249875 (10804-78109): Alternaria incomplexa strain BMP 0042 chromosome ... (38% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_3.svg">
NW_020194484 (7340-83776): Amorphotheca resinae ATCC 22711 unplaced genomic s... (41% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c3_4.svg">
NC_049565 (2694595-2813931): Talaromyces rugulosus chromosome V, complete seq... (24% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_5.svg">
NW_026055699 (512819-580124): Alternaria ventricosa strain BMP 2768 chromosom... (36% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_6.svg">
NW_022474205 (7019281-7087036): Venustampulla echinocandica strain BP 5553 ch... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_7.svg">
NW_025408687 (532325-600393): Hirsutella rhossiliensis strain HR02 chromosome... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_8.svg">
NW_025953791 (566181-633989): Annulohypoxylon truncatum strain CBS 140777 unp... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_9.svg">
NW_026249941 (3336813-3404118): Alternaria arbusti strain BMP 1465 chromosome... (33% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c3_10.svg">
NW_021167084 (1837567-1907553): Sodiomyces alkalinus F11 unplaced genomic sca... (20% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r4c3-download">Download graphic</button>
</div>
<div id="clusterblast-r4c3-svg"></div>

</div>
</div>

<div class="body-details-section r4c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-254"></div> <span class="help-tooltip" id="cb-known-help-254">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c3_1.svg">
shimalactone A/shimalactone B
</option>

</select>
<button id="knownclusterblast-r4c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c3-svg"></div>

</div>
</div>

<div class="body-details-section r4c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-255"></div> <span class="help-tooltip" id="cb-sub-help-255">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-258"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-258">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-domain-bubble-svg" data-filename="scaffold_4_r4c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (3938254 - 4057426): neighbouring T1PKS-NRPS-NRPS-like
</option>



<option value="CC4">
Candidate 4 (3938254 - 3986169): single T1PKS
</option>



<option value="CC5">
Candidate 5 (3978734 - 4030187): chemical hybrid NRPS-T1PKS
</option>



<option value="CC6">
Candidate 6 (4011672 - 4057426): single NRPS-like
</option>


</select>

<div class="details-svg" id="r4c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-260"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-260">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-pfam-details-svg" data-filename="scaffold_4_r4c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-261"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-261">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-tigrfam-details-svg" data-filename="scaffold_4_r4c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c3-nrps_pks_products" data-name="r4c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c3-nrps_pks_monomers" data-name="r4c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-256"></div> <span class="help-tooltip" id="nrps-structures-help-256">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 3938254 - 4057426:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c3-3" class="smiles-canvas" data-smiles="C([*])C(-O)C([*])C(-O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk) + (pk - X) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=x&amp;nrps3=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)C([*])C(-O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 3938254 - 3986169:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c3-4" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c3-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 5, location 3978734 - 4030187:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c3-5" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c3-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 6, location 4011672 - 4057426:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c3-6" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c3-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-257"></div> <span class="help-tooltip" id="nrps-monomers-help-257">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1249</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene1249 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (540..851)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1249 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1249-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Methoxymalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1249-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 70.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 62.4</dt>

<dd></dd><dt>Malonyl-CoA: 48.7</dt>

<dd></dd><dt>Propionyl-CoA: 36.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 32.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 31.1</dt>

<dd></dd><dt>fatty_acid: 30.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 23.9</dt>

<dd></dd><dt>inactive: 22.6</dt>

<dd></dd><dt>Benzoyl-CoA: 16.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 15.3</dt>

<dd></dd><dt>CHC-CoA: 13.3</dt>

<dd></dd><dt>Acetyl-CoA: -1.2</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

<strong><span class="serif">input.path1.gene1259</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene1259 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (561..875)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1259 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1259-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1259-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 90.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 81.6</dt>

<dd></dd><dt>Isobutyryl-CoA: 69.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 61.7</dt>

<dd></dd><dt>Propionyl-CoA: 45.4</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 33.6</dt>

<dd></dd><dt>Malonyl-CoA: 32.3</dt>

<dd></dd><dt>Benzoyl-CoA: 30.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 29.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 28.2</dt>

<dd></dd><dt>Acetyl-CoA: 20.3</dt>

<dd></dd><dt>inactive: 19.0</dt>

<dd></dd><dt>fatty_acid: 17.8</dt>

<dd></dd><dt>CHC-CoA: 13.9</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2747..3154)</strong>: X
<div class="collapser collapser-target-input-path1-gene1259 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1259-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIAYSIESQK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene1266</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1266 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (51..371)</strong>: X
<div class="collapser collapser-target-input-path1-gene1266 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene1266-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">-HCYMGHQLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r5c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-512"></div> <span class="help-tooltip" id="region_svg-help-512">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c1-svg" data-filename="scaffold_5_r5c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 50,556 - 98,255 nt. (total: 47,700 nt)

<a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-513"></div> <span class="help-tooltip" id="focus-panel-help-513">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c1-gene-table-details" data-name="r5c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c1-nrps_pks" data-name="r5c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c1-nrps-pks-bubbles" data-name="r5c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c1-tigrfam-details" data-name="r5c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-270"></div> <span class="help-tooltip" id="gt-help-tooltip-help-270">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-262"></div> <span class="help-tooltip" id="nrps-domain-help-262">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-details-svg" data-filename="scaffold_5_r5c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-263"></div> <span class="help-tooltip" id="MIBiG-help-263">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c1"></div>
<div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>hydroxycinnamoyltyramine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>hydroxycinnamoyltyramine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-264"></div> <span class="help-tooltip" id="cb-general-help-264">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c1_1.svg">
NW_006271969 (4860304-4927848): Cordyceps militaris CM01 unplaced genomic sca... (38% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_2.svg">
NW_021167090 (669974-737132): Sodiomyces alkalinus F11 unplaced genomic scaff... (32% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_3.svg">
NW_022985016 (85063-152593): Dothidotthia symphoricarpi CBS 119687 unplaced g... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_4.svg">
NC_049565 (1715696-1783101): Talaromyces rugulosus chromosome V, complete seq... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_5.svg">
NW_006271969 (1967760-2079673): Cordyceps militaris CM01 unplaced genomic sca... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_6.svg">
NW_017971444 (92882-160254): Talaromyces atroroseus strain IBT 11181 chromoso... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_7.svg">
NT_165977 (3031183-3103199): Chaetomium globosum CBS 148.51 scaffold 2 genomi... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_8.svg">
NW_015622519 (2644762-2718843): Exophiala spinifera strain CBS 89968 unplaced... (7% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_9.svg">
NW_015971652 (274258-347739): Fonsecaea multimorphosa CBS 102226 unplaced gen... (7% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_10.svg">
NW_020290905 (239903-308208): Aspergillus niger CBS 101883 unplaced genomic s... (6% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r5c1-download">Download graphic</button>
</div>
<div id="clusterblast-r5c1-svg"></div>

</div>
</div>

<div class="body-details-section r5c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-265"></div> <span class="help-tooltip" id="cb-known-help-265">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-266"></div> <span class="help-tooltip" id="cb-sub-help-266">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-269"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-269">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-domain-bubble-svg" data-filename="scaffold_5_r5c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (50555 - 98255): single T1PKS
</option>


</select>

<div class="details-svg" id="r5c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-271"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-271">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="scaffold_5_r5c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-272"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-272">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-tigrfam-details-svg" data-filename="scaffold_5_r5c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c1-nrps_pks_products" data-name="r5c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c1-nrps_pks_monomers" data-name="r5c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-267"></div> <span class="help-tooltip" id="nrps-structures-help-267">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 50555 - 98255:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-268"></div> <span class="help-tooltip" id="nrps-monomers-help-268">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene24</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene24 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (549..878)</strong>: pk
<div class="collapser collapser-target-input-path1-gene24 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene24-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>2-Methylbutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene24-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 81.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 77.1</dt>

<dd></dd><dt>Malonyl-CoA: 58.9</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 46.6</dt>

<dd></dd><dt>Isobutyryl-CoA: 41.8</dt>

<dd></dd><dt>Propionyl-CoA: 35.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 27.0</dt>

<dd></dd><dt>fatty_acid: 17.0</dt>

<dd></dd><dt>Benzoyl-CoA: 15.0</dt>

<dd></dd><dt>CHC-CoA: 13.8</dt>

<dd></dd><dt>trans-1,2-CPDA: 11.6</dt>

<dd></dd><dt>inactive: 10.4</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.8</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-514"></div> <span class="help-tooltip" id="region_svg-help-514">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c2-svg" data-filename="scaffold_5_r5c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 463,589 - 516,413 nt. (total: 52,825 nt)

<a class="cluster-rules-header" id="r5c2-rules-header" href="#r5c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-515"></div> <span class="help-tooltip" id="focus-panel-help-515">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c2-gene-table-details" data-name="r5c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c2-nrps_pks" data-name="r5c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c2-MIBiG-cluster-compare" data-name="r5c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c2-clusterblast" data-name="r5c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c2-knownclusterblast" data-name="r5c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c2-subclusterblast" data-name="r5c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c2-nrps-pks-bubbles" data-name="r5c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c2-pfam-details" data-name="r5c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c2-tigrfam-details" data-name="r5c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-281"></div> <span class="help-tooltip" id="gt-help-tooltip-help-281">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-273"></div> <span class="help-tooltip" id="nrps-domain-help-273">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-details-svg" data-filename="scaffold_5_r5c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-274"></div> <span class="help-tooltip" id="MIBiG-help-274">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c2"></div>
<div id="comparison-MIBiG-r5c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001545: 19-17507">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001545/index.html#r1c1" target="_new">BGC0001545</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>chrysogine</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002199: 0-17425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002199/index.html#r1c1" target="_new">BGC0002199</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Alkaloid</td>

<td>nanangelenin A</td>


<td>Aspergillus nanangensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001545: 19-17507">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001545/index.html#r1c1" target="_new">BGC0001545</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>chrysogine</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002199: 0-17425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002199/index.html#r1c1" target="_new">BGC0002199</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Alkaloid</td>
<td>nanangelenin A</td>
<td>Aspergillus nanangensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-275"></div> <span class="help-tooltip" id="cb-general-help-275">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c2_1.svg">
NC_035791 (633219-706856): Pochonia chlamydosporia 170 chromosome 2, whole ge... (26% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c2_2.svg">
NC_026474 (5505232-5623287): Fusarium graminearum PH-1 chromosome 1, whole ge... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_3.svg">
NW_026054647 (100851-185503): Ophidiomyces ophidiicola strain CBS 122913 chro... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c2_4.svg">
NW_025953788 (1126759-1218489): Annulohypoxylon truncatum strain CBS 140777 u... (6% of genes show similarity), NRPS,indole,terpene
</option>

<option value = "svg/clusterblast_r5c2_5.svg">
NW_024468937 (1436807-1502743): Cucurbitaria berberidis CBS 394.84 unplaced g... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c2_6.svg">
NW_025091283 (4319746-4671991): Aspergillus pseudoviridinutans IFM 55266 DNA,... (1% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r5c2_7.svg">
NW_006271970 (340089-399189): Cordyceps militaris CM01 unplaced genomic scaff... (8% of genes show similarity), fungal-RiPP
</option>

<option value = "svg/clusterblast_r5c2_8.svg">
NW_006912904 (1152366-1183721): Capronia epimyces CBS 606.96 aczIQ-supercont1... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r5c2_9.svg">
NW_025408668 (1200492-1268825): Hirsutella rhossiliensis strain HR02 chromoso... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c2_10.svg">
NW_021167090 (669974-737132): Sodiomyces alkalinus F11 unplaced genomic scaff... (8% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r5c2-download">Download graphic</button>
</div>
<div id="clusterblast-r5c2-svg"></div>

</div>
</div>

<div class="body-details-section r5c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-276"></div> <span class="help-tooltip" id="cb-known-help-276">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-277"></div> <span class="help-tooltip" id="cb-sub-help-277">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-280"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-280">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-domain-bubble-svg" data-filename="scaffold_5_r5c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (463588 - 516413): single NRPS
</option>


</select>

<div class="details-svg" id="r5c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-282"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-282">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-pfam-details-svg" data-filename="scaffold_5_r5c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-283"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-283">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-tigrfam-details-svg" data-filename="scaffold_5_r5c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c2-nrps_pks_products" data-name="r5c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c2-nrps_pks_monomers" data-name="r5c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-278"></div> <span class="help-tooltip" id="nrps-structures-help-278">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 463588 - 516413:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(Cc1ccccc1)C(=O)N1C(CCC1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Phe - Pro)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,pro" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,phe*,pro*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(Cc1ccccc1)C(=O)N1C(CCC1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-279"></div> <span class="help-tooltip" id="nrps-monomers-help-279">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene148</span></strong>: X - Phe - Pro
<div class="collapser collapser-target-input-path1-gene148 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,pro" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,pro*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (347..742)</strong>: X
<div class="collapser collapser-target-input-path1-gene148 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene148-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVIIVFSMLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1426..1821)</strong>: Phe
<div class="collapser collapser-target-input-path1-gene148 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-input-path1-gene148-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIIVLGGVSK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DAIVLGGVCK</span> (53% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (3264..3657)</strong>: Pro
<div class="collapser collapser-target-input-path1-gene148 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Pro
<div class="collapser collapser-target-input-path1-gene148-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>pro (Pro)</dd>
<dt>Single AA prediction:</dt>
<dd>pro (Pro)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVLFCGLICK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Pro <span class="serif">DVLFCGLICK</span> (91% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 3 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-516"></div> <span class="help-tooltip" id="region_svg-help-516">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c3-svg" data-filename="scaffold_5_r5c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 578,855 - 639,595 nt. (total: 60,741 nt)

<a class="cluster-rules-header" id="r5c3-rules-header" href="#r5c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c3-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r5c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-517"></div> <span class="help-tooltip" id="focus-panel-help-517">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c3-gene-table-details" data-name="r5c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c3-MIBiG-cluster-compare" data-name="r5c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c3-clusterblast" data-name="r5c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c3-knownclusterblast" data-name="r5c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c3-subclusterblast" data-name="r5c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c3-pfam-details" data-name="r5c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c3-tigrfam-details" data-name="r5c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-288"></div> <span class="help-tooltip" id="gt-help-tooltip-help-288">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-284"></div> <span class="help-tooltip" id="MIBiG-help-284">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c3"></div>
<div id="comparison-MIBiG-r5c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001621: 3190-27169">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001621/index.html#r1c1" target="_new">BGC0001621</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>imizoquin A, imizoquin B, imizoquin C, imizoquin D, TMC-2A, TMC-2B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002261: 51-38175">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002261/index.html#r1c1" target="_new">BGC0002261</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.45, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000140: 112-14048">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000140/index.html#r1c1" target="_new">BGC0000140</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.45, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>4-hexadecanoyl-3-hydroxy-2-(hydroxymethyl)-2H-furan-5-one</td>


<td>Streptomyces sp. 88-682</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001040: 0-107379">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001040/index.html#r1c1" target="_new">BGC0001040</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.43, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>NRP, Polyketide</td>

<td>rapamycin</td>


<td>Streptomyces rapamycinicus NRRL 5491</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001621: 3190-27169">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001621/index.html#r1c1" target="_new">BGC0001621</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP</td>
<td>imizoquin A, imizoquin B, imizoquin C, imizoquin D, TMC-2A, TMC-2B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002261: 51-38175">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002261/index.html#r1c1" target="_new">BGC0002261</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.45, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000140: 112-14048">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000140/index.html#r1c1" target="_new">BGC0000140</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.45, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>4-hexadecanoyl-3-hydroxy-2-(hydroxymethyl)-2H-furan-5-one</td>
<td>Streptomyces sp. 88-682</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001040: 0-107379">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001040/index.html#r1c1" target="_new">BGC0001040</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.43, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>NRP, Polyketide</td>
<td>rapamycin</td>
<td>Streptomyces rapamycinicus NRRL 5491</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-285"></div> <span class="help-tooltip" id="cb-general-help-285">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c3_1.svg">
NW_025953252 (947231-1128644): Annulohypoxylon maeteangense strain CBS 123835... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_2.svg">
NW_019154049 (138062-255179): Pochonia chlamydosporia 170 chromosome Unknown ... (4% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r5c3_3.svg">
NC_072840 (5720970-5812093): Fusarium oxysporum Fo47 chromosome I, complete s... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_4.svg">
NC_026474 (5505232-5623287): Fusarium graminearum PH-1 chromosome 1, whole ge... (6% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_5.svg">
NC_038012 (5664495-5755610): Fusarium venenatum strain A3/5 genome assembly, ... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_6.svg">
NC_058399 (5498869-5589983): Fusarium poae strain DAOMC 252244 chromosome 1, ... (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_7.svg">
NC_036622 (502259-593382): Fusarium fujikuroi IMI 58289 draft genome, chromos... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_8.svg">
NC_070544 (835792-926898): Fusarium falciforme chromosome 1, complete sequence (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_9.svg">
NC_070529 (504504-595611): Fusarium keratoplasticum isolate Fu6.1 chromosome ... (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c3_10.svg">
NC_058387 (525966-617087): Fusarium musae strain F31 chromosome 1, whole geno... (10% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r5c3-download">Download graphic</button>
</div>
<div id="clusterblast-r5c3-svg"></div>

</div>
</div>

<div class="body-details-section r5c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-286"></div> <span class="help-tooltip" id="cb-known-help-286">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-287"></div> <span class="help-tooltip" id="cb-sub-help-287">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-289"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-289">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-pfam-details-svg" data-filename="scaffold_5_r5c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-290"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-290">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-tigrfam-details-svg" data-filename="scaffold_5_r5c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r5c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 4 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-518"></div> <span class="help-tooltip" id="region_svg-help-518">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c4-svg" data-filename="scaffold_5_r5c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,851,347 - 1,898,221 nt. (total: 46,875 nt)

<a class="cluster-rules-header" id="r5c4-rules-header" href="#r5c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c4-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-519"></div> <span class="help-tooltip" id="focus-panel-help-519">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c4-gene-table-details" data-name="r5c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c4-nrps_pks" data-name="r5c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c4-MIBiG-cluster-compare" data-name="r5c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c4-clusterblast" data-name="r5c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c4-knownclusterblast" data-name="r5c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c4-subclusterblast" data-name="r5c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c4-nrps-pks-bubbles" data-name="r5c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c4-pfam-details" data-name="r5c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-299"></div> <span class="help-tooltip" id="gt-help-tooltip-help-299">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-291"></div> <span class="help-tooltip" id="nrps-domain-help-291">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-details-svg" data-filename="scaffold_5_r5c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-292"></div> <span class="help-tooltip" id="MIBiG-help-292">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c4"></div>
<div id="comparison-MIBiG-r5c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>yanuthone D</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001037: 0-18799">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001037/index.html#r1c1" target="_new">BGC0001037</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP, Polyketide</td>

<td>pseurotin, azaspirene</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>yanuthone D</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001037: 0-18799">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001037/index.html#r1c1" target="_new">BGC0001037</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP, Polyketide</td>
<td>pseurotin, azaspirene</td>
<td>Aspergillus fumigatus Af293</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-293"></div> <span class="help-tooltip" id="cb-general-help-293">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c4_1.svg">
NW_026054642 (296356-363048): Ophidiomyces ophidiicola strain CBS 122913 chro... (30% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c4_2.svg">
NW_025091217 (15487-82066): Aspergillus viridinutans IFM 47045 DNA, scaffold ... (62% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c4_3.svg">
NW_025408676 (2911047-2980048): Hirsutella rhossiliensis strain HR02 chromoso... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c4_4.svg">
NW_025091286 (2424447-2527542): Aspergillus pseudoviridinutans IFM 55266 DNA,... (7% of genes show similarity), NRPS,T1PKS,other
</option>

<option value = "svg/clusterblast_r5c4_5.svg">
NC_030957 (3250063-3360781): Colletotrichum higginsianum IMI 349063 chromosom... (5% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_6.svg">
NW_026250021 (1864413-1931647): Hypoxylon trugodes strain CBS 135444 unplaced... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c4_7.svg">
NW_024468940 (1557228-1624623): Cucurbitaria berberidis CBS 394.84 unplaced g... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c4_8.svg">
NW_003052500 (3318567-3442588): Uncinocarpus reesii 1704 scaffold 1 genomic s... (5% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c4_9.svg">
NW_006921541 (103572-212868): Pseudocercospora fijiensis CIRAD86 unplaced gen... (15% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c4_10.svg">
NC_054849 (2647776-2721226): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (11% of genes show similarity), T1PKS,terpene
</option>

</select>
<button id= "clusterblast-r5c4-download">Download graphic</button>
</div>
<div id="clusterblast-r5c4-svg"></div>

</div>
</div>

<div class="body-details-section r5c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-294"></div> <span class="help-tooltip" id="cb-known-help-294">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-295"></div> <span class="help-tooltip" id="cb-sub-help-295">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-298"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-298">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-domain-bubble-svg" data-filename="scaffold_5_r5c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (1851346 - 1898221): single T1PKS
</option>


</select>

<div class="details-svg" id="r5c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-300"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-300">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-pfam-details-svg" data-filename="scaffold_5_r5c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c4-nrps_pks_products" data-name="r5c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c4-nrps_pks_monomers" data-name="r5c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-296"></div> <span class="help-tooltip" id="nrps-structures-help-296">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 1851346 - 1898221:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c4-4" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-297"></div> <span class="help-tooltip" id="nrps-monomers-help-297">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene561</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene561 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (550..867)</strong>: pk
<div class="collapser collapser-target-input-path1-gene561 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene561-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Propionyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene561-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 69.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 60.9</dt>

<dd></dd><dt>Malonyl-CoA: 54.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 38.9</dt>

<dd></dd><dt>Propionyl-CoA: 29.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 26.4</dt>

<dd></dd><dt>Benzoyl-CoA: 14.2</dt>

<dd></dd><dt>fatty_acid: 13.5</dt>

<dd></dd><dt>inactive: 12.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 12.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 9.9</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.3</dt>

<dd></dd><dt>CHC-CoA: 8.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r6c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-520"></div> <span class="help-tooltip" id="region_svg-help-520">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c1-svg" data-filename="scaffold_6_r6c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 57,797 nt. (total: 57,797 nt)

<a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r6c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r6c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-521"></div> <span class="help-tooltip" id="focus-panel-help-521">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c1-gene-table-details" data-name="r6c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-305"></div> <span class="help-tooltip" id="gt-help-tooltip-help-305">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-301"></div> <span class="help-tooltip" id="MIBiG-help-301">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c1"></div>
<div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001580: 126-73935">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001580/index.html#r1c1" target="_new">BGC0001580</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.43, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>ebelactone</td>


<td>Kitasatospora aburaviensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001580: 126-73935">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001580/index.html#r1c1" target="_new">BGC0001580</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.43, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>Polyketide</td>
<td>ebelactone</td>
<td>Kitasatospora aburaviensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-302"></div> <span class="help-tooltip" id="cb-general-help-302">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c1_1.svg">
NW_006271970 (4894883-4985904): Cordyceps militaris CM01 unplaced genomic sca... (22% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c1-download">Download graphic</button>
</div>
<div id="clusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-303"></div> <span class="help-tooltip" id="cb-known-help-303">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-304"></div> <span class="help-tooltip" id="cb-sub-help-304">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-306"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-306">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="scaffold_6_r6c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r6c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 2 - T1PKS,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-522"></div> <span class="help-tooltip" id="region_svg-help-522">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c2-svg" data-filename="scaffold_6_r6c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 292,495 - 380,465 nt. (total: 87,971 nt)

<a class="cluster-rules-header" id="r6c2-rules-header" href="#r6c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c2-rules">fungal-RiPP-like: DUF3328<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r6c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-523"></div> <span class="help-tooltip" id="focus-panel-help-523">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c2-gene-table-details" data-name="r6c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c2-nrps_pks" data-name="r6c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c2-MIBiG-cluster-compare" data-name="r6c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c2-clusterblast" data-name="r6c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c2-knownclusterblast" data-name="r6c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c2-subclusterblast" data-name="r6c2-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r6c2-nrps-pks-bubbles" data-name="r6c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c2-pfam-details" data-name="r6c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c2-tigrfam-details" data-name="r6c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-315"></div> <span class="help-tooltip" id="gt-help-tooltip-help-315">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-307"></div> <span class="help-tooltip" id="nrps-domain-help-307">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-details-svg" data-filename="scaffold_6_r6c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-308"></div> <span class="help-tooltip" id="MIBiG-help-308">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c2"></div>
<div id="comparison-MIBiG-r6c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.34</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>cichorine</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001122: 0-43663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001122/index.html#r1c1" target="_new">BGC0001122</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.46, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>






<td class="cc-heat-cell" title="0.36 (id:0.45, order:0.26, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.23</td>
<td>NRP, Polyketide</td>

<td>dihydroisoflavipucine, isoflavipucine</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>oosporein</td>


<td>Beauveria bassiana ARSEF 2860</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Nodulisporium sp. ATCC74245</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>naphthalene</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>cichorine</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>oosporein</td>
<td>Beauveria bassiana ARSEF 2860</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene, Polyketide</td>
<td>ascochlorin</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002614: 1538-32674">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002614/index.html#r1c1" target="_new">BGC0002614</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>subglutinol A, sesquicillin A, colletotrichin, metarhizin A</td>
<td>Penicillium shearii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>emodin</td>
<td>Escovopsis weberi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000030: 0-18435">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000030/index.html#r1c1" target="_new">BGC0000030</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>bikaverin</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-309"></div> <span class="help-tooltip" id="cb-general-help-309">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c2_1.svg">
NC_026475 (3526767-3593357): Fusarium graminearum PH-1 chromosome 2, whole ge... (12% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c2_2.svg">
NC_066261 (236482-342132): Aspergillus nidulans FGSC A4 chromosome V (6% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_3.svg">
NW_024467526 (3357689-3444377): Aspergillus versicolor CBS 583.65 unplaced ge... (7% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c2_4.svg">
NW_023336279 (711499-880798): Aspergillus tubingensis WU-2223L DNA, scaffold ... (3% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r6c2_5.svg">
NZ_CP019572 (2480141-2530188): Rhodococcus sp. MTM3W5.2 voucher Culture colle... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_6.svg">
NZ_JMLO01000001 (539340-589945): Rhodococcus sp. UNC363MFTsu5.1 BR31DRAFT sca... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_7.svg">
NW_024467537 (92677-199333): Aspergillus versicolor CBS 583.65 unplaced genom... (4% of genes show similarity), T1PKS,fungal-RiPP
</option>

<option value = "svg/clusterblast_r6c2_8.svg">
NW_022983479 (1031372-1097841): Aaosphaeria arxii CBS 175.79 unplaced genomic... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_9.svg">
NZ_CP021748 (458486-577960): Streptomyces alboflavus strain MDJK44 chromosome... (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_10.svg">
NW_016017383 (3643102-3709927): Sphaerulina musiva SO2202 unplaced genomic sc... (7% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r6c2-download">Download graphic</button>
</div>
<div id="clusterblast-r6c2-svg"></div>

</div>
</div>

<div class="body-details-section r6c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-310"></div> <span class="help-tooltip" id="cb-known-help-310">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-311"></div> <span class="help-tooltip" id="cb-sub-help-311">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r6c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-314"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-314">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-domain-bubble-svg" data-filename="scaffold_6_r6c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (292494 - 380465): neighbouring fungal-RiPP-like-T1PKS
</option>



<option value="CC4">
Candidate 4 (333045 - 380465): single T1PKS
</option>


</select>

<div class="details-svg" id="r6c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-316"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-316">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-pfam-details-svg" data-filename="scaffold_6_r6c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-317"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-317">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-tigrfam-details-svg" data-filename="scaffold_6_r6c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c2-nrps_pks_products" data-name="r6c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c2-nrps_pks_monomers" data-name="r6c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r6c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-312"></div> <span class="help-tooltip" id="nrps-structures-help-312">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 292494 - 380465:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c2-2" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 333045 - 380465:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c2-4" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c2-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-313"></div> <span class="help-tooltip" id="nrps-monomers-help-313">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene118</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene118 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (514..805)</strong>: mal
<div class="collapser collapser-target-input-path1-gene118 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene118-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene118-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 81.1</dt>

<dd></dd><dt>inactive: 52.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 46.8</dt>

<dd></dd><dt>Methylmalonyl-CoA: 36.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 24.7</dt>

<dd></dd><dt>Propionyl-CoA: 22.1</dt>

<dd></dd><dt>CHC-CoA: 22.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 19.2</dt>

<dd></dd><dt>Benzoyl-CoA: 15.1</dt>

<dd></dd><dt>Isobutyryl-CoA: 12.3</dt>

<dd></dd><dt>Acetyl-CoA: 12.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 9.8</dt>

<dd></dd><dt>fatty_acid: 9.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>

<div class="page" id='r6c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-524"></div> <span class="help-tooltip" id="region_svg-help-524">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c3-svg" data-filename="scaffold_6_r6c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 864,414 - 909,766 nt. (total: 45,353 nt)

<a class="cluster-rules-header" id="r6c3-rules-header" href="#r6c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-525"></div> <span class="help-tooltip" id="focus-panel-help-525">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c3-gene-table-details" data-name="r6c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c3-nrps_pks" data-name="r6c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c3-MIBiG-cluster-compare" data-name="r6c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c3-clusterblast" data-name="r6c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c3-knownclusterblast" data-name="r6c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c3-subclusterblast" data-name="r6c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c3-nrps-pks-bubbles" data-name="r6c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c3-pfam-details" data-name="r6c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-326"></div> <span class="help-tooltip" id="gt-help-tooltip-help-326">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-318"></div> <span class="help-tooltip" id="nrps-domain-help-318">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-details-svg" data-filename="scaffold_6_r6c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-319"></div> <span class="help-tooltip" id="MIBiG-help-319">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c3"></div>
<div id="comparison-MIBiG-r6c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001518: 0-23499">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001518/index.html#r1c1" target="_new">BGC0001518</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>astellolide A</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876: 71-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Other</td>

<td>neopolyoxin C</td>


<td>Streptomyces tendae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717: 0-19662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>okaramine B</td>


<td>Penicillium simplicissimum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001518: 0-23499">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001518/index.html#r1c1" target="_new">BGC0001518</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>astellolide A</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876: 71-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>neopolyoxin C</td>
<td>Streptomyces tendae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717: 0-19662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>okaramine B</td>
<td>Penicillium simplicissimum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-320"></div> <span class="help-tooltip" id="cb-general-help-320">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c3_1.svg">
NC_054850 (2918415-2983028): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c3_2.svg">
NW_019170454 (685594-749291): Aspergillus glaucus CBS 516.65 unplaced genomic... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c3_3.svg">
NC_054861 (1916756-1987814): Aspergillus puulaauensis MK2 DNA, chromosome 5, ... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c3_4.svg">
NC_057367 (973431-1037220): Aspergillus chevalieri M1 DNA, chromosome 6, near... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c3_5.svg">
NW_024467524 (3810963-3881818): Aspergillus versicolor CBS 583.65 unplaced ge... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c3_6.svg">
NW_024467304 (552618-615729): Aspergillus ruber CBS 135680 unplaced genomic s... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c3_7.svg">
NC_007196 (3573152-3653216): Aspergillus fumigatus Af293 chromosome 3, whole ... (6% of genes show similarity), NRPS,isocyanide-nrp
</option>

<option value = "svg/clusterblast_r6c3_8.svg">
NW_020292165 (42547-121840): Aspergillus ibericus CBS 121593 unplaced genomic... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c3_9.svg">
NC_054859 (3374101-3450218): Aspergillus puulaauensis MK2 DNA, chromosome 3, ... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c3_10.svg">
NW_025091266 (1102530-1174184): Aspergillus udagawae IFM 46973 DNA, scaffold ... (7% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r6c3-download">Download graphic</button>
</div>
<div id="clusterblast-r6c3-svg"></div>

</div>
</div>

<div class="body-details-section r6c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-321"></div> <span class="help-tooltip" id="cb-known-help-321">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-322"></div> <span class="help-tooltip" id="cb-sub-help-322">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-325"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-325">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-domain-bubble-svg" data-filename="scaffold_6_r6c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (864413 - 909766): single NRPS
</option>


</select>

<div class="details-svg" id="r6c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-327"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-327">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-pfam-details-svg" data-filename="scaffold_6_r6c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c3-nrps_pks_products" data-name="r6c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c3-nrps_pks_monomers" data-name="r6c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-323"></div> <span class="help-tooltip" id="nrps-structures-help-323">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 864413 - 909766:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c3-5" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c3-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-324"></div> <span class="help-tooltip" id="nrps-monomers-help-324">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene288</span></strong>: X
<div class="collapser collapser-target-input-path1-gene288 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (329..720)</strong>: X
<div class="collapser collapser-target-input-path1-gene288 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr
<div class="collapser collapser-target-input-path1-gene288-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">VVTAVVLKGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 4 - NRPS,indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-526"></div> <span class="help-tooltip" id="region_svg-help-526">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c4-svg" data-filename="scaffold_6_r6c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 966,154 - 1,011,305 nt. (total: 45,152 nt)

<a class="cluster-rules-header" id="r6c4-rules-header" href="#r6c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c4-rules">indole: (indsynth or dmat or indole_PTase)<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-527"></div> <span class="help-tooltip" id="focus-panel-help-527">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c4-gene-table-details" data-name="r6c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c4-nrps_pks" data-name="r6c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c4-MIBiG-cluster-compare" data-name="r6c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c4-clusterblast" data-name="r6c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c4-knownclusterblast" data-name="r6c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c4-subclusterblast" data-name="r6c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c4-pfam-details" data-name="r6c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-334"></div> <span class="help-tooltip" id="gt-help-tooltip-help-334">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-328"></div> <span class="help-tooltip" id="nrps-domain-help-328">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c4-details-svg" data-filename="scaffold_6_r6c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-329"></div> <span class="help-tooltip" id="MIBiG-help-329">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c4"></div>
<div id="comparison-MIBiG-r6c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">indole</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000930: 2573-28230">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000930/index.html#r1c1" target="_new">BGC0000930</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>deoxynivalenol</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001811: 32-25672">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001811/index.html#r1c1" target="_new">BGC0001811</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>trichodiene-11-one</td>


<td>Fusarium asiaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001989: 0-7977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001989/index.html#r1c1" target="_new">BGC0001989</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Other</td>

<td>pulcherrimin</td>


<td>Kluyveromyces lactis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>preaspterpenacid I</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002321: 0-6666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002321/index.html#r1c1" target="_new">BGC0002321</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>pulcherriminic acid, pulcherrimin, cyclodileucine, compound 3</td>


<td>Metschnikowia aff. pulcherrima</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>HEx-tc3 terpene</td>


<td>Botrytis cinerea T4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001278: 588-54488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001278/index.html#r1c1" target="_new">BGC0001278</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.29</td>
<td>Terpene</td>

<td>nivalenol, deoxynivalenol, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-decalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>


<td>Fusarium sporotrichioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.44, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>KK-1</td>


<td>Curvularia clavata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001989: 0-7977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001989/index.html#r1c1" target="_new">BGC0001989</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>pulcherrimin</td>
<td>Kluyveromyces lactis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>preaspterpenacid I</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002321: 0-6666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002321/index.html#r1c1" target="_new">BGC0002321</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>pulcherriminic acid, pulcherrimin, cyclodileucine, compound 3</td>
<td>Metschnikowia aff. pulcherrima</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>citreohybridonol</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000930: 2573-28230">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000930/index.html#r1c1" target="_new">BGC0000930</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>deoxynivalenol</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001811: 32-25672">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001811/index.html#r1c1" target="_new">BGC0001811</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>trichodiene-11-one</td>
<td>Fusarium asiaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>HEx-tc3 terpene</td>
<td>Botrytis cinerea T4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685: 127-25775">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>brassicicene C</td>
<td>Alternaria brassicicola ATCC 96836</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-330"></div> <span class="help-tooltip" id="cb-general-help-330">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c4_1.svg">
NW_017971445 (712163-779607): Talaromyces atroroseus strain IBT 11181 chromos... (28% of genes show similarity), NRPS,indole
</option>

<option value = "svg/clusterblast_r6c4_2.svg">
NW_025091284 (3966613-4033341): Aspergillus pseudoviridinutans IFM 55266 DNA,... (18% of genes show similarity), NRPS,indole
</option>

<option value = "svg/clusterblast_r6c4_3.svg">
NW_025091265 (3587345-3654076): Aspergillus udagawae IFM 46973 DNA, scaffold ... (17% of genes show similarity), NRPS,indole
</option>

<option value = "svg/clusterblast_r6c4_4.svg">
NC_007196 (3400193-3466923): Aspergillus fumigatus Af293 chromosome 3, whole ... (16% of genes show similarity), NRPS,indole
</option>

<option value = "svg/clusterblast_r6c4_5.svg">
NW_022987222 (836896-929755): Trematosphaeria pertusa strain CBS 122368 unpla... (10% of genes show similarity), NRPS,indole,terpene
</option>

<option value = "svg/clusterblast_r6c4_6.svg">
NW_025091266 (1883427-2035482): Aspergillus udagawae IFM 46973 DNA, scaffold ... (11% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c4_7.svg">
NW_022985439 (2668911-2735131): Mytilinidion resinicola strain CBS 304.34 unp... (20% of genes show similarity), NRPS,indole
</option>

<option value = "svg/clusterblast_r6c4_8.svg">
NZ_CP054468 (565714-703444): Burkholderia gladioli strain CGB10 chromosome 1,... (6% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c4_9.svg">
NC_072841 (2188172-2273307): Fusarium oxysporum Fo47 chromosome II, complete ... (6% of genes show similarity), NRPS,T1PKS,indole
</option>

<option value = "svg/clusterblast_r6c4_10.svg">
NZ_AP022821 (787260-861278): Halomonas meridiana strain Slthf1 chromosome, co... (4% of genes show similarity), NRP-metallophore,NRPS
</option>

</select>
<button id= "clusterblast-r6c4-download">Download graphic</button>
</div>
<div id="clusterblast-r6c4-svg"></div>

</div>
</div>

<div class="body-details-section r6c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-331"></div> <span class="help-tooltip" id="cb-known-help-331">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c4_1.svg">
sirodesmin PL
</option>

</select>
<button id="knownclusterblast-r6c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c4-svg"></div>

</div>
</div>

<div class="body-details-section r6c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-332"></div> <span class="help-tooltip" id="cb-sub-help-332">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-335"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-335">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c4-pfam-details-svg" data-filename="scaffold_6_r6c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c4-nrps_pks_monomers" data-name="r6c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r6c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-333"></div> <span class="help-tooltip" id="nrps-monomers-help-333">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene324</span></strong>: X
<div class="collapser collapser-target-input-path1-gene324 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (17..220)</strong>: X
<div class="collapser collapser-target-input-path1-gene324 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r6c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 5 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-528"></div> <span class="help-tooltip" id="region_svg-help-528">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c5-svg" data-filename="scaffold_6_r6c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,397,566 - 1,453,710 nt. (total: 56,145 nt)

<a class="cluster-rules-header" id="r6c5-rules-header" href="#r6c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c5-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-529"></div> <span class="help-tooltip" id="focus-panel-help-529">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c5-gene-table-details" data-name="r6c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c5-nrps_pks" data-name="r6c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c5-MIBiG-cluster-compare" data-name="r6c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c5-clusterblast" data-name="r6c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c5-knownclusterblast" data-name="r6c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c5-subclusterblast" data-name="r6c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c5-nrps-pks-bubbles" data-name="r6c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c5-pfam-details" data-name="r6c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c5-tigrfam-details" data-name="r6c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-344"></div> <span class="help-tooltip" id="gt-help-tooltip-help-344">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-336"></div> <span class="help-tooltip" id="nrps-domain-help-336">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-details-svg" data-filename="scaffold_6_r6c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-337"></div> <span class="help-tooltip" id="MIBiG-help-337">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c5"></div>
<div id="comparison-MIBiG-r6c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>oxepinamide D</td>


<td>Aspergillus ustus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>oxepinamide D</td>
<td>Aspergillus ustus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-338"></div> <span class="help-tooltip" id="cb-general-help-338">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c5_1.svg">
NW_006271970 (4182342-4255710): Cordyceps militaris CM01 unplaced genomic sca... (57% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_2.svg">
NW_021167094 (207435-275256): Sodiomyces alkalinus F11 unplaced genomic scaff... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c5_3.svg">
NW_011942171 (5162432-5375378): Metarhizium robertsii ARSEF 23 MAA Scf 3, who... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c5_4.svg">
NW_017971433 (2380693-2447085): Talaromyces atroroseus strain IBT 11181 chrom... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_5.svg">
NW_025091266 (1102530-1174184): Aspergillus udagawae IFM 46973 DNA, scaffold ... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_6.svg">
NW_025091283 (359926-455746): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c5_7.svg">
NW_011942151 (1102940-1173579): Metarhizium robertsii ARSEF 23 MAA Scf 11, wh... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_8.svg">
NW_024467526 (7938-201848): Aspergillus versicolor CBS 583.65 unplaced genomi... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_9.svg">
NC_035790 (380514-527376): Pochonia chlamydosporia 170 chromosome 1, whole ge... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c5_10.svg">
NW_020290885 (1149889-1231194): Aspergillus niger CBS 101883 unplaced genomic... (10% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r6c5-download">Download graphic</button>
</div>
<div id="clusterblast-r6c5-svg"></div>

</div>
</div>

<div class="body-details-section r6c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-339"></div> <span class="help-tooltip" id="cb-known-help-339">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-340"></div> <span class="help-tooltip" id="cb-sub-help-340">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-343"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-343">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-domain-bubble-svg" data-filename="scaffold_6_r6c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC9">
Candidate 9 (1397565 - 1453710): single NRPS
</option>


</select>

<div class="details-svg" id="r6c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-345"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-345">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-pfam-details-svg" data-filename="scaffold_6_r6c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-346"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-346">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-tigrfam-details-svg" data-filename="scaffold_6_r6c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c5-nrps_pks_products" data-name="r6c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c5-nrps_pks_monomers" data-name="r6c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-341"></div> <span class="help-tooltip" id="nrps-structures-help-341">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 9, location 1397565 - 1453710:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c5-9" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Leu - Ala - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,leu*,ala*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c5-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-342"></div> <span class="help-tooltip" id="nrps-monomers-help-342">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene468</span></strong>: X - Leu - Ala - X
<div class="collapser collapser-target-input-path1-gene468 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,ala*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (37..442)</strong>: X
<div class="collapser collapser-target-input-path1-gene468 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-input-path1-gene468-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVMEVGTPTK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Å match)</dd>
<dd>Orn <span class="serif">DVMELSSITK</span> (47% 8Å match)</dd>
<dd>Pro <span class="serif">DVMLVGAVNK</span> (44% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1130..1528)</strong>: Leu
<div class="collapser collapser-target-input-path1-gene468 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-input-path1-gene468-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALDTGMATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (71% 8Å match)</dd>
<dd>Phe or Trp <span class="serif">DAMIGGMVTK</span> (47% 8Å match)</dd>
<dd>Leu <span class="serif">DALYLGMMCK</span> (44% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2218..2622)</strong>: Ala
<div class="collapser collapser-target-input-path1-gene468 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-input-path1-gene468-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLCLHIMTFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3309..3712)</strong>: X
<div class="collapser collapser-target-input-path1-gene468 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene468-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALDTGMATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Å match)</dd>
<dd>Phe or Trp <span class="serif">DAMIGGMVTK</span> (50% 8Å match)</dd>
<dd>Leu <span class="serif">DALYLGMMCK</span> (47% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 6 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-530"></div> <span class="help-tooltip" id="region_svg-help-530">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c6-svg" data-filename="scaffold_6_r6c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,458,851 - 3,502,639 nt. (total: 43,789 nt)

<a class="cluster-rules-header" id="r6c6-rules-header" href="#r6c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c6-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-531"></div> <span class="help-tooltip" id="focus-panel-help-531">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c6-gene-table-details" data-name="r6c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c6-nrps_pks" data-name="r6c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c6-MIBiG-cluster-compare" data-name="r6c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c6-clusterblast" data-name="r6c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c6-knownclusterblast" data-name="r6c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c6-subclusterblast" data-name="r6c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c6-nrps-pks-bubbles" data-name="r6c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c6-pfam-details" data-name="r6c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c6-tigrfam-details" data-name="r6c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-355"></div> <span class="help-tooltip" id="gt-help-tooltip-help-355">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-347"></div> <span class="help-tooltip" id="nrps-domain-help-347">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-details-svg" data-filename="scaffold_6_r6c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-348"></div> <span class="help-tooltip" id="MIBiG-help-348">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c6"></div>
<div id="comparison-MIBiG-r6c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002660: 0-23622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002660/index.html#r1c1" target="_new">BGC0002660</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>pentacitidin A, pentacitidin B</td>


<td>Chitinophaga eiseniae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP, Polyketide</td>

<td>myxoprincomide-c506</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349: 0-24145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.41, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>NRP</td>

<td>erythrochelin</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.41, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>NRP</td>

<td>fragin</td>


<td>Burkholderia cenocepacia H111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002660: 0-23622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002660/index.html#r1c1" target="_new">BGC0002660</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>pentacitidin A, pentacitidin B</td>
<td>Chitinophaga eiseniae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP, Polyketide</td>
<td>myxoprincomide-c506</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349: 0-24145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.41, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>NRP</td>
<td>erythrochelin</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.41, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>NRP</td>
<td>fragin</td>
<td>Burkholderia cenocepacia H111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-349"></div> <span class="help-tooltip" id="cb-general-help-349">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c6_1.svg">
NW_024468909 (3998907-4062650): Cryphonectria parasitica EP155 unplaced genom... (11% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c6_2.svg">
NW_001809795 (231995-296216): Meyerozyma guilliermondii ATCC 6260 scaffold 6 ... (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c6_3.svg">
NW_026249859 (1367401-1431033): Alternaria incomplexa strain BMP 0042 chromos... (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c6_4.svg">
NZ_RFFH01000004 (285300-329908): Nocardia stercoris strain NEAU-LL90 Scaffold... (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c6_5.svg">
NW_025764711 (468396-559417): Emericellopsis atlantica unplaced genomic scaff... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_6.svg">
NW_007361001 (2569111-2660065): Glarea lozoyensis ATCC 20868 chromosome Unkno... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_7.svg">
NW_017567004 (112351-143605): Saitoella complicata NRRL Y-17804 unplaced geno... (9% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c6_8.svg">
NW_022986854 (382972-474393): Westerdykella ornata strain CBS 379.55 unplaced... (4% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c6-download">Download graphic</button>
</div>
<div id="clusterblast-r6c6-svg"></div>

</div>
</div>

<div class="body-details-section r6c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-350"></div> <span class="help-tooltip" id="cb-known-help-350">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-351"></div> <span class="help-tooltip" id="cb-sub-help-351">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-354"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-354">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-domain-bubble-svg" data-filename="scaffold_6_r6c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC10">
Candidate 10 (3458850 - 3502639): single NRPS-like
</option>


</select>

<div class="details-svg" id="r6c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-356"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-356">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-pfam-details-svg" data-filename="scaffold_6_r6c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-357"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-357">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-tigrfam-details-svg" data-filename="scaffold_6_r6c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c6-nrps_pks_products" data-name="r6c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c6-nrps_pks_monomers" data-name="r6c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-352"></div> <span class="help-tooltip" id="nrps-structures-help-352">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 10, location 3458850 - 3502639:
<div class="collapser collapser-target-CC10 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c6-10" class="smiles-canvas" data-smiles="NC(CCCC(=O)O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Aad)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c6-10-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCC(=O)O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-353"></div> <span class="help-tooltip" id="nrps-monomers-help-353">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1101</span></strong>: Aad
<div class="collapser collapser-target-input-path1-gene1101 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (22..383)</strong>: Aad
<div class="collapser collapser-target-input-path1-gene1101 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Aad
<div class="collapser collapser-target-input-path1-gene1101-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPRHLVARSK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Aad <span class="serif">DPRHFVMRSK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 7 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-532"></div> <span class="help-tooltip" id="region_svg-help-532">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c7-svg" data-filename="scaffold_6_r6c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,511,972 - 3,565,093 nt. (total: 53,122 nt)

<a class="cluster-rules-header" id="r6c7-rules-header" href="#r6c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c7-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-533"></div> <span class="help-tooltip" id="focus-panel-help-533">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c7-gene-table-details" data-name="r6c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c7-nrps_pks" data-name="r6c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c7-MIBiG-cluster-compare" data-name="r6c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c7-clusterblast" data-name="r6c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c7-knownclusterblast" data-name="r6c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c7-subclusterblast" data-name="r6c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c7-nrps-pks-bubbles" data-name="r6c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c7-pfam-details" data-name="r6c7-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-366"></div> <span class="help-tooltip" id="gt-help-tooltip-help-366">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-358"></div> <span class="help-tooltip" id="nrps-domain-help-358">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c7-details-svg" data-filename="scaffold_6_r6c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-359"></div> <span class="help-tooltip" id="MIBiG-help-359">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c7"></div>
<div id="comparison-MIBiG-r6c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>pyrrolizixenamide A</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>NRP</td>

<td>pseudomonine</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.19</td>
<td>NRP</td>

<td>KK-1</td>


<td>Curvularia clavata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002424: 0-19621">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002424/index.html#r1c1" target="_new">BGC0002424</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>saccharochelin A, saccharochelin B, saccharochelin C, saccharochelin D, saccharochelin E, saccharochelin F, saccharochelin G, saccharochelin H</td>


<td>Saccharothrix sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002160: 880-28810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002160/index.html#r1c1" target="_new">BGC0002160</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>aspergillicin A, aspergillicin F, aspergillicin G</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>pyrrolizixenamide A</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>NRP</td>
<td>pseudomonine</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001636: 2807-73774">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001636/index.html#r1c1" target="_new">BGC0001636</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>KK-1</td>
<td>Curvularia clavata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002424: 0-19621">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002424/index.html#r1c1" target="_new">BGC0002424</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>saccharochelin A, saccharochelin B, saccharochelin C, saccharochelin D, saccharochelin E, saccharochelin F, saccharochelin G, saccharochelin H</td>
<td>Saccharothrix sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002160: 880-28810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002160/index.html#r1c1" target="_new">BGC0002160</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>aspergillicin A, aspergillicin F, aspergillicin G</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-360"></div> <span class="help-tooltip" id="cb-general-help-360">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c7_1.svg">
NC_070554 (2627617-2701809): Fusarium falciforme chromosome 11, complete sequ... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_2.svg">
NC_072849 (2666227-2740414): Fusarium oxysporum Fo47 chromosome X, complete s... (33% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_3.svg">
NW_024467525 (4378280-4483078): Aspergillus versicolor CBS 583.65 unplaced ge... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c7_4.svg">
NW_024467859 (5395537-5469740): Aspergillus sydowii CBS 593.65 unplaced genom... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_5.svg">
NC_054858 (33608-143759): Aspergillus puulaauensis MK2 DNA, chromosome 2, nea... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c7_6.svg">
NW_022985570 (41698-115964): Pseudovirgaria hyperparasitica strain CBS 121739... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_7.svg">
NW_001820829 (1536004-1610195): Sclerotinia sclerotiorum 1980 UF-70 scaffold ... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_8.svg">
NC_049561 (2485042-2559251): Talaromyces rugulosus chromosome I, complete seq... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_9.svg">
NW_020291491 (170449-238474): Aspergillus aculeatinus CBS 121060 unplaced gen... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_10.svg">
NW_024468980 (278081-345365): Aspergillus fijiensis CBS 313.89 unplaced genom... (12% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r6c7-download">Download graphic</button>
</div>
<div id="clusterblast-r6c7-svg"></div>

</div>
</div>

<div class="body-details-section r6c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-361"></div> <span class="help-tooltip" id="cb-known-help-361">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c7-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c7_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c7_1.svg">
epichloenin A
</option>

<option value = "svg/knownclusterblast_r6c7_2.svg">
epichloenin A
</option>

</select>
<button id="knownclusterblast-r6c7-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c7-svg"></div>

</div>
</div>

<div class="body-details-section r6c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-362"></div> <span class="help-tooltip" id="cb-sub-help-362">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-365"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-365">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c7-domain-bubble-svg" data-filename="scaffold_6_r6c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC11">
Candidate 11 (3511971 - 3565093): single NRPS
</option>


</select>

<div class="details-svg" id="r6c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-367"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-367">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c7-pfam-details-svg" data-filename="scaffold_6_r6c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c7-nrps_pks_products" data-name="r6c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c7-nrps_pks_monomers" data-name="r6c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-363"></div> <span class="help-tooltip" id="nrps-structures-help-363">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 11, location 3511971 - 3565093:
<div class="collapser collapser-target-CC11 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c7-11" class="smiles-canvas" data-smiles="NC(CCC(=O)N)C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gln - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c7-11-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCC(=O)N)C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-364"></div> <span class="help-tooltip" id="nrps-monomers-help-364">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1119</span></strong>: X - Gln - X
<div class="collapser collapser-target-input-path1-gene1119 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (1..313)</strong>: X
<div class="collapser collapser-target-input-path1-gene1119 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene1119-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMTLWGGIFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Cyc-D-Gly <span class="serif">DAYLWGGVFK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>





<dd><strong>AMP-binding (1003..1412)</strong>: Gln
<div class="collapser collapser-target-input-path1-gene1119 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gln
<div class="collapser collapser-target-input-path1-gene1119-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVRSTGVINK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gln <span class="serif">DACSTGVINK</span> (85% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2115..2522)</strong>: X
<div class="collapser collapser-target-input-path1-gene1119 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe, Trp, Ph-Gly, Tyr, bOH-Tyr
<div class="collapser collapser-target-input-path1-gene1119-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aromatic side chain (Phe, Trp, Ph-Gly, Tyr, bOH-Tyr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVGGAGAVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>E-ahmohOrn <span class="serif">DVGGGGVIGK</span> (85% 8Å match)</dd>
<dd>Trp <span class="serif">DVGMVGAVAK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>














</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r7c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-534"></div> <span class="help-tooltip" id="region_svg-help-534">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c1-svg" data-filename="scaffold_7_r7c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,607 - 51,243 nt. (total: 45,637 nt)

<a class="cluster-rules-header" id="r7c1-rules-header" href="#r7c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r7c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-535"></div> <span class="help-tooltip" id="focus-panel-help-535">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c1-gene-table-details" data-name="r7c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c1-nrps_pks" data-name="r7c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c1-MIBiG-cluster-compare" data-name="r7c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c1-clusterblast" data-name="r7c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c1-knownclusterblast" data-name="r7c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c1-subclusterblast" data-name="r7c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c1-nrps-pks-bubbles" data-name="r7c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c1-pfam-details" data-name="r7c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c1-tigrfam-details" data-name="r7c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-376"></div> <span class="help-tooltip" id="gt-help-tooltip-help-376">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-368"></div> <span class="help-tooltip" id="nrps-domain-help-368">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-details-svg" data-filename="scaffold_7_r7c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-369"></div> <span class="help-tooltip" id="MIBiG-help-369">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c1"></div>
<div id="comparison-MIBiG-r7c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.39</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876: 71-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Other</td>

<td>neopolyoxin C</td>


<td>Streptomyces tendae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876: 71-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>neopolyoxin C</td>
<td>Streptomyces tendae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-370"></div> <span class="help-tooltip" id="cb-general-help-370">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c1_1.svg">
NW_020167533 (924875-991241): Pseudogymnoascus destructans isolate 20631-21 u... (50% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c1_2.svg">
NC_007196 (862437-944474): Aspergillus fumigatus Af293 chromosome 3, whole ge... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c1_3.svg">
NC_054698 (836604-936663): Aspergillus flavus NRRL3357 chromosome 8, complete... (19% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c1_4.svg">
NW_019170457 (891496-957720): Aspergillus glaucus CBS 516.65 unplaced genomic... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c1_5.svg">
NC_036442 (803563-869335): Aspergillus oryzae RIB40 DNA, chromosome 8 (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c1_6.svg">
NW_025091265 (1175759-1262422): Aspergillus udagawae IFM 46973 DNA, scaffold ... (17% of genes show similarity), NI-siderophore,NRPS
</option>

<option value = "svg/clusterblast_r7c1_7.svg">
NW_025091284 (1562771-1677454): Aspergillus pseudoviridinutans IFM 55266 DNA,... (13% of genes show similarity), NI-siderophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c1_8.svg">
NC_066257 (766757-833124): Aspergillus nidulans FGSC A4 chromosome I (27% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c1_9.svg">
NW_024467863 (1585733-1652038): Aspergillus sydowii CBS 593.65 unplaced genom... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c1_10.svg">
NC_054861 (3243484-3309791): Aspergillus puulaauensis MK2 DNA, chromosome 5, ... (23% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r7c1-download">Download graphic</button>
</div>
<div id="clusterblast-r7c1-svg"></div>

</div>
</div>

<div class="body-details-section r7c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-371"></div> <span class="help-tooltip" id="cb-known-help-371">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-372"></div> <span class="help-tooltip" id="cb-sub-help-372">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-375"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-375">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-domain-bubble-svg" data-filename="scaffold_7_r7c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (5606 - 51243): single NRPS
</option>


</select>

<div class="details-svg" id="r7c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-377"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-377">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-pfam-details-svg" data-filename="scaffold_7_r7c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-378"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-378">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-tigrfam-details-svg" data-filename="scaffold_7_r7c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c1-nrps_pks_products" data-name="r7c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c1-nrps_pks_monomers" data-name="r7c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-373"></div> <span class="help-tooltip" id="nrps-structures-help-373">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 5606 - 51243:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-374"></div> <span class="help-tooltip" id="nrps-monomers-help-374">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene6</span></strong>: X
<div class="collapser collapser-target-input-path1-gene6 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (5..412)</strong>: X
<div class="collapser collapser-target-input-path1-gene6 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Z-ahmohOrn
<div class="collapser collapser-target-input-path1-gene6-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aromatic side chain (Phe, Trp, Ph-Gly, Tyr, bOH-Tyr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVDGGGGIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">DVDGGGGIGK</span> (94% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-536"></div> <span class="help-tooltip" id="region_svg-help-536">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c2-svg" data-filename="scaffold_7_r7c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 120,190 - 144,256 nt. (total: 24,067 nt)

<a class="cluster-rules-header" id="r7c2-rules-header" href="#r7c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r7c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-537"></div> <span class="help-tooltip" id="focus-panel-help-537">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c2-gene-table-details" data-name="r7c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c2-MIBiG-cluster-compare" data-name="r7c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c2-clusterblast" data-name="r7c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c2-knownclusterblast" data-name="r7c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c2-subclusterblast" data-name="r7c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r7c2-pfam-details" data-name="r7c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c2-tigrfam-details" data-name="r7c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-383"></div> <span class="help-tooltip" id="gt-help-tooltip-help-383">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-379"></div> <span class="help-tooltip" id="MIBiG-help-379">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c2"></div>
<div id="comparison-MIBiG-r7c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Other</td>

<td>hydroxycinnamoyltyramine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675: 0-4976">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>(+)-T-muurolol</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>hydroxycinnamoyltyramine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675: 0-4976">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>(+)-T-muurolol</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-380"></div> <span class="help-tooltip" id="cb-general-help-380">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c2_1.svg">
NW_025091284 (1355867-1424289): Aspergillus pseudoviridinutans IFM 55266 DNA,... (18% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r7c2_2.svg">
NC_064679 (5392829-5459152): Colletotrichum lupini chromosome 6, complete seq... (27% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r7c2_3.svg">
NC_054693 (3931870-3995086): Aspergillus flavus NRRL3357 chromosome 3, comple... (14% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r7c2_4.svg">
NC_036437 (4008631-4071847): Aspergillus oryzae RIB40 DNA, chromosome 3 (14% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r7c2_5.svg">
NW_026054650 (362143-429610): Ophidiomyces ophidiicola strain CBS 122913 chro... (15% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c2_6.svg">
NC_026475 (6633069-6664256): Fusarium graminearum PH-1 chromosome 2, whole ge... (15% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_7.svg">
NC_058400 (5864309-5895503): Fusarium poae strain DAOMC 252244 chromosome 2, ... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_8.svg">
NC_038013 (5603203-5634406): Fusarium venenatum strain A3/5 genome assembly, ... (16% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_9.svg">
NC_070540 (2401382-2432591): Fusarium keratoplasticum isolate Fu6.1 chromosom... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_10.svg">
NW_025408683 (1085865-1154571): Hirsutella rhossiliensis strain HR02 chromoso... (16% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r7c2-download">Download graphic</button>
</div>
<div id="clusterblast-r7c2-svg"></div>

</div>
</div>

<div class="body-details-section r7c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-381"></div> <span class="help-tooltip" id="cb-known-help-381">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r7c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r7c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r7c2_1.svg">
trichodiene-11-one
</option>

<option value = "svg/knownclusterblast_r7c2_2.svg">
nivalenol/vomitoxin/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-deacetylcalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

<option value = "svg/knownclusterblast_r7c2_3.svg">
nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

<option value = "svg/knownclusterblast_r7c2_4.svg">
deoxynivalenol
</option>

</select>
<button id="knownclusterblast-r7c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r7c2-svg"></div>

</div>
</div>

<div class="body-details-section r7c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-382"></div> <span class="help-tooltip" id="cb-sub-help-382">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r7c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-384"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-384">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-pfam-details-svg" data-filename="scaffold_7_r7c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-385"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-385">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-tigrfam-details-svg" data-filename="scaffold_7_r7c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r7c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 3 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-538"></div> <span class="help-tooltip" id="region_svg-help-538">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c3-svg" data-filename="scaffold_7_r7c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 534,078 - 581,937 nt. (total: 47,860 nt)

<a class="cluster-rules-header" id="r7c3-rules-header" href="#r7c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r7c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-539"></div> <span class="help-tooltip" id="focus-panel-help-539">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c3-gene-table-details" data-name="r7c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c3-nrps_pks" data-name="r7c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c3-MIBiG-cluster-compare" data-name="r7c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c3-clusterblast" data-name="r7c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c3-knownclusterblast" data-name="r7c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c3-subclusterblast" data-name="r7c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c3-nrps-pks-bubbles" data-name="r7c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c3-pfam-details" data-name="r7c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-394"></div> <span class="help-tooltip" id="gt-help-tooltip-help-394">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-386"></div> <span class="help-tooltip" id="nrps-domain-help-386">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-details-svg" data-filename="scaffold_7_r7c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-387"></div> <span class="help-tooltip" id="MIBiG-help-387">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c3"></div>
<div id="comparison-MIBiG-r7c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002185: 100-28222">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002185/index.html#r1c1" target="_new">BGC0002185</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>gregatin A</td>


<td>Penicillium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002219: 0-26397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002219/index.html#r1c1" target="_new">BGC0002219</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Terpene</td>

<td>(-)-(R)-nephthenol</td>


<td>Stereum hirsutum FP-91666 SS1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001907: 74-38528">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001907/index.html#r1c1" target="_new">BGC0001907</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>chromane</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002185: 100-28222">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002185/index.html#r1c1" target="_new">BGC0002185</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>gregatin A</td>
<td>Penicillium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002219: 0-26397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002219/index.html#r1c1" target="_new">BGC0002219</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene</td>
<td>(-)-(R)-nephthenol</td>
<td>Stereum hirsutum FP-91666 SS1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001907: 74-38528">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001907/index.html#r1c1" target="_new">BGC0001907</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>chromane</td>
<td>Daldinia eschscholzii IFB-TL01</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-388"></div> <span class="help-tooltip" id="cb-general-help-388">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c3_1.svg">
NW_025953807 (113785-185922): Annulohypoxylon truncatum strain CBS 140777 unp... (25% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r7c3_2.svg">
NW_025953250 (2330314-2401531): Annulohypoxylon maeteangense strain CBS 12383... (25% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r7c3_3.svg">
NC_026474 (10853503-10932309): Fusarium graminearum PH-1 chromosome 1, whole ... (12% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_4.svg">
NW_024467525 (3144728-3217090): Aspergillus versicolor CBS 583.65 unplaced ge... (17% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_5.svg">
NC_054858 (1386967-1459328): Aspergillus puulaauensis MK2 DNA, chromosome 2, ... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_6.svg">
NC_049565 (5160731-5287596): Talaromyces rugulosus chromosome V, complete seq... (9% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_7.svg">
NW_016017388 (1492810-1594298): Sphaerulina musiva SO2202 unplaced genomic sc... (9% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c3_8.svg">
NC_066261 (362715-434651): Aspergillus nidulans FGSC A4 chromosome V (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c3_9.svg">
NW_013550603 (4299149-4365274): Rhinocladiella mackenziei CBS 650.93 unplaced... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c3_10.svg">
NW_007360993 (564614-637444): Glarea lozoyensis ATCC 20868 chromosome Unknown... (18% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r7c3-download">Download graphic</button>
</div>
<div id="clusterblast-r7c3-svg"></div>

</div>
</div>

<div class="body-details-section r7c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-389"></div> <span class="help-tooltip" id="cb-known-help-389">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r7c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r7c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r7c3_1.svg">
fusarielin H
</option>

<option value = "svg/knownclusterblast_r7c3_2.svg">
burnettiene A/preburnettiene B/preburnettiene A
</option>

</select>
<button id="knownclusterblast-r7c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r7c3-svg"></div>

</div>
</div>

<div class="body-details-section r7c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-390"></div> <span class="help-tooltip" id="cb-sub-help-390">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-393"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-393">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-domain-bubble-svg" data-filename="scaffold_7_r7c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (534077 - 581937): single T1PKS
</option>


</select>

<div class="details-svg" id="r7c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-395"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-395">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-pfam-details-svg" data-filename="scaffold_7_r7c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c3-nrps_pks_products" data-name="r7c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c3-nrps_pks_monomers" data-name="r7c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-391"></div> <span class="help-tooltip" id="nrps-structures-help-391">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 534077 - 581937:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c3-3" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-392"></div> <span class="help-tooltip" id="nrps-monomers-help-392">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene172</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene172 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (563..878)</strong>: pk
<div class="collapser collapser-target-input-path1-gene172 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene172-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene172-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 83.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 64.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 53.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 42.5</dt>

<dd></dd><dt>Benzoyl-CoA: 32.4</dt>

<dd></dd><dt>Malonyl-CoA: 31.8</dt>

<dd></dd><dt>Propionyl-CoA: 23.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 21.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 18.6</dt>

<dd></dd><dt>fatty_acid: 15.9</dt>

<dd></dd><dt>CHC-CoA: 12.5</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: -2.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 4 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-540"></div> <span class="help-tooltip" id="region_svg-help-540">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c4-svg" data-filename="scaffold_7_r7c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 700,447 - 721,871 nt. (total: 21,425 nt)

<a class="cluster-rules-header" id="r7c4-rules-header" href="#r7c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c4-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r7c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-541"></div> <span class="help-tooltip" id="focus-panel-help-541">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c4-gene-table-details" data-name="r7c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c4-MIBiG-cluster-compare" data-name="r7c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c4-clusterblast" data-name="r7c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c4-knownclusterblast" data-name="r7c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c4-subclusterblast" data-name="r7c4-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r7c4-pfam-details" data-name="r7c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-400"></div> <span class="help-tooltip" id="gt-help-tooltip-help-400">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-396"></div> <span class="help-tooltip" id="MIBiG-help-396">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c4"></div>
<div id="comparison-MIBiG-r7c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>pimara-8(14),15-diene</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.52, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>sartorypyrone A</td>


<td>Aspergillus felis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.51, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Terpene, Polyketide</td>

<td>subglutinol A, subglutinol B</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.50, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>


<td>Penicillium paxilli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.51, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>chevalone E</td>


<td>Aspergillus versicolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.51, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>shearinine D</td>


<td>Penicillium janthinellum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>pimara-8(14),15-diene</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.52, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>sartorypyrone A</td>
<td>Aspergillus felis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.51, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Terpene, Polyketide</td>
<td>subglutinol A, subglutinol B</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.50, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Terpene</td>
<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>
<td>Penicillium paxilli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.51, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>chevalone E</td>
<td>Aspergillus versicolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.51, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Terpene</td>
<td>shearinine D</td>
<td>Penicillium janthinellum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-397"></div> <span class="help-tooltip" id="cb-general-help-397">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c4_1.svg">
NW_006271971 (242847-274242): Cordyceps militaris CM01 unplaced genomic scaff... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c4_2.svg">
NC_070533 (2808830-2840090): Fusarium keratoplasticum isolate Fu6.1 chromosom... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c4_3.svg">
NC_070548 (2879257-2910520): Fusarium falciforme chromosome 5, complete sequence (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c4_4.svg">
NW_011942182 (2894477-2966843): Metarhizium robertsii ARSEF 23 MAA Scf 4, who... (13% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r7c4_5.svg">
NW_014574713 (814784-887446): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (15% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r7c4_6.svg">
NW_006711170 (271878-302853): Trichoderma reesei QM6a unplaced genomic scaffo... (42% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c4_7.svg">
NW_024065901 (1622993-1655154): Colletotrichum karsti strain CkLH20 chromosom... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c4_8.svg">
NW_021167089 (220979-252224): Sodiomyces alkalinus F11 unplaced genomic scaff... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c4_9.svg">
NW_006917094 (1335307-1366507): Pestalotiopsis fici W106-1 unplaced genomic s... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c4_10.svg">
NC_043743 (1411668-1442907): Pyricularia pennisetigena strain Br36 chromosome... (27% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r7c4-download">Download graphic</button>
</div>
<div id="clusterblast-r7c4-svg"></div>

</div>
</div>

<div class="body-details-section r7c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-398"></div> <span class="help-tooltip" id="cb-known-help-398">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-399"></div> <span class="help-tooltip" id="cb-sub-help-399">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r7c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-401"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-401">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c4-pfam-details-svg" data-filename="scaffold_7_r7c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r7c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 5 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-542"></div> <span class="help-tooltip" id="region_svg-help-542">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c5-svg" data-filename="scaffold_7_r7c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 996,901 - 1,046,628 nt. (total: 49,728 nt)

<a class="cluster-rules-header" id="r7c5-rules-header" href="#r7c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c5-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r7c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-543"></div> <span class="help-tooltip" id="focus-panel-help-543">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c5-gene-table-details" data-name="r7c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c5-nrps_pks" data-name="r7c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c5-MIBiG-cluster-compare" data-name="r7c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c5-clusterblast" data-name="r7c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c5-knownclusterblast" data-name="r7c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c5-subclusterblast" data-name="r7c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c5-nrps-pks-bubbles" data-name="r7c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c5-pfam-details" data-name="r7c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c5-tigrfam-details" data-name="r7c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-410"></div> <span class="help-tooltip" id="gt-help-tooltip-help-410">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-402"></div> <span class="help-tooltip" id="nrps-domain-help-402">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-details-svg" data-filename="scaffold_7_r7c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-403"></div> <span class="help-tooltip" id="MIBiG-help-403">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c5"></div>
<div id="comparison-MIBiG-r7c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>


<td>Chaetomium olivaceum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>xenematide</td>


<td>Xenorhabdus nematophila AN6/1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP, Alkaloid</td>

<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>


<td>Pseudomonas fluorescens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002538: 0-17732">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002538/index.html#r1c1" target="_new">BGC0002538</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>chaetolivacine A, chaetolivacine B, chaetolivacine C</td>
<td>Chaetomium olivaceum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>xenematide</td>
<td>Xenorhabdus nematophila AN6/1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP, Alkaloid</td>
<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>
<td>Pseudomonas fluorescens</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-404"></div> <span class="help-tooltip" id="cb-general-help-404">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c5_1.svg">
NW_021167081 (267870-339989): Sodiomyces alkalinus F11 unplaced genomic scaff... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c5_2.svg">
NW_025091284 (4592542-4724179): Aspergillus pseudoviridinutans IFM 55266 DNA,... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c5_3.svg">
NW_006763054 (106798-184191): Drepanopeziza brunnea f. sp. &#39;multigermtubi&#39; MB... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c5_4.svg">
NW_024467874 (99002-166751): Aspergillus sydowii CBS 593.65 unplaced genomic ... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c5_5.svg">
NW_025091224 (1326221-1421444): Aspergillus viridinutans IFM 47045 DNA, scaff... (6% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c5_6.svg">
NW_003052500 (3318567-3442588): Uncinocarpus reesii 1704 scaffold 1 genomic s... (5% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c5_7.svg">
NW_024067594 (118838-184232): Penicillium roqueforti strain LCP96 04111 chrom... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c5_8.svg">
NC_030957 (3250063-3360781): Colletotrichum higginsianum IMI 349063 chromosom... (5% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c5_9.svg">
NZ_CM001439 (2826594-2847755): Saccharomonospora marina XMU15 chromosome, who... (10% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c5_10.svg">
NW_017253367 (2050017-2132786): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (5% of genes show similarity), isocyanide-nrp
</option>

</select>
<button id= "clusterblast-r7c5-download">Download graphic</button>
</div>
<div id="clusterblast-r7c5-svg"></div>

</div>
</div>

<div class="body-details-section r7c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-405"></div> <span class="help-tooltip" id="cb-known-help-405">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-406"></div> <span class="help-tooltip" id="cb-sub-help-406">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-409"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-409">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-domain-bubble-svg" data-filename="scaffold_7_r7c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (996900 - 1046628): single NRPS
</option>


</select>

<div class="details-svg" id="r7c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-411"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-411">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-pfam-details-svg" data-filename="scaffold_7_r7c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-412"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-412">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-tigrfam-details-svg" data-filename="scaffold_7_r7c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c5-nrps_pks_products" data-name="r7c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c5-nrps_pks_monomers" data-name="r7c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-407"></div> <span class="help-tooltip" id="nrps-structures-help-407">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 996900 - 1046628:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c5-5" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c5-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-408"></div> <span class="help-tooltip" id="nrps-monomers-help-408">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene319</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene319 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (14..326)</strong>: pk
<div class="collapser collapser-target-input-path1-gene319 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene319-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Isobutyryl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene319-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 91.5</dt>

<dd></dd><dt>Malonyl-CoA: 90.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 82.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 78.1</dt>

<dd></dd><dt>Propionyl-CoA: 53.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 38.4</dt>

<dd></dd><dt>inactive: 37.5</dt>

<dd></dd><dt>Benzoyl-CoA: 29.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 19.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 19.7</dt>

<dd></dd><dt>Acetyl-CoA: 18.5</dt>

<dd></dd><dt>fatty_acid: 11.0</dt>

<dd></dd><dt>CHC-CoA: 10.8</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 5.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>









<dd><strong>AMP-binding (2143..2553)</strong>: X
<div class="collapser collapser-target-input-path1-gene319 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene319-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMHITGGIIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DMHMLGVIIK</span> (41% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 6 - NRP-metallophore,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-544"></div> <span class="help-tooltip" id="region_svg-help-544">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c6-svg" data-filename="scaffold_7_r7c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,735,616 - 1,795,950 nt. (total: 60,335 nt)

<a class="cluster-rules-header" id="r7c6-rules-header" href="#r7c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c6-rules">NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r7c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-545"></div> <span class="help-tooltip" id="focus-panel-help-545">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c6-gene-table-details" data-name="r7c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c6-nrps_pks" data-name="r7c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c6-MIBiG-cluster-compare" data-name="r7c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c6-clusterblast" data-name="r7c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c6-knownclusterblast" data-name="r7c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c6-subclusterblast" data-name="r7c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c6-nrps-pks-bubbles" data-name="r7c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c6-pfam-details" data-name="r7c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c6-tigrfam-details" data-name="r7c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-421"></div> <span class="help-tooltip" id="gt-help-tooltip-help-421">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-413"></div> <span class="help-tooltip" id="nrps-domain-help-413">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c6-details-svg" data-filename="scaffold_7_r7c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-414"></div> <span class="help-tooltip" id="MIBiG-help-414">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c6"></div>
<div id="comparison-MIBiG-r7c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.41</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>






<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.30</td>
<td>NRP, Polyketide</td>

<td>myxoprincomide-c506</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>






<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>






<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>pseudomonine</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349: 0-24145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>






<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>erythrochelin</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP, Polyketide</td>
<td>myxoprincomide-c506</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410: 162-16011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>NRP</td>
<td>pseudomonine</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349: 0-24145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>NRP</td>
<td>erythrochelin</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-415"></div> <span class="help-tooltip" id="cb-general-help-415">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c6_1.svg">
NW_023336198 (10188-84865): Geosmithia morbida isolate 1262 chromosome Unknow... (33% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c6_2.svg">
NW_006711178 (9878-84649): Trichoderma reesei QM6a unplaced genomic scaffold ... (31% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c6_3.svg">
NC_070530 (3742001-3826089): Fusarium keratoplasticum isolate Fu6.1 chromosom... (41% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r7c6_4.svg">
NC_070545 (3816730-3900832): Fusarium falciforme chromosome 2, complete sequence (41% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r7c6_5.svg">
NC_036623 (2195750-2279651): Fusarium fujikuroi IMI 58289 draft genome, chrom... (38% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r7c6_6.svg">
NC_058401 (2692264-2776229): Fusarium poae strain DAOMC 252244 chromosome 3, ... (33% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r7c6_7.svg">
NC_058389 (2698749-2782755): Fusarium musae strain F31 chromosome 3, whole ge... (44% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r7c6_8.svg">
NC_026476 (2004097-2087970): Fusarium graminearum PH-1 chromosome 3, whole ge... (29% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r7c6_9.svg">
NW_011942149 (5526781-5608010): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (38% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r7c6_10.svg">
NW_014574715 (339608-420863): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (38% of genes show similarity), NRP-metallophore,NRPS
</option>

</select>
<button id= "clusterblast-r7c6-download">Download graphic</button>
</div>
<div id="clusterblast-r7c6-svg"></div>

</div>
</div>

<div class="body-details-section r7c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-416"></div> <span class="help-tooltip" id="cb-known-help-416">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-417"></div> <span class="help-tooltip" id="cb-sub-help-417">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-420"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-420">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c6-domain-bubble-svg" data-filename="scaffold_7_r7c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (1735615 - 1795950): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r7c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-422"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-422">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c6-pfam-details-svg" data-filename="scaffold_7_r7c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-423"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-423">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c6-tigrfam-details-svg" data-filename="scaffold_7_r7c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c6-nrps_pks_products" data-name="r7c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c6-nrps_pks_monomers" data-name="r7c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-418"></div> <span class="help-tooltip" id="nrps-structures-help-418">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 1735615 - 1795950:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c6-6" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c6-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-419"></div> <span class="help-tooltip" id="nrps-monomers-help-419">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene538</span></strong>: X - X
<div class="collapser collapser-target-input-path1-gene538 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>AMP-binding (717..1108)</strong>: X
<div class="collapser collapser-target-input-path1-gene538 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene538-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFSVAGNIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DVFSVALVYK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2323..2721)</strong>: X
<div class="collapser collapser-target-input-path1-gene538 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-input-path1-gene538-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVLDIGAIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DVLSFGAIIK</span> (65% 8Å match)</dd>
<dd>Phe or Trp <span class="serif">DVLSIGAVVK</span> (62% 8Å match)</dd>
<dd>Arg <span class="serif">DVESIGAIAK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>














</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_7 - Region 7 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-546"></div> <span class="help-tooltip" id="region_svg-help-546">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_7.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c7-svg" data-filename="scaffold_7_r7c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,231,056 - 2,280,851 nt. (total: 49,796 nt)

<a class="cluster-rules-header" id="r7c7-rules-header" href="#r7c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c7-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r7c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-547"></div> <span class="help-tooltip" id="focus-panel-help-547">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c7-gene-table-details" data-name="r7c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c7-nrps_pks" data-name="r7c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c7-MIBiG-cluster-compare" data-name="r7c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c7-clusterblast" data-name="r7c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c7-knownclusterblast" data-name="r7c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c7-subclusterblast" data-name="r7c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c7-nrps-pks-bubbles" data-name="r7c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c7-pfam-details" data-name="r7c7-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-432"></div> <span class="help-tooltip" id="gt-help-tooltip-help-432">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-424"></div> <span class="help-tooltip" id="nrps-domain-help-424">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c7-details-svg" data-filename="scaffold_7_r7c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-425"></div> <span class="help-tooltip" id="MIBiG-help-425">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c7"></div>
<div id="comparison-MIBiG-r7c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>phomoidride</td>


<td>fungal sp. ATCC 74256</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>epipyriculol</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>trichoxide</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>flavoglaucin</td>


<td>Aspergillus ruber CBS 135680</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:0.39, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>phomoidride</td>
<td>fungal sp. ATCC 74256</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>epipyriculol</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>flavoglaucin</td>
<td>Aspergillus ruber CBS 135680</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:0.39, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-426"></div> <span class="help-tooltip" id="cb-general-help-426">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c7_1.svg">
NW_006271970 (1369951-1438884): Cordyceps militaris CM01 unplaced genomic sca... (61% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c7_2.svg">
NW_025408667 (6621596-6715308): Hirsutella rhossiliensis strain HR02 chromoso... (35% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c7_3.svg">
NW_024468918 (1133295-1261954): Cryphonectria parasitica EP155 unplaced genom... (9% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r7c7_4.svg">
NW_006711153 (459708-526750): Trichoderma reesei QM6a unplaced genomic scaffo... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c7_5.svg">
NC_070539 (1227558-1294574): Fusarium keratoplasticum isolate Fu6.1 chromosom... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c7_6.svg">
NC_070554 (1471044-1538049): Fusarium falciforme chromosome 11, complete sequ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c7_7.svg">
NC_035790 (8523905-8591015): Pochonia chlamydosporia 170 chromosome 1, whole ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c7_8.svg">
NW_025091220 (124401-191227): Aspergillus viridinutans IFM 47045 DNA, scaffol... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c7_9.svg">
NW_009276938 (1486712-1553486): Verticillium dahliae VdLs.17 supercont1.5 gen... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c7_10.svg">
NW_013581198 (762797-842941): Aureobasidium namibiae CBS 147.97 unplaced geno... (7% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r7c7-download">Download graphic</button>
</div>
<div id="clusterblast-r7c7-svg"></div>

</div>
</div>

<div class="body-details-section r7c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-427"></div> <span class="help-tooltip" id="cb-known-help-427">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-428"></div> <span class="help-tooltip" id="cb-sub-help-428">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-431"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-431">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c7-domain-bubble-svg" data-filename="scaffold_7_r7c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (2231055 - 2280851): single T1PKS
</option>


</select>

<div class="details-svg" id="r7c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-433"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-433">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c7-pfam-details-svg" data-filename="scaffold_7_r7c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c7-nrps_pks_products" data-name="r7c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c7-nrps_pks_monomers" data-name="r7c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-429"></div> <span class="help-tooltip" id="nrps-structures-help-429">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 2231055 - 2280851:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c7-7" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c7-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-430"></div> <span class="help-tooltip" id="nrps-monomers-help-430">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene687</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene687 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (537..847)</strong>: pk
<div class="collapser collapser-target-input-path1-gene687 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene687-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Isobutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene687-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 64.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 60.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 42.4</dt>

<dd></dd><dt>Propionyl-CoA: 40.2</dt>

<dd></dd><dt>Malonyl-CoA: 37.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 24.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 23.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 14.1</dt>

<dd></dd><dt>inactive: 13.5</dt>

<dd></dd><dt>Benzoyl-CoA: 12.2</dt>

<dd></dd><dt>fatty_acid: 7.6</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r8c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_8 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-548"></div> <span class="help-tooltip" id="region_svg-help-548">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_8.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r8c1-svg" data-filename="scaffold_8_r8c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,913,883 - 1,959,398 nt. (total: 45,516 nt)

<a class="cluster-rules-header" id="r8c1-rules-header" href="#r8c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r8c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r8c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-549"></div> <span class="help-tooltip" id="focus-panel-help-549">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r8c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r8c1-gene-table-details" data-name="r8c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r8c1-nrps_pks" data-name="r8c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r8c1-MIBiG-cluster-compare" data-name="r8c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r8c1-clusterblast" data-name="r8c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r8c1-knownclusterblast" data-name="r8c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r8c1-subclusterblast" data-name="r8c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r8c1-pfam-details" data-name="r8c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r8c1-tigrfam-details" data-name="r8c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r8c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-440"></div> <span class="help-tooltip" id="gt-help-tooltip-help-440">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r8c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r8c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r8c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r8c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r8c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-434"></div> <span class="help-tooltip" id="nrps-domain-help-434">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c1-details-svg" data-filename="scaffold_8_r8c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r8c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r8c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-435"></div> <span class="help-tooltip" id="MIBiG-help-435">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r8c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r8c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r8c1"></div>
<div id="comparison-MIBiG-r8c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.55, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002619: 0-42148">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002619/index.html#r1c1" target="_new">BGC0002619</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.57, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>mycophenolic acid</td>


<td>Penicillium brevicompactum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868: 0-11041">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.49, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Other (Phenazine)</td>

<td>iodinin</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002089: 100-18365">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002089/index.html#r1c1" target="_new">BGC0002089</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>thaxtomin D, thaxtomin A, thaxtomin C, thaxtomin B</td>


<td>Streptomyces scabiei 87.22</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.49, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>1,6-phenazinedimethanol, fontizine A</td>


<td>Serratia fonticola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392: 0-27717">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.45, order:0.29, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>mirubactin</td>


<td>Actinosynnema mirum DSM 43827</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000454: 24-19884">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000454/index.html#r1c1" target="_new">BGC0000454</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.49, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>vanchrobactin</td>


<td>Vibrio anguillarum RV22</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000460: 0-23711">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000460/index.html#r1c1" target="_new">BGC0000460</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.50, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>vulnibactin</td>


<td>Vibrio vulnificus CMCP6</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r8c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.55, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002619: 0-42148">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002619/index.html#r1c1" target="_new">BGC0002619</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.57, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>mycophenolic acid</td>
<td>Penicillium brevicompactum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868: 0-11041">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.49, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Other (Phenazine)</td>
<td>iodinin</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002089: 100-18365">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002089/index.html#r1c1" target="_new">BGC0002089</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>NRP</td>
<td>thaxtomin D, thaxtomin A, thaxtomin C, thaxtomin B</td>
<td>Streptomyces scabiei 87.22</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.49, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>NRP</td>
<td>1,6-phenazinedimethanol, fontizine A</td>
<td>Serratia fonticola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392: 0-27717">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.45, order:0.29, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>NRP</td>
<td>mirubactin</td>
<td>Actinosynnema mirum DSM 43827</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000454: 24-19884">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000454/index.html#r1c1" target="_new">BGC0000454</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.49, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>NRP</td>
<td>vanchrobactin</td>
<td>Vibrio anguillarum RV22</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000460: 0-23711">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000460/index.html#r1c1" target="_new">BGC0000460</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.50, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>NRP</td>
<td>vulnibactin</td>
<td>Vibrio vulnificus CMCP6</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r8c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-436"></div> <span class="help-tooltip" id="cb-general-help-436">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r8c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r8c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r8c1_1.svg">
NW_006271972 (846900-910624): Cordyceps militaris CM01 unplaced genomic scaff... (59% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_2.svg">
NW_025408674 (2031871-2095823): Hirsutella rhossiliensis strain HR02 chromoso... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_3.svg">
NC_030957 (4696242-4786854): Colletotrichum higginsianum IMI 349063 chromosom... (20% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r8c1_4.svg">
NC_070546 (1097748-1161429): Fusarium falciforme chromosome 3, complete sequence (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_5.svg">
NC_070531 (1081663-1145343): Fusarium keratoplasticum isolate Fu6.1 chromosom... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_6.svg">
NC_036627 (1441489-1505176): Fusarium fujikuroi IMI 58289 draft genome, chrom... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_7.svg">
NC_058392 (2751608-2815071): Fusarium musae strain F31 chromosome 6, whole ge... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_8.svg">
NC_072845 (1566921-1630612): Fusarium oxysporum Fo47 chromosome VI, complete ... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_9.svg">
NC_058400 (1372797-1436602): Fusarium poae strain DAOMC 252244 chromosome 2, ... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r8c1_10.svg">
NC_038013 (1254253-1318050): Fusarium venenatum strain A3/5 genome assembly, ... (10% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r8c1-download">Download graphic</button>
</div>
<div id="clusterblast-r8c1-svg"></div>

</div>
</div>

<div class="body-details-section r8c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-437"></div> <span class="help-tooltip" id="cb-known-help-437">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r8c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-438"></div> <span class="help-tooltip" id="cb-sub-help-438">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r8c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-441"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-441">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c1-pfam-details-svg" data-filename="scaffold_8_r8c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r8c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r8c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-442"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-442">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c1-tigrfam-details-svg" data-filename="scaffold_8_r8c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r8c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r8c1-nrps_pks_monomers" data-name="r8c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r8c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-439"></div> <span class="help-tooltip" id="nrps-monomers-help-439">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene567</span></strong>: X
<div class="collapser collapser-target-input-path1-gene567 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (58..470)</strong>: X
<div class="collapser collapser-target-input-path1-gene567 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene567-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">IFNSTAAPVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DFESTAAVYK</span> (29% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r8c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_8 - Region 2 - T1PKS,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-550"></div> <span class="help-tooltip" id="region_svg-help-550">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_8.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r8c2-svg" data-filename="scaffold_8_r8c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,125,889 - 2,201,955 nt. (total: 76,067 nt)

<a class="cluster-rules-header" id="r8c2-rules-header" href="#r8c2">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r8c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r8c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-551"></div> <span class="help-tooltip" id="focus-panel-help-551">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r8c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r8c2-gene-table-details" data-name="r8c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r8c2-nrps_pks" data-name="r8c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r8c2-MIBiG-cluster-compare" data-name="r8c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r8c2-clusterblast" data-name="r8c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r8c2-knownclusterblast" data-name="r8c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r8c2-subclusterblast" data-name="r8c2-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r8c2-nrps-pks-bubbles" data-name="r8c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r8c2-pfam-details" data-name="r8c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r8c2-tigrfam-details" data-name="r8c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r8c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-451"></div> <span class="help-tooltip" id="gt-help-tooltip-help-451">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r8c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r8c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r8c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r8c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r8c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-443"></div> <span class="help-tooltip" id="nrps-domain-help-443">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c2-details-svg" data-filename="scaffold_8_r8c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r8c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r8c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-444"></div> <span class="help-tooltip" id="MIBiG-help-444">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r8c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r8c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r8c2"></div>
<div id="comparison-MIBiG-r8c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.27</td>
<td>Terpene, Polyketide</td>

<td>calidoustene A, calidoustene B, calidoustene C</td>


<td>Aspergillus calidoustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000684: 369-21746">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000684/index.html#r1c1" target="_new">BGC0000684</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.46, order:0.32, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>






<td class="cc-heat-cell" title="0.40 (id:0.49, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>asperthecin</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>epipyriculol</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>HEx-pks1 polyketide</td>


<td>Paraphaeosphaeria sporulosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.48, order:0.41, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>esperamicin</td>


<td>Actinomadura verrucosospora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002729: 0-27318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002729/index.html#r1c1" target="_new">BGC0002729</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>sordarial</td>


<td>Neurospora crassa OR74A</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r8c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>HEx-pks1 polyketide</td>
<td>Paraphaeosphaeria sporulosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>epipyriculol</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>flavoglaucin</td>
<td>Aspergillus ruber CBS 135680</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002729: 0-27318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002729/index.html#r1c1" target="_new">BGC0002729</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>sordarial</td>
<td>Neurospora crassa OR74A</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141: 262-37660">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>4-epi-15-epi-brefeldin A</td>
<td>Penicillium brefeldianum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r8c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-445"></div> <span class="help-tooltip" id="cb-general-help-445">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r8c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r8c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r8c2_1.svg">
NW_006271971 (2425175-2516249): Cordyceps militaris CM01 unplaced genomic sca... (22% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r8c2_2.svg">
NW_006271969 (1967760-2079673): Cordyceps militaris CM01 unplaced genomic sca... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r8c2_3.svg">
NW_022985016 (85063-152593): Dothidotthia symphoricarpi CBS 119687 unplaced g... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_4.svg">
NC_049565 (1715696-1783101): Talaromyces rugulosus chromosome V, complete seq... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_5.svg">
NW_013550604 (1398013-1484027): Rhinocladiella mackenziei CBS 650.93 unplaced... (19% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r8c2_6.svg">
NW_006271969 (4860304-4927848): Cordyceps militaris CM01 unplaced genomic sca... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_7.svg">
NW_021167090 (669974-737132): Sodiomyces alkalinus F11 unplaced genomic scaff... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_8.svg">
NC_007196 (322855-390985): Aspergillus fumigatus Af293 chromosome 3, whole ge... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_9.svg">
NW_025091224 (1326221-1421444): Aspergillus viridinutans IFM 47045 DNA, scaff... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_10.svg">
NW_020290905 (239903-308208): Aspergillus niger CBS 101883 unplaced genomic s... (10% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r8c2-download">Download graphic</button>
</div>
<div id="clusterblast-r8c2-svg"></div>

</div>
</div>

<div class="body-details-section r8c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-446"></div> <span class="help-tooltip" id="cb-known-help-446">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r8c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-447"></div> <span class="help-tooltip" id="cb-sub-help-447">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r8c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-450"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-450">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c2-domain-bubble-svg" data-filename="scaffold_8_r8c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r8c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (2125888 - 2201955): neighbouring T1PKS-fungal-RiPP-like
</option>



<option value="CC3">
Candidate 3 (2125888 - 2173531): single T1PKS
</option>


</select>

<div class="details-svg" id="r8c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r8c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r8c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-452"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-452">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c2-pfam-details-svg" data-filename="scaffold_8_r8c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r8c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r8c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-453"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-453">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c2-tigrfam-details-svg" data-filename="scaffold_8_r8c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r8c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r8c2-nrps_pks_products" data-name="r8c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r8c2-nrps_pks_monomers" data-name="r8c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r8c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-448"></div> <span class="help-tooltip" id="nrps-structures-help-448">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 2125888 - 2201955:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r8c2-2" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r8c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 2125888 - 2173531:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r8c2-3" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r8c2-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r8c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-449"></div> <span class="help-tooltip" id="nrps-monomers-help-449">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene634</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene634 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (572..900)</strong>: pk
<div class="collapser collapser-target-input-path1-gene634 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene634-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>2-Methylbutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene634-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 74.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 71.1</dt>

<dd></dd><dt>Malonyl-CoA: 51.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 49.3</dt>

<dd></dd><dt>Isobutyryl-CoA: 35.2</dt>

<dd></dd><dt>Propionyl-CoA: 33.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 23.5</dt>

<dd></dd><dt>fatty_acid: 12.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 11.3</dt>

<dd></dd><dt>inactive: 10.9</dt>

<dd></dd><dt>CHC-CoA: 10.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.1</dt>

<dd></dd><dt>Benzoyl-CoA: 6.2</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/fungi_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://fungismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/fungi_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>