
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>Cmilitaris_ATCC34164 - 40 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/fungi.css">
<meta property="og:title" content="Cmilitaris_ATCC34164 - 7 record(s) - 40 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/fungi_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://fungismash.secondarymetabolites.org/upload/fungi-a3179b02-62d0-4874-a715-63b60a935ed6/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://fungismash.secondarymetabolites.org/">
<img src="images/fungi_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://fungismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="Cmilitaris_ATCC34164.zip">Download all results</a></li>

<li><a href="Cmilitaris_ATCC34164.gbk">Download GenBank summary file</a></li>
<li><a href="Cmilitaris_ATCC34164.json">Download JSON results file</a></li>

<li><a href="fungi-a3179b02-62d0-4874-a715-63b60a935ed6.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton RiPP fungal-RiPP-like r1c1">
<a href="#r1c1">1.1</a>
</div>

<div class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">1.2</a>
</div>



<div class="regbutton PKS T1PKS r2c1">
<a href="#r2c1">2.1</a>
</div>

<div class="regbutton terpene terpene r2c2">
<a href="#r2c2">2.2</a>
</div>

<div class="regbutton PKS T1PKS r2c3">
<a href="#r2c3">2.3</a>
</div>



<div class="regbutton hybrid r3c1">
<a href="#r3c1">3.1</a>
</div>

<div class="regbutton NRPS isocyanide-nrp r3c2">
<a href="#r3c2">3.2</a>
</div>

<div class="regbutton NRPS NRPS r3c3">
<a href="#r3c3">3.3</a>
</div>

<div class="regbutton hybrid r3c4">
<a href="#r3c4">3.4</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r3c5">
<a href="#r3c5">3.5</a>
</div>



<div class="regbutton terpene terpene r4c1">
<a href="#r4c1">4.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">4.2</a>
</div>

<div class="regbutton NRPS NRPS-like r4c3">
<a href="#r4c3">4.3</a>
</div>

<div class="regbutton NRPS NRPS r4c4">
<a href="#r4c4">4.4</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r4c5">
<a href="#r4c5">4.5</a>
</div>



<div class="regbutton hybrid r5c1">
<a href="#r5c1">5.1</a>
</div>

<div class="regbutton other indole r5c2">
<a href="#r5c2">5.2</a>
</div>

<div class="regbutton other isocyanide r5c3">
<a href="#r5c3">5.3</a>
</div>

<div class="regbutton NRPS r5c4">
<a href="#r5c4">5.4</a>
</div>

<div class="regbutton PKS T1PKS r5c5">
<a href="#r5c5">5.5</a>
</div>

<div class="regbutton hybrid r5c6">
<a href="#r5c6">5.6</a>
</div>

<div class="regbutton hybrid r5c7">
<a href="#r5c7">5.7</a>
</div>



<div class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">6.1</a>
</div>

<div class="regbutton NRPS NRPS r6c2">
<a href="#r6c2">6.2</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r6c3">
<a href="#r6c3">6.3</a>
</div>

<div class="regbutton terpene terpene r6c4">
<a href="#r6c4">6.4</a>
</div>

<div class="regbutton NRPS NRPS-like r6c5">
<a href="#r6c5">6.5</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r6c6">
<a href="#r6c6">6.6</a>
</div>

<div class="regbutton NRPS NRPS r6c7">
<a href="#r6c7">6.7</a>
</div>

<div class="regbutton PKS T1PKS r6c8">
<a href="#r6c8">6.8</a>
</div>

<div class="regbutton PKS T1PKS r6c9">
<a href="#r6c9">6.9</a>
</div>



<div class="regbutton hybrid r7c1">
<a href="#r7c1">7.1</a>
</div>

<div class="regbutton terpene terpene r7c2">
<a href="#r7c2">7.2</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r7c3">
<a href="#r7c3">7.3</a>
</div>

<div class="regbutton NRPS NRPS r7c4">
<a href="#r7c4">7.4</a>
</div>

<div class="regbutton NRPS NRPS r7c5">
<a href="#r7c5">7.5</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r7c6">
<a href="#r7c6">7.6</a>
</div>

<div class="regbutton hybrid r7c7">
<a href="#r7c7">7.7</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r7c8">
<a href="#r7c8">7.8</a>
</div>

<div class="regbutton NRPS NAPAA r7c9">
<a href="#r7c9">7.9</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>CP023328.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="CP023328.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton RiPP fungal-RiPP-like r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">417,601</td>
<td class="digits table-split-left">478,581</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,614,824</td>
<td class="digits table-split-left">1,662,459</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>CP023327.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="CP023327.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton PKS T1PKS r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,078</td>
<td class="digits table-split-left">48,855</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001866/1" target="_blank">viriditoxin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 55%, #ffffff00 55%)">55%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton terpene terpene r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,326,010</td>
<td class="digits table-split-left">2,347,398</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton PKS T1PKS r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,417,368</td>
<td class="digits table-split-left">2,464,117</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002063/1" target="_blank">cryptosporioptide B/cryptosporioptide A/cryptosporioptide C</a></td>
<td>Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>CP023326.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="CP023326.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton hybrid r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">22,531</td>
<td class="digits table-split-left">75,306</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001136/1" target="_blank">desmethylbassianin</a></td>
<td>NRP+Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton NRPS isocyanide-nrp r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide-nrp" target="_blank">isocyanide-nrp</a>
</td>
<td class="digits">334,262</td>
<td class="digits table-split-left">391,696</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton NRPS NRPS r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,030,044</td>
<td class="digits table-split-left">1,074,412</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton hybrid r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,257,328</td>
<td class="digits table-split-left">1,346,792</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001278/1" target="_blank">nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton RiPP fungal-RiPP-like r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">2,834,759</td>
<td class="digits table-split-left">2,895,794</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>CP023322.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="CP023322.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton terpene terpene r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">189,498</td>
<td class="digits table-split-left">211,796</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001248/1" target="_blank">clavaric acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">615,487</td>
<td class="digits table-split-left">676,459</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c3">
<td class="regbutton NRPS NRPS-like r4c3">
<a href="#r4c3">Region&nbsp4.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">869,949</td>
<td class="digits table-split-left">913,940</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c4">
<td class="regbutton NRPS NRPS r4c4">
<a href="#r4c4">Region&nbsp4.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,586,874</td>
<td class="digits table-split-left">1,632,367</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c5">
<td class="regbutton RiPP fungal-RiPP-like r4c5">
<a href="#r4c5">Region&nbsp4.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,965,926</td>
<td class="digits table-split-left">5,026,565</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000313/1" target="_blank">beauvericin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 30%, #ffffff00 30%)">30%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>CP023325.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="CP023325.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton hybrid r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,181,633</td>
<td class="digits table-split-left">1,249,181</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton other indole r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">1,787,500</td>
<td class="digits table-split-left">1,809,048</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton other isocyanide r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide" target="_blank">isocyanide</a>
</td>
<td class="digits">2,963,228</td>
<td class="digits table-split-left">3,005,268</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c4">
<td class="regbutton NRPS r5c4">
<a href="#r5c4">Region&nbsp5.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,037,327</td>
<td class="digits table-split-left">3,097,372</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c5">
<td class="regbutton PKS T1PKS r5c5">
<a href="#r5c5">Region&nbsp5.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">3,688,839</td>
<td class="digits table-split-left">3,737,299</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c6">
<td class="regbutton hybrid r5c6">
<a href="#r5c6">Region&nbsp5.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,455,302</td>
<td class="digits table-split-left">4,524,605</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002259/1" target="_blank">beauveriolide I/beauveriolide III</a></td>
<td>Polyketide+NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c7">
<td class="regbutton hybrid r5c7">
<a href="#r5c7">Region&nbsp5.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,705,473</td>
<td class="digits table-split-left">5,766,444</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>CP023323.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="CP023323.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">822,578</td>
<td class="digits table-split-left">866,868</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton NRPS NRPS r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,344,291</td>
<td class="digits table-split-left">1,400,349</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c3">
<td class="regbutton RiPP fungal-RiPP-like r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,754,981</td>
<td class="digits table-split-left">1,816,068</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c4">
<td class="regbutton terpene terpene r6c4">
<a href="#r6c4">Region&nbsp6.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,135,448</td>
<td class="digits table-split-left">2,156,930</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000667/1" target="_blank">PR-toxin</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c5">
<td class="regbutton NRPS NRPS-like r6c5">
<a href="#r6c5">Region&nbsp6.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,271,609</td>
<td class="digits table-split-left">3,315,469</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c6">
<td class="regbutton RiPP fungal-RiPP-like r6c6">
<a href="#r6c6">Region&nbsp6.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,636,275</td>
<td class="digits table-split-left">3,711,474</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c7">
<td class="regbutton NRPS NRPS r6c7">
<a href="#r6c7">Region&nbsp6.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,090,598</td>
<td class="digits table-split-left">4,135,469</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c8">
<td class="regbutton PKS T1PKS r6c8">
<a href="#r6c8">Region&nbsp6.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,327,223</td>
<td class="digits table-split-left">4,374,354</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c9">
<td class="regbutton PKS T1PKS r6c9">
<a href="#r6c9">Region&nbsp6.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">5,790,965</td>
<td class="digits table-split-left">5,840,010</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>CP023324.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-7" data-filename="CP023324.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-7" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r7c1">
<td class="regbutton hybrid r7c1">
<a href="#r7c1">Region&nbsp7.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">190,427</td>
<td class="digits table-split-left">271,267</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c2">
<td class="regbutton terpene terpene r7c2">
<a href="#r7c2">Region&nbsp7.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1,515,240</td>
<td class="digits table-split-left">1,536,807</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c3">
<td class="regbutton RiPP fungal-RiPP-like r7c3">
<a href="#r7c3">Region&nbsp7.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,708,929</td>
<td class="digits table-split-left">3,769,829</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c4">
<td class="regbutton NRPS NRPS r7c4">
<a href="#r7c4">Region&nbsp7.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,298,434</td>
<td class="digits table-split-left">4,343,860</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002710/1" target="_blank">metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c5">
<td class="regbutton NRPS NRPS r7c5">
<a href="#r7c5">Region&nbsp7.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,726,125</td>
<td class="digits table-split-left">4,770,472</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c6">
<td class="regbutton RiPP fungal-RiPP-like r7c6">
<a href="#r7c6">Region&nbsp7.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">5,186,674</td>
<td class="digits table-split-left">5,247,751</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c7">
<td class="regbutton hybrid r7c7">
<a href="#r7c7">Region&nbsp7.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">6,136,346</td>
<td class="digits table-split-left">6,188,673</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001738/1" target="_blank">phomasetin</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>

</tr>
<tr class="linked-row even" data-anchor="#r7c8">
<td class="regbutton RiPP fungal-RiPP-like r7c8">
<a href="#r7c8">Region&nbsp7.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">6,225,089</td>
<td class="digits table-split-left">6,286,233</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c9">
<td class="regbutton NRPS NAPAA r7c9">
<a href="#r7c9">Region&nbsp7.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">6,983,456</td>
<td class="digits table-split-left">7,017,684</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">ε-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton RiPP fungal-RiPP-like r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">417,601</td>
<td class="digits table-split-left">478,581</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,614,824</td>
<td class="digits table-split-left">1,662,459</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton PKS T1PKS r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,078</td>
<td class="digits table-split-left">48,855</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001866/1" target="_blank">viriditoxin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 55%, #ffffff00 55%)">55%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton terpene terpene r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,326,010</td>
<td class="digits table-split-left">2,347,398</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton PKS T1PKS r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">2,417,368</td>
<td class="digits table-split-left">2,464,117</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002063/1" target="_blank">cryptosporioptide B/cryptosporioptide A/cryptosporioptide C</a></td>
<td>Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>

</tr>


<tr class="linked-row even" data-anchor="#r3c1">
<td class="regbutton hybrid r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">22,531</td>
<td class="digits table-split-left">75,306</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001136/1" target="_blank">desmethylbassianin</a></td>
<td>NRP+Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c2">
<td class="regbutton NRPS isocyanide-nrp r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide-nrp" target="_blank">isocyanide-nrp</a>
</td>
<td class="digits">334,262</td>
<td class="digits table-split-left">391,696</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c3">
<td class="regbutton NRPS NRPS r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,030,044</td>
<td class="digits table-split-left">1,074,412</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c4">
<td class="regbutton hybrid r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,257,328</td>
<td class="digits table-split-left">1,346,792</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001278/1" target="_blank">nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>

</tr>
<tr class="linked-row even" data-anchor="#r3c5">
<td class="regbutton RiPP fungal-RiPP-like r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">2,834,759</td>
<td class="digits table-split-left">2,895,794</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton terpene terpene r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">189,498</td>
<td class="digits table-split-left">211,796</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001248/1" target="_blank">clavaric acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton RiPP fungal-RiPP-like r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">615,487</td>
<td class="digits table-split-left">676,459</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c3">
<td class="regbutton NRPS NRPS-like r4c3">
<a href="#r4c3">Region&nbsp4.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">869,949</td>
<td class="digits table-split-left">913,940</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c4">
<td class="regbutton NRPS NRPS r4c4">
<a href="#r4c4">Region&nbsp4.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,586,874</td>
<td class="digits table-split-left">1,632,367</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c5">
<td class="regbutton RiPP fungal-RiPP-like r4c5">
<a href="#r4c5">Region&nbsp4.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">4,965,926</td>
<td class="digits table-split-left">5,026,565</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000313/1" target="_blank">beauvericin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 30%, #ffffff00 30%)">30%</td>

</tr>


<tr class="linked-row even" data-anchor="#r5c1">
<td class="regbutton hybrid r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,181,633</td>
<td class="digits table-split-left">1,249,181</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c2">
<td class="regbutton other indole r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">1,787,500</td>
<td class="digits table-split-left">1,809,048</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c3">
<td class="regbutton other isocyanide r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#isocyanide" target="_blank">isocyanide</a>
</td>
<td class="digits">2,963,228</td>
<td class="digits table-split-left">3,005,268</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c4">
<td class="regbutton NRPS r5c4">
<a href="#r5c4">Region&nbsp5.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">3,037,327</td>
<td class="digits table-split-left">3,097,372</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c5">
<td class="regbutton PKS T1PKS r5c5">
<a href="#r5c5">Region&nbsp5.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">3,688,839</td>
<td class="digits table-split-left">3,737,299</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c6">
<td class="regbutton hybrid r5c6">
<a href="#r5c6">Region&nbsp5.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,455,302</td>
<td class="digits table-split-left">4,524,605</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002259/1" target="_blank">beauveriolide I/beauveriolide III</a></td>
<td>Polyketide+NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r5c7">
<td class="regbutton hybrid r5c7">
<a href="#r5c7">Region&nbsp5.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,705,473</td>
<td class="digits table-split-left">5,766,444</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">822,578</td>
<td class="digits table-split-left">866,868</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton NRPS NRPS r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,344,291</td>
<td class="digits table-split-left">1,400,349</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c3">
<td class="regbutton RiPP fungal-RiPP-like r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,754,981</td>
<td class="digits table-split-left">1,816,068</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c4">
<td class="regbutton terpene terpene r6c4">
<a href="#r6c4">Region&nbsp6.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">2,135,448</td>
<td class="digits table-split-left">2,156,930</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000667/1" target="_blank">PR-toxin</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c5">
<td class="regbutton NRPS NRPS-like r6c5">
<a href="#r6c5">Region&nbsp6.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">3,271,609</td>
<td class="digits table-split-left">3,315,469</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c6">
<td class="regbutton RiPP fungal-RiPP-like r6c6">
<a href="#r6c6">Region&nbsp6.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,636,275</td>
<td class="digits table-split-left">3,711,474</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c7">
<td class="regbutton NRPS NRPS r6c7">
<a href="#r6c7">Region&nbsp6.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,090,598</td>
<td class="digits table-split-left">4,135,469</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c8">
<td class="regbutton PKS T1PKS r6c8">
<a href="#r6c8">Region&nbsp6.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,327,223</td>
<td class="digits table-split-left">4,374,354</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c9">
<td class="regbutton PKS T1PKS r6c9">
<a href="#r6c9">Region&nbsp6.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">5,790,965</td>
<td class="digits table-split-left">5,840,010</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r7c1">
<td class="regbutton hybrid r7c1">
<a href="#r7c1">Region&nbsp7.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">190,427</td>
<td class="digits table-split-left">271,267</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c2">
<td class="regbutton terpene terpene r7c2">
<a href="#r7c2">Region&nbsp7.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1,515,240</td>
<td class="digits table-split-left">1,536,807</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row even" data-anchor="#r7c3">
<td class="regbutton RiPP fungal-RiPP-like r7c3">
<a href="#r7c3">Region&nbsp7.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">3,708,929</td>
<td class="digits table-split-left">3,769,829</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c4">
<td class="regbutton NRPS NRPS r7c4">
<a href="#r7c4">Region&nbsp7.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,298,434</td>
<td class="digits table-split-left">4,343,860</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002710/1" target="_blank">metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>

</tr>
<tr class="linked-row even" data-anchor="#r7c5">
<td class="regbutton NRPS NRPS r7c5">
<a href="#r7c5">Region&nbsp7.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">4,726,125</td>
<td class="digits table-split-left">4,770,472</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c6">
<td class="regbutton RiPP fungal-RiPP-like r7c6">
<a href="#r7c6">Region&nbsp7.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">5,186,674</td>
<td class="digits table-split-left">5,247,751</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c7">
<td class="regbutton hybrid r7c7">
<a href="#r7c7">Region&nbsp7.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">6,136,346</td>
<td class="digits table-split-left">6,188,673</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001738/1" target="_blank">phomasetin</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r7c8">
<td class="regbutton RiPP fungal-RiPP-like r7c8">
<a href="#r7c8">Region&nbsp7.8</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">6,225,089</td>
<td class="digits table-split-left">6,286,233</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r7c9">
<td class="regbutton NRPS NAPAA r7c9">
<a href="#r7c9">Region&nbsp7.9</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">6,983,456</td>
<td class="digits table-split-left">7,017,684</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">ε-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


</tbody>
</table>
</div>
</div>


</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023328.1 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-356"></div> <span class="help-tooltip" id="region_svg-help-356">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023328.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="CP023328.1_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 417,601 - 478,581 nt. (total: 60,981 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-357"></div> <span class="help-tooltip" id="focus-panel-help-357">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c1-tigrfam-details" data-name="r1c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-5"></div> <span class="help-tooltip" id="gt-help-tooltip-help-5">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.43, order:0.27, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001402: 590-34242">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001402/index.html#r1c1" target="_new">BGC0001402</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP</td>

<td>cyclochlorotine, hydroxycyclochlorotine A, cyclotine, cyclochlorotine B, deoxy-Cyclochlorotine</td>


<td>Talaromyces islandicus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.43, order:0.27, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001402: 590-34242">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001402/index.html#r1c1" target="_new">BGC0001402</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP</td>
<td>cyclochlorotine, hydroxycyclochlorotine A, cyclotine, cyclochlorotine B, deoxy-Cyclochlorotine</td>
<td>Talaromyces islandicus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c1_1.svg">
NW_006271974 (287596-378578): Cordyceps militaris CM01 unplaced genomic scaff... (52% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_2.svg">
NW_007360990 (964758-1055299): Glarea lozoyensis ATCC 20868 chromosome Unknow... (22% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r1c1_3.svg">
NW_011942160 (5753348-5844373): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_4.svg">
NC_035794 (2151549-2242577): Pochonia chlamydosporia 170 chromosome 5, whole ... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_5.svg">
NW_017971443 (180850-345227): Talaromyces atroroseus strain IBT 11181 chromos... (3% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_6.svg">
NW_025091268 (382059-493390): Aspergillus udagawae IFM 46973 DNA, scaffold 5,... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_7.svg">
NC_007197 (3596722-3687366): Aspergillus fumigatus Af293 chromosome 4, whole ... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_8.svg">
NW_022985511 (1077816-1171333): Neohortaea acidophila strain CBS 113389 unpla... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_9.svg">
NW_024468909 (2246932-2338289): Cryphonectria parasitica EP155 unplaced genom... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r1c1_10.svg">
NC_072349 (1077128-1157364): Talaromyces marneffei chromosome 2, complete seq... (6% of genes show similarity), isocyanide-nrp
</option>

</select>
<button id= "clusterblast-r1c1-download">Download graphic</button>
</div>
<div id="clusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-6"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-6">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="CP023328.1_r1c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-7"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-7">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-tigrfam-details-svg" data-filename="CP023328.1_r1c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r1c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023328.1 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-358"></div> <span class="help-tooltip" id="region_svg-help-358">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023328.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c2-svg" data-filename="CP023328.1_r1c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,614,824 - 1,662,459 nt. (total: 47,636 nt)

<a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r1c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-359"></div> <span class="help-tooltip" id="focus-panel-help-359">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c2-gene-table-details" data-name="r1c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c2-nrps_pks" data-name="r1c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-14"></div> <span class="help-tooltip" id="gt-help-tooltip-help-14">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-8"></div> <span class="help-tooltip" id="nrps-domain-help-8">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-details-svg" data-filename="CP023328.1_r1c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-9"></div> <span class="help-tooltip" id="MIBiG-help-9">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c2"></div>
<div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.47, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.47, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>
<td>Aspergillus niger</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-10"></div> <span class="help-tooltip" id="cb-general-help-10">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c2_1.svg">
NW_006271975 (2582207-2649723): Cordyceps militaris CM01 unplaced genomic sca... (58% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_2.svg">
NZ_CP042266 (6917394-7296053): Streptomyces qinzhouensis strain SSL-25 chromo... (9% of genes show similarity), NRP-metallophore,NRPS,NRPS-like,T1PKS,other,terpene
</option>

<option value = "svg/clusterblast_r1c2_3.svg">
NZ_CP031194 (8221320-8396998): Streptomyces paludis strain GSSD-12 chromosome... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c2_4.svg">
NZ_CP073778 (1381856-1573614): Streptomyces lavendulae subsp. lavendulae stra... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_5.svg">
NZ_CP058693 (110837-483494): Streptomyces sp. Rer75 chromosome (4% of genes show similarity), T1PKS,butyrolactone,other,pyrrolidine,terpene
</option>

<option value = "svg/clusterblast_r1c2_6.svg">
NZ_CP033581 (898034-1174147): Streptomyces sp. ADI95-16 chromosome, complete ... (6% of genes show similarity), LAP,NRP-metallophore,NRPS,RRE-containing,T1PKS,thiopeptide
</option>

<option value = "svg/clusterblast_r1c2_7.svg">
NZ_CP071139 (7832793-8024104): Streptomyces nojiriensis strain JCM 3382 chrom... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_8.svg">
NZ_CP115936 (711670-902918): Streptomyces sp. CGMCC 4.1796 chromosome, comple... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_9.svg">
NZ_CP011664 (1057301-1248574): Streptomyces sp. Mg1 chromosome 1, complete se... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c2_10.svg">
NZ_JAFLQY010000003 (105434-296683): Streptomyces sp. KCTC 0041BP Streptomyces... (14% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r1c2-download">Download graphic</button>
</div>
<div id="clusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-11"></div> <span class="help-tooltip" id="cb-known-help-11">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-12"></div> <span class="help-tooltip" id="cb-sub-help-12">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-15"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-15">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="CP023328.1_r1c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c2-nrps_pks_monomers" data-name="r1c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r1c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-13"></div> <span class="help-tooltip" id="nrps-monomers-help-13">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene406</span></strong>: X
<div class="collapser collapser-target-input-path1-gene406 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (83..490)</strong>: X
<div class="collapser collapser-target-input-path1-gene406 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe, Trp, Ph-Gly, Tyr, bOH-Tyr
<div class="collapser collapser-target-input-path1-gene406-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aromatic side chain (Phe, Trp, Ph-Gly, Tyr, bOH-Tyr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">TLGGVTMGVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene408</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene408 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (557..876)</strong>: pk
<div class="collapser collapser-target-input-path1-gene408 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene408-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Methoxymalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene408-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 73.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 56.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 49.6</dt>

<dd></dd><dt>Malonyl-CoA: 38.9</dt>

<dd></dd><dt>Propionyl-CoA: 32.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 28.5</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 20.2</dt>

<dd></dd><dt>fatty_acid: 20.1</dt>

<dd></dd><dt>inactive: 18.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 16.7</dt>

<dd></dd><dt>Benzoyl-CoA: 15.9</dt>

<dd></dd><dt>CHC-CoA: 15.1</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023327.1 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-360"></div> <span class="help-tooltip" id="region_svg-help-360">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023327.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="CP023327.1_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,078 - 48,855 nt. (total: 47,778 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-361"></div> <span class="help-tooltip" id="focus-panel-help-361">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-nrps_pks" data-name="r2c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c1-nrps-pks-bubbles" data-name="r2c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-24"></div> <span class="help-tooltip" id="gt-help-tooltip-help-24">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-16"></div> <span class="help-tooltip" id="nrps-domain-help-16">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-details-svg" data-filename="CP023327.1_r2c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-17"></div> <span class="help-tooltip" id="MIBiG-help-17">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene, Polyketide</td>

<td>subglutinol A, subglutinol B</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>eupenifeldin</td>


<td>Phoma sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001827: 0-43539">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001827/index.html#r1c1" target="_new">BGC0001827</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>xenovulene A</td>


<td>Sarocladium sp. &#39;schorii&#39;</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001446: 0-15813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001446/index.html#r1c1" target="_new">BGC0001446</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>asparasone A</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.46, order:0.57, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, Polyketide</td>
<td>subglutinol A, subglutinol B</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>eupenifeldin</td>
<td>Phoma sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001827: 0-43539">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001827/index.html#r1c1" target="_new">BGC0001827</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>xenovulene A</td>
<td>Sarocladium sp. &#39;schorii&#39;</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001446: 0-15813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001446/index.html#r1c1" target="_new">BGC0001446</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>asparasone A</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.46, order:0.57, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-18"></div> <span class="help-tooltip" id="cb-general-help-18">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NW_006271969 (6181520-6248454): Cordyceps militaris CM01 unplaced genomic sca... (62% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NW_025091218 (4642-192860): Aspergillus viridinutans IFM 47045 DNA, scaffold ... (15% of genes show similarity), NRPS-like,T1PKS,indole
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NW_025091283 (4319746-4671991): Aspergillus pseudoviridinutans IFM 55266 DNA,... (9% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NW_025091267 (57500-124807): Aspergillus udagawae IFM 46973 DNA, scaffold 4, ... (27% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NC_049562 (5050410-5117184): Talaromyces rugulosus chromosome II, complete se... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NC_030957 (3095408-3162021): Colletotrichum higginsianum IMI 349063 chromosom... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NC_064681 (3222189-3288600): Colletotrichum lupini chromosome 8, complete seq... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NC_063019 (3034218-3101038): Fulvia fulva chromosome 8, complete sequence (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NW_011942151 (20096-254382): Metarhizium robertsii ARSEF 23 MAA Scf 11, whole... (4% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NW_017264193 (1450064-1517296): Xylona heveae TC161 unplaced genomic scaffold... (26% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-19"></div> <span class="help-tooltip" id="cb-known-help-19">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c1_1.svg">
viriditoxin
</option>

</select>
<button id="knownclusterblast-r2c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-20"></div> <span class="help-tooltip" id="cb-sub-help-20">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-23"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-23">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-domain-bubble-svg" data-filename="CP023327.1_r2c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (1077 - 48855): single T1PKS
</option>


</select>

<div class="details-svg" id="r2c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-25"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-25">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="CP023327.1_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-26"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-26">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="CP023327.1_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c1-nrps_pks_products" data-name="r2c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c1-nrps_pks_monomers" data-name="r2c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-21"></div> <span class="help-tooltip" id="nrps-structures-help-21">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 1077 - 48855:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-22"></div> <span class="help-tooltip" id="nrps-monomers-help-22">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene6</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene6 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (778..1076)</strong>: mal
<div class="collapser collapser-target-input-path1-gene6 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene6-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>inactive: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene6-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 96.5</dt>

<dd></dd><dt>Methylmalonyl-CoA: 77.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 76.8</dt>

<dd></dd><dt>inactive: 54.6</dt>

<dd></dd><dt>Propionyl-CoA: 44.5</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 33.8</dt>

<dd></dd><dt>CHC-CoA: 33.4</dt>

<dd></dd><dt>Acetyl-CoA: 26.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 25.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 22.6</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 19.1</dt>

<dd></dd><dt>Benzoyl-CoA: 18.3</dt>

<dd></dd><dt>fatty_acid: 13.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 12.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r2c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023327.1 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-362"></div> <span class="help-tooltip" id="region_svg-help-362">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023327.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c2-svg" data-filename="CP023327.1_r2c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,326,010 - 2,347,398 nt. (total: 21,389 nt)

<a class="cluster-rules-header" id="r2c2-rules-header" href="#r2c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-363"></div> <span class="help-tooltip" id="focus-panel-help-363">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c2-gene-table-details" data-name="r2c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c2-MIBiG-cluster-compare" data-name="r2c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c2-clusterblast" data-name="r2c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c2-knownclusterblast" data-name="r2c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c2-subclusterblast" data-name="r2c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c2-pfam-details" data-name="r2c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-31"></div> <span class="help-tooltip" id="gt-help-tooltip-help-31">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-27"></div> <span class="help-tooltip" id="MIBiG-help-27">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c2"></div>
<div id="comparison-MIBiG-r2c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.52, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.52, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>pimara-8(14),15-diene</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>preaspterpenacid I</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.51, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>


<td>Penicillium paxilli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.51, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Terpene, Polyketide</td>

<td>subglutinol A, subglutinol B</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.52, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.52, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16β-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Terpene</td>
<td>pimara-8(14),15-diene</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>preaspterpenacid I</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.51, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Terpene</td>
<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>
<td>Penicillium paxilli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.51, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Terpene, Polyketide</td>
<td>subglutinol A, subglutinol B</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-28"></div> <span class="help-tooltip" id="cb-general-help-28">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c2_1.svg">
NW_006271971 (242847-274242): Cordyceps militaris CM01 unplaced genomic scaff... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_2.svg">
NC_063073 (1136454-1167807): Purpureocillium takamizusanense chromosome 6, co... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_3.svg">
NC_070533 (2808830-2840090): Fusarium keratoplasticum isolate Fu6.1 chromosom... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_4.svg">
NC_036626 (1833202-1864459): Fusarium fujikuroi IMI 58289 draft genome, chrom... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_5.svg">
NC_072844 (1849949-1881206): Fusarium oxysporum Fo47 chromosome V, complete s... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_6.svg">
NC_058399 (9867240-9898491): Fusarium poae strain DAOMC 252244 chromosome 1, ... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_7.svg">
NC_026474 (9777926-9809879): Fusarium graminearum PH-1 chromosome 1, whole ge... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_8.svg">
NC_035796 (1863677-1895105): Pochonia chlamydosporia 170 chromosome 7, whole ... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_9.svg">
NC_058390 (2633992-2665249): Fusarium musae strain F31 chromosome 4, whole ge... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c2_10.svg">
NW_011942182 (2894477-2966843): Metarhizium robertsii ARSEF 23 MAA Scf 4, who... (13% of genes show similarity), NRPS-like,terpene
</option>

</select>
<button id= "clusterblast-r2c2-download">Download graphic</button>
</div>
<div id="clusterblast-r2c2-svg"></div>

</div>
</div>

<div class="body-details-section r2c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-29"></div> <span class="help-tooltip" id="cb-known-help-29">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-30"></div> <span class="help-tooltip" id="cb-sub-help-30">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-32"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-32">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-pfam-details-svg" data-filename="CP023327.1_r2c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r2c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023327.1 - Region 3 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-364"></div> <span class="help-tooltip" id="region_svg-help-364">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023327.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c3-svg" data-filename="CP023327.1_r2c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,417,368 - 2,464,117 nt. (total: 46,750 nt)

<a class="cluster-rules-header" id="r2c3-rules-header" href="#r2c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-365"></div> <span class="help-tooltip" id="focus-panel-help-365">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c3-gene-table-details" data-name="r2c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c3-nrps_pks" data-name="r2c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c3-MIBiG-cluster-compare" data-name="r2c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c3-clusterblast" data-name="r2c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c3-knownclusterblast" data-name="r2c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c3-subclusterblast" data-name="r2c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c3-nrps-pks-bubbles" data-name="r2c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c3-pfam-details" data-name="r2c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c3-tigrfam-details" data-name="r2c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-41"></div> <span class="help-tooltip" id="gt-help-tooltip-help-41">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-33"></div> <span class="help-tooltip" id="nrps-domain-help-33">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-details-svg" data-filename="CP023327.1_r2c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-34"></div> <span class="help-tooltip" id="MIBiG-help-34">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c3"></div>
<div id="comparison-MIBiG-r2c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Nodulisporium sp. ATCC74245</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Polyketide</td>

<td>emodin</td>


<td>Escovopsis weberi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>grayanic acid</td>


<td>Cladonia grayi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001865: 0-36183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001865/index.html#r1c1" target="_new">BGC0001865</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>elsinochrome C</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Nodulisporium sp. ATCC74245</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>emodin</td>
<td>Escovopsis weberi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>grayanic acid</td>
<td>Cladonia grayi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001865: 0-36183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001865/index.html#r1c1" target="_new">BGC0001865</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>elsinochrome C</td>
<td>Parastagonospora nodorum SN15</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-35"></div> <span class="help-tooltip" id="cb-general-help-35">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c3_1.svg">
NW_006271971 (91199-157949): Cordyceps militaris CM01 unplaced genomic scaffo... (42% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c3_2.svg">
NW_017263595 (627002-693843): Mollisia scopiformis unplaced genomic scaffold ... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c3_3.svg">
NW_025408670 (3613020-3679633): Hirsutella rhossiliensis strain HR02 chromoso... (30% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r2c3_4.svg">
NW_011942182 (3703888-3770743): Metarhizium robertsii ARSEF 23 MAA Scf 4, who... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c3_5.svg">
NW_025408669 (7752252-7819709): Hirsutella rhossiliensis strain HR02 chromoso... (17% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c3_6.svg">
NW_007360988 (416544-482798): Glarea lozoyensis ATCC 20868 chromosome Unknown... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c3_7.svg">
NW_022985010 (1623664-1725341): Dothidotthia symphoricarpi CBS 119687 unplace... (12% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r2c3_8.svg">
NW_017971432 (1168267-1234841): Talaromyces atroroseus strain IBT 11181 chrom... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c3_9.svg">
NC_049564 (4920448-4987078): Talaromyces rugulosus chromosome IV, complete se... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r2c3_10.svg">
NC_072350 (1386622-1453217): Talaromyces marneffei chromosome 3, complete seq... (14% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r2c3-download">Download graphic</button>
</div>
<div id="clusterblast-r2c3-svg"></div>

</div>
</div>

<div class="body-details-section r2c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-36"></div> <span class="help-tooltip" id="cb-known-help-36">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c3_1.svg">
cryptosporioptide B/cryptosporioptide A/cryptosporioptide C
</option>

<option value = "svg/knownclusterblast_r2c3_2.svg">
neosartorin
</option>

<option value = "svg/knownclusterblast_r2c3_3.svg">
agnestin A/agnestin B
</option>

<option value = "svg/knownclusterblast_r2c3_4.svg">
4-chloropinselin/pinselin/chloromonilinic acid B/chloromonilinic acid D/4-hydroxyvertixanthone
</option>

</select>
<button id="knownclusterblast-r2c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c3-svg"></div>

</div>
</div>

<div class="body-details-section r2c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-37"></div> <span class="help-tooltip" id="cb-sub-help-37">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-40"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-40">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-domain-bubble-svg" data-filename="CP023327.1_r2c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (2417367 - 2464117): single T1PKS
</option>


</select>

<div class="details-svg" id="r2c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-42"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-42">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-pfam-details-svg" data-filename="CP023327.1_r2c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-43"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-43">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-tigrfam-details-svg" data-filename="CP023327.1_r2c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c3-nrps_pks_products" data-name="r2c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c3-nrps_pks_monomers" data-name="r2c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-38"></div> <span class="help-tooltip" id="nrps-structures-help-38">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 2417367 - 2464117:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c3-3" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-39"></div> <span class="help-tooltip" id="nrps-monomers-help-39">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene633</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene633 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (917..1213)</strong>: mal
<div class="collapser collapser-target-input-path1-gene633 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene633-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene633-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 91.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 51.3</dt>

<dd></dd><dt>inactive: 48.2</dt>

<dd></dd><dt>Methylmalonyl-CoA: 37.7</dt>

<dd></dd><dt>Propionyl-CoA: 23.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 20.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 15.6</dt>

<dd></dd><dt>Benzoyl-CoA: 14.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 9.4</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.2</dt>

<dd></dd><dt>CHC-CoA: 8.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023326.1 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-366"></div> <span class="help-tooltip" id="region_svg-help-366">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023326.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="CP023326.1_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 22,531 - 75,306 nt. (total: 52,776 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-367"></div> <span class="help-tooltip" id="focus-panel-help-367">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-nrps_pks" data-name="r3c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c1-nrps-pks-bubbles" data-name="r3c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c1-tigrfam-details" data-name="r3c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-52"></div> <span class="help-tooltip" id="gt-help-tooltip-help-52">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-44"></div> <span class="help-tooltip" id="nrps-domain-help-44">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-details-svg" data-filename="CP023326.1_r3c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-45"></div> <span class="help-tooltip" id="MIBiG-help-45">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.71</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001049: 0-19176">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001049/index.html#r1c1" target="_new">BGC0001049</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.59, order:0.57, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.59, order:0.57, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.69</td>
<td>NRP, Polyketide</td>

<td>tenellin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001136: 0-20147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001136/index.html#r1c1" target="_new">BGC0001136</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.59, order:0.51, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.59, order:0.51, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>NRP, Polyketide</td>

<td>desmethylbassianin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>Polyketide, NRP</td>

<td>xenoacremone A</td>


<td>Xenoacremonium sp. BF-2018a</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001049: 0-19176">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001049/index.html#r1c1" target="_new">BGC0001049</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.59, order:0.57, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP, Polyketide</td>
<td>tenellin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001136: 0-20147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001136/index.html#r1c1" target="_new">BGC0001136</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.59, order:0.51, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Polyketide</td>
<td>desmethylbassianin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide, NRP</td>
<td>xenoacremone A</td>
<td>Xenoacremonium sp. BF-2018a</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-46"></div> <span class="help-tooltip" id="cb-general-help-46">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c1_1.svg">
NW_006271975 (27980-100581): Cordyceps militaris CM01 unplaced genomic scaffo... (72% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c1_2.svg">
NW_026250021 (2041365-2122437): Hypoxylon trugodes strain CBS 135444 unplaced... (11% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r3c1_3.svg">
NW_025091288 (77418-218033): Aspergillus pseudoviridinutans IFM 55266 DNA, sc... (10% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r3c1_4.svg">
NW_003052497 (1083806-1155882): Uncinocarpus reesii 1704 scaffold 4 genomic s... (12% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c1_5.svg">
NW_017263610 (256865-328702): Mollisia scopiformis unplaced genomic scaffold ... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c1_6.svg">
NC_054696 (3134268-3301036): Aspergillus flavus NRRL3357 chromosome 6, comple... (10% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r3c1_7.svg">
NC_036436 (608747-764003): Aspergillus oryzae RIB40 DNA, chromosome 2 (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c1_8.svg">
NC_018218 (5508545-5577675): Zymoseptoria tritici IPO323 chromosome 1, whole ... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c1_9.svg">
NW_014013670 (106490-178646): Trichoderma virens Gv29-8 chromosome Unknown TR... (18% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c1_10.svg">
NC_038014 (6525771-6597702): Fusarium venenatum strain A3/5 genome assembly, ... (10% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r3c1-download">Download graphic</button>
</div>
<div id="clusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-47"></div> <span class="help-tooltip" id="cb-known-help-47">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c1_1.svg">
desmethylbassianin
</option>

<option value = "svg/knownclusterblast_r3c1_2.svg">
tenellin
</option>

<option value = "svg/knownclusterblast_r3c1_3.svg">
fumosorinone
</option>

<option value = "svg/knownclusterblast_r3c1_4.svg">
ilicicolin H
</option>

<option value = "svg/knownclusterblast_r3c1_5.svg">
ilicicolin H/ilicicolin J/8-epi-ilicicolin H
</option>

</select>
<button id="knownclusterblast-r3c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-48"></div> <span class="help-tooltip" id="cb-sub-help-48">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-51"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-51">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-domain-bubble-svg" data-filename="CP023326.1_r3c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (22530 - 75306): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r3c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-53"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-53">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="CP023326.1_r3c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-54"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-54">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-tigrfam-details-svg" data-filename="CP023326.1_r3c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c1-nrps_pks_products" data-name="r3c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c1-nrps_pks_monomers" data-name="r3c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-49"></div> <span class="help-tooltip" id="nrps-structures-help-49">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 22530 - 75306:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC(Cc1ccc(O)cc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk - Tyr)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk,tyr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*,tyr*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC(Cc1ccc(O)cc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-50"></div> <span class="help-tooltip" id="nrps-monomers-help-50">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene16</span></strong>: pk - Tyr
<div class="collapser collapser-target-input-path1-gene16 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>PKS_AT (458..779)</strong>: pk
<div class="collapser collapser-target-input-path1-gene16 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene16-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>2-Methylbutyryl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene16-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 101.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 80.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 66.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 55.4</dt>

<dd></dd><dt>Benzoyl-CoA: 51.3</dt>

<dd></dd><dt>Propionyl-CoA: 39.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 35.6</dt>

<dd></dd><dt>Malonyl-CoA: 32.1</dt>

<dd></dd><dt>Acetyl-CoA: 28.8</dt>

<dd></dd><dt>trans-1,2-CPDA: 23.1</dt>

<dd></dd><dt>CHC-CoA: 22.4</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 22.0</dt>

<dd></dd><dt>fatty_acid: 14.5</dt>

<dd></dd><dt>inactive: 10.5</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>













<dd><strong>AMP-binding (3043..3458)</strong>: Tyr
<div class="collapser collapser-target-input-path1-gene16 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Tyr
<div class="collapser collapser-target-input-path1-gene16-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMVICGCAAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DMVICGCAAK</span> (97% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023326.1 - Region 2 - isocyanide-nrp <div class="help-container"> <div class="help-icon" data-id="region_svg-help-368"></div> <span class="help-tooltip" id="region_svg-help-368">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023326.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c2-svg" data-filename="CP023326.1_r3c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 334,262 - 391,696 nt. (total: 57,435 nt)

<a class="cluster-rules-header" id="r3c2-rules-header" href="#r3c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c2-rules">isocyanide-nrp: (DIT1_PvcA and minimum(2, [AMP-binding, PP-binding, Transferase]))</div>

<div class="region-svg-container">
<div id='r3c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-369"></div> <span class="help-tooltip" id="focus-panel-help-369">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c2-gene-table-details" data-name="r3c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c2-nrps_pks" data-name="r3c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c2-MIBiG-cluster-compare" data-name="r3c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c2-clusterblast" data-name="r3c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c2-knownclusterblast" data-name="r3c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c2-subclusterblast" data-name="r3c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c2-nrps-pks-bubbles" data-name="r3c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c2-pfam-details" data-name="r3c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-63"></div> <span class="help-tooltip" id="gt-help-tooltip-help-63">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-55"></div> <span class="help-tooltip" id="nrps-domain-help-55">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-details-svg" data-filename="CP023326.1_r3c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-56"></div> <span class="help-tooltip" id="MIBiG-help-56">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c2"></div>
<div id="comparison-MIBiG-r3c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">isocyanide-nrp</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>citreohybridonol</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.43, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>yanuthone D</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001508: 25-29346">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001508/index.html#r1c1" target="_new">BGC0001508</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>andrastin A</td>


<td>Penicillium roqueforti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002615: 973-23784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002615/index.html#r1c1" target="_new">BGC0002615</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Terpene</td>

<td>4-oxomacrophorin A, macrophorin A, 5&#39;-epimacrophorin B</td>


<td>Penicillium terrestre</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002612: 0-30932">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002612/index.html#r1c1" target="_new">BGC0002612</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>anditomin</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002537: 100-26716">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002537/index.html#r1c1" target="_new">BGC0002537</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide, Terpene</td>

<td>emeridone F</td>


<td>Aspergillus sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001436: 25-56534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001436/index.html#r1c1" target="_new">BGC0001436</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.42, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>cornexistin</td>


<td>Paecilomyces divaricatus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Terpene</td>
<td>citreohybridonol</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.43, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>yanuthone D</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001508: 25-29346">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001508/index.html#r1c1" target="_new">BGC0001508</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>andrastin A</td>
<td>Penicillium roqueforti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002615: 973-23784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002615/index.html#r1c1" target="_new">BGC0002615</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Terpene</td>
<td>4-oxomacrophorin A, macrophorin A, 5&#39;-epimacrophorin B</td>
<td>Penicillium terrestre</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002612: 0-30932">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002612/index.html#r1c1" target="_new">BGC0002612</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Terpene</td>
<td>anditomin</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002537: 100-26716">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002537/index.html#r1c1" target="_new">BGC0002537</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide, Terpene</td>
<td>emeridone F</td>
<td>Aspergillus sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001436: 25-56534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001436/index.html#r1c1" target="_new">BGC0001436</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.42, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Polyketide</td>
<td>cornexistin</td>
<td>Paecilomyces divaricatus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-57"></div> <span class="help-tooltip" id="cb-general-help-57">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c2_1.svg">
NW_006271975 (307084-386893): Cordyceps militaris CM01 unplaced genomic scaff... (48% of genes show similarity), isocyanide-nrp
</option>

<option value = "svg/clusterblast_r3c2_2.svg">
NW_024467533 (148342-258756): Aspergillus versicolor CBS 583.65 unplaced geno... (4% of genes show similarity), NRPS,NRPS-like,betalactone
</option>

<option value = "svg/clusterblast_r3c2_3.svg">
NC_054860 (4191964-4296027): Aspergillus puulaauensis MK2 DNA, chromosome 4, ... (4% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r3c2_4.svg">
NZ_CADEWY010000008 (83164-125000): Burkholderia gladioli strain BCC1678, whol... (5% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c2_5.svg">
NZ_CADEXU010000005 (317718-359554): Burkholderia gladioli strain BCC1709, who... (5% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c2_6.svg">
NZ_CADEWI010000007 (18512-60348): Burkholderia gladioli strain BCC1649, whole... (5% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c2_7.svg">
NZ_CADEWM010000008 (294023-335859): Burkholderia gladioli strain BCC1687, who... (5% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c2_8.svg">
NZ_CADEWR010000004 (309100-350936): Burkholderia gladioli strain BCC1669, who... (5% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c2_9.svg">
NZ_CADEWW010000007 (385875-427711): Burkholderia gladioli strain BCC1668, who... (5% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r3c2_10.svg">
NZ_CADEXC010000021 (99760-141596): Burkholderia gladioli strain BCC1688, whol... (5% of genes show similarity), isocyanide
</option>

</select>
<button id= "clusterblast-r3c2-download">Download graphic</button>
</div>
<div id="clusterblast-r3c2-svg"></div>

</div>
</div>

<div class="body-details-section r3c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-58"></div> <span class="help-tooltip" id="cb-known-help-58">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-59"></div> <span class="help-tooltip" id="cb-sub-help-59">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-62"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-62">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-domain-bubble-svg" data-filename="CP023326.1_r3c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (334261 - 391696): single isocyanide-nrp
</option>


</select>

<div class="details-svg" id="r3c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-64"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-64">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-pfam-details-svg" data-filename="CP023326.1_r3c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c2-nrps_pks_products" data-name="r3c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c2-nrps_pks_monomers" data-name="r3c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-60"></div> <span class="help-tooltip" id="nrps-structures-help-60">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 334261 - 391696:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-61"></div> <span class="help-tooltip" id="nrps-monomers-help-61">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene83</span></strong>: X
<div class="collapser collapser-target-input-path1-gene83 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (411..821)</strong>: X
<div class="collapser collapser-target-input-path1-gene83 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene83-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GAMSCSLFIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023326.1 - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-370"></div> <span class="help-tooltip" id="region_svg-help-370">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023326.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c3-svg" data-filename="CP023326.1_r3c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,030,044 - 1,074,412 nt. (total: 44,369 nt)

<a class="cluster-rules-header" id="r3c3-rules-header" href="#r3c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-371"></div> <span class="help-tooltip" id="focus-panel-help-371">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c3-gene-table-details" data-name="r3c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c3-nrps_pks" data-name="r3c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c3-MIBiG-cluster-compare" data-name="r3c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c3-clusterblast" data-name="r3c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c3-knownclusterblast" data-name="r3c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c3-subclusterblast" data-name="r3c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c3-pfam-details" data-name="r3c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c3-tigrfam-details" data-name="r3c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-71"></div> <span class="help-tooltip" id="gt-help-tooltip-help-71">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-65"></div> <span class="help-tooltip" id="nrps-domain-help-65">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-details-svg" data-filename="CP023326.1_r3c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-66"></div> <span class="help-tooltip" id="MIBiG-help-66">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c3"></div>
<div id="comparison-MIBiG-r3c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404: 1968-56662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>penicillin</td>


<td>Penicillium chrysogenum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>xenematide</td>


<td>Xenorhabdus nematophila AN6/1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>sansalvamide</td>


<td>[Nectria] haematococca mpVI 77-13-4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404: 1968-56662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>penicillin</td>
<td>Penicillium chrysogenum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825: 0-14850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>xenematide</td>
<td>Xenorhabdus nematophila AN6/1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>sansalvamide</td>
<td>[Nectria] haematococca mpVI 77-13-4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-67"></div> <span class="help-tooltip" id="cb-general-help-67">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c3_1.svg">
NW_006271969 (2236886-2301135): Cordyceps militaris CM01 unplaced genomic sca... (63% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c3_2.svg">
NW_009276938 (1486712-1553486): Verticillium dahliae VdLs.17 supercont1.5 gen... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_3.svg">
NW_022987240 (51101-118095): Trematosphaeria pertusa strain CBS 122368 unplac... (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c3_4.svg">
NC_070534 (1074625-1138530): Fusarium keratoplasticum isolate Fu6.1 chromosom... (20% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r3c3_5.svg">
NZ_CP047318 (5618751-5678859): Streptomyces sp. HM190 chromosome, complete ge... (4% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c3_6.svg">
NZ_CP049780 (4864858-4908809): Streptomyces sp. JB150 chromosome (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c3_7.svg">
NZ_BMVV01000017 (56440-100403): Streptomyces omiyaensis strain JCM 4806 seque... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c3_8.svg">
NZ_LT607413 (4729322-4815787): Micromonospora echinospora strain DSM 43816 ch... (3% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r3c3_9.svg">
NZ_PNXQ01000016 (500910-548524): Paenibacillus terrae strain PIC167 Contig 01... (5% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r3c3_10.svg">
NZ_MAUJ01000002 (439421-527978): Pseudoalteromonas luteoviolacea strain IPB1 ... (3% of genes show similarity), NRPS,NRPS-like
</option>

</select>
<button id= "clusterblast-r3c3-download">Download graphic</button>
</div>
<div id="clusterblast-r3c3-svg"></div>

</div>
</div>

<div class="body-details-section r3c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-68"></div> <span class="help-tooltip" id="cb-known-help-68">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-69"></div> <span class="help-tooltip" id="cb-sub-help-69">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-72"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-72">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-pfam-details-svg" data-filename="CP023326.1_r3c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-73"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-73">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-tigrfam-details-svg" data-filename="CP023326.1_r3c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c3-nrps_pks_monomers" data-name="r3c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r3c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-70"></div> <span class="help-tooltip" id="nrps-monomers-help-70">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene230</span></strong>: X
<div class="collapser collapser-target-input-path1-gene230 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (226..661)</strong>: X
<div class="collapser collapser-target-input-path1-gene230 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene230-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLMFVGIMCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAMLVGAACK</span> (59% 8Å match)</dd>
<dd>Hty <span class="serif">DLGFTGAVCK</span> (56% 8Å match)</dd>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r3c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023326.1 - Region 4 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-372"></div> <span class="help-tooltip" id="region_svg-help-372">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023326.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c4-svg" data-filename="CP023326.1_r3c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,257,328 - 1,346,792 nt. (total: 89,465 nt)

<a class="cluster-rules-header" id="r3c4-rules-header" href="#r3c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r3c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-373"></div> <span class="help-tooltip" id="focus-panel-help-373">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c4-gene-table-details" data-name="r3c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c4-nrps_pks" data-name="r3c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c4-MIBiG-cluster-compare" data-name="r3c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c4-clusterblast" data-name="r3c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c4-knownclusterblast" data-name="r3c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c4-subclusterblast" data-name="r3c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c4-nrps-pks-bubbles" data-name="r3c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r3c4-pfam-details" data-name="r3c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-82"></div> <span class="help-tooltip" id="gt-help-tooltip-help-82">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-74"></div> <span class="help-tooltip" id="nrps-domain-help-74">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-details-svg" data-filename="CP023326.1_r3c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-75"></div> <span class="help-tooltip" id="MIBiG-help-75">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c4"></div>
<div id="comparison-MIBiG-r3c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>Other</td>

<td>itaconic acid</td>


<td>Ustilago maydis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.38</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.38</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002606: 0-21662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002606/index.html#r1c1" target="_new">BGC0002606</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.36</td>
<td>NRP, Polyketide</td>

<td>ochratoxin A</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>oxepinamide D</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002208: 764-29749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002208/index.html#r1c1" target="_new">BGC0002208</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>oxepinamide F</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.34</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.34</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger CBS 101883</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002749: 1722-17470">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002749/index.html#r1c1" target="_new">BGC0002749</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.33</td>
<td>NRP, Polyketide</td>

<td>aspcandine</td>


<td>Aspergillus candidus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.33</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>Other</td>
<td>itaconic acid</td>
<td>Ustilago maydis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>lupeol</td>
<td>Lotus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717: 0-19662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>okaramine B</td>
<td>Penicillium simplicissimum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-76"></div> <span class="help-tooltip" id="cb-general-help-76">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c4_1.svg">
NW_006271969 (1967760-2079673): Cordyceps militaris CM01 unplaced genomic sca... (60% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c4_2.svg">
NW_022985016 (85063-152593): Dothidotthia symphoricarpi CBS 119687 unplaced g... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c4_3.svg">
NW_013550604 (1398013-1484027): Rhinocladiella mackenziei CBS 650.93 unplaced... (19% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r3c4_4.svg">
NW_025091283 (4319746-4671991): Aspergillus pseudoviridinutans IFM 55266 DNA,... (6% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r3c4_5.svg">
NW_025091284 (1562771-1677454): Aspergillus pseudoviridinutans IFM 55266 DNA,... (13% of genes show similarity), NI-siderophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c4_6.svg">
NW_006271969 (4860304-4927848): Cordyceps militaris CM01 unplaced genomic sca... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c4_7.svg">
NW_017971432 (1544947-1608598): Talaromyces atroroseus strain IBT 11181 chrom... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c4_8.svg">
NW_022983478 (652553-879885): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (6% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c4_9.svg">
NW_020291494 (375619-484726): Aspergillus aculeatinus CBS 121060 unplaced gen... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c4_10.svg">
NW_024469024 (26525-267638): Aspergillus fijiensis CBS 313.89 unplaced genomi... (4% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

</select>
<button id= "clusterblast-r3c4-download">Download graphic</button>
</div>
<div id="clusterblast-r3c4-svg"></div>

</div>
</div>

<div class="body-details-section r3c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-77"></div> <span class="help-tooltip" id="cb-known-help-77">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c4_1.svg">
nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

<option value = "svg/knownclusterblast_r3c4_2.svg">
nivalenol/vomitoxin/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-deacetylcalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

</select>
<button id="knownclusterblast-r3c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c4-svg"></div>

</div>
</div>

<div class="body-details-section r3c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-78"></div> <span class="help-tooltip" id="cb-sub-help-78">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-81"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-81">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-domain-bubble-svg" data-filename="CP023326.1_r3c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (1257327 - 1346792): neighbouring NRPS-T1PKS
</option>



<option value="CC5">
Candidate 5 (1257327 - 1300651): single NRPS
</option>



<option value="CC6">
Candidate 6 (1298844 - 1346792): single T1PKS
</option>


</select>

<div class="details-svg" id="r3c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r3c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-83"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-83">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-pfam-details-svg" data-filename="CP023326.1_r3c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c4-nrps_pks_products" data-name="r3c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c4-nrps_pks_monomers" data-name="r3c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r3c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-79"></div> <span class="help-tooltip" id="nrps-structures-help-79">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 1257327 - 1346792:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c4-4" class="smiles-canvas" data-smiles="NC(C)C(=O)C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ala) + (pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C)C(=O)C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 5, location 1257327 - 1300651:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c4-5" class="smiles-canvas" data-smiles="NC(C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ala)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c4-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 6, location 1298844 - 1346792:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c4-6" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c4-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-80"></div> <span class="help-tooltip" id="nrps-monomers-help-80">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene296</span></strong>: Ala
<div class="collapser collapser-target-input-path1-gene296 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (1..319)</strong>: Ala
<div class="collapser collapser-target-input-path1-gene296 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-input-path1-gene296-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DADGCWVGTK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene310</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene310 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (572..901)</strong>: pk
<div class="collapser collapser-target-input-path1-gene310 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene310-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene310-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 86.6</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 83.3</dt>

<dd></dd><dt>Malonyl-CoA: 60.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 56.6</dt>

<dd></dd><dt>Propionyl-CoA: 47.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 47.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 43.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 30.3</dt>

<dd></dd><dt>CHC-CoA: 30.0</dt>

<dd></dd><dt>fatty_acid: 19.9</dt>

<dd></dd><dt>Benzoyl-CoA: 15.7</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 7.5</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r3c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023326.1 - Region 5 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-374"></div> <span class="help-tooltip" id="region_svg-help-374">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023326.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c5-svg" data-filename="CP023326.1_r3c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,834,759 - 2,895,794 nt. (total: 61,036 nt)

<a class="cluster-rules-header" id="r3c5-rules-header" href="#r3c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c5-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-375"></div> <span class="help-tooltip" id="focus-panel-help-375">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c5-gene-table-details" data-name="r3c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c5-MIBiG-cluster-compare" data-name="r3c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c5-clusterblast" data-name="r3c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c5-knownclusterblast" data-name="r3c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c5-subclusterblast" data-name="r3c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c5-pfam-details" data-name="r3c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c5-tigrfam-details" data-name="r3c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-88"></div> <span class="help-tooltip" id="gt-help-tooltip-help-88">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-84"></div> <span class="help-tooltip" id="MIBiG-help-84">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c5"></div>
<div id="comparison-MIBiG-r3c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000030: 0-18435">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000030/index.html#r1c1" target="_new">BGC0000030</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>bikaverin</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001404: 0-28758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001404/index.html#r1c1" target="_new">BGC0001404</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>sorbicillin</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002190: 548-34110">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002190/index.html#r1c1" target="_new">BGC0002190</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>sorbicillin, 2&#39;3&#39;-dihydrosorbicillin, sorbicillinol, oxosorbicillinol, bisvertinolone (spelled bisvertinolon in paper), dihydrobisvertinolone, tetrahydrobisvertinolone, bisorbicillinol, 2&#39;3&#39;-dihydrosorbicillinol</td>


<td>Acremonium chrysogenum ATCC 11550</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001738: 0-24326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001738/index.html#r1c1" target="_new">BGC0001738</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>phomasetin</td>


<td>Pyrenochaetopsis sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.46, order:0.36, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>nidulanin A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002197: 1638-28433">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002197/index.html#r1c1" target="_new">BGC0002197</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.46, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide, NRP</td>

<td>terrestric acid</td>


<td>Penicillium crustosum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>viguiepinol</td>


<td>Streptomyces sp. KO-3988</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002262: 0-50372">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002262/index.html#r1c1" target="_new">BGC0002262</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>trienylfuranol A, sporothriolide, dihydrosporothriolide, deoxysporothric acid, dihydroisosporothric acid, sporothric acid, isosporothric acid, sporochartine B</td>


<td>Hypomontagnella monticulosa</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000030: 0-18435">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000030/index.html#r1c1" target="_new">BGC0000030</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>bikaverin</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001404: 0-28758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001404/index.html#r1c1" target="_new">BGC0001404</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>sorbicillin</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002190: 548-34110">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002190/index.html#r1c1" target="_new">BGC0002190</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>sorbicillin, 2&#39;3&#39;-dihydrosorbicillin, sorbicillinol, oxosorbicillinol, bisvertinolone (spelled bisvertinolon in paper), dihydrobisvertinolone, tetrahydrobisvertinolone, bisorbicillinol, 2&#39;3&#39;-dihydrosorbicillinol</td>
<td>Acremonium chrysogenum ATCC 11550</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001738: 0-24326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001738/index.html#r1c1" target="_new">BGC0001738</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>NRP, Polyketide</td>
<td>phomasetin</td>
<td>Pyrenochaetopsis sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.46, order:0.36, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>NRP</td>
<td>nidulanin A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002197: 1638-28433">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002197/index.html#r1c1" target="_new">BGC0002197</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.46, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide, NRP</td>
<td>terrestric acid</td>
<td>Penicillium crustosum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>viguiepinol</td>
<td>Streptomyces sp. KO-3988</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002262: 0-50372">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002262/index.html#r1c1" target="_new">BGC0002262</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>trienylfuranol A, sporothriolide, dihydrosporothriolide, deoxysporothric acid, dihydroisosporothric acid, sporothric acid, isosporothric acid, sporochartine B</td>
<td>Hypomontagnella monticulosa</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-85"></div> <span class="help-tooltip" id="cb-general-help-85">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c5_1.svg">
NW_025764704 (762962-853638): Emericellopsis atlantica unplaced genomic scaff... (17% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_2.svg">
NW_024067566 (523062-613732): Penicillium roqueforti strain LCP96 04111 chrom... (15% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_3.svg">
NW_024468936 (4943083-5033757): Cucurbitaria berberidis CBS 394.84 unplaced g... (14% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_4.svg">
NW_022985018 (595640-686314): Dothidotthia symphoricarpi CBS 119687 unplaced ... (15% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_5.svg">
NC_064672 (2020083-2159961): Colletotrichum lupini chromosome 1, complete seq... (12% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_6.svg">
NW_009258210 (60061-249466): Heterobasidion irregulare TC 32-1 unplaced genom... (17% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_7.svg">
NW_024467220 (23402-114183): Dacryopinax primogenitus strain DJM-731 SS1 chro... (20% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_8.svg">
NW_022985549 (2326994-2419159): Pseudovirgaria hyperparasitica strain CBS 121... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_9.svg">
NW_009258206 (1553961-1650456): Heterobasidion irregulare TC 32-1 unplaced ge... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c5_10.svg">
NW_015971635 (1692172-1793674): Fonsecaea multimorphosa CBS 102226 unplaced g... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c5-download">Download graphic</button>
</div>
<div id="clusterblast-r3c5-svg"></div>

</div>
</div>

<div class="body-details-section r3c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-86"></div> <span class="help-tooltip" id="cb-known-help-86">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-87"></div> <span class="help-tooltip" id="cb-sub-help-87">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-89"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-89">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-pfam-details-svg" data-filename="CP023326.1_r3c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-90"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-90">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c5-tigrfam-details-svg" data-filename="CP023326.1_r3c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r4c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023322.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-376"></div> <span class="help-tooltip" id="region_svg-help-376">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023322.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c1-svg" data-filename="CP023322.1_r4c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 189,498 - 211,796 nt. (total: 22,299 nt)

<a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r4c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-377"></div> <span class="help-tooltip" id="focus-panel-help-377">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c1-gene-table-details" data-name="r4c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c1-tigrfam-details" data-name="r4c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-95"></div> <span class="help-tooltip" id="gt-help-tooltip-help-95">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-91"></div> <span class="help-tooltip" id="MIBiG-help-91">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c1"></div>
<div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001248: 1352-4192">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001248/index.html#r1c1" target="_new">BGC0001248</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>clavaric acid</td>


<td>Hypholoma sublateritium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.43, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Other</td>

<td>sarasinoside</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Terpene</td>

<td>lupeol</td>


<td>Lotus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002402: 203-37725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002402/index.html#r1c1" target="_new">BGC0002402</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>yossosides</td>


<td>Spinacia oleracea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>thalianol, thaliandiol</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000686: 0-16277">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000686/index.html#r1c1" target="_new">BGC0000686</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.47, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Terpene</td>

<td>helvolic acid</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314: 1711-56297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Terpene</td>

<td>tirucalla</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000273: 50-42876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000273/index.html#r1c1" target="_new">BGC0000273</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.47, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>Polyketide, Saccharide</td>

<td>steffimycin D</td>


<td>Streptomyces steffisburgensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002722: 3-337945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002722/index.html#r1c1" target="_new">BGC0002722</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.45, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>Alkaloid, Terpene, Saccharide</td>

<td>α-chaconine, α-solanine</td>


<td>Solanum tuberosum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001248: 1352-4192">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001248/index.html#r1c1" target="_new">BGC0001248</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>clavaric acid</td>
<td>Hypholoma sublateritium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.43, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Other</td>
<td>sarasinoside</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Terpene</td>
<td>lupeol</td>
<td>Lotus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002402: 203-37725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002402/index.html#r1c1" target="_new">BGC0002402</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Terpene</td>
<td>yossosides</td>
<td>Spinacia oleracea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Terpene</td>
<td>thalianol, thaliandiol</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000686: 0-16277">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000686/index.html#r1c1" target="_new">BGC0000686</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.47, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Terpene</td>
<td>helvolic acid</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314: 1711-56297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Terpene</td>
<td>tirucalla</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000273: 50-42876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000273/index.html#r1c1" target="_new">BGC0000273</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.47, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>Polyketide, Saccharide</td>
<td>steffimycin D</td>
<td>Streptomyces steffisburgensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002722: 3-337945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002722/index.html#r1c1" target="_new">BGC0002722</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.45, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>Alkaloid, Terpene, Saccharide</td>
<td>α-chaconine, α-solanine</td>
<td>Solanum tuberosum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-92"></div> <span class="help-tooltip" id="cb-general-help-92">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c1_1.svg">
NW_006271977 (650506-682810): Cordyceps militaris CM01 unplaced genomic scaff... (64% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r4c1_2.svg">
NW_014574706 (185039-217400): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (53% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r4c1_3.svg">
NC_035794 (3381868-3414218): Pochonia chlamydosporia 170 chromosome 5, whole ... (77% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r4c1_4.svg">
NW_014013668 (153970-222143): Trichoderma virens Gv29-8 chromosome Unknown TR... (31% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_5.svg">
NC_054389 (10607680-10640044): Drechmeria coniospora strain ARSEF 6962 chromo... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r4c1_6.svg">
NW_025408667 (5243071-5275419): Hirsutella rhossiliensis strain HR02 chromoso... (77% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r4c1_7.svg">
NC_043740 (4056696-4089123): Pyricularia pennisetigena strain Br36 chromosome... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r4c1_8.svg">
NW_006711154 (931670-999559): Trichoderma reesei QM6a unplaced genomic scaffo... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_9.svg">
NC_057320 (3125192-3157585): Ustilaginoidea virens chromosome 5, complete seq... (54% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r4c1_10.svg">
NW_023336201 (447629-481710): Geosmithia morbida isolate 1262 chromosome Unkn... (38% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r4c1-download">Download graphic</button>
</div>
<div id="clusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-93"></div> <span class="help-tooltip" id="cb-known-help-93">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c1_1.svg">
clavaric acid
</option>

</select>
<button id="knownclusterblast-r4c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-94"></div> <span class="help-tooltip" id="cb-sub-help-94">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r4c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-96"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-96">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="CP023322.1_r4c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-97"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-97">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-tigrfam-details-svg" data-filename="CP023322.1_r4c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r4c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023322.1 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-378"></div> <span class="help-tooltip" id="region_svg-help-378">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023322.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c2-svg" data-filename="CP023322.1_r4c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 615,487 - 676,459 nt. (total: 60,973 nt)

<a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-379"></div> <span class="help-tooltip" id="focus-panel-help-379">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c2-gene-table-details" data-name="r4c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c2-clusterblast" data-name="r4c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c2-pfam-details" data-name="r4c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r4c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-102"></div> <span class="help-tooltip" id="gt-help-tooltip-help-102">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-98"></div> <span class="help-tooltip" id="MIBiG-help-98">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c2"></div>
<div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>auriculamide</td>


<td>Herpetosiphon aurantiacus DSM 785</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.10</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002509: 94-21985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002509/index.html#r1c1" target="_new">BGC0002509</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.42, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>montamide A, capsimycin, clifednamide A, frontalamide A, combamide A</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002294: 412-30500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002294/index.html#r1c1" target="_new">BGC0002294</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Other</td>

<td>streptozotocin</td>


<td>Streptomyces achromogenes subsp. streptozoticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001057: 2684-46050">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001057/index.html#r1c1" target="_new">BGC0001057</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.44, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>NRP, Polyketide</td>

<td>zearalenone</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000190: 0-28713">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000190/index.html#r1c1" target="_new">BGC0000190</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.44, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>A-74528</td>


<td>Streptomyces sp. SANK 61196</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000227: 1098-38839">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000227/index.html#r1c1" target="_new">BGC0000227</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.42, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>granaticin</td>


<td>Streptomyces violaceoruber</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>NRP</td>
<td>auriculamide</td>
<td>Herpetosiphon aurantiacus DSM 785</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002509: 94-21985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002509/index.html#r1c1" target="_new">BGC0002509</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.42, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>NRP</td>
<td>montamide A, capsimycin, clifednamide A, frontalamide A, combamide A</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002294: 412-30500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002294/index.html#r1c1" target="_new">BGC0002294</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Other</td>
<td>streptozotocin</td>
<td>Streptomyces achromogenes subsp. streptozoticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001057: 2684-46050">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001057/index.html#r1c1" target="_new">BGC0001057</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.44, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>NRP, Polyketide</td>
<td>zearalenone</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Polyketide</td>
<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000190: 0-28713">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000190/index.html#r1c1" target="_new">BGC0000190</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.44, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Polyketide</td>
<td>A-74528</td>
<td>Streptomyces sp. SANK 61196</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000227: 1098-38839">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000227/index.html#r1c1" target="_new">BGC0000227</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.42, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Polyketide</td>
<td>granaticin</td>
<td>Streptomyces violaceoruber</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-99"></div> <span class="help-tooltip" id="cb-general-help-99">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c2_1.svg">
NW_006271977 (184210-275183): Cordyceps militaris CM01 unplaced genomic scaff... (34% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_2.svg">
NW_023336282 (1703318-1900820): Aspergillus tubingensis WU-2223L DNA, scaffol... (2% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_3.svg">
NW_020292039 (3332166-3473157): Aspergillus vadensis CBS 113365 unplaced geno... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_4.svg">
NW_022985457 (182988-273904): Mytilinidion resinicola strain CBS 304.34 unpla... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_5.svg">
NW_022987217 (2027599-2119214): Trematosphaeria pertusa strain CBS 122368 unp... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_6.svg">
NW_017253365 (654740-748459): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_7.svg">
NW_013550604 (816316-907011): Rhinocladiella mackenziei CBS 650.93 unplaced g... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_8.svg">
NW_017263601 (538564-636121): Mollisia scopiformis unplaced genomic scaffold ... (5% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_9.svg">
NW_026054545 (823442-914479): Daldinia decipiens strain CBS 113046 unplaced g... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_10.svg">
NW_025953037 (2770655-2861749): Daldinia caldariorum strain CBS 122874 unplac... (9% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r4c2-download">Download graphic</button>
</div>
<div id="clusterblast-r4c2-svg"></div>

</div>
</div>

<div class="body-details-section r4c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-100"></div> <span class="help-tooltip" id="cb-known-help-100">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-101"></div> <span class="help-tooltip" id="cb-sub-help-101">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-103"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-103">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-pfam-details-svg" data-filename="CP023322.1_r4c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r4c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023322.1 - Region 3 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-380"></div> <span class="help-tooltip" id="region_svg-help-380">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023322.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c3-svg" data-filename="CP023322.1_r4c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 869,949 - 913,940 nt. (total: 43,992 nt)

<a class="cluster-rules-header" id="r4c3-rules-header" href="#r4c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c3-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-381"></div> <span class="help-tooltip" id="focus-panel-help-381">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c3-gene-table-details" data-name="r4c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c3-nrps_pks" data-name="r4c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c3-MIBiG-cluster-compare" data-name="r4c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c3-clusterblast" data-name="r4c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c3-knownclusterblast" data-name="r4c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c3-subclusterblast" data-name="r4c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c3-nrps-pks-bubbles" data-name="r4c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c3-pfam-details" data-name="r4c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c3-tigrfam-details" data-name="r4c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-112"></div> <span class="help-tooltip" id="gt-help-tooltip-help-112">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-104"></div> <span class="help-tooltip" id="nrps-domain-help-104">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-details-svg" data-filename="CP023322.1_r4c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-105"></div> <span class="help-tooltip" id="MIBiG-help-105">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c3"></div>
<div id="comparison-MIBiG-r4c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>ustethylin A</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002190: 548-34110">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002190/index.html#r1c1" target="_new">BGC0002190</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>sorbicillin, 2&#39;3&#39;-dihydrosorbicillin, sorbicillinol, oxosorbicillinol, bisvertinolone (spelled bisvertinolon in paper), dihydrobisvertinolone, tetrahydrobisvertinolone, bisorbicillinol, 2&#39;3&#39;-dihydrosorbicillinol</td>


<td>Acremonium chrysogenum ATCC 11550</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>eupenifeldin</td>


<td>Phoma sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002491: 0-17030">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002491/index.html#r1c1" target="_new">BGC0002491</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>myxochelin B, myxochelin A, pseudochelin A</td>


<td>Pseudoalteromonas piscicida</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002206: 173-38907">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002206/index.html#r1c1" target="_new">BGC0002206</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>harziphilone, t22azaphilone, isoharziphilone-1, isoharziphilone-2, compound 4, compound 1</td>


<td>Trichoderma guizhouense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>ustethylin A</td>
<td>Aspergillus ustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002190: 548-34110">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002190/index.html#r1c1" target="_new">BGC0002190</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>sorbicillin, 2&#39;3&#39;-dihydrosorbicillin, sorbicillinol, oxosorbicillinol, bisvertinolone (spelled bisvertinolon in paper), dihydrobisvertinolone, tetrahydrobisvertinolone, bisorbicillinol, 2&#39;3&#39;-dihydrosorbicillinol</td>
<td>Acremonium chrysogenum ATCC 11550</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Terpene</td>
<td>eupenifeldin</td>
<td>Phoma sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002491: 0-17030">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002491/index.html#r1c1" target="_new">BGC0002491</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>NRP</td>
<td>myxochelin B, myxochelin A, pseudochelin A</td>
<td>Pseudoalteromonas piscicida</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002206: 173-38907">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002206/index.html#r1c1" target="_new">BGC0002206</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Polyketide</td>
<td>harziphilone, t22azaphilone, isoharziphilone-1, isoharziphilone-2, compound 4, compound 1</td>
<td>Trichoderma guizhouense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-106"></div> <span class="help-tooltip" id="cb-general-help-106">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c3_1.svg">
NW_006271972 (141643-205500): Cordyceps militaris CM01 unplaced genomic scaff... (53% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c3_2.svg">
NC_070534 (194101-306469): Fusarium keratoplasticum isolate Fu6.1 chromosome ... (7% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r4c3_3.svg">
NW_025764711 (689887-753562): Emericellopsis atlantica unplaced genomic scaff... (8% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c3_4.svg">
NC_070549 (252971-316778): Fusarium falciforme chromosome 6, complete sequence (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c3_5.svg">
NC_072850 (2763613-2827398): Fusarium oxysporum Fo47 chromosome XI, complete ... (14% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c3_6.svg">
NC_035793 (242100-305951): Pochonia chlamydosporia 170 chromosome 4, whole ge... (8% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c3_7.svg">
NC_054691 (37037-214255): Aspergillus flavus NRRL3357 chromosome 1, complete ... (8% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r4c3_8.svg">
NW_014013673 (128037-191883): Trichoderma virens Gv29-8 chromosome Unknown TR... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c3_9.svg">
NW_024468936 (3257142-3320914): Cucurbitaria berberidis CBS 394.84 unplaced g... (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c3_10.svg">
NW_022985382 (9951-73992): Macroventuria anomochaeta strain CBS 525.71 unplac... (8% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r4c3-download">Download graphic</button>
</div>
<div id="clusterblast-r4c3-svg"></div>

</div>
</div>

<div class="body-details-section r4c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-107"></div> <span class="help-tooltip" id="cb-known-help-107">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-108"></div> <span class="help-tooltip" id="cb-sub-help-108">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-111"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-111">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-domain-bubble-svg" data-filename="CP023322.1_r4c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (869948 - 913940): single NRPS-like
</option>


</select>

<div class="details-svg" id="r4c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-113"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-113">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-pfam-details-svg" data-filename="CP023322.1_r4c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-114"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-114">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c3-tigrfam-details-svg" data-filename="CP023322.1_r4c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c3-nrps_pks_products" data-name="r4c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c3-nrps_pks_monomers" data-name="r4c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-109"></div> <span class="help-tooltip" id="nrps-structures-help-109">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 869948 - 913940:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c3-3" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-110"></div> <span class="help-tooltip" id="nrps-monomers-help-110">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene231</span></strong>: X
<div class="collapser collapser-target-input-path1-gene231 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (51..367)</strong>: X
<div class="collapser collapser-target-input-path1-gene231 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene231-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">-HCYMGHQLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023322.1 - Region 4 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-382"></div> <span class="help-tooltip" id="region_svg-help-382">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023322.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c4-svg" data-filename="CP023322.1_r4c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,586,874 - 1,632,367 nt. (total: 45,494 nt)

<a class="cluster-rules-header" id="r4c4-rules-header" href="#r4c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-383"></div> <span class="help-tooltip" id="focus-panel-help-383">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c4-gene-table-details" data-name="r4c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c4-nrps_pks" data-name="r4c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c4-MIBiG-cluster-compare" data-name="r4c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c4-clusterblast" data-name="r4c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c4-knownclusterblast" data-name="r4c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c4-subclusterblast" data-name="r4c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c4-pfam-details" data-name="r4c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c4-tigrfam-details" data-name="r4c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-121"></div> <span class="help-tooltip" id="gt-help-tooltip-help-121">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-115"></div> <span class="help-tooltip" id="nrps-domain-help-115">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-details-svg" data-filename="CP023322.1_r4c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-116"></div> <span class="help-tooltip" id="MIBiG-help-116">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c4"></div>
<div id="comparison-MIBiG-r4c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.22</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.55, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>marneral</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>hydroxycinnamoyltyramine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002619: 0-42148">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002619/index.html#r1c1" target="_new">BGC0002619</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.57, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>mycophenolic acid</td>


<td>Penicillium brevicompactum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868: 0-11041">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.49, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Other (Phenazine)</td>

<td>iodinin</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.55, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>hydroxycinnamoyltyramine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002619: 0-42148">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002619/index.html#r1c1" target="_new">BGC0002619</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.57, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>mycophenolic acid</td>
<td>Penicillium brevicompactum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868: 0-11041">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.49, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Other (Phenazine)</td>
<td>iodinin</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-117"></div> <span class="help-tooltip" id="cb-general-help-117">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c4_1.svg">
NW_006271972 (846900-910624): Cordyceps militaris CM01 unplaced genomic scaff... (50% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_2.svg">
NW_025408674 (2031871-2095823): Hirsutella rhossiliensis strain HR02 chromoso... (39% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_3.svg">
NC_030957 (4696242-4786854): Colletotrichum higginsianum IMI 349063 chromosom... (17% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c4_4.svg">
NC_070531 (1081663-1145343): Fusarium keratoplasticum isolate Fu6.1 chromosom... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_5.svg">
NC_070546 (1097748-1161429): Fusarium falciforme chromosome 3, complete sequence (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_6.svg">
NC_036627 (1441489-1505176): Fusarium fujikuroi IMI 58289 draft genome, chrom... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_7.svg">
NC_072845 (1566921-1630612): Fusarium oxysporum Fo47 chromosome VI, complete ... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_8.svg">
NC_058400 (1372797-1436602): Fusarium poae strain DAOMC 252244 chromosome 2, ... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_9.svg">
NC_058392 (2751608-2815071): Fusarium musae strain F31 chromosome 6, whole ge... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r4c4_10.svg">
NW_017263587 (265177-438368): Mollisia scopiformis unplaced genomic scaffold ... (4% of genes show similarity), NRPS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r4c4-download">Download graphic</button>
</div>
<div id="clusterblast-r4c4-svg"></div>

</div>
</div>

<div class="body-details-section r4c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-118"></div> <span class="help-tooltip" id="cb-known-help-118">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-119"></div> <span class="help-tooltip" id="cb-sub-help-119">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-122"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-122">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-pfam-details-svg" data-filename="CP023322.1_r4c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-123"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-123">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c4-tigrfam-details-svg" data-filename="CP023322.1_r4c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c4-nrps_pks_monomers" data-name="r4c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r4c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-120"></div> <span class="help-tooltip" id="nrps-monomers-help-120">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene444</span></strong>: X
<div class="collapser collapser-target-input-path1-gene444 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (37..466)</strong>: X
<div class="collapser collapser-target-input-path1-gene444 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene444-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">IFNSSAAPVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r4c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023322.1 - Region 5 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-384"></div> <span class="help-tooltip" id="region_svg-help-384">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023322.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c5-svg" data-filename="CP023322.1_r4c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,965,926 - 5,026,565 nt. (total: 60,640 nt)

<a class="cluster-rules-header" id="r4c5-rules-header" href="#r4c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c5-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r4c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-385"></div> <span class="help-tooltip" id="focus-panel-help-385">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c5-gene-table-details" data-name="r4c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c5-MIBiG-cluster-compare" data-name="r4c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c5-clusterblast" data-name="r4c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c5-knownclusterblast" data-name="r4c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c5-subclusterblast" data-name="r4c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c5-pfam-details" data-name="r4c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r4c5-tigrfam-details" data-name="r4c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r4c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-128"></div> <span class="help-tooltip" id="gt-help-tooltip-help-128">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-124"></div> <span class="help-tooltip" id="MIBiG-help-124">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c5"></div>
<div id="comparison-MIBiG-r4c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.61, order:0.46, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000624: 46-7740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000624/index.html#r1c1" target="_new">BGC0000624</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>salivaricin CRL1328 α peptide, salivaricin CRL1328 β peptide</td>


<td>Lactobacillus salivarius</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.45, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>nidulanin A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002747: 549-28042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002747/index.html#r1c1" target="_new">BGC0002747</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>lentinulin A</td>


<td>Lentinula edodes</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001402: 590-34242">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001402/index.html#r1c1" target="_new">BGC0001402</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>cyclochlorotine, hydroxycyclochlorotine A, cyclotine, cyclochlorotine B, deoxy-Cyclochlorotine</td>


<td>Talaromyces islandicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>β-Diketones</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002330: 0-16516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002330/index.html#r1c1" target="_new">BGC0002330</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>landornamide A</td>


<td>Kamptonema sp. PCC 6506</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.61, order:0.46, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000624: 46-7740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000624/index.html#r1c1" target="_new">BGC0000624</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>RiPP</td>
<td>salivaricin CRL1328 α peptide, salivaricin CRL1328 β peptide</td>
<td>Lactobacillus salivarius</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.45, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>NRP</td>
<td>nidulanin A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002747: 549-28042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002747/index.html#r1c1" target="_new">BGC0002747</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>RiPP</td>
<td>lentinulin A</td>
<td>Lentinula edodes</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001402: 590-34242">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001402/index.html#r1c1" target="_new">BGC0001402</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP</td>
<td>cyclochlorotine, hydroxycyclochlorotine A, cyclotine, cyclochlorotine B, deoxy-Cyclochlorotine</td>
<td>Talaromyces islandicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>β-Diketones</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002330: 0-16516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002330/index.html#r1c1" target="_new">BGC0002330</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>RiPP</td>
<td>landornamide A</td>
<td>Kamptonema sp. PCC 6506</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-125"></div> <span class="help-tooltip" id="cb-general-help-125">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c5_1.svg">
NW_006271972 (3966194-4059400): Cordyceps militaris CM01 unplaced genomic sca... (53% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_2.svg">
NW_025091264 (5067321-5160178): Aspergillus udagawae IFM 46973 DNA, scaffold ... (12% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_3.svg">
NW_017264196 (550250-647543): Xylona heveae TC161 unplaced genomic scaffold L... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_4.svg">
NC_049566 (4842785-4935536): Talaromyces rugulosus chromosome VI, complete se... (17% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_5.svg">
NC_064655 (2136069-2230273): Penicillium oxalicum strain HP7-1 chromosome VI,... (10% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_6.svg">
NC_064676 (3053739-3126636): Colletotrichum lupini chromosome 3, complete seq... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r4c5_7.svg">
NW_024467835 (2629885-2660821): Aspergillus wentii DTO 134E9 unplaced genomic... (14% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r4c5_8.svg">
NW_017387255 (344948-435980): Fonsecaea erecta strain CBS 125763 chromosome U... (5% of genes show similarity), betalactone,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_9.svg">
NW_008481819 (221274-312185): Cladophialophora carrionii CBS 160.54 unplaced ... (5% of genes show similarity), betalactone,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c5_10.svg">
NW_006913045 (4824330-4915217): Cladophialophora yegresii CBS 114405 aczIN-su... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r4c5-download">Download graphic</button>
</div>
<div id="clusterblast-r4c5-svg"></div>

</div>
</div>

<div class="body-details-section r4c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-126"></div> <span class="help-tooltip" id="cb-known-help-126">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c5-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c5_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c5_1.svg">
beauvericin
</option>

</select>
<button id="knownclusterblast-r4c5-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c5-svg"></div>

</div>
</div>

<div class="body-details-section r4c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-127"></div> <span class="help-tooltip" id="cb-sub-help-127">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-129"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-129">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c5-pfam-details-svg" data-filename="CP023322.1_r4c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r4c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-130"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-130">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c5-tigrfam-details-svg" data-filename="CP023322.1_r4c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r4c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r5c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023325.1 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-386"></div> <span class="help-tooltip" id="region_svg-help-386">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023325.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c1-svg" data-filename="CP023325.1_r5c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,181,633 - 1,249,181 nt. (total: 67,549 nt)

<a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-387"></div> <span class="help-tooltip" id="focus-panel-help-387">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c1-gene-table-details" data-name="r5c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c1-nrps_pks" data-name="r5c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c1-nrps-pks-bubbles" data-name="r5c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-139"></div> <span class="help-tooltip" id="gt-help-tooltip-help-139">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-131"></div> <span class="help-tooltip" id="nrps-domain-help-131">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-details-svg" data-filename="CP023325.1_r5c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-132"></div> <span class="help-tooltip" id="MIBiG-help-132">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c1"></div>
<div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002639: 0-8541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002639/index.html#r1c1" target="_new">BGC0002639</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.54</td>
<td>NRP</td>

<td>phospho-tyrolose, tyrolose, tyrolitol, tyrocitabine-626, acyl-tyrocitabine-668, acyl-tyrocitabine-696, acyl-tyrocitabine-724, acyl-tyrocitabine-752</td>


<td>Lactobacillus iners LEAF 2052A-d</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP, Polyketide</td>

<td>himeic acid A</td>


<td>Aspergillus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000287: 10-13001">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000287/index.html#r1c1" target="_new">BGC0000287</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>NRP</td>

<td>L-2-amino-4-methoxy-trans-3-butenoic acid</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21-β-H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>6-MSA, aculin A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002639: 0-8541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002639/index.html#r1c1" target="_new">BGC0002639</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>phospho-tyrolose, tyrolose, tyrolitol, tyrocitabine-626, acyl-tyrocitabine-668, acyl-tyrocitabine-696, acyl-tyrocitabine-724, acyl-tyrocitabine-752</td>
<td>Lactobacillus iners LEAF 2052A-d</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>himeic acid A</td>
<td>Aspergillus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-133"></div> <span class="help-tooltip" id="cb-general-help-133">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c1_1.svg">
NW_006271971 (3503471-3590640): Cordyceps militaris CM01 unplaced genomic sca... (70% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_2.svg">
NW_020290887 (673812-764439): Aspergillus niger CBS 101883 unplaced genomic s... (28% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_3.svg">
NW_023336287 (1029426-1127137): Aspergillus tubingensis WU-2223L DNA, scaffol... (29% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c1_4.svg">
NW_020291816 (340965-433020): Aspergillus costaricaensis CBS 115574 unplaced ... (29% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_5.svg">
NW_020292069 (316400-408453): Aspergillus vadensis CBS 113365 unplaced genomi... (30% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_6.svg">
NW_020291625 (99659-196841): Aspergillus piperis CBS 112811 unplaced genomic ... (29% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c1_7.svg">
NC_054853 (2119459-2216438): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (25% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c1_8.svg">
NW_013581207 (483365-550922): Aureobasidium namibiae CBS 147.97 unplaced geno... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c1_9.svg">
NW_022474207 (2338451-2410505): Venustampulla echinocandica strain BP 5553 ch... (19% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c1_10.svg">
NW_013566983 (689939-757505): Aureobasidium subglaciale EXF-2481 unplaced gen... (15% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r5c1-download">Download graphic</button>
</div>
<div id="clusterblast-r5c1-svg"></div>

</div>
</div>

<div class="body-details-section r5c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-134"></div> <span class="help-tooltip" id="cb-known-help-134">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-135"></div> <span class="help-tooltip" id="cb-sub-help-135">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-138"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-138">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-domain-bubble-svg" data-filename="CP023325.1_r5c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (1181632 - 1249181): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r5c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-140"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-140">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="CP023325.1_r5c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c1-nrps_pks_products" data-name="r5c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c1-nrps_pks_monomers" data-name="r5c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-136"></div> <span class="help-tooltip" id="nrps-structures-help-136">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 1181632 - 1249181:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-137"></div> <span class="help-tooltip" id="nrps-monomers-help-137">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene281</span></strong>: X
<div class="collapser collapser-target-input-path1-gene281 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (244..681)</strong>: X
<div class="collapser collapser-target-input-path1-gene281 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene281-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLIFVGIMAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DALFVGLVAK</span> (56% 8Å match)</dd>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (53% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene284</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene284 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (532..850)</strong>: pk
<div class="collapser collapser-target-input-path1-gene284 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene284-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene284-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 67.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 58.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 39.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 29.5</dt>

<dd></dd><dt>Malonyl-CoA: 20.6</dt>

<dd></dd><dt>Propionyl-CoA: 18.5</dt>

<dd></dd><dt>fatty_acid: 18.4</dt>

<dd></dd><dt>Benzoyl-CoA: 15.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 15.3</dt>

<dd></dd><dt>CHC-CoA: 11.8</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2921..3329)</strong>: X
<div class="collapser collapser-target-input-path1-gene284 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene284-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIDTNGNVWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGVVWK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023325.1 - Region 2 - indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-388"></div> <span class="help-tooltip" id="region_svg-help-388">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023325.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c2-svg" data-filename="CP023325.1_r5c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,787,500 - 1,809,048 nt. (total: 21,549 nt)

<a class="cluster-rules-header" id="r5c2-rules-header" href="#r5c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c2-rules">indole: (indsynth or dmat or indole_PTase)</div>

<div class="region-svg-container">
<div id='r5c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-389"></div> <span class="help-tooltip" id="focus-panel-help-389">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c2-gene-table-details" data-name="r5c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c2-MIBiG-cluster-compare" data-name="r5c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c2-clusterblast" data-name="r5c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c2-knownclusterblast" data-name="r5c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c2-subclusterblast" data-name="r5c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r5c2-pfam-details" data-name="r5c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c2-tigrfam-details" data-name="r5c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-145"></div> <span class="help-tooltip" id="gt-help-tooltip-help-145">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-141"></div> <span class="help-tooltip" id="MIBiG-help-141">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c2"></div>
<div id="comparison-MIBiG-r5c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">indole</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide, NRP</td>

<td>xenoacremone A</td>


<td>Xenoacremonium sp. BF-2018a</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000811: 0-22152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000811/index.html#r1c1" target="_new">BGC0000811</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Alkaloid</td>

<td>fumigaclavine C</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001738: 0-24326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001738/index.html#r1c1" target="_new">BGC0001738</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>phomasetin</td>


<td>Pyrenochaetopsis sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.48, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>flavoglaucin</td>


<td>Aspergillus ruber CBS 135680</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>chaetoglobosins</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Terpene</td>

<td>HEx-tc1 terpene</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.42, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001255: 1908-38951">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001255/index.html#r1c1" target="_new">BGC0001255</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>NRP, Polyketide</td>

<td>equisetin</td>


<td>Fusarium heterosporum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002505: 784-24646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002505/index.html#r1c1" target="_new">BGC0002505</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide, NRP</td>
<td>xenoacremone A</td>
<td>Xenoacremonium sp. BF-2018a</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000811: 0-22152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000811/index.html#r1c1" target="_new">BGC0000811</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Alkaloid</td>
<td>fumigaclavine C</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001738: 0-24326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001738/index.html#r1c1" target="_new">BGC0001738</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP, Polyketide</td>
<td>phomasetin</td>
<td>Pyrenochaetopsis sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.48, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>flavoglaucin</td>
<td>Aspergillus ruber CBS 135680</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>chaetoglobosins</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Terpene</td>
<td>HEx-tc1 terpene</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.42, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001255: 1908-38951">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001255/index.html#r1c1" target="_new">BGC0001255</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>NRP, Polyketide</td>
<td>equisetin</td>
<td>Fusarium heterosporum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-142"></div> <span class="help-tooltip" id="cb-general-help-142">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c2_1.svg">
NW_006271971 (2425175-2516249): Cordyceps militaris CM01 unplaced genomic sca... (13% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c2_2.svg">
NW_024067578 (417738-485177): Penicillium roqueforti strain LCP96 04111 chrom... (8% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r5c2_3.svg">
NW_017253378 (136942-168366): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (50% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r5c2_4.svg">
NW_006917091 (1121274-1194772): Pestalotiopsis fici W106-1 unplaced genomic s... (7% of genes show similarity), NRPS-like,T1PKS,indole
</option>

<option value = "svg/clusterblast_r5c2_5.svg">
NW_025408670 (3613020-3679633): Hirsutella rhossiliensis strain HR02 chromoso... (15% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r5c2_6.svg">
NC_049562 (250751-282212): Talaromyces rugulosus chromosome II, complete sequ... (16% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r5c2_7.svg">
NW_017971432 (1309170-1381501): Talaromyces atroroseus strain IBT 11181 chrom... (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c2_8.svg">
NW_022985018 (1161025-1297869): Dothidotthia symphoricarpi CBS 119687 unplace... (4% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r5c2_9.svg">
NC_054863 (3079018-3155320): Aspergillus puulaauensis MK2 DNA, chromosome 7, ... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c2_10.svg">
NW_006711169 (17423-81332): Trichoderma reesei QM6a unplaced genomic scaffold... (11% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r5c2-download">Download graphic</button>
</div>
<div id="clusterblast-r5c2-svg"></div>

</div>
</div>

<div class="body-details-section r5c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-143"></div> <span class="help-tooltip" id="cb-known-help-143">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-144"></div> <span class="help-tooltip" id="cb-sub-help-144">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r5c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-146"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-146">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-pfam-details-svg" data-filename="CP023325.1_r5c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-147"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-147">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-tigrfam-details-svg" data-filename="CP023325.1_r5c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r5c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023325.1 - Region 3 - isocyanide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-390"></div> <span class="help-tooltip" id="region_svg-help-390">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023325.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c3-svg" data-filename="CP023325.1_r5c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,963,228 - 3,005,268 nt. (total: 42,041 nt)

<a class="cluster-rules-header" id="r5c3-rules-header" href="#r5c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c3-rules">isocyanide: cds(TauD and DIT1_PvcA)</div>

<div class="region-svg-container">
<div id='r5c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-391"></div> <span class="help-tooltip" id="focus-panel-help-391">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c3-gene-table-details" data-name="r5c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c3-MIBiG-cluster-compare" data-name="r5c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c3-clusterblast" data-name="r5c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c3-knownclusterblast" data-name="r5c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c3-subclusterblast" data-name="r5c3-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r5c3-pfam-details" data-name="r5c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-152"></div> <span class="help-tooltip" id="gt-help-tooltip-help-152">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-148"></div> <span class="help-tooltip" id="MIBiG-help-148">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c3"></div>
<div id="comparison-MIBiG-r5c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">isocyanide</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000923: 943-6060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000923/index.html#r1c1" target="_new">BGC0000923</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>paerucumarin</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001594: 0-38842">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001594/index.html#r1c1" target="_new">BGC0001594</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Alkaloid</td>

<td>fischerindole L</td>


<td>Fischerella muscicola UTEX 1829</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Alkaloid</td>

<td>fischerindole</td>


<td>Fischerella sp. SAG 46.79</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.41, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>


<td>Hapalosiphon welwitschii UTEX B 1830</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Alkaloid</td>

<td>ambiguine H isonitrile</td>


<td>Fischerella ambigua UTEX 1903</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>oviedomycin</td>


<td>Streptomyces antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001886: 0-36704">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001886/index.html#r1c1" target="_new">BGC0001886</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>secalonic acids</td>


<td>Claviceps purpurea 20.1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000923: 943-6060">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000923/index.html#r1c1" target="_new">BGC0000923</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>paerucumarin</td>
<td>Pseudomonas aeruginosa PAO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001594: 0-38842">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001594/index.html#r1c1" target="_new">BGC0001594</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Alkaloid</td>
<td>fischerindole L</td>
<td>Fischerella muscicola UTEX 1829</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Alkaloid</td>
<td>fischerindole</td>
<td>Fischerella sp. SAG 46.79</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.41, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>
<td>Hapalosiphon welwitschii UTEX B 1830</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001126: 256-41538">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001126/index.html#r1c1" target="_new">BGC0001126</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole J isonitrile, ambiguine A isonitrile, ambiguine B isonitrile, ambiguine C isonitrile, ambiguine D isonitrile, ambiguine E isonitrile, ambiguine K isonitrile, ambiguine L isonitrile, ambiguine I isonitrile, ambiguine J isonitrile</td>
<td>Fischerella ambigua UTEX 1903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001612: 0-41276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001612/index.html#r1c1" target="_new">BGC0001612</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Alkaloid</td>
<td>ambiguine H isonitrile</td>
<td>Fischerella ambigua UTEX 1903</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>oviedomycin</td>
<td>Streptomyces antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001886: 0-36704">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001886/index.html#r1c1" target="_new">BGC0001886</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>secalonic acids</td>
<td>Claviceps purpurea 20.1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-149"></div> <span class="help-tooltip" id="cb-general-help-149">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c3_1.svg">
NC_054389 (8457870-8566158): Drechmeria coniospora strain ARSEF 6962 chromoso... (28% of genes show similarity), T1PKS,isocyanide
</option>

<option value = "svg/clusterblast_r5c3_2.svg">
NW_014574715 (467040-529181): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (38% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c3_3.svg">
NW_011942149 (5418619-5480759): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (38% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c3_4.svg">
NW_006271969 (5556555-5618595): Cordyceps militaris CM01 unplaced genomic sca... (50% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c3_5.svg">
NC_063070 (2362618-2424749): Purpureocillium takamizusanense chromosome 3, co... (33% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c3_6.svg">
NC_035790 (6741031-6803169): Pochonia chlamydosporia 170 chromosome 1, whole ... (27% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c3_7.svg">
NW_006711178 (114520-207586): Trichoderma reesei QM6a unplaced genomic scaffo... (16% of genes show similarity), isocyanide,terpene
</option>

<option value = "svg/clusterblast_r5c3_8.svg">
NW_025408686 (565240-627429): Hirsutella rhossiliensis strain HR02 chromosome... (35% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c3_9.svg">
NC_057319 (4219732-4281896): Ustilaginoidea virens chromosome 4, complete seq... (14% of genes show similarity), isocyanide
</option>

<option value = "svg/clusterblast_r5c3_10.svg">
NW_024468916 (67569-143215): Cryphonectria parasitica EP155 unplaced genomic ... (16% of genes show similarity), isocyanide,terpene
</option>

</select>
<button id= "clusterblast-r5c3-download">Download graphic</button>
</div>
<div id="clusterblast-r5c3-svg"></div>

</div>
</div>

<div class="body-details-section r5c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-150"></div> <span class="help-tooltip" id="cb-known-help-150">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-151"></div> <span class="help-tooltip" id="cb-sub-help-151">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r5c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-153"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-153">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-pfam-details-svg" data-filename="CP023325.1_r5c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r5c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023325.1 - Region 4 - NRP-metallophore,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-392"></div> <span class="help-tooltip" id="region_svg-help-392">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023325.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c4-svg" data-filename="CP023325.1_r5c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,037,327 - 3,097,372 nt. (total: 60,046 nt)

<a class="cluster-rules-header" id="r5c4-rules-header" href="#r5c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c4-rules">NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-393"></div> <span class="help-tooltip" id="focus-panel-help-393">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c4-gene-table-details" data-name="r5c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c4-nrps_pks" data-name="r5c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c4-MIBiG-cluster-compare" data-name="r5c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c4-clusterblast" data-name="r5c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c4-knownclusterblast" data-name="r5c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c4-subclusterblast" data-name="r5c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c4-nrps-pks-bubbles" data-name="r5c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c4-pfam-details" data-name="r5c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c4-tigrfam-details" data-name="r5c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-162"></div> <span class="help-tooltip" id="gt-help-tooltip-help-162">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-154"></div> <span class="help-tooltip" id="nrps-domain-help-154">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-details-svg" data-filename="CP023325.1_r5c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-155"></div> <span class="help-tooltip" id="MIBiG-help-155">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c4"></div>
<div id="comparison-MIBiG-r5c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.41</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250: 0-14073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-156"></div> <span class="help-tooltip" id="cb-general-help-156">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c4_1.svg">
NW_006711178 (9878-84649): Trichoderma reesei QM6a unplaced genomic scaffold ... (40% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c4_2.svg">
NW_023336198 (10188-84865): Geosmithia morbida isolate 1262 chromosome Unknow... (42% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r5c4_3.svg">
NW_006271969 (5451891-5533992): Cordyceps militaris CM01 unplaced genomic sca... (58% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c4_4.svg">
NC_035790 (6855176-6936375): Pochonia chlamydosporia 170 chromosome 1, whole ... (55% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c4_5.svg">
NC_070545 (3816730-3900832): Fusarium falciforme chromosome 2, complete sequence (52% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c4_6.svg">
NC_070530 (3742001-3826089): Fusarium keratoplasticum isolate Fu6.1 chromosom... (52% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c4_7.svg">
NW_014574715 (339608-420863): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (50% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c4_8.svg">
NW_011942149 (5526781-5608010): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (50% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c4_9.svg">
NC_026476 (2004097-2087970): Fusarium graminearum PH-1 chromosome 3, whole ge... (37% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r5c4_10.svg">
NC_057319 (3036502-3117459): Ustilaginoidea virens chromosome 4, complete seq... (42% of genes show similarity), NRP-metallophore,NRPS
</option>

</select>
<button id= "clusterblast-r5c4-download">Download graphic</button>
</div>
<div id="clusterblast-r5c4-svg"></div>

</div>
</div>

<div class="body-details-section r5c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-157"></div> <span class="help-tooltip" id="cb-known-help-157">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-158"></div> <span class="help-tooltip" id="cb-sub-help-158">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-161"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-161">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-domain-bubble-svg" data-filename="CP023325.1_r5c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (3037326 - 3097372): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r5c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-163"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-163">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-pfam-details-svg" data-filename="CP023325.1_r5c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-164"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-164">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c4-tigrfam-details-svg" data-filename="CP023325.1_r5c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c4-nrps_pks_products" data-name="r5c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c4-nrps_pks_monomers" data-name="r5c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-159"></div> <span class="help-tooltip" id="nrps-structures-help-159">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 3037326 - 3097372:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c4-4" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-160"></div> <span class="help-tooltip" id="nrps-monomers-help-160">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene792</span></strong>: X
<div class="collapser collapser-target-input-path1-gene792 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (29..284)</strong>: X
<div class="collapser collapser-target-input-path1-gene792 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-input-path1-gene792-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFEMI-ILK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Cys <span class="serif">DLFEMSLIWK</span> (32% 8Å match)</dd>
<dd>Ala <span class="serif">DVFIYAAILK</span> (29% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">input.path1.gene794</span></strong>: X - X
<div class="collapser collapser-target-input-path1-gene794 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (34..307)</strong>: X
<div class="collapser collapser-target-input-path1-gene794 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-input-path1-gene794-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFSVAGNIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DVFSVALVYK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>









<dd><strong>AMP-binding (1341..1739)</strong>: X
<div class="collapser collapser-target-input-path1-gene794 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-input-path1-gene794-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVLDIGAIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DVLSFGAIIK</span> (65% 8Å match)</dd>
<dd>Phe or Trp <span class="serif">DVLSIGAVVK</span> (62% 8Å match)</dd>
<dd>Arg <span class="serif">DVESIGAIAK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>














</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023325.1 - Region 5 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-394"></div> <span class="help-tooltip" id="region_svg-help-394">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023325.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c5-svg" data-filename="CP023325.1_r5c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,688,839 - 3,737,299 nt. (total: 48,461 nt)

<a class="cluster-rules-header" id="r5c5-rules-header" href="#r5c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c5-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-395"></div> <span class="help-tooltip" id="focus-panel-help-395">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c5-gene-table-details" data-name="r5c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c5-nrps_pks" data-name="r5c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c5-MIBiG-cluster-compare" data-name="r5c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c5-clusterblast" data-name="r5c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c5-knownclusterblast" data-name="r5c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c5-subclusterblast" data-name="r5c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c5-pfam-details" data-name="r5c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c5-tigrfam-details" data-name="r5c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-171"></div> <span class="help-tooltip" id="gt-help-tooltip-help-171">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-165"></div> <span class="help-tooltip" id="nrps-domain-help-165">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c5-details-svg" data-filename="CP023325.1_r5c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-166"></div> <span class="help-tooltip" id="MIBiG-help-166">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c5"></div>
<div id="comparison-MIBiG-r5c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>hydroxycinnamoyltyramine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001304: 0-15094">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001304/index.html#r1c1" target="_new">BGC0001304</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>aflavarin</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>hydroxycinnamoyltyramine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001304: 0-15094">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001304/index.html#r1c1" target="_new">BGC0001304</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>aflavarin</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-167"></div> <span class="help-tooltip" id="cb-general-help-167">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c5_1.svg">
NW_006271969 (4860304-4927848): Cordyceps militaris CM01 unplaced genomic sca... (55% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_2.svg">
NW_021167090 (669974-737132): Sodiomyces alkalinus F11 unplaced genomic scaff... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_3.svg">
NW_022985016 (85063-152593): Dothidotthia symphoricarpi CBS 119687 unplaced g... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_4.svg">
NC_049565 (1715696-1783101): Talaromyces rugulosus chromosome V, complete seq... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_5.svg">
NW_006271969 (1967760-2079673): Cordyceps militaris CM01 unplaced genomic sca... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c5_6.svg">
NC_018213 (82527-151813): Zymoseptoria tritici IPO323 chromosome 6, whole gen... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_7.svg">
NW_006912904 (3671749-3746762): Capronia epimyces CBS 606.96 aczIQ-supercont1... (9% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r5c5_8.svg">
NT_165977 (3031183-3103199): Chaetomium globosum CBS 148.51 scaffold 2 genomi... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_9.svg">
NW_024467841 (224613-292322): Aspergillus wentii DTO 134E9 unplaced genomic s... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c5_10.svg">
NW_024467860 (3001170-3069516): Aspergillus sydowii CBS 593.65 unplaced genom... (7% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r5c5-download">Download graphic</button>
</div>
<div id="clusterblast-r5c5-svg"></div>

</div>
</div>

<div class="body-details-section r5c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-168"></div> <span class="help-tooltip" id="cb-known-help-168">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-169"></div> <span class="help-tooltip" id="cb-sub-help-169">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-172"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-172">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c5-pfam-details-svg" data-filename="CP023325.1_r5c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-173"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-173">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c5-tigrfam-details-svg" data-filename="CP023325.1_r5c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c5-nrps_pks_monomers" data-name="r5c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r5c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-170"></div> <span class="help-tooltip" id="nrps-monomers-help-170">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene967</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene967 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (579..904)</strong>: pk
<div class="collapser collapser-target-input-path1-gene967 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene967-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>2-Methylbutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene967-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 83.6</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 78.6</dt>

<dd></dd><dt>Malonyl-CoA: 58.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 56.8</dt>

<dd></dd><dt>Propionyl-CoA: 29.1</dt>

<dd></dd><dt>Benzoyl-CoA: 23.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 21.1</dt>

<dd></dd><dt>fatty_acid: 15.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 15.0</dt>

<dd></dd><dt>CHC-CoA: 12.4</dt>

<dd></dd><dt>inactive: 11.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 9.5</dt>

<dd></dd><dt>Acetyl-CoA: 7.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: -3.4</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r5c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023325.1 - Region 6 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-396"></div> <span class="help-tooltip" id="region_svg-help-396">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023325.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c6-svg" data-filename="CP023325.1_r5c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,455,302 - 4,524,605 nt. (total: 69,304 nt)

<a class="cluster-rules-header" id="r5c6-rules-header" href="#r5c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c6-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-397"></div> <span class="help-tooltip" id="focus-panel-help-397">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c6-gene-table-details" data-name="r5c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c6-nrps_pks" data-name="r5c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c6-MIBiG-cluster-compare" data-name="r5c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c6-clusterblast" data-name="r5c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c6-knownclusterblast" data-name="r5c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c6-subclusterblast" data-name="r5c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c6-nrps-pks-bubbles" data-name="r5c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c6-pfam-details" data-name="r5c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c6-tigrfam-details" data-name="r5c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-182"></div> <span class="help-tooltip" id="gt-help-tooltip-help-182">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-174"></div> <span class="help-tooltip" id="nrps-domain-help-174">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c6-details-svg" data-filename="CP023325.1_r5c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-175"></div> <span class="help-tooltip" id="MIBiG-help-175">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c6"></div>
<div id="comparison-MIBiG-r5c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>





<td class="cc-heat-cell" title="0.83 (id:0.64, order:1.00, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.63, order:0.76, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.82</td>
<td>Polyketide, NRP</td>

<td>beauveriolide I, beauveriolide III</td>


<td>Cordyceps militaris CM01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002203: 0-33767">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002203/index.html#r1c1" target="_new">BGC0002203</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.56, order:0.69, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.56, order:0.57, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.55</td>
<td>NRP, Polyketide, Other</td>

<td>beauveriolide B, beauveriolide C, beauveriolide D</td>


<td>Beauveria bassiana ARSEF 2860</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001290: 0-37424">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001290/index.html#r1c1" target="_new">BGC0001290</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.48, order:0.58, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.49, order:0.64, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.44</td>
<td>NRP</td>

<td>emericellamide A, emericellamide B</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.41</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.38</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.38</td>
<td>Polyketide</td>

<td>naphthalene</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>


<td class="cc-heat-cell" title="0.83 (id:0.64, order:1.00, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
<td>0.83</td>
<td>Polyketide, NRP</td>
<td>beauveriolide I, beauveriolide III</td>
<td>Cordyceps militaris CM01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002203: 0-33767">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002203/index.html#r1c1" target="_new">BGC0002203</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.56, order:0.69, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP, Polyketide, Other</td>
<td>beauveriolide B, beauveriolide C, beauveriolide D</td>
<td>Beauveria bassiana ARSEF 2860</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001290: 0-37424">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001290/index.html#r1c1" target="_new">BGC0001290</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.48, order:0.58, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>emericellamide A, emericellamide B</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-176"></div> <span class="help-tooltip" id="cb-general-help-176">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c6_1.svg">
NW_006271969 (4087046-4175862): Cordyceps militaris CM01 unplaced genomic sca... (64% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_2.svg">
NW_025091271 (18416-119910): Aspergillus udagawae IFM 46973 DNA, scaffold 8, ... (21% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_3.svg">
NC_066263 (4200452-4292984): Aspergillus nidulans FGSC A4 chromosome VII (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_4.svg">
NT_165979 (2272028-2354621): Chaetomium globosum CBS 148.51 scaffold 4 genomi... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_5.svg">
NC_038012 (6505506-6611069): Fusarium venenatum strain A3/5 genome assembly, ... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_6.svg">
NC_058399 (6406982-6512577): Fusarium poae strain DAOMC 252244 chromosome 1, ... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_7.svg">
NC_072354 (38624-176736): Talaromyces marneffei chromosome 7, complete sequence (12% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_8.svg">
NC_054859 (4418286-4502506): Aspergillus puulaauensis MK2 DNA, chromosome 3, ... (12% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_9.svg">
NW_022985363 (1607118-1709581): Macroventuria anomochaeta strain CBS 525.71 u... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c6_10.svg">
NW_025091283 (359926-455746): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (7% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r5c6-download">Download graphic</button>
</div>
<div id="clusterblast-r5c6-svg"></div>

</div>
</div>

<div class="body-details-section r5c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-177"></div> <span class="help-tooltip" id="cb-known-help-177">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r5c6-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r5c6_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r5c6_1.svg">
beauveriolide I/beauveriolide III
</option>

<option value = "svg/knownclusterblast_r5c6_2.svg">
beauveriolide B/beauveriolide C/beauveriolide D
</option>

<option value = "svg/knownclusterblast_r5c6_3.svg">
emericellamide A/emericellamide B
</option>

</select>
<button id="knownclusterblast-r5c6-download">Download graphic</button>
</div>
<div id="knownclusterblast-r5c6-svg"></div>

</div>
</div>

<div class="body-details-section r5c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-178"></div> <span class="help-tooltip" id="cb-sub-help-178">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-181"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-181">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c6-domain-bubble-svg" data-filename="CP023325.1_r5c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (4455301 - 4524605): neighbouring NRPS-T1PKS
</option>



<option value="CC7">
Candidate 7 (4455301 - 4510608): single NRPS
</option>



<option value="CC8">
Candidate 8 (4477018 - 4524605): single T1PKS
</option>


</select>

<div class="details-svg" id="r5c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c6-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-183"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-183">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c6-pfam-details-svg" data-filename="CP023325.1_r5c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-184"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-184">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c6-tigrfam-details-svg" data-filename="CP023325.1_r5c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c6-nrps_pks_products" data-name="r5c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c6-nrps_pks_monomers" data-name="r5c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-179"></div> <span class="help-tooltip" id="nrps-structures-help-179">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 4455301 - 4524605:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c6-6" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Ala - D-Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala,d-leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,ala*,d-leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c6-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 7, location 4455301 - 4510608:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c6-7" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Ala - D-Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala,d-leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,ala*,d-leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c6-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 8, location 4477018 - 4524605:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c6-8" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Ala - D-Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala,d-leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,ala*,d-leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c6-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-180"></div> <span class="help-tooltip" id="nrps-monomers-help-180">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1194</span></strong>: X - Ala - Leu
<div class="collapser collapser-target-input-path1-gene1194 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala,leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*,leu*" target="_new">relaxed</a>
</dd>
<br>







<dd><strong>AMP-binding (532..807)</strong>: X
<div class="collapser collapser-target-input-path1-gene1194 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1194-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAFLMIGIIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DAYLMIGIIK</span> (97% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1824..2214)</strong>: Ala
<div class="collapser collapser-target-input-path1-gene1194 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-input-path1-gene1194-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLGIVGGMIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DLGIVGGMIK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2919..3310)</strong>: Leu
<div class="collapser collapser-target-input-path1-gene1194 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-input-path1-gene1194-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Cyclic aliphatic chain (polar NH2 group) (Pro, Hpr)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMHMLGVIIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DMHMLGVIIK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">input.path1.gene1197</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene1197 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (479..798)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1197 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1197-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
<dd>inactive: 54.2%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1197-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 72.6</dt>

<dd></dd><dt>Methylmalonyl-CoA: 68.8</dt>

<dd></dd><dt>Malonyl-CoA: 46.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 41.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 35.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.9</dt>

<dd></dd><dt>Propionyl-CoA: 23.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.7</dt>

<dd></dd><dt>fatty_acid: 16.2</dt>

<dd></dd><dt>Benzoyl-CoA: 15.9</dt>

<dd></dd><dt>CHC-CoA: 10.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 10.7</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023325.1 - Region 7 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-398"></div> <span class="help-tooltip" id="region_svg-help-398">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023325.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c7-svg" data-filename="CP023325.1_r5c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,705,473 - 5,766,444 nt. (total: 60,972 nt)

<a class="cluster-rules-header" id="r5c7-rules-header" href="#r5c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c7-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-399"></div> <span class="help-tooltip" id="focus-panel-help-399">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c7-gene-table-details" data-name="r5c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c7-nrps_pks" data-name="r5c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c7-MIBiG-cluster-compare" data-name="r5c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c7-clusterblast" data-name="r5c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c7-knownclusterblast" data-name="r5c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c7-subclusterblast" data-name="r5c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c7-nrps-pks-bubbles" data-name="r5c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c7-pfam-details" data-name="r5c7-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-193"></div> <span class="help-tooltip" id="gt-help-tooltip-help-193">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-185"></div> <span class="help-tooltip" id="nrps-domain-help-185">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c7-details-svg" data-filename="CP023325.1_r5c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-186"></div> <span class="help-tooltip" id="MIBiG-help-186">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c7"></div>
<div id="comparison-MIBiG-r5c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.60</td>
<td>Polyketide</td>

<td>6-MSA, aculin A</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.58</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.58</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.57</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.51</td>
<td>Polyketide</td>

<td>pyripyropene A</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000120: 194-40016">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000120/index.html#r1c1" target="_new">BGC0000120</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.49</td>
<td>Polyketide</td>

<td>patulin</td>


<td>Penicillium expansum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.47</td>
<td>Polyketide</td>

<td>yanuthone D</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.46</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>Terpene</td>

<td>lupeol</td>


<td>Lotus japonicus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>6-MSA, aculin A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000129: 0-23079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000129/index.html#r1c1" target="_new">BGC0000129</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>pyripyropene A</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000120: 194-40016">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000120/index.html#r1c1" target="_new">BGC0000120</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>patulin</td>
<td>Penicillium expansum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>yanuthone D</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002644: 0-7903">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002644/index.html#r1c1" target="_new">BGC0002644</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other, Terpene</td>
<td>domoic acid, isodomoic acid A, isodomoic acid B, isodomoic acid C, 7&#39;-carboxy-L-NGG, prekainic acid, kainic acid</td>
<td>Chondria armata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-187"></div> <span class="help-tooltip" id="cb-general-help-187">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c7_1.svg">
NW_006271971 (3503471-3590640): Cordyceps militaris CM01 unplaced genomic sca... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c7_2.svg">
NW_020291625 (99659-196841): Aspergillus piperis CBS 112811 unplaced genomic ... (14% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c7_3.svg">
NW_020292069 (316400-408453): Aspergillus vadensis CBS 113365 unplaced genomi... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c7_4.svg">
NW_023336287 (1029426-1127137): Aspergillus tubingensis WU-2223L DNA, scaffol... (14% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c7_5.svg">
NW_020291816 (340965-433020): Aspergillus costaricaensis CBS 115574 unplaced ... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c7_6.svg">
NW_020290887 (673812-764439): Aspergillus niger CBS 101883 unplaced genomic s... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r5c7_7.svg">
NW_013581207 (483365-550922): Aureobasidium namibiae CBS 147.97 unplaced geno... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c7_8.svg">
NW_013566983 (689939-757505): Aureobasidium subglaciale EXF-2481 unplaced gen... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r5c7_9.svg">
NC_054853 (2119459-2216438): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (12% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r5c7_10.svg">
NW_022474207 (2338451-2410505): Venustampulla echinocandica strain BP 5553 ch... (14% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r5c7-download">Download graphic</button>
</div>
<div id="clusterblast-r5c7-svg"></div>

</div>
</div>

<div class="body-details-section r5c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-188"></div> <span class="help-tooltip" id="cb-known-help-188">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-189"></div> <span class="help-tooltip" id="cb-sub-help-189">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-192"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-192">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c7-domain-bubble-svg" data-filename="CP023325.1_r5c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC9">
Candidate 9 (5705472 - 5766444): neighbouring T1PKS-NRPS
</option>



<option value="CC10">
Candidate 10 (5705472 - 5752929): single T1PKS
</option>



<option value="CC11">
Candidate 11 (5719095 - 5766444): single NRPS
</option>


</select>

<div class="details-svg" id="r5c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-194"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-194">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c7-pfam-details-svg" data-filename="CP023325.1_r5c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c7-nrps_pks_products" data-name="r5c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c7-nrps_pks_monomers" data-name="r5c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-190"></div> <span class="help-tooltip" id="nrps-structures-help-190">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 9, location 5705472 - 5766444:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c7-9" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c7-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 10, location 5705472 - 5752929:
<div class="collapser collapser-target-CC10 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c7-10" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c7-10-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 11, location 5719095 - 5766444:
<div class="collapser collapser-target-CC11 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c7-11" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c7-11-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-191"></div> <span class="help-tooltip" id="nrps-monomers-help-191">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1515</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene1515 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (566..884)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1515 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1515-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1515-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 67.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 60.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 50.2</dt>

<dd></dd><dt>Malonyl-CoA: 42.8</dt>

<dd></dd><dt>fatty_acid: 29.4</dt>

<dd></dd><dt>Propionyl-CoA: 27.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 16.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 15.4</dt>

<dd></dd><dt>Benzoyl-CoA: 14.2</dt>

<dd></dd><dt>Acetyl-CoA: 13.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 6.4</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">input.path1.gene1519</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1519 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (205..645)</strong>: X
<div class="collapser collapser-target-input-path1-gene1519 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1519-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALTVGGIVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DALAVGEVVK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r6c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-400"></div> <span class="help-tooltip" id="region_svg-help-400">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c1-svg" data-filename="CP023323.1_r6c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 822,578 - 866,868 nt. (total: 44,291 nt)

<a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-401"></div> <span class="help-tooltip" id="focus-panel-help-401">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c1-gene-table-details" data-name="r6c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c1-tigrfam-details" data-name="r6c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-201"></div> <span class="help-tooltip" id="gt-help-tooltip-help-201">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-195"></div> <span class="help-tooltip" id="nrps-domain-help-195">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="CP023323.1_r6c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-196"></div> <span class="help-tooltip" id="MIBiG-help-196">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c1"></div>
<div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>pyrrolizixenamide A</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>pyrrolizixenamide A</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-197"></div> <span class="help-tooltip" id="cb-general-help-197">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c1_1.svg">
NW_006271970 (4654617-4718544): Cordyceps militaris CM01 unplaced genomic sca... (61% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_2.svg">
NW_024468935 (6362317-6425953): Cucurbitaria berberidis CBS 394.84 unplaced g... (14% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_3.svg">
NW_017306191 (726629-790263): Alternaria alternata strain SRC1lrK2f unplaced ... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_4.svg">
NW_024468910 (2525048-2615941): Cryphonectria parasitica EP155 unplaced genom... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c1_5.svg">
NC_064655 (797649-865606): Penicillium oxalicum strain HP7-1 chromosome VI, w... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c1_6.svg">
NW_024067573 (273246-341219): Penicillium roqueforti strain LCP96 04111 chrom... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c1_7.svg">
NC_030958 (2311099-2374368): Colletotrichum higginsianum IMI 349063 chromosom... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_8.svg">
NW_026054555 (234652-296885): Daldinia decipiens strain CBS 113046 unplaced g... (8% of genes show similarity), isocyanide
</option>

</select>
<button id= "clusterblast-r6c1-download">Download graphic</button>
</div>
<div id="clusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-198"></div> <span class="help-tooltip" id="cb-known-help-198">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-199"></div> <span class="help-tooltip" id="cb-sub-help-199">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-202"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-202">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="CP023323.1_r6c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-203"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-203">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-tigrfam-details-svg" data-filename="CP023323.1_r6c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c1-nrps_pks_monomers" data-name="r6c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r6c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-200"></div> <span class="help-tooltip" id="nrps-monomers-help-200">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene213</span></strong>: Aad
<div class="collapser collapser-target-input-path1-gene213 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (22..472)</strong>: Aad
<div class="collapser collapser-target-input-path1-gene213 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Aad
<div class="collapser collapser-target-input-path1-gene213-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPRHFV-RAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Aad <span class="serif">DPRHFVMRAK</span> (71% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r6c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-402"></div> <span class="help-tooltip" id="region_svg-help-402">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c2-svg" data-filename="CP023323.1_r6c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,344,291 - 1,400,349 nt. (total: 56,059 nt)

<a class="cluster-rules-header" id="r6c2-rules-header" href="#r6c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-403"></div> <span class="help-tooltip" id="focus-panel-help-403">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c2-gene-table-details" data-name="r6c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c2-nrps_pks" data-name="r6c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c2-MIBiG-cluster-compare" data-name="r6c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c2-clusterblast" data-name="r6c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c2-knownclusterblast" data-name="r6c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c2-subclusterblast" data-name="r6c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c2-nrps-pks-bubbles" data-name="r6c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c2-pfam-details" data-name="r6c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c2-tigrfam-details" data-name="r6c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-212"></div> <span class="help-tooltip" id="gt-help-tooltip-help-212">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-204"></div> <span class="help-tooltip" id="nrps-domain-help-204">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-details-svg" data-filename="CP023323.1_r6c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-205"></div> <span class="help-tooltip" id="MIBiG-help-205">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c2"></div>
<div id="comparison-MIBiG-r6c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>oxepinamide D</td>


<td>Aspergillus ustus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>oxepinamide D</td>
<td>Aspergillus ustus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-206"></div> <span class="help-tooltip" id="cb-general-help-206">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c2_1.svg">
NW_006271970 (4182342-4255710): Cordyceps militaris CM01 unplaced genomic sca... (57% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_2.svg">
NW_021167094 (207435-275256): Sodiomyces alkalinus F11 unplaced genomic scaff... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_3.svg">
NW_011942171 (5162432-5375378): Metarhizium robertsii ARSEF 23 MAA Scf 3, who... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c2_4.svg">
NW_017971433 (2380693-2447085): Talaromyces atroroseus strain IBT 11181 chrom... (30% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_5.svg">
NW_025091283 (359926-455746): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c2_6.svg">
NW_025091266 (1102530-1174184): Aspergillus udagawae IFM 46973 DNA, scaffold ... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_7.svg">
NW_011942151 (1102940-1173579): Metarhizium robertsii ARSEF 23 MAA Scf 11, wh... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_8.svg">
NC_035790 (380514-527376): Pochonia chlamydosporia 170 chromosome 1, whole ge... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_9.svg">
NW_024467526 (7938-201848): Aspergillus versicolor CBS 583.65 unplaced genomi... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c2_10.svg">
NW_024468995 (361260-526376): Aspergillus fijiensis CBS 313.89 unplaced genom... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c2-download">Download graphic</button>
</div>
<div id="clusterblast-r6c2-svg"></div>

</div>
</div>

<div class="body-details-section r6c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-207"></div> <span class="help-tooltip" id="cb-known-help-207">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-208"></div> <span class="help-tooltip" id="cb-sub-help-208">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-211"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-211">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-domain-bubble-svg" data-filename="CP023323.1_r6c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (1344290 - 1400349): single NRPS
</option>


</select>

<div class="details-svg" id="r6c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-213"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-213">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-pfam-details-svg" data-filename="CP023323.1_r6c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-214"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-214">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-tigrfam-details-svg" data-filename="CP023323.1_r6c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c2-nrps_pks_products" data-name="r6c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c2-nrps_pks_monomers" data-name="r6c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-209"></div> <span class="help-tooltip" id="nrps-structures-help-209">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 1344290 - 1400349:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - Leu - Ala - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,leu*,ala*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-210"></div> <span class="help-tooltip" id="nrps-monomers-help-210">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene327</span></strong>: X - Leu - Ala - X
<div class="collapser collapser-target-input-path1-gene327 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,ala*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (38..442)</strong>: X
<div class="collapser collapser-target-input-path1-gene327 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-input-path1-gene327-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVMEVGTPTK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Å match)</dd>
<dd>Orn <span class="serif">DVMELSSITK</span> (47% 8Å match)</dd>
<dd>Pro <span class="serif">DVMLVGAVNK</span> (44% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1127..1526)</strong>: Leu
<div class="collapser collapser-target-input-path1-gene327 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-input-path1-gene327-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>leu (Leu)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALDTGMATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (71% 8Å match)</dd>
<dd>Phe or Trp <span class="serif">DAMIGGMVTK</span> (47% 8Å match)</dd>
<dd>Leu <span class="serif">DALYLGMMCK</span> (44% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2216..2614)</strong>: Ala
<div class="collapser collapser-target-input-path1-gene327 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-input-path1-gene327-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLCLHIMTFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3289..3691)</strong>: X
<div class="collapser collapser-target-input-path1-gene327 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene327-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALDTGMATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVSDTGAPTK</span> (68% 8Å match)</dd>
<dd>Phe or Trp <span class="serif">DAMIGGMVTK</span> (50% 8Å match)</dd>
<dd>Leu <span class="serif">DALYLGMMCK</span> (47% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 3 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-404"></div> <span class="help-tooltip" id="region_svg-help-404">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c3-svg" data-filename="CP023323.1_r6c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,754,981 - 1,816,068 nt. (total: 61,088 nt)

<a class="cluster-rules-header" id="r6c3-rules-header" href="#r6c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c3-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r6c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-405"></div> <span class="help-tooltip" id="focus-panel-help-405">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c3-gene-table-details" data-name="r6c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c3-MIBiG-cluster-compare" data-name="r6c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c3-clusterblast" data-name="r6c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c3-knownclusterblast" data-name="r6c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c3-subclusterblast" data-name="r6c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c3-pfam-details" data-name="r6c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c3-tigrfam-details" data-name="r6c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-219"></div> <span class="help-tooltip" id="gt-help-tooltip-help-219">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-215"></div> <span class="help-tooltip" id="MIBiG-help-215">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c3"></div>
<div id="comparison-MIBiG-r6c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.44, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001423: 0-41128">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001423/index.html#r1c1" target="_new">BGC0001423</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide C</td>


<td>Myxococcus hansupus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001421: 0-47099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001421/index.html#r1c1" target="_new">BGC0001421</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide B</td>


<td>Myxococcus sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001424: 0-41589">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001424/index.html#r1c1" target="_new">BGC0001424</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide C</td>


<td>Myxococcus virescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001428: 0-44682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001428/index.html#r1c1" target="_new">BGC0001428</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001426: 0-44134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001426/index.html#r1c1" target="_new">BGC0001426</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001427: 0-44152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001427/index.html#r1c1" target="_new">BGC0001427</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001425: 0-44135">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001425/index.html#r1c1" target="_new">BGC0001425</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.44, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>NRP, Polyketide</td>

<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>


<td>Fusarium heterosporum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001422: 0-42748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001422/index.html#r1c1" target="_new">BGC0001422</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.47, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus fulvus HW-1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.44, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001423: 0-41128">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001423/index.html#r1c1" target="_new">BGC0001423</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide C</td>
<td>Myxococcus hansupus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001421: 0-47099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001421/index.html#r1c1" target="_new">BGC0001421</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide B</td>
<td>Myxococcus sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001424: 0-41589">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001424/index.html#r1c1" target="_new">BGC0001424</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide C</td>
<td>Myxococcus virescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001428: 0-44682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001428/index.html#r1c1" target="_new">BGC0001428</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001426: 0-44134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001426/index.html#r1c1" target="_new">BGC0001426</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001427: 0-44152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001427/index.html#r1c1" target="_new">BGC0001427</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001425: 0-44135">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001425/index.html#r1c1" target="_new">BGC0001425</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.47, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000992: 667-59922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000992/index.html#r1c1" target="_new">BGC0000992</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.44, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>NRP, Polyketide</td>
<td>fusaridione A, (3R,5S)-5-(4-hydroxybenzyl)-3-methyl-3-((2E,4E,6E,8E,10E)-4,8,10-trimethyldodeca-2,4,6,8,10-pentaenoyl)pyrrolidine-2,4-dione</td>
<td>Fusarium heterosporum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001422: 0-42748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001422/index.html#r1c1" target="_new">BGC0001422</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.47, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus fulvus HW-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-216"></div> <span class="help-tooltip" id="cb-general-help-216">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c3_1.svg">
NW_006271970 (3761055-3936010): Cordyceps militaris CM01 unplaced genomic sca... (25% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_2.svg">
NC_035794 (1709791-1793960): Pochonia chlamydosporia 170 chromosome 5, whole ... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c3_3.svg">
NW_024468978 (129841-221990): Aspergillus fijiensis CBS 313.89 unplaced genom... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_4.svg">
NC_054856 (3727884-3894446): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (3% of genes show similarity), NI-siderophore,NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_5.svg">
NW_023336279 (145289-236294): Aspergillus tubingensis WU-2223L DNA, scaffold ... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_6.svg">
NW_020292053 (616043-708384): Aspergillus vadensis CBS 113365 unplaced genomi... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_7.svg">
NW_014574622 (178902-269178): Penicillium digitatum Pd1 unplaced genomic scaf... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_8.svg">
NW_020290899 (145522-237868): Aspergillus niger CBS 101883 unplaced genomic s... (6% of genes show similarity), NI-siderophore,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_9.svg">
NW_017971443 (180850-345227): Talaromyces atroroseus strain IBT 11181 chromos... (3% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c3_10.svg">
NW_017253361 (212602-309441): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (11% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c3-download">Download graphic</button>
</div>
<div id="clusterblast-r6c3-svg"></div>

</div>
</div>

<div class="body-details-section r6c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-217"></div> <span class="help-tooltip" id="cb-known-help-217">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-218"></div> <span class="help-tooltip" id="cb-sub-help-218">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-220"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-220">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-pfam-details-svg" data-filename="CP023323.1_r6c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-221"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-221">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-tigrfam-details-svg" data-filename="CP023323.1_r6c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r6c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 4 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-406"></div> <span class="help-tooltip" id="region_svg-help-406">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c4-svg" data-filename="CP023323.1_r6c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,135,448 - 2,156,930 nt. (total: 21,483 nt)

<a class="cluster-rules-header" id="r6c4-rules-header" href="#r6c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c4-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r6c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-407"></div> <span class="help-tooltip" id="focus-panel-help-407">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c4-gene-table-details" data-name="r6c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c4-MIBiG-cluster-compare" data-name="r6c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c4-clusterblast" data-name="r6c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c4-knownclusterblast" data-name="r6c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c4-subclusterblast" data-name="r6c4-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r6c4-pfam-details" data-name="r6c4-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-226"></div> <span class="help-tooltip" id="gt-help-tooltip-help-226">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-222"></div> <span class="help-tooltip" id="MIBiG-help-222">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c4"></div>
<div id="comparison-MIBiG-r6c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>betaestacin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>preaspterpenacid I</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685: 127-25775">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>brassicicene C</td>


<td>Alternaria brassicicola ATCC 96836</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001604: 5569-24234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001604/index.html#r1c1" target="_new">BGC0001604</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Terpene</td>

<td>gibberellin</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002651: 0-16858">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002651/index.html#r1c1" target="_new">BGC0002651</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Terpene</td>

<td>sordaricin B</td>


<td>Rosellinia necatrix</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Terpene</td>

<td>citreohybridonol</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Terpene</td>

<td>marneral</td>


<td>Arabidopsis thaliana</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>preaspterpenacid I</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685: 127-25775">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>brassicicene C</td>
<td>Alternaria brassicicola ATCC 96836</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001604: 5569-24234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001604/index.html#r1c1" target="_new">BGC0001604</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>gibberellin</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002651: 0-16858">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002651/index.html#r1c1" target="_new">BGC0002651</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene</td>
<td>sordaricin B</td>
<td>Rosellinia necatrix</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001547: 100-20696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001547/index.html#r1c1" target="_new">BGC0001547</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>citreohybridonol</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-223"></div> <span class="help-tooltip" id="cb-general-help-223">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c4_1.svg">
NW_006271970 (3541960-3572703): Cordyceps militaris CM01 unplaced genomic sca... (87% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c4_2.svg">
NW_016017385 (214033-282604): Sphaerulina musiva SO2202 unplaced genomic scaf... (20% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r6c4_3.svg">
NW_019154049 (138062-255179): Pochonia chlamydosporia 170 chromosome Unknown ... (9% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r6c4_4.svg">
NC_036441 (277745-308902): Aspergillus oryzae RIB40 DNA, chromosome 7 (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c4_5.svg">
NC_054697 (293793-324950): Aspergillus flavus NRRL3357 chromosome 7, complete... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c4_6.svg">
NW_017306216 (296572-327678): Alternaria alternata strain SRC1lrK2f unplaced ... (42% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c4_7.svg">
NW_013566992 (141321-209375): Aureobasidium subglaciale EXF-2481 unplaced gen... (17% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c4_8.svg">
NW_025953826 (180766-254516): Annulohypoxylon truncatum strain CBS 140777 unp... (19% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c4_9.svg">
NW_024468936 (1367361-1398427): Cucurbitaria berberidis CBS 394.84 unplaced g... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r6c4_10.svg">
NW_024468918 (1133295-1261954): Cryphonectria parasitica EP155 unplaced genom... (9% of genes show similarity), T1PKS,terpene
</option>

</select>
<button id= "clusterblast-r6c4-download">Download graphic</button>
</div>
<div id="clusterblast-r6c4-svg"></div>

</div>
</div>

<div class="body-details-section r6c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-224"></div> <span class="help-tooltip" id="cb-known-help-224">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c4_1.svg">
PR-toxin
</option>

</select>
<button id="knownclusterblast-r6c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c4-svg"></div>

</div>
</div>

<div class="body-details-section r6c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-225"></div> <span class="help-tooltip" id="cb-sub-help-225">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r6c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-227"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-227">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c4-pfam-details-svg" data-filename="CP023323.1_r6c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r6c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 5 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-408"></div> <span class="help-tooltip" id="region_svg-help-408">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c5-svg" data-filename="CP023323.1_r6c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,271,609 - 3,315,469 nt. (total: 43,861 nt)

<a class="cluster-rules-header" id="r6c5-rules-header" href="#r6c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c5-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-409"></div> <span class="help-tooltip" id="focus-panel-help-409">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c5-gene-table-details" data-name="r6c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c5-nrps_pks" data-name="r6c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c5-MIBiG-cluster-compare" data-name="r6c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c5-clusterblast" data-name="r6c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c5-knownclusterblast" data-name="r6c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c5-subclusterblast" data-name="r6c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c5-nrps-pks-bubbles" data-name="r6c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c5-pfam-details" data-name="r6c5-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c5-tigrfam-details" data-name="r6c5-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-236"></div> <span class="help-tooltip" id="gt-help-tooltip-help-236">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-228"></div> <span class="help-tooltip" id="nrps-domain-help-228">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-details-svg" data-filename="CP023323.1_r6c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-229"></div> <span class="help-tooltip" id="MIBiG-help-229">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c5"></div>
<div id="comparison-MIBiG-r6c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>





<td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>


<td>0.43</td>
<td>NRP</td>

<td>choline</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308: 0-7851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>aureusimine A, aureusimine B, aureusimine C</td>


<td>Staphylococcus aureus subsp. aureus str. JKD6008</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>


<td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
<td>0.74</td>
<td>NRP</td>
<td>choline</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308: 0-7851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>aureusimine A, aureusimine B, aureusimine C</td>
<td>Staphylococcus aureus subsp. aureus str. JKD6008</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-230"></div> <span class="help-tooltip" id="cb-general-help-230">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c5_1.svg">
NW_006271970 (2419165-2483026): Cordyceps militaris CM01 unplaced genomic sca... (52% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c5_2.svg">
NW_014013657 (238627-342899): Trichoderma virens Gv29-8 chromosome Unknown TR... (17% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c5_3.svg">
NC_054390 (745744-925575): Drechmeria coniospora strain ARSEF 6962 chromosome... (8% of genes show similarity), NRPS,NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c5_4.svg">
NW_006711169 (17423-81332): Trichoderma reesei QM6a unplaced genomic scaffold... (23% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c5_5.svg">
NW_025764707 (691343-755195): Emericellopsis atlantica unplaced genomic scaff... (20% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c5_6.svg">
NC_070532 (4966088-5118746): Fusarium keratoplasticum isolate Fu6.1 chromosom... (9% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c5_7.svg">
NC_070547 (4757164-4894085): Fusarium falciforme chromosome 4, complete sequence (9% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c5_8.svg">
NW_011942160 (555242-619088): Metarhizium robertsii ARSEF 23 MAA Scf 2, whole... (20% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c5_9.svg">
NW_014574716 (3095388-3159234): Metarhizium brunneum ARSEF 3297 chromosome Un... (18% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c5_10.svg">
NC_035791 (1143313-1234197): Pochonia chlamydosporia 170 chromosome 2, whole ... (9% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c5-download">Download graphic</button>
</div>
<div id="clusterblast-r6c5-svg"></div>

</div>
</div>

<div class="body-details-section r6c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-231"></div> <span class="help-tooltip" id="cb-known-help-231">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c5-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c5_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c5_1.svg">
choline
</option>

</select>
<button id="knownclusterblast-r6c5-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c5-svg"></div>

</div>
</div>

<div class="body-details-section r6c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-232"></div> <span class="help-tooltip" id="cb-sub-help-232">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-235"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-235">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-domain-bubble-svg" data-filename="CP023323.1_r6c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (3271608 - 3315469): single NRPS-like
</option>


</select>

<div class="details-svg" id="r6c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-237"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-237">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-pfam-details-svg" data-filename="CP023323.1_r6c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c5-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-238"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-238">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c5-tigrfam-details-svg" data-filename="CP023323.1_r6c5_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c5-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c5-nrps_pks_products" data-name="r6c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c5-nrps_pks_monomers" data-name="r6c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-233"></div> <span class="help-tooltip" id="nrps-structures-help-233">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 3271608 - 3315469:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c5-5" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c5-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-234"></div> <span class="help-tooltip" id="nrps-monomers-help-234">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene845</span></strong>: Phe
<div class="collapser collapser-target-input-path1-gene845 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (13..418)</strong>: Phe
<div class="collapser collapser-target-input-path1-gene845 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-input-path1-gene845-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>phe (Phe)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--WLWNVEAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 6 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-410"></div> <span class="help-tooltip" id="region_svg-help-410">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c6-svg" data-filename="CP023323.1_r6c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,636,275 - 3,711,474 nt. (total: 75,200 nt)

<a class="cluster-rules-header" id="r6c6-rules-header" href="#r6c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c6-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r6c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-411"></div> <span class="help-tooltip" id="focus-panel-help-411">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c6-gene-table-details" data-name="r6c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c6-MIBiG-cluster-compare" data-name="r6c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c6-clusterblast" data-name="r6c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c6-knownclusterblast" data-name="r6c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c6-subclusterblast" data-name="r6c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c6-pfam-details" data-name="r6c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c6-tigrfam-details" data-name="r6c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-243"></div> <span class="help-tooltip" id="gt-help-tooltip-help-243">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-239"></div> <span class="help-tooltip" id="MIBiG-help-239">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c6"></div>
<div id="comparison-MIBiG-r6c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>ascofuranone</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002219: 0-26397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002219/index.html#r1c1" target="_new">BGC0002219</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>(-)-(R)-nephthenol</td>


<td>Stereum hirsutum FP-91666 SS1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Other</td>

<td>itaconic acid</td>


<td>Ustilago maydis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Alkaloid</td>

<td>huperzine A</td>


<td>Shiraia sp. slf14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002721: 182-542486">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002721/index.html#r1c1" target="_new">BGC0002721</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Saccharide</td>

<td>α-hydroxynitrile glucoside, β-hydroxynitrile glucoside, γ-hydroxynitrile glucoside</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>armillyl orsellinate, 8α-hydroxy-6-protoilludene</td>


<td>Armillaria gallica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.47, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>HEx-tc3 terpene</td>


<td>Botrytis cinerea T4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.15</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>yanuthone D</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001924: 618-5310">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001924/index.html#r1c1" target="_new">BGC0001924</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>ascofuranone</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002219: 0-26397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002219/index.html#r1c1" target="_new">BGC0002219</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>(-)-(R)-nephthenol</td>
<td>Stereum hirsutum FP-91666 SS1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>itaconic acid</td>
<td>Ustilago maydis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Alkaloid</td>
<td>huperzine A</td>
<td>Shiraia sp. slf14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002721: 182-542486">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002721/index.html#r1c1" target="_new">BGC0002721</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Saccharide</td>
<td>α-hydroxynitrile glucoside, β-hydroxynitrile glucoside, γ-hydroxynitrile glucoside</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>armillyl orsellinate, 8α-hydroxy-6-protoilludene</td>
<td>Armillaria gallica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>HEx-tc3 terpene</td>
<td>Botrytis cinerea T4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>yanuthone D</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-240"></div> <span class="help-tooltip" id="cb-general-help-240">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c6_1.svg">
NW_006271970 (2031211-2136459): Cordyceps militaris CM01 unplaced genomic sca... (52% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_2.svg">
NW_011371361 (1937664-2035052): Paracoccidioides brasiliensis Pb18 unplaced g... (17% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_3.svg">
NW_015440948 (109188-206314): Paracoccidioides lutzii Pb01 unplaced genomic s... (18% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_4.svg">
NT_165982 (1008305-1103578): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (20% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_5.svg">
NW_017263587 (265177-438368): Mollisia scopiformis unplaced genomic scaffold ... (8% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_6.svg">
NC_030959 (4385741-4495530): Colletotrichum higginsianum IMI 349063 chromosom... (14% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_7.svg">
NW_026250024 (1787317-1950532): Hypoxylon trugodes strain CBS 135444 unplaced... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_8.svg">
NW_025953252 (947231-1128644): Annulohypoxylon maeteangense strain CBS 123835... (3% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c6_9.svg">
NC_035796 (212423-303459): Pochonia chlamydosporia 170 chromosome 7, whole ge... (13% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r6c6_10.svg">
NW_024468986 (554990-661675): Aspergillus fijiensis CBS 313.89 unplaced genom... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r6c6-download">Download graphic</button>
</div>
<div id="clusterblast-r6c6-svg"></div>

</div>
</div>

<div class="body-details-section r6c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-241"></div> <span class="help-tooltip" id="cb-known-help-241">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-242"></div> <span class="help-tooltip" id="cb-sub-help-242">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-244"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-244">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-pfam-details-svg" data-filename="CP023323.1_r6c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-245"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-245">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c6-tigrfam-details-svg" data-filename="CP023323.1_r6c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r6c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 7 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-412"></div> <span class="help-tooltip" id="region_svg-help-412">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c7-svg" data-filename="CP023323.1_r6c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,090,598 - 4,135,469 nt. (total: 44,872 nt)

<a class="cluster-rules-header" id="r6c7-rules-header" href="#r6c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c7-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-413"></div> <span class="help-tooltip" id="focus-panel-help-413">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c7-gene-table-details" data-name="r6c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c7-nrps_pks" data-name="r6c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c7-MIBiG-cluster-compare" data-name="r6c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c7-clusterblast" data-name="r6c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c7-knownclusterblast" data-name="r6c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c7-subclusterblast" data-name="r6c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c7-nrps-pks-bubbles" data-name="r6c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c7-pfam-details" data-name="r6c7-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c7-tigrfam-details" data-name="r6c7-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-254"></div> <span class="help-tooltip" id="gt-help-tooltip-help-254">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-246"></div> <span class="help-tooltip" id="nrps-domain-help-246">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c7-details-svg" data-filename="CP023323.1_r6c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-247"></div> <span class="help-tooltip" id="MIBiG-help-247">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c7"></div>
<div id="comparison-MIBiG-r6c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058: 11-12613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.41, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide, NRP</td>

<td>mutanocyclin</td>


<td>Streptococcus mutans B04Sm5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002287: 24540-40025">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002287/index.html#r1c1" target="_new">BGC0002287</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.41, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>mutanocyclin, leuvalin, tyrvalin</td>


<td>Streptococcus mutans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>pyrrolizixenamide A</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002226: 138-12428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002226/index.html#r1c1" target="_new">BGC0002226</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>acu-dioxomorpholine B, acu-dioxomorpholine A</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058: 11-12613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.41, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide, NRP</td>
<td>mutanocyclin</td>
<td>Streptococcus mutans B04Sm5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002287: 24540-40025">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002287/index.html#r1c1" target="_new">BGC0002287</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.41, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>mutanocyclin, leuvalin, tyrvalin</td>
<td>Streptococcus mutans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>pyrrolizixenamide A</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002226: 138-12428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002226/index.html#r1c1" target="_new">BGC0002226</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>acu-dioxomorpholine B, acu-dioxomorpholine A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-248"></div> <span class="help-tooltip" id="cb-general-help-248">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c7_1.svg">
NW_006271970 (1632104-1696976): Cordyceps militaris CM01 unplaced genomic sca... (54% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_2.svg">
NW_022156716 (287324-376248): Pyricularia grisea strain NI907 chromosome Unkn... (8% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r6c7_3.svg">
NZ_CP077093 (3609174-3681297): Pseudomonas vanderleydeniana strain RW8P3 chro... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_4.svg">
NZ_CP010026 (730799-778179): Paraburkholderia fungorum strain ATCC BAA-463 ch... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_5.svg">
NZ_RRUE01000001 (972066-1112935): Lautropia dentalis strain KCOM 2505 KCOM250... (2% of genes show similarity), NRP-metallophore,NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r6c7_6.svg">
NZ_CP099647 (926083-973463): Paraburkholderia fungorum strain OTU2SAUBB1 chro... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_7.svg">
NZ_JACIIK010000022 (23038-70427): Paraburkholderia fungorum strain SEMIA 4013... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_8.svg">
NZ_CP028829 (730806-778186): Paraburkholderia fungorum strain UM TBK chromoso... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_9.svg">
NZ_CP014578 (923071-970565): Paraburkholderia phytofirmans OLGA172 chromosome... (6% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r6c7_10.svg">
NZ_CP099623 (1141171-1188551): Paraburkholderia fungorum strain OTU2BAGNBA1 c... (6% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r6c7-download">Download graphic</button>
</div>
<div id="clusterblast-r6c7-svg"></div>

</div>
</div>

<div class="body-details-section r6c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-249"></div> <span class="help-tooltip" id="cb-known-help-249">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-250"></div> <span class="help-tooltip" id="cb-sub-help-250">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-253"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-253">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c7-domain-bubble-svg" data-filename="CP023323.1_r6c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (4090597 - 4135469): single NRPS
</option>


</select>

<div class="details-svg" id="r6c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-255"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-255">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c7-pfam-details-svg" data-filename="CP023323.1_r6c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c7-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-256"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-256">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c7-tigrfam-details-svg" data-filename="CP023323.1_r6c7_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c7-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c7-nrps_pks_products" data-name="r6c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c7-nrps_pks_monomers" data-name="r6c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-251"></div> <span class="help-tooltip" id="nrps-structures-help-251">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 4090597 - 4135469:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c7-7" class="smiles-canvas" data-smiles="NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c7-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-252"></div> <span class="help-tooltip" id="nrps-monomers-help-252">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1068</span></strong>: Ile
<div class="collapser collapser-target-input-path1-gene1068 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (498..892)</strong>: Ile
<div class="collapser collapser-target-input-path1-gene1068 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-input-path1-gene1068-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALHSGAVIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DALHSGAVIK</span> (82% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 8 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-414"></div> <span class="help-tooltip" id="region_svg-help-414">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c8-svg" data-filename="CP023323.1_r6c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,327,223 - 4,374,354 nt. (total: 47,132 nt)

<a class="cluster-rules-header" id="r6c8-rules-header" href="#r6c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c8-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r6c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-415"></div> <span class="help-tooltip" id="focus-panel-help-415">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c8-gene-table-details" data-name="r6c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c8-nrps_pks" data-name="r6c8-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c8-MIBiG-cluster-compare" data-name="r6c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c8-clusterblast" data-name="r6c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c8-knownclusterblast" data-name="r6c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c8-subclusterblast" data-name="r6c8-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c8-nrps-pks-bubbles" data-name="r6c8-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c8-pfam-details" data-name="r6c8-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-265"></div> <span class="help-tooltip" id="gt-help-tooltip-help-265">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c8-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-257"></div> <span class="help-tooltip" id="nrps-domain-help-257">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c8-details-svg" data-filename="CP023323.1_r6c8_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c8-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-258"></div> <span class="help-tooltip" id="MIBiG-help-258">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c8-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c8-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c8"></div>
<div id="comparison-MIBiG-r6c8-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>phomoidride</td>


<td>fungal sp. ATCC 74256</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene, Polyketide</td>

<td>calidoustene A, calidoustene B, calidoustene C</td>


<td>Aspergillus calidoustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>epipyriculol</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>HEx-pks1 polyketide</td>


<td>Paraphaeosphaeria sporulosa</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c8-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>phomoidride</td>
<td>fungal sp. ATCC 74256</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>epipyriculol</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>HEx-pks1 polyketide</td>
<td>Paraphaeosphaeria sporulosa</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-259"></div> <span class="help-tooltip" id="cb-general-help-259">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c8_1.svg">
NW_006271970 (1369951-1438884): Cordyceps militaris CM01 unplaced genomic sca... (55% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c8_2.svg">
NW_024468918 (1133295-1261954): Cryphonectria parasitica EP155 unplaced genom... (9% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c8_3.svg">
NW_006711153 (459708-526750): Trichoderma reesei QM6a unplaced genomic scaffo... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c8_4.svg">
NW_025408667 (6621596-6715308): Hirsutella rhossiliensis strain HR02 chromoso... (30% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c8_5.svg">
NC_070539 (1227558-1294574): Fusarium keratoplasticum isolate Fu6.1 chromosom... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c8_6.svg">
NC_070554 (1471044-1538049): Fusarium falciforme chromosome 11, complete sequ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c8_7.svg">
NC_035790 (8523905-8591015): Pochonia chlamydosporia 170 chromosome 1, whole ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c8_8.svg">
NW_025091220 (124401-191227): Aspergillus viridinutans IFM 47045 DNA, scaffol... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c8_9.svg">
NW_013581198 (762797-842941): Aureobasidium namibiae CBS 147.97 unplaced geno... (7% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r6c8_10.svg">
NW_025953257 (328883-396554): Annulohypoxylon maeteangense strain CBS 123835 ... (16% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r6c8-download">Download graphic</button>
</div>
<div id="clusterblast-r6c8-svg"></div>

</div>
</div>

<div class="body-details-section r6c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-260"></div> <span class="help-tooltip" id="cb-known-help-260">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-261"></div> <span class="help-tooltip" id="cb-sub-help-261">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c8-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-264"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-264">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c8-domain-bubble-svg" data-filename="CP023323.1_r6c8_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c8-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC8">
Candidate 8 (4327222 - 4374354): single T1PKS
</option>


</select>

<div class="details-svg" id="r6c8-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c8-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-266"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-266">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c8-pfam-details-svg" data-filename="CP023323.1_r6c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c8-nrps_pks_products" data-name="r6c8-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c8-nrps_pks_monomers" data-name="r6c8-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c8-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-262"></div> <span class="help-tooltip" id="nrps-structures-help-262">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 8, location 4327222 - 4374354:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c8-8" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c8-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c8-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-263"></div> <span class="help-tooltip" id="nrps-monomers-help-263">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1128</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene1128 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (498..808)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1128 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1128-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Isobutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1128-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 68.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 62.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 48.7</dt>

<dd></dd><dt>Propionyl-CoA: 43.8</dt>

<dd></dd><dt>Malonyl-CoA: 38.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 27.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 21.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 20.2</dt>

<dd></dd><dt>Benzoyl-CoA: 18.3</dt>

<dd></dd><dt>fatty_acid: 7.6</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c9' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023323.1 - Region 9 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-416"></div> <span class="help-tooltip" id="region_svg-help-416">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023323.1.region009.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c9-svg" data-filename="CP023323.1_r6c9.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,790,965 - 5,840,010 nt. (total: 49,046 nt)

<a class="cluster-rules-header" id="r6c9-rules-header" href="#r6c9">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c9-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r6c9-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-417"></div> <span class="help-tooltip" id="focus-panel-help-417">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c9">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c9-gene-table-details" data-name="r6c9-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c9-nrps_pks" data-name="r6c9-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c9-MIBiG-cluster-compare" data-name="r6c9-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c9-clusterblast" data-name="r6c9-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c9-knownclusterblast" data-name="r6c9-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c9-subclusterblast" data-name="r6c9-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c9-nrps-pks-bubbles" data-name="r6c9-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c9-pfam-details" data-name="r6c9-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c9-tigrfam-details" data-name="r6c9-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c9-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-275"></div> <span class="help-tooltip" id="gt-help-tooltip-help-275">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c9">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c9-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c9-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c9">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c9-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-267"></div> <span class="help-tooltip" id="nrps-domain-help-267">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c9-details-svg" data-filename="CP023323.1_r6c9_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c9-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c9-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-268"></div> <span class="help-tooltip" id="MIBiG-help-268">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c9-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c9-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c9"></div>
<div id="comparison-MIBiG-r6c9-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000012: 1892-19092">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000012/index.html#r1c1" target="_new">BGC0000012</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>alternapyrone</td>


<td>Alternaria solani</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>abscisic acid</td>


<td>Leptosphaeria maculans JN3</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c9-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000012: 1892-19092">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000012/index.html#r1c1" target="_new">BGC0000012</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>alternapyrone</td>
<td>Alternaria solani</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>abscisic acid</td>
<td>Leptosphaeria maculans JN3</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c9-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-269"></div> <span class="help-tooltip" id="cb-general-help-269">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c9-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c9_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c9_1.svg">
NW_022987226 (946899-1015102): Trematosphaeria pertusa strain CBS 122368 unpl... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c9_2.svg">
NW_022986863 (466665-534682): Westerdykella ornata strain CBS 379.55 unplaced... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c9_3.svg">
NW_006917091 (1555707-1623710): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c9_4.svg">
NW_017253364 (1410639-1535276): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (4% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r6c9_5.svg">
NW_021167080 (3127431-3195689): Sodiomyces alkalinus F11 unplaced genomic sca... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c9_6.svg">
NW_024468938 (1558469-1626421): Cucurbitaria berberidis CBS 394.84 unplaced g... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c9_7.svg">
NW_025953250 (276988-345443): Annulohypoxylon maeteangense strain CBS 123835 ... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c9_8.svg">
NW_022987230 (2761-70419): Trematosphaeria pertusa strain CBS 122368 unplaced... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c9_9.svg">
NT_165982 (3358274-3426284): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c9_10.svg">
NW_026054544 (710053-825119): Daldinia decipiens strain CBS 113046 unplaced g... (6% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r6c9-download">Download graphic</button>
</div>
<div id="clusterblast-r6c9-svg"></div>

</div>
</div>

<div class="body-details-section r6c9-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-270"></div> <span class="help-tooltip" id="cb-known-help-270">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c9-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-271"></div> <span class="help-tooltip" id="cb-sub-help-271">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c9-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-274"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-274">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c9-domain-bubble-svg" data-filename="CP023323.1_r6c9_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c9-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC9">
Candidate 9 (5790964 - 5840010): single T1PKS
</option>


</select>

<div class="details-svg" id="r6c9-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c9-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c9-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-276"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-276">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c9-pfam-details-svg" data-filename="CP023323.1_r6c9_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c9-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c9-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-277"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-277">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c9-tigrfam-details-svg" data-filename="CP023323.1_r6c9_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c9-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c9-nrps_pks_products" data-name="r6c9-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c9-nrps_pks_monomers" data-name="r6c9-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c9-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-272"></div> <span class="help-tooltip" id="nrps-structures-help-272">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 9, location 5790964 - 5840010:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c9-9" class="smiles-canvas" data-smiles="C(C)CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-redmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-redmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-redmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c9-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C(C)CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c9-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-273"></div> <span class="help-tooltip" id="nrps-monomers-help-273">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1520</span></strong>: mal
<div class="collapser collapser-target-input-path1-gene1520 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (524..838)</strong>: mal
<div class="collapser collapser-target-input-path1-gene1520 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1520-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Propionyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1520-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 76.5</dt>

<dd></dd><dt>Methylmalonyl-CoA: 74.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 70.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 41.5</dt>

<dd></dd><dt>inactive: 38.1</dt>

<dd></dd><dt>Propionyl-CoA: 33.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 30.6</dt>

<dd></dd><dt>Benzoyl-CoA: 30.6</dt>

<dd></dd><dt>fatty_acid: 17.5</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 12.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 10.2</dt>

<dd></dd><dt>CHC-CoA: 8.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 7.0</dt>

<dd></dd><dt>Acetyl-CoA: 5.4</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r7c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-418"></div> <span class="help-tooltip" id="region_svg-help-418">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c1-svg" data-filename="CP023324.1_r7c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 190,427 - 271,267 nt. (total: 80,841 nt)

<a class="cluster-rules-header" id="r7c1-rules-header" href="#r7c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r7c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-419"></div> <span class="help-tooltip" id="focus-panel-help-419">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c1-gene-table-details" data-name="r7c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c1-nrps_pks" data-name="r7c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c1-MIBiG-cluster-compare" data-name="r7c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c1-clusterblast" data-name="r7c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c1-knownclusterblast" data-name="r7c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c1-subclusterblast" data-name="r7c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c1-nrps-pks-bubbles" data-name="r7c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c1-pfam-details" data-name="r7c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c1-tigrfam-details" data-name="r7c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-286"></div> <span class="help-tooltip" id="gt-help-tooltip-help-286">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-278"></div> <span class="help-tooltip" id="nrps-domain-help-278">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-details-svg" data-filename="CP023324.1_r7c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-279"></div> <span class="help-tooltip" id="MIBiG-help-279">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c1"></div>
<div id="comparison-MIBiG-r7c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.62</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.55</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.54</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.54</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.54</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.53</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.52</td>
<td>Polyketide</td>

<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.52</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.52</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.44, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>






<td class="cc-heat-cell" title="0.35 (id:0.44, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.51</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.46, order:0.69, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Glarea lozoyensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-280"></div> <span class="help-tooltip" id="cb-general-help-280">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c1_1.svg">
NW_006271974 (2321315-2405136): Cordyceps militaris CM01 unplaced genomic sca... (75% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c1_2.svg">
NW_020194484 (7340-83776): Amorphotheca resinae ATCC 22711 unplaced genomic s... (41% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r7c1_3.svg">
NW_026249875 (10804-78109): Alternaria incomplexa strain BMP 0042 chromosome ... (38% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_4.svg">
NW_026055699 (512819-580124): Alternaria ventricosa strain BMP 2768 chromosom... (36% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_5.svg">
NW_022474205 (7019281-7087036): Venustampulla echinocandica strain BP 5553 ch... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_6.svg">
NW_025408687 (532325-600393): Hirsutella rhossiliensis strain HR02 chromosome... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_7.svg">
NW_026249941 (3336813-3404118): Alternaria arbusti strain BMP 1465 chromosome... (33% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_8.svg">
NW_025544748 (424962-492810): Colletotrichum gloeosporioides strain Lc1 chrom... (25% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_9.svg">
NW_025953791 (566181-633989): Annulohypoxylon truncatum strain CBS 140777 unp... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c1_10.svg">
NW_021167084 (1837567-1907553): Sodiomyces alkalinus F11 unplaced genomic sca... (20% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r7c1-download">Download graphic</button>
</div>
<div id="clusterblast-r7c1-svg"></div>

</div>
</div>

<div class="body-details-section r7c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-281"></div> <span class="help-tooltip" id="cb-known-help-281">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-282"></div> <span class="help-tooltip" id="cb-sub-help-282">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-285"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-285">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-domain-bubble-svg" data-filename="CP023324.1_r7c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (190426 - 271267): neighbouring NRPS-T1PKS
</option>



<option value="CC2">
Candidate 2 (190426 - 241925): chemical hybrid NRPS-T1PKS
</option>



<option value="CC3">
Candidate 3 (223321 - 271267): single T1PKS
</option>


</select>

<div class="details-svg" id="r7c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-287"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-287">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-pfam-details-svg" data-filename="CP023324.1_r7c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-288"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-288">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c1-tigrfam-details-svg" data-filename="CP023324.1_r7c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c1-nrps_pks_products" data-name="r7c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c1-nrps_pks_monomers" data-name="r7c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-283"></div> <span class="help-tooltip" id="nrps-structures-help-283">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 190426 - 271267:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X) + (pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 190426 - 241925:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c1-2" class="smiles-canvas" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 223321 - 271267:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c1-3" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-284"></div> <span class="help-tooltip" id="nrps-monomers-help-284">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene52</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene52 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (490..800)</strong>: pk
<div class="collapser collapser-target-input-path1-gene52 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene52-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene52-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 88.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 78.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 72.0</dt>

<dd></dd><dt>Malonyl-CoA: 67.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 59.9</dt>

<dd></dd><dt>Propionyl-CoA: 48.6</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 43.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 35.1</dt>

<dd></dd><dt>Benzoyl-CoA: 24.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 24.4</dt>

<dd></dd><dt>inactive: 22.3</dt>

<dd></dd><dt>CHC-CoA: 21.0</dt>

<dd></dd><dt>fatty_acid: 19.4</dt>

<dd></dd><dt>Acetyl-CoA: 9.2</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2673..3082)</strong>: X
<div class="collapser collapser-target-input-path1-gene52 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene52-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIAYSIESQK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">input.path1.gene54</span></strong>: pk
<div class="collapser collapser-target-input-path1-gene54 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (512..823)</strong>: pk
<div class="collapser collapser-target-input-path1-gene54 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene54-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>2-Methylbutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene54-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 73.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 64.6</dt>

<dd></dd><dt>Malonyl-CoA: 55.8</dt>

<dd></dd><dt>Propionyl-CoA: 38.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 33.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 27.6</dt>

<dd></dd><dt>fatty_acid: 26.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 24.9</dt>

<dd></dd><dt>inactive: 19.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 18.3</dt>

<dd></dd><dt>Benzoyl-CoA: 15.0</dt>

<dd></dd><dt>CHC-CoA: 13.2</dt>

<dd></dd><dt>Acetyl-CoA: -1.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: -6.3</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-420"></div> <span class="help-tooltip" id="region_svg-help-420">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c2-svg" data-filename="CP023324.1_r7c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,515,240 - 1,536,807 nt. (total: 21,568 nt)

<a class="cluster-rules-header" id="r7c2-rules-header" href="#r7c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r7c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-421"></div> <span class="help-tooltip" id="focus-panel-help-421">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c2-gene-table-details" data-name="r7c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c2-MIBiG-cluster-compare" data-name="r7c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c2-clusterblast" data-name="r7c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c2-knownclusterblast" data-name="r7c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c2-subclusterblast" data-name="r7c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r7c2-pfam-details" data-name="r7c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c2-tigrfam-details" data-name="r7c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-293"></div> <span class="help-tooltip" id="gt-help-tooltip-help-293">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-289"></div> <span class="help-tooltip" id="MIBiG-help-289">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c2"></div>
<div id="comparison-MIBiG-r7c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.53, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>squalestatin S1</td>


<td>Aspergillus sp. Z5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>cytolysin ClyLl, cytolysin ClyLs</td>


<td>Plasmid pAD1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.45, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.45, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>microcyclamide</td>


<td>Microcystis aeruginosa NIES-298</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>carbapenem MM4550</td>


<td>Streptomyces argenteolus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.52, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>squalestatin S1</td>


<td>Phoma sp. MF5453</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>gramicidin S</td>


<td>Aneurinibacillus migulanus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>





<td class="cc-heat-cell" title="0.15 (id:0.45, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>sorangipyranone</td>


<td>Myxococcales bacterium</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.53, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>squalestatin S1</td>
<td>Aspergillus sp. Z5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504: 606-7471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>cytolysin ClyLl, cytolysin ClyLs</td>
<td>Plasmid pAD1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.45, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.45, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>RiPP</td>
<td>microcyclamide</td>
<td>Microcystis aeruginosa NIES-298</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>carbapenem MM4550</td>
<td>Streptomyces argenteolus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.52, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Polyketide</td>
<td>squalestatin S1</td>
<td>Phoma sp. MF5453</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP</td>
<td>gramicidin S</td>
<td>Aneurinibacillus migulanus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>


<td class="cc-heat-cell" title="0.15 (id:0.45, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
<td>0.15</td>
<td>Polyketide</td>
<td>sorangipyranone</td>
<td>Myxococcales bacterium</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-290"></div> <span class="help-tooltip" id="cb-general-help-290">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c2_1.svg">
NW_006271974 (1078266-1109834): Cordyceps militaris CM01 unplaced genomic sca... (36% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_2.svg">
NC_070529 (5465471-5497016): Fusarium keratoplasticum isolate Fu6.1 chromosom... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_3.svg">
NC_070544 (5794890-5826434): Fusarium falciforme chromosome 1, complete sequence (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_4.svg">
NC_058389 (1466023-1497573): Fusarium musae strain F31 chromosome 3, whole ge... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_5.svg">
NC_036623 (3601013-3632563): Fusarium fujikuroi IMI 58289 draft genome, chrom... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_6.svg">
NC_058402 (6845329-6876878): Fusarium poae strain DAOMC 252244 chromosome 4, ... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_7.svg">
NC_026477 (6663266-6694813): Fusarium graminearum PH-1 chromosome 4, whole ge... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_8.svg">
NC_038015 (6636088-6667634): Fusarium venenatum strain A3/5 genome assembly, ... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_9.svg">
NW_025764705 (1077863-1109318): Emericellopsis atlantica unplaced genomic sca... (28% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c2_10.svg">
NW_023336189 (374372-405746): Geosmithia morbida isolate 1262 chromosome Unkn... (44% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r7c2-download">Download graphic</button>
</div>
<div id="clusterblast-r7c2-svg"></div>

</div>
</div>

<div class="body-details-section r7c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-291"></div> <span class="help-tooltip" id="cb-known-help-291">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r7c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r7c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r7c2_1.svg">
squalestatin S1
</option>

</select>
<button id="knownclusterblast-r7c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r7c2-svg"></div>

</div>
</div>

<div class="body-details-section r7c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-292"></div> <span class="help-tooltip" id="cb-sub-help-292">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r7c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-294"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-294">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-pfam-details-svg" data-filename="CP023324.1_r7c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-295"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-295">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c2-tigrfam-details-svg" data-filename="CP023324.1_r7c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r7c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 3 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-422"></div> <span class="help-tooltip" id="region_svg-help-422">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c3-svg" data-filename="CP023324.1_r7c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,708,929 - 3,769,829 nt. (total: 60,901 nt)

<a class="cluster-rules-header" id="r7c3-rules-header" href="#r7c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c3-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r7c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-423"></div> <span class="help-tooltip" id="focus-panel-help-423">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c3-gene-table-details" data-name="r7c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c3-MIBiG-cluster-compare" data-name="r7c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c3-clusterblast" data-name="r7c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c3-knownclusterblast" data-name="r7c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c3-subclusterblast" data-name="r7c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c3-pfam-details" data-name="r7c3-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c3-tigrfam-details" data-name="r7c3-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-300"></div> <span class="help-tooltip" id="gt-help-tooltip-help-300">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-296"></div> <span class="help-tooltip" id="MIBiG-help-296">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c3"></div>
<div id="comparison-MIBiG-r7c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001621: 3190-27169">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001621/index.html#r1c1" target="_new">BGC0001621</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>imizoquin A, imizoquin B, imizoquin C, imizoquin D, TMC-2A, TMC-2B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001565: 171-90079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001565/index.html#r1c1" target="_new">BGC0001565</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.48, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>cyclosporin C</td>


<td>Beauveria felina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001515: 0-63902">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001515/index.html#r1c1" target="_new">BGC0001515</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.49, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP</td>

<td>aspercryptins</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000334: 1-107683">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000334/index.html#r1c1" target="_new">BGC0000334</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.48, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP</td>

<td>cyclosporin</td>


<td>Tolypocladium inflatum NRRL 8044</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002539: 47-37339">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002539/index.html#r1c1" target="_new">BGC0002539</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.53, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Alkaloid</td>

<td>chaetoglobosin P, chaetoglobosin K, chaetoglobosin A</td>


<td>Discosia rubi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001724: 0-71056">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001724/index.html#r1c1" target="_new">BGC0001724</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.51, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>oxaleimide C</td>


<td>Penicillium oxalicum 114-2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000304: 9933-51438">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000304/index.html#r1c1" target="_new">BGC0000304</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.48, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>apicidin</td>


<td>Fusarium incarnatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002611: 1682-51377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002611/index.html#r1c1" target="_new">BGC0002611</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.49, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>mulundocandin</td>


<td>Aspergillus mulundensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001621: 3190-27169">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001621/index.html#r1c1" target="_new">BGC0001621</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>NRP</td>
<td>imizoquin A, imizoquin B, imizoquin C, imizoquin D, TMC-2A, TMC-2B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001565: 171-90079">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001565/index.html#r1c1" target="_new">BGC0001565</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.48, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>NRP</td>
<td>cyclosporin C</td>
<td>Beauveria felina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001515: 0-63902">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001515/index.html#r1c1" target="_new">BGC0001515</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.49, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP</td>
<td>aspercryptins</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000334: 1-107683">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000334/index.html#r1c1" target="_new">BGC0000334</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.48, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP</td>
<td>cyclosporin</td>
<td>Tolypocladium inflatum NRRL 8044</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002539: 47-37339">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002539/index.html#r1c1" target="_new">BGC0002539</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.53, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Alkaloid</td>
<td>chaetoglobosin P, chaetoglobosin K, chaetoglobosin A</td>
<td>Discosia rubi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001724: 0-71056">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001724/index.html#r1c1" target="_new">BGC0001724</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.51, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>NRP, Polyketide</td>
<td>oxaleimide C</td>
<td>Penicillium oxalicum 114-2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000304: 9933-51438">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000304/index.html#r1c1" target="_new">BGC0000304</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.48, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>NRP</td>
<td>apicidin</td>
<td>Fusarium incarnatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002611: 1682-51377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002611/index.html#r1c1" target="_new">BGC0002611</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.49, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP</td>
<td>mulundocandin</td>
<td>Aspergillus mulundensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-297"></div> <span class="help-tooltip" id="cb-general-help-297">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c3_1.svg">
NW_006271973 (1574268-1668763): Cordyceps militaris CM01 unplaced genomic sca... (48% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c3_2.svg">
NW_025408667 (6621596-6715308): Hirsutella rhossiliensis strain HR02 chromoso... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c3_3.svg">
NW_014013666 (749888-843435): Trichoderma virens Gv29-8 chromosome Unknown TR... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c3_4.svg">
NW_017263581 (2155038-2256236): Mollisia scopiformis unplaced genomic scaffol... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c3_5.svg">
NC_072349 (1251120-1365955): Talaromyces marneffei chromosome 2, complete seq... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c3_6.svg">
NW_014574704 (587836-655796): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c3_7.svg">
NW_011942160 (5112978-5180935): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c3_8.svg">
NW_015971586 (138-90635): Cladophialophora bantiana CBS 173.52 unplaced genom... (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c3_9.svg">
NC_054856 (3727884-3894446): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (3% of genes show similarity), NI-siderophore,NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c3_10.svg">
NW_024468915 (2226591-2323212): Cryphonectria parasitica EP155 unplaced genom... (9% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r7c3-download">Download graphic</button>
</div>
<div id="clusterblast-r7c3-svg"></div>

</div>
</div>

<div class="body-details-section r7c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-298"></div> <span class="help-tooltip" id="cb-known-help-298">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-299"></div> <span class="help-tooltip" id="cb-sub-help-299">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-301"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-301">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-pfam-details-svg" data-filename="CP023324.1_r7c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c3-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-302"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-302">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c3-tigrfam-details-svg" data-filename="CP023324.1_r7c3_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c3-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r7c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 4 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-424"></div> <span class="help-tooltip" id="region_svg-help-424">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c4-svg" data-filename="CP023324.1_r7c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,298,434 - 4,343,860 nt. (total: 45,427 nt)

<a class="cluster-rules-header" id="r7c4-rules-header" href="#r7c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r7c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-425"></div> <span class="help-tooltip" id="focus-panel-help-425">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c4-gene-table-details" data-name="r7c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c4-nrps_pks" data-name="r7c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c4-MIBiG-cluster-compare" data-name="r7c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c4-clusterblast" data-name="r7c4-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c4-knownclusterblast" data-name="r7c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c4-subclusterblast" data-name="r7c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c4-nrps-pks-bubbles" data-name="r7c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c4-pfam-details" data-name="r7c4-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c4-tigrfam-details" data-name="r7c4-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-311"></div> <span class="help-tooltip" id="gt-help-tooltip-help-311">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-303"></div> <span class="help-tooltip" id="nrps-domain-help-303">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c4-details-svg" data-filename="CP023324.1_r7c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-304"></div> <span class="help-tooltip" id="MIBiG-help-304">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c4"></div>
<div id="comparison-MIBiG-r7c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.56, order:0.53, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.56, order:0.53, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c4-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-305"></div> <span class="help-tooltip" id="cb-general-help-305">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c4-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c4_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c4_1.svg">
NW_006271973 (996861-1062288): Cordyceps militaris CM01 unplaced genomic scaf... (57% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_2.svg">
NW_014013664 (48182-113471): Trichoderma virens Gv29-8 chromosome Unknown TRI... (64% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_3.svg">
NW_006711167 (506611-572053): Trichoderma reesei QM6a unplaced genomic scaffo... (78% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_4.svg">
NW_011942160 (3770676-3836079): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_5.svg">
NW_014574716 (6263628-6329031): Metarhizium brunneum ARSEF 3297 chromosome Un... (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_6.svg">
NC_070551 (2536076-2601948): Fusarium falciforme chromosome 8, complete sequence (58% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_7.svg">
NC_070536 (2319263-2385135): Fusarium keratoplasticum isolate Fu6.1 chromosom... (62% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_8.svg">
NC_057319 (2334015-2399373): Ustilaginoidea virens chromosome 4, complete seq... (52% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_9.svg">
NC_035791 (4601700-4667019): Pochonia chlamydosporia 170 chromosome 2, whole ... (47% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c4_10.svg">
NW_021167080 (1267789-1333877): Sodiomyces alkalinus F11 unplaced genomic sca... (57% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r7c4-download">Download graphic</button>
</div>
<div id="clusterblast-r7c4-svg"></div>

</div>
</div>

<div class="body-details-section r7c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-306"></div> <span class="help-tooltip" id="cb-known-help-306">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r7c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r7c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r7c4_1.svg">
metachelin C/metachelin A/metachelin A-CE/metachelin B/dimerumic acid 11-mannoside/dimerumic acid
</option>

<option value = "svg/knownclusterblast_r7c4_2.svg">
dimethylcoprogen
</option>

</select>
<button id="knownclusterblast-r7c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r7c4-svg"></div>

</div>
</div>

<div class="body-details-section r7c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-307"></div> <span class="help-tooltip" id="cb-sub-help-307">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-310"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-310">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c4-domain-bubble-svg" data-filename="CP023324.1_r7c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (4298433 - 4343860): single NRPS
</option>


</select>

<div class="details-svg" id="r7c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c4-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c4-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-312"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-312">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c4-pfam-details-svg" data-filename="CP023324.1_r7c4_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c4-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c4-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-313"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-313">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c4-tigrfam-details-svg" data-filename="CP023324.1_r7c4_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c4-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c4-nrps_pks_products" data-name="r7c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c4-nrps_pks_monomers" data-name="r7c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-308"></div> <span class="help-tooltip" id="nrps-structures-help-308">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 4298433 - 4343860:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c4-6" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c4-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-309"></div> <span class="help-tooltip" id="nrps-monomers-help-309">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1193</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1193 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (84..484)</strong>: X
<div class="collapser collapser-target-input-path1-gene1193 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1193-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVDHGGAVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">DVDGGGGIGK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

<strong><span class="serif">input.path1.gene1194</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1194 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (70..460)</strong>: X
<div class="collapser collapser-target-input-path1-gene1194 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Z-ahmohOrn
<div class="collapser collapser-target-input-path1-gene1194-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">FGLHIASTCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">FGLHIASTCK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 5 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-426"></div> <span class="help-tooltip" id="region_svg-help-426">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c5-svg" data-filename="CP023324.1_r7c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,726,125 - 4,770,472 nt. (total: 44,348 nt)

<a class="cluster-rules-header" id="r7c5-rules-header" href="#r7c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c5-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r7c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-427"></div> <span class="help-tooltip" id="focus-panel-help-427">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c5-gene-table-details" data-name="r7c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c5-nrps_pks" data-name="r7c5-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c5-MIBiG-cluster-compare" data-name="r7c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c5-clusterblast" data-name="r7c5-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c5-knownclusterblast" data-name="r7c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c5-subclusterblast" data-name="r7c5-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c5-nrps-pks-bubbles" data-name="r7c5-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c5-pfam-details" data-name="r7c5-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-322"></div> <span class="help-tooltip" id="gt-help-tooltip-help-322">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c5-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-314"></div> <span class="help-tooltip" id="nrps-domain-help-314">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-details-svg" data-filename="CP023324.1_r7c5_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c5-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-315"></div> <span class="help-tooltip" id="MIBiG-help-315">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c5"></div>
<div id="comparison-MIBiG-r7c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c5-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-316"></div> <span class="help-tooltip" id="cb-general-help-316">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c5-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c5_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c5_1.svg">
NC_057367 (973431-1037220): Aspergillus chevalieri M1 DNA, chromosome 6, near... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c5_2.svg">
NW_019170454 (685594-749291): Aspergillus glaucus CBS 516.65 unplaced genomic... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c5_3.svg">
NW_024467304 (552618-615729): Aspergillus ruber CBS 135680 unplaced genomic s... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c5_4.svg">
NC_007196 (3573152-3653216): Aspergillus fumigatus Af293 chromosome 3, whole ... (6% of genes show similarity), NRPS,isocyanide-nrp
</option>

<option value = "svg/clusterblast_r7c5_5.svg">
NC_054850 (2918415-2983028): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c5_6.svg">
NC_054861 (1916756-1987814): Aspergillus puulaauensis MK2 DNA, chromosome 5, ... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c5_7.svg">
NW_024467524 (3810963-3881818): Aspergillus versicolor CBS 583.65 unplaced ge... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c5_8.svg">
NW_024467861 (3117513-3226103): Aspergillus sydowii CBS 593.65 unplaced genom... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c5_9.svg">
NC_054859 (3374101-3450218): Aspergillus puulaauensis MK2 DNA, chromosome 3, ... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c5_10.svg">
NW_026054668 (57810-121125): Ophidiomyces ophidiicola strain CBS 122913 chrom... (10% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r7c5-download">Download graphic</button>
</div>
<div id="clusterblast-r7c5-svg"></div>

</div>
</div>

<div class="body-details-section r7c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-317"></div> <span class="help-tooltip" id="cb-known-help-317">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-318"></div> <span class="help-tooltip" id="cb-sub-help-318">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c5-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-321"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-321">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-domain-bubble-svg" data-filename="CP023324.1_r7c5_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c5-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC7">
Candidate 7 (4726124 - 4770472): single NRPS
</option>


</select>

<div class="details-svg" id="r7c5-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c5-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c5-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-323"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-323">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c5-pfam-details-svg" data-filename="CP023324.1_r7c5_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c5-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c5-nrps_pks_products" data-name="r7c5-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c5-nrps_pks_monomers" data-name="r7c5-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c5-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-319"></div> <span class="help-tooltip" id="nrps-structures-help-319">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 7, location 4726124 - 4770472:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c5-7" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c5-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c5-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-320"></div> <span class="help-tooltip" id="nrps-monomers-help-320">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1294</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1294 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (182..575)</strong>: X
<div class="collapser collapser-target-input-path1-gene1294 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr
<div class="collapser collapser-target-input-path1-gene1294-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">VVTAVVLKGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 6 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-428"></div> <span class="help-tooltip" id="region_svg-help-428">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c6-svg" data-filename="CP023324.1_r7c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,186,674 - 5,247,751 nt. (total: 61,078 nt)

<a class="cluster-rules-header" id="r7c6-rules-header" href="#r7c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c6-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r7c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-429"></div> <span class="help-tooltip" id="focus-panel-help-429">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c6-gene-table-details" data-name="r7c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c6-MIBiG-cluster-compare" data-name="r7c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c6-clusterblast" data-name="r7c6-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c6-knownclusterblast" data-name="r7c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c6-subclusterblast" data-name="r7c6-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c6-pfam-details" data-name="r7c6-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c6-tigrfam-details" data-name="r7c6-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-328"></div> <span class="help-tooltip" id="gt-help-tooltip-help-328">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-324"></div> <span class="help-tooltip" id="MIBiG-help-324">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c6"></div>
<div id="comparison-MIBiG-r7c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.46, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001246: 1296-28428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001246/index.html#r1c1" target="_new">BGC0001246</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.54, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>trans-resorcylide</td>


<td>Sarocladium zeae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.46, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001246: 1296-28428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001246/index.html#r1c1" target="_new">BGC0001246</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.54, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Polyketide</td>
<td>trans-resorcylide</td>
<td>Sarocladium zeae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c6-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-325"></div> <span class="help-tooltip" id="cb-general-help-325">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c6-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c6_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c6_1.svg">
NW_006271969 (3108218-3200931): Cordyceps militaris CM01 unplaced genomic sca... (66% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c6_2.svg">
NW_023336188 (2131192-2194925): Geosmithia morbida isolate 1262 chromosome Un... (52% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r7c6_3.svg">
NC_038014 (4378948-4442770): Fusarium venenatum strain A3/5 genome assembly, ... (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r7c6_4.svg">
NC_058401 (4731897-4795714): Fusarium poae strain DAOMC 252244 chromosome 3, ... (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r7c6_5.svg">
NC_026476 (4083977-4147801): Fusarium graminearum PH-1 chromosome 3, whole ge... (13% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r7c6_6.svg">
NT_165982 (1135729-1226820): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c6_7.svg">
NW_025953812 (404505-501170): Annulohypoxylon truncatum strain CBS 140777 unp... (9% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c6_8.svg">
NW_025953252 (947231-1128644): Annulohypoxylon maeteangense strain CBS 123835... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c6_9.svg">
NW_022983481 (741577-810039): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c6_10.svg">
NW_026249943 (1889762-1956344): Alternaria arbusti strain BMP 1465 chromosome... (15% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r7c6-download">Download graphic</button>
</div>
<div id="clusterblast-r7c6-svg"></div>

</div>
</div>

<div class="body-details-section r7c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-326"></div> <span class="help-tooltip" id="cb-known-help-326">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-327"></div> <span class="help-tooltip" id="cb-sub-help-327">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c6-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-329"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-329">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c6-pfam-details-svg" data-filename="CP023324.1_r7c6_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c6-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c6-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-330"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-330">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c6-tigrfam-details-svg" data-filename="CP023324.1_r7c6_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c6-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r7c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 7 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-430"></div> <span class="help-tooltip" id="region_svg-help-430">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c7-svg" data-filename="CP023324.1_r7c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,136,346 - 6,188,673 nt. (total: 52,328 nt)

<a class="cluster-rules-header" id="r7c7-rules-header" href="#r7c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c7-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r7c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-431"></div> <span class="help-tooltip" id="focus-panel-help-431">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c7-gene-table-details" data-name="r7c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c7-nrps_pks" data-name="r7c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c7-MIBiG-cluster-compare" data-name="r7c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c7-clusterblast" data-name="r7c7-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c7-knownclusterblast" data-name="r7c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c7-subclusterblast" data-name="r7c7-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c7-nrps-pks-bubbles" data-name="r7c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r7c7-pfam-details" data-name="r7c7-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-339"></div> <span class="help-tooltip" id="gt-help-tooltip-help-339">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-331"></div> <span class="help-tooltip" id="nrps-domain-help-331">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c7-details-svg" data-filename="CP023324.1_r7c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-332"></div> <span class="help-tooltip" id="MIBiG-help-332">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c7"></div>
<div id="comparison-MIBiG-r7c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Polyketide</td>

<td>FR901512</td>


<td>fungal sp. No.14919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.64</td>
<td>Polyketide</td>

<td>verrucosidin</td>


<td>Penicillium polonicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002230: 9359-28771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002230/index.html#r1c1" target="_new">BGC0002230</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Polyketide, NRP</td>

<td>tolypyridone C</td>


<td>Tolypocladium sp. 49Y</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.62</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.62</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001026: 1-19101">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001026/index.html#r1c1" target="_new">BGC0001026</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>NRP, Polyketide</td>

<td>NG-391</td>


<td>Metarhizium anisopliae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002192: 0-9693">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002192/index.html#r1c1" target="_new">BGC0002192</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>FR901512</td>
<td>fungal sp. No.14919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002745: 541-30364">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002745/index.html#r1c1" target="_new">BGC0002745</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>verrucosidin</td>
<td>Penicillium polonicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002230: 9359-28771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002230/index.html#r1c1" target="_new">BGC0002230</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide, NRP</td>
<td>tolypyridone C</td>
<td>Tolypocladium sp. 49Y</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001026: 1-19101">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001026/index.html#r1c1" target="_new">BGC0001026</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP, Polyketide</td>
<td>NG-391</td>
<td>Metarhizium anisopliae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c7-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-333"></div> <span class="help-tooltip" id="cb-general-help-333">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c7-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c7_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c7_1.svg">
NW_024467862 (1993078-2065565): Aspergillus sydowii CBS 593.65 unplaced genom... (22% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c7_2.svg">
NW_017264196 (2263112-2335444): Xylona heveae TC161 unplaced genomic scaffold... (35% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c7_3.svg">
NC_072842 (4536206-4608469): Fusarium oxysporum Fo47 chromosome III, complete... (20% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c7_4.svg">
NC_036624 (299708-371970): Fusarium fujikuroi IMI 58289 draft genome, chromos... (18% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c7_5.svg">
NW_017306203 (371193-443658): Alternaria alternata strain SRC1lrK2f unplaced ... (20% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c7_6.svg">
NW_017263583 (13540-86418): Mollisia scopiformis unplaced genomic scaffold LY... (22% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c7_7.svg">
NC_058388 (4682141-4750888): Fusarium musae strain F31 chromosome 2, whole ge... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r7c7_8.svg">
NW_022474219 (264912-357113): Venustampulla echinocandica strain BP 5553 chro... (22% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r7c7_9.svg">
NC_049563 (4296847-4376181): Talaromyces rugulosus chromosome III, complete s... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r7c7_10.svg">
NW_025953831 (148948-216846): Annulohypoxylon truncatum strain CBS 140777 unp... (13% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r7c7-download">Download graphic</button>
</div>
<div id="clusterblast-r7c7-svg"></div>

</div>
</div>

<div class="body-details-section r7c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-334"></div> <span class="help-tooltip" id="cb-known-help-334">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r7c7-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r7c7_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r7c7_1.svg">
phomasetin
</option>

<option value = "svg/knownclusterblast_r7c7_2.svg">
equisetin
</option>

</select>
<button id="knownclusterblast-r7c7-download">Download graphic</button>
</div>
<div id="knownclusterblast-r7c7-svg"></div>

</div>
</div>

<div class="body-details-section r7c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-335"></div> <span class="help-tooltip" id="cb-sub-help-335">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-338"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-338">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c7-domain-bubble-svg" data-filename="CP023324.1_r7c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r7c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC9">
Candidate 9 (6136345 - 6188673): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r7c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r7c7-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r7c7-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-340"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-340">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c7-pfam-details-svg" data-filename="CP023324.1_r7c7_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c7-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c7-nrps_pks_products" data-name="r7c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r7c7-nrps_pks_monomers" data-name="r7c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r7c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-336"></div> <span class="help-tooltip" id="nrps-structures-help-336">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 9, location 6136345 - 6188673:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r7c7-9" class="smiles-canvas" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r7c7-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r7c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-337"></div> <span class="help-tooltip" id="nrps-monomers-help-337">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1719</span></strong>: pk - X
<div class="collapser collapser-target-input-path1-gene1719 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (540..858)</strong>: pk
<div class="collapser collapser-target-input-path1-gene1719 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-input-path1-gene1719-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Methoxymalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-input-path1-gene1719-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 87.6</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 71.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 67.3</dt>

<dd></dd><dt>Malonyl-CoA: 44.6</dt>

<dd></dd><dt>Propionyl-CoA: 43.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 35.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 32.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 31.7</dt>

<dd></dd><dt>fatty_acid: 28.4</dt>

<dd></dd><dt>CHC-CoA: 26.6</dt>

<dd></dd><dt>Benzoyl-CoA: 22.9</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 22.2</dt>

<dd></dd><dt>Acetyl-CoA: 15.1</dt>

<dd></dd><dt>inactive: 12.4</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>













<dd><strong>AMP-binding (3017..3428)</strong>: X
<div class="collapser collapser-target-input-path1-gene1719 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-input-path1-gene1719-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIDVFGNLAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Tyr <span class="serif">DFESFGNIAK</span> (59% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r7c8' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 8 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-432"></div> <span class="help-tooltip" id="region_svg-help-432">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region008.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c8-svg" data-filename="CP023324.1_r7c8.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,225,089 - 6,286,233 nt. (total: 61,145 nt)

<a class="cluster-rules-header" id="r7c8-rules-header" href="#r7c8">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c8-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r7c8-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-433"></div> <span class="help-tooltip" id="focus-panel-help-433">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c8">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c8-gene-table-details" data-name="r7c8-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c8-MIBiG-cluster-compare" data-name="r7c8-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c8-clusterblast" data-name="r7c8-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c8-knownclusterblast" data-name="r7c8-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c8-subclusterblast" data-name="r7c8-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c8-pfam-details" data-name="r7c8-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r7c8-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-345"></div> <span class="help-tooltip" id="gt-help-tooltip-help-345">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c8">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c8-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c8-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c8">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c8-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-341"></div> <span class="help-tooltip" id="MIBiG-help-341">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<div><p>No matches found.</p></div>

</div>
</div>



<div class="body-details-section r7c8-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-342"></div> <span class="help-tooltip" id="cb-general-help-342">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c8-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c8_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c8_1.svg">
NW_006271973 (266414-357915): Cordyceps militaris CM01 unplaced genomic scaff... (50% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c8_2.svg">
NW_024468919 (637636-749466): Cryphonectria parasitica EP155 unplaced genomic... (10% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c8_3.svg">
NC_072350 (525276-617173): Talaromyces marneffei chromosome 3, complete sequence (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c8_4.svg">
NW_022985510 (2150254-2227692): Neohortaea acidophila strain CBS 113389 unpla... (6% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r7c8_5.svg">
NW_022985463 (11775-102765): Mytilinidion resinicola strain CBS 304.34 unplac... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c8_6.svg">
NW_017253367 (1229921-1261277): Paraphaeosphaeria sporulosa strain AP3s5-JAC2... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r7c8_7.svg">
NW_024467832 (4333687-4396473): Aspergillus wentii DTO 134E9 unplaced genomic... (6% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r7c8-download">Download graphic</button>
</div>
<div id="clusterblast-r7c8-svg"></div>

</div>
</div>

<div class="body-details-section r7c8-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-343"></div> <span class="help-tooltip" id="cb-known-help-343">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r7c8-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-344"></div> <span class="help-tooltip" id="cb-sub-help-344">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c8-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-346"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-346">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c8-pfam-details-svg" data-filename="CP023324.1_r7c8_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c8-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r7c9' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> CP023324.1 - Region 9 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-434"></div> <span class="help-tooltip" id="region_svg-help-434">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = CP023324.1.region009.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r7c9-svg" data-filename="CP023324.1_r7c9.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,983,456 - 7,017,684 nt. (total: 34,229 nt)

<a class="cluster-rules-header" id="r7c9-rules-header" href="#r7c9">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r7c9-rules">NAPAA: (AMP-binding and TIGR02353)</div>

<div class="region-svg-container">
<div id='r7c9-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-435"></div> <span class="help-tooltip" id="focus-panel-help-435">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r7c9">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r7c9-gene-table-details" data-name="r7c9-gene-table-details">
Gene overview
</div>



<div class="body-details-header r7c9-nrps_pks" data-name="r7c9-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r7c9-MIBiG-cluster-compare" data-name="r7c9-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r7c9-clusterblast" data-name="r7c9-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r7c9-knownclusterblast" data-name="r7c9-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r7c9-subclusterblast" data-name="r7c9-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r7c9-pfam-details" data-name="r7c9-pfam-details">
Pfam domains
</div>

<div class="body-details-header r7c9-tigrfam-details" data-name="r7c9-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r7c9-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-353"></div> <span class="help-tooltip" id="gt-help-tooltip-help-353">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r7c9">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r7c9-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r7c9-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r7c9">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r7c9-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-347"></div> <span class="help-tooltip" id="nrps-domain-help-347">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c9-details-svg" data-filename="CP023324.1_r7c9_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r7c9-details-svg"></div>
</div>
</div>



<div class="body-details-section r7c9-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-348"></div> <span class="help-tooltip" id="MIBiG-help-348">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r7c9-selector">Analysis type:</label>
<select id="comparison-MIBiG-r7c9-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r7c9"></div>
<div id="comparison-MIBiG-r7c9-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NAPAA</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>zealexin B2, zealexin C2</td>


<td>Zea mays</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.52, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>ε-Poly-L-lysine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001679: 12-12481">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001679/index.html#r1c1" target="_new">BGC0001679</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:0.52, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>N-Acetyltryptophan</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.46, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r7c9-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Terpene</td>
<td>zealexin B2, zealexin C2</td>
<td>Zea mays</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.52, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>ε-Poly-L-lysine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001679: 12-12481">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001679/index.html#r1c1" target="_new">BGC0001679</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:0.52, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>N-Acetyltryptophan</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.46, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r7c9-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-349"></div> <span class="help-tooltip" id="cb-general-help-349">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r7c9-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r7c9_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r7c9_1.svg">
NW_006271975 (1702597-1751671): Cordyceps militaris CM01 unplaced genomic sca... (53% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r7c9_2.svg">
NW_011942149 (6466303-6515308): Metarhizium robertsii ARSEF 23 MAA Scf 1, who... (16% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r7c9_3.svg">
NC_054389 (12430017-12539558): Drechmeria coniospora strain ARSEF 6962 chromo... (7% of genes show similarity), NAPAA,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r7c9_4.svg">
NC_035791 (818908-980602): Pochonia chlamydosporia 170 chromosome 2, whole ge... (3% of genes show similarity), NAPAA,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c9_5.svg">
NC_018207 (430001-619419): Zymoseptoria tritici IPO323 chromosome 12, whole g... (3% of genes show similarity), NAPAA,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r7c9_6.svg">
NZ_CP039432 (2236443-2270412): Rhodococcus sp. SGAir0479 chromosome, complete... (11% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r7c9_7.svg">
NZ_CP031455 (1608048-1824050): Streptomyces olivoreticuli subsp. olivoreticul... (2% of genes show similarity), NAPAA,NRPS,T1PKS,terpene,transAT-PKS
</option>

<option value = "svg/clusterblast_r7c9_8.svg">
NZ_JAMHJQ010000003 (169212-203274): Streptomyces sp. 35G-GA-8 Scaffold3, whol... (6% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r7c9_9.svg">
NZ_CP118574 (3689879-3724055): Streptomyces sp. WZ-12 chromosome, complete ge... (6% of genes show similarity), NAPAA
</option>

<option value = "svg/clusterblast_r7c9_10.svg">
NZ_BMMN01000006 (211516-245575): Microbispora bryophytorum strain CGMCC 4.713... (7% of genes show similarity), NAPAA
</option>

</select>
<button id= "clusterblast-r7c9-download">Download graphic</button>
</div>
<div id="clusterblast-r7c9-svg"></div>

</div>
</div>

<div class="body-details-section r7c9-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-350"></div> <span class="help-tooltip" id="cb-known-help-350">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r7c9-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r7c9_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r7c9_1.svg">
ε-Poly-L-lysine
</option>

</select>
<button id="knownclusterblast-r7c9-download">Download graphic</button>
</div>
<div id="knownclusterblast-r7c9-svg"></div>

</div>
</div>

<div class="body-details-section r7c9-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-351"></div> <span class="help-tooltip" id="cb-sub-help-351">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r7c9-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-354"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-354">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c9-pfam-details-svg" data-filename="CP023324.1_r7c9_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r7c9-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r7c9-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-355"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-355">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r7c9-tigrfam-details-svg" data-filename="CP023324.1_r7c9_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r7c9-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r7c9-nrps_pks_monomers" data-name="r7c9-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r7c9-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-352"></div> <span class="help-tooltip" id="nrps-monomers-help-352">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">input.path1.gene1976</span></strong>: X
<div class="collapser collapser-target-input-path1-gene1976 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (159..419)</strong>: X
<div class="collapser collapser-target-input-path1-gene1976 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-input-path1-gene1976-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAESIGTVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DAEDIGTVVK</span> (68% 8Å match)</dd>
<dd>Gln or Tyr <span class="serif">DAESVGTVVK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/fungi_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://fungismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/fungi_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>